<h3 id=protein_count>Protein count</h3>
Number of protein entries associated with this proteome: UniProtKB entries for regular proteomes or UniParc entries for redundant proteomes ( [more...](http://www.uniprot.org/help/proteome%5Fredundancy) )

<h3 id=genome_assembly>Genome assembly ID</h3>
Identifier for the genome assembly ( [more...](https://www.ensembl.org/Help/Faq?id=216) )

<h3 id=proteome_components>Proteome components</h3>
Genomic components encoding the proteome

<h3 id=proteome_gene_count>Gene count</h3>
This is the total number of unique genes found in the proteome set, algorithmically computed. For each gene, a single representative protein sequence is chosen from the proteome. Where possible, reviewed (Swiss-Prot) protein sequences are chosen as the representatives.

<h3 id=pan_proteome>Pan proteome</h3>
A pan proteome is the full set of proteins thought to be expressed by a group of highly related organisms (e.g. multiple strains of the same bacterial species).

[More...](http://pir.georgetown.edu/rps/pp.shtml)

<h3 id=share_button>Share button</h3>
The 'Share' button allows you to share a custom URL including the columns in your result view.

[More...](http://www.uniprot.org/help/api%5Fqueries)

<h3 id=evidenceToolTip>Evidence Tooltip</h3>
An evidence describes the source of an annotation, e.g. an experiment that has been published in the scientific literature, an orthologous protein, a record from another database, etc.

[More...](http://www.uniprot.org/manual/evidences)

<h3 id=uniprot ECO303>uniprot ECO303</h3>
Manually curated information that is based on statements in scientific articles for which there is no experimental support.

[More...](http://www.uniprot.org/manual/evidences#ECO:0000303)

<h3 id=uniprot ECO305>uniprot ECO305</h3>
Manually curated information which has been inferred by a curator based on his/her scientific knowledge or on the scientific content of an article.

[More...](http://www.uniprot.org/manual/evidences#ECO:0000305)

<h3 id=uniprot ECO250>uniprot ECO250</h3>
Manually curated information which has been propagated from a related experimentally characterized protein.

[More...](http://www.uniprot.org/manual/evidences#ECO:0000250)

<h3 id=uniprot ECO255>uniprot ECO255</h3>
Manual validated information which has been generated by the UniProtKB automatic annotation system.

[More...](http://www.uniprot.org/manual/evidences#ECO:0000255)

<h3 id=uniprot ECO256>uniprot ECO256</h3>
Information which has been generated by the UniProtKB automatic annotation system, without manual validation.

[More...](http://www.uniprot.org/manual/evidences#ECO:0000256)

<h3 id=uniprot ECO259>uniprot ECO259</h3>
Information which has been generated by the UniProtKB automatic annotation system, without manual validation.

[More...](http://www.uniprot.org/manual/evidences#ECO:0000259)

<h3 id=uniprot ECO269>uniprot ECO269</h3>
Manually curated information for which there is published experimental evidence.

[More...](http://www.uniprot.org/manual/evidences#ECO:0000269)

<h3 id=uniprot ECO312>uniprot ECO312</h3>
Manually validated information which has been imported from another database.

[More...](http://www.uniprot.org/manual/evidences#ECO:0000312)

<h3 id=uniprot ECO313>uniprot ECO313</h3>
Information which has been imported from another database using automatic procedures.

[More...](http://www.uniprot.org/manual/evidences#ECO:0000313)

<h3 id=uniprot ECO7744>uniprot ECO7744</h3>
Manually validated information inferred from a combination of experimental and computational evidence.

[More...](http://www.uniprot.org/manual/evidences#ECO:0007744)

<h3 id=uniprot ECO7829>uniprot ECO7829</h3>
Information inferred from a combination of experimental and computational evidence, without manual validation.

[More...](http://www.uniprot.org/manual/evidences#ECO:0000213)

<h3 id=blast_annotation>blast_annotation</h3>
Control which [Sequence annotation](http://www.uniprot.org/manual/sequence%5Fannotation) is colored on the local alignment. This helps you see if essential amino acids are conserved. This is only available if there is known sequence annotation for the query or match sequence in the range convered by the local alignment.

<h3 id=MolAnnNote>Molecule-specific annotation note</h3>
Annotation specific to this molecule.

<h3 id=blast_aminoacid_properties>BLAST amino acid properties</h3>
Color the amino acids in the local alignment on the basis of their physical and chemical properties.

<h3 id=blast_selected_annotation>BLAST selected annotation</h3>
This shows annotation of the match sequence. The shown columns are customizable using [customize display](http://www.uniprot.org/help/customize) .

<h3 id=blast_graphical>BLAST graphical</h3>
A graphical overview of the position of the alignment on the query sequence. Green means a high identity alignment, colors trending to red mean an alignment with a lower identity.

<h3 id=blast_position>BLAST Position</h3>
The position of the alignment in relation to the query and match sequence. If these positions differ greatly one should take care before drawing biological conclusions.

<h3 id=blast_match_length>BLAST match length</h3>
The full length of the hit sequence, that was found in the database, in aminoacids. That was submitted to BLAST. If this number is significantly larger than the length of the alignment one should take care before transferring annotation.

<h3 id=blast_query_length>BLAST query length</h3>
The full length of the query sequence in aminoacids. That was submitted to BLAST. If this number is significantly larger than the length of the alignment one should take care in transferring annotation.

<h3 id=blast_positives>BLAST Positives</h3>
The percentage of aminoacids that at their position in the alignment lead to a positive score in the scoring matrix. The higher this number the better.

<h3 id=blast_identity>BLAST Identity</h3>
The identity is the percentage of amino acids that are identical on their position between the query and match sequence. The higher this number the better.

<h3 id=blast_e-value>BLAST E-value</h3>
The e-value is the number of expected alignments between your query sequence and the database of the same quality as this one. In general the lower the e-value the better. The e-value indicates statistical significance not biological. If two sequences are aligned in a BLAST search against a large database the alignment will have a higher reported e-value then when the same two sequence are found in small database.

<h3 id=blast_score>BLAST score</h3>
The raw score reported by BLAST. This score depends on the scoring matrix that was used. In general longer alignments tend to have a higher score. A high score in combination with a high identity percentage tends to indicate a higher quality alignment.

<h3 id=blast_select_query_sequence>BLAST query select box</h3>
The 'select query sequence' checkbox allows you to select the sequence you submitted for further analysis. Example: you blasted a sequence and want to do a multiple alignment with the sequence you submitted and five other interesting hits.

-   Use the 'select query sequence' checkbox to select your query sequence. The query sequence identifier appears in the green bar at the bottom of the page.
-   Click on the checkboxes next to the five other hits of interest.
-   Click on the 'align' button in the green bar (bottom right). You get a multiple sequence alignment including your query sequence and the five interesting BLAST hits you selected.

<h3 id=blast_query>BLAST query</h3>
This shows you the query sequence that was used by BLAST, in fasta format, whatever you submitted a sequence or a UniProt identifier.

<h3 id=blast_program>BLAST program</h3>
This gives you information on the BLAST program used (including the version and release date).

-   BlastP: search the protein database using a protein query (no translation)
-   BlastX: search the protein database using a translated nucleotide query. The submitted nucleotide sequence is translated in all 6 reading frames, before being compared to the protein database.

<h3 id=blast_post_filter>Post BLAST filter</h3>
The BLAST filter allows you to your BLAST results to specific subsets, i.e. by filtering by database section or taxonomy. When doing a BLAST against UniRef, you can also show only the seed sequence annotation. The filter can be removed by clicking on the reset button.

<h3 id=seed-sequence>Seed sequence</h3>
The seed sequence is the longest sequence in a cluster.

The UniRef databases are generated in a hierarchical fashion where:

-   UniRef100 clusters are generated first using sequences from UniProtKB and UniParc databases
-   UniRef90 clusters are generated using UniRef100 seed sequences (longest sequence in a cluster)
-   UniRef50 clusters are generated using UniRef90 seed sequences

<h3 id=representative-sequence>Representative sequence</h3>
The representative sequence is the top sequence in a cluster.

All the proteins in each cluster are ranked to facilitate the selection of a biologically relevant representative for the cluster. The representative sequence (ranked first) is selected based on the following criteria:

1.  quality of annotation: order of preference is a member from UniProtKB/Swiss-Prot then UniProtKB/TrEMBL and last is UniParc
2.  meaningful name: members with protein names that do not contain words such as hypothetical, probable etc. are preferred
3.  organism: entries from model organisms are preferred
4.  sequence length: longest sequence is preferred

<h3 id=checksum>Sequence checksum</h3>
The checksum is a form of redundancy check that is calculated from the sequence. It is useful for tracking sequence updates.

It should be noted that while, in theory, two different sequences could have the same checksum value, the likelihood that this would happen is extremely low.

However UniProtKB may contain entries with identical sequences in case of multiple genes (paralogs).

The checksum is computed as the sequence 64-bit Cyclic Redundancy Check value (CRC64) using the generator polynomial: x <sup>64</sup> + x <sup>4</sup> + x <sup>3</sup> + x + 1. The algorithm is described in the ISO 3309 standard.

Press W.H., Flannery B.P., Teukolsky S.A. and Vetterling W.T.  
**Cyclic redundancy and other checksums**  
[Numerical recipes in C 2nd ed., pp896-902, Cambridge University Press (1993)](http://www.nrbook.com/b/bookcpdf.php) )

<h3 id=Web resources>Web resources</h3>
CC WEB RESOURCE

Description of a cross-reference to a network database or resource for a specific protein

<h3 id=FT CHAIN>Chain</h3>
The extent of a polypeptide chain in the mature protein.

In reviewed entries, it is present:

1.  For proteins that are not processed (those for which the mature protein sequence corresponds to the translated cDNA sequence) the CHAIN covers the whole protein sequence.
2.  For processed proteins, it describes the mature part of the protein once preprotein parts such as propeptides and signal sequences have been removed

For TrEMBL, it is present only in those entries where the submitters to the nucleotide sequence databases have provided this information for processed proteins.

<h3 id=FT DOMAIN>Domain</h3>
Extent of a domain, which is defined as a specific combination of secondary structures organized into a characteristic three-dimensional structure or fold.

<h3 id=GO ic>IC: Inferred by Curator</h3>
Inferred by Curator

Used for cases where an annotation is not supported by any evidence but can be reasonably inferred by a curator from other GO annotations for which evidence  
is available.

More information in the [GO evidence code guide](http://geneontology.org/page/guide-go-evidence-codes#ic)

<h3 id=GO iba>IBA: Inferred from Biological aspect of Ancestor</h3>
Inferred from Biological aspect of Ancestor

A type of phylogenetic evidence whereby an aspect of a descendent is inferred through the characterization of an aspect of a ancestral gene.

More information in the [GO evidence code guide](http://geneontology.org/page/guide-go-evidence-codes#iba)

<h3 id=GO ibd>IBD: Inferred from Biological aspect of Descendent</h3>
Inferred from Biological aspect of Descendant

A type of phylogenetic evidence whereby an aspect of an ancestral gene is inferred through the characterization of an aspect of a descendant gene.

More information in the [GO evidence code guide](http://geneontology.org/page/guide-go-evidence-codes#ibd)

<h3 id=GO ida>IDA: Inferred from Direct Assay</h3>
Inferred from Direct Assay

Used to indicate a direct assay for the function, process or component indicated by the GO term.

More information in the [GO evidence code guide](http://geneontology.org/page/guide-go-evidence-codes#ida)

<h3 id=GO iea>IEA: Inferred from Electronic Annotation</h3>
Inferred from Electronic Annotation

Used for annotations that depend directly on computation or automated transfer of annotations from a database. These annotations have not been manually checked to verify their accuracy.

More information in the [GO evidence code guide](http://geneontology.org/page/guide-go-evidence-codes#iea)

<h3 id=GO iep>IEP: Inferred from Expression Pattern</h3>
Inferred from Expression Pattern

Covers cases where the annotation is inferred from the timing or location of expression of a gene.

More information in the [GO evidence code guide](http://geneontology.org/page/guide-go-evidence-codes#iep)

<h3 id=GO igc>IGC: Inferred from Genomic Context</h3>
Inferred from Genomic Context

Used when information about the genomic context of a gene product forms part of the evidence for a particular annotation. Genomic context includes, but is not limited to, such things as identity of the genes neighboring the gene product in question (i.e. synteny), operon structure, and phylogenetic or other whole genome analysis.

More information in the [GO evidence code guide](http://geneontology.org/page/guide-go-evidence-codes#igc)

<h3 id=GO igi>IGI: Inferred from Genetic Interaction</h3>
Inferred from Genetic Interaction

Used to describe "traditional" genetic interactions such as suppressors and synthetic lethals as well as other techniques such as functional complementation, rescue experiments, or inferences about a gene drawn from the phenotype of a mutation in a different gene.

More information in the [GO evidence code guide](http://geneontology.org/page/guide-go-evidence-codes#igi)

<h3 id=GO ikr>IKR: Inferred from Key Residues</h3>
Inferred from Key Residues

A type of phylogenetic evidence characterized by the loss of key sequence residues.

More information in the [GO evidence code guide](http://geneontology.org/page/guide-go-evidence-codes#ikr)

<h3 id=GO imp>IMP: Inferred from Mutant Phenotype</h3>
Inferred from Mutant Phenotype

Describes annotations that are concluded from looking at variations or changes in a gene product such as mutations or abnormal levels and includes techniques such as knockouts, overexpression, anti-sense experiments and use of specific protein inhibitors.

More information in the [GO evidence code guide](http://geneontology.org/page/guide-go-evidence-codes#imp)

<h3 id=GO ipi>IPI: Inferred from Physical Interaction</h3>
Inferred from Physical Interaction

Covers physical interactions between the gene product of interest and another molecule (or ion, or complex).

More information in the [GO evidence code guide](http://geneontology.org/page/guide-go-evidence-codes#ipi)

<h3 id=GO ird>IRD: Inferred from Rapid Divergence</h3>
Inferred from Rapid Divergence

A type of phylogenetic evidence characterized by rapid divergence from ancestral sequence.

More information in the [GO evidence code guide](http://geneontology.org/page/guide-go-evidence-codes#ird)

<h3 id=GO iss>ISS: Inferred from Sequence or Structural Similarity</h3>
Inferred from Sequence or Structural Similarity  
Used for any analysis based on sequence alignment, structure comparison, or evaluation of sequence features such as composition.

More information in the [GO evidence code guide](http://geneontology.org/page/guide-go-evidence-codes#iss)

<h3 id=GO nas>NAS: Non-traceable Author Statement</h3>
Non-traceable Author Statement

Used for statements in the abstract, introduction or discussion of a paper that cannot be traced back to another publication.

More information in the [GO evidence code guide](http://geneontology.org/page/guide-go-evidence-codes#nas)

<h3 id=GO nd>ND: No biological Data available</h3>
No biological Data available

Used for annotations to the three GO root terms "molecular\_function", "biological\_process", or "cellular\_component". This reference documents the fact that a database curator has reviewed the literature describing a gene product but found no biologically useful information relating to the ontology in question (molecular function, biological process, or cellular component) as of the date  
indicated.

More information in the [GO evidence code guide](http://geneontology.org/page/guide-go-evidence-codes#nd)

<h3 id=GO rca>RCA: inferred from Reviewed Computational Analysis</h3>
Inferred from Reviewed Computational Analysis

Used for annotations based on a non-sequence-based computational method where the results have been reviewed by an author or a curator.

More information in the [GO evidence code guide](http://geneontology.org/page/guide-go-evidence-codes#rca)

<h3 id=GO tas>TAS: Traceable Author Statement</h3>
Traceable Author Statement

Used for information from review articles where the original experiments are traceable through that article and also for information from text books or dictionaries.

More information in the [GO evidence code guide](http://geneontology.org/page/guide-go-evidence-codes#tas)

