
---
title: Alternative products
categories: Sequence,manual
---

This subsection of the 'Sequence' section lists the alternative protein sequences (isoforms) that can be generated from the same gene by a single or by the combination of up to four biological events (alternative promoter usage, alternative splicing, alternative initiation and ribosomal frameshifting). Additionally, this section gives relevant information on each alternative protein isoform. This section is only present in reviewed entries, i.e. in UniProtKB/Swiss-Prot.

### 1\. General information

We give the number of alternative protein sequences described in the entry and the biological event(s) which lead to the production of these isoforms (alternative promoter usage, alternative splicing, alternative initiation and/or ribosomal frameshifting). Additional information is provided, if available.  
  
Example: [P04150](http://www.uniprot.org/uniprot/P04150#sequences)

### 2\. Isoform-specific information

For each isoform, we provide the following information:

#### Identifier

Each isoform is characterized by a unique identifier, which is composed of the primary accession number of the entry, followed by a dash and a number.  
  
Example: [P04150-2](http://www.uniprot.org/uniprot/P04150#sequences)

When alternative protein sequences differ significantly, we create separate entries and list all isoforms in each of them. Consequently, isoforms produced from a single gene listed in one entry may have identifiers derived from different primary accession numbers.  
  
Examples: [P42166](http://www.uniprot.org/uniprot/P42166#sequences), [P42167](http://www.uniprot.org/uniprot/P42167#sequences)

#### Name and synomym(s)

We provide the name and synomym(s) of the isoform, which are used in the literature or assigned by Swiss-Prot annotators. By default, the isoforms are numbered.

#### Description of the sequence differences

We clearly describe the alternative sequences with respect to the [canonical sequence](http://www.uniprot.org/help/canonical%5Fand%5Fisoforms) displayed by default in the entry.  
  
_Note_: The so-called 'canonical' sequence is the most prevalent sequence and/or the most similar sequence among orthologous species. All [positional information](http://www.uniprot.org/help/sequence%5Fannotation) in the entry refers to it.

_Note_: If an isoform has been reported to exist in the literature, but we are not able to determine its sequence, we add the tag 'Sequence is not available'.  
  
Example: [P21836](http://www.uniprot.org/uniprot/P21836#sequences) (Isoform H)

#### Additional comments

Some information in free text can be provided to complement the above items, such as:

**Description of the biological event(s) producing the isoform (optional)**

Example: [P04150](http://www.uniprot.org/uniprot/P04150#sequences)

**Information about the evidence for the existence of the isoform**

We include alternative protein sequences produced by alternative splicing only if there is experimental evidence that they exist (at least one publication) or if there is at least one mRNA with correct intron/exon boundaries. The tag 'No experimental confirmation available' is added in the 'Note=' topic when the protein sequence derives from the translation of a single mRNA. This tag is thus often used for isoforms that have been identified by large scale cDNA sequencing projects. We consider the isoform to be 'confirmed' if it has been isolated at least twice (from different cDNA libraries) or when the same isoform is also found in a closely related species.  
  
Example: [Q62141](http://www.uniprot.org/uniprot/Q62141#sequences)

Note that alternative sequences produced by alternative promoter usage, alternative initiation and ribosomal frameshifting are only annotated if their existence has been experimentally proven and documented in at least in one publication.

**Description of conflict(s), variant(s) and/or posttranslational modification(s)**

In most cases, this type of annotations are described in the specific subsection, positions being provided with respect to the canonical sequence. However, some variations may be isoform-specific, i.e. the amino acids involved are not present in the canonical sequence. These variations are then described in the 'Alternative products' section and the position(s) refer to the sequence of the actual isoform.  
  
Example: [P08631](http://www.uniprot.org/uniprot/P08631#sequences)

Other information can be found in various sections such as ['Function'](http://www.uniprot.org/manual/function), ['Expression'](http://www.uniprot.org/manual/tissue%5Fspecificity), etc.

For the description of the isoforms at the amino acid level, see the ['Alternative sequence'](http://www.uniprot.org/manual/var%5Fseq) subsection.

Isoform sequences can be **downloaded in FASTA format** from our [FTP download index page](http://www.uniprot.org/downloads) (choose the file: 'Isoform sequences').

Related keywords: [Alternative splicing](http://www.uniprot.org/keywords/25), [Alternative promoter usage](http://www.uniprot.org/keywords/877), [Alternative initiation](http://www.uniprot.org/keywords/24), [Ribosomal frameshifting](http://www.uniprot.org/keywords/688).

#### Related documents

[What is the canonical sequence? Are all isoforms described in one entry? How can I retrieve them?](http://www.uniprot.org/faq/30)

[How are protein sequence variety and protein diversity represented in UniProtKB?](http://www.uniprot.org/faq/21)

[How do I get the nucleotide sequence that corresponds to the canonical UniProtKB sequence?](http://www.uniprot.org/faq/35)

[Cross-references for isoform sequences](http://www.uniprot.org/help/isoform%5Fcrossreferences)

[How to retrieve a complete set of protein sequences?](http://www.uniprot.org/faq/38)

[Automatic gene-centric isoform mapping for eukaryotic reference proteome entries](http://www.uniprot.org/help/gene%5Fcentric%5Fisoform%5Fmapping)

[What are UniProtKB's criteria for defining a CDS as a protein?](http://www.uniprot.org/faq/25)

Related terms: splice variant, splice isoform
        