---
title: How is orthology established in UniProtKB/Swiss-Prot?
categories: UniProtKB,Family_and_domains,Cross-references,faq
---

**How is orthology established in UniProtKB/Swiss-Prot?**

Information is combined from a variety of complementary sources to establish orthology relationships. Potential orthologs are initially identified using sequence similarity search programs such as BLAST. Orthology relationships are then verified manually using a combination of resources including scientific literature, sequence analysis tools, phylogenetic and comparative genomics databases such as Ensembl Compara, and other specialised databases such as species-specific collections.

See also:

- [What is HAMAP?](http://www.uniprot.org/faq/16)
- [How is protein family membership assigned in UniProtKB?](http://www.uniprot.org/faq/41)
- [Is there a download file available where all UniProt IDs from X.laevis are matched to their human equivalents (homologs)? How can I obtain an ortholog mapping of human proteins to S.pombe proteins?](http://www.uniprot.org/help/orthologs%5Fbetween%5Ftwo%5Fspecies)
- [Sequence similarities](http://www.uniprot.org/manual/sequence%5Fsimilarities)