<div>
 <p>
  <strong>
   What are the criteria for defining a CDS as a real protein, i.e. for inclusion in UniProtKB?
  </strong>
 </p>
 <p>
  Most protein sequences are derived from translations of CoDing Sequence (CDS) derived from gene predictions. A CoDing Sequence (CDS) is a region of DNA or RNA whose sequence determines the sequence of amino acids in a protein. It should not be mixed up with an Open Reading Frame (ORF), which is a continuous stretch of DNA codons that begins with a start codon and ends at a STOP codon. All CDS are ORFs, but not all ORFs are CDS...
 </p>
 <p>
  Some of the predicted CDSs exhibit strong sequence similarity to known proteins in closely related species. For other proteins there is experimental evidence, such as Edman sequencing, clear identification by mass spectrometry (MSI), X-ray or NMR structure, detection by antibodies, etc. However, for some other proteins, there is no evidence at all. To indicate these different levels of evidence for the existence of a protein, we have introduced the PE (Protein Existence) line (see
  <a href="http://www.uniprot.org/docs/pe%5Fcriteria">
   the protein existence criteria
  </a>
  ).
 </p>
 <p>
  Note that the PE line does not describe the accuracy or correctness of a sequence displayed in UniProtKB, but the evidence for the existence of a protein. It may happen that the protein sequence is not entirely accurate, especially for sequences derived from gene predictions from genomic sequences.
 </p>
 <p>
  <strong>
   What are UniProtKB's criteria for defining a CDS as 'not a real protein'?
  </strong>
 </p>
 <p>
  Gene prediction performance largely depends on current biological knowledge. We use bioinformatics tools to align the proposed CDS with the latest version of nucleic acid sequences (genomic and RNA/ESTs). We sometimes suggest that proposed CDS or ORFs are incorrectly predicted protein sequences. Our evidence can include the presence of new longer or shorter RNAs (fused or split predicted gene(s)), absence of RNA (even in other species), and/or wrong intron/exon boundaries (in Eukaryota). Some other protein sequences may have been identified as pseudogenes in the literature. When there is enough evidence that these CDSs are not real proteins, we remove them from UniProtKB.
 </p>
 <p>
  See also:
 </p>
 <ul>
  <li>
   <a href="http://www.uniprot.org/help/sequence%5Forigin">
    Where do the UniProtKB protein sequences come from?
   </a>
  </li>
  <li>
   <a href="http://www.uniprot.org/help/uniprotkb%5Fcoverage">
    Does UniProtKB contain all protein sequences?
   </a>
  </li>
  <li>
   <a href="http://www.uniprot.org/help/dubious%5Fsequences">
    Why do we keep dubious sequences in UniProtKB? How to discard them from a protein set?
   </a>
  </li>
  <li>
   <a href="http://www.uniprot.org/help/deleted%5Faccessions">
    Why have some UniProtKB accession numbers been deleted? How can I track them?
   </a>
  </li>
  <li>
   <a href="http://www.uniprot.org/manual/sequences">
    Sequences
   </a>
  </li>
  <li>
   <a href="http://www.uniprot.org/manual/var%5Fseq">
    Alternative sequence
   </a>
  </li>
 </ul>
</div>