<div>
 <p>
  The Fungal protein annotation project focuses on the manual annotation of fungi-specific proteins and protein families.
 </p>
 <p>
  See:
  <a href="http://www.uniprot.org/faq/45">
   How do we manually annotate a UniProtKB entry?
  </a>
 </p>
 <h3 id="The_fungal_kingdom">
  The fungal kingdom
 </h3>
 <p>
  Species of the fungal kingdom occupy almost every ecological niche, and include important human, animal and plant pathogens. They are of tremendous industrial importance. Not only are they employed in beer, wine, bread and cheese manufacturing, but they are also natural producers of antibiotics (i.e. penicillin), and can be engineered to make biotechnologically relevant enzymes like hemicellulases, vitamins, and other industrially important secondary metabolites. In addition, fungi share many important biological processes with higher eukaryotes, and their experimental tractability make certain species of fungi among the most useful model systems in cell biology.
 </p>
 <h3 id="Priorities">
  Priorities
 </h3>
 <p>
  Our major manual curation efforts are directed towards the annotation of proteins from the well-studied model yeasts
  <em>
   <a href="http://www.uniprot.org/taxonomy/4932">
    Saccharomyces cerevisiae
   </a>
  </em>
  and
  <em>
   <a href="http://www.uniprot.org/taxonomy/4896">
    Schizosaccharomyces pombe
   </a>
  </em>
  as well as the annotation of representative members of fungal-specific protein families from other model fungi like
  <em>
   <a href="http://www.uniprot.org/taxonomy/162425">
    Aspergillus nidulans
   </a>
  </em>
  and
  <em>
   <a href="http://www.uniprot.org/taxonomy/5141">
    Neurospora crassa
   </a>
  </em>
  , or the human pathogens
  <em>
   <a href="http://www.uniprot.org/taxonomy/5476">
    Candida albicans
   </a>
  </em>
  and
  <em>
   <a href="http://www.uniprot.org/taxonomy/5207">
    Cryptococcus neoformans
   </a>
  </em>
  . This is done with special emphasis on proteins of particular interest within these individual species and also on proteins belonging to conserved fungal families, but which have no discernable members in the two model yeasts
  <em>
   S. cerevisiae
  </em>
  and
  <em>
   S. pombe
  </em>
  . This allows an even broader coverage of biochemically characterized fungal proteins in our knowledgebase.
 </p>
 <ul>
  <li>
   All manually reviewed
   <em>
    Saccharomyces cerevisiae
   </em>
   entries can be found
   <a href="http://www.uniprot.org/uniprot/?query=organism:4932+AND+reviewed:yes">
    here
   </a>
   <a href="http://www.uniprot.org/biocuration%5Fproject/fungi/statistics/#Saccharomycescerevisiae">
    (statistics)
   </a>
  </li>
  <li>
   All manually reviewed
   <em>
    Schizosaccharomyces pombe
   </em>
   entries can be found
   <a href="http://www.uniprot.org/uniprot/?query=organism:4896+AND+reviewed:yes">
    here
   </a>
   <a href="http://www.uniprot.org/biocuration%5Fproject/fungi/statistics/#Schizosaccharomycespombe">
    (statistics)
   </a>
  </li>
  <li>
   All manually reviewed
   <em>
    Aspergillus nidulans
   </em>
   entries can be found
   <a href="http://www.uniprot.org/uniprot/?query=organism:162425+AND+reviewed:yes">
    here
   </a>
  </li>
  <li>
   All manually reviewed
   <em>
    Neurospora crassa
   </em>
   entries can be found
   <a href="http://www.uniprot.org/uniprot/?query=organism:5141+AND+reviewed:yes">
    here
   </a>
  </li>
  <li>
   All manually reviewed
   <em>
    Candida albicans
   </em>
   entries can be found
   <a href="http://www.uniprot.org/uniprot/?query=organism:5476+AND+reviewed:yes">
    here
   </a>
  </li>
  <li>
   All manually reviewed
   <em>
    Cryptococcus neoformans
   </em>
   entries can be found
   <a href="http://www.uniprot.org/uniprot/?query=organism:5207+AND+reviewed:yes">
    here
   </a>
  </li>
 </ul>
 <h3 id="Annotation_propagation">
  Annotation propagation
 </h3>
 <p>
  With the availability of increasing numbers of sequenced fungal genomes, the scope of this annotation project was extended to cover the identification and annotation of the orthologs of characterized proteins in all fungal species. Orthologs are identified by similarity searches and phylogenetic analysis, and annotation from characterized proteins is transferred to other family members using strict rules, to avoid propagation of inaccurate information and preserve the high standard of UniProtKB/Swiss-Prot manual annotation.
 </p>
 <h2 id="About_yeasts">
  About yeasts
 </h2>
 <h3 id="Budding_yeast">
  Budding yeast
 </h3>
 <p>
  <em>
   S. cerevisiae
  </em>
  has always been at the forefront of biochemical research. Its ease of cultivation coupled with its genetic tractability made it an organism of choice in the early days of research, and it has retained its pre-eminence in the post-genomics area: many innovative and large-scale investigative techniques have been pioneered in
  <em>
   S. cerevisiae
  </em>
  in recent years. Therefore yeast remains one of the (if not THE) best-characterized eukaryotic organisms, and the functions of orthologous proteins in other organisms are often inferred at least in part based on studies of their yeast counterpart. For this reason, the Fungal Protein Annotation project has prioritized
  <em>
   S. cerevisiae
  </em>
  annotation and pursued the goal of completely annotating all yeast ORFs thought to code for a protein, resulting in the
  <a href="http://www.uniprot.org/uniprot/?query=taxonomy:4932%20keyword:1185">
   <em>
    S. cerevisiae
   </em>
   proteome
  </a>
  <br/>
  integration into UniProtKB/Swiss-Prot in
  <a href="http://www.uniprot.org/news/2007/01/09/release">
   early 2007
  </a>
  .
 </p>
 <h3 id="Fission_yeast">
  Fission yeast
 </h3>
 <p>
  Another yeast that was recognized early on as a valuable model organism for the study of conserved pathways and cell mechanisms is
  <em>
   Schizosaccharomyces pombe
  </em>
  , also commonly known as fission yeast.
  <em>
   S. pombe
  </em>
  shares many features with cells of more complicated eukaryotes and has long been used as a model for research into cell cycle progression and cell division.
  <em>
   S. pombe
  </em>
  grows by length extension and divides by medial fission, properties that allow for facile determination of cell cycle stage. Gene sequence comparisons and phylogenetic analyses have suggested that fission yeast diverged from budding yeast around 330-420 million years ago, and from Metazoa and plants around 1,000-1,200 million years ago, with some gene sequences as divergent between the two yeasts as they are from their human homologs. The wealth of biochemical information for
  <em>
   S. pombe
  </em>
  proteins, its interesting phylogenetic position and, not at least, the fact that it was the second sequenced genome of a fungus in 2002 prompted us to direct special annotation efforts to this species, which resulted in the
  <a href="http://www.uniprot.org/uniprot/?query=taxonomy:4896%20keyword:1185">
   <em>
    S. pombe
   </em>
   proteome
  </a>
  integration into UniProtKB/Swiss-Prot
  <a href="http://www.uniprot.org/news/2009/05/05/release">
   in 2009
  </a>
  .
 </p>
</div>