<div>
 <p>
  Each of the 3 UniProt databases - UniProtKB (Swiss-Prot and TrEMBL), UniParc and UniRef - is 'non-redundant'. However, the definition of 'redundancy' varies among the 3.
 </p>
 <p>
  <strong>
   Summary:
  </strong>
 </p>
 <p>
  Non-redundancy means in:
 </p>
 <ul>
  <li>
   UniProtKB/TrEMBL: one record for 100% identical full-length sequences in one species;
  </li>
  <li>
   UniProtKB/Swiss-Prot: one record per gene in one species;
  </li>
  <li>
   UniParc: one record for 100% identical sequences over the entire length, regardless of the species;
  </li>
  <li>
   UniRef100: one record for 100% identical sequences, including fragments, regardless of the species.
  </li>
 </ul>
 <p>
  <strong>
   UniProtKB
  </strong>
 </p>
 <p>
  - UniProtKB/TrEMBL is 'non-redundant' in the sense that all identical, full-length protein sequences, provided they come from the same species, are represented in a single record. Fragments, isoforms, variants and so on, encoded by the same gene, are stored in separate entries.
 </p>
 <p>
  - UniProtKB/Swiss-Prot is 'non-redundant' in the sense that all protein products encoded by one gene in a given species are represented in a single record. This includes alternative splicing isoforms, fragments, genetic variations, sequence conflicts, etc. Differences between sequence reports are analyzed, fully documented and reported in the entry. Cross-references to the original submissions to EMBL-Bank/GenBank/DDBJ databases are kept (see for instance,
  <a href="http://www.uniprot.org/uniprot/Q9BXB7#cross-references">
   Q9BXB7
  </a>
  ).
 </p>
 <p>
  See also:
 </p>
 <ul>
  <li>
   <a href="http://www.uniprot.org/help/protein%5Fdiversity">
    How are protein sequence variety and protein diversity represented in UniProtKB?
   </a>
  </li>
  <li>
   <a href="http://www.uniprot.org/help/canonical%5Fand%5Fisoforms">
    What is the canonical sequence? Are all isoforms described in one entry?
   </a>
  </li>
  <li>
   <a href="http://www.uniprot.org/help/proteome%5Fredundancy">
    Reducing proteome redundancy
   </a>
  </li>
 </ul>
 <p>
  Remarks:
 </p>
 <ul>
  <li>
   When different genes in the same species give rise to the same protein sequence, they were merged in a single UniProtKB/Swiss-Prot record and the gene names listed in the gene name subsection. See for example the entry for human histone H3.1 (
   <a href="http://www.uniprot.org/uniprot/P68431#names%5Fand%5Ftaxonomy">
    P68431
   </a>
   ). However, we tend to demerge many of these entries, and for newly annotated proteins, generate separate sequence entries in case of multiple genes coding for identical protein sequences, e.g.
   <a href="http://www.uniprot.org/uniprot/?query=replaces:P08409">
    P08409
   </a>
   .
  </li>
  <li>
   'Non-redundancy' in UniProtKB/Swiss-Prot implies that identical sequences are represented in a single record. However, if identical protein sequences are produced in different species, they are stored in different records (e.g.
   <a href="http://www.uniprot.org/uniprot/?query=(cluster%3A(member%3AP62975+identity%3A1.0)+length%3A76)+AND+reviewed%3Ayes">
    ubiquitin
   </a>
   ).
  </li>
  <li>
   Manual annotation in UniProtKB/Swiss-Prot consists of merging all UniProtKB/TrEMBL entries corresponding to protein products encoded by the same gene and documenting all differences. Once the manually annotated UniProtKB/TrEMBL entries are integrated into UniProtKB/Swiss-Prot, they are automatically removed from UniProtKB/TrEMBL. In this sense, Swiss-Prot and TrEMBL are not redundant with one another. However, merging of new UniProtKB/TrEMBL to existing UniProtKB/Swiss-Prot entries is performed manually - at least partially - in order to ensure that all information contained in the UniProtKB/TrEMBL entry is correctly integrated into UniProtKB/Swiss-Prot. For example, additional publications present in the new UniProtKB/TrEMBL entry have to be critically reviewed and the UniProtKB/Swiss-Prot entry updated accordingly. Given this manual curation effort and the increasing amount of protein sequence data from high-throughput cDNA or genome sequencing projects, it is not possible to ensure non-redundancy at this level. Therefore there can be 100% identical sequences for the same species in UniProtKB/Swiss-Prot and UniProtKB/TrEMBL.
  </li>
 </ul>
 <p>
  <strong>
   UniParc
  </strong>
 </p>
 <p>
  <a href="http://www.uniprot.org/help/uniparc">
   UniParc
  </a>
  is 'non-redundant' in the sense that all identical protein sequences are stored in a single record regardless of the species. Each record is characterized by a unique identifier, UPI. For example, identical ubiquitin sequences from various organisms can be found in UniParc record
  <a href="http://www.uniprot.org/uniparc/UPI00000006C4">
   UPI00000006C4
  </a>
  .
 </p>
 <p>
  <strong>
   UniRef
  </strong>
 </p>
 <p>
  Three
  <a href="http://www.uniprot.org/help/uniref">
   UniRef
  </a>
  databases - UniRef100, UniRef90 and UniRef50 - merge sequences automatically across species.
 </p>
 <p>
  UniRef100 is 'non-redundant' in the sense that identical sequences and subfragments are presented as a single entry. For example, see the UniRef100 cluster for
  <a href="http://www.uniprot.org/uniref/?query=member%3aP62975+identity:1.0">
   ubiquitin
  </a>
  , which contains 100% identical sequences from many different organisms.
 </p>
</div>