<div>
 <p>
  This section displays by default the canonical protein sequence and upon request all isoforms described in the entry. It also includes information pertinent to the sequence(s), including
  <a href="http://www.uniprot.org/help/sequence%5Flength">
   length
  </a>
  and
  <a href="http://www.uniprot.org/help/sequences">
   molecular weight
  </a>
  . The information is filed in different subsections. The current subsections and their content are listed below:
 </p>
 <div class="grid">
  <table>
   <tr>
    <td>
     <strong>
      Subsection
     </strong>
    </td>
    <td>
     <strong>
      Content
     </strong>
    </td>
   </tr>
   <tr>
    <td>
     <a href="http://www.uniprot.org/manual/sequence%5Fstatus">
      Sequence status
     </a>
    </td>
    <td>
     Indicates if the protein sequence is complete or not
    </td>
   </tr>
   <tr>
    <td>
     <a href="http://www.uniprot.org/manual/sequence%5Fprocessing">
      Sequence processing
     </a>
    </td>
    <td>
     Indicates if the mature form of a protein is derived by processing of a precursor
    </td>
   </tr>
   <tr>
    <td>
     <a href="http://www.uniprot.org/manual/sequences">
      Sequence
     </a>
    </td>
    <td>
     Canonical protein sequence data and list of described protein isoforms
    </td>
   </tr>
   <tr>
    <td>
     <a href="http://www.uniprot.org/manual/alternative%5Fproducts">
      Alternative products
     </a>
    </td>
    <td>
     Description of the different proteins generated from the same gene
    </td>
   </tr>
   <tr>
    <td>
     <a href="http://www.uniprot.org/help/gene%5Fcentric%5Fisoform%5Fmapping">
      Computationally mapped potential isoform sequences
     </a>
    </td>
    <td>
     Automatic gene-centric isoform mapping for eukaryotic reference proteome sequences
    </td>
   </tr>
   <tr>
    <td>
     <a href="http://www.uniprot.org/manual/sequence%5Fcaution">
      Sequence caution
     </a>
    </td>
    <td>
     Warning about possible errors related to the protein sequence
    </td>
   </tr>
   <tr>
    <td>
     <a href="http://www.uniprot.org/manual/rna%5Fediting">
      RNA editing
     </a>
    </td>
    <td>
     Description of amino acid change(s) due to RNA editing
    </td>
   </tr>
   <tr>
    <td>
     <a href="http://www.uniprot.org/manual/mass%5Fspectrometry">
      Mass spectrometry
     </a>
    </td>
    <td>
     Information derived from mass spectrometry experiments
    </td>
   </tr>
   <tr>
    <td>
     <a href="http://www.uniprot.org/manual/polymorphism">
      Polymorphism
     </a>
    </td>
    <td>
     Description of polymorphism(s)
    </td>
   </tr>
   <tr>
    <td>
     <a href="http://www.uniprot.org/manual/variant">
      Natural variant
     </a>
    </td>
    <td>
     Natural variants documented for the protein
    </td>
   </tr>
   <tr>
    <td>
     <a href="http://www.uniprot.org/manual/var%5Fseq">
      Alternative sequence
     </a>
    </td>
    <td>
     Amino acid change(s) producing alternate protein isoforms
    </td>
   </tr>
   <tr>
    <td>
     <a href="http://www.uniprot.org/manual/unsure">
      Sequence uncertainty
     </a>
    </td>
    <td>
     Regions of uncertainty in the sequence
    </td>
   </tr>
   <tr>
    <td>
     <a href="http://www.uniprot.org/manual/conflict">
      Sequence conflict
     </a>
    </td>
    <td>
     Description of sequence discrepancies of unknown origin
    </td>
   </tr>
   <tr>
    <td>
     <a href="http://www.uniprot.org/manual/non%5Fcons">
      Non-adjacent residues
     </a>
    </td>
    <td>
     Indicates that two residues in a sequence are not consecutive
    </td>
   </tr>
   <tr>
    <td>
     <a href="http://www.uniprot.org/manual/non%5Fter">
      Non-terminal residue
     </a>
    </td>
    <td>
     The sequence is incomplete. Indicate that a residue is not the terminal residue of the complete protein
    </td>
   </tr>
   <tr>
    <td>
     <a href="http://www.uniprot.org/manual/non%5Fstd">
      Non-standard residue
     </a>
    </td>
    <td>
     Occurence of non-standard amino acids (selenocysteine and pyrrolysine) in the protein sequence
    </td>
   </tr>
   <tr>
    <td>
     <a href="http://www.uniprot.org/database/?query=category%3A%22Sequence+databases">
      Sequence databases
     </a>
    </td>
    <td>
     Selection of
     <a href="http://www.uniprot.org/manual/cross%5Freferences%5Fsection">
      cross-references
     </a>
     that point to data collections other than UniProtKB
    </td>
   </tr>
   <tr>
    <td>
     <a href="http://www.uniprot.org/database/?query=category%3A%22Genome+annotation+databases">
      Genome annotation databases
     </a>
    </td>
    <td>
     Selection of
     <a href="http://www.uniprot.org/manual/cross%5Freferences%5Fsection">
      cross-references
     </a>
     that point to data collections other than UniProtKB
    </td>
   </tr>
   <tr>
    <td>
     <a href="http://www.uniprot.org/database/?query=category%3A%22Genetic+variation+databases">
      Genetic variation databases
     </a>
    </td>
    <td>
     Selection of
     <a href="http://www.uniprot.org/manual/cross%5Freferences%5Fsection">
      cross-references
     </a>
     that point to data collections other than UniProtKB
    </td>
   </tr>
   <tr>
    <td>
     <a href="http://www.uniprot.org/keywords/KW-9997">
      Keywords 'Coding sequence diversity'
     </a>
    </td>
    <td>
     Selection of controlled vocabulary which summarises the content of an entry
    </td>
   </tr>
  </table>
 </div>
</div>