<div>
 <p>
  This subsection of the 'PTM / Processing' section describes the extent of a transit peptide.
 </p>
 <p>
  Transit peptides are responsible for the transport of a protein encoded by a nuclear gene to a particular organelle. To date, we report transit peptides for the following organelles:
 </p>
 <ul>
  <li>
   Mitochondrion
  </li>
  <li>
   Apicoplast
  </li>
  <li>
   Chromoplast
  </li>
  <li>
   Chloroplast
  </li>
  <li>
   Cyanelle
  </li>
  <li>
   Thylakoid
  </li>
  <li>
   Amyloplast
  </li>
  <li>
   Peroxisome
  </li>
  <li>
   Glyoxysome
  </li>
  <li>
   Hydrogenosome
  </li>
 </ul>
 <p>
  Cyanelles are the plastids of glaucocystophyte algae.
 </p>
 <p>
  Peroxisomes, hydrogenosomes, glyoxysomes and glycosomes constitute the class of microbodies, which are defined as single membrane-bounded small organelles. Proteins can be targeted to microbodies via an N-terminal transit peptide (which is rare), or via a C-terminal motif (which is more common).
 </p>
 <p>
  Chromoplasts are plastids containing pigments other than chlorophyll. Found in flowers, petals and fruits.
 </p>
 <p>
  Transit peptides are generally cleaved from the mature protein and so will logically be found in proteins tagged with the 'Sequence processing' information: 'The displayed sequence is further processed into a mature form.'.
 </p>
 <p>
  The particular organelle to which a protein is targeted is indicated in the
  <a href="http://www.uniprot.org/manual/subcellular_location">
   'Subcellular location'
  </a>
  subsection and by the presence of a specific keyword.
 </p>
 <h3 id="1__Annotation_of_experimentally_proven_transit_peptides_">
  1. Annotation of experimentally proven transit peptides:
 </h3>
 <p>
  We annotate experimentally proven transit peptides when the cleavage site has been determined by direct protein sequencing.
  <br/>
  Example:
  <a href="http://www.uniprot.org/uniprot/P15690#ptm_processing">
   P15690
  </a>
 </p>
 <p>
  Large-scale proteomics data are used to annotate transit peptides through the combination of experimental and computational methods, tagged with evidence from
  <a href="http://www.uniprot.org/help/evidences#ECO:00007744">
   'Combined sources'
  </a>
  .
  <br/>
  Example:
  <a href="http://www.uniprot.org/uniprot/P28331#ptm_processing">
   P28331
  </a>
 </p>
 <p>
  This information can then be propagated
  <a href="http://www.uniprot.org/help/evidences#ECO:0000250">
   'By similarity'
  </a>
  to closely related homologs provided that the transit peptide sequence is conserved.
  <br/>
  Example:
  <a href="http://www.uniprot.org/uniprot/Q0MQG1#ptm_processing">
   Q0MQG1
  </a>
 </p>
 <p>
  When a protein contains a transit peptide (according to experimental data or its similarity with a family of proteins), but the precise cleavage position has not been experimentally determined, we use a question mark instead of a precise position.
  <br/>
  Example:
  <a href="http://www.uniprot.org/uniprot/Q6GQ48#ptm%5Fprocessing">
   Q6GQ48
  </a>
 </p>
 <h3 id="2__Annotation_of_predicted_transit_peptides_">
  2. Annotation of predicted transit peptides:
 </h3>
 <p>
  We also annotate transit peptides which are predicted by the application of the predictive tools
  <a href="http://mitf.cbrc.jp/MitoFates/cgi-bin/top.cgi">
   Mitofates
  </a>
  ,
  <a href="http://urgi.versailles.inra.fr/predotar/predotar.html">
   Predotar
  </a>
  and
  <a href="http://www.cbs.dtu.dk/services/TargetP/">
   TargetP
  </a>
  , but only when such predictions are consistent with the known or presumed subcellular location of the protein concerned. The predicted positions of the transit peptide are annotated with evidence
  <a href="http://www.uniprot.org/help/evidences#ECO:0000255">
   'Sequence analysis'
  </a>
  .
  <br/>
  Example:
  <a href="http://www.uniprot.org/uniprot/P91929#ptm_processing">
   P91929
  </a>
 </p>
 <p>
  See also:
 </p>
 <ul>
  <li>
   <a href="http://www.uniprot.org/manual/evidences">
    Evidence
   </a>
  </li>
  <li>
   <a href="http://www.uniprot.org/manual/peptide">
    Peptide (describing small active peptides)
   </a>
  </li>
  <li>
   <a href="http://www.uniprot.org/help/sequence%5Fannotation">
    Sequence annotation (features)
   </a>
  </li>
 </ul>
 <p>
  Related keywords:
  <br/>
  <a href="http://www.uniprot.org/keywords/809">
   Transit peptide
  </a>
  ,
  <br/>
  <a href="http://www.uniprot.org/keywords/150">
   Chloroplast
  </a>
  ,
  <br/>
  <a href="http://www.uniprot.org/keywords/194">
   Cyanelle
  </a>
  ,
  <br/>
  <a href="http://www.uniprot.org/keywords/330">
   Glyoxysome
  </a>
  ,
  <br/>
  <a href="http://www.uniprot.org/keywords/377">
   Hydrogenosome
  </a>
  ,
  <br/>
  <a href="http://www.uniprot.org/keywords/496">
   Mitochondrion
  </a>
  ,
  <br/>
  <a href="http://www.uniprot.org/keywords/576">
   Peroxisome
  </a>
  ,
  <br/>
  <a href="http://www.uniprot.org/keywords/793">
   Thylakoid
  </a>
  .
 </p>
</div>