---
title: UniProt release 11.1
categories: release,Taxonomy,Chordata
---

## Headlines

### 4,000 bovine entries in UniProtKB/Swiss-Prot

UniProtKB/Swiss-Prot is happy to announce the annotation of [over 4,000 entries](http://www.uniprot.org/uniprot/?query=organism:9913+reviewed:yes) of a very popular animal in Switzerland, almost a national symbol: [Bos taurus](http://www.uniprot.org/taxonomy/9913) , in other words, the cow.

Those of you who have visited the Swiss Alps may know that their gorgeous scenery is definitely associated with the sound of cowbells in summer pasture. Similarly, the modern biology landscape would be poorer without bovine sequences, obviously not in a decorative role, but as a key element for our understanding of human biology.

The domesticated cow is extensively used in biomedical research, as an animal model and also as a source of biological material. Remember that bovine insulin was the first sequenced protein and was used for decades to treat diabetes. The first draft of the bovine genome sequence was released in October 2004 by the Human Genome Sequencing Center of the Baylor College of Medicine. The human and bovine genomes are more similarly organized than when either is compared to the mouse. Despite its interest, only a few large scale cDNA sequencing projects have been initiated. Currently more than 70% of the UniProtKB/Swiss-Prot bovine sequences come from translation of cDNA sequences produced the NIH [Mammalian Gene Collection](http://mgc.nci.nih.gov/) and the [Agricultural Research Service](http://www.pubmedcentral.nih.gov/articlerender.fcgi?tool=pubmed&pubmedid=16305752) , US Department of Agriculture.

## UniProtKB News

### Cross-references to PeptideAtlas

Cross-references to the [PeptideAtlas](http://www.peptideatlas.org/) database [have been added](http://www.uniprot.org/uniprot/?query=database:peptideatlas) . PeptideAtlas is a multi-organism, publicly accessible compendium of peptides that have been identified in a large set of tandem mass spectrometry proteomics experiments. All results of sequence searching have subsequently been processed through PeptideProphet to derive a probability of correct identification for all results in a uniform manner to insure a high quality database. All peptides have been mapped to Ensembl and can be viewed as custom tracks on the Ensembl Genome Browser.

The format of the explicit links in the flat file is:

<table><colgroup><col style="width: 50%" /><col style="width: 50%" /></colgroup><tbody><tr class="odd"><td>Resource abbreviation</td><td>PeptideAtlas</td></tr><tr class="even"><td>Resource identifier</td><td>UniProtKB accession number.</td></tr><tr class="odd"><td>Example</td><td><pre><code>P08524:
DR   PeptideAtlas; P08524; -.</code></pre></td></tr></tbody></table>

### Cross-references to DisProt

Cross-references to the [Database of Protein Disorder](http://www.disprot.org/) (DisProt) [have been added](http://www.uniprot.org/uniprot/?query=database:disprot) . The Database of Protein Disorder (DisProt) is a curated database that provides information about proteins that lack fixed 3D structure in their putatively native states, either in their entirety or in part. DisProt is a collaborative effort between Center for Computational Biology and Bioinformatics at Indiana University School of Medicine and Center for Information Science and Technology at Temple University.

The format of the explicit links in the flat file is:

<table><colgroup><col style="width: 50%" /><col style="width: 50%" /></colgroup><tbody><tr class="odd"><td>Resource abbreviation</td><td>DisProt</td></tr><tr class="even"><td>Resource identifier</td><td>DisProt accession number.</td></tr><tr class="odd"><td>Example</td><td><pre><code>P07293:
DR   DisProt; DP00228; -.
DR   DisProt; DP00440; -.</code></pre></td></tr></tbody></table>
