---
title: UniProt release 14.5
categories: release,Plants
---

## Headlines

### The plastid: the most important organelle!

The world is full of plastids. Most of us know the green photosynthetic chloroplast which houses the machinery that fixes CO <sub>2</sub> (with O <sub>2</sub> as a "mere" by-product) and synthesizes sugars, lipids, amino acids, etc.; in short, the basis of our food chain. Found in plants and algae, chloroplasts are absolutely essential to life as we know it.

Plastids contain DNA; they are the remnants of a cyanobacterium that was engulfed by a eukaryotic heterotroph which had previously engulfed an alphaproteobacteria which eventually became the mitochondrion. These are primary endosymbiotic events; the organism that was taken up by the host was not digested but survived in the cytoplasm, eventually transferring genes to the host nucleus and being in effect enslaved. Most of these transferred gene products are imported back into their respective organelles using transit peptides. Plastids now encode between 28 and 250 protein-coding genes. The primary plastid endosymbiosis gave rise to 3 lineages: green algae, red algae and the glaucophytes. Subsequent engulfment of green or red algae by other eukaryotes has given rise to secondary endosymbionts, which in some cases have been engulfed again, sometimes with plastid replacement, to give an array of tertiary endosymbionts. These secondary and tertiary events gave rise to (among others) cryptophytes, diatoms, heterokont algae and apicocomplexa which are organisms that are no longer photosynthetic such as Plasmodium. To further complicate matters, it was thought that there were only 2 primary endosymbiotic events; [recent work](http://view.ncbi.nlm.nih.gov/pubmed/18356055) , however, on a thecate amoeba, Paulinella chromatophora, has cast doubt on this assumption.

Due to their small size, plastids are easily sequenced. A list of fully sequenced plastid genomes, their genes and the nomenclature of known plastid-encoded proteins can be found in our document [plastid.txt](http://www.uniprot.org/docs/plastid) .

In UniProtKB, we indicate whether a protein is encoded by plastid, mitochondrial or plasmid DNA in the 'Names and origin' section, 'Encoded on' subsection (OG line in the flat file). 6 categories have been created for plastids:

-   'Plastid; Chloroplast' indicates the organism is photosynthetic, whether of primary, secondary or higher endosymbiotic events.
-   'Plastid; Non-photosynthetic plastid' is used when the organism is from a photosynthetic lineage but genetically unable to photosynthesize, as happens with some parasitic plants (Epifagus virginiana, Aneura mirabilis), a parastic "green" algae (Helicosporidium sp. subsp. Simulium jonesii) and a euglenoid (Astasia longa).
-   'Plastid; Cyanelle' is used for the plastid of the glaucophyte algae. It has the remnants of a cell wall between its surrounding membranes.
-   'Plastid; Apicoplast' is used for plastids from the non-photosynthetic Apicocomplexan parasites such as Plasmodium, Toxoplasma and Eimeria which cause malaria, toxoplasmosis and coccidian diseases respectively. Although the plastid remnant has a reduced coding capacity, it is essential for cell survival and is interesting as a drug target.
-   'Plastid; Chromatophore' is used for the plastid of the thecate amoeba Paulinella chromatophora, which has a very large endosymbiont genome (1.0 Mb, encoding almost 900 proteins).
-   'Plastid' (without any qualifier) is used for some parasitic plants (mostly from the genus Cuscuta) which may be briefly photosynthetic when very young.

Currently, in UniProtKB/Swiss-Prot, there are close to 11'000 entries encoded by a [plastid](http://www.uniprot.org/uniprot/?query=organelle:plastid+AND+reviewed:yes) genome; 10'130 by [chloroplasts](http://www.uniprot.org/uniprot/?query=organelle:Chloroplast+AND+reviewed:yes) , 145 by [cyanelles](http://www.uniprot.org/uniprot/?query=organelle:Cyanelle+AND+reviewed:yes) , 142 by [non-photosynthetic plastids](http://www.uniprot.org/uniprot/?query=organelle:Non-photosynthetic+plastid+AND+reviewed:yes) , 18 by [apicoplasts](http://www.uniprot.org/uniprot/?query=organelle:Apicoplast+AND+reviewed:yes) , 22 by [chromatophores](http://www.uniprot.org/uniprot/?query=organelle:Chromatophore+AND+reviewed:yes) and 165 by [unspecified types of plastids](http://www.uniprot.org/uniprot/?query=(organelle:plastid+AND+reviewed:yes)+NOT+organelle:Apicoplast+NOT+organelle:Chloroplast+NOT+organelle:Chromatophore+NOT+organelle:Cyanelle+NOT+organelle:Non-photosynthetic+plastid) .

## UniProtKB News

### Changes concerning keywords

New keywords:

-   [Amyloidosis](http://www.uniprot.org/keywords/KW-1008)
-   [Dyskeratosis congenita](http://www.uniprot.org/keywords/KW-1011)
-   [Hearing](http://www.uniprot.org/keywords/KW-1009)
-   [Kartagener syndrome](http://www.uniprot.org/keywords/KW-1012)
-   [Non-syndromic deafness](http://www.uniprot.org/keywords/KW-1010)

Modified keyword:

-   Tegument protein -&gt; [Virion tegument](http://www.uniprot.org/keywords/KW-0920)

Deleted keyword:

-   Structural protein

## Website News

### New UniParc query field 'isoform'

The existing query field `uniprot` allows you to search UniParc for the [canonical sequence](http://www.uniprot.org/help/canonical_and_isoforms) of a UniProtKB entry, e.g. `uniprot:P00750` . With the new query field `isoform` you can retrieve the UniParc record that corresponds to the sequence of a specific UniProtKB isoform, e.g. `isoform:P00750-2` or you can retrieve all isoforms of a UniProtKB entry, e.g. `isoform:P00750-*` .

This can also be done with the website's toolbar:

1.  Select **Search in** : `Sequence Archive (UniParc)`
2.  Click on **Fields »** to open the query builder
3.  Select **Field** : `UniProtKB isoform ID`
4.  Type the identifier, e.g. `P00750-2`
5.  Click on **Add & Search**

### Programmatic search for UniRef and UniParc identifiers in UniProtKB

The URLs to search for UniRef and UniParc identifiers in UniProtKB are going to change in the following way:

<table style="width:99%;"><colgroup><col style="width: 16%" /><col style="width: 40%" /><col style="width: 41%" /></colgroup><thead><tr class="header"><th></th><th style="text-align: left;">Valid until release 14.6</th><th style="text-align: left;">Valid from release 14.5</th></tr></thead><tbody><tr class="odd"><td>UniRef</td><td style="text-align: left;"><code>cluster:*</code><br />
e.g. <code>cluster:UniRef50_Q8WZ42</code></td><td style="text-align: left;"><code>cluster:(*)</code><br />
e.g. <code>cluster:(UniRef50_Q8WZ42)</code></td></tr><tr class="even"><td>UniParc</td><td style="text-align: left;"><code>sequence:*</code><br />
e.g. <code>sequence:UPI0000D7E631</code></td><td style="text-align: left;"><code>sequence:(*)</code><br />
e.g. <code>sequence:(UPI0000D7E631)</code></td></tr></tbody></table>

Please change your queries before release 14.6 by adding parentheses around the identifier.

The web interface for searching UniRef and UniParc identifiers in UniProtKB remains unchanged:

1.  Select **Search in** : `Protein Knowledgebase (UniProtKB)`
2.  Click on **Fields »** to open the query builder
3.  Select **Field** : `UniRef ID` (or `UniParc ID` )
4.  Type the identifier, e.g. `UniRef50_Q8WZ42`
5.  Click on **Add & Search**
