---
title: UniProt release 15.1
categories: release,Viruses
---

## Headlines

### Hepatitis Delta virus, a living fossil virus of the old RNA world?

[Hepatitis delta virus](http://viralzone.expasy.org/all_by_species/175.html) (HDV) is unique in virology, and continues to fascinate since its discovery 30 years ago. HDV is a defective virus parasiting [hepatitis B virus](http://viralzone.expasy.org/all_by_species/101.html) (HBV) infected cells. The clinical significance of HDV infection is more severe acute and chronic liver disease than that caused by HBV alone.

Only 1'680 nucleotides long, the HDV genome is the smallest known to infect man. The virus comprises one single gene, encoding the small Hepatitis Delta Antigen ( [S-HDAg](http://www.uniprot.org/uniprot/P0C6L3) ). To compensate for this limited protein-coding capacity, HDV relies on a unique molecular mechanism to hijack host functions and the extraordinary dynamics of its RNA genome.

All known RNA viruses code for an RNA-dependent RNA polymerase to replicate/transcribe their genome, since eukaryotic host cells are unable to replicate RNA genomes. All but HDV; surprisingly, the S-HDAg seems to modify the activity of human DNA-dependent RNA polymerase II, [turning it into an RNA-dependent RNA polymerase](http://view.ncbi.nlm.nih.gov/pubmed/18032511) ! Not only is this activity unique in molecular biology, but it also has many implications in the field of molecular evolution: life is thought to have started as RNA. HDV highlights the potential ability of human RNA polymerase II to switch back to an activity presumably forgotten for hundreds of millions of years.

HDV genome replication is further pushing its nostalgia for the ancient RNA world. Rolling circle genome replication produces a ssRNA composed of numerous repeats of the viral genome. All viruses known to use the rolling circle replication rely on proteins to cleave the genome concatemer. All but HDV; cleavage occurs via an autocatalytic ribozyme activity encoded in the RNA genome.

HDV needs HBV co-infection only to borrow its capsid and budding mechanism. This function is carried out by a longer isoform of HDAg with an additional 19 to 20 amino acids ( [L-HDAg](http://www.uniprot.org/uniprot/P29996) ). Again HDV relies on a unique mechanism to produce this isoform; the genomic RNA is edited at one specific site by a human [RNA adenosine deaminase](http://www.uniprot.org/uniprot/P55265) (ADAR1). Somehow, edited genomes are unable to replicate, assuring that the unedited version remains predominant.

The lesson from this quite unusual virus is that evolution does not always result in the creation of new tools, but sometimes it allows an existing tool to learn old and long forgotten tricks

## UniProtKB News

### Cross-references to CAZy

Cross-references have been added to the Carbohydrate-Active enZymes database [CAZy](http://www.cazy.org/) . CAZy describes the families of structurally-related catalytic and carbohydrate-binding modules (or functional domains) of enzymes that degrade, modify, or create glycosidic bonds.

CAZy is available at <http://www.cazy.org/> .

The format of the explicit links in the flat file is:

<table><colgroup><col style="width: 36%" /><col style="width: 63%" /></colgroup><tbody><tr class="odd"><td>Resource abbreviation</td><td>CAZy</td></tr><tr class="even"><td>Resource identifier</td><td>CAZy family number.</td></tr><tr class="odd"><td>Optional information 1</td><td>CAZy family name.</td></tr><tr class="even"><td>Examples</td><td><pre><code>P30590:
DR   CAZy; GT2; Glycosyltransferase Family 2.

P32775:
DR   CAZy; CBM48; Carbohydrate-Binding Module Family 48.
DR   CAZy; GH13; Glycoside Hydrolase Family 13.</code></pre></td></tr></tbody></table>

### Changes concerning keywords

New keyword:

-   [Pontocerebellar hypoplasia](http://www.uniprot.org/keywords/KW-1021)

### Changes in [subcellular location controlled vocabulary](http://www.uniprot.org/docs/subcell)

New subcellular locations:

-   Host
-   Host apical cell membrane
-   Host caveola
-   Host cell
-   Host cell envelope
-   Host cell inner membrane
-   Host cell junction
-   Host cell membrane
-   Host cell outer membrane
-   Host cell projection
-   Host cell surface
-   Host cell wall
-   Host cellular thylakoid
-   Host cellular thylakoid membrane
-   Host cis-Golgi network
-   Host cis-Golgi network membrane
-   Host cytoplasm
-   Host cytoplasmic vesicle
-   Host cytoplasmic vesicle membrane
-   Host cytoskeleton
-   Host cytosol
-   Host endoplasmic reticulum
-   Host endoplasmic reticulum lumen
-   Host endoplasmic reticulum membrane
-   Host endoplasmic reticulum-Golgi intermediate compartment
-   Host endoplasmic reticulum-Golgi intermediate compartment membrane
-   Host endosome
-   Host endosome membrane
-   Host extracellular space
-   Host filopodium
-   Host Golgi apparatus
-   Host Golgi apparatus membrane
-   Host intracytoplasmic membrane
-   Host late endosome
-   Host late endosome membrane
-   Host lipid droplet
-   Host lipid droplet membrane
-   Host lysosome
-   Host lysosome membrane
-   Host membrane
-   Host microsome
-   Host microsome membrane
-   Host mitochondrion
-   Host mitochondrion envelope
-   Host mitochondrion inner membran
-   Host mitochondrion membrane
-   Host mitochondrion outer membrane
-   Host nucleolus
-   Host nucleoplasm
-   Host nucleus
-   Host nucleus envelope
-   Host nucleus inner membrane
-   Host nucleus lamina
-   Host nucleus matrix
-   Host nucleus membrane
-   Host perinuclear region
-   Host periplasm
-   Host plasmodesma
-   Host rough endoplasmic reticulum
-   Host rough endoplasmic reticulum membrane
