---
title: UniProt release 2014_03
categories: human,Prokaryotes
---

<h2 id="Headline">
 Headline
</h2>
<h3 id="Minority_report">
 Minority report
</h3>
<p>
 We are a minority in our own body. Over 90% of our cells are actually not human, but microbial. The majority of these microbes reside in the gut. The gut microbiota is typically dominated by bacteria, more specifically by
 <a href="http://www.ncbi.nlm.nih.gov/pubmed/17183312">
  Bacteroidetes and Firmicutes
 </a>
 . The exact composition of gut microbiota varies between individuals and depends upon lifestyle, diet, hygienic preferences, use of antibiotics, etc. Gut microbes have a
 <a href="http://www.ncbi.nlm.nih.gov/pubmed/15790844">
  profound influence on human physiology and nutrition
 </a>
 . Among others, they contribute to harvesting energy from food.
</p>
<p>
 All guidelines for a healthy diet emphasize the necessity of eating fruit, vegetables and whole grains. These products are rich in dietary fibers, i.e. non-starch polysaccharides, most of which cannot be digested by the hydrolases encoded by our genome. Our inherent ability to digest carbohydrates is restricted to starch and simple saccharides, not xyloglucans (XyGs), a family of highly branched plant cell wall polysaccharides, which are abundant in plants. In view of the prevalence of XyGs in our diet, the mechanism of degradation of these complex polysaccharides by bacteria was expected to be important to human energy acquisition, but until recently it was still unclear.
 <a href="http://www.ncbi.nlm.nih.gov/pubmed/24463512">
  Very interesting work by Larsbrink et al.
 </a>
 , published in February, sheds light on XyG metabolism. The authors identified a polysaccharide utilization locus (PUL) in the genome of a common human gut symbiont,
 <a href="http://www.uniprot.org/taxonomy/411476">
  <em>
   Bacteroides ovatus
  </em>
 </a>
 . PUL is transcriptionally upregulated in response to growth on galactoxyloglucan. It is predicted to encode
 <a href="http://www.uniprot.org/uniprot/?query=24463512">
  10 genes
 </a>
 , including
 <a href="http://www.uniprot.org/uniprot/?query=organism:411476+and+name:BoGH">
  8 glycoside hydrolases
 </a>
 . All of them were subjected to in-depth molecular characterization through reverse genetics,
 <em>
  in vitro
 </em>
 protein biochemistry and enzymology. Finally, the
 <a href="http://www.pdb.org/pdb/explore/explore.do?pdbId=3ZMR">
  3D structure
 </a>
 of the endo-xyloglucanase
 <a href="http://www.uniprot.org/uniprot/A7LXT7">
  BoGH5A
 </a>
 , which generates short XyG oligosaccharides, was solved. This study unraveled all the details of the enzymatic pathways by which the most common dietary polysaccharides are digested in our gut.
</p>
<p>
 Although XyG utilization loci (XyGULs) have been identified in only a few other gut-resident Bacteroidetes, including
 <em>
  B. cellulosyliticus
 </em>
 ,
 <em>
  B. uniformis
 </em>
 ,
 <em>
  B. fluxus
 </em>
 ,
 <em>
  Dysgonomonas mossii
 </em>
 and
 <em>
  D. gadei
 </em>
 , most human beings harbor at least one of these Bacteroides XyGULs in their gut, suggesting their importance in human nutrition.
</p>
<p>
 The importance of the gut microbiome goes far beyond an active role in food digestion. It also acts on intestinal function, promoting gut-associated lymphoid tissue maturation, tissue regeneration, gut motility, and morphogenesis of the vascular system surrounding the gut. It additionally affects
 <a href="http://www.ncbi.nlm.nih.gov/pubmed/23435359">
  many other physiopathological aspects
 </a>
 , such as the nervous system and bone homeostasis. Not surprisingly, changes in the microbiota composition or a complete lack of a gut microbiota has been shown to affect metabolism, tissue homeostasis and behavior.
</p>
<p>
 As of this release, manually reviewed
 <a href="http://www.uniprot.org/uniprot/?query=citation:24463512">
  <em>
   B. ovatus
  </em>
  XyGUL gene products
 </a>
 are available in UniProtKB/Swiss-Prot. Let's bet that they will be followed by many more proteins encoded by our other genome(s) in the near future.
</p>
<h2 id="UniProtKB_news">
 UniProtKB news
</h2>
<h3 id="Cross_references_for_isoform_sequences">
 Cross-references for isoform sequences
</h3>
<p>
 Some of the resources to which we link contain information that is specific to an
 <a href="http://www.uniprot.org/manual/alternative%5Fproducts">
  isoform sequence
 </a>
 and where this is known we now indicate the corresponding UniProtKB isoform sequence identifier in our cross-references as described below. The first resources for which we provide such isoform-specific cross-references are
 <a href="http://www.ensembl.org/">
  Ensembl
 </a>
 and
 <a href="http://genome.ucsc.edu">
  UCSC
 </a>
 .
</p>
<h4 id="Text_format">
 Text format
</h4>
<p>
 The UniProtKB isoform sequence identifier is shown in square brackets at the end of the DR line as an optional field:
</p>
<pre>DR   ResourceAbbreviation; ResourceIdentifier(; AdditionalField)+. [IsoId]
</pre>
<p>
 Examples:
</p>
<pre>DR   Ensembl; ENST00000281772; ENSP00000281772; ENSG00000144445. [A0AUZ9-1]
DR   Ensembl; ENST00000418791; ENSP00000405724; ENSG00000144445. [A0AUZ9-2]
DR   Ensembl; ENST00000452086; ENSP00000401408; ENSG00000144445. [A0AUZ9-3]
DR   Ensembl; ENST00000457374; ENSP00000393432; ENSG00000144445. [A0AUZ9-3]
</pre>
<pre>DR   UCSC; uc002vds.3; human. [A0AUZ9-1]
DR   UCSC; uc002vdt.3; human. [A0AUZ9-2]
DR   UCSC; uc002vdx.1; human. [A0AUZ9-4]
</pre>
<h4 id="XML_format">
 XML format
</h4>
<p>
 To show the UniProtKB isoform sequence identifier in
 <code>
  dbReference
 </code>
 elements, we added an optional
 <code>
  molecule
 </code>
 element to the
 <code>
  dbReferenceType
 </code>
 . For consistency, we also changed the type of the
 <code>
  molecule
 </code>
 element that is found in the
 <code>
  commentType
 </code>
 . The XSD has been changed as highlited below:
</p>
<pre><code>&lt;xs:complexType name="commentType"&gt;
    ...
                &lt;xs:sequence&gt;
                    &lt;xs:annotation&gt;
                        &lt;xs:documentation&gt;Used in 'subcellular location' annotations.&lt;/xs:documentation&gt;
                    &lt;/xs:annotation&gt;</code>
                    <code style="color: red;">&lt;!-- &lt;xs:element name="molecule" type="xs:string" minOccurs="0"/&gt; --&gt;
                    &lt;xs:element name="molecule" type="moleculeType" minOccurs="0"/&gt;</code>
                    <code>&lt;xs:element name="subcellularLocation" type="subcellularLocationType" maxOccurs="unbounded"/&gt;
                &lt;/xs:sequence&gt;
    ...
    &lt;xs:complexType name="dbReferenceType"&gt;
    ...
        &lt;xs:sequence&gt;</code>
            <code style="color: red;">&lt;xs:element name="molecule" type="moleculeType" minOccurs="0"/&gt;</code>
            <code>&lt;xs:element name="property" type="propertyType" minOccurs="0" maxOccurs="unbounded"/&gt;
        &lt;/xs:sequence&gt;
        ...
    &lt;/xs:complexType&gt;
    ...</code>
    <code style="color: red;">&lt;xs:complexType name="moleculeType"&gt;
        &lt;xs:annotation&gt;
            &lt;xs:documentation&gt;Describes a molecule by name or unique identifier.&lt;/xs:documentation&gt;
        &lt;/xs:annotation&gt;
        &lt;xs:simpleContent&gt;
            &lt;xs:extension base="xs:string"&gt;
                &lt;xs:attribute name="id" type="xs:string" use="optional"/&gt;
            &lt;/xs:extension&gt;
        &lt;/xs:simpleContent&gt;
    &lt;/xs:complexType&gt;</code>
</pre>
<p>
 Examples:
</p>
<pre>&lt;dbReference type="Ensembl" id="ENST00000281772"&gt;
  &lt;molecule id="A0AUZ9-1"/&gt;
  &lt;property type="protein sequence ID" value="ENSP00000281772"/&gt;
  &lt;property type="gene ID" value="ENSG00000144445"/&gt;
&lt;/dbReference&gt;
&lt;dbReference type="Ensembl" id="ENST00000418791"&gt;
  &lt;molecule id="A0AUZ9-2"/&gt;
  &lt;property type="protein sequence ID" value="ENSP00000405724"/&gt;
  &lt;property type="gene ID" value="ENSG00000144445"/&gt;
&lt;/dbReference&gt;
&lt;dbReference type="Ensembl" id="ENST00000452086"&gt;
  &lt;molecule id="A0AUZ9-3"/&gt;
  &lt;property type="protein sequence ID" value="ENSP00000401408"/&gt;
  &lt;property type="gene ID" value="ENSG00000144445"/&gt;
&lt;/dbReference&gt;
&lt;dbReference type="Ensembl" id="ENST00000457374"&gt;
  &lt;molecule id="A0AUZ9-3"/&gt;
  &lt;property type="protein sequence ID" value="ENSP00000393432"/&gt;
  &lt;property type="gene ID" value="ENSG00000144445"/&gt;
&lt;/dbReference&gt;
</pre>
<pre>&lt;dbReference type="UCSC" id="uc002vds.3"&gt;
  &lt;molecule id="A0AUZ9-1"/&gt;
  &lt;property type="organism name" value="human"/&gt;
&lt;/dbReference&gt;
&lt;dbReference type="UCSC" id="uc002vdt.3"&gt;
  &lt;molecule id="A0AUZ9-2"/&gt;
  &lt;property type="organism name" value="human"/&gt;
&lt;/dbReference&gt;
&lt;dbReference type="UCSC" id="uc002vdx.1"&gt;
  &lt;molecule id="A0AUZ9-4"/&gt;
  &lt;property type="organism name" value="human"/&gt;
&lt;/dbReference&gt;
</pre>
<h3 id="Changes_to_the__a_href___docs_humdisease__controlled_vocabulary_of_human_diseases__a_">
 Changes to the
 <a href="http://www.uniprot.org/docs/humdisease">
  controlled vocabulary of human diseases
 </a>
</h3>
<p>
 New diseases:
</p>
<ul>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-04003">
   Alzheimer disease 18
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-04014">
   Arrhythmogenic right ventricular dysplasia, familial, 13
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-04009">
   Arthrogryposis, distal, 5
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-04008">
   Bleeding disorder, platelet-type 17
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-04010">
   Cataract 41
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-03996">
   Combined oxidative phosphorylation deficiency 18
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-04002">
   Combined oxidative phosphorylation deficiency 19
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-04018">
   Complement factor B deficiency
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-04006">
   Congenital disorder of glycosylation 1W
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-04007">
   Congenital disorder of glycosylation 1X
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-04012">
   Deafness, X-linked, 6
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-03997">
   Fanconi renotubular syndrome 3
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-04000">
   Immunodeficiency 15
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-04001">
   Immunodeficiency 16
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-04005">
   Keratoderma, palmoplantar, Nagashima type
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-04015">
   L-ferritin deficiency
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-03998">
   Macular degeneration, age-related, 15
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-04004">
   Mental retardation, autosomal recessive 40
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-04013">
   Microphthalmia, syndromic, 1
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-04011">
   Ritscher-Schinzel syndrome
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-04016">
   Spastic ataxia 2, autosomal recessive
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-04017">
   Spastic ataxia 3, autosomal recessive
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-03999">
   Verheij syndrome
  </a>
 </li>
</ul>
<p>
 Modified diseases:
</p>
<ul>
 <li>
  Autoimmune polyendocrine syndrome 1 -&gt;
  <a href="http://www.uniprot.org/diseases/DI-01198">
   Autoimmune polyendocrine syndrome 1, with or without reversible metaphyseal dysplasia
  </a>
 </li>
 <li>
  Spastic ataxia autosomal recessive 4 -&gt;
  <a href="http://www.uniprot.org/diseases/DI-02952">
   Spastic ataxia 4, autosomal recessive
  </a>
 </li>
 <li>
  Spastic ataxia autosomal recessive 5 -&gt;
  <a href="http://www.uniprot.org/diseases/DI-03374">
   Spastic ataxia 5, autosomal recessive
  </a>
 </li>
</ul>
<h3 id="Changes_to_the__a_href___docs_ptmlist__controlled_vocabulary_for_PTMs__a_">
 Changes to the
 <a href="http://www.uniprot.org/docs/ptmlist">
  controlled vocabulary for PTMs
 </a>
</h3>
<p>
 New terms for the feature key 'Cross-link' ('CROSSLNK' in the flat file):
</p>
<ul>
 <li>
  Isoaspartyl lysine isopeptide (Lys-Asp)
 </li>
</ul>