---
title: UniProt release 2015_05
categories: human
---

<h2 id="Headline">
 Headline
</h2>
<h3 id="A_never_ending_race_between_evolution_and_genomic_integrity">
 A never-ending race between evolution and genomic integrity
</h3>
<p>
 <a href="http://www.ncbi.nlm.nih.gov/pubmed/15016989,20591532">
  Primate evolution has been accompanied by several waves of retrotransposon insertions
 </a>
 . Nowadays about 50% of our genome is composed of endogenous retroelements (EREs). Although many of them have lost their transposition ability, some remain quite active. For instance, among the 500,000 copies of long interspersed element-1 (LINE1 or L1) present in the human genome, about 100 are retrotransposition-competent, and over 40 of them are
 <a href="http://www.ncbi.nlm.nih.gov/pubmed/12682288,21801021">
  highly active
 </a>
 . Other EREs, such as short interspersed nuclear elements (SINEs), including Alu repeats, and SINE-VNTR-Alu (SVA), a composite hominid-restricted ERE, also actively move in the genome. It is currently estimated that new, non-parental L1 integrations occur in
 <a href="http://www.ncbi.nlm.nih.gov/pubmed/20488934,20813032">
  nearly 1/100 births
 </a>
 and roughly every 20th newborn baby has a
 <a href="http://www.ncbi.nlm.nih.gov/pubmed/16522357">
  new Alu retrotransposon somewhere in its DNA
 </a>
 .
</p>
<p>
 Obviously having DNA jumping around our genome may be quite harmful and our cells work hard to repress EREs. Transcriptional silencing is controlled by
 <a href="http://www.uniprot.org/uniprot/Q13263">
  TRIM28
 </a>
 and
 <a href="http://www.uniprot.org/uniprot/?query=family:%22krueppel+C2H2-type+zinc-finger+protein+family%22+AND+domain:%22KRAB+domain%2A%22+AND+organism:9606+AND+reviewed:yes">
  KRAB domain-containing Zinc finger proteins
 </a>
 (KRAB-ZNFs). TRIM28 forms a repressive complex (KAP1 complex) by interacting with
 <a href="http://www.uniprot.org/uniprot/Q12873">
  CHD3
 </a>
 , a subunit of the nucleosome remodeling and deacetylation (NuRD) complex, and
 <a href="http://www.uniprot.org/uniprot/SETDB1">
  SETDB1
 </a>
 , which specifically methylates histone H3 at 'Lys-9', inducing heterochromatinization. KRAB-ZNFs bind DNA and recruit the KAP1 complex to target sites.
</p>
<p>
 KRAB-ZNF genes are one of the fastest growing gene families in primates, possibly to limit the activity of newly emerged ERE classes. This hypothesis has gained support in
 <a href="http://www.ncbi.nlm.nih.gov/pubmed/25274305">
  an elegant study recently published in Nature
 </a>
 . In this article, Jacobs et al. used a heterologous cell system in which murine embryonic stem cells harbored a copy of human chromosome 11, which contains a number of EREs, including SVA and the L1 subfamily L1PA. In this cellular environment, the primate-specific EREs were derepressed. Individual overexpression of highly expressed human KRAB-ZNFs, confirmed by reporter gene assays, allowed the identification of genes involved in the repression of specific ERE (sub)families:
 <a href="http://www.uniprot.org/uniprot/Q05481">
  ZNF91
 </a>
 and
 <a href="http://www.uniprot.org/uniprot/P35789">
  ZNF93
 </a>
 which acted on SVA and L1PA4, respectively. The authors then traced back the phylogenic history of these genes in the primate lineage and analyzed the parallel evolution of their target EREs. They could show that a new wave of L1PA insertions in great ape genomes was made possible through the deletion of a 129-bp element in L1PA3, which destroyed the ZNF93-binding site. This could be interpreted as an ERE response to a series of structural changes in ZNF93 that occurred soon before and improved host repression of L1PA activity.
</p>
<p>
 In conclusion, the expansion of a new ERE drives the evolution of a host repressor which leads to a subsequent change in ERE to escape repression, and so on. It is a never-ending race of our genome with itself, which leads inexorably to greater and greater complexity.
</p>
<p>
 As of this release, updated human
 <a href="http://www.uniprot.org/uniprot/Q05481">
  ZNF91
 </a>
 and
 <a href="http://www.uniprot.org/uniprot/P35789">
  ZNF93
 </a>
 entries are available in UniProtKB/Swiss-Prot.
</p>
<h2 id="UniProtKB_news">
 UniProtKB news
</h2>
<h3 id="Removal_of_IPI_species_proteome_data_sets_from_FTP_site">
 Removal of IPI species proteome data sets from FTP site
</h3>
<p>
 Since the closure of IPI in 2011, UniProt has provided proteome data sets for IPI species on its FTP site. In UniProt release
 <a href="http://www.uniprot.org/news/2015/03/04/release">
  2015_03
 </a>
 , we have started to provide new
 <a href="ftp://ftp.uniprot.org/pub/databases/uniprot/current%5Frelease/knowledgebase/reference%5Fproteomes/README">
  data sets for reference proteomes
 </a>
 which cover also the IPI species and we have now removed the old 'proteomes' FTP directory that contained only data for the IPI species.
</p>
<h3 id="UniProtKB_XSD_change_for_evidence_attribution">
 UniProtKB XSD change for evidence attribution
</h3>
<p>
 We have made the following changes to the UniProtKB XSD to allow a more fine-grained attribution of evidence to the parts of
 <code>
  comment
 </code>
 annotations that contain "free-text" descriptions:
</p>
<ul>
 <li>
  The cardinality of all existing
  <code>
   text
  </code>
  elements was changed from
  <code>
   maxOccurs="1"
  </code>
  to
  <code>
   maxOccurs="unbounded"
  </code>
  .
 </li>
 <li>
  The
  <code>
   phDependence
  </code>
  ,
  <code>
   redoxPotential
  </code>
  and
  <code>
   temperatureDependence
  </code>
  child elements of the
  <code>
   bpcCommentGroup
  </code>
  now have a sequence of
  <code>
   text
  </code>
  child elements.
 </li>
 <li>
  The
  <code>
   note
  </code>
  child element of the
  <code>
   isoformType
  </code>
  was replaced by a sequence of
  <code>
   text
  </code>
  child elements.
 </li>
</ul>
<p>
 The XSD changes are highlighted in red color below:
</p>
<pre>&lt;xs:complexType name="commentType"&gt;
        ...
            &lt;xs:element name="text" type="evidencedStringType" minOccurs="0" <code style="color: red;">maxOccurs="unbounded"</code>/&gt;
        ...
    <code>&lt;xs:group name="bpcCommentGroup"&gt;
       ...
             &lt;xs:element name="absorption" minOccurs="0"&gt;
                ...
                        &lt;xs:element name="text" type="evidencedStringType" minOccurs="0" </code><code style="color: red;">maxOccurs="unbounded"</code>/&gt;
                ...
            <code>&lt;xs:element name="kinetics" minOccurs="0"&gt;
                ...
                        &lt;xs:element name="text" type="evidencedStringType" minOccurs="0" </code><code style="color: red;">maxOccurs="unbounded"</code>/&gt;
                ...

            <code style="color: red;">&lt;!-- The following 3 elements will in future each have a sequence of &lt;text&gt; child elements:
            &lt;xs:element name="phDependence" type="evidencedStringType" minOccurs="0"/&gt;
            &lt;xs:element name="redoxPotential" type="evidencedStringType" minOccurs="0"/&gt;
            &lt;xs:element name="temperatureDependence" type="evidencedStringType" minOccurs="0"/&gt;
            --&gt;
            &lt;xs:element name="phDependence" minOccurs="0"&gt;
                &lt;xs:complexType&gt;
                    &lt;xs:sequence&gt;
                        &lt;xs:element name="text" type="evidencedStringType" maxOccurs="unbounded"/&gt;
                    &lt;/xs:sequence&gt;
                &lt;/xs:complexType&gt;
            &lt;/xs:element&gt;
            &lt;xs:element name="redoxPotential" minOccurs="0"&gt;
                &lt;xs:complexType&gt;
                    &lt;xs:sequence&gt;
                        &lt;xs:element name="text" type="evidencedStringType" maxOccurs="unbounded"/&gt;
                    &lt;/xs:sequence&gt;
                &lt;/xs:complexType&gt;
            &lt;/xs:element&gt;
            &lt;xs:element name="temperatureDependence" minOccurs="0"&gt;
                &lt;xs:complexType&gt;
                    &lt;xs:sequence&gt;
                        &lt;xs:element name="text" type="evidencedStringType" maxOccurs="unbounded"/&gt;
                    &lt;/xs:sequence&gt;
                &lt;/xs:complexType&gt;
            &lt;/xs:element&gt;</code>
        ...
    <code>&lt;xs:complexType name="isoformType"&gt;
        ...</code>
            <code style="color: red;">&lt;!-- The &lt;note&gt; element will be replaced by a sequence of &lt;text&gt; elements:
            &lt;xs:element name="note" minOccurs="0"&gt;
                &lt;xs:complexType&gt;
                    &lt;xs:simpleContent&gt;
                        &lt;xs:extension base="xs:string"&gt;
                            &lt;xs:attribute name="evidence" type="intListType" use="optional"/&gt;
                        &lt;/xs:extension&gt;
                    &lt;/xs:simpleContent&gt;
                &lt;/xs:complexType&gt;
            &lt;/xs:element&gt;
            --&gt;
            &lt;xs:element name="text" type="evidencedStringType" minOccurs="0" maxOccurs="unbounded"/&gt;</code>
</pre>
<h3 id="Cross_references_to_BioMuta">
 Cross-references to BioMuta
</h3>
<p>
 Cross-references have been added to
 <a href="https://hive.biochemistry.gwu.edu/tools/biomuta/">
  BioMuta
 </a>
 , a curated single-nucleotide variation and disease association database.
</p>
<p>
 BioMuta is available at
 <a href="https://hive.biochemistry.gwu.edu/tools/biomuta/">
  https://hive.biochemistry.gwu.edu/tools/biomuta/
 </a>
 .
</p>
<p>
 The format of the explicit links is:
</p>
<table>
 <tr>
  <th>
   Resource abbreviation
  </th>
  <td>
   BioMuta
  </td>
 </tr>
 <tr>
  <th>
   Resource identifier
  </th>
  <td>
   Gene name.
  </td>
 </tr>
</table>
<p>
 Example:
 <a href="http://www.uniprot.org/uniprot/P02787">
  P02787
 </a>
</p>
<p>
 <a href="http://www.uniprot.org/uniprot/?query=database%3Abiomuta&amp;sort=score">
  Show all entries having a cross-reference to BioMuta.
 </a>
</p>
<h4 id="Text_format">
 Text format
</h4>
<p>
 Example:
 <a href="http://www.uniprot.org/uniprot/P02787.txt">
  P02787
 </a>
</p>
<pre>DR   BioMuta; TF; -.
</pre>
<h4 id="XML_format">
 XML format
</h4>
<p>
 Example:
 <a href="http://www.uniprot.org/uniprot/P02787.xml">
  P02787
 </a>
</p>
<pre>&lt;dbReference type="BioMuta" id="TF"/&gt;
</pre>
<h3 id="Changes_to_the__a_href___docs_humdisease__controlled_vocabulary_of_human_diseases__a_">
 Changes to the
 <a href="http://www.uniprot.org/docs/humdisease">
  controlled vocabulary of human diseases
 </a>
</h3>
<p>
 New diseases:
</p>
<ul>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-04344">
   Amelogenesis imperfecta 1A
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-04338">
   Amelogenesis imperfecta 1H
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-04343">
   Charcot-Marie-Tooth disease 2T
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-04347">
   CODAS syndrome
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-04332">
   Combined oxidative phosphorylation deficiency 23
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-04330">
   Combined oxidative phosphorylation deficiency 24
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-04341">
   Epilepsy, progressive myoclonic 8
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-04337">
   Fibrosis of extraocular muscles, congenital, 5
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-04331">
   Growth hormone deficiency, isolated partial
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-04327">
   Lethal congenital contracture syndrome 6
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-04334">
   Lissencephaly 6, with microcephaly
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-04329">
   Long QT syndrome 14
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-04328">
   Long QT syndrome 15
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-04340">
   Myasthenic syndrome, congenital, with tubular aggregates, 3
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-04339">
   Myasthenic syndrome, congenital, without tubular aggregates
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-04333">
   Myopathy, isolated mitochondrial, autosomal dominant
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-04342">
   Myopathy, vacuolar, with CASQ1 aggregates
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-04336">
   Nephronophthisis 19
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-04348">
   Pontocerebellar hypoplasia 5
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-04345">
   Spinal muscular atrophy, Jokela type
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-04335">
   Thrombocytopenia 5
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-04346">
   Ventriculomegaly with cystic kidney disease
  </a>
 </li>
</ul>
<p>
 Modified diseases:
</p>
<ul>
 <li>
  Abdominal obesity-metabolic syndrome 4 -&gt;
  <a href="http://www.uniprot.org/diseases/DI-04219">
   Lipodystrophy, familial partial, 6
  </a>
 </li>
 <li>
  Cerebral arteriopathy with subcortical infarcts and leukoencephalopathy, autosomal recessive -&gt;
  <a href="http://www.uniprot.org/diseases/DI-02549">
   Cerebral arteriopathy, autosomal recessive, with subcortical infarcts and leukoencephalopathy
  </a>
 </li>
 <li>
  Congenital fibrosis of extraocular muscles 1 -&gt;
  <a href="http://www.uniprot.org/diseases/DI-00352">
   Fibrosis of extraocular muscles, congenital, 1
  </a>
 </li>
 <li>
  Congenital fibrosis of extraocular muscles 2 -&gt;
  <a href="http://www.uniprot.org/diseases/DI-00353">
   Fibrosis of extraocular muscles, congenital, 2
  </a>
 </li>
 <li>
  Congenital fibrosis of extraocular muscles 3A -&gt;
  <a href="http://www.uniprot.org/diseases/DI-02509">
   Fibrosis of extraocular muscles, congenital, 3A
  </a>
 </li>
 <li>
  Dystonia 5 -&gt;
  <a href="http://www.uniprot.org/diseases/DI-00415">
   Dystonia, dopa-responsive
  </a>
 </li>
 <li>
  Familial progressive hyperpigmentation -&gt;
  <a href="http://www.uniprot.org/diseases/DI-02576">
   Hyperpigmentation with or without hypopigmentation, familial progressive
  </a>
 </li>
 <li>
  GTP cyclohydrolase 1 deficiency -&gt;
  <a href="http://www.uniprot.org/diseases/DI-00538">
   Hyperphenylalaninemia, BH4-deficient, B
  </a>
 </li>
 <li>
  Hyperproinsulinemia, familial -&gt;
  <a href="http://www.uniprot.org/diseases/DI-01585">
   Hyperproinsulinemia
  </a>
 </li>
 <li>
  Methylcrotonoyl-CoA carboxylase 1 deficiency -&gt;
  <a href="http://www.uniprot.org/diseases/DI-00742">
   3-methylcrotonoyl-CoA carboxylase 1 deficiency
  </a>
 </li>
 <li>
  Methylcrotonoyl-CoA carboxylase 2 deficiency -&gt;
  <a href="http://www.uniprot.org/diseases/DI-00743">
   3-methylcrotonoyl-CoA carboxylase 2 deficiency
  </a>
 </li>
 <li>
  Short stature, idiopathic, autosomal -&gt;
  <a href="http://www.uniprot.org/diseases/DI-02300">
   Growth hormone insensitivity, partial
  </a>
 </li>
 <li>
  Tetraamelia, autosomal recessive -&gt;
  <a href="http://www.uniprot.org/diseases/DI-01262">
   Tetraamelia syndrome, autosomal recessive
  </a>
 </li>
</ul>
<h3 id="Changes_to_the__a_href___docs_ptmlist__controlled_vocabulary_for_PTMs__a_">
 Changes to the
 <a href="http://www.uniprot.org/docs/ptmlist">
  controlled vocabulary for PTMs
 </a>
</h3>
<p>
 New term for the feature key 'Lipidation' ('LIPID' in the flat file):
</p>
<ul>
 <li>
  O-palmitoleyl serine
 </li>
</ul>