---
title: UniProt release 2017_08
categories: Human
---

<h2 id="Headline">
 Headline
</h2>
<h3 id="Curation_of_human_immunoglobulin_genes__a_fruitful_collaboration_between_UniProtKB_Swiss_Prot_and_IMGT__174_">
 Curation of human immunoglobulin genes: a fruitful collaboration between UniProtKB/Swiss-Prot and IMGT®
</h3>
<p>
 The existence of an agent in the blood that could neutralize diphteria toxin was
 <a href="https://www.ncbi.nlm.nih.gov/pubmed/1749380">
  reported as early as 1890
 </a>
 . Over a century after this major discovery, much is known about immunoglobulins (IG) or antibodies. They are
 <a href="https://www.ncbi.nlm.nih.gov/pubmed/20176268">
  large heterodimeric proteins
 </a>
 made up of 2 heavy (H) chains and 2 light (L) kappa or lambda chains, held together by disulfide bonds to form a 'Y' shaped molecule. Each chain comprises one variable (V) domain at the N-terminal end and one or several (for L and H, respectively) constant (C) domains. The antigen binding site is formed by the V domain of one H chain, together with that of its associated L chain. Thus, each immunoglobulin has 2 antigen binding sites with remarkable affinity for a particular antigen. Each variable domain is encoded by a variable (V) gene, a diversity (D) gene (only for H) and a joining (J) gene which are assembled by a process called
 <a href="https://www.ncbi.nlm.nih.gov/pubmed/24600447">
  V-(D)-J rearrangement
 </a>
 and can then be subjected to somatic hypermutations which, after exposure to antigen and selection, allow affinity maturation for a particular antigen. The resulting rearranged V-(D)-J genes are further spliced to C genes. The C region determines the effector properties and the mechanism used to destroy the antigen, such as activation of complement or binding to Fc receptors. An immunoglobulin is encoded by 7 genes (IGHV, IGHD, IGHJ, IGHC for the H chain and IGKV, IGKJ, IGKC for a kappa or IGLV, IGLJ or IGLC for a lambda L chain). The human genome contains 176 functional immunoglobulin genes clustered in 3 loci, IGH on chromosome 14 (50 V, 23 D, 6 J and 9 C), IGK on chromosome 2 (40 V, 5 J and 1 C) and IGL on chromosome 22 (32 V, 5 J and 5 C). During the development of B cells, the mechanisms of diversity involved in the immunoglobulin synthesis (combinatorial V-(D)-J diversity, junctional diversity and somatic hypermutations) lead to the huge potential antibody repertoire of each individual, estimated to comprise 10
 <sup>
  12
 </sup>
 different immunoglobulins, the limiting factor being only the number of B cells that an organism is genetically programmed to produce.
</p>
<p>
 In 2008, we announced the
 <a href="http://www.uniprot.org/news/2008/09/02/release">
  first draft of the complete human proteome in UniProtKB/Swiss-Prot
 </a>
 , and have been continuing to update this resource ever since. Recent work performed in collaboration with the IMGT® team has included a thorough review and update of the immunoglobulin genes, for which we now present a representative set of full-length germline immunoglobulin protein sequences. 15 entries showing the sequence of all
 <a href="http://www.uniprot.org/uniprot/?query=keyword:KW-0392+AND+keyword:KW-0181+AND+name:immunoglobulin+AND+organism:9606+AND+reviewed:yes">
  C gene products
 </a>
 and 122 representing all
 <a href="http://www.uniprot.org/uniprot/?query=keyword:KW-0394+AND+keyword:KW-0181+AND+name:immunoglobulin+AND+organism:9606+AND+reviewed:yes">
  V gene products
 </a>
 are now publicly available. These entries can be retrieved with the keyword
 <a href="http://www.uniprot.org/keywords/KW-0392">
  'Immunoglobulin C region'
 </a>
 and
 <a href="http://www.uniprot.org/keywords/KW-0394">
  'Immunoglobulin V region'
 </a>
 , respectively. D and J gene products are extremely small, with an average of 5 amino acids for D genes and 15-30 for J. In other words, they are too short to be informative on their own. Therefore we have decided to curate a
 <a href="http://www.uniprot.org/uniprot/P0DOY5">
  single peptide representative of D gene products
 </a>
 and 3 of
 <a href="http://www.uniprot.org/uniprot/?query=accession:A0A0C4DH62+OR+accession:A0A0A0MT89+OR+accession:A0A0A0MT76">
  J gene products
 </a>
 , one for H chains and 2 for L chains kappa and lambda. As for other human proteins, the sequences shown match the translation of the reference genome (Genome Reference Consortium GRCh38/hg38). The nomenclature used is the official one from IMGT/GENE-DB, approved by HGNC and endorsed by NCBI Gene and the IUIS-Nomenclature SubCommittee. Cross-references were implemented in the 141 UniProtKB/Swiss-Prot immunoglobulin entries, providing direct access to the
 <a href="https://www.ncbi.nlm.nih.gov/pubmed/25378316">
  dedicated IMGT® resource
 </a>
 and its comprehensive sequence repertoire, which currently describes 927 alleles from 462 functional and non-functional genes together with a wealth of additional information concerning immunoglobulins. Reciprocal links to UniProtKB from
 <a href="http://www.imgt.org">
  IMGT®
 </a>
 ensure easy navigation between both resources.
</p>
<p>
 We also provide several examples of
 <a href="http://www.uniprot.org/uniprot/?query=(keyword:KW-0392+AND+reviewed:yes+AND+organism:9606)+NOT+keyword:KW-0181">
  full-length rearranged immunoglobulins
 </a>
 . Among the 10
 <sup>
  12
 </sup>
 predicted sequences, we have selected some of those that have been entirely sequenced at the amino acid level. However, the representation of the full repertoire is beyond the scope of our knowledgebase and UniProtKB users interested in these complex molecules are advised to visit
 <a href="http://www.imgt.org">
  IMGT®
 </a>
 .
</p>
<p>
 We would like take this opportunity to thank Marie-Paule Lefranc, Sofia Kossida and the IMGT® team for this fruitful collaboration, which is beneficial not only for both resources, but hopefully also for the scientific community as a whole.
</p>
<h3 id="Cross_references_to_ELM">
 Cross-references to ELM
</h3>
<p>
 Cross-references have been added to the Eukaryotic Linear Motif (ELM) resource for functional sites in proteins.
</p>
<p>
 ELM is available at
 <a href="http://elm.eu.org">
  http://elm.eu.org
 </a>
 .
</p>
<p>
 The format of the explicit links is:
</p>
<table>
 <tr>
  <th>
   Resource abbreviation
  </th>
  <td>
   ELM
  </td>
 </tr>
 <tr>
  <th>
   Resource identifier
  </th>
  <td>
   UniProtKB accession number
  </td>
 </tr>
</table>
<p>
 Example:
 <a href="http://www.uniprot.org/uniprot/P12931#cross%5Freferences">
  P12931
 </a>
</p>
<p>
 <a href="http://www.uniprot.org/uniprot/?query=database:elm&amp;sort=score">
  Show all entries having a cross-reference to ELM.
 </a>
</p>
<h4 id="Text_format">
 Text format
</h4>
<p>
 Example:
 <a href="http://www.uniprot.org/uniprot/P12931.txt">
  P12931
 </a>
</p>
<pre>DR   ELM; P12931; -.
</pre>
<h4 id="XML_format">
 XML format
</h4>
<p>
 Example:
 <a href="http://www.uniprot.org/uniprot/P12931.xml">
  P12931
 </a>
</p>
<pre>&lt;dbReference type="ELM" id="P12931"/&gt;
</pre>
<h4 id="RDF_format">
 RDF format
</h4>
<p>
 Example:
 <a href="http://www.uniprot.org/uniprot/P12931.ttl">
  P12931
 </a>
</p>
<pre>uniprot:P12931
  rdfs:seeAlso &lt;http://purl.uniprot.org/elm/P12931&gt; .
&lt;http://purl.uniprot.org/elm/P12931&gt;
  rdf:type up:Resource ;
  up:database &lt;http://purl.uniprot.org/database/ELM&gt; .
</pre>
<h3 id="Changes_to_the__a_href___docs_humdisease__controlled_vocabulary_of_human_diseases__a_">
 Changes to the
 <a href="http://www.uniprot.org/docs/humdisease">
  controlled vocabulary of human diseases
 </a>
</h3>
<p>
 New diseases:
</p>
<ul>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-05002">
   46,XX sex reversal 4
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-05003">
   Adrenal insufficiency, NR5A1-related
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-04998">
   Arthrogryposis multiplex congenita, neurogenic, with myelin defect
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-05005">
   Cardiac valvular defect, developmental
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-05007">
   Cleft palate, cardiac defects, and mental retardation
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-04997">
   Intellectual developmental disorder with dysmorphic facies, seizures, and distal limb anomalies
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-04996">
   Intellectual developmental disorder with gastrointestinal difficulties and high pain threshold
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-05004">
   Neurodevelopmental disorder with microcephaly, hypotonia, and variable brain anomalies
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-05008">
   Pancreatic lipase deficiency
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-05006">
   Retinitis pigmentosa with or without skeletal anomalies
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-04999">
   Specific granule deficiency 1
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-05000">
   Specific granule deficiency 2
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-05001">
   Structural heart defects and renal anomalies syndrome
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-04995">
   Townes-Brocks syndrome 2
  </a>
 </li>
</ul>
<p>
 Modified diseases:
</p>
<ul>
 <li>
  2-methyl-3-hydroxybutyryl-CoA dehydrogenase deficiency -&gt;
  <a href="http://www.uniprot.org/diseases/DI-00001">
   HDS10 mitochondrial disease
  </a>
 </li>
 <li>
  Cataract 23 -&gt;
  <a href="http://www.uniprot.org/diseases/DI-01874">
   Cataract 23, multiple types
  </a>
 </li>
 <li>
  Cataract 30 -&gt;
  <a href="http://www.uniprot.org/diseases/DI-03825">
   Cataract 30, multiple types
  </a>
 </li>
 <li>
  Cataract 33 -&gt;
  <a href="http://www.uniprot.org/diseases/DI-01235">
   Cataract 33, multiple types
  </a>
 </li>
 <li>
  Hereditary hyperferritinemia-cataract syndrome -&gt;
  <a href="http://www.uniprot.org/diseases/DI-01718">
   Hyperferritinemia with or without cataract
  </a>
 </li>
 <li>
  Leukoencephalopathy, with dystonia and motor neuropathy -&gt;
  <a href="http://www.uniprot.org/diseases/DI-02987">
   Leukoencephalopathy with dystonia and motor neuropathy
  </a>
 </li>
 <li>
  Townes-Brocks syndrome -&gt;
  <a href="http://www.uniprot.org/diseases/DI-02376">
   Townes-Brocks syndrome 1
  </a>
 </li>
</ul>
<p>
 Deleted diseases
</p>
<ul>
 <li>
  Mental retardation, X-linked, syndromic, 10
 </li>
</ul>
<h3 id="Changes_in__a_href___docs_subcell__subcellular_location_controlled_vocabulary__a_">
 Changes in
 <a href="http://www.uniprot.org/docs/subcell">
  subcellular location controlled vocabulary
 </a>
</h3>
<p>
 New subcellular locations:
</p>
<ul>
 <li>
  <a href="http://www.uniprot.org/locations/SL-0491">
   Anammoxosome
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/locations/SL-0492">
   Anammoxosome membrane
  </a>
 </li>
</ul>
<h2 id="UniParc_news">
 UniParc news
</h2>
<h3 id="UniParc_XSD_change_for_InterPro_annotations">
 UniParc XSD change for InterPro annotations
</h3>
<p>
 To reduce the sequence redundancy in UniProtKB, we apply a procedure to
 <a href="http://www.uniprot.org/help/proteome%5Fredundancy">
  identify highly redundant proteomes
 </a>
 within selected species groups to exclude them from UniProtKB. Their sequences are still available for download from the
 <a href="http://www.uniprot.org/uniparc">
  UniParc
 </a>
 sequence archive, which stores protein sequences that are 100% identical and the same length in a single record, with cross-references to the source database where the protein exists. UniParc also includes basic annotation data (taxonomy,
 <a href="http://www.uniprot.org/news/2014/07/09/release">
  gene and protein names
 </a>
 ,
 <a href="http://www.uniprot.org/news/2015/03/04/release">
  proteome identifier and component
 </a>
 ) to allow users interested in redundant proteomes to retrieve meaningful data sets, and we have now further enhanced UniParc with
 <a href="http://www.ebi.ac.uk/interpro/">
  InterPro
 </a>
 annotations and for this purpose extended the UniParc XSD with new elements and types as shown below in red color:
</p>
<pre>&lt;xs:element name="entry"&gt;
  &lt;xs:complexType&gt;
   &lt;xs:sequence&gt;
       ...
    <code style="color: red;">&lt;xs:element name="signatureSequenceMatch" type="seqFeatureType" minOccurs="0" maxOccurs="unbounded"/&gt;</code>
                ...
   <code>&lt;/xs:sequence&gt;</code>
            ...
  <code>&lt;/xs:complexType&gt;
 &lt;/xs:element&gt;</code>
    ...
    <code style="color: red;">&lt;xs:complexType name="seqFeatureType"&gt;
        &lt;xs:sequence&gt;
   &lt;xs:element name="ipr" type="seqFeatureGroupType" minOccurs="0" maxOccurs="1"/&gt;
   &lt;xs:element name="lcn" type="locationType" minOccurs="1" maxOccurs="unbounded"/&gt;
        &lt;/xs:sequence&gt;
        &lt;xs:attribute name="database" type="xs:string" use="required"/&gt;
        &lt;xs:attribute name="id" type="xs:string" use="required"/&gt;
    &lt;/xs:complexType&gt;</code>

    <code style="color: red;">&lt;xs:complexType name="seqFeatureGroupType"&gt;
        &lt;xs:attribute name="name" type="xs:string"/&gt;
        &lt;xs:attribute name="id" type="xs:string" use="required"/&gt;
    &lt;/xs:complexType&gt;</code>

 <code style="color: red;">&lt;xs:complexType name="locationType"&gt;
  &lt;xs:attribute name="start" type="xs:int" use="required"/&gt;
  &lt;xs:attribute name="end" type="xs:int" use="required"/&gt;
 &lt;/xs:complexType&gt;</code>
</pre>