---
title: UniProt release 2018_08
---

## Headline

### Human brain development: slow and steady wins the race

As mammals, we share most of our physiological processes with other animals and these similarities allow the wide use of model organisms for medical research purposes. Yet there is something special about us, abstract thought, creativity, art, culture, something linked to our big brains. The increase in size and complexity of our cerebral cortex happened recently on the evolution time scale, i.e. after the *Homo* lineage split apart from that of other related primates. In order to try to understand this distinctive feature of ours, several new [human-specific genes involved in corticogenesis](https://www.ncbi.nlm.nih.gov/pubmed/29856955,27584858) have been identified. They are produced by [segmental duplications](https://www.ncbi.nlm.nih.gov/pubmed/27584858) , but their functional impact on brain development remains mysterious. However, among these genes are [3 nearly identical NOTCH2 paralogs](https://www.ncbi.nlm.nih.gov/pubmed/29856954,29856955) , called [NOTCH2NLA, B and C](http://www.uniprot.org/uniprot/?query=accession:q7z3s9+OR+accession:p0dpk3+OR+accession:p0dpk4) for which functional clues have been recently obtained. The evolutionary history of NOTCH2NL genes is peculiar. [NOTCH2 partial duplication occurred](https://www.ncbi.nlm.nih.gov/pubmed/29856954) prior to the last common ancestor of human, chimpanzee, and gorilla (some 14 million years ago) leading to the creation of a truncated inactive copy, called NOTCH2NL (standing for Notch homolog 2 N-terminal-like). In the hominin lineage, some 3 to 4 million years ago, the NOTCH2 dopplegänger was repaired by gene conversion and duplicated, creating 3 new human-specific active genes NOTCH2NLA, B and C. This timeframe corresponds to the early stages of the expansion of the human neocortex.

NOTCH2NLA, B and C are expressed in radial glia neural stem cells during cortical development. These cells undergo multiple cycles of regenerative, mostly asymmetric, cell divisions, leading to the generation of diverse types of neurons while maintaining a pool of progenitors. NOTCH2NL gene expression activates the NOTCH signaling pathway, down-regulates neuronal differentiation genes, and delays the differentiation of neuronal progenitors, increasing their number, all of which ultimately results in an increase in neurons. In this context, slow development produces a huge benefit.

The chromosome 1q21.1 region hosting NOTCH2NL genes is associated with [chromosome 1q21.1 deletion](https://www.omim.org/entry/612474) / [duplication syndromes](https://www.omim.org/entry/612475) , where duplications are associated with macrocephaly and autism, and deletions with microcephaly and schizophrenia. [11 patients were analyzed](https://www.ncbi.nlm.nih.gov/pubmed/29856954) : those with microcephaly had NOTCH2NLA and/or NOTCH2NLB deletions, while the macrocephaly cases were consistent with NOTCH2NLA and/or NOTCH2NLB duplications. If confirmed, these results are consistent with a crucial role for NOTCH2NL genes in human neocortex development. [Thus, the emergence of human-specific NOTCH2NL genes may have contributed to the rapid evolution of the larger human neocortex, at the expense of susceptibility to recurrent neurodevelopmental disorders](https://www.ncbi.nlm.nih.gov/pubmed/29856954) .

Using our big brains, we have annotated all [3 NOTCH2NL gene products](http://www.uniprot.org/uniprot/?query=accession:q7z3s9+OR+accession:p0dpk3+OR+accession:p0dpk4) in UniProtKB/Swiss-Prot and they are publicly available as of this release.

## UniProtKB news

### Change of the annotation topic 'Enzyme regulation' to 'Activity regulation'

In UniProtKB entries, the topic 'Enzyme regulation' was used to display information about factors that regulate the activity of enzymes, but also of transporters and microbial transcription factors. To clarify the situation, we have renamed this topic to 'Activity regulation'.

#### Text format

Example: [P02730](http://www.uniprot.org/uniprot/P02730.txt)

Previous format:

    CC   -!- ENZYME REGULATION: Phenyl isothiocyanate inhibits anion transport
    CC       in vitro.

New format:

    CC   -!- ACTIVITY REGULATION: Phenyl isothiocyanate inhibits anion transport
    CC       in vitro.

#### XML format

Example: [P02730](http://www.uniprot.org/uniprot/P02730.xml)

Previous format:

    <comment type="enzyme regulation">
      <text>Phenyl isothiocyanate inhibits anion transport in vitro.</text>
    </comment>

New format:

    <comment type="activity regulation">
      <text>Phenyl isothiocyanate inhibits anion transport in vitro.</text>
    </comment>

#### RDF format

Example: [P02730](http://www.uniprot.org/uniprot/P02730.ttl)

Previous format:

    uniprot:P02730
      up:annotation <P02730#SIPC58AB4FDB0DD7DCA> .

    <P02730#SIPC58AB4FDB0DD7DCA>
      rdf:type up:Enzyme_Regulation_Annotation ;
      rdfs:comment "Phenyl isothiocyanate inhibits anion transport in vitro." .

New format:

    uniprot:P02730
      up:annotation <P02730#SIPC58AB4FDB0DD7DCA> .

    <P02730#SIPC58AB4FDB0DD7DCA>
      rdf:type up:Activity_Regulation_Annotation ;
      rdfs:comment "Phenyl isothiocyanate inhibits anion transport in vitro." .

### Change to the cross-references to Bgee

We have introduced an additional field in the cross-references to the [Bgee](https://bgee.org) database to indicate the expression pattern of the gene.

#### Text format

Example: [P10361](http://www.uniprot.org/uniprot/P10361.txt)

    DR   Bgee; ENSRNOG00000010756; Expressed in 10 organ(s), highest expression level in spleen.

#### XML format

Example: [P10361](http://www.uniprot.org/uniprot/P10361.xml)

    <dbReference type="Bgee" id="ENSRNOG00000010756">
      <property type="expression patterns" value="Expressed in 10 organ(s), highest expression level in spleen"/>
    </dbReference>

This change does not affect the XSD, but may nevertheless require code changes.

#### RDF format

Example: [P10361](http://www.uniprot.org/uniprot/P10361.ttl)

    uniprot:P10361
      rdfs:seeAlso <http://purl.uniprot.org/bgee/ENSRNOG00000010756> .
    <http://purl.uniprot.org/bgee/ENSRNOG00000010756>
      rdf:type up:Resource ;
      up:database <http://purl.uniprot.org/database/Bgee> ;
      rdfs:comment "Expressed in 10 organ(s), highest expression level in spleen" .

### Changes to the [controlled vocabulary of human diseases](http://www.uniprot.org/docs/humdisease)

New diseases:

-   [Combined oxidative phosphorylation deficiency 36](http://www.uniprot.org/diseases/DI-05238)
-   [Congenital disorder of glycosylation with defective fucosylation](http://www.uniprot.org/diseases/DI-05266)
-   [Deafness, autosomal recessive, 26](http://www.uniprot.org/diseases/DI-05262)
-   [Deafness, autosomal recessive, 57](http://www.uniprot.org/diseases/DI-05260)
-   [Deafness, autosomal recessive, 109](http://www.uniprot.org/diseases/DI-05261)
-   [Deafness, congenital heart defects, and posterior embryotoxon](http://www.uniprot.org/diseases/DI-05252)
-   [Developmental delay, intellectual disability, obesity, and dysmorphic features](http://www.uniprot.org/diseases/DI-05259)
-   [Ehlers-Danlos syndrome, classic-like, 2](http://www.uniprot.org/diseases/DI-05255)
-   [Epileptic encephalopathy, early infantile, 63](http://www.uniprot.org/diseases/DI-05248)
-   [Epileptic encephalopathy, early infantile, 64](http://www.uniprot.org/diseases/DI-05265)
-   [Hyperostosis cranialis interna](http://www.uniprot.org/diseases/DI-05257)
-   [Jaberi-Elahi syndrome](http://www.uniprot.org/diseases/DI-05251)
-   [Leukodystrophy, hypomyelinating, 15](http://www.uniprot.org/diseases/DI-05239)
-   [Leukodystrophy, hypomyelinating, 16](http://www.uniprot.org/diseases/DI-05245)
-   [Microcephaly 21, primary, autosomal recessive](http://www.uniprot.org/diseases/DI-05234)
-   [Microcephaly 22, primary, autosomal recessive](http://www.uniprot.org/diseases/DI-05235)
-   [Microcephaly 23, primary, autosomal recessive](http://www.uniprot.org/diseases/DI-05236)
-   [Multiple mitochondrial dysfunctions syndrome 6](http://www.uniprot.org/diseases/DI-05241)
-   [Neurodevelopmental disorder with spastic quadriplegia and brain abnormalities with or without seizures](http://www.uniprot.org/diseases/DI-05249)
-   [Oocyte maturation defect 5](http://www.uniprot.org/diseases/DI-05264)
-   [Osteogenesis imperfecta 18](http://www.uniprot.org/diseases/DI-05240)
-   [Premature ovarian failure 14](http://www.uniprot.org/diseases/DI-05263)
-   [Rh-null, amorph type](http://www.uniprot.org/diseases/DI-05246)
-   [Short-rib thoracic dysplasia 7/20 with polydactyly, digenic](http://www.uniprot.org/diseases/DI-05258)
-   [Spermatogenic failure 24](http://www.uniprot.org/diseases/DI-05256)
-   [Spermatogenic failure 25](http://www.uniprot.org/diseases/DI-05242)
-   [Spermatogenic failure 26](http://www.uniprot.org/diseases/DI-05243)
-   [Spermatogenic failure 27](http://www.uniprot.org/diseases/DI-05244)
-   [Spinocerebellar ataxia 47](http://www.uniprot.org/diseases/DI-05237)
-   [Spondyloepimetaphyseal dysplasia, Di Rocco type](http://www.uniprot.org/diseases/DI-05247)
-   [Tumoral calcinosis, hyperphosphatemic, familial, 2](http://www.uniprot.org/diseases/DI-05253)
-   [Tumoral calcinosis, hyperphosphatemic, familial, 3](http://www.uniprot.org/diseases/DI-05254)
-   [Ververi-Brady syndrome](http://www.uniprot.org/diseases/DI-05250)

Deleted diseases

-   Ehlers-Danlos syndrome 7B

## UniProt website news

### New advanced search interface

We have revamped the advanced search interface to make it easier for you to browse the different search fields and options within the dropdown menus. Most importantly, there is now a search box right at the top when you open the blue dropdown menu that allows you to type a concept name (e.g. "structure") and receive some autocompleted suggestions from which you can then select the most suitable one:

![image](https://github.com/ebi-uniprot/uniprot-manual/raw/main/images/2018-09-12-release-5.png)

### Automatic gene-centric isoform mapping for eukaryotic reference proteome entries

Some proteomes have been (manually and algorithmically) selected as [reference proteomes](http://www.uniprot.org/help/reference%5Fproteome) . They cover well-studied model organisms and other organisms of interest for biomedical research and phylogeny. In this context, we provide [data sets for reference proteomes](ftp://ftp.uniprot.org/pub/databases/uniprot/current%5Frelease/knowledgebase/reference%5Fproteomes) where only one form of a protein, usually the best annotated version in UniProtKB, is present. The relationships identified when generating these data sets are now also used when displaying individual entries on the UniProt website:

A single gene can code for multiple proteins through biological events such as [alternative splicing, initiation and promoter usage](http://www.uniprot.org/help/alternative%5Fproducts) . While the UniProtKB/Swiss-Prot expert curation process includes the identification and review of different forms of a protein and their description in a single UniProtKB/Swiss-Prot entry, its focus is the functional annotation of proteins. For this reason, not all potential isoforms of a protein that are available in UniProtKB/TrEMBL can be reviewed and merged into a single entry. This results in a larger number of UniProtKB entries than genes for many of the eukaryotic reference proteomes. In order to identify potential isoforms that have not (yet) been reviewed by a biocurator, we have established an **automatic [gene-centric mapping](http://www.uniprot.org/help/gene%5Fcentric%5Fisoform%5Fmapping) between entries from eukaryotic reference proteomes that are likely to belong to the same gene** . This mapping is based on gene identifiers from Ensembl, EnsemblGenomes and model organism databases and, in cases where none of these are available, on gene names assigned by the original sequencing projects.

Example: [Q15286](http://www.uniprot.org/uniprot/Q15286#sequences)
