<div>
 <h4 id="Standard_operating_procedure__SOP__for_UniProt_manual_curation">
  Standard operating procedure (SOP) for UniProt manual curation
 </h4>
 <p>
  This document describes the manual curation procedure used by the UniProt Consortium members. The UniProt manual curation process comprises manual review of results from a range of sequence analysis programs and literature curation of experimental data as well as attribution of all information to its original source.
 </p>
 <p>
  UniProtKB/Swiss-Prot document:
  <a href="http://www.uniprot.org/docs/sop%5Fmanual%5Fcuration.pdf">
   Standard operating procedure (SOP) for UniProt manual curation
  </a>
  <br/>
  UniProt web page:
  <a href="http://www.uniprot.org/program/">
   UniProt manual annotation program
  </a>
 </p>
 <p>
  <a href="http://www.uniprot.org/help/manual%5Fcuration">
   How do we manually annotate a UniProtKB entry?
  </a>
  <br/>
  <a href="http://www.uniprot.org/help/uniprotkb%5Fsections">
   Why is UniProtKB composed of 2 sections, UniProtKB/Swiss-Prot and UniProtKB/TrEMBL?
  </a>
 </p>
 <h4 id="Protein_naming_guidelines">
  Protein naming guidelines
 </h4>
 <p>
  Ambiguities regarding gene/protein names are a major problem in the literature and in the sequence databases which tend to propagate the confusion. As administrators of UniProt we feel that we can play a major role in standardization of protein nomenclature.
 </p>
 <p>
  The European Bioinformatics Institute (EMBL-EBI), the National Center for Biotechnology Information (NCBI), the Protein Information Resource (PIR) and the Swiss Institute for Bioinformatics (SIB) have worked together to produce a shared set of protein naming guidelines. These guidelines are intended for use by anyone who wants to name a protein and aim to promote consistent nomenclature which is indispensable for communication, literature searching and data retrieval:
 </p>
 <p>
  UniProtKB/Swiss-Prot document:
  <a href="http://www.uniprot.org/docs/International%5FProtein%5FNomenclature%5FGuidelines.pdf">
   International protein nomenclature guidelines
  </a>
 </p>
 <p>
  User manual:
  <a href="http://www.uniprot.org/manual/protein%5Fnames">
   Protein names
  </a>
 </p>
 <p>
  <a href="http://www.uniprot.org/help/different%5Fprotein%5Fgene%5Fnames">
   Why does the UniProtKB use so many different names for the same protein?
  </a>
 </p>
 <p>
  The document
  <a href="http://www.uniprot.org/docs/nomlist">
   'Protein nomenclature publication list'
  </a>
  lists references that are important in defining the nomenclature or terminology relative to proteins in general and in particular on specific family or groups of proteins.
 </p>
 <h4 id="Criteria_description_for_protein_existence">
  Criteria description for protein existence
 </h4>
 <p>
  Some of protein sequences exhibit strong similarity to known proteins in closely related species. For other proteins there is experimental evidence, such as Edman sequencing, clear identification by mass spectrometry (MSI), X-ray or NMR structure, detection by antibodies, etc. However, for some other proteins, there is no evidence at all. To indicate these different levels of evidence for the existence of a protein, we have introduced a PE (Protein Existence) level. The following document lists the criteria used to assign a protein existence (PE) level to entries.
 </p>
 <p>
  UniProtKB/Swiss-Prot document:
  <a href="http://www.uniprot.org/docs/pe%5Fcriteria">
   Criteria used to assign the PE level of entries
  </a>
  <br/>
  User manual:
  <a href="http://www.uniprot.org/manual/protein%5Fexistence">
   Protein existence
  </a>
 </p>
 <p>
  <a href="http://www.uniprot.org/help/dubious%5Fsequences">
   Why do we keep dubious sequences in UniProtKB? How to discard them from a protein set?
  </a>
 </p>
</div>