<div>
 <p>
  <strong>
   What is the canonical sequence?
  </strong>
 </p>
 <p>
  Each UniProtKB/Swiss-Prot entry contains all curated protein products encoded by a given gene in a given species or strain. For each UniProtKB/Swiss-Prot entry, we choose a canonical (or representative) sequence for display that should conform to at least one of the following criteria:
 </p>
 <ol>
  <li>
   It is functional;
  </li>
  <li>
   It is widely expressed;
  </li>
  <li>
   It is encoded by conserved exons found in orthologous sequences;
  </li>
  <li>
   It is identical to consensus sequences chosen by other resources and genome curation efforts such as CCDS and MANE (see also
   <a href="http://www.uniprot.org/help/human%5Fproteome">
    UniProt's human proteome
   </a>
   )
  </li>
  <li>
   In the absence of any information, we choose the longest sequence.
  </li>
 </ol>
 <p>
  Sequences chosen according to these criteria generally allow the description of the majority of functionally important
  <a href="http://www.uniprot.org/help/domain">
   domains
  </a>
  ,
  <a href="http://www.uniprot.org/help/motif">
   motifs
  </a>
  ,
  <a href="http://www.uniprot.org/help/site">
   sites
  </a>
  , and
  <a href="http://www.uniprot.org/help/ptm%5Fprocessing%5Fsection">
   post-translational modifications
  </a>
  , naturally occurring
  <a href="http://www.uniprot.org/help/variant">
   variants
  </a>
  with functional and clinical significance, and other
  <a href="http://www.uniprot.org/help/sequence%5Fannotation">
   sequence features
  </a>
  .
 </p>
 <p>
  Additional information can be found in the
  <a href="http://www.uniprot.org/help/var%5Fseq">
   'Alternative sequence'
  </a>
  subsection.
 </p>
 <p>
  The various UniProtKB distribution formats (Flat Text, XML, RDF/XML) display only the canonical sequence. The website's
  <a href="http://www.uniprot.org/help/sequences%5Fsection">
   'Sequences'
  </a>
  section displays the canonical sequence, but for convenience it offers also a view of the isoforms that are described in the 'Alternative sequence' subsection.
 </p>
 <p>
  <strong>
   Are all isoforms described in one UniProtKB/Swiss-Prot entry?
  </strong>
 </p>
 <p>
  Whenever possible, all the protein products encoded by one gene in a given species are described in a single UniProtKB/Swiss-Prot entry, including isoforms generated by alternative splicing, alternative promoter usage, and alternative translation initiation (*). However, some alternative splicing isoforms derived from the same gene share only a few exons, if any at all, the same for some 'trans-splicing' events. In these cases, the divergence is obviously too important to merge all protein sequences into a single entry and the isoforms have to be described in separate 'external' entries.
 </p>
 <p>
  Example:
  <a href="http://www.uniprot.org/uniprot/P42284#sequences">
   isoforms derived from the lola gene (Drosophila melanogaster)
  </a>
 </p>
 <p>
  (*) Important remark: Due to the increase of sequence data coming from large-scale sequencing projects, UniProtKB/TrEMBL may contain additional predicted sequences encoded by genes which are described in a UniProtKB/Swiss-Prot entry.
 </p>
 <p>
  <strong>
   How can I retrieve isoform sequences?
  </strong>
 </p>
 <p>
  Alternative sequences, described in either single or separate entries, are all available for Blast searches.
 </p>
 <p>
  Isoform sequences can be downloaded in FASTA format from our
  <a href="http://www.uniprot.org/downloads">
   FTP download index page
  </a>
  (choose the file: 'Isoform sequences').
 </p>
 <p>
  Query-derived sets of canonical sequences alone or canonical and isoform sequences can also be downloaded in FASTA format (see
  <a href="http://www.uniprot.org/help/retrieve%5Fsets">
   How to retrieve sets of UniProtKB protein sequences?
  </a>
  ).
 </p>
 <p>
  See also:
 </p>
 <ul>
  <li>
   <a href="http://www.uniprot.org/help/cds%5Fprotein%5Fdefinition">
    What are UniProtKB's criteria for defining a CDS as a protein?
   </a>
  </li>
  <li>
   <a href="http://www.uniprot.org/help/protein%5Fdiversity">
    How are protein sequence variety and protein diversity represented in UniProtKB?
   </a>
  </li>
  <li>
   <a href="http://www.uniprot.org/manual/sequence%5Flength">
    Sequence length
   </a>
  </li>
  <li>
   <a href="http://www.uniprot.org/manual/alternative%5Fproducts">
    Alternative products
   </a>
  </li>
  <li>
   <a href="http://www.uniprot.org/manual/var%5Fseq">
    Alternative sequence
   </a>
  </li>
  <li>
   <a href="http://www.uniprot.org/manual/conflict">
    Sequence conflict
   </a>
  </li>
 </ul>
</div>