<div>
 <p>
  <strong>
   How do I get the nucleotide sequence that corresponds to the canonical UniProtKB sequence?
  </strong>
 </p>
 <p>
  You cannot! Although more than 95% of the known protein sequences derive from DNA translation, there is no
  <strong>
   single
  </strong>
  nucleic acid reference sequence for a given UniProtKB/Swiss-Prot protein sequence.
 </p>
 <p>
  The
  <a href="http://www.uniprot.org/help/canonical%5Fand%5Fisoforms">
   canonical protein sequence
  </a>
  is the outcome of thorough curation work, which often involves the merge of various sequences encoded by the same gene (in one species). In the annotation process, the most correct amino acid sequences are chosen and discrepancies are analyzed and documented, most frequently in the 'Sequence' section (in the flat file FT keys: VAR_SEQ, CONFLICT, VARIANT, ...). Additional information can be found in the 'Alternative sequence' subsection.
 </p>
 <p>
  The annotated coding sequences (CDS) submitted to the EMBL-Bank/GenBank/DDBJ databases that show the most severe discrepancies with the UniProtKB/Swiss-Prot canonical sequence are tagged in the 'Sequence databases' subsection of the
  <a href="http://www.uniprot.org/help/cross%5Freferences%5Fsection">
   Cross-reference section
  </a>
  (EMBL-Bank/GenBank/DDBJ) as frameshifts, erroneous prediction, etc., with an optional comment in the subsection 'Sequence caution'.
 </p>
 <p>
  In most cases, in the absence of annotation of 'Sequence conflict', 'Variant', 'Alternative products', 'Sequence caution', etc., the nucleic acid sequences listed in the 'Sequence databases' subsection, when translated, correspond to the canonical sequence, provided they are not fragments.
 </p>
 <p>
  There are some exceptions. Some entries may display a full-length protein sequence, while the available nucleic acid sequences are only fragments. In this case, the manual annotation process comes into the picture: by combining the sequences of submitted fragmentary CDS and/or of the genomic sequence, taking into account the similarity with orthologous proteins (in a closely related species), one can predict the full-length protein sequence.
 </p>
 <p>
  In addition, some
  <a href="http://www.uniprot.org/uniprot/?query=reviewed:yes+keyword:KW-0903+NOT+database:embl">
   entries
  </a>
  exclusively contain protein sequence data resulting from direct protein sequencing and thus do not have cross-references to any nucleic acid sequence.
 </p>
 <p>
  <strong>
   How do I get the nucleotide sequence that corresponds to the UniProtKB alternative sequences?
  </strong>
 </p>
 <p>
  There is currently no way to directly find the nucleotide sequence that corresponds to the alternative protein sequences described in a UniProtKB entry.
 </p>
 <p>
  However, some of the resources to which we link contain information that is specific to an isoform sequence. Where this is known, we indicate the corresponding UniProtKB
  <a href="http://www.uniprot.org/help/isoform%5Fcrossreferences">
   isoform sequence identifier in cross-references
  </a>
  .
 </p>
 <p>
  See also:
 </p>
 <ul>
  <li>
   <a href="http://www.uniprot.org/help/canonical%5Fand%5Fisoforms">
    What is the canonical sequence? Are all isoforms described in one entry?
   </a>
  </li>
  <li>
   <a href="http://www.uniprot.org/help/redundancy">
    How redundant are the UniProt databases?
   </a>
  </li>
  <li>
   <a href="http://www.uniprot.org/manual/sequences">
    Sequences
   </a>
  </li>
  <li>
   <a href="http://www.uniprot.org/manual/var%5Fseq">
    Alternative sequence
   </a>
  </li>
  <li>
   <a href="http://www.uniprot.org/manual/conflict">
    Sequence conflict
   </a>
  </li>
  <li>
   <a href="http://www.uniprot.org/help/cross%5Freferences%5Fsection">
    Cross-references section
   </a>
  </li>
  <li>
   <a href="http://www.uniprot.org/help/isoform%5Fcrossreferences">
    Cross-references for isoform sequences
   </a>
  </li>
 </ul>
</div>