<div>
 <p>
  The Chordata protein annotation project focuses on the manual annotation of chordata-specific proteins as well as those that are widely conserved. The aim of this project is twofold: 1. to keep the existing human entries up-to-date and 2. to broaden the manual annotation to other vertebrate species, especially model organisms, including great apes, cow, mouse, rat, chicken, zebrafish, as well as
  <em>
   Xenopus laevis
  </em>
  and
  <em>
   Xenopus tropicalis
  </em>
  .
 </p>
 <p>
  See:
  <a href="http://www.uniprot.org/faq/45">
   How do we manually annotate a UniProtKB entry?
  </a>
 </p>
 <h3 id="Update_of_the_human_proteome">
  Update of the human proteome
 </h3>
 <p>
  A draft of the
  <a href="http://www.uniprot.org/uniprot/?query=organism:9606+AND+keyword:kW-1185+AND+reviewed:yes">
   human proteome
  </a>
  has been available in UniProtKB/Swiss-Prot since 2008 and one of the current priorities of the Chordata protein annotation project is to improve the quality of human sequences provided.
 </p>
 <p>
  See:
  <a href="http://www.uniprot.org/faq/48">
   What is UniProt's human proteome?
  </a>
 </p>
 <p>
  To this aim, we are updating sequences which show discrepancies with those predicted from the genome sequence. Dubious isoforms, sequences based on experimental artefacts and protein products derived from erroneous gene model predictions are also revisited. This work is in part done in collaboration with the Hinxton Sequence Forum (HSF), which allows active exchange between UniProt,
  <a href="http://www.sanger.ac.uk/HGP/havana/">
   HAVANA
  </a>
  ,
  <a href="http://www.ensembl.org/index.html">
   Ensembl
  </a>
  and
  <a href="http://www.genenames.org/">
   HGNC
  </a>
  groups, as well as with
  <a href="http://www.ncbi.nlm.nih.gov/RefSeq/">
   RefSeq database
  </a>
  . UniProt is a member of the
  <a href="http://www.ncbi.nlm.nih.gov/projects/CCDS/CcdsBrowse.cgi">
   Consensus CDS project
  </a>
  and we are in the process of reviewing our records to support convergence towards a standard set of protein annotation.
 </p>
 <p>
  We also continuously update human entries with functional annotation, including novel structural, post-translational modification, interaction and enzymatic activity data. In order to identify candidates for re-annotation, we use, among others, information extraction tools such as the
  <a href="http://string.embl.de/">
   STRING database
  </a>
  . In addition, we regularly add new sequence variants and maintain disease information. Indeed, this annotation project includes the Variation Annotation project, the goal of which is to annotate all known human genetic diseases and disease-linked protein variants, as well as neutral polymorphisms.
 </p>
 <ul>
  <li>
   All manually reviewed human entries can be found
   <a href="http://www.uniprot.org/uniprot/?query=organism:9606+AND+reviewed:yes">
    here
   </a>
   <a href="http://www.uniprot.org/biocuration%5Fproject/Chordata/statistics/#Homosapiens">
    (statistics)
   </a>
  </li>
 </ul>
 <h3 id="Annotation_of_other_mammalian_and_chordata_proteins">
  Annotation of other mammalian and chordata proteins
 </h3>
 <p>
  In addition to the review of the human proteome, other mammalian and non-mammalian chordata proteins are increasingly being manually annotated with special emphasis on species such as
  <em>
   Xenopus laevis
  </em>
  ,
  <em>
   Xenopus tropicalis
  </em>
  and
  <em>
   Danio rerio
  </em>
  (Zebrafish) which are important model organisms for studying embryonic development and cell biology. We work in close collaboration with species-specific resources and model organism databases, such as
  <a href="http://www.genenames.org/">
   HGNC
  </a>
  ,
  <a href="http://www.informatics.jax.org/">
   MGI
  </a>
  ,
  <a href="http://rgd.mcw.edu/">
   RGD
  </a>
  ,
  <a href="http://zfin.org/">
   Zfin
  </a>
  and
  <a href="http://www.xenbase.org/">
   Xenbase
  </a>
  , to ensure consistency between UniProt and these resources.
 </p>
 <ul>
  <li>
   All manually reviewed mouse entries can be found
   <a href="http://www.uniprot.org/uniprot/?query=organism:10090+AND+reviewed:yes">
    here
   </a>
   <a href="http://www.uniprot.org/biocuration%5Fproject/Chordata/statistics/#Musmusculus">
    (statistics)
   </a>
  </li>
  <li>
   All manually reviewed rat entries can be found
   <a href="http://www.uniprot.org/uniprot/?query=organism:10116+AND+reviewed:yes">
    here
   </a>
   <a href="http://www.uniprot.org/biocuration%5Fproject/Chordata/statistics/#Rattusnorvegicus">
    (statistics)
   </a>
  </li>
  <li>
   All manually reviewed
   <em>
    Xenopus laevis
   </em>
   entries can be found
   <a href="http://www.uniprot.org/uniprot/?query=organism:8355+AND+reviewed:yes">
    here
   </a>
  </li>
  <li>
   All manually reviewed
   <em>
    Xenopus tropicalis
   </em>
   entries can be found
   <a href="http://www.uniprot.org/uniprot/?query=organism:8364+AND+reviewed:yes">
    here
   </a>
  </li>
  <li>
   All manually reviewed
   <em>
    Danio rerio
   </em>
   (Zebrafish) entries can be found
   <a href="http://www.uniprot.org/uniprot/?query=organism:7955+AND+reviewed:yes">
    here
   </a>
  </li>
 </ul>
</div>