<div>
 <p>
  The following is a description of FASTA headers for UniProtKB (including alternative isoforms), UniRef, UniParc and archived UniProtKB versions. NCBI's program formatdb (in particular its -o option) is compatible with the UniProtKB fasta headers.
 </p>
 <h4 id="UniProtKB">
  UniProtKB
 </h4>
 <pre>&gt;db|UniqueIdentifier|EntryName ProteinName OS=OrganismName OX=OrganismIdentifier [GN=GeneName ]PE=ProteinExistence SV=SequenceVersion
</pre>
 <p>
  Where:
 </p>
 <ul>
  <li>
   <em>
    db
   </em>
   is 'sp' for UniProtKB/Swiss-Prot and 'tr' for UniProtKB/TrEMBL.
  </li>
  <li>
   <em>
    UniqueIdentifier
   </em>
   is the primary
   <a href="http://www.uniprot.org/help/accession%5Fnumbers">
    accession number
   </a>
   of the UniProtKB entry.
  </li>
  <li>
   <em>
    EntryName
   </em>
   is the
   <a href="http://www.uniprot.org/help/entry%5Fname">
    entry name
   </a>
   of the UniProtKB entry.
  </li>
  <li>
   <em>
    ProteinName
   </em>
   is the
   <a href="http://www.uniprot.org/help/protein%5Fnames">
    recommended name
   </a>
   of the UniProtKB entry as annotated in the
   <code>
    RecName
   </code>
   field. For UniProtKB/TrEMBL entries without a
   <code>
    RecName
   </code>
   field, the
   <code>
    SubName
   </code>
   field is used. In case of multiple
   <code>
    SubNames
   </code>
   , the first one is used. The 'precursor' attribute is excluded, 'Fragment' is included with the name if applicable.
  </li>
  <li>
   <em>
    OrganismName
   </em>
   is the
   <a href="http://www.uniprot.org/help/organism-name">
    scientific name of the organism
   </a>
   of the UniProtKB entry.
  </li>
  <li>
   <em>
    OrganismIdentifier
   </em>
   is the
   <a href="http://www.uniprot.org/help/taxonomic%5Fidentifier">
    unique identifier of the source organism, assigned by the NCBI
   </a>
   .
  </li>
  <li>
   <em>
    GeneName
   </em>
   is the first
   <a href="http://www.uniprot.org/help/gene%5Fname">
    gene name
   </a>
   of the UniProtKB entry. If there is no gene name,
   <code>
    OrderedLocusName
   </code>
   or
   <code>
    ORFname
   </code>
   , the
   <code>
    GN
   </code>
   field is not listed.
  </li>
  <li>
   <em>
    ProteinExistence
   </em>
   is the numerical value describing the
   <a href="http://www.uniprot.org/help/protein%5Fexistence">
    evidence for the existence
   </a>
   of the protein.
  </li>
  <li>
   <em>
    SequenceVersion
   </em>
   is the
   <a href="http://www.uniprot.org/help/entry%5Fhistory">
    version number
   </a>
   of the sequence.
  </li>
 </ul>
 <p>
  Examples:
 </p>
 <pre>&gt;sp|Q8I6R7|ACN2_ACAGO Acanthoscurrin-2 (Fragment) OS=Acanthoscurria gomesiana OX=115339 GN=acantho2 PE=1 SV=1
&gt;sp|P27748|ACOX_CUPNH Acetoin catabolism protein X OS=Cupriavidus necator (strain ATCC 17699 / H16 / DSM 428 / Stanier 337) OX=381666 GN=acoX PE=4 SV=2
&gt;sp|P04224|HA22_MOUSE H-2 class II histocompatibility antigen, E-K alpha chain OS=Mus musculus OX=10090 PE=1 SV=1

&gt;tr|Q3SA23|Q3SA23_9HIV1 Protein Nef (Fragment) OS=Human immunodeficiency virus 1  OX=11676 GN=nef PE=3 SV=1
&gt;tr|Q8N2H2|Q8N2H2_HUMAN cDNA FLJ90785 fis, clone THYRO1001457, moderately similar to H.sapiens protein kinase C mu OS=Homo sapiens OX=9606 PE=2 SV=1
</pre>
 <p>
  <strong>
   Alternative isoforms (this only applies to UniProtKB/Swiss-Prot):
  </strong>
  <pre>&gt;sp|IsoID|EntryName Isoform IsoformName of ProteinName OS=OrganismName OX=OrganismIdentifier[ GN=GeneName]
</pre>
 </p>
 <p>
  Where:
 </p>
 <ul>
  <li>
   <em>
    IsoID
   </em>
   is the isoform identifier as assigned in the
   <a href="http://www.uniprot.org/help/alternative%5Fproducts">
    ALTERNATIVE PRODUCTS
   </a>
   section of the UniProtKB entry.
  </li>
  <li>
   <em>
    IsoformName
   </em>
   is the isoform name as annotated in the ALTERNATIVE PRODUCTS
   <code>
    Name
   </code>
   field of the UniProtKB entry.
  </li>
 </ul>
 <p>
  <em>
   ProteinExistence
  </em>
  and
  <em>
   SequenceVersion
  </em>
  do not apply to alternative isoforms (
  <em>
   ProteinExistence
  </em>
  is dependent on the number of cDNA sequences, which is not known for individual isoforms).
 </p>
 <p>
  Example:
 </p>
 <pre>&gt;sp|Q4R572-2|1433B_MACFA Isoform Short of 14-3-3 protein beta/alpha OS=Macaca fascicularis OX=9541 GN=YWHAB
</pre>
 <h4 id="UniRef">
  UniRef
 </h4>
 <pre>&gt;UniqueIdentifier ClusterName n=Members Tax=TaxonName TaxID=TaxonIdentifier RepID=RepresentativeMember
</pre>
 <p>
  Where:
 </p>
 <ul>
  <li>
   <em>
    UniqueIdentifier
   </em>
   is the primary accession number of the UniRef cluster.
  </li>
  <li>
   <em>
    ClusterName
   </em>
   is the name of the UniRef cluster.
  </li>
  <li>
   <em>
    Members
   </em>
   is the number of UniRef cluster members.
  </li>
  <li>
   <em>
    TaxonName
   </em>
   is the scientific name of the lowest common taxon shared by all UniRef cluster members.
  </li>
  <li>
   <em>
    TaxonIdentifier
   </em>
   is the NCBI taxonomy identifier of the lowest common taxon shared by all UniRef cluster members.
  </li>
  <li>
   <em>
    RepresentativeMember
   </em>
   is the entry name of the representative member of the UniRef cluster.
  </li>
 </ul>
 <p>
  Example:
 </p>
 <pre>&gt;UniRef50_Q9K794 Putative AgrB-like protein n=2 Tax=Bacillus TaxID=1386 RepID=AGRB_BACHD
</pre>
 <h4 id="UniParc">
  UniParc
 </h4>
 <pre>&gt;UniqueIdentifier status=Status
</pre>
 <p>
  Where:
 </p>
 <ul>
  <li>
   <em>
    UniqueIdentifier
   </em>
   is the primary accession number of the UniParc entry.
  </li>
  <li>
   <em>
    Status
   </em>
   is 'active' if the UniParc entry has at least one active cross-reference, and 'inactive' if it does not have any active cross-references.
  </li>
 </ul>
 <p>
  Example:
 </p>
 <pre>&gt;UPI0000000005 status=active
</pre>
 <h4 id="Archived_UniProtKB_sequence_versions">
  Archived UniProtKB sequence versions
 </h4>
 <pre>&gt;db|UniqueIdentifier archived from Release ReleaseNumber ReleaseDate SV=SequenceVersion
</pre>
 <p>
  Where:
 </p>
 <ul>
  <li>
   <em>
    db
   </em>
   is 'sp' for UniProtKB/Swiss-Prot and 'tr' for UniProtKB/TrEMBL.
  </li>
  <li>
   <em>
    UniqueIdentifier
   </em>
   is the primary accession number of the UniProtKB entry.
  </li>
  <li>
   <em>
    ReleaseNumber
   </em>
   refers to the release from which the sequence was archived (Swiss-Prot or TrEMBL release numbers for releases prior to the first UniProt release, and both UniProt and Swiss-Prot or TrEMBL release numbers for releases after the first UniProt release).
  </li>
  <li>
   <em>
    ReleaseDate
   </em>
   is the date of the release form which the sequence was archived.
  </li>
  <li>
   <em>
    SequenceVersion
   </em>
   is the version number of the sequence.
  </li>
 </ul>
 <p>
  Examples:
 </p>
 <p>
  'pre-UniProt':
 </p>
 <pre>&gt;sp|P05067 archived from Release 18.0 01-MAY-1991 SV=3
&gt;tr|Q55167 archived from Release 17.0 01-JUN-2001 SV=1
</pre>
 <p>
  'post-UniProt':
 </p>
 <pre>&gt;sp|P05067 archived from Release 9.2/51.2 28-NOV-2006 SV=3
&gt;tr|A0RTJ8 archived from Release 11.0/36.0 29-MAY-2007 SV=1
</pre>
 <p>
  Related terms: FASTA header, FASTA format
 </p>
</div>