<div>
 <p>
  In eukaryotic reference proteomes, unreviewed entries that are likely to belong to the same gene are computationally mapped, based on gene identifiers from Ensembl, EnsemblGenomes and model organism databases.
 </p>
 <p>
  Some proteomes have been (manually and algorithmically) selected as
  <a href="http://www.uniprot.org/help/reference%5Fproteome">
   reference proteomes
  </a>
  . They cover well-studied model organisms and other organisms of interest for biomedical research and phylogeny. In this context, we provide
  <a href="https://ftp.uniprot.org/pub/databases/uniprot/current%5Frelease/knowledgebase/reference%5Fproteomes">
   data sets for reference proteomes
  </a>
  where only one form of a protein, usually the best annotated version in UniProtKB, is present. The relationships identified when generating these data sets are also used when displaying individual entries on the UniProt website:
 </p>
 <p>
  A single gene can code for multiple proteins through biological events such as
  <a href="http://www.uniprot.org/help/alternative%5Fproducts">
   alternative splicing, initiation and promoter usage
  </a>
  . While the UniProtKB/Swiss-Prot expert curation process includes the identification and review of different forms of a protein and their description in a single UniProtKB/Swiss-Prot entry, its focus is the functional annotation of proteins. For this reason, not all potential isoforms of a protein that are available in UniProtKB/TrEMBL can be reviewed and merged into a single entry. This results in a larger number of UniProtKB entries than genes for many of the eukaryotic reference proteomes. In order to identify potential isoforms that have not (yet) been reviewed by a biocurator, we have established an
  <strong>
   automatic gene-centric mapping between entries from eukaryotic reference proteomes that are likely to belong to the same gene
  </strong>
  . This mapping is based on gene identifiers from Ensembl, EnsemblGenomes and model organism databases and, in cases where none of these are available, on gene names assigned by the original sequencing projects.
 </p>
 <p>
  See also:
 </p>
 <ul>
  <li>
   <a href="http://www.uniprot.org/help/uniprotkb%5Fsections">
    Why is UniProtKB composed of 2 sections, UniProtKB/Swiss-Prot and UniProtKB/TrEMBL?
   </a>
  </li>
  <li>
   <a href="http://www.uniprot.org/help/manual%5Fcuration">
    How do we manually annotate a UniProtKB entry?
   </a>
  </li>
  <li>
   <a href="http://www.uniprot.org/help/canonical%5Fand%5Fisoforms">
    What is the canonical sequence? Are all isoforms described in one entry?
   </a>
  </li>
  <li>
   <a href="http://www.uniprot.org/help/reference%5Fproteome">
    What are reference proteomes?
   </a>
  </li>
  <li>
   <a href="http://www.uniprot.org/help/sequences">
    Sequences
   </a>
  </li>
  <li>
   <a href="http://www.uniprot.org/manual/alternative%5Fproducts">
    Alternative products
   </a>
  </li>
 </ul>
</div>