<div>
 <p>
  This subsection of the 'PTM / Processing' section specifies the position and type of each modified residue excluding
  <a href="http://www.uniprot.org/manual/lipid">
   lipids
  </a>
  ,
  <a href="http://www.uniprot.org/manual/carbohyd">
   glycans
  </a>
  and
  <a href="http://www.uniprot.org/manual/crosslnk">
   protein cross-links
  </a>
  .
 </p>
 <p>
  Common modifications include phosphorylation, methylation, acetylation, amidation, formation of pyrrolidone carboxylic acid, isomerization, hydroxylation, sulfation, flavin-binding, cysteine oxidation and nitrosylation.
 </p>
 <p>
  We describe the chemical nature of the modified residue using a controlled vocabulary (see the document
  <a href="http://www.uniprot.org/docs/ptmlist">
   'Controlled vocabulary of posttranslational modifications (PTM)'
  </a>
  ).
 </p>
 <p>
  We provide additional information concerning the modification, such as:
 </p>
 <ol>
  <li>
   the
   <strong>
    form of the protein
   </strong>
   which undergoes the modification; this may be either a specific
   <strong>
    isoform
   </strong>
   , a particular
   <strong>
    processed or modified form
   </strong>
   of the protein, or a specific
   <strong>
    sequence variant
   </strong>
   ;
   <br/>
   Examples:
   <a href="http://www.uniprot.org/uniprot/P41500#ptm%5Fprocessing">
    P41500
   </a>
   (isoform),
   <a href="http://www.uniprot.org/uniprot/P84715#ptm%5Fprocessing">
    P84715
   </a>
   (processed form),
   <a href="http://www.uniprot.org/uniprot/P68871#ptm%5Fprocessing">
    P68871
   </a>
   (sequence variant)
  </li>
  <li>
   the
   <strong>
    enzyme
   </strong>
   which carries out the modification ('by...'). For proteins of infectious organisms, such as viruses, phages and bacteria, we also indicate whether the modification is carried out by a
   <strong>
    host
   </strong>
   protein;
   <br/>
   Examples:
   <a href="http://www.uniprot.org/uniprot/P03279#ptm%5Fprocessing">
    P03279
   </a>
   (modifying enzyme indicated),
   <a href="http://www.uniprot.org/uniprot/Q53EZ4#ptm%5Fprocessing">
    Q53EZ4
   </a>
   (viral protein modified by host protein)
  </li>
  <li>
   information on the
   <strong>
    frequency
   </strong>
   of the modification or the
   <strong>
    relationship with another feature
   </strong>
   ('partial', 'alternate', 'transient'). The term
   <strong>
    'partial'
   </strong>
   indicates that not all protein molecules are modified,
   <strong>
    'alternate'
   </strong>
   means that the same amino acid can be modified in more than one way, and
   <strong>
    'transient'
   </strong>
   is applied to exceptional cases of otherwise stable modifications. For partial modifications, we do not propagate this comment to homologous proteins and we do not specify the fraction of proteins modified, as this may depend on the experimental conditions.
   <br/>
   Examples:
   <a href="http://www.uniprot.org/uniprot/P16157#ptm%5Fprocessing">
    P16157
   </a>
   (partial modification),
   <a href="http://www.uniprot.org/uniprot/Q9UKV3#ptm%5Fprocessing">
    Q9UKV3
   </a>
   (alternate modification),
   <a href="http://www.uniprot.org/uniprot/Q9FX54#ptm%5Fprocessing">
    Q9FX54
   </a>
   (transient modification)
  </li>
 </ol>
 <p>
  See also the subsection
  <a href="http://www.uniprot.org/help/post-translational%5Fmodification">
   Post-translational modifications
  </a>
  for additional information on modifications for which
  <strong>
   position-specific data is not yet available
  </strong>
  .
 </p>
 <p>
  <strong>
   Data sources and propagation of modified residue annotation
  </strong>
 </p>
 <p>
  We annotate experimentally determined sites of modification. These are propagated 'By similarity' to related orthologs provided the following criteria are met:
 </p>
 <ol>
  <li>
   the modification should be necessary for protein function (and therefore likely to occur in related organisms);
  </li>
  <li>
   the enzyme which performs the modification must exist in the related organism, or the same type of modification must have already been observed (this condition is mandatory);
  </li>
  <li>
   the modified residue and the surrounding region should be conserved in the orthologous protein (this condition is also mandatory).
  </li>
 </ol>
 <p>
  Propagation is generally restricted to closely related species, e.g. among mammals or bacteria from the same taxonomic group. We usually do not propagate information concerning modified residues among plants or unicellular fungi. Specific details regarding the type of modification may be generalized during propagation. For instance, a particular lysine can be subjected to mono-or dimethylation in
  <em>
   E.coli
  </em>
  strain K12, but this information is not propagated to the
  <em>
   E.coli
  </em>
  O157:H7 orthologous entry, where we simply indicate that methylation occurs.
  <br/>
  Example:
  <a href="http://www.uniprot.org/uniprot/P0CE47#ptm_processing">
   <span class="caps">
    P0CE47
   </span>
  </a>
  (source entry),
  <a href="http://www.uniprot.org/uniprot/P0A6N3#ptm_processing">
   <span class="caps">
    P0A6N3
   </span>
  </a>
  (orthologous entry)
 </p>
 <p>
  <strong>
   Sites that are not modified
  </strong>
 </p>
 <p>
  When a site is found not to be modified, and if this is of biological significance, we indicate it in the 'Site' subsection. This information is not propagated in related entries.
  <br/>
  Examples:
  <a href="http://www.uniprot.org/uniprot/Q10471#ptm_processing">
   Q10471
  </a>
  ,
  <a href="http://www.uniprot.org/uniprot/P62152#ptm_processing">
   P62152
  </a>
  ,
  <a href="http://www.uniprot.org/uniprot/P32457#ptm_processing">
   P32457
  </a>
  ,
  <a href="http://www.uniprot.org/uniprot/P07173#ptm_processing">
   P07173
  </a>
  .
 </p>
 <p>
  <strong>
   Unknown sites bearing unknown modifications
  </strong>
 </p>
 <p>
  Here is an example of a feature where the identity of the amino acid is unknown (an X is shown at this position in the sequence) and the only information concerning the modification is that the N-terminus is blocked:
  <a href="http://www.uniprot.org/uniprot/P80979#ptm%5Fprocessing">
   P80979
  </a>
  (Blocked amino end (Xaa)).
 </p>
 <h3 id="1__Phosphorylation">
  1. Phosphorylation
 </h3>
 <p>
  Phosphorylation refers to the transfer of a gamma-phosphate to an amino acid. It is a key mechanism for signaling in both eukaryotic and prokaryotic cells. It can occur on a number of cytoplasmic and nuclear residues, i.e. on the hydroxyl group of serine, threonine or tyrosine, on the nitrogen of arginine, histidine or lysine, on the carboxyl group of aspartate, or on the sulfhydryl group of cysteine.
 </p>
 <p>
  Related keyword:
  <a href="http://www.uniprot.org/keywords/597">
   Phosphoprotein
  </a>
 </p>
 <p>
  Phosphorylation is frequent on serine, threonine, and tyrosine from eukaryotic proteins, serine phosphorylation being the most common. Phosphorylation of histidine and aspartate is known to occur as part of the two-component signalling in prokaryotes and has also been described in eukaryotes (mainly fungi and plants).
 </p>
 <p>
  Example:
  <a href="http://www.uniprot.org/uniprot/O15350#ptm%5Fprocessing">
   O15350
  </a>
 </p>
 <p>
  Since phosphorylation (phosphoserine, phosphothreonine and phosphotyrosine) is a reversible modification, phosphorylation sites are never annotated as 'partial'.
 </p>
 <p>
  Examples:
  <a href="http://www.uniprot.org/uniprot/Q9RQQ9#ptm%5Fprocessing">
   Q9RQQ9
  </a>
  ,
  <a href="http://www.uniprot.org/uniprot/P02662#ptm%5Fprocessing">
   P02662
  </a>
  ,
  <a href="http://www.uniprot.org/uniprot/P04859#ptm%5Fprocessing">
   P04859
  </a>
  ,
  <a href="http://www.uniprot.org/uniprot/P68433#ptm%5Fprocessing">
   P68433
  </a>
 </p>
 <p>
  Histidine can be phosphorylated on either of its two nitrogen atoms. We refer to 'Pros-phosphohistidine', when phosphorylation occurs on the nitrogen atom that is closest to the alpha-carbon and 'Tele-phosphohistidine', where it occurs on the most distal one. When the exact position of phosphate attachment on histidine is not known we simply use the term 'Phosphohistidine'.
 </p>
 <p>
  Examples:
  <a href="http://www.uniprot.org/uniprot/P0AA06#ptm_processing">
   P0AA06
  </a>
  ,
  <a href="http://www.uniprot.org/uniprot/P39928#ptm_processing">
   P39928
  </a>
  ,
  <a href="http://www.uniprot.org/uniprot/P16575#ptm_processing">
   P16575
  </a>
  ,
  <a href="http://www.uniprot.org/uniprot/P26762#ptm_processing">
   P26762
  </a>
  ,
  <a href="http://www.uniprot.org/uniprot/P0A0E2#ptm_processing">
   P0A0E2
  </a>
 </p>
 <p>
  Note that phosphorylation of aspartate follows a specific syntax:
 </p>
 <p>
  Example:
  <a href="http://www.uniprot.org/uniprot/P04042#ptm_processing">
   P04042
  </a>
 </p>
 <p>
  We annotate experimentally determined phosphorylation sites and transfer this information to related orthologs as described. We do not annotate predicted phosphorylation sites. When transferring annotation regarding phosphorylation, we do not usually specify the kinase responsible in the orthologous entry, except when the modification is part of a precise, well-studied transduction pathway. For phosphohistidine, we propagate the position of the phosphorylated nitrogen atom.
 </p>
 <h3 id="2__Methylation">
  2. Methylation
 </h3>
 <p>
  Cytoplasmic and nuclear proteins can be enzymatically modified in several ways by the addition of methyl groups from S-adenosylmethionine. Methylation reactions occurring on carboxyl groups can be reversible and modulate the activity of the target protein, while those on nitrogen atoms at the N-terminus and on side-chains are usually irreversible.
 </p>
 <p>
  Related keyword:
  <a href="http://www.uniprot.org/keywords/488">
   Methylation
  </a>
 </p>
 <h4 id="Carboxyl_methylation">
  Carboxyl methylation
 </h4>
 <p>
  Carboxyl methylation can occur either on a C-terminal cysteine, leucine or lysine residue, or on the side chain of a glutamate residue (or glutamine, after deamidation). It can affect protein-protein interactions and protein function.
 </p>
 <p>
  Examples:
  <a href="http://www.uniprot.org/uniprot/P67775#ptm_processing">
   P67775
  </a>
  ,
  <a href="http://www.uniprot.org/uniprot/P02994#ptm_processing">
   P02994
  </a>
 </p>
 <p>
  Cysteine carboxymethylation frequently occurs after prenylation of the CAAX (Cys - aliphatic_twice - any residue) sequence and proteolytic cleavage of the C-A bond.
 </p>
 <p>
  Example:
  <a href="http://www.uniprot.org/uniprot/P34068#ptm_processing">
   P34068
  </a>
 </p>
 <p>
  In prokaryotes, glutamate methyl ester formation plays a major role in chemotactic signal transduction. We indicate whether the glutamate methyl ester is formed either from glutamate or from glutamine.
 </p>
 <p>
  Example:
  <a href="http://www.uniprot.org/uniprot/P07018#ptm%5Fprocessing">
   P07018
  </a>
 </p>
 <h4 id="Nitrogen_methylation">
  Nitrogen methylation
 </h4>
 <p>
  Nitrogen methylation can occur on the N-terminus of a polypeptide chain on a phenylalanine, isoleucine, leucine, methionine, tyrosine, proline or alanine residue. It can also occur on the side chain of lysine, arginine or histidine residues. In eukaryotes, arginine and lysine methylation have been found mainly on histones and play an important role in signal transduction processes, nuclear transport and regulation of transcription.
 </p>
 <p>
  Examples:
  <a href="http://www.uniprot.org/uniprot/P0A7N9#ptm_processing">
   P0A7N9
  </a>
  ,
  <a href="http://www.uniprot.org/uniprot/P00873#ptm_processing">
   P00873
  </a>
 </p>
 <h4 id="Histidine_methylation">
  Histidine methylation
 </h4>
 <p>
  Histidine methylation can occur on two positions, which are specified as 'Pros-methylhistidine' and 'Tele-methylhistidine'. When the exact position of the methylation is not known, we simply indicate 'Methylhistidine'.
 </p>
 <p>
  Examples:
  <a href="http://www.uniprot.org/uniprot/P31725#ptm_processing">
   P31725
  </a>
  ,
  <a href="http://www.uniprot.org/uniprot/P62739#ptm_processing">
   P62739
  </a>
  ,
  <a href="http://www.uniprot.org/uniprot/P24020#ptm_processing">
   P24020
  </a>
 </p>
 <h4 id="Lysine_methylation">
  Lysine methylation
 </h4>
 <p>
  Lysine can be mono-, di-, or tri-methylated. We describe each of these modifications as 'N6-methyllysine', 'N6,N6-dimethyllysine' and 'N6,N6,N6-trimethyllysine'. If the number of methyl groups is unknown, we simply indicate 'N6-methylated lysine'. This general description is also used when propagating the information 'By similarity' to homologous proteins.
 </p>
 <p>
  Example:
  <a href="http://www.uniprot.org/uniprot/P13538#ptm_processing">
   P13538
  </a>
 </p>
 <p>
  Lysine methylation can compete with acetylation on the same residue. In this case, the modifications are described as 'alternate'.
 </p>
 <h4 id="Arginine_methylation">
  Arginine methylation
 </h4>
 <p>
  Arginine can be monomethylated or dimethylated in two ways which leads to four different descriptions: 'Omega-N-methylarginine', 'Dimethylated arginine', 'Symmetric dimethylarginine' and 'Asymmetric dimethylarginine'. If the residue is dimethylated, but the position of the methyl groups is unknown, it is annotated as 'Dimethylated arginine'. If the number of methyl groups is unknown, the ambiguous description 'Omega-N-methylated arginine' is used. This general description is also used when propagating information to homologous proteins.
 </p>
 <p>
  Examples:
  <a href="http://www.uniprot.org/uniprot/Q07666#ptm_processing">
   Q07666
  </a>
  ,
  <a href="http://www.uniprot.org/uniprot/P11940#ptm_processing">
   P11940
  </a>
  ,
  <a href="http://www.uniprot.org/uniprot/Q60487#ptm_processing">
   Q60487
  </a>
  ,
  <a href="http://www.uniprot.org/uniprot/P45481#ptm_processing">
   P45481
  </a>
 </p>
 <h4 id="Other_rare_examples_of_side_chain_methylation">
  Other rare examples of side chain methylation
 </h4>
 <p>
  Examples:
  <a href="http://www.uniprot.org/uniprot/P00318#ptm_processing">
   P00318
  </a>
  ,
  <a href="http://www.uniprot.org/uniprot/P0A287#ptm_processing">
   P0A287
  </a>
 </p>
 <h3 id="3__Acetylation">
  3. Acetylation
 </h3>
 <p>
  We annotate both N-terminal acetylation and acetylation on internal residues.
 </p>
 <p>
  Related keyword:
  <a href="http://www.uniprot.org/keywords/7">
   Acetylation
  </a>
 </p>
 <h4 id="N_terminal_acetylation">
  N-terminal acetylation
 </h4>
 <p>
  N-terminal acetylation is one of the most common post-translational modifications in eukaryotes, but it is rare in prokaryotes. It refers to the addition of an acetyl group from acetyl-CoA to the alpha-amino group of the first residue of a protein, often after the cleavage of the initiator methionine. The most commonly acetylated residues are glycine, alanine, serine or threonine. This reaction occurs in the cytosol. Methionine residues can also be modified if the next residue is an aspartate, glutamate, leucine, isoleucine, tryptophan, phenylalanine or asparagine residue. Note that the modified position may not correspond to the first amino acid of the displayed sequence if N-terminal acetylation occurs after proteolytic processing of the chain.
 </p>
 <p>
  We annotate experimentally determined sites of N-terminal acetylation and this information is propagated 'By similarity' to homologous proteins in related species.
 </p>
 <p>
  Examples:
  <a href="http://www.uniprot.org/uniprot/P23542#ptm_processing">
   P23542
  </a>
  ,
  <a href="http://www.uniprot.org/uniprot/P68251#ptm_processing">
   P68251
  </a>
  ,
  <a href="http://www.uniprot.org/uniprot/P01201#ptm_processing">
   P01201
  </a>
  ,
  <a href="http://www.uniprot.org/uniprot/P41682#ptm_processing">
   P41682
  </a>
  ,
  <a href="http://www.uniprot.org/uniprot/Q71SP7#ptm_processing">
   Q71SP7
  </a>
 </p>
 <h4 id="Internal_acetylation">
  Internal acetylation
 </h4>
 <p>
  Internal acetylation is the addition of a N-alpha-acetyl group from acetyl-CoA to the side chain of a lysine residue. In eukaryotes, it generally takes place in the nucleus and affects mainly, but not exclusively, histones. It also occurs in prokaryotes.
 </p>
 <p>
  Lysine acetylation can compete with acetylation on the same residue, in which case both modifications are described as 'alternate'.
 </p>
 <p>
  We annotate experimentally determined sites of internal acetylation and propagate the information 'By similarity' to homologous proteins in related species.
 </p>
 <p>
  Examples:
  <a href="http://www.uniprot.org/uniprot/P0C0S9#ptm_processing">
   P0C0S9
  </a>
  ,
  <a href="http://www.uniprot.org/uniprot/Q12158#ptm_processing">
   Q12158
  </a>
  ,
  <a href="http://www.uniprot.org/uniprot/Q9NHD5#ptm_processing">
   Q9NHD5
  </a>
  ,
  <a href="http://www.uniprot.org/uniprot/Q8ZKF6#ptm_processing">
   Q8ZKF6
  </a>
  ,
  <a href="http://www.uniprot.org/uniprot/Q88EH6#ptm_processing">
   Q88EH6
  </a>
 </p>
 <h3 id="4__Amidation">
  4. Amidation
 </h3>
 <p>
  The C-terminus of secreted proteins is often modified by cleavage between the amino group and the alpha carbon of a C-terminal glycine, resulting in the amidation of the precedent amino acid. This modification protects the C-terminus from degradation by proteases. Amidated proteins contain a signal peptide and are often processed prior to amidation in order to expose a glycine at C-terminus. Amidation has been observed in eukaryotes including mammals, non-mammalian vertebrates and insects, but not in plants.
 </p>
 <p>
  Related keyword:
  <a href="http://www.uniprot.org/keywords/27">
   Amidation
  </a>
 </p>
 <p>
  The sequence including the amidated position generally conforms to the consensus cleavage sequence for proconvertases, which is G[RK] or G[RK][RK]. Following proconvertase cleavage, removal of the pair of basic residues is accomplished by carboxypeptidase H, while the terminal glycine is converted to a new C-terminal amide by the bifunctional peptidylglycine alpha-amidating monooxygenase (PAM).
 </p>
 <p>
  We annotate experimentally determined sites of amidation, which may be propagated to homologous proteins in related species. For each amidation, we specify the position of the glycine which provides the amide group, provided it is present within the mature protein chain. We also specify the position at which the protein is cleaved, if that is known, and the limits of the peptide which is removed by cleavage. For aspartate and glutamate amides the position of the amide is also indicated as the side chains of these amino acids can theoretically be modified to amides by a different reaction.
 </p>
 <p>
  Examples:
  <a href="http://www.uniprot.org/uniprot/P58913#ptm_processing">
   P58913
  </a>
  ,
  <a href="http://www.uniprot.org/uniprot/P09859#ptm_processing">
   P09859
  </a>
 </p>
 <h4 id="Terminal_amidation">
  Terminal amidation
 </h4>
 <p>
  Examples:
  <a href="http://www.uniprot.org/uniprot/P69148#ptm_processing">
   P69148
  </a>
  ,
  <a href="http://www.uniprot.org/uniprot/P82387#ptm_processing">
   P82387
  </a>
 </p>
 <h4 id="Glutamate_amidation">
  Glutamate amidation
 </h4>
 <p>
  Example:
  <a href="http://www.uniprot.org/uniprot/P20481#ptm_processing">
   P20481
  </a>
 </p>
 <h3 id="5__Pyrrolidone_carboxylic_acid">
  5. Pyrrolidone carboxylic acid
 </h3>
 <p>
  The N-terminal glutamine of extracellular and multi-pass membrane proteins can be modified by cyclization of the glutamine via condensation of the alpha-amino group with the side-chain carboxyl group. Modified proteins show an increased half-life. Modified proteins of this type cannot be sequenced by the Edman method, they are blocked.
 </p>
 <p>
  This modification can also occur from a glutamate residue but this seems to be extremely rare, and may be correlated with an extreme acidic context of the protein involved. We specify it by indicating 'Pyrrolidone carboxylic acid (Glu)' in the 'Description' field.
 </p>
 <p>
  This modification has been observed in eukaryotes (including mammals, plants, insects), archaea (including halobacteria) and bacteria (including proteobacteria and actinobacteria).
  <br/>
  Synonyms: pyrrolidone carboxylic acid (Pca), pyroglutamic acid (Pga), pyroglutamate (pyro-Glu, pGlu, Pyr).
  <br/>
  <em>
   Note
  </em>
  : Pyro-Glu is often indicated in papers as 'pGlu' and sometimes, in one-letter code as "U", although this is now used for selenocysteine. In figures of publications, it may be cited as Z, pQ or E.
 </p>
 <p>
  Examples:
  <a href="http://www.uniprot.org/uniprot/P30233#ptm_processing">
   P30233
  </a>
  ,
  <a href="http://www.uniprot.org/uniprot/P68000#ptm_processing">
   P68000
  </a>
  ,
  <a href="http://www.uniprot.org/uniprot/P02945#ptm_processing">
   P02945
  </a>
 </p>
 <p>
  Related keyword:
  <a href="http://www.uniprot.org/keywords/873">
   Pyrrolidone carboxylic acid
  </a>
 </p>
 <p>
  We annotate experimentally identified pyrrolidone carboxylic acid modifications, which may be propagated to homologous proteins in related species. When the N-terminal glutamine of an extracellular protein is known to be blocked, we annotate it as a pyrrolidone carboxylic acid with the evidence 'Curated'.
 </p>
 <p>
  Example:
  <a href="http://www.uniprot.org/uniprot/P12111#ptm_processing">
   P12111
  </a>
 </p>
 <h3 id="6__Isomerization">
  6. Isomerization
 </h3>
 <p>
  In the translation process, only L-amino acids are incorporated into nascent polypeptides by the ribosomes. However, a variety of amino acids in secreted peptides are post-translationally converted to the D-form, probably via a deprotonation-reprotonation mechanism at the alpha-carbon. This modification has a strong effect on protein conformation and thus on protein activity and interactions. This modification has been found in secreted neuropeptides, toxins and hormones from molluscs, frogs, crustaceans, arachnids, and in bacterial lantibiotics. Synonyms: racemization, epimerisation, stereoinversion.
 </p>
 <p>
  Example:
  <a href="http://www.uniprot.org/uniprot/P35904#ptm_processing">
   P35904
  </a>
 </p>
 <p>
  Related keyword:
  <a href="http://www.uniprot.org/keywords/208">
   D-amino acid
  </a>
 </p>
 <p>
  D-alanine can originate from both alanine and serine. The name of the original amino acid is given in brackets in the 'Description' field.
 </p>
 <p>
  Examples:
  <a href="http://www.uniprot.org/uniprot/O93456#ptm_processing">
   O93456
  </a>
  ,
  <a href="http://www.uniprot.org/uniprot/P23826#ptm_processing">
   P23826
  </a>
 </p>
 <p>
  2 stereoisomers of L-isoleucine can exist: D-allo-isoleucine and D-threo-isoleucine. However, only the allo-isomer has been observed so far.
 </p>
 <p>
  Example:
  <a href="http://www.uniprot.org/uniprot/P29006#ptm_processing">
   P29006
  </a>
 </p>
 <p>
  When D-amino acids are found as partners of a cross-link, it is indicated in the 'Cross-link' subsection.
  <br/>
  Examples:
  <a href="http://www.uniprot.org/uniprot/P08136#ptm_processing">
   P08136
  </a>
  ,
  <a href="http://www.uniprot.org/uniprot/O07623#ptm_processing">
   O07623
  </a>
 </p>
 <h3 id="7__Hydroxylation">
  7. Hydroxylation
 </h3>
 <p>
  The modified amino acids are generally extracellular (secreted proteins, extracellular matrix proteins, multi-pass membrane proteins).
 </p>
 <p>
  In animals, 3-hydroxyproline, 4-hydroxyproline and 5-hydroxylysine are mostly found in collagen and collagen-like domain-containing proteins. 4-hydroxyprolines have been shown to stabilize the collagen triple helix. Some of the hydroxylysines are further modified by O-glycosylation. In plants 4-hydroxyproline is mostly found in structural components of cell walls (hydroxyproline-rich glycoproteins, e.g. extensins, proline-rich proteins, arabinogalactan proteins). In most of the extensin and extensin-like domain-containing proteins, hydroxyprolines are further O-glycosylated. 4-hydroxyproline is also found in secreted neuropeptides, toxins, anti-fungal and anti-bacterial peptides of several molluscs, insects and plants.
 </p>
 <p>
  3-hydroxyaspartate and 3-hydroxyasparagine have been found in EGF-like domain-containing proteins, e.g. blood coagulation protein factors VII, IX and X, proteins C, S, and Z, the LDL receptor and thrombomodulin.
 </p>
 <p>
  Related keyword:
  <a href="http://www.uniprot.org/keywords/379">
   Hydroxylation
  </a>
 </p>
 <h4 id="Hydroxyproline">
  Hydroxyproline
 </h4>
 <p>
  Both C3 and C4 of proline can be hydroxylated. The carbon bearing the hydroxyl group is indicated in the 'Description' field, when known. In the absence of precise information, we use the ambiguous description 'Hydroxyproline'. Note that hydroxyproline can be further glycosylated.
 </p>
 <p>
  <strong>
   4-hydroxyproline
  </strong>
 </p>
 <p>
  Examples:
  <a href="http://www.uniprot.org/uniprot/Q5E9E3#ptm_processing">
   Q5E9E3
  </a>
  ,
  <a href="http://www.uniprot.org/uniprot/P60245#ptm_processing">
   P60245
  </a>
 </p>
 <p>
  <strong>
   3-hydroxyproline
  </strong>
 </p>
 <p>
  Examples:
  <a href="http://www.uniprot.org/uniprot/P30754#ptm_processing">
   P30754
  </a>
  ,
  <a href="http://www.uniprot.org/uniprot/Q9S8M0#ptm_processing">
   Q9S8M0
  </a>
  ,
  <a href="http://www.uniprot.org/uniprot/Q16665#ptm_processing">
   Q16665
  </a>
 </p>
 <p>
  The information on hydroxyproline may be propagated 'By similarity' to closely related organisms.
 </p>
 <p>
  Example:
  <a href="http://www.uniprot.org/uniprot/Q15848#ptm_processing">
   Q15848
  </a>
 </p>
 <h4 id="Hydroxylysine">
  Hydroxylysine
 </h4>
 <p>
  <strong>
   5-hydroxylysine
  </strong>
 </p>
 <p>
  Example:
  <a href="http://www.uniprot.org/uniprot/Q3Y5Z3#ptm_processing">
   Q3Y5Z3
  </a>
 </p>
 <p>
  <em>
   Note
  </em>
  : Hydroxylysine can be glycosylated.
 </p>
 <p>
  <strong>
   4,5-dihydroxylysine, 3',4'-dihydroxyphenylalanine (DOPA), 3,4-dihydroxyarginine
  </strong>
 </p>
 <p>
  Examples:
  <a href="http://www.uniprot.org/uniprot/O18495#ptm_processing">
   O18495
  </a>
  ,
  <a href="http://www.uniprot.org/uniprot/Q25460#ptm_processing">
   Q25460
  </a>
  ,
  <a href="http://www.uniprot.org/uniprot/O18496#ptm_processing">
   O18496
  </a>
 </p>
 <h4 id="Hydroxyasparagine">
  Hydroxyasparagine
 </h4>
 <p>
  <strong>
   3-hydroxyasparagine
  </strong>
  ,
  <strong>
   3-hydroxyaspartate
  </strong>
  .
 </p>
 <p>
  Examples:
  <a href="http://www.uniprot.org/uniprot/Q8CG16#ptm_processing">
   Q8CG16
  </a>
  ,
  <a href="http://www.uniprot.org/uniprot/O88278#ptm_processing">
   O88278
  </a>
 </p>
 <h3 id="8__Sulfation">
  8. Sulfation
 </h3>
 <p>
  Extracellular tyrosines of secreted and multi-pass membrane proteins can
  <br/>
  be modified by the addition of a sulfate group. Cytoplasmic serine and threonine residues can also undergo sulfation, although very rarely. Although the function of this modification has not yet been fully elucidated, it may serve to enhance protein stability and modulate protein-protein interactions. When carbohydrates attached to proteins are sulfated, we indicate this fact in the '
  <a href="http://www.uniprot.org/manual/post-translational_modification">
   Post-translational modification
  </a>
  ' subsection. Sulfation has been observed in eukaryotes only.
 </p>
 <p>
  Related keyword:
  <a href="http://www.uniprot.org/keywords/765">
   Sulfation
  </a>
 </p>
 <p>
  Carbohydrates attached to proteins can also be sulfated, this is annotated in the 'Post-translational modification' subsection, but won't lead to the attribution of the keyword 'Sulfation', which applies only to the modification of the protein itself, but not to the attached carbohydrate groups.
 </p>
 <h4 id="Tyrosine_sulfation">
  Tyrosine sulfation
 </h4>
 <p>
  Example:
  <a href="http://www.uniprot.org/uniprot/Q7LZ52#ptm_processing">
   Q7LZ52
  </a>
 </p>
 <h4 id="Serine_and_threonine_sulfation_">
  Serine and threonine sulfation:
 </h4>
 <p>
  Examples:
  <a href="http://www.uniprot.org/uniprot/Q01974#ptm_processing">
   Q01974
  </a>
  ,
  <a href="http://www.uniprot.org/uniprot/Q8IIJ9#ptm_processing">
   Q8IIJ9
  </a>
 </p>
 <p>
  We annotate not only experimentally determined sites of tyrosine sulfation (as well as that on serine and threonine residues), but also tyrosine sulfation sites predicted with the
  <a href="http://web.expasy.org/sulfinator/">
   'Sulfinator'
  </a>
  tool. Sulfation prediction is taken into account only when this modification is known to occur on the protein concerned but the exact site is not known. The annotation of a predicted site is flagged with 'Sequence Analysis' (Sequence Model).
  <br/>
  Example:
  <a href="http://www.uniprot.org/uniprot/P30443#ptm_processing">
   P30443
  </a>
 </p>
 <h3 id="9__Flavin_binding">
  9. Flavin-binding
 </h3>
 <p>
  Enzymes involved in electron transport, oxidation and reduction often contain a flavin group (flavin mononucleotide [FMN]) or flavin adenine dinucleotide [FAD]) as a cofactor. When the cofactor is covalently bound to the protein, this is considered as a post-translational modification and is annotated in the 'Amino acid modifications' subsection. When the cofactor is not covalently bound, the binding region is annotated in the
  <a href="http://www.uniprot.org/manual/np%5Fbind">
   'Nucleotide binding'
  </a>
  subsection. The flavin can be transferred to the hydroxyl group of a serine, threonine or tyrosine, to the one or the other nitrogen of a histidine or to the sulfhydryl group of a cysteine. Flavin-binding has been observed in all organisms.
 </p>
 <p>
  Examples:
  <a href="http://www.uniprot.org/uniprot/P21398#ptm%5Fprocessing">
   P21398
  </a>
  ,
  <a href="http://www.uniprot.org/uniprot/P09788#ptm%5Fprocessing">
   P09788
  </a>
  ,
  <a href="http://www.uniprot.org/uniprot/Q9UI17#ptm%5Fprocessing">
   Q9UI17
  </a>
  ,
  <a href="http://www.uniprot.org/uniprot/Q752Y3#ptm%5Fprocessing">
   Q752Y3
  </a>
  ,
  <a href="http://www.uniprot.org/uniprot/Q9K0M5#ptm%5Fprocessing">
   Q9K0M5
  </a>
  ,
  <a href="http://www.uniprot.org/uniprot/Q9KPS2#ptm%5Fprocessing">
   Q9KPS2
  </a>
  ,
  <a href="http://www.uniprot.org/uniprot/O34627#ptm%5Fprocessing">
   O34627
  </a>
  ,
  <a href="http://www.uniprot.org/uniprot/Q48303#ptm%5Fprocessing">
   Q48303
  </a>
  ,
  <a href="http://www.uniprot.org/uniprot/P40875#ptm%5Fprocessing">
   P40875
  </a>
 </p>
 <p>
  Related keywords:
  <a href="http://www.uniprot.org/keywords/285">
   Flavoprotein
  </a>
  ,
  <a href="http://www.uniprot.org/keywords/274">
   FAD
  </a>
  ,
  <a href="http://www.uniprot.org/keywords/288">
   FMN
  </a>
 </p>
 <h3 id="10__Cysteine_oxidation_and_nitrosylation">
  10. Cysteine oxidation and nitrosylation
 </h3>
 <p>
  Sulfur occurs in many different oxidation states in biological systems. In response to mild oxidative stress, reactive oxygen and nitrogen species, such as peroxides, superoxide, nitric oxide or peroxinitrite, can oxidize cytoplasmic cysteines to cysteine sulfenic acid (-SOH), S-nitrosocysteine (-SNO) and sulfinic acid (-SO2H)(single, single and double oxidation state, respectively). These modifications can alter protein activity, protein-protein interactions, or protein stability.
 </p>
 <p>
  Cysteine sulfenic acid is extremely reactive. It is generally stabilized
  <br/>
  by either the formation of a disulfide bond (in this case the residue is called a cystine) or glutathione binding or reduced by specific enzymes to cysteine. Disulfide bonds are annotated in the '
  <a href="http://www.uniprot.org/manual/disulfid">
   Disulfide bond
  </a>
  ' subsection.
 </p>
 <p>
  In response to severe oxidative stress, cysteines are irreversibly oxidized to cysteine sulfonic acid (-SO3) or cysteic acid, which generally leads to protein inactivation and degradation. We do not annotate this modification.
 </p>
 <h4 id="Sulfenic_and_sulfinic_acid">
  Sulfenic and sulfinic acid
 </h4>
 <p>
  Examples:
  <a href="http://www.uniprot.org/uniprot/P13448#ptm%5Fprocessing">
   P13448
  </a>
  ,
  <a href="http://www.uniprot.org/uniprot/P03122#ptm%5Fprocessing">
   P03122
  </a>
 </p>
 <p>
  Related keyword:
  <a href="http://www.uniprot.org/keywords/558">
   Oxidation
  </a>
 </p>
 <h4 id="S_nitrosocysteine">
  S-nitrosocysteine
 </h4>
 <p>
  Examples:
  <a href="http://www.uniprot.org/uniprot/P68871#ptm%5Fprocessing">
   P68871
  </a>
  ,
  <a href="http://www.uniprot.org/uniprot/P0ACQ6#ptm%5Fprocessing">
   P0ACQ6
  </a>
 </p>
 <p>
  Related keyword:
  <a href="http://www.uniprot.org/keywords/702">
   S-nitrosylation
  </a>
 </p>
 <h4 id="Related_documents">
  Related documents
 </h4>
 <p>
  The nature of the post-translationally formed amino acid is annotated by using a controlled vocabulary. The currently defined list of controlled vocabulary, as well as other information, such as the target amino acid, the related keyword, the taxonomic range and the subcellular location of the modification, are available in
  <a href="http://www.uniprot.org/docs/ptmlist">
   ptmlist.txt
  </a>
  document. Links to the RESID database are also provided to help gain a better insight into every modification.
 </p>
 <p>
  See also:
  <a href="http://www.uniprot.org/manual/evidences">
   Evidence
  </a>
  ,
  <a href="http://www.uniprot.org/help/post-translational%5Fmodification">
   Post-translational modifications
  </a>
  ,
  <a href="http://www.uniprot.org/help/advanced%5Fsearch">
   Advanced search
  </a>
 </p>
</div>