<div>
 <p>
  <strong>
   How is orthology established in UniProtKB/Swiss-Prot?
  </strong>
 </p>
 <p>
  Information is combined from a variety of complementary sources to establish orthology relationships. Potential orthologs are initially identified using sequence similarity search programs such as BLAST. Orthology relationships are then verified manually using a combination of resources including scientific literature, sequence analysis tools, phylogenetic and comparative genomics databases such as Ensembl Compara, and other specialised databases such as species-specific collections.
 </p>
 <p>
  See also:
 </p>
 <ul>
  <li>
   <a href="http://www.uniprot.org/help/hamap">
    What is HAMAP?
   </a>
  </li>
  <li>
   <a href="http://www.uniprot.org/help/family%5Fmembership">
    How is protein family membership assigned in UniProtKB?
   </a>
  </li>
  <li>
   <a href="http://www.uniprot.org/help/orthologs%5Fbetween%5Ftwo%5Fspecies">
    Is there a download file available where all UniProt IDs from X.laevis are matched to their human equivalents (homologs)? How can I obtain an ortholog mapping of human proteins to S.pombe proteins?
   </a>
  </li>
  <li>
   <a href="http://www.uniprot.org/manual/sequence%5Fsimilarities">
    Sequence similarities
   </a>
  </li>
 </ul>
</div>