<div>
 <p>
  The Prokaryotic protein annotation project focuses on the manual annotation of bacterial and archaeal-specific proteins and protein families.
 </p>
 <p>
  See:
  <a href="http://www.uniprot.org/faq/45">
   How do we manually annotate a UniProtKB entry?
  </a>
 </p>
 <p>
  Our major effort is currently directed towards the annotation of proteins from the already well-characterized model bacteria
  <em>
   <a href="http://www.uniprot.org/taxonomy/83333">
    Escherichia coli
   </a>
  </em>
  and
  <em>
   <a href="http://www.uniprot.org/taxonomy/224308">
    Bacillus subtilis
   </a>
  </em>
  , as well as the annotation of pathogens such as
  <em>
   <a href="http://www.uniprot.org/taxonomy/1773">
    Mycobacterium tuberculosis
   </a>
  </em>
  .
 </p>
 <ul>
  <li>
   All manually reviewed
   <em>
    Escherichia coli
   </em>
   entries can be found
   <a href="http://www.uniprot.org/uniprot/?query=organism:83333+AND+reviewed:yes">
    here
   </a>
   <a href="http://www.uniprot.org/biocuration%5Fproject/Prokaryotes/statistics#Escherichiacoli">
    (statistics)
   </a>
  </li>
  <li>
   All manually reviewed
   <em>
    Bacillus subtilis
   </em>
   entries can be found
   <a href="http://www.uniprot.org/uniprot/?query=organism:224308+AND+reviewed:yes">
    here
   </a>
   <a href="http://www.uniprot.org/biocuration%5Fproject/Prokaryotes/statistics#Bacillussubtilis">
    (statistics)
   </a>
  </li>
  <li>
   All manually reviewed
   <em>
    Mycobacterium tuberculosis
   </em>
   entries can be found
   <a href="http://www.uniprot.org/uniprot/?query=organism:1773+AND+reviewed:yes">
    here
   </a>
   <a href="http://www.uniprot.org/biocuration%5Fproject/Prokaryotes/statistics#Mycobacteriumtuberculosis">
    (statistics)
   </a>
  </li>
 </ul>
 <h3 id="High_quality_automated_annotation_propagation">
  High-quality automated annotation propagation
 </h3>
 <p>
  Due to the quantity of data produced today thanks to next-generation sequencing and the ever increasing rate of genome sequencing, it is no longer possible to manually annotate even a small portion of these genomes, despite the considerable demand for corrected and annotated proteomes. To enrich their annotation in UniProtKB, we developed
  <a href="http://hamap.expasy.org/">
   HAMAP
  </a>
  (High-quality Automated and Manual Annotation of Proteins), whose goal is to automatically annotate a significant percentage of the huge amount of proteins originating from genome sequencing projects. This automatic annotation pipeline, based on a collection of family profiles and manually created annotation rules, is only applied in cases where it can produce the same quality as manual annotation would, that is for proteins that are part of well-defined families or subfamilies. By this we mean protein families which have a well-defined function and which are well conserved at the sequence level.
 </p>
</div>