<div>
 <h4 id="How_to_cite_us">
  How to cite us
 </h4>
 <p>
  If you find UniProt useful, please consider citing our latest publication:
 </p>
 <p>
  The UniProt Consortium
  <br/>
  <strong>
   UniProt: the universal protein knowledgebase in 2021
  </strong>
  <br/>
  <a href="https://academic.oup.com/nar/advance-article/doi/10.1093/nar/gkaa1100/6006196">
   Nucleic Acids Res. 49:D1 (2021)
  </a>
 </p>
 <p>
  ...or choose the publication that best covers the UniProt aspects or components you used in your work:
 </p>
 <h4 id="2019">
  2019
 </h4>
 <p>
  The UniProt Consortium
  <br/>
  <strong>
   UniProt: a worldwide hub of protein knowledge
  </strong>
  <br/>
  <a href="https://doi.org/10.1093/nar/gky1049">
   Nucleic Acids Res. 47:D506-515 (2019)
  </a>
 </p>
 <p>
  Morgat A, Lombardot T, Coudert E, Axelsen K, Neto TB, Gehant S, Bansal P, Bolleman J, Gasteiger E, de Castro E, Baratin D, Pozzato M, Xenarios I, Poux S, Redaschi N, Bridge A, UniProt Consortium.
  <br/>
  <strong>
   Enzyme annotation in UniProtKB using Rhea
  </strong>
  <br/>
  <a href="https://dx.doi.org/10.1093/bioinformatics/btz817">
   Bioinformatics 36(6):1896-1901 (2019)
  </a>
 </p>
 <h4 id="2018">
  2018
 </h4>
 <p>
  The UniProt Consortium
  <br/>
  <strong>
   UniProt: the universal protein knowledgebase
  </strong>
  <br/>
  <a href="http://dx.doi.org/doi:10.1093/nar/gky092">
   Nucleic Acids Res. 46:2699 (2018)
  </a>
 </p>
 <p>
  Pichler K, Warner K, Magrane M, UniProt Consortium
  <br/>
  <strong>
   SPIN: Submitting Sequences Determined at Protein Level to UniProt
  </strong>
  <br/>
  <a href="https://doi.org/10.1002/cpbi.52">
   Curr. Protoc. Bioinformatics 62(1):e52 (2018)
  </a>
 </p>
 <h4 id="2017">
  2017
 </h4>
 <p>
  The UniProt Consortium
  <br/>
  <strong>
   UniProt: the universal protein knowledgebase
  </strong>
  <br/>
  <a href="http://dx.doi.org/doi:10.1093/nar/gkw1099">
   Nucleic Acids Res. 45:D158-D169 (2017)
  </a>
 </p>
 <p>
  Chen C, Huang H, Wu CH.
  <br/>
  <strong>
   Protein Bioinformatics Databases and Resources
  </strong>
  <br/>
  <a href="https://www.ncbi.nlm.nih.gov/pubmed/28150231">
   Methods Mol. Biol. 1558:3-39 (2017)
  </a>
 </p>
 <p>
  Ding R, Boutet E, Lieberherr D, Schneider M, Tognolli M, Wu CH, Vijay-Shanker K, Arighi CN.
  <br/>
  <strong>
   eGenPub, a text mining system for extending computationally mapped bibliography for UniProt Knowledgebase by capturing centrality
  </strong>
  <br/>
  <a href="https://doi.org/10.1093/database/bax081">
   Database (Oxford) (2017)
  </a>
 </p>
 <p>
  Nightingale A, Antunes R, Alpi E, Bursteinas B, Gonzales L, Liu W, Luo J, Qi G, Turner E, Martin M.
  <br/>
  <strong>
   The Proteins API: accessing key integrated protein and genome information
  </strong>
  <br/>
  <a href="https://academic.oup.com/nar/article-lookup/doi/10.1093/nar/gkx237">
   Nucleic Acids Res. 45:W539-W544 (2017)
  </a>
 </p>
 <p>
  Poux S, Arighi, CN, Magrane M, Bateman A, Wei C-H, Lu Z, Boutet E, Bye-A-Jee H, Famiglietti ML, Roechert B., UniProt Consortium
  <br/>
  <strong>
   On expert curation and scalability: UniProtKB/Swiss-Prot as a case study
  </strong>
  <br/>
  <a href="https://academic.oup.com/bioinformatics/article/33/21/3454/3964379" title="2017">
   Bioinformatics 33:3454-3460
  </a>
 </p>
 <p>
  Pundir S, Martin MJ, O'Donovan C.
  <br/>
  <strong>
   UniProt Protein Knowledgebase
  </strong>
  <br/>
  <a href="https://link.springer.com/protocol/10.1007%2F978-1-4939-6783-4%5F2">
   Methods Mol. Biol. 1558:41-55 (2017)
  </a>
 </p>
 <p>
  Watkins X, Garcia LJ, Pundir S, Martin MJ, UniProt Consortium
  <br/>
  <strong>
   ProtVista: visualization of protein sequence annotations
  </strong>
  <br/>
  <a href="https://academic.oup.com/bioinformatics/article-lookup/doi/10.1093/bioinformatics/btx120">
   Bioinformatics 33:2040-2041 (2017)
  </a>
 </p>
 <p>
  Zaru R, Magrane M, O'Donovan C, UniProt Consortium
  <br/>
  <strong>
   From the research laboratory to the database: the Caenorhabditis elegans kinome in UniProtKB
  </strong>
  <br/>
  <a href="http://www.biochemj.org/content/474/4/493">
   Biochem J. 474(4):493-515 (2017)
  </a>
 </p>
 <h4 id="2016">
  2016
 </h4>
 <p>
  Boutet E, Lieberherr D, Tognolli M, Schneider M, Bansal P, Bridge AJ, Poux S, Bougueleret L, Xenarios I.
  <br/>
  <strong>
   UniProtKB/Swiss-Prot, the Manually Annotated Section of the UniProt KnowledgeBase: How to Use the Entry View
  </strong>
  <br/>
  <a href="https://www.ncbi.nlm.nih.gov/pubmed/26519399">
   Methods Mol. Biol. 1374:23-54 (2016)
  </a>
 </p>
 <p>
  Breuza L, Poux S, Estreicher A, Famiglietti ML, Magrane M, Tognolli M, Bridge A, Baratin D, Redaschi N, UniProt Consortium
  <br/>
  <strong>
   The UniProtKB guide to the human proteome
  </strong>
  <br/>
  <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4761109/">
   Database (Oxford) (2016)
  </a>
 </p>
 <p>
  Bursteinas B, Britto R, Bely B, Auchincloss A, Rivoire C, Redaschi N, O'Donovan C, Martin MJ.
  <br/>
  <strong>
   Minimizing proteome redundancy in the UniProt Knowledgebase
  </strong>
  <br/>
  <a href="https://www.ncbi.nlm.nih.gov/pubmed/28025334">
   Database (Oxford) (2016)
  </a>
 </p>
 <p>
  Chen C, Huang H, Mazumder R, Natale DA, McGarvey PB, Zhang J, Polson SW, Wang Y, Wu CH, UniProt Consortium
  <br/>
  <strong>
   Computational Clustering for Viral Reference Proteomes
  </strong>
  <br/>
  <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=PMC4920120">
   Bioinformatics 32:2041-2043 (2016)
  </a>
 </p>
 <p>
  Dogan T, MacDougall A, Saidi R, Poggioli D, Bateman A, O'Donovan C, Martin MJ.
  <br/>
  <strong>
   UniProt-DAAC: domain architecture alignment and classification, a new method for automatic functional annotation in UniProtKB
  </strong>
  <br/>
  <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4965628/">
   Bioinformatics 32:2264-2271 (2016)
  </a>
 </p>
 <p>
  Kilic S, Sagitova DM, Wolfish S, Bely B, Courtot M, Ciufo S, Tatusova T, O'Donovan C, Chibucos MC, Martin MJ, Erill I.
  <br/>
  <strong>
   From data repositories to submission portals: rethinking the role of domain-specific databases in CollecTF
  </strong>
  <br/>
  <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4843526/">
   Database (Oxford) (2016)
  </a>
 </p>
 <p>
  Pundir S, Martin MJ, O'Donovan C, UniProt Consortium
  <br/>
  <strong>
   UniProt Tools
  </strong>
  <br/>
  <a href="https://www.ncbi.nlm.nih.gov/pubmed/27010333">
   Curr. Protoc. Bioinformatics 53:1.29.1-1.29.15 (2016)
  </a>
 </p>
 <h4 id="2015">
  2015
 </h4>
 <p>
  Alpi E, Griss J, da Silva AW, Bely B, Antunes R, Zellner H, Rios D, O'Donovan C, Vizcaino JA, Martin MJ.
  <br/>
  <strong>
   Analysis of the tryptic search space in UniProt databases
  </strong>
  <br/>
  <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4298651/">
   Proteomics 15:48-57 (2015)
  </a>
 </p>
 <p>
  Bastian FB, Chibucos MC, Gaudet P, Giglio M, Holliday GL, Huang H, Lewis SE, Niknejad A, Orchard S, Poux S, Skunca N, Robinson-Rechavi M.
  <br/>
  <strong>
   The Confidence Information Ontology: a step towards a standard for asserting confidence in annotations
  </strong>
  <br/>
  <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4425939/">
   Database (Oxford) bav043 (2015)
  </a>
 </p>
 <p>
  Foulger RE, Osumi-Sutherland D, McIntosh BK, Hulo C, Masson P, Poux S, Le Mercier P, Lomax J.
  <br/>
  <strong>
   Representing virus-host interactions and other multi-organism processes in the Gene Ontology
  </strong>
  <br/>
  <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4517558/">
   BMC Microbiol. 15:146 (2015)
  </a>
 </p>
 <p>
  Holliday GL, Bairoch A, Bagos PG, Chatonnet A, Craik DJ, Finn RD, Henrissat B, Landsman D, Manning G, Nagano N, O'Donovan C, Pruitt KD, Rawlings ND, Saier M, Sowdhamini R, Spedding M, Srinivasan N, Vriend G, Babbitt PC, Bateman A.
  <br/>
  <strong>
   Key challenges for the creation and maintenance of specialist protein resources
  </strong>
  <br/>
  <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4446195/">
   Proteins 83:1005-1013 (2015)
  </a>
 </p>
 <p>
  Huntley RP, Sawford T, Mutowo-Meullenet P, Shypitsyna A, Bonilla C, Martin MJ, O'Donovan C.
  <br/>
  <strong>
   The GOA database: gene Ontology annotation updates for 2015
  </strong>
  <br/>
  <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4383930/">
   Nucleic Acids Res. 43:d1057-1063 (2015)
  </a>
 </p>
 <p>
  Pedruzzi I, Rivoire C, Auchincloss AH, Coudert E, Keller G, de Castro E, Baratin D, Cuche BA, Bougueleret L, Poux S, Redaschi N, Xenarios I, Bridge A.
  <br/>
  <strong>
   HAMAP in 2015: updates to the protein family classification and annotation system
  </strong>
  <br/>
  <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4383873/">
   Nucleic Acids Res. 43:D1064-1070 (2015)
  </a>
 </p>
 <p>
  Pundir S, Magrane M, Martin MJ, O'Donovan C, UniProt Consortium
  <br/>
  <strong>
   Searching and Navigating UniProt Databases
  </strong>
  <br/>
  <a href="https://www.ncbi.nlm.nih.gov/pubmed/26088053">
   Curr. Protoc. Bioinformatics 50:1.27.1-1.27.10 (2015)
  </a>
 </p>
 <p>
  Suzek BE, Wang Y, Huang H, McGarvey PB, Wu CH, UniProt Consortium
  <br/>
  <strong>
   UniRef clusters: a comprehensive and scalable alternative for improving sequence similarity searches
  </strong>
  <br/>
  <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4375400/">
   Bioinformatics 31:926-932 (2015)
  </a>
 </p>
 <p>
  The UniProt Consortium
  <br/>
  <strong>
   UniProt: a hub for protein information
  </strong>
  <br/>
  <a href="https://www.ncbi.nlm.nih.gov/pubmed/25348405">
   Nucleic Acids Res. 43:D204-212 (2015)
  </a>
 </p>
 <h4 id="2014">
  2014
 </h4>
 <p>
  Famiglietti ML, Estreicher A, Gos A, Bolleman J, Gehant S, Breuza L, Bridge A, Poux S, Redaschi N, Bougueleret L, Xenarios I.
  <br/>
  <strong>
   Genetic Variations and Diseases in UniProtKB/Swiss-Prot: The Ins and Outs of Expert Manual Curation.
  </strong>
  <br/>
  <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4107114/">
   Hum. Mutat. 35:927-935 (2014)
  </a>
 </p>
 <p>
  Huntley RP, Sawford T, Martin MJ, O'Donovan C.
  <br/>
  <strong>
   Understanding how and why the Gene Ontology and its annotations evolve: the GO within UniProt
  </strong>
  <br/>
  <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3995153/">
   Gigascience 3:4 (2014)
  </a>
 </p>
 <p>
  Masson P, Hulo C, de Castro E, Foulger R, Poux S, Bridge A, Lomax J, Bougueleret L, Xenarios I, Le Mercier P.
  <br/>
  <strong>
   An Integrated Ontology Resource to Explore and Study Host-Virus Relationships
  </strong>
  <br/>
  <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4169452/">
   PLoS One 9(9):e108075 (2014)
  </a>
 </p>
 <p>
  Poux S, Magrane M, Arighi CN, Bridge A, O'Donovan C, Laiho K, The UniProt Consortium
  <br/>
  <strong>
   Expert curation in UniProtKB: a case study on dealing with conflicting and erroneous data
  </strong>
  <br/>
  <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3950660/">
   Database (Oxford) bau016 (2014)
  </a>
 </p>
 <p>
  The UniProt Consortium
  <br/>
  <strong>
   Activities at the Universal Protein Resource (UniProt)
  </strong>
  <br/>
  <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=PMC3965022">
   Nucl. Acids Res. 42:D191-D198 (2014)
  </a>
 </p>
 <p>
  Welter D, MacArthur J, Morales J, Burdett T, Hall P, Junkins H, Klemm A, Flicek P, Manolio T, Hindorff L, Parkinson H.
  <br/>
  <strong>
   The NHGRI GWAS Catalog, a curated resource of SNP-trait associations
  </strong>
  <br/>
  <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=PMC3965119">
   Nucleic Acids Res. 42:D1001-D1006 (2014)
  </a>
 </p>
 <h4 id="2013">
  2013
 </h4>
 <p>
  Chen C, Li Z, Huang H, Suzek BE, Wu CH; UniProt Consortium
  <br/>
  <strong>
   A fast Peptide Match service for UniProt Knowledgebase
  </strong>
  <br/>
  <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3799477/">
   Bioinformatics 29:2808-2809 (2013)
  </a>
 </p>
 <p>
  Mutowo-Meullenet P, Huntley RP, Dimmer EC, Alam-Faruque Y, Sawford T, Martin MJ, O'Donovan C, Apweiler R.
  <br/>
  <strong>
   Use of Gene Ontology Annotation to understand the peroxisome proteome in humans
  </strong>
  <br/>
  <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3548334/">
   Database (Oxford) bas062 (2013)
  </a>
 </p>
 <p>
  Pedruzzi I, Rivoire C, Auchincloss AH, Coudert E, Keller G, de Castro E, Baratin D, Cuche BA, Bougueleret L, Poux S, Redaschi N, Xenarios I, Bridge A, UniProt Consortium
  <br/>
  <strong>
   HAMAP in 2013, new developments in the protein family classification and annotation system
  </strong>
  <br/>
  <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=PMC3531088">
   Nucl. Acids Res. 41:D584-D589 (2013)
  </a>
 </p>
 <p>
  The UniProt Consortium
  <br/>
  <strong>
   Update on activities at the Universal Protein Resource (UniProt) in 2013
  </strong>
  <br/>
  <a href="http://dx.doi.org/doi:10.1093/nar/gks1068">
   Nucleic Acids Res. 41: D43-D47 (2013)
  </a>
 </p>
 <p>
  Velankar S, Dana JM, Jacobsen J, van Ginkel G, Gane PJ, Luo J, Oldfield TJ, O'Donovan C, Martin MJ, Kleywegt GJ.
  <br/>
  <strong>
   SIFTS: Structure Integration with Function, Taxonomy and Sequences resource
  </strong>
  <br/>
  <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=PMC3531078">
   Nucleic Acids Res. 41:D483-D4839 (2013)
  </a>
 </p>
 <p>
  Veuthey AL, Bridge A, Gobeill J, Ruch P, McEntyre JR, Bougueleret L, Xenarios I.
  <br/>
  <strong>
   Application of text-mining for updating protein post-translational modification annotation in UniProtKB
  </strong>
  <br/>
  <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=PMC3660268">
   BMC Bioinformatics 14:104 (2013)
  </a>
 </p>
 <h4 id="2012">
  2012
 </h4>
 <p>
  Burge S, Attwood TK, Bateman A, Berardini TZ, Cherry M, O'Donovan C, Xenarios I, Gaudet P.
  <br/>
  <strong>
   Biocurators and Biocuration: surveying the 21st century challenges
  </strong>
  <br/>
  <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=PMC3308150">
   Database (Oxford) (2012)
  </a>
 </p>
 <p>
  Dimmer EC, Huntley RP, Alam-Faruque Y, Sawford T, O'Donovan C, Martin MJ, Bely B, Browne P, Mun Chan W, Eberhardt R, Gardner M, Laiho K, Legge D, Magrane M, Pichler K, Poggioli D, Sehra H, Auchincloss A, Axelsen K, Blatter MC, Boutet E, Braconi-Quintaje S, Breuza L, Bridge A, Coudert E, Estreicher A, Famiglietti L, Ferro-Rojas S, Feuermann M, Gos A, Gruaz-Gumowski N, Hinz U, Hulo C, James J, Jimenez S, Jungo F, Keller G, Lemercier P, Lieberherr D, Masson P, Moinat M, Pedruzzi I, Poux S, Rivoire C, Roechert B, Schneider M, Stutz A, Sundaram S, Tognolli M, Bougueleret L, Argoud-Puy G, Cusin I, Duek-Roggli P, Xenarios I, Apweiler R.
  <br/>
  <strong>
   The UniProt-GO Annotation database in 2011
  </strong>
  <br/>
  <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=PMC3245010">
   Nucleic Acids Res. 40:D565-D570 (2012)
  </a>
 </p>
 <p>
  Jungo F, Bougueleret L, Xenarios I, Poux S.
  <br/>
  <strong>
   The UniProtKB/Swiss-Prot Tox-Prot program: A central hub of integrated venom protein data
  </strong>
  <br/>
  <a href="http://dx.doi.org/10.1016/j.toxicon.2012.03.010">
   Toxicon 60:551-557 (2012)
  </a>
 </p>
 <p>
  Schneider M, Poux S, UniProt Consortium
  <br/>
  <strong>
   UniProtKB amid the turmoil of plant proteomics research front
  </strong>
  <br/>
  <a href="https://dx.doi.org/doi:10.3389/fpls.2012.00270">
   Front. Plant Sci. 3:270-270 (2012)
  </a>
 </p>
 <p>
  Wang Q, Arighi CN, King BL, Polson SW, Vincent J, Chen C, Huang H, Kingham BF, Page ST, Rendino MF, Thomas WK, Udwary DW, Wu CH.
  <br/>
  <strong>
   North East Bioinformatics Collaborative Curation Team. Community annotation and bioinformatics workforce development in concert--Little Skate Genome Annotation Workshops and Jamborees
  </strong>
  <br/>
  <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=PMC3308154">
   Database (Oxford) (2012)
  </a>
 </p>
 <p>
  The UniProt Consortium
  <br/>
  <strong>
   Reorganizing the protein space at the Universal Protein Resource (UniProt)
  </strong>
  <br/>
  <a href="http://dx.doi.org/doi:10.1093/nar/gkr981">
   Nucleic Acids Res. 40: D71-D75 (2012)
  </a>
 </p>
 <h4 id="2011">
  2011
 </h4>
 <p>
  Alam-Faruque Y, Huntley RP, Khodiyar VK, Camon EB, Dimmer EC, Sawford T, Martin MJ, O'Donovan C, Talmud PJ, Scambler P, Apweiler R, Lovering RC.
  <br/>
  <strong>
   The impact of focused Gene Ontology curation of specific mammalian systems
  </strong>
  <br/>
  <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=PMC3235096">
   Plos One (2011)
  </a>
 </p>
 <p>
  Burmester A, Shelest E, Glockner G, Heddergott C, Schindler S, Staib P, Heidel A, Felder M, Petzold A, Szafranski K, Feuermann M, Pedruzzi I, Priebe S, Groth M, Winkler R, Li W, Kniemeyer O, Schroeckh V, Hertweck C, Hube B, White TC, Platzer M, Guthke R, Heitman J, Wostemeyer J, Zipfel PF, Monod M, Brakhage AA.
  <br/>
  <strong>
   Comparative and functional genomics provide insights into the pathogenicity of dermatophytic fungi
  </strong>
  <br/>
  <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=PMC3091305">
   Genome Biol. 12:R7 (2011)
  </a>
 </p>
 <p>
  Chen C, Natale DA, Finn RD, Huang H, Zhang J, Wu CH, Mazumder R.
  <br/>
  <strong>
   Representative Proteomes: a stable, scalable and unbiased proteome set for sequence analysis and functional annotation
  </strong>
  <br/>
  <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=PMC3083393">
   PLOS One (2011)
  </a>
 </p>
 <p>
  Griss J, Martin M, O'Donovan C, Apweiler R, Hermjakob H, Vizcaino JA.
  <br/>
  <strong>
   Consequences of the discontinuation of the International Protein Index (IPI) database and its substitution by the UniProtKB "complete proteome" sets
  </strong>
  <br/>
  <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=PMC3556690">
   Proteomics 11:4434-4438 (2011)
  </a>
 </p>
 <p>
  Huang H, McGarvey PB, Suzek BE, Mazumder R, Zhang J, Chen Y, Wu CH.
  <br/>
  <strong>
   A comprehensive protein-centric ID mapping service for molecular data integration
  </strong>
  <br/>
  <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=PMC3072559">
   Bioinformatics 27:1190-1191 (2011)
  </a>
 </p>
 <p>
  Klimke W, O'Donovan C, White O, Brister JR, Clark K, Fedorov B, Mizrachi I, Pruitt KD, Tatusova T.
  <br/>
  <strong>
   Solving the Problem: Genome Annotation Standards before the Data Deluge
  </strong>
  <br/>
  <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=PMC3236044">
   Stand. Genomic Sci. 5:168-193 (2011)
  </a>
 </p>
 <p>
  Magrane M, UniProt Consortium
  <br/>
  <strong>
   UniProt Knowledgebase: a hub of integrated protein data
  </strong>
  <br/>
  <a href="https://dx.doi.org/doi:10.1093/database/bar009">
   Database, 2011: bar009 (2011)
  </a>
 </p>
 <p>
  O'Donovan C, Apweiler R.
  <br/>
  <strong>
   A guide to UniProt for protein scientists
  </strong>
  <br/>
  <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=PMID%3A+21082425">
   Methods Mol. Biol. 694:25-35 (2011)
  </a>
 </p>
 <p>
  Patient S, Martin M.
  <br/>
  <strong>
   Annotating UniProt metagenomic and environmental sequences in UniMES
  </strong>
  <br/>
  In: Bioinformatics 2011 - Proceedings of the International Conference on Bioinformatics Models, Methods and Algorithms. SciTePress 367-368 (2011)
 </p>
 <p>
  The UniProt Consortium
  <br/>
  <strong>
   Ongoing and future developments at the Universal Protein Resource
  </strong>
  <br/>
  <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=PMC3013648">
   Nucleic Acids Res. 39:D214-D219 (2011)
  </a>
 </p>
 <p>
  Vasudevan S, Vinayaka CR, Natale DA, Huang H, Kahsay RY, Wu CH.
  <br/>
  <strong>
   Structure-guided rule-based annotation of protein functional sites in UniProt Knowledgebase
  </strong>
  <br/>
  <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=PMID%3A+21082430">
   Methods Mol. Biol. 694:91-105 (2011)
  </a>
 </p>
 <h4 id="2010">
  2010
 </h4>
 <p>
  Hinz U, UniProt Consortium
  <br/>
  <strong>
   From protein sequences to 3D-structures and beyond: the example of the UniProt knowledgebase
  </strong>
  <br/>
  <a href="http://dx.doi.org/10.1007/s00018-009-0229-6">
   Cell. Mol. Life Sci. 67:1049-1064 (2010)
  </a>
 </p>
 <p>
  Jungo F, Estreicher A, Bairoch A, Bougueleret L, Xenarios I.
  <br/>
  <strong>
   Animal Toxins: How is complexity represented in databases?
  </strong>
  <br/>
  <a href="http://dx.doi.org/doi:10.3390/toxins2020261">
   Toxins 2:262-282 (2010)
  </a>
 </p>
 <p>
  The UniProt Consortium
  <br/>
  <strong>
   The Universal Protein Resource (UniProt) in 2010
  </strong>
  <br/>
  <a href="http://nar.oxfordjournals.org/content/38/suppl%5F1/D142.long">
   Nucleic Acids Res. 38:D142-D148 (2010)
  </a>
 </p>
 <h4 id="2009">
  2009
 </h4>
 <p>
  Cochrane G, Martin MJ, Apweiler R.
  <br/>
  <strong>
   Public data resources as a foundation for a worldwide genomics data infrastructure
  </strong>
  <br/>
  In: Metagenomics: Theory, methods, and applications. Marco D (Ed), pp 79-105 (2011)
 </p>
 <p>
  Jain E, Bairoch A, Duvaud S, Phan I, Redaschi N, Suzek BE, Martin MJ, McGarvey P, Gasteiger E.
  <br/>
  <strong>
   Infrastructure for the life sciences: design and implementation of the UniProt website
  </strong>
  <br/>
  <a href="http://dx.doi.org/doi:10.1186/1471-2105-10-136">
   BMC Bioinformatics 10:136 (2009)
  </a>
 </p>
 <p>
  Lima T, Auchincloss AH, Coudert E, Keller G, Michoud K, Rivoire C, Bulliard V, de Castro E, Lachaize C, Baratin D, Phan I, Bougueleret L, Bairoch A.
  <br/>
  <strong>
   HAMAP: a database of completely sequenced microbial proteome sets and manually curated microbial protein families in UniProtKB/Swiss-Prot
  </strong>
  <br/>
  <a href="http://nar.oxfordjournals.org/content/37/suppl%5F1/D471.long">
   Nucleic Acids Res. 37:D471-D478 (2009)
  </a>
 </p>
 <p>
  Schneider M, Lane L, Boutet E, Lieberherr D, Tognolli M, Bougueleret L, Bairoch A.
  <br/>
  <strong>
   The UniProtKB/Swiss-Prot knowledgebase and its Plant Proteome Annotation Program
  </strong>
  <br/>
  <a href="http://dx.doi.org/10.1016/j.jprot.2008.11.010">
   J. Proteomics 72:567-573 (2009)
  </a>
 </p>
 <p>
  The UniProt Consortium
  <br/>
  <strong>
   The Universal Protein Resource (UniProt)
  </strong>
  <br/>
  <a href="http://nar.oxfordjournals.org/content/37/suppl%5F1/D169.long">
   Nucleic Acids Res. 37:D169-D174 (2009)
  </a>
 </p>
 <h4 id="2008">
  2008
 </h4>
 <p>
  Braconi Quintaje S, Orchard S.
  <br/>
  <strong>
   The annotation of both human and mouse kinomes in UniProtKB/Swiss-Prot: one small step in manual annotation, one giant leap for full comprehension of genomes
  </strong>
  <br/>
  <a href="http://dx.doi.org/10.1074/mcp.R700001-MCP200">
   Mol. Cell. Proteomics 7:1409-1419 (2008)
  </a>
 </p>
 <p>
  Bridge A, Veuthey A-L.
  <br/>
  <strong>
   Resources for functional annotation
  </strong>
  <br/>
  In: Modern genome annotation: The biosapiens network. Frishman D, Valencia A (eds), pp 139-64, Springer, Cambridge, UK (2008)
 </p>
 <p>
  David FP, Yip YL.
  <br/>
  <strong>
   SSMap: a new UniProt-PDB mapping resource for the curation of structural-related information in the UniProt/Swiss-Prot Knowledgebase
  </strong>
  <br/>
  <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2567350/">
   BMC Bioinformatics 23:391 (2008)
  </a>
 </p>
 <p>
  Hinz U, Bairoch A.
  <br/>
  <strong>
   The impact of 3d structures on a protein knowledgebase: From proteins to systems
  </strong>
  <br/>
  In: Structural proteomics and its impact on the life sciences. Sussman JL, Silman I (eds), World Scientific Publishing, Israel (2008)
 </p>
 <p>
  Mottaz A, Yip YL, Ruch P, Veuthey AL.
  <br/>
  <strong>
   Mapping proteins to disease terminologies: from UniProt to MeSH
  </strong>
  <br/>
  <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2367626/">
   BMC Bioinformatics 29:S3 (2008)
  </a>
 </p>
 <p>
  Mulder NJ, Kersey P, Pruess M, Apweiler R.
  <br/>
  <strong>
   In silico characterization of proteins: UniProt, InterPro and Integr8
  </strong>
  <br/>
  <a href="https://www.ncbi.nlm.nih.gov/pubmed/18219596">
   Mol. Biotechnol. 38:165-177 (2008)
  </a>
 </p>
 <p>
  Patient S, Wieser D, Kleen M, Kretschmann E, Martin M, Apweiler R.
  <br/>
  <strong>
   UniProtJAPI: a remote API for accessing UniProt data
  </strong>
  <br/>
  <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=uniprotjapi">
   Bioinformatics 24:1321-1322 (2008)
  </a>
 </p>
 <p>
  The UniProt Consortium
  <br/>
  <strong>
   The Universal Protein Resource (UniProt)
  </strong>
  <br/>
  <a href="http://nar.oxfordjournals.org/content/36/suppl%5F1/D190.long">
   Nucleic Acids Res. 36:D190-D195 (2008)
  </a>
 </p>
 <p>
  Yip YL, Famiglietti M, Gos A, Duek PD, David FP, Gateau A, Bairoch A.
  <br/>
  <strong>
   Annotating single amino acid polymorphisms in the UniProt/Swiss-Prot knowledgebase
  </strong>
  <br/>
  <a href="https://www.ncbi.nlm.nih.gov/pubmed/18175334">
   Hum. Mutat. 29:361-366 (2008)
  </a>
 </p>
 <h4 id="2007">
  2007
 </h4>
 <p>
  Boutet E, Lieberherr D, Tognolli M, Schneider M, Bairoch A.
  <br/>
  <strong>
   UniProtKB/Swiss-Prot
  </strong>
  <br/>
  <a href="https://www.ncbi.nlm.nih.gov/pubmed/18287689">
   Methods Mol. Biol. 406:89-112 (2007)
  </a>
 </p>
 <p>
  Hu ZZ, Valencia JC, Huang H, Chi A, Shabanowitz J, Hearing VJ, Appella E, Wu C.
  <br/>
  <strong>
   Comparative Bioinformatics Analyses and Profiling of Lysosome-Related Organelle Proteomes
  </strong>
  <br/>
  <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC1828028/">
   Int. J. Mass Spectrom. 259:147-160 (2007)
  </a>
 </p>
 <p>
  Mueller M, Martens L, Apweiler R.
  <br/>
  <strong>
   Annotating the human proteome: beyond establishing a parts list
  </strong>
  <br/>
  <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=Annotating+the+human+proteome%3A+beyond+astablishing">
   Biochim. Biophys. Acta 1774:175-191 (2007)
  </a>
 </p>
 <p>
  Qiu P, Wang J, Ray Liu KJ, Hu ZZ, Wu CH.
  <br/>
  <strong>
   Dependence network modeling for biomarker identification
  </strong>
  <br/>
  <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=dependence+network+modeling+for+biomarker+identification">
   Bioinformatics 23:198-206 (2007)
  </a>
 </p>
 <p>
  Rice Annotation Project, Itoh T, Tanaka T, Barrero RA, Yamasaki C, Fujii Y, Hilton PB, Antonio BA, Aono H, Apweiler R, Bruskiewich R, Bureau T, Burr F, Costa de Oliveira A, Fuks G, Habara T, Haberer G, Han B, Harada E, Hiraki AT, Hirochika H, Hoen D, Hokari H, Hosokawa S, Hsing YI, Ikawa H, Ikeo K, Imanishi T, Ito Y, Jaiswal P, Kanno M, Kawahara Y, Kawamura T, Kawashima H, Khurana JP, Kikuchi S, Komatsu S, Koyanagi KO, Kubooka H, Lieberherr D, Lin YC, Lonsdale D, Matsumoto T, Matsuya A, McCombie WR, Messing J, Miyao A, Mulder N, Nagamura Y, Nam J, Namiki N, Numa H, Nurimoto S, O'Donovan C, Ohyanagi H, Okido T, Oota S, Osato N, Palmer LE, Quetier F, Raghuvanshi S, Saichi N, Sakai H, Sakai Y, Sakata K, Sakurai T, Sato F, Sato Y, Schoof H, Seki M, Shibata M, Shimizu Y, Shinozaki K, Shinso Y, Singh NK, Smith-White B, Takeda J, Tanino M, Tatusova T, Thongjuea S, Todokoro F, Tsugane M, Tyagi AK, Vanavichit A, Wang A, Wing RA, Yamaguchi K, Yamamoto M, Yamamoto N, Yu Y, Zhang H, Zhao Q, Higo K, Burr B, Gojobori T, Sasaki T.
  <br/>
  <strong>
   Curated genome annotation of Oryza sativa ssp. japonica and comparative genome analysis with Arabidopsis thaliana
  </strong>
  <br/>
  <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC1781349/">
   Genome Res. 17:175-183 (2007)
  </a>
 </p>
 <p>
  Suzek BE, Huang H, McGarvey P, Mazumder R, Wu CH.
  <br/>
  <strong>
   UniRef: Comprehensive and Non-Redundant UniProt Reference Clusters
  </strong>
  <br/>
  <a href="http://bioinformatics.oxfordjournals.org/content/23/10/1282.long">
   Bioinformatics 23:1282-1288 (2007)
  </a>
 </p>
 <p>
  The UniProt Consortium
  <br/>
  <strong>
   The Universal Protein Resource (UniProt)
  </strong>
  <br/>
  <a href="http://nar.oxfordjournals.org/content/35/suppl%5F1/D193.long">
   Nucleic Acids Res. 35:D193-D197 (2007)
  </a>
 </p>
 <p>
  Yip YL, Lachenal N, Pillet V, Veuthey AL.
  <br/>
  <strong>
   Retrieving mutation-specific information for human proteins in UniProt/Swiss-Prot Knowledgebase
  </strong>
  <br/>
  <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=retrieving+mutation-specific+information+for+human+proteins+in+uniprot">
   J. Bioinform. Comput. Biol. 5:1215-1231 (2007)
  </a>
 </p>
 <h4 id="2006">
  2006
 </h4>
 <p>
  Chi A, Valencia JC, Hu ZZ, Watabe H, Yamaguchi H, Mangini NJ, Huang H, Canfield VA, Cheng KC, Yang F, Abe R, Yamagishi S, Shabanowitz J, Hearing VJ, Wu C, Appella E, Hunt DF.
  <br/>
  <strong>
   Proteomic and bioinformatic characterization of the biogenesis and function of melanosomes
  </strong>
  <br/>
  <a href="https://www.ncbi.nlm.nih.gov/pubmed/17081065">
   J. Proteome Res. 5:3135-3144 (2006)
  </a>
 </p>
 <p>
  Han B, Obradovic Z, Hu ZZ, Wu CH, Vucetic S.
  <br/>
  <strong>
   Substring selection for biomedical document classification
  </strong>
  <br/>
  <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=substring+selection+for+biomedical+document+classification">
   Bioinformatics 22:2136-2142 (2006)
  </a>
 </p>
 <p>
  Leinonen R, Nardone F, Zhu W, Apweiler R.
  <br/>
  <strong>
   UniSave: the UniProtKB Sequence/Annotation Version database
  </strong>
  <br/>
  <a href="http://bioinformatics.oxfordjournals.org/content/22/10/1284.long">
   Bioinformatics 22:1284-1285 (2006)
  </a>
 </p>
 <p>
  Liu H, Hu ZZ, Torii M, Wu C, Friedman CJ.
  <br/>
  <strong>
   Quantitative assessment of dictionary-based protein named entity tagging
  </strong>
  <br/>
  <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC1561801/">
   Am. Med. Inform. Assoc. 13:497-507 (2006)
  </a>
 </p>
 <p>
  Liu HF, Hu ZZ, Zhang J, Wu CH.
  <br/>
  <strong>
   BioThesaurus: a web-based thesaurus of protein and gene names
  </strong>
  <br/>
  <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=BioThesaurus%3A+a+web-based+thesaurus+of+protein+and+gene+names">
   Bioinformatics 22:103-105 (2006)
  </a>
 </p>
 <p>
  Nikolskaya AN, Arighi C, Huang H, Barker WC, Wu CH.
  <br/>
  <strong>
   PIRSF family classification system for protein functional and evolutionary analysis
  </strong>
  <br/>
  <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2674652/">
   Evolutionary Bioinformatics Online 2:209-221 (2006)
  </a>
 </p>
 <p>
  Petrova NV, Wu CH.
  <br/>
  <strong>
   Prediction of catalytic residues using Support Vector Machine with selected protein sequence and structural properties
  </strong>
  <br/>
  <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC1534064/">
   BMC Bioinformatics 7:312 (2006)
  </a>
 </p>
 <p>
  Pruess M, Kersey P, Kulikova T, Apweiler R.
  <br/>
  <strong>
   Databases and resources for in silico proteome analysis
  </strong>
  <br/>
  <a href="https://www.ncbi.nlm.nih.gov/pubmed/16929689">
   Methods Biochem. Anal. 49:397-414 (2006)
  </a>
 </p>
 <p>
  Schneider M, Bairoch A, Wu CH, Apweiler R.
  <br/>
  <strong>
   Plant Protein Annotation in the UniProt Knowledgebase
  </strong>
  <br/>
  <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC1104161/">
   Plant Physiology 138:59-66 (2006)
  </a>
 </p>
 <p>
  Whitfield EJ, Pruess M, Apweiler R.
  <br/>
  <strong>
   Bioinformatics database infrastructure for biotechnology research
  </strong>
  <br/>
  <a href="https://www.ncbi.nlm.nih.gov/pubmed/16757051">
   Biotechnol. 124:629-639 (2006)
  </a>
 </p>
 <p>
  Wu CH, Apweiler R, Bairoch A, Natale DA, Barker WC, Boeckmann B, Ferro S, Gasteiger E, Huang H, Lopez R, Magrane M, Martin MJ, Mazumder R, O'Donovan C, Redaschi N, Suzek B.
  <br/>
  <strong>
   The Universal Protein Resource (UniProt): an expanding universe of protein information
  </strong>
  <br/>
  <a href="http://nar.oxfordjournals.org/content/34/suppl%5F1/D187.long">
   Nucleic Acids Res. 34:D187-D191 (2006)
  </a>
 </p>
 <p>
  Yuan X, Hu ZZ, Wu HT, Torii M, Narayanaswamy M, Ravikumar KE, Vijay-Shanker K, Wu CH.
  <br/>
  <strong>
   An online literature mining tool for protein phosphorylation
  </strong>
  <br/>
  <a href="https://www.ncbi.nlm.nih.gov/pubmed/16644790">
   Bioinformatics 22:1668-1669 (2006)
  </a>
 </p>
 <h4 id="2005">
  2005
 </h4>
 <p>
  Bairoch A, Apweiler R, Wu CH, Barker WC, Boeckmann B, Ferro S, Gasteiger E, Huang H, Lopez R, Magrane M, Martin MJ, Natale DA, O'Donovan C, Redaschi N, Yeh LS.
  <br/>
  <strong>
   The Universal Protein Resource (UniProt)
  </strong>
  <br/>
  <a href="http://dx.doi.org/10.1093/nar/gki070">
   Nucleic Acids Res. 33:D154-D159 (2005)
  </a>
 </p>
 <p>
  Boeckmann B, Blatter M-C, Famiglietti L, Hinz U, Lane L, Roechert B, Bairoch A.
  <br/>
  <strong>
   Protein variety and functional diversity: Swiss-Prot annotation in its biological context
  </strong>
  <br/>
  <a href="http://dx.doi.org/10.1016/j.crvi.2005.06.001">
   C. R. Biol. 328:882-899 (2005)
  </a>
 </p>
 <p>
  Hu ZZ, Narayanaswamy M, Ravikumar KE, Vijay-Shanker K, Wu CH.
  <br/>
  <strong>
   Literature mining and database annotation of protein phosphorylation using a rule-based system
  </strong>
  <br/>
  <a href="https://www.ncbi.nlm.nih.gov/pubmed/15814565">
   Bioinformatics 21:2759-2765 (2005)
  </a>
 </p>
 <p>
  Jones P, Vinod N, Down T, Hackmann A, Kahari A, Kretschmann E, Wieder D, Hermjakob H, Apweiler R.
  <br/>
  <strong>
   Dasty and UniProt DAS: a perfect pair for protein feature visualization
  </strong>
  <br/>
  <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=dasty+and%20uniprot%20das">
   Bioinformatics 21:3198-3199 (2005)
  </a>
 </p>
 <p>
  Jungo F, Bairoch A.
  <br/>
  <strong>
   Tox-Prot, the toxin protein annotation program of the Swiss-Prot protein knowledgebase
  </strong>
  <br/>
  <a href="http://dx.doi.org/10.1016/j.toxicon.2004.10.018">
   Toxicon 45:293-301 (2005)
  </a>
 </p>
 <p>
  Schneider M, Bairoch A, Wu CH, Apweiler R.
  <br/>
  <strong>
   Plant Protein Annotation in the UniProt Knowledgebase
  </strong>
  <br/>
  <a href="http://dx.doi.org/10.1104/pp.104.058933">
   Plant Physiol. 138:59-66 (2005)
  </a>
 </p>
 <h4 id="2004">
  2004
 </h4>
 <p>
  Apweiler R, Bairoch A, Wu CH.
  <br/>
  <strong>
   Protein sequence databases
  </strong>
  <br/>
  <a href="http://dx.doi.org/10.1016/j.cbpa.2003.12.004">
   Curr. Opin. Chem. Biol. 8:76-80 (2004)
  </a>
 </p>
 <p>
  Apweiler R, Bairoch A, Wu CH, Barker WC, Boeckmann B, Ferro S, Gasteiger E, Huang H, Lopez R, Magrane M, Martin MJ, Natale DA, O'Donovan C, Redaschi N, Yeh LS.
  <br/>
  <strong>
   UniProt: the Universal Protein knowledgebase
  </strong>
  <br/>
  <a href="http://dx.doi.org/10.1093/nar/gkh131">
   Nucleic Acids Res. 32:D115-D119 (2004)
  </a>
 </p>
 <p>
  Bairoch A, Boeckmann B, Ferro S, Gasteiger E.
  <br/>
  <strong>
   Swiss-Prot: juggling between evolution and stability
  </strong>
  <br/>
  <a href="https://view.ncbi.nlm.nih.gov/pubmed/15153305">
   Brief. Bioinform. 5:39-55 (2004)
  </a>
 </p>
 <p>
  Farriol-Mathis N, Garavelli JS, Boeckmann B, Duvaud S, Gasteiger E, Gateau A, Veuthey A-L, Bairoch A.
  <br/>
  <strong>
   Annotation of post-translational modifications in the Swiss-Prot knowledgebase
  </strong>
  <br/>
  <a href="http://dx.doi.org/10.1002/pmic.200300764">
   Proteomics 4:1537-1550 (2004)
  </a>
 </p>
 <p>
  Leinonen R, Diez FG, Binns D, Fleischmann W, Lopez R, Apweiler R.
  <br/>
  <strong>
   UniProt Archive
  </strong>
  <br/>
  <a href="http://dx.doi.org/10.1093/bioinformatics/bth191">
   Bioinformatics 20:3236-3237 (2004)
  </a>
 </p>
 <p>
  Wieser D, Kretschmann E, Apweiler R.
  <br/>
  <strong>
   Filtering Erroneous Protein Annotation
  </strong>
  <br/>
  <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=PMID%3A+15262818">
   Bioinformatics 20:i342-i347 (2004)
  </a>
 </p>
 <p>
  Yip YL, Scheib H, Diemand AV, Gattiker A, Famiglietti LM, Gasteiger E, Bairoch A.
  <br/>
  <strong>
   The Swiss-Prot variant page and the ModSNP database: a resource for sequence and structure information on human protein variants
  </strong>
  <br/>
  <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=yip+scheib+diemand">
   Hum. Mutat. 23:464-470 (2004)
  </a>
 </p>
 <h4 id="2003_and_earlier">
  2003 and earlier
 </h4>
 <p>
  Boeckmann B, Bairoch A, Apweiler R, Blatter M-C, Estreicher A, Gasteiger E, Martin MJ, Michoud K, O'Donovan C, Phan I, Pilbout S, Schneider M.
  <br/>
  <strong>
   The Swiss-Prot protein knowledgebase and its supplement TrEMBL in 2003
  </strong>
  <br/>
  <a href="http://view.ncbi.nlm.nih.gov/pubmed/12520024">
   Nucleic Acids Res. 31:365-370 (2003)
  </a>
 </p>
 <p>
  Gattiker A, Michoud K, Rivoire C, Auchincloss AH, Coudert E, Lima T, Kersey P, Pagni M, Sigrist CJ, Lachaize C, Veuthey A-L, Gasteiger E, Bairoch A.
  <br/>
  <strong>
   Automated annotation of microbial proteomes in Swiss-Prot
  </strong>
  <br/>
  <a href="https://view.ncbi.nlm.nih.gov/pubmed/12798039">
   Comput. Biol. Chem. 27:49-58 (2003)
  </a>
 </p>
 <p>
  Wu CH, Huang H, Yeh L-S, Barker WC.
  <br/>
  <strong>
   Protein family classification and functional annotation
  </strong>
  <br/>
  <a href="https://www.ncbi.nlm.nih.gov/pubmed/12798038">
   Computational Biology and Chemistry 27:37-47 (2003)
  </a>
 </p>
 <p>
  O'Donovan C, Martin MJ, Gattiker A, Gasteiger E, Bairoch A, Apweiler R.
  <br/>
  <strong>
   High-quality protein knowledge resource: Swiss-Prot and TrEMBL
  </strong>
  <br/>
  <a href="http://view.ncbi.nlm.nih.gov/pubmed/12230036">
   Brief. Bioinform. 3:275-284 (2002)
  </a>
 </p>
 <p>
  Apweiler R.
  <br/>
  <strong>
   Functional information in Swiss-Prot: the basis for large-scale characterisation of protein sequences
  </strong>
  <br/>
  <a href="http://view.ncbi.nlm.nih.gov/pubmed/11465066">
   Brief. Bioinform. 2:9-18 (2001)
  </a>
 </p>
 <p>
  Gasteiger E, Jung E, Bairoch A.
  <br/>
  <strong>
   Swiss-Prot: connecting biomolecular knowledge via a protein database
  </strong>
  <br/>
  <a href="http://view.ncbi.nlm.nih.gov/pubmed/11488411">
   Curr. Issues Mol. Biol. 3:47-55 (2001)
  </a>
 </p>
 <p>
  Jung E, Veuthey A-L, Gasteiger E, Bairoch A.
  <br/>
  <strong>
   Annotation of glycoproteins in the Swiss-Prot database
  </strong>
  <br/>
  <a href="http://view.ncbi.nlm.nih.gov/pubmed/11680872">
   Proteomics 1:262-268 (2001)
  </a>
 </p>
 <p>
  O'Donovan C, Apweiler R, Bairoch A.
  <br/>
  <strong>
   The human proteomics initiative (HPI)
  </strong>
  <br/>
  <a href="http://view.ncbi.nlm.nih.gov/pubmed/11301130">
   Trends Biotechnol. 19:178-181 (2001)
  </a>
 </p>
 <p>
  Kersey P, Hermjakob H, Apweiler R.
  <br/>
  <strong>
   VARSPLIC: alternatively-spliced protein sequences derived from Swiss-Prot and TrEMBL
  </strong>
  <br/>
  <a href="http://view.ncbi.nlm.nih.gov/pubmed/11159319">
   Bioinformatics 16:1048-1049 (2000)
  </a>
 </p>
 <p>
  Fleischmann W, Moeller S, Gateau A, Apweiler R.
  <br/>
  <strong>
   A novel method for automatic functional annotation of proteins
  </strong>
  <br/>
  <a href="http://view.ncbi.nlm.nih.gov/pubmed/10222410">
   Bioinformatics 15:228-233 (1999)
  </a>
 </p>
 <p>
  Junker VL, Apweiler R, Bairoch A.
  <br/>
  <strong>
   Representation of functional information in the Swiss-Prot data bank
  </strong>
  <br/>
  <a href="http://view.ncbi.nlm.nih.gov/pubmed/10746001">
   Bioinformatics 15:1066-1067 (1999)
  </a>
 </p>
 <p>
  Moeller S, Leser U, Fleischmann W, Apweiler R.
  <br/>
  <strong>
   EDITtoTrEMBL: a distributed approach to high-quality automated protein sequence annotation
  </strong>
  <br/>
  <a href="http://view.ncbi.nlm.nih.gov/pubmed/10222409">
   Bioinformatics 15:219-227 (1999)
  </a>
 </p>
 <p>
  O'Donovan C, Martin MJ, Glemet E, Codani JJ, Apweiler R.
  <br/>
  <strong>
   Removing redundancy in Swiss-Prot and TrEMBL
  </strong>
  <br/>
  <a href="http://view.ncbi.nlm.nih.gov/pubmed/10222414">
   Bioinformatics 15:258-259 (1999)
  </a>
 </p>
 <p>
  Bairoch A, Apweiler R.
  <br/>
  <strong>
   The Swiss-Prot protein sequence database: its relevance to human molecular medical research
  </strong>
  <br/>
  <a href="http://view.ncbi.nlm.nih.gov/pubmed/9181472">
   J. Mol. Med. 75:312-316 (1997)
  </a>
 </p>
</div>