<div>
 <p>
  <a href="https://www.youtube.com/watch?v=UPaConHNP7E">
   Tutorial/Video
  </a>
 </p>
 <p>
  Select the
  <em>
   Blast
  </em>
  tab of the toolbar to run a sequence similarity search with the BLAST (Basic Local Alignment Search Tool) program:
 </p>
 <ol>
  <li>
   Enter either a protein or nucleotide sequence (raw sequence or fasta format) or a UniProt identifier into the form field.
  </li>
  <li>
   Click the
   <em>
    Blast
   </em>
   button.
  </li>
 </ol>
 <div id="blast-identifiers">
  <p>
   The following kinds of UniProt identifiers are supported:
  </p>
  <table>
   <tr>
    <td>
     <strong>
      P00750
     </strong>
    </td>
    <td>
     UniProtKB entry
    </td>
   </tr>
   <tr>
    <td>
     <strong>
      P00750-2
     </strong>
    </td>
    <td>
     UniProtKB entry isoform sequence
    </td>
   </tr>
   <tr>
    <td>
     <strong>
      P00750[1-20]
     </strong>
    </td>
    <td>
     Part of UniProtKB entry sequence, from its 1st to 20th amino acid residue (inclusive)
    </td>
   </tr>
   <tr>
    <td>
     <strong>
      A4_HUMAN
     </strong>
    </td>
    <td>
     UniProtKB entry name
    </td>
   </tr>
   <tr>
    <td>
     <strong>
      UPI0000000001
     </strong>
    </td>
    <td>
     UniParc entry
    </td>
   </tr>
   <tr>
    <td>
     <strong>
      UniRef100_P00750
     </strong>
    </td>
    <td>
     UniRef entry
    </td>
   </tr>
  </table>
 </div>
 <p>
  If you select the
  <em>
   Blast
  </em>
  tab of the toolbar from a UniProtKB, UniRef or UniParc entry page, the current sequence is prefilled in the form.
 </p>
 <p>
  Jobs have unique identifiers, which (depending on the job type) can be used in queries (e.g. to get the intersection of two sequence similarity searches). Job identifiers and the related data are kept for 7 days, and are then deleted.
 </p>
 <h4 id="options">
  Options
 </h4>
 <table>
  <tr>
   <td>
    <strong>
     Database
    </strong>
   </td>
   <td>
    Database against which the search is performed: UniProtKB or clusters of sequences with 100%, 90% or 50% identity.
   </td>
  </tr>
  <tr>
   <td>
    <strong>
     Threshold
    </strong>
   </td>
   <td>
    The expectation value (E) threshold is a statistical measure of the number of expected matches in a random database. The lower the e-value, the more likely the match is to be significant. E-values between 0.1 and 10 are generally dubious, and over 10 are unlikely to have biological significance. In all cases, those matches need to be verified manually. You may need to increase the E threshold if you have a very short query sequence, to detect very weak similarities, or similarities in a short region, or if your sequence has a low complexity region and you use the "filter" option
   </td>
  </tr>
  <tr>
   <td>
    <strong>
     Matrix
    </strong>
   </td>
   <td>
    The matrix assigns a probability score for each position in an alignment. The BLOSUM matrix assigns a probability score for each position in an alignment that is based on the frequency with which that substitution is known to occur among consensus blocks within related proteins. BLOSUM62 is among the best of the available matrices for detecting weak protein similarities. The PAM set of matrices is also available. If "Auto" is set, the matrix will be selected depending on the query sequence length.
   </td>
  </tr>
  <tr>
   <td>
    <strong>
     Filtering
    </strong>
   </td>
   <td>
    Low-complexity regions (e.g. stretches of cysteine in
    <a href="http://www.uniprot.org/uniprot/Q03751">
     Q03751
    </a>
    , or hydrophobic regions in membrane proteins) tend to produce spurious, insignificant matches with sequences in the database which have the same kind of low-complexity regions, but are unrelated biologically. If "Filter low complexity regions" is selected, the query sequence will be run through the program SEG, and all amino acids in low-complexity regions will be replaced by X's.
   </td>
  </tr>
  <tr>
   <td>
    <strong>
     Gapped
    </strong>
   </td>
   <td>
    This will allow gaps to be introduced in the sequences when the comparison is done.
   </td>
  </tr>
  <tr>
   <td>
    <strong>
     Hits
    </strong>
   </td>
   <td>
    Limits the number of returned alignments.
   </td>
  </tr>
 </table>
 <h4 id="related-services">
  Related services
 </h4>
 <table>
  <tr>
   <td>
    <strong>
     <a href="http://web.expasy.org/blast/">
      BLAST
     </a>
     (ExPASy)
    </strong>
   </td>
  </tr>
  <tr>
   <td>
    Sequence similarity search
   </td>
  </tr>
  <tr>
   <td>
    <strong>
     <a href="https://www.ebi.ac.uk/Tools/sss/ncbiblast/">
      BLAST
     </a>
     (EBI)
    </strong>
   </td>
  </tr>
  <tr>
   <td>
    Sequence similarity search
   </td>
  </tr>
  <tr>
   <td>
    <strong>
     <a href="https://www.ebi.ac.uk/Tools/sss/fasta/">
      FASTA
     </a>
     (EBI)
    </strong>
   </td>
  </tr>
  <tr>
   <td>
    Sequence similarity search
   </td>
  </tr>
  <tr>
   <td>
    <strong>
     <a href="http://pir.georgetown.edu/iproclass/index.shtml">
      iProClass
     </a>
     (PIR)
    </strong>
   </td>
  </tr>
  <tr>
   <td>
    Query a sequence against the iProClass database, which contains both sequences and protein families
   </td>
  </tr>
  <tr>
   <td>
    <strong>
     <a href="https://www.ebi.ac.uk/interpro/search/sequence/">
      InterProScan
     </a>
     (EBI)
    </strong>
   </td>
  </tr>
  <tr>
   <td>
    Query a sequence against the InterPro database, which contains protein families, domains, and motifs
   </td>
  </tr>
  <tr>
   <td>
    <strong>
     <a href="http://prosite.expasy.org/scanprosite/">
      ScanProsite
     </a>
     (ExPASy)
    </strong>
   </td>
  </tr>
  <tr>
   <td>
    Scan a protein sequence for patterns, or use a pattern to scan protein sequences
   </td>
  </tr>
  <tr>
   <td>
    <strong>
     <a href="https://www.ebi.ac.uk/Tools/hmmer/">
      HMMER
     </a>
     (EBI)
    </strong>
   </td>
  </tr>
  <tr>
   <td>
    Sequence Similarity Search
   </td>
  </tr>
 </table>
</div>