<div>
 <p>
  This subsection of the 'Sequence' section reports difference(s) between the protein sequence shown in the UniProtKB entry and other available protein sequences derived from the same gene.
 </p>
 <p>
  The sequence discrepancies described in this subsection are generally severe and thus distinct from those that are described in the
  <a href="http://www.uniprot.org/manual/conflict">
   'Sequence conflict'
  </a>
  subsection. In this subsection, we list the differences between the
  <a href="http://www.uniprot.org/help/canonical%5Fand%5Fisoforms">
   canonical sequence
  </a>
  , displayed by default in the entry, and the sequence reported in the indicated reference (be it a paper, a submitted sequence or a prediction).
 </p>
 <p>
  We annotate 6 different types of sequence discrepancies in this subsection:
 </p>
 <ul>
  <li>
   <strong>
    Frameshift
   </strong>
   : discrepancies are due to the insertion or deletion of one or more nucleotides in the underlying cDNA or genomic sequence relative to the canonical sequence.
   <br/>
   Example:
   <a href="http://www.uniprot.org/uniprot/O14467#sequences">
    O14467
   </a>
  </li>
  <li>
   <strong>
    Erroneous initiation codon
   </strong>
   : discrepancies are due to an erroneous initiation codon choice in the submitted sequence. The erroneous initiation codon may correspond to an internal codon and the sequence should be extended N-terminally. Conversely, the submitted sequence may be too long and has to be N-terminally shortened to match the canonical sequence.
   <br/>
   Example:
   <a href="http://www.uniprot.org/uniprot/Q7L2H7#sequences">
    Q7L2H7
   </a>
  </li>
  <li>
   <strong>
    Erroneous termination codon
   </strong>
   : the termination codon of the submitted sequence differs from that of the sequence displayed. A sequencing error introduced a stop codon, the sequence should be C-terminally extended by "translation" into a particular amino acid to match the canonical sequence. Conversely, the sequence may contain an amino acid instead of a bona fide termination signal. Finally, a stop codon should have been translated into a non-standard amino acid, either selenocysteine (Sec) or pyrrolysine (Pyl).
   <br/>
   Example:
   <a href="http://www.uniprot.org/uniprot/Q9Y6D0#sequences">
    Q9Y6D0
   </a>
  </li>
  <li>
   <strong>
    Erroneous gene model prediction
   </strong>
   : discrepancies are due to an erroneous gene model prediction. The predicted protein sequence (from the start to the stop codons and including all exon / intron boundaries) does not match the canonical sequence.
   <br/>
   Example:
   <a href="http://www.uniprot.org/uniprot/Q7XR80#sequences">
    Q7XR80
   </a>
  </li>
  <li>
   <strong>
    Erroneous translation
   </strong>
   : discrepancies are due to erroneous ORF assignement, or the CDS is thought to be in a different region of the mRNA, or a wrong genetic code has been used, etc.
   <br/>
   Example:
   <a href="http://www.uniprot.org/uniprot/Q96GX2#sequences">
    Q96GX2
   </a>
  </li>
  <li>
   <strong>
    Miscellaneous discrepancy
   </strong>
   : this category includes intron retention, chimeric DNA, unusual initiator codon (in eukaryotes), etc.
   <br/>
   Examples:
   <a href="http://www.uniprot.org/uniprot/P49762#sequences">
    P49762
   </a>
   ,
   <a href="http://www.uniprot.org/uniprot/Q8TBF5#sequences">
    Q8TBF5
   </a>
  </li>
 </ul>
 <p>
  Note that the cross-references linking to the problematic nucleotide sequences are tagged in the
  <a href="http://www.uniprot.org/manual/cross%5Freferences%5Fsection">
   'Cross-references'
  </a>
  section (Sequence databases).
 </p>
</div>