<div>
 <h3 id="Visualization">
  Visualization
 </h3>
 <ul>
  <li>
   ProtVista
   <br/>
   A Web Component which uses Nightingale components to display protein sequence information.
   <br/>
   URL:
   <a href="https://github.com/ebi-webcomponents/protvista-uniprot">
    https://github.com/ebi-webcomponents/protvista-uniprot
   </a>
   <br/>
   License: MIT License
  </li>
 </ul>
 <ul>
  <li>
   Nightingale
   <br/>
   Visualization Web Components for biological data.
   <br/>
   URL:
   <a href="https://github.com/ebi-webcomponents/nightingale">
    https://github.com/ebi-webcomponents/nightingale
   </a>
   <br/>
   License: MIT License
  </li>
 </ul>
 <ul>
  <li>
   Franklin pattern library
   <br/>
   React and Zurb Foundation based design system for life sciences web applications.
   <br/>
   URL:
   <a href="https://github.com/ebi-uniprot/franklin-sites">
    https://github.com/ebi-uniprot/franklin-sites
   </a>
   <br/>
   License: MIT License
  </li>
 </ul>
 <h3 id="Command_line_tools">
  Command-line tools
 </h3>
 <ul>
  <li>
   Swissknife
   <br/>
   Object-oriented Perl library for parsing the UniProtKB text format.
   <br/>
   URL:
   <a href="http://swissknife.sourceforge.net/">
    http://swissknife.sourceforge.net/
   </a>
   <br/>
   License:
   <a href="https://www.gnu.org/licenses/old-licenses/gpl-2.0.en.html">
    GPLv2
   </a>
   or any later version
  </li>
 </ul>
 <ul>
  <li>
   Varsplic
   <br/>
   A Perl script to generate additional records from UniProtKB/Swiss-Prot, one for each alternative isoform curated by UniProtKB/Swiss-Prot biocurators
   <br/>
   URL:
   <a href="ftp://ftp.ebi.ac.uk/pub/software/uniprot/varsplic/">
    ftp://ftp.ebi.ac.uk/pub/software/uniprot/varsplic/
   </a>
   <br/>
   License:
   <a href="https://www.gnu.org/licenses/old-licenses/gpl-2.0.en.html">
    GPLv2
   </a>
   or any later version
  </li>
 </ul>
 <ul>
  <li>
   unirefxml2fasta.pl
   <br/>
   A Perl script to extract UniRef sequences from UniRef XML files.
   <br/>
   URL:
   <a href="https://proteininformationresource.org/download/uniref/xml2fasta/">
    https://proteininformationresource.org/download/uniref/xml2fasta/
   </a>
   <br/>
   License:
   <a href="https://www.gnu.org/licenses/gpl-3.0.html">
    GPLv3
   </a>
  </li>
 </ul>
 <ul>
  <li>
   Peptide Match
   <br/>
   A command line tool that allows to quickly retrieve all occurrences of a given query peptide from a customized protein sequence database (see
   <a href="https://pubmed.ncbi.nlm.nih.gov/23958731/">
    PMID:23958731
   </a>
   for more details).
   <br/>
   URL:
   <a href="https://research.bioinformatics.udel.edu/peptidematch/commandlinetool.jsp">
    https://research.bioinformatics.udel.edu/peptidematch/commandlinetool.jsp
   </a>
   <br/>
   License:
   <a href="https://www.gnu.org/licenses/gpl-3.0.html">
    GPLv3
   </a>
  </li>
 </ul>
 <h3 id="Automatic_annotation">
  Automatic annotation
 </h3>
 <p>
  <a href="http://www.uniprot.org/help/automatic%5Fannotation">
   UniProt's automatic annotation pipeline
  </a>
  enhances the unreviewed records in UniProtKB by enriching them with automatic classification and annotation. A stand-alone version of this system, and parts thereof, are available to annotate protein sequences.
 </p>
 <ul>
  <li>
   UniFIRE
   <br/>
   UniFIRE (The UniProt Functional annotation Inference Rule Engine) is an engine to execute rules in the UniProt Rule Markup Language (URML) format. It can be used to execute the UniProt annotation rules (
   <a href="http://www.uniprot.org/help/unirule">
    UniRule
   </a>
   and
   <a href="http://www.uniprot.org/help/arba">
    ARBA
   </a>
   ).
   <br/>
   URL:
   <a href="https://gitlab.ebi.ac.uk/uniprot-public/unifire">
    https://gitlab.ebi.ac.uk/uniprot-public/unifire
   </a>
   <br/>
   License:
   <a href="http://www.apache.org/licenses/LICENSE-2.0">
    Apache License 2.0
   </a>
  </li>
 </ul>
 <ul>
  <li>
   PIRsitePredict
   <br/>
   PIRsitePredict predicts functional residues within uncharacterized proteins using position-specific site conditional template annotation rules (PIR Site Rules) derived from known protein structural and experimental data curated by structural biologists. PIR Site Rules are part of
   <a href="http://www.uniprot.org/help/unirule">
    UniRule
   </a>
   .
   <br/>
   URL:
   <a href="https://research.bioinformatics.udel.edu/PIRSitePredict/download">
    https://research.bioinformatics.udel.edu/PIRSitePredict/download
   </a>
   <br/>
   License:
   <a href="https://creativecommons.org/licenses/by-nc-sa/4.0/">
    CC BY-NC-SA 4.0
   </a>
  </li>
 </ul>
 <ul>
  <li>
   HAMAP as SPARQL rules
   <br/>
   <a href="https://hamap.expasy.org/">
    HAMAP
   </a>
   is a system for the classification and annotation of protein sequences. It consists of a collection of expert-curated protein family signatures and rules that specify annotations that apply to family members. HAMAP rules are part of
   <a href="http://www.uniprot.org/help/unirule">
    UniRule
   </a>
   and are also published in the W3C standard SPARQL 1.1 syntax to allow to execute the rules with freely available SPARQL engines (see
   <a href="https://pubmed.ncbi.nlm.nih.gov/32034905/">
    PMID:32034905
   </a>
   for more details).
   <br/>
   URL:
   <a href="https://github.com/sib-swiss/HAMAP-SPARQL">
    https://github.com/sib-swiss/HAMAP-SPARQL
   </a>
   <br/>
   License:
   <a href="https://creativecommons.org/publicdomain/zero/1.0/">
    CC0 1.0
   </a>
   (supplementary code),
   <a href="https://creativecommons.org/licenses/by-nd/4.0/">
    CC-BY-ND 4.0
   </a>
   (rules)
  </li>
 </ul>
</div>