<div>
 <p>
  The two sections of UniProtKB - UniProtKB/Swiss-Prot and UniProtKB/TrEMBL - give access to
  <a href="http://www.uniprot.org/help/sequence%5Forigin">
   most protein sequences which are available to the public
  </a>
  . However, UniProtKB excludes the following protein sequences:
 </p>
 <ol>
  <li>
   Most non-germline
   <a href="http://www.uniprot.org/help/immunoglobulins">
    immunoglobulins
   </a>
   and T-cell receptors
  </li>
  <li>
   Synthetic sequences
  </li>
  <li>
   Most patent application sequences
  </li>
  <li>
   Small fragments encoded from nucleotide sequence (&lt;8 amino acids)
  </li>
  <li>
   Pseudogenes
  </li>
  <li>
   Sequences from
   <a href="http://www.uniprot.org/help/proteome%5Fredundancy">
    redundant proteomes
   </a>
  </li>
  <li>
   Sequences from proteomes that NCBI genomes/RefSeq considers to be
   <a href="https://www.ncbi.nlm.nih.gov/assembly/help/anomnotrefseq/">
    low quality assemblies
   </a>
   , i.e.
   <a href="http://www.uniprot.org/help/proteome%5Fexclusion%5Freasons">
    excluded proteomes
   </a>
  </li>
  <li>
   Fusion/truncated proteins
  </li>
  <li>
   Not real proteins
  </li>
 </ol>
 <p>
  The first 5 are identified automatically by the UniProtKB/TrEMBL creation program and never enter UniProtKB. However some proteins belonging to these classes are also identified during the UniProtKB/Swiss-Prot annotation process by the curators and then removed from UniProtKB.
 </p>
 <p>
  Protein sequences originating from proteomes that are considered
  <a href="http://www.uniprot.org/help/proteome%5Fredundancy">
   redundant
  </a>
  or
  <a href="https://www.ncbi.nlm.nih.gov/assembly/help/anomnotrefseq/">
   low quality
  </a>
  are identified automatically at every release, and are either removed from UniProtKB/TrEMBL, or never enter UniProtKB but stay in
  <a href="http://www.uniprot.org/help/uniparc">
   UniParc
  </a>
  .
 </p>
 <p>
  Fusion/truncated proteins and those classified as not real proteins are only manually identified by the curators and removed from UniProtKB/TrEMBL or UniProtKB/Swiss-Prot. All these excluded sequences are available at
  <a href="http://www.uniprot.org/help/uniparc">
   UniParc
  </a>
  . The corresponding UniParc entries have been flagged with the reason for the absence of that sequence from UniProtKB.
 </p>
 <p>
  See also:
 </p>
 <ul>
  <li>
   <a href="http://www.uniprot.org/help/sequence%5Forigin">
    Where do the UniProtKB protein sequences come from?
   </a>
  </li>
  <li>
   <a href="http://www.uniprot.org/help/cds%5Fprotein%5Fdefinition">
    What are UniProtKB's criteria for defining a CDS as a protein?
   </a>
  </li>
  <li>
   <a href="http://www.uniprot.org/help/deleted%5Faccessions">
    Why have some UniProtKB accession numbers been deleted? How can I track them?
   </a>
  </li>
  <li>
   <a href="http://www.uniprot.org/help/dubious%5Fsequences">
    Why do we keep dubious sequences in UniProtKB? How to discard them from a protein set?
   </a>
  </li>
  <li>
   <a href="http://www.uniprot.org/manual/sequences">
    Sequences
   </a>
  </li>
 </ul>
</div>