---
title: Alternative products
categories: Sequence,manual
---

This subsection of the 'Sequence' section lists the alternative protein sequences (isoforms) that can be generated from the same gene by a single or by the combination of up to four biological events (alternative promoter usage, alternative splicing, alternative initiation and ribosomal frameshifting). Additionally, this section gives relevant information on each alternative protein isoform.

In addition to expertly curated isoforms, present in reviewed entries only, this section may include [computationally mapped potential isoform sequences](http://www.uniprot.org/help/gene%5Fcentric%5Fisoform%5Fmapping) if the protein is a member of a eukaryotic reference proteome (reviewed or unreviewed).

The following text refers to expert-curated isoforms in reviewed entries, i.e.Â in UniProtKB/Swiss-Prot.

### 1. General information {\#1\_\_General\_information}

We give the number of alternative protein sequences described in the entry and the biological event(s) which lead to the production of these isoforms (alternative promoter usage, alternative splicing, alternative initiation and/or ribosomal frameshifting). Additional information is provided, if available.  
Example: [P04150](http://www.uniprot.org/uniprotkb/P04150#sequences)

### 2. Isoform-specific information {\#2\_\_Isoform\_specific\_information}

For each isoform, we provide the following information:

#### Identifier

Each isoform is characterized by a unique identifier, which is composed of the primary accession number of the entry, followed by a dash and a number.  
Example: [P04150-2](http://www.uniprot.org/uniprotkb/P04150#sequences)

When alternative protein sequences differ significantly, we create separate entries and list all isoforms in each of them. Consequently, isoforms produced from a single gene listed in one entry may have identifiers derived from different primary accession numbers.  
Examples: [P42166](http://www.uniprot.org/uniprotkb/P42166#sequences) , [P42167](http://www.uniprot.org/uniprotkb/P42167#sequences)

#### Name and synomym(s)

We provide the name and synomym(s) of the isoform, which are used in the literature or assigned by Swiss-Prot biocurators. In the absence of a commonly used name, consecutive numbers are used in the "Name" field.

#### Description of the sequence differences

We clearly describe the alternative sequences with respect to the [canonical sequence](http://www.uniprot.org/help/canonical%5Fand%5Fisoforms) displayed by default in the entry.  
*Note* : The so-called 'canonical' sequence is selected on the basis of several criteria including its function, expression, and/or conservation across orthologous species. Unless otherwise stated, [positional information](http://www.uniprot.org/help/sequence%5Fannotation) in the entry refers to it.

*Note* : If an isoform has been reported to exist in the literature, but we are not able to determine its sequence, we add the tag 'Sequence is not available'.  
Example: [P21836](http://www.uniprot.org/uniprotkb/P21836#sequences) (Isoform H)

#### Description of characteristics that differ from the canonical form

When isoforms (or products of proteolytic cleavage) are known to differ significantly in their function, localization or any other characteristics with respect to the canonical sequence, these are described in the corresponding sections of the entry. In case of positional annotations such as PTMs or variants, positions are provided with respect to the isoform sequence they refer to.

Examples: [P19021](http://www.uniprot.org/uniprotkb/P19021#ptm%5Fprocessing) , [Q9NZU7](http://www.uniprot.org/uniprotkb/Q9NZU7#ptm%5Fprocessing) , [Q13510](http://www.uniprot.org/uniprotkb/Q13510##pathology%5Fand%5Fbiotech)

#### Information in free text can be provided to complement the above items

Example: [P04150](http://www.uniprot.org/uniprotkb/P04150#sequences)

### Notes

For the description of the isoforms at the amino acid level, see the ['Alternative sequence'](http://www.uniprot.org/manual/var%5Fseq) subsection.

Isoform sequences can be **downloaded in FASTA format** from our [download page](http://www.uniprot.org/downloads) (choose the file: 'Isoform sequences').

Related keywords: [Alternative splicing](http://www.uniprot.org/keywords/25) , [Alternative promoter usage](http://www.uniprot.org/keywords/877) , [Alternative initiation](http://www.uniprot.org/keywords/24) , [Ribosomal frameshifting](http://www.uniprot.org/keywords/688) .

#### Related documents

[What is the canonical sequence? Are all isoforms described in one entry? How can I retrieve them?](http://www.uniprot.org/help/canonical%5Fand%5Fisoforms)

[How are protein sequence variety and protein diversity represented in UniProtKB?](http://www.uniprot.org/help/protein%5Fdiversity)

[How do I get the nucleotide sequence that corresponds to the canonical UniProtKB sequence?](http://www.uniprot.org/help/canonical%5Fnucleotide)

[Cross-references for isoform sequences](http://www.uniprot.org/help/isoform%5Fcrossreferences)

[How to retrieve a complete set of protein sequences?](http://www.uniprot.org/help/retrieve%5Fsets)

[Automatic gene-centric isoform mapping for eukaryotic reference proteome entries](http://www.uniprot.org/help/gene%5Fcentric%5Fisoform%5Fmapping)

[What are UniProtKB's criteria for defining a CDS as a protein?](http://www.uniprot.org/help/cds%5Fprotein%5Fdefinition)

Related terms: splice variant, splice isoform
