<div>
 <pre>

                              TrEMBL Release Notes
                              Release 8, November 1998

    EMBL Outstation
    European Bioinformatics Institute (EBI)
    Wellcome Trust Genome Campus
    Hinxton
    Cambridge CB10 1SD
    United Kingdom

    Telephone: (+44 1223) 494 444
    Fax: (+44 1223) 494 468
    Electronic mail address: DATALIB@EBI.AC.UK
    WWW server: http://www.ebi.ac.uk/

    Amos Bairoch
    Swiss Institute of Bioinformatics (SIB)
    Centre Medical Universitaire
    1, rue Michel Servet
    1211 Geneva 4
    Switzerland

    Telephone: (+41 22) 784 40 82
    Fax: (+41 22) 702 55 02
    Electronic mail address: BAIROCH@CMU.UNIGE.CH
    WWW server: http://www.expasy.ch/


    Acknowledgements

    TrEMBL has been prepared by:

    o  Rolf Apweiler, Sergio Contrino, Wolfgang Fleischmann, Gill Fraser,
       Henning Hermjakob, Vivien Junker, Stephanie Kappus, Youla
       Karavidopoulou, Fiona Lang, Michele Magrane, Maria Jesus Martin,
       Steffen Moeller, Nicoletta Mitaritonna and Claire O'Donovan at
       the EMBL Outstation - European Bioinformatics Institute (EBI) in
       Hinxton, UK;
    o  Amos Bairoch and Alain Gateau at the Swiss Institute of Bioinformatics
       in Geneva, Switzerland.

    Notes

    This manual and the database it accompanies may be copied and
    redistributed freely, without advance permission, provided
    that this statement is reproduced with each copy.

    Citation

    If you  want to  cite  TrEMBL  in  a  publication  please  use  the
    following reference:

              Bairoch A., and Apweiler R.
              The SWISS-PROT protein sequence data bank and its
              supplement TrEMBL in 1998.
              Nucleic Acids Res. 26:38-42(1998).

                         1. Introduction


TrEMBL is a computer-annotated protein sequence database supplementing the
SWISS-PROT Protein Sequence Data Bank. TrEMBL contains the translations of
all coding sequences (CDS) present in the EMBL Nucleotide Sequence Database
not yet integrated in SWISS-PROT. TrEMBL can be considered as a preliminary
section of SWISS-PROT. For all TrEMBL entries which should finally be
upgraded to the standard SWISS-PROT quality, SWISS-PROT accession numbers
have been assigned.

                        2. Why a supplement to SWISS-PROT?

The ongoing gene sequencing and mapping projects have dramatically
increased the number of protein sequences to be incorporated into SWISS-PROT.
We do not want to dilute the quality standards of SWISS-PROT by incorporating
sequences without proper sequence analysis and annotation, but we do want to
make the sequences available as fast as possible. TrEMBL achieves this second
goal, and is a major step in the process of speeding up subsequent
upgrading of annotation to the standard SWISS-PROT quality.
To address the problem of redundancy, the translations of all coding
sequences (CDS) in the EMBL Nucleotide Sequence Database already included
in SWISS-PROT have been removed from TrEMBL.

We name this supplement TrEMBL (Translation from EMBL), since the tools
used to create the translations of the CDS are based on the program
'trembl' written by Thure Etzold at the EMBL.

                             3. The Release

This TrEMBL release is created from the EMBL Nucleotide Sequence Database
release 56 and contains 224'543 sequence entries, comprising 60'188'661 amino
acids. To minimize redundancy, the translations of all coding sequences (CDS)
in the EMBL Nucleotide Sequence Database already included in SWISS-PROT 36
have been removed from TrEMBL release 8.

TrEMBL is split in two main sections; SP-TrEMBL and REM-TrEMBL:

SP-TrEMBL (SWISS-PROT TrEMBL) contains the entries (180'763) which should be
eventually incorporated into SWISS-PROT. SWISS-PROT accession numbers have
been assigned for all SP-TrEMBL entries.

SP-TrEMBL is organized in subsections:

arc.dat (Archea):               7397 entries
fun.dat (Fungi):                6007 entries
hum.dat (Human):                7688 entries
inv.dat (Invertebrates):       22829 entries
mam.dat (Other Mammals):        2892 entries
mhc.dat (MHC proteins):         3985 entries
org.dat (Organelles):          14230 entries
phg.dat (Bacteriophages):       1824 entries
pln.dat (Plants):              14749 entries
pro.dat (Prokaryotes):         39777 entries
rod.dat (Rodents):              6923 entries
unc.dat (Unclassified):           44 entries
vrl.dat (Viruses):             48472 entries
vrt.dat (Other Vertebrates):    3946 entries

20'082 new entries have been integrated in SP-TrEMBL. 922 sequences of
SP-TrEMBL entries have been updated and in 124'261 cases the annotation
has been updated.
In the document deleteac.txt you will find a list of all accession numbers
which appeared earlier in the TrEMBL data bank, but have been deleted from the
database.

REM-TrEMBL (REMaining TrEMBL) contains the entries (43'780) that we do
not want to include in SWISS-PROT. REM-TrEMBL entries have no accession
numbers. This section is organized in five subsections:

   1) Immunoglobulins and T-cell receptors (Immuno.dat)
      Most REM-TrEMBL entries are  immunoglobulins and  T-cell receptors. We
      stopped entering immunoglobulins and T-cell receptors into SWISS-PROT,
      because we only want to keep  the  germ line gene derived translations
      of these proteins in SWISS-PROT and not all known somatic recombinated
      variations of  these proteins.  We would like to  create a specialized
      database  dealing  with  these  sequences  as a further  supplement to
      SWISS-PROT  and  keep  only a  representative  cross-section of  these
      proteins in SWISS-PROT.

   2) Synthetic sequences (Synth.dat)
      Another category of data, which will not be included in SWISS-PROT are
      synthetic sequences.  Again, we do not want to  leave these entries in
      TrEMBL. Ideally one should build a specialized database for artificial
      sequences as a further supplement to SWISS-PROT.

   3) Patent application sequences (Patent.dat)
      A third  subsection consists of  coding sequences captured from patent
      applications.  A thorough  survey of  these  entries  have shown  that
      apart from a rather small minority  (which in  most cases have already
      been integrated in SWISS-PROT), most of these sequences contain either
      erroneous data or concern artificially generated sequences outside the
      scope of SWISS-PROT.

   4) Small fragments (Smalls.dat)
      Another  subsection  consists of fragments  with less than eight amino
      acids.

   5) CDS not coding for real proteins (Pseudo.dat)
      The last subsection consists of  CDS translations where we have strong
      evidence to believe that these CDS are not coding for real proteins.


                4. Format Differences Between SWISS-PROT and TrEMBL

The format and conventions used by TrEMBL follow as closely as possible
that of SWISS-PROT. Hence, it is not necessary to produce an additional
user manual and extensive release notes for TrEMBL. The information given
in the SWISS-PROT release notes and user manual are in general valid for
TrEMBL. The differences are mentioned below.

The general structure of an entry is identical in SWISS-PROT and TrEMBL.
The data class used in TrEMBL (in the ID line) is always 'PRELIMINARY',
whereas in SWISS-PROT it is always 'STANDARD'.

Differences in line types present in SWISS-PROT and TrEMBL:

The ID line (IDentification):

The entry name used in SP-TrEMBL is the same as the Accession Number of the
entry. The entry name used in REM-TrEMBL is the PID tagged to the
corresponding CDS in the EMBL Nucleotide Sequence Database. 'PID' stands for
the "Protein IDentification" number. It is a number that you will find in
EMBL nucleotide sequence entries in a qualifier called "/db_xref" which is
tagged to every CDS in the nucleotide database. Example:

   FT   CDS            54..1382
   FT                  /note="ribulose-1,5-bisphosphate carboxylase/
   FT                  oxygenase activase precursor"
   FT                  /db_xref="PID:g1006835"

The DT line (DaTe)

The format of the DT lines that serve to indicate when an entry was
created and updated are identical to that defined in SWISS-PROT; but the
DT lines in TrEMBL are referring to the TEMBL release. The difference is
shown in the example below.

    DT lines in a SWISS-PROT entry:

    DT   01-JAN-1988 (REL. 06, CREATED)
    DT   01-JUL-1989 (REL. 11, LAST SEQUENCE UPDATE)
    DT   01-AUG-1992 (REL. 23, LAST ANNOTATION UPDATE)

    DT lines in a TrEMBL entry:

    DT   01-NOV-1996 (TREMBLREL. 01, CREATED)
    DT   01-NOV-1996 (TREMBLREL. 01, LAST SEQUENCE UPDATE)
    DT   01-FEB-1997 (TREMBLREL. 02, LAST ANNOTATION UPDATE)

                5. Weekly updates of TrEMBL and non-redundant data sets

Weekly cumulative updates of TrEMBL are available by anonymous FTP and
from the EBI SRS server.
We also produce every week a complete non-redundant protein sequence
collection by providing three compressed files (these are in the directory
/pub/databases/sp_tr_nrdb on the EBI FTP server and in databases/sp_tr_nrdb
on the ExPASy server): sprot.dat.Z, trembl.dat.Z and trembl_new.dat.Z.
This set of non-redundant files is especially important for two types of
users:
(i) Managers of similarity search services. They can now provide what is
currently the most comprehensive and non-redundant data set of protein
sequences.
(ii) Anybody wanting to update their full copy of SWISS-PROT + TrEMBL to
their own schedule without having to wait for full releases of SWISS-PROT
or of TrEMBL.

                6. Access/Data Distribution

FTP server:     ftp.ebi.ac.uk/pub/databases/trembl
SRS server:     http://srs.ebi.ac.uk/

TrEMBL is also available on the SWISS-PROT CD-ROM.
SWISS-PROT + TrEMBL is searchable on the following servers at the EBI:
FASTA3  (http://www2.ebi.ac.uk/fasta3/)
BLAST2  (http://www2.ebi.ac.uk/blast2/)
Bic_sw  (http://www2.ebi.ac.uk/bic_sw/)
Scanps  (http://www2.ebi.ac.uk/scanps/)
SSearch (http://www2.ebi.ac.uk/ssearch3/)


  7. Description of changes made to TrEMBL since release 7.

7.1  Switch to the NCBI taxonomy

To standardize the taxonomies used by different databases we have switched
to the NCBI taxonomy, which is already used by the DDBJ/EMBL/GenBank
nucleotide sequence databases.

7.2  Introduction of RT lines

We have introduced a new line type, the RT (Reference Title) line.
This optional line is placed between the RA and RL line. The RT line gives the
title of the paper (or other work) as accurately as possible given the limitations
of the computer character set. The form used is that which is used in a
citation rather than that displayed at the top of the published paper. For
instance, where journals capitalize major title words this is not preserved.
The title is enclosed in double quotes, and may continue over several
lines if necessary. Note these lines are in mixed case. The title lines are
terminated by a semicolon. An example of the use of RT lines is shown below:

 RT   "Sequence analysis of the genome of the unicellular cyanobacterium
 RT   Synechocystis sp. strain PCC6803. I. Sequence features in the 1 Mb
 RT   region from map positions 64% to 92% of the genome.";


                8. Planned changes

8.1  Extension of the accession number system

As explained in detail in the SWISS-PROT release notes under 2.3, we will
extend the accession number system when the 'O' series is used up. This can
be anticipated for January 1999.
  </pre>
</div>