<div>
 <pre>

                              TrEMBL Release Notes
                              Release 15, October 2000

    EMBL Outstation
    European Bioinformatics Institute (EBI)
    Wellcome Trust Genome Campus
    Hinxton
    Cambridge CB10 1SD
    United Kingdom

    Telephone: (+44 1223) 494 444
    Fax: (+44 1223) 494 468
    Electronic mail address: DATALIB@EBI.AC.UK
    WWW server: http://www.ebi.ac.uk/

    Swiss Institute of Bioinformatics (SIB)
    Centre Medical Universitaire
    1, rue Michel Servet
    1211 Geneva 4
    Switzerland

    Telephone: (+41 22) 702 50 50
    Fax: (+41 22) 702 58 58
    Electronic mail address: BAIROCH@CMU.UNIGE.CH
    WWW server: http://www.expasy.ch/


    Acknowledgements

    TrEMBL has been prepared by:

    o  Rolf Apweiler, Kirsty Bates, Margaret Biswas, Sergio Contrino,
       Kirill Degtyarenko, Wolfgang Fleischmann, Gill Fraser, Cathy Gedman,
       Henning Hermjakob, Vivien Junker, Alexander Kanapin, Youla
       Karavidopoulou, Paul Kersey, Fiona Lang, Minna Lehvaslaiho,
       Michele Magrane, Maria Jesus Martin, Nicoletta Mitaritonna, Virginie
       Mittard, Steffen Moeller, Nicola Mulder, Claire O'Donovan, John F.
       O'Rourke, Isabelle Phan, Sandrine Pilbout, Lucia Rodriguez-Monge,
       Eleanor Whitfield and Allyson Williams
       at the EMBL Outstation - European Bioinformatics Institute (EBI)
       in Hinxton, UK;
    o  Amos Bairoch and Alain Gateau at the Swiss Institute of Bioinformatics
       in Geneva, Switzerland.

    Copyright Notice
    TrEMBL copyright (c) 2000 EMBL-EBI
    This manual and the database it accompanies may be copied and
    redistributed freely, without advance permission, provided
    that this copyright statement is reproduced with each copy.

    Citation

    If you  want to  cite  TrEMBL  in  a  publication  please  use  the
    following reference:


              Bairoch A, and Apweiler R.
              The SWISS-PROT protein sequence data bank and its supplement
              TrEMBL in 2000.
              Nucl. Acids Res. 28:45-48(2000).


                         1. Introduction


TrEMBL is a computer-annotated protein sequence database supplementing the
SWISS-PROT Protein Sequence Data Bank. TrEMBL contains the translations of
all coding sequences (CDS) present in the EMBL Nucleotide Sequence Database
not yet integrated in SWISS-PROT. TrEMBL can be considered as a preliminary
section of SWISS-PROT. For all TrEMBL entries which should finally be
upgraded to the standard SWISS-PROT quality, SWISS-PROT accession numbers
have been assigned.

                        2. Why a supplement to SWISS-PROT?

The ongoing gene sequencing and mapping projects have dramatically
increased the number of protein sequences to be incorporated into SWISS-PROT.
We do not want to dilute the quality standards of SWISS-PROT by incorporating
sequences without proper sequence analysis and annotation, but we do want to
make the sequences available as fast as possible. TrEMBL achieves this second
goal, and is a major step in the process of speeding up subsequent
upgrading of annotation to the standard SWISS-PROT quality.
To address the problem of redundancy, the translations of all coding
sequences (CDS) in the EMBL Nucleotide Sequence Database already included
in SWISS-PROT have been removed from TrEMBL.


                             3. The Release

This TrEMBL release was created from the EMBL Nucleotide Sequence Database
release 64 and contains 431'424 sequence entries, comprising 124'294'926 amino
acids. To minimize redundancy, the translations of all coding sequences (CDS)
in the EMBL Nucleotide Sequence Database already included in SWISS-PROT release
39 and updates up to 01.9.2000 have been removed from TrEMBl release 15.

TrEMBL is split in two main sections: SP-TrEMBL and REM-TrEMBL:
SP-TrEMBL (SWISS-PROT TrEMBL) contains the entries (374'700) which should be
eventually incorporated into SWISS-PROT. SWISS-PROT accession numbers have
been assigned for all SP-TrEMBL entries.

SP-TrEMBL is organized in subsections:

arc.dat (Archaea):             11801 entries
fun.dat (Fungi):               10994 entries
hum.dat (Human):               18327 entries
inv.dat (Invertebrates):       51398 entries
mam.dat (Other Mammals):        6332 entries
mhc.dat (MHC proteins):         6111 entries
org.dat (Organelles):          32082 entries
phg.dat (Bacteriophages):       3016 entries
pln.dat (Plants):              40283 entries
pro.dat (Prokaryotes):         88169 entries
rod.dat (Rodents):             11849 entries
unc.dat (Unclassified):           51 entries
vrl.dat (Viruses):             86365 entries
vrt.dat (Other Vertebrates):    7922 entries

79'958 new entries have been integrated in SP-TrEMBL. The sequences of
621 SP-TrEMBL entries have been updated and the annotation has been
updated in 70'559 entries.

In the document deleteac.txt, you will find a list of all accession numbers
which were previously present in TrEMBL, but which have now been deleted from
the database.

REM-TrEMBL (REMaining TrEMBL) contains the entries (56'724) that we do
not want to include in SWISS-PROT. REM-TrEMBL entries have no accession
numbers. This section is organized in six subsections:

   1) Immunoglobulins and T-cell receptors (Immuno.dat)
      Most REM-TrEMBL entries are  immunoglobulins and  T-cell receptors. We
      stopped entering immunoglobulins and T-cell receptors into SWISS-PROT,
      because we only want to keep  the  germ line gene derived translations
      of these proteins in SWISS-PROT and not all known somatic recombinated
      variations of  these proteins.  We would like to  create a specialized
      database  dealing  with  these  sequences  as a further  supplement to
      SWISS-PROT  and  keep  only a  representative  cross-section of  these
      proteins in SWISS-PROT.

   2) Synthetic sequences (Synth.dat)
      Another category of data, which will not be included in SWISS-PROT are
      synthetic sequences.  Again, we do not want to  leave these entries in
      TrEMBL. Ideally one should build a specialized database for artificial
      sequences as a further supplement to SWISS-PROT.

   3) Patent application sequences (Patent.dat)
      A third  subsection consists of  coding sequences captured from patent
      applications.  A thorough  survey of  these  entries  have shown  that
      apart from a rather small minority  (which in  most cases have already
      been integrated in SWISS-PROT), most of these sequences contain either
      erroneous data or concern artificially generated sequences outside the
      scope of SWISS-PROT.

   4) Small fragments (Smalls.dat)
      Another  subsection  consists of fragments  with less than eight amino
      acids.

   5) CDS not coding for real proteins (Pseudo.dat)
      This subsection consists of CDS translations where we have strong
      evidence to believe that these CDS are not coding for real proteins.

   6) Truncated proteins (Truncated.dat)
      The last subsection consists of truncated proteins which result from
      events like mutations introducing a stop codon leading to the truncation
      of the protein product.

                4. Format Differences Between SWISS-PROT and TrEMBL

The format and conventions used by TrEMBL follow as closely as possible
that of SWISS-PROT. Hence, it is not necessary to produce an additional
user manual and extensive release notes for TrEMBL. The information given
in the SWISS-PROT release notes and user manual are in general valid for
TrEMBL. The differences are mentioned below.

The general structure of an entry is identical in SWISS-PROT and TrEMBL.
The data class used in TrEMBL (in the ID line) is always 'PRELIMINARY',
whereas in SWISS-PROT it is always 'STANDARD'.

Differences in line types present in SWISS-PROT and TrEMBL:

The ID line (IDentification):

The entry name used in SP-TrEMBL is the same as the Accession Number of the
entry. The entry name used in REM-TrEMBL is the stable part of the protein_id
tagged to the corresponding CDS in the EMBL Nucleotide Sequence Database.
'protein_id' stands for the "Protein Identification" number. It is a number
that you will find in the feature table of the EMBL nucleotide sequence
entries in a qualifier called "/protein_id" which is tagged to every CDS.

Example:

FT   CDS             339..1514
FT                   /codon_start=1
FT                   /db_xref="PID:g1256015"
FT                   /product="dystrobrevin-epsilon"
FT                   /protein_id="AAC50431.1"

The protein_id is defined as follows in the The DDBJ/EMBL/GenBank Feature Table
Definition documentation
Qualifier          /protein_id
Definition         Protein Identifier, issued by International collaborators.
                   This qualifier consists of a stable ID portion (3+5 format
                   with 3 position letters and 5 numbers) plus a version
                   number after the decimal point.

Value format       &lt;identifier&gt;
Example            /protein_id="AAA12345.1"
Comment            When the protein sequence encoded by the CDS changes, only
                   the version number of the /protein_id value is incremented.
                   The stable part of the /protein_id remains unchanged and
                   as a result will permanently be associated with a given
                   protein. This qualifier is valid only on CDS features
                   which translate into a valid protein.


The DT line (DaTe)

The format of the DT lines that serve to indicate when an entry was
created and updated are identical to that defined in SWISS-PROT; but the
DT lines in TrEMBL refer to the TrEMBL release. The difference is
shown in the example below.

    DT lines in a SWISS-PROT entry:

    DT   01-JAN-1988 (Rel. 06, Created)
    DT   01-JUL-1989 (Rel. 11, Last sequence update)
    DT   01-AUG-1992 (Rel. 23, Last annotation update)

    DT lines in a TrEMBL entry:

    DT   01-NOV-1996 (TrEMBLrel. 01, Created)
    DT   01-NOV-1996 (TrEMBLrel. 01, Last sequence update)
    DT   01-FEB-1997 (TrEMBLrel. 02, Last annotation update)

                5. Weekly updates of TrEMBL and non-redundant data sets

Weekly cumulative updates of TrEMBL are available by anonymous FTP and
from the EBI SRS server.

We also produce every week a complete non-redundant protein sequence
collection by providing three compressed files (these are in the directory
/pub/databases/sp_tr_nrdb on the EBI FTP server and in databases/sp_tr_nrdb
on the ExPASy server): sprot.dat.Z, trembl.dat.Z and trembl_new.dat.Z.
This set of non-redundant files is especially important for two types of
users:
(i) Managers of similarity search services. They can now provide what is
currently the most comprehensive and non-redundant data set of protein
sequences.
(ii) Anybody wanting to update their full copy of SWISS-PROT + TrEMBL to
their own schedule without having to wait for full releases of SWISS-PROT
or of TrEMBL.

We also recently introduced Varsplic Expand which is a program to generate
"expanded" sequences from SWISS-PROT records i.e. sequences including the
variants specified by the varsplic, variant and conflict annotations. New
records are produced in either pseduo-SWISS-PROT or FASTA format for each
specified variant. More information and the data is available at
ftp://ftp.ebi.ac.uk/pub/databases/sp_tr_nrdb/

                6. Access/Data Distribution

FTP server:     ftp.ebi.ac.uk/pub/databases/trembl
SRS server:     http://srs.ebi.ac.uk/

TrEMBL is also available on the SWISS-PROT CD-ROM.
SWISS-PROT + TrEMBL is searchable on the following servers at the EBI:

FASTA3  (http://www.ebi.ac.uk/fasta3/)
BLAST2  (http://www.ebi.ac.uk/blast2/)
Bic_sw  (http://www.ebi.ac.uk/bic_sw/)
Scanps  (http://www.ebi.ac.uk/scanps/)
MPSrch  (http://www.ebi.ac.uk/MPsrch/)


  7. Description of changes made to TrEMBL since release 14.

7.1  Changes concerning cross-references (DR line)

We have added cross-references from TrEMBL to the REBASE database
available at ftp://ftp.neb.com/pub/rebase and ftp://ftp.ebi.ac.uk/pub/databases/rebase

A list of all crossreferences in TrEMBL to other databases is provided below:
AARHUS/GHENT-2DPAGE;
EMBL
FLYBASE
GCRDB
HSSP
MENDEL
MGD
MIM
PDB
INTERPRO
PFAM
PIR
PRINTS
PRODOM
PROSITE
SGD
SUBTILIST
TIGR
TRANSFAC
WORMPEP
ZFIN

7.2 Introduction of a new line type and code (OX line)

The OX (Organism taxonomy cross-reference) line contains
cross-references to taxonomy resource(s) for the organism(s)
listed in the OS line(s) of the entry.
The format of the OX line is:

OX   TAXONOMY_RESOURCE_NAME=IDENTIFIER_1[, IDENTIFIER_2,...
IDENTIFIER_N];

Where the currently defined taxonomic resources are:

Name:     NCBI_TaxID
Resource: National Center for Biotechnology Information Taxonomy Browser

- The identifiers are listed in the order of the corresponding organisms
in the OS line(s).
- In the rare case where all the information will not fit on a single
line more than one OX line is used.

Examples:
OX   NCBI_TaxID=256;
OX   NCBI_TaxID=126566, 38, 846, 23412;

The OX line(s) occurs after the OC lines in the entry line order.


7.3 Modification of the RX line type

We extended the scope of the RX line to include PubMed
as well as Medline. We also changed the syntax of this
line.
An example is given below:

RX  MEDLINE=99382246; PubMed=10448073;

  8. Planned Changes

8.1 Please note that the TrEMBL files will be distributed from the EBI's ftp
    server as gzip files (*.gz) as of the next release (Rel.16) instead of
    compress (*.Z) files. Please contact us if you have any queries
    regarding this.




  </pre>
</div>