<div>
 <pre>

                              TrEMBL Release Notes
                              Release 24, June 2003


EMBL Outstation
European Bioinformatics Institute (EBI)
Wellcome Trust Genome Campus
Hinxton
Cambridge CB10 1SD
United Kingdom

Telephone: (+44 1223) 494 444
Fax: (+44 1223) 494 468
Electronic mail address: datalib@ebi.ac.uk/swissprot@ebi.ac.uk
WWW server: http://www.ebi.ac.uk/

Swiss Institute of Bioinformatics (SIB)
Centre Medical Universitaire
1, rue Michel Servet
1211 Geneva 4
Switzerland

Telephone: (+41 22) 702 50 50
Fax: (+41 22) 702 58 58
Electronic mail address: Amos.Bairoch@isb-sib.ch
WWW server: http://www.expasy.org/


Acknowledgements

UniProtKB/TrEMBL has been prepared by:

o Maria Jesus Martin, Claire O'Donovan, Nicola Althorpe, Rolf
Apweiler, Daniel Barrell, Kirsty Bates, Paul Browne, Kirill
Degtyarenko, Ruth Eberhardt, Gill Fraser, Alexander Fedetov, Andre
Hackmann, Alexander Kanapin, Youla Karavidopoulou, Paul Kersey, Ernst
Kretschmann, Kati Laiho, Minna Lehvaslaiho, Michele Magrane, Michelle
McHale, Virginie Mittard, Nicola Mulder, John F. O'Rourke, Markiyan
Oliynyk, Sandra Orchard, Astrid Rakow, Sandra van den Broek, Eleanor
Whitfield and Allyson Williams at the EMBL Outstation - European
Bioinformatics Institute (EBI) in Hinxton, UK;
o Amos Bairoch, Alexandre Gattiker, Karine Michoud, Isabelle Phan and
Sandrine Pilbout at the Swiss Institute of Bioinformatics in Geneva,
Switzerland.

Copyright Notice
TrEMBL copyright (c) 2003 EMBL-EBI
This manual and the database it accompanies may be copied and
redistributed freely, without advance permission, provided
that this copyright statement is reproduced with each copy.

Citation

If you want to cite TrEMBL in a publication please use
the following reference:

Boeckmann B., Bairoch A., Apweiler R., Blatter M., Estreicher A.,
Gasteiger E., Martin M.J., Michoud K., O'Donovan C., Phan I.,
Pilbout S., and Schneider M. (2003)
The Swiss-Prot protein knowledgebase and its supplement TrEMBL in
2003.
Nucleic Acids Res. 31:365-370.


1. Introduction


UniProtKB/TrEMBL is a computer-annotated protein sequence database
complementing the UniProtKB/Swiss-Prot Protein Knowledgebase. UniProtKB/TrEMBL contains the translations of all coding sequences (CDS) present in the
EMBL/GenBank/DDBJ Nucleotide Sequence Databases and also protein
sequences extracted from the literature or submitted to Swiss-Prot,
which are not yet integrated into UniProtKB/Swiss-Prot. For all UniProtKB/TrEMBL entries which should finally be upgraded to the standard UniProtKB/Swiss-Prot quality,
UniProtKB/Swiss-Prot accession numbers have been assigned.

2. Why a complement to UniProtKB/Swiss-Prot?

The ongoing gene sequencing and mapping projects have dramatically
increased the number of protein sequences to be incorporated into
UniProtKB/Swiss-Prot. We do not want to dilute the quality standards of UniProtKB/Swiss-Prot by incorporating sequences without proper sequence analysis and
annotation, but we do want to make the sequences available as quickly as
possible. UniProtKB/TrEMBL achieves this second goal, and is a major step in the
process of speeding up subsequent upgrading of annotation to the standard
UniProtKB/Swiss-Prot quality. To address the problem of redundancy, the translations
of all coding sequences (CDS) in the EMBL Nucleotide Sequence Database
already included in Swiss-Prot have been removed from UniProtKB/TrEMBL.


3. The Release

This UniProtKB/TrEMBL release has been produced in synch with UniProtKB/Swiss-Prot release 41.13. It was created from the EMBL Nucleotide Sequence Database release 74 and updates until the 16-May-2003 and contains 1'043'240 entries and 310'062'472
amino acids.

TrEMBL is split in two main sections: SP-TrEMBL and REM-TrEMBL:
SP-TrEMBL (Swiss-Prot TrEMBL) contains the entries (944'868) which should
be eventually incorporated into UniProtKB/Swiss-Prot. UniProtKB/Swiss-Prot accession numbers have been assigned for all SP-TrEMBL entries.

SP-TrEMBL is organized in subsections:

arc.dat (Archaea): 1796 entries
arp.dat (Complete Archaeal proteomes): 31367 entries
fun.dat (Fungi): 17715 entries
hum.dat (Human): 35721 entries
inv.dat (Invertebrates): 84696 entries
mam.dat (Other Mammals): 13529 entries
mhc.dat (MHC proteins): 9252 entries
org.dat (Organelles): 81685 entries
phg.dat (Bacteriophages): 8188 entries
pln.dat (Plants): 86647 entries
pro.dat (Prokaryotes): 86373 entries
prp.dat (Complete Prokaryotic Proteomes): 245103 entries
rod.dat (Rodents): 42386 entries
unc.dat (Unclassified): 341 entries
vrl.dat (Viruses): 89781 entries
vrt.dat (Other Vertebrates): 18607 entries
vrv.dat (Retroviruses): 91681 entries

124'197 new entries have been integrated in SP-TrEMBL. The sequences of
1509 SP-TrEMBL entries have been updated and the annotation has been
updated in 651'912 entries.

In the document deleteac.txt, you will find a list of all accession numbers
which were previously present in TrEMBL, but which have now been deleted from
the database.

REM-TrEMBL (REMaining TrEMBL) contains the entries (98'372) that we do
not want to include in UniProtKB/Swiss-Prot. REM-TrEMBL entries do not have UniProtKB/Swiss-Prot accession numbers. Instead the stable ID portion of the protein_id present in the source EMBL/DDBJ/GenBank nucleotide sequence database entries is
used as the ID and accession number.This section is organized in six
subsections:

1) Immunoglobulins and T-cell receptors (Immuno.dat)
Most REM-TrEMBL entries are immunoglobulins and T-cell receptors. We
stopped entering immunoglobulins and T-cell receptors into Swiss-Prot,
because we only want to keep the germ line gene derived translations
of these proteins in Swiss-Prot and not all known somatic recombinated
variations of these proteins. We would like to create a specialized
database dealing with these sequences as a further supplement to
Swiss-Prot and keep only a representative cross-section of these
proteins in Swiss-Prot.

2) Synthetic sequences (Synth.dat)
Another category of data, which will not be included in UniProtKB/Swiss-Prot are
synthetic sequences. Again, we do not want to leave these entries in
UniProtKB/TrEMBL. Ideally one should build a specialized database for artificial
sequences as a further supplement to Swiss-Prot.

3) Patent application sequences (Patent.dat)
A third subsection consists of coding sequences captured from patent
applications. A thorough survey of these entries have shown that
apart from a rather small minority (which in most cases have already
been integrated in UniProtKB/Swiss-Prot), most of these sequences contain either
erroneous data or concern artificially generated sequences outside the
scope of UniProtKB/Swiss-Prot.

4) Small fragments (Smalls.dat)
Another subsection consists of fragments with less than eight amino
acids.

5) CDS not coding for real proteins (Pseudo.dat)
This subsection consists of CDS translations where we have strong
evidence to believe that these CDS are not coding for real proteins.

6) Truncated proteins (Truncated.dat)
The last subsection consists of truncated proteins which result from
events like mutations introducing a stop codon leading to the truncation
of the protein product.

4. Format Differences Between UniProtKB/Swiss-Prot and UniProtKB/TrEMBL

The format and conventions used by UniProtKB/TrEMBL follow as closely as possible
that of UniProtKB/Swiss-Prot. Hence, it is not necessary to produce an additional
user manual and extensive release notes for UniProtKB/TrEMBL. The information given
in the UniProtKB/Swiss-Prot release notes and user manual are in general valid for
UniProtKB/TrEMBL. The differences are mentioned below.

The general structure of an entry is identical in UniProtKB/Swiss-Prot and UniProtKB/TrEMBL. The data class used in TrEMBL (in the ID line) is always 'PRELIMINARY', whereas in UniProtKB/Swiss-Prot it is always 'STANDARD'.

Differences in line types present in UniProtKB/Swiss-Prot and UniProtKB/TrEMBL:

The ID line (IDentification):

The entry name used in SP-TrEMBL is the same as the Accession Number of the
entry. The entry name used in REM-TrEMBL is the stable part of the protein_id
tagged to the corresponding CDS in the EMBL Nucleotide Sequence Database.
'protein_id' stands for the "Protein Identification" number. It is a number
that you will find in the feature table of the EMBL nucleotide sequence
entries in a qualifier called "/protein_id" which is tagged to every CDS.

Example:

FT CDS 339..1514
FT /codon_start=1
FT /db_xref="PID:g1256015"
FT /product="dystrobrevin-epsilon"
FT /protein_id="AAC50431.1"

The protein_id is defined as follows in the The DDBJ/EMBL/GenBank Feature Table
Definition documentation
Qualifier /protein_id
Definition Protein Identifier, issued by International collaborators.
This qualifier consists of a stable ID portion (3+5 format
with 3 position letters and 5 numbers) plus a version
number after the decimal point.

Value format &lt;identifier&gt;
Example /protein_id="AAA12345.1"
Comment When the protein sequence encoded by the CDS changes, only
the version number of the /protein_id value is incremented.
The stable part of the /protein_id remains unchanged and
as a result will permanently be associated with a given
protein. This qualifier is valid only on CDS features
which translate into a valid protein.


The DT line (DaTe)

The format of the DT lines that serve to indicate when an entry was
created and updated are identical to that defined in Swiss-Prot; but the
DT lines in UniProtKB/TrEMBL refer to the UniProtKB/TrEMBL release. The difference is shown in the example below.

DT lines in a UniProtKB/Swiss-Prot entry:

DT 01-JAN-1988 (Rel. 06, Created)
DT 01-JUL-1989 (Rel. 11, Last sequence update)
DT 01-AUG-1992 (Rel. 23, Last annotation update)

DT lines in a TrEMBL entry:

DT 01-NOV-1996 (TrEMBLrel. 01, Created)
DT 01-NOV-1999 (TrEMBLrel. 12, Last sequence update)
DT 28-FEB-2003 (TrEMBLrel. 23, Last annotation update)

5. Weekly updates of UniProtKB/TrEMBL and non-redundant data sets

5.1 TrEMBL updates

Weekly cumulative updates of UniProtKB/TrEMBL are available by anonymous FTP and
from the EBI SRS server.

5.2 SPTr

We also produce every week a complete non-redundant protein sequence
collection by providing three compressed files (these are in the directory
/pub/databases/sp_tr_nrdb on the EBI FTP server and in databases/sp_tr_nrdb
on the ExPASy server): sprot.dat.gz, trembl.dat.gz and trembl_new.dat.gz.
This set of non-redundant files is especially important for two types of
users:
(i) Managers of similarity search services. They can now provide what is
currently the most comprehensive and non-redundant data set of protein
sequences.
(ii) Anybody wanting to update their full copy of Swiss-Prot + TrEMBL to
their own schedule without having to wait for full releases of Swiss-Prot
or TrEMBL.

5.3 XML

A version of UniProtKB/Swiss-Prot and UniProtKB/TrEMBL in XML format has been developed and is provided with this release. More information is available at
http://www.ebi.ac.uk/swissprot/SP-ML and the data can be downloaded
from ftp://ftp.ebi.ac.uk/pub/databases/trembl/xml and
ftp://ftp.ebi.ac.uk/pub/databases/sp_tr_nrdb/xml

We would welcome any feedback from the user community.

5.4 Varsplic Expand

We also provide Varsplic Expand which is a program to generate
" expanded" sequences from UniProtKB/Swiss-Prot and UniProtKB/TrEMBL records i.e. sequences including the variants specified by the varsplic, variant and conflict
annotations. New records are produced in either pseudo-Swiss-Prot or
FASTA format for each specified variant. More information and the data is
available at ftp://ftp.ebi.ac.uk/pub/databases/sp_tr_nrdb/

6. Access/Data Distribution

FTP server: ftp.ebi.ac.uk/pub/databases/trembl
SRS server: http://srs.ebi.ac.uk/

UniProtKB/TrEMBL is also available on the UniProtKB/Swiss-Prot CD-ROM.
UniProtKB/Swiss-Prot + UniProtKB/TrEMBL is searchable on the following servers at the EBI:

FASTA3 (http://www.ebi.ac.uk/fasta33/)
BLAST2 (http://www.ebi.ac.uk/blast2/)
Scanps (http://www.ebi.ac.uk/scanps/)
MPSrch (http://www.ebi.ac.uk/MPsrch/)

For each UniProtKB/TrEMBL release, a synchronized version of the concurrent UniProtKB/Swiss-Prot release is distributed at ftp.ebi.ac.uk/pub/databases/trembl/swissprot/

7. Description of changes made to TrEMBL since release 23.

7.1 Changes concerning cross-references (DR line)

We have added cross-references from UniProtKB/TrEMBL to three new databases.

7.1.1 GK

We have added cross-references to the Genome Knowledgebase (GK)
(available at http://www.genomeknowledgebase.org)

7.1.2 GO

We have added cross-references to the Gene Ontology database (GO)
(available at http://www.geneontology.org).

7.1.3 PIRSF

We have added cross-references to PIR superfamilies, an integrated
protein classification database
(available at http://pir.georgetown.edu/iproclass)


8. Forthcoming changes

8.1 Evidence tags

We are continuing with the introduction of evidence tags to UniProtKB/Swiss-Prot and
UniProtKB/TrEMBL entries. The aim of this is to allow users to see where data items
came from and to enable Swiss-Prot staff to automatically update data if
the underlying evidence changes. This is ongoing internally and the evidence
tags are visible in the XML version of UniProtKB/Swiss-Prot and UniProtKB/TrEMBL.
For more information, please see
http://www.ebi.ac.uk/swissprot/SP-ML/evidence.html
We would welcome any feedback from the user community.

8.2 Conversion of TrEMBL to mixed case

Most of the DE (DEscription), GN (Gene Name), RC (Reference Comment)
and CC (Comment) lines have been converted to mixed case internally. The
conversion is ongoing and will be made public as the conversion of each
line type reaches a satisfactory stage. A mixed case version of the DE
line was made public in release 22 and the RC line in release 23.

8.3 Version of SPTr in XML format:

We intend to provide an XML version of SPTr updated monthly

8.4 New format of comment line (CC) topics

We are continuing a major overhaul of various comment line topics. We would
like the majority of the information stored to be usable by computer
programs (while remaining human-readable). We are therefore standardizing
the format of the topics.
'ALTERNATIVE PRODUCTS' was the first topic to be modified.

For more information, please see
http://us.expasy.org/sprot/relnotes/sp_news.html

8.5 Modifications concerning the feature table (FT line)

We are investigating a major effort in the annotation of posttranslational
modifications, which has an effect on various feature keys and feature
descriptions.
In this release, the feature key 'CROSSLNK' has been
introduced to describe bonds between amino acids, which are formed
posttranslationally within a peptide or between peptides, such as isopeptidic
bonds, carbon-carbon-linkages, carbon-nitrogen linkages, thioether bonds,
thoiether bonds, thiolester bonds, and backbone condensations.

For more information, please see
http://us.expasy.org/sprot/relnotes/sp_news.html

8.6 Extension of the entry name format

Currently TrEMBL has it's accession number as the entry name. It is
intended to extend this to have the accession number as the protein
name component of the entry name (having elongated the mnemonic code
from 4 characters to 6) and to assign the mnemonic species identification
code of at most 5 alphanumeric characters as Swiss-Prot currently does.


9. General announcements

9.1 REM-TrEMBL

It has been decided to discontinue the production of REM-TrEMBL. The next
release will be the last one. This is because REM-TrEMBL has been fully
integrated into our new database resource UniParc which will be made
public in the near future. We will continue to produce releases of
SP-TrEMBL quarterly.

9.2 What's new?

We have introduced a new resource for users to enable us to communicate
effectively between releases about what is new in UniProtKB/Swiss-Prot and UniProtKB/TrEMBL.
This is available at: http://us.expasy.org/sprot/relnotes/sp_news.html

  </pre>
</div>