<div>
 <h2>
  Headlines
 </h2>
 <h3>
  Extinct organisms and Swiss-Prot...
 </h3>
 <p>
  Did you know that Swiss-Prot contains proteins originating from extinct organisms?
     Since the beginning of the 90s, various groups have sequenced gene fragments
     from a variety of extinct organisms. Most of the time, the resulting sequences
     are too small or too fragmentary to be translated into protein sequences. But
     this is not always the case, and we harbor a few complete or partial sequences
      originating from species that existed on earth in various periods of time.
 </p>
 <p>
  For example we have a RuBisCO large subunit from a fossil leaf of a Miocene
      (17-20 Myr old) Magnolia,
  <a href="http://www.uniprot.org/uniprot/P30828">
   P30828
  </a>
  .
 </p>
 <p>
  Much more recent is a complete cytochrome b sequence from a Siberian
      mammoth,
  <a href="http://www.uniprot.org/uniprot/P92658">
   P92658
  </a>
  .
 </p>
 <p>
  But what is more interesting for those interested in the longevity of
        proteins, is the complete sequence of an osteocalcin from a steppe bison.
        This sequence was ontained by mass spectrometry directly from
        permafrost fossilized bones, about 55-56 Kyr old,
  <a href="http://www.uniprot.org/uniprot/P83489">
   P83489
  </a>
  .
 </p>
 <h2>
  UniProtKB news
 </h2>
 <h3>
  Cross-references to Oxford GlycoProteomics 2-DE database (OGP)
 </h3>
 <p>
  We have added cross-references to the Oxford GlycoProteomics 2-DE database (OGP), available
at
  <a href="http://proteomewww.bioch.ox.ac.uk/2d/2d.html">
   http://proteomewww.bioch.ox.ac.uk/2d/2d.html
  </a>
  .
 </p>
 <p>
  The identifiers of the appropriate DR line are:
 </p>
 <table>
  <tr>
   <th>
    Resource abbreviation
   </th>
   <td>
    OGP
   </td>
  </tr>
  <tr>
   <th>
    Resource identifier
   </th>
   <td>
    OGP's unique identifier for a protein, which is identical to the
      Swiss-Prot primary AC number of that protein.
   </td>
  </tr>
  <tr>
   <th>
    Example
   </th>
   <td>
    <pre><a href="http://www.uniprot.org/uniprot/P31946#section_x-ref">P31946</a>:
DR   OGP; P31946; -.</pre>
   </td>
  </tr>
 </table>
 <h3>
  Cross-references to 2-DE database of rat heart
 </h3>
 <p>
  We have added cross-references to the 2-DE database of rat heart, at German Heart Institute Berlin, available
at
  <a href="http://www.mpiib-berlin.mpg.de/2D-PAGE/RAT-HEART/2d/">
   http://www.mpiib-berlin.mpg.de/2D-PAGE/RAT-HEART/2d/
  </a>
  .
 </p>
 <p>
  The identifiers of the appropriate DR line are:
 </p>
 <table>
  <tr>
   <th>
    Resource abbreviation
   </th>
   <td>
    Rat-heart-2DPAGE
   </td>
  </tr>
  <tr>
   <th>
    Resource identifier
   </th>
   <td>
    Rat-heart-2DPAGE's unique identifier for a protein, which is identical to the
      Swiss-Prot primary AC number of that protein.
   </td>
  </tr>
  <tr>
   <th>
    Example
   </th>
   <td>
    <pre><a href="http://www.uniprot.org/uniprot/P03996#section_x-ref">P03996</a>:
DR   Rat-heart-2DPAGE; P03996; -.</pre>
   </td>
  </tr>
 </table>
 <h3>
  Changes concerning keywords
 </h3>
 <p>
  New keyword:
 </p>
 <ul>
  <li>
   <a href="http://www.uniprot.org/keywords/KW-0901">
    Leber congenital amaurosis
   </a>
  </li>
  <li>
   <a href="http://www.uniprot.org/keywords/KW-0902">
    Two-component regulatory system
   </a>
  </li>
 </ul>
 <br/>
 <h3>
  Changes concerning the
  <a href="http://www.uniprot.org/docs/ptmlist">
   controlled vocabulary for PTMs
  </a>
 </h3>
 <p>
  We are continuously overhauling the annotation of post-translational modifications (PTMs).
Phosphorylation sites are described in the description field of the feature key MOD_RES, all entries
with such a site contain the keyword 'Phosphorylation'. The initially defined controlled
vocabulary for phosphorylation sites is listed below:
 </p>
 <pre>
Phosphocysteine
4-aspartylphosphate
Phosphohistidine
Tele-phosphohistidine
Pros-phosphohistidine
Phosphoserine
Phosphothreonine
Phosphotyrosine
</pre>
</div>