<div>
 <h2>
  Headlines
 </h2>
 <h3>
  Over 100'000 prokaryotic entries in UniProtKB/Swiss-Prot
 </h3>
 <p>
  We have reached &gt;100'000 prokaryotic (bacterial and archaeal) entries in UniProtKB/Swiss-Prot.
Of these, just over 10'000 are archaeal entries.
To deal with the enormous increase in the amount of available prokaryotic protein sequences,
 the Swiss-Prot group started the
  <a href="http://www.expasy.org/sprot/hamap/">
   HAMAP project
  </a>
  , which aims to automatically annotate, with a high-throughput but with no decrease in quality, proteins from complete microbial genomes that belong to a family (well-defined or uncharacterized).
 </p>
 <p>
  The HAMAP annotation system is based on
  <a href="http://www.expasy.org/sprot/hamap/families.html">
   manually curated family rules
  </a>
  , which contain the information,
 derived from searches of the available literature, that can be safely propagated to all members of the family.
 Profiles are generated from an alignment of seed members; these profiles, in turn, are used to scan all available
 sequences in Swiss-Prot and TrEMBL to identify family members. The goal of the system is not to search for distant
 sequence similarities, but to annotate only the proteins that can be conservatively assigned to a HAMAP family.
 Cases and conditions are included in most rules so that warnings are generated if some conserved features, such
 as active sites or metal-binding amino acids, are not present in a given protein sequence, or if there are other
 problems, such as size or taxonomic range. All entries that contain warnings are subjected to manual verification.
In fact, since the implementation of the system and for the time being, ALL the proteins that have been annotated
using the HAMAP family rules have been manually verified to check the reliability of the HAMAP annotation module.
 </p>
 <p>
  More than 1'200 family rules are available, and almost 70'000 prokaryotic entries belong to one (or more) HAMAP families.
 From the
  <a href="http://www.expasy.org/sprot/hamap/">
   HAMAP website
  </a>
  it is possible to scan protein sequences for matches against HAMAP families, and it is also possible to submit a whole proteome, by confidential ftp, to be scanned against the collection of HAMAP family rules.
 </p>
 <br/>
 <h2>
  UniProtKB News
 </h2>
 <h3>
  Changes concerning keywords
 </h3>
 <p>
  New keywords:
 </p>
 <ul>
  <li>
   <a href="http://www.uniprot.org/keywords/KW-0937">
    Abscisic acid biosynthesis
   </a>
  </li>
  <li>
   <a href="http://www.uniprot.org/keywords/KW-0938">
    Abscisic acid signaling pathway
   </a>
  </li>
  <li>
   <a href="http://www.uniprot.org/keywords/KW-0936">
    Ethylene signaling pathway
   </a>
  </li>
  <li>
   <a href="http://www.uniprot.org/keywords/KW-0935">
    Progressive external ophthalmoplegia
   </a>
  </li>
 </ul>
 <p>
  Deleted keywords:
 </p>
 <ul>
  <li>
   Dehydrin
  </li>
 </ul>
</div>