<div>
 <p>
  <a href="http://www.expasy.org/txt/old-rel/relnotes.49.htm">
   Full statistics and release notes
  </a>
 </p>
 <h2>
  Headlines
 </h2>
 <h3>
  UniProtKB/Swiss-Prot major release (49.0)
 </h3>
 <p>
  Release 49.0 of 07-Feb-2006 of UniProtKB/Swiss-Prot contains
     207'132 sequence entries, comprising 75'438'310 amino acids
     abstracted from 139'151 references. 12'815 sequences have been
     added since release 48, the sequence data of
     991 existing entries has been updated and the annotations of all
     entries have been revised. This represents an increase of 7%.
 </p>
 <p>
  Many improvements were carried out in the last 5 months.
     In particular, we have changed from showing only the dates
     corresponding to full UniProtKB releases in the DT lines to
     displaying the date of the biweekly release at which an
     entry is integrated or updated. We dropped the information
     concerning the release number and introduced entry and
     sequence version numbers in the DT lines.
 </p>
 <p>
  Cross-references to several databases have been added,
     and we have changed our copyright statement.
     (Please read below for further details)
 </p>
 <p>
  <a href="http://www.expasy.org/txt/old-rel/relnotes.49.htm">
   Full statistics and release notes
  </a>
 </p>
 <h2>
  UniProtKB News
 </h2>
 <h3>
  Dates and Versions
 </h3>
 <p>
  We changed from showing only the dates corresponding to full UniProtKB
    releases in the DT lines to displaying the date of the biweekly release at which
    an entry is integrated or updated. We dropped the information concerning the
    release number and introduced entry and sequence version numbers in the DT
    lines.
 </p>
 <p>
  The new format of the three DT lines is:
 </p>
 <pre>
DT   DD-MMM-YYYY, integrated into UniProtKB/<em>database_name</em>.
DT   DD-MMM-YYYY, sequence version <em>version_number</em>.
DT   DD-MMM-YYYY, entry version <em>version_number</em>.
</pre>
 <p>
  Example for UniProtKB/Swiss-Prot:
 </p>
 <pre>
DT   01-JAN-1998, integrated into UniProtKB/Swiss-Prot.
DT   15-OCT-2001, sequence version 3.
DT   01-APR-2004, entry version 14.
</pre>
 <p>
  Example for UniProtKB/TrEMBL:
 </p>
 <pre>
DT   01-FEB-1999, integrated into UniProtKB/TrEMBL.
DT   15-OCT-2000, sequence version 2.
DT   15-DEC-2004, entry version 5.
</pre>
 <p>
  The sequence version number of an entry is incremented by one when its amino
    acid sequence is modified. The entry version number is incremented by one
    whenever any data in the flat file representation of the entry is modified.
 </p>
 <p>
  We retrofitted the entry and sequence version numbers, as well as all dates,
    using archived UniProtKB releases.
 </p>
 <h3>
  Addition of a feature (FT) key CHAIN over the whole sequence length
 </h3>
 <p>
  The feature key
  <a href="http://www.uniprot.org/manual/chain">
   CHAIN
  </a>
  was previously only used to describe processed protein sequences. Now we added,
    in the UniProtKB/Swiss-Prot database, a "FT CHAIN" to all the entries having neither a
    "FT CHAIN" nor a "FT PEPTIDE". This led to the addition of a "FT CHAIN", covering
    the full length of the sequence, to more than 170 000 entries. In this way, in
    UniProtKB/Swiss-Prot, all the mature proteins will be described in the feature
    lines.
 </p>
 <h3>
  Release of a new document about post-translational modifications
 </h3>
 <p>
  The controlled vocabulary for the post-translational modifications (PTMs)
    that are annotated in the UniProtKB feature table has moved from the UniProtKB
    User Manual to a separate document,
  <a href="http://www.uniprot.org/docs/ptmlist">
   ptmlist.txt
  </a>
  .
 </p>
 <p>
  The document contains, for each individual modification, the controlled
    vocabulary term and its associated feature key, and additional information, such
    as the amino acid that can be modified and the mass difference, the position on
    the protein sequence, the taxonomic distribution, the protein location,
    associated keyword(s), as well as links to the UniProtKB entries that contain
    the annotation, and to the corresponding entry in the RESID Database of Protein
    Modifications.
 </p>
 <h3>
  Copyright
 </h3>
 We have changed the copyright statement in all UniProt Knowledgebase entries.

    All UniProtKB/Swiss-Prot and UniProtKB/TrEMBL entries, as well as all
 <a href="http://www.uniprot.org/docs/">
  documents
 </a>
 now contain the statement
 <pre>
CC   -----------------------------------------------------------------------
CC   Copyrighted by the UniProt Consortium, see http://www.uniprot.org/terms
CC   Distributed under the Creative Commons Attribution-NoDerivs License
CC   -----------------------------------------------------------------------
</pre>
 <h3>
  Cross-References to PptaseDB
 </h3>
 <p>
  Cross-references have been added to the
  <a href="http://vigen.biochem.vt.edu/p3d/p3d.htm">
   Prokaryotic Protein Phosphatase Database
  </a>
  , a database which provides information concerning
    prokaryotic and archaeal phosphatases for which experimental evidence exists
    demonstrating phosphatase activity. The Prokaryotic Protein Phosphatase Database is available at
  <a href="http://vigen.biochem.vt.edu/p3d/p3d.htm">
   http://vigen.biochem.vt.edu/p3d/p3d.htm
  </a>
  .
 </p>
 <p>
  The format of the explicit links in the flat file is:
 </p>
 <table>
  <tr>
   <th>
    Resource abbreviation
   </th>
   <td>
    PptaseDB
   </td>
  </tr>
  <tr>
   <th>
    Resource identifier
   </th>
   <td>
    PptaseDB unique phosphatase identifier.
   </td>
  </tr>
  <tr>
   <th>
    Example
   </th>
   <td>
    <pre><a href="http://www.uniprot.org/uniprot/O52787#section_x-ref">O52787</a>:
DR   PptaseDB; P3D040495; -.</pre>
   </td>
  </tr>
 </table>
 <h3>
  Cross-references to BioCyc
 </h3>
 <p>
  Cross-references have been added to
  <a href="http://www.biocyc.org/">
   BioCyc
  </a>
  ,
    a collection of Pathway/Genome Databases. Each Pathway/Genome
    Database describes the genome and metabolic pathways of a single organism,
    with the exception of the MetaCyc database, which is a reference source on
    metabolic pathways from many organisms.
    BioCyc is available at
  <a href="http://www.biocyc.org/">
   http://www.biocyc.org/
  </a>
  .
 </p>
 <p>
  Implicit links to the EcoCyc database have already been provided before in the
    NiceProt view of relevant Swiss-Prot entries on ExPASy.
 </p>
 <p>
  The format of the explicit links in the flat file is:
 </p>
 <table>
  <tr>
   <th>
    Resource abbreviation
   </th>
   <td>
    BioCyc
   </td>
  </tr>
  <tr>
   <th>
    Resource identifier
   </th>
   <td>
    BioCyc database code and identifier, separated by a colon.
   </td>
  </tr>
  <tr>
   <th>
    Examples
   </th>
   <td>
    <pre><a href="http://www.uniprot.org/uniprot/P21170#section_x-ref">P21170</a>:
DR   BioCyc; EcoCyc:ARGDECARBOXBIO-MONOMER; -.

<a href="http://www.uniprot.org/uniprot/Q9HCC0#section_x-ref">Q9HCC0</a>:
DR   BioCyc; MetaCyc:MONOMER-10082; -.</pre>
   </td>
  </tr>
 </table>
</div>