<div>
 <h2>
  Headlines
 </h2>
 <h3>
  Major update of influenza A viruses: H5N1 pathogenicity
 </h3>
 <p>
  Influenza A viruses are named depending on their surface protein subtype, H
    for hemagglutinin and N for neuraminidase. There are 16 known H subtypes and 9
    known N subtypes for influenza A virus, all of them infect birds, a few such as
    H1N1, H1N2 and H3N2 can infect human.
 </p>
 <p>
  'Avian influenza' is used to name viruses commonly restricted to birds, such
    as H5,H7,H9,... subtypes. Most avian influenza subtypes cause very mild diseases,
    but the H5 and H7 subtypes can cause outbreaks involving massive deaths in domestic
    poultry. During these outbreaks, sporadic transmission to human has been reported.
    Fortunately humans are dead end hosts for these viruses, i.e. infected humans do
    not transmit the virus.
    Although few human cases of H7N7 and H9N2 have been documented, the major threat
    remains the H5N1 subtype.
 </p>
 <p>
  H5N1 is not a new virus, it was isolated on birds in Scotland back in 1959
    (hemagglutinin:
  <a href="http://www.uniprot.org/uniprot/P09345">
   P09345
  </a>
  ). It became famous after the
    first  big outbreak in 1997 in Hong Kong, where 1.5 millions of poultry were
    affected and destroyed, and 18 human cases occurred, six of whom died
    (hemagglutinin:
  <a href="http://www.uniprot.org/uniprot/O56140">
   O56140
  </a>
  ). This was the first time an
    avian influenza A virus  transmission directly from birds to humans had been found.
 </p>
 <p>
  In 2003 two cases of H5N1 occurred in Hong Kong, one fatal. How or where these
    two family members were infected  was not determined.
    In 2004 and 2005, severe outbreaks happened in Thailand, Vietnam, Cambodia and
    Indonesia, for a total of 130 human cases, 70 of whom died. Most of these cases
    occurred as a result of people having direct or close contact with infected poultry,
    however a few cases of human-to-human spread of H5N1 have occurred.
 </p>
 <p>
  Why is H5N1 so deadly in poultry and humans?
    Presumably because of small sequence variations in hemagglutinin.
 </p>
 <p>
  Hemaglutinin is present at the virion surface, and its function both to
    bind cellular receptor and induce fusion of viral and target cell membrane.
    In order to be able to promote fusion, the protein must be cleaved. In common
    influenza A viruses, the cleavage site is specific to proteases present
    in the respiratory tract. Hence influenza is restricted to infect this organ.
 </p>
 <p>
  H5 and H7 have a completely different cleavage site,  rich in arginine and
    lysine residues (RRRKKR in Hong Kong 1997:
  <a href="http://www.uniprot.org/uniprot/O56140">
   O56140
  </a>
  ),
    which can be processed by ubiquitous proteases: furins.
    This result in an infection of almost all host organs, and an acute pathology
    which can be quickly fatal.
 </p>
 <p>
  Few antiviral drugs are effective against influenza. Zanamivir (Relenza)
    and oseltamivir (Tamiflu) are inhibitors of the neuraminidase
    (e.g.
  <a href="http://www.uniprot.org/uniprot/Q9W7Y7">
   Q9W7Y7
  </a>
  ), amantadine and rimantadine are
    inhibitors of ion channel M2 protein (e.g.
  <a href="http://www.uniprot.org/uniprot/O70632">
   O70632
  </a>
  ).
    Unfortunately drug resistance evolves rapidly, and already a case of H5N1 resistant
    to Tamiflu has been reported
    (
  <a href="http://view.ncbi.nlm.nih.gov/pubmed/16228009">
   Nature 437:1108-1108(2005)
  </a>
  ).
 </p>
 <p>
  The following are examples of updated influenza entries:
 </p>
 <p>
  H5N1 isolated from human, in Hong Kong 1997:
 </p>
 Hemagglutinin: HEMA_IAHO3 (
 <a href="http://www.uniprot.org/uniprot/O56140">
  O56140
 </a>
 )
    Neuraminidase:  NRAM_IAHO3 (
 <a href="http://www.uniprot.org/uniprot/Q9W7Y7">
  Q9W7Y7
 </a>
 )
    M1: M1_IAHO3 (
 <a href="http://www.uniprot.org/uniprot/Q77Y95">
  Q77Y95
 </a>
 )
    M2: M2_IAHO3 (
 <a href="http://www.uniprot.org/uniprot/O70632">
  O70632
 </a>
 )
    NS1 : NS1_IAHO3 (
 <a href="http://www.uniprot.org/uniprot/O56264">
  O56264
 </a>
 )
    NEP: NEP_IAHO3 (
 <a href="http://www.uniprot.org/uniprot/O56263">
  O56263
 </a>
 )
    PB1-F2 : PB1F2_IAHO3 (
 <a href="http://www.uniprot.org/uniprot/P0C0U0">
  P0C0U0
 </a>
 )
    Nucleoprotein: NCAP_IAHO3 (
 <a href="http://www.uniprot.org/uniprot/O92784">
  O92784
 </a>
 )
           PA : PA_IAHO3 (
 <a href="http://www.uniprot.org/uniprot/O89752">
  O89752
 </a>
 )
    PB1 : RDRP_IAHO3 (
 <a href="http://www.uniprot.org/uniprot/Q9WLS3">
  Q9WLS3
 </a>
 )
    PB2: PB2_IAHO3 (
 <a href="http://www.uniprot.org/uniprot/O56266">
  O56266
 </a>
 )
 <p>
  H5N1 isolated in 1959 on chicken:
 </p>
 <p>
  Hemagglutinin: HEMA_IACKS (
  <a href="http://www.uniprot.org/uniprot/P09345">
   P09345
  </a>
  )
 </p>
 <br/>
 <br/>
 <h2>
  UniProtKB News
 </h2>
 <h3>
  Format change in the dbxref.txt document file
 </h3>
 <p>
  The
  <a href="http://www.uniprot.org/docs/dbxref">
   dbxref.txt
  </a>
  file lists the names and
    abbreviations and URLs of all databases cross-referenced in the UniProt Knowledgebase.
    We have added a new mandatory field, "
  <strong>
   Cat
  </strong>
  ". This field contains the database
     category,
    and will allow us to display cross-references in our entry view in a more user-friendly
    and explicit manner.
 </p>
 <p>
  Currently used categories are:
 </p>
 <ul>
  <li>
   2D gel databases
  </li>
  <li>
   3D structure databases
  </li>
  <li>
   Enzyme and pathway databases
  </li>
  <li>
   Family and domain databases
  </li>
  <li>
   Gene expression databases
  </li>
  <li>
   Ontologies
  </li>
  <li>
   Organism-specific gene databases
  </li>
  <li>
   Other
  </li>
  <li>
   PTM databases
  </li>
  <li>
   Polymorphism databases
  </li>
  <li>
   Protein family/group databases
  </li>
  <li>
   Protein-protein interaction databases
  </li>
  <li>
   Sequence databases
  </li>
 </ul>
 <p>
  Example:
 </p>
 <pre>
Abbrev: EcoGene
Name  : Escherichia coli strain K12 genome database
LinkTp: Explicit
Server: http://www.ecogene.org/
Db_URL: www.ecogene.org/geneInfo.php?eg_id=%s
<strong>Cat   : Organism-specific gene databases</strong>
</pre>
 <h3>
  Changes concerning keywords
 </h3>
 <p>
  Modified keywords:
 </p>
 <ul>
  <li>
   Alzheimer's disease -&gt;
   <a href="http://www.uniprot.org/keywords/KW-0026">
    Alzheimer disease
   </a>
  </li>
  <li>
   Cockayne's syndrome -&gt;
   <a href="http://www.uniprot.org/keywords/KW-0172">
    Cockayne syndrome
   </a>
  </li>
  <li>
   Down's syndrome -&gt;
   <a href="http://www.uniprot.org/keywords/KW-0241">
    Down syndrome
   </a>
  </li>
  <li>
   Parkinson's disease -&gt;
   <a href="http://www.uniprot.org/keywords/KW-0907">
    Parkinson disease
   </a>
  </li>
  <li>
   Vitamin B12 -&gt;
   <a href="http://www.uniprot.org/keywords/KW-0846">
    Cobalamin
   </a>
  </li>
 </ul>
 <p>
  Deleted keywords:
 </p>
 <ul>
  <li>
   Chorion
  </li>
  <li>
   Multigene family
  </li>
  <li>
   Myelin
  </li>
  <li>
   Seminal vesicle
  </li>
  <li>
   Sperm
  </li>
  <li>
   T-cell
  </li>
  <li>
   Testis
  </li>
 </ul>
</div>