<div>
 <h2>
  Headlines
 </h2>
 <h3>
  Major update of a re-emerging pathogen: Dengue virus.
 </h3>
 <p>
  Dengue is a mosquito-borne virus found in
      tropical and sub-tropical regions around the world, predominantly in urban and
      semi-urban areas in south-east Asia, Africa, and South America.
 </p>
 <p>
  During the 1970s, the disease had become sporadic due
      to an active vector control program. Moreover since the 1980s, both the virus and his
      vector have re-emerged and spread even wider than before: the disease is now
      found in more than 100 countries. The reasons for this re-emergence probably include the
      growing extension of urban areas and the suspension of the vector control program.
 </p>
 <p>
  The virus is transmitted to humans through the bite of an infected
  <em>
   <a href="http://www.uniprot.org/taxonomy/7159">
    Aedes aegypti
   </a>
  </em>
  mosquito
      and subsequently replicates in skin dendritic cells before infecting lymph
      nodes and blood cells. The symptoms are fever and pain, that can be sustained
      for up to 7 days. In rare cases, human infection leads to dengue haemorrhagic
      fever (DHF), a potentially fatal complication. Today, DHF affects most Asian
      countries and has become a leading cause of hospitalisation and death among
      children.
 </p>
 <p>
  Some 2500 million people -- two fifths of
      the world's population -- are now at risk from dengue. WHO currently estimates
      there may be 50 million cases of dengue infection worldwide every year. The
      mild autumn of 2006 has favoured the spread of the vector and has been responsible for a
  <a href="http://en.wikipedia.org/wiki/2006_Dengue_outbreak_in_India">
   major outbreak of dengue in India
  </a>
  ,
      with many reported cases in New Delhi.
 </p>
 <p>
  The growing number of
  <a href="http://www.uniprot.org/uniprot/?query=taxonomy:%22dengue+virus+group%22">
   dengue virus sequences
  </a>
  ( currently over 3400 in UniProtKB/TrEMBL ) and the absence of taxonomic
      nomenclature complicates the identification of medical samples.
 </p>
 <p>
  In the current UniProtKB/Swiss-Prot release, we have adopted a systematic nomenclature for
  <a href="http://www.uniprot.org/taxonomy/?query=dengue+AND+virus+AND+reviewed%3ayes">
   28 representative
      dengue strains
  </a>
  , where we indicate the country and the year of isolation besides the
      strain name.
 </p>
 <p>
  Example:
  <strong>
   Dengue virus type 2 (strain TH-36)
  </strong>
  <br/>
  becomes:
  <strong>
   Dengue virus type 2 (strain Thailand/TH-36/1958)
  </strong>
 </p>
 <p>
  The virus (+)RNA genome codes for a
      single polyprotein, cleaved in more than 12 products. 32 representative dengue
      virus polyproteins have been annotated and are available in UniProtKB/Swiss-Prot.
      (ex:
  <a href="http://www.uniprot.org/uniprot/P33478">
   P33478
  </a>
  )
 </p>
 <br/>
 <h2>
  UniProtKB News
 </h2>
 <h3>
  Protein Spotlight document
 </h3>
 <p>
  <a href="http://www.expasy.org/spotlight/">
   Protein Spotlight
  </a>
  (ISSN 1424-4721 -
  <a href="http://www.expasy.org/spotlight/">
   http://www.expasy.org/spotlight/
  </a>
  )
     is a monthly review written by the Swiss-Prot team of the SIB Swiss Institute
     of Bioinformatics. Spotlight articles describe a specific protein or family of proteins on an informal tone.
 </p>
 <p>
  The
  <a href="http://www.uniprot.org/docs/protspot">
   protsprot document
  </a>
  lists for each Protein Spotlight article
     the corresponding UniProtKB/Swiss-Prot entries that it cites.
 </p>
 <h3>
  Cross-references to DIP
 </h3>
 <p>
  Cross-references have been added to the
  <a href="http://dip.doe-mbi.ucla.edu/">
   Database of interacting proteins
  </a>
  . The DIP database catalogs experimentally determined interactions between
     proteins. It combines information from a variety of sources to create a single, consistent set of protein-protein
     interactions. The data stored within the DIP database were curated, both, manually by expert curators and also
     automatically using computational approaches that utilize the knowledge about the protein-protein interaction
     networks extracted from the most reliable, core subset of the DIP data.
 </p>
 <p>
  The DIP is available at
  <a href="http://dip.doe-mbi.ucla.edu/">
   http://dip.doe-mbi.ucla.edu/
  </a>
  .
 </p>
 <p>
  The format of the explicit links in the flat file is:
 </p>
 <table>
  <tr>
   <th>
    Resource abbreviation
   </th>
   <td>
    DIP
   </td>
  </tr>
  <tr>
   <th>
    Resource identifier
   </th>
   <td>
    DIP accession number.
   </td>
  </tr>
  <tr>
   <th>
    Examples
   </th>
   <td>
    <pre><a href="http://www.uniprot.org/uniprot/Q9W1K5#section_x-ref">Q9W1K5</a>:
DR   DIP; DIP:19601N; -.

<a href="http://www.uniprot.org/uniprot/P41597#section_x-ref">P41597</a>:
DR   DIP; DIP:5833N; -.
DR   DIP; DIP:5839N; -.</pre>
   </td>
  </tr>
 </table>
 <h3>
  Cross-references to Reactome
 </h3>
 <p>
  The resource identifier of the cross-references to the
  <a href="http://www.reactome.org/">
   Reactome
  </a>
  database has been
    modified. The resource identifier was a Reactome unique identifier
    for a protein, which was identical to the Swiss-Prot primary AC
    number of that protein. Now it is a stable Reactome identifier. In
    addition, the pathway name is given as an optional information
    field.
 </p>
 <table>
  <tr>
   <th>
    Resource identifier
   </th>
   <td>
    Reactome identifier.
   </td>
  </tr>
  <tr>
   <th>
    Optional information 1
   </th>
   <td>
    Pathway name.
   </td>
  </tr>
  <tr>
   <th>
    Examples
   </th>
   <td>
    <pre><a href="http://www.uniprot.org/uniprot/P61978#section_x-ref">P61978</a>:
DR   Reactome; REACT_1675.1; mRNA Processing.
DR   Reactome; REACT_71.1; Gene Expression.

<a href="http://www.uniprot.org/uniprot/P62191#section_x-ref">P62191</a>:
DR   Reactome; REACT_152.2; Cell Cycle, Mitotic.
DR   Reactome; REACT_1538.1; Cell Cycle Checkpoints.
DR   Reactome; REACT_383.2; DNA Replication.
DR   Reactome; REACT_6185.3; HIV Infection.
DR   Reactome; REACT_6850.1; Cdc20:Phospho-APC/C mediated degradation of Cyclin A.</pre>
   </td>
  </tr>
 </table>
 <h3>
  Cross-references to GermOnline
 </h3>
 <p>
  The resource identifier of the cross-references to the
  <a href="http://germonline.unibas.ch/">
   GermOnline
  </a>
  database has
    been modified. The resource identifier was a GermOnline identifier
    for a gene. Now it is a gene identifier from any source,
    e.g. Ensembl or model organism database. In addition, the organism
    name is is given as an optional information field.
 </p>
 <table>
  <tr>
   <th>
    Resource identifier
   </th>
   <td>
    Gene identifier from any source, e.g. Ensembl or model organism database.
   </td>
  </tr>
  <tr>
   <th>
    Optional information 1
   </th>
   <td>
    Organism name.
   </td>
  </tr>
  <tr>
   <th>
    Examples
   </th>
   <td>
    <pre><a href="http://www.uniprot.org/uniprot/P02766#section_x-ref">P02766</a>:
DR   GermOnline; ENSG00000118271; Homo sapiens.

<a href="http://www.uniprot.org/uniprot/P32559#section_x-ref">P32559</a>:
DR   GermOnline; YMR023C; Saccharomyces cerevisiae.</pre>
   </td>
  </tr>
 </table>
 <h3>
  Changes concerning keywords
 </h3>
 <p>
  New keyword:
 </p>
 <ul>
  <li>
   <a href="http://www.uniprot.org/keywords/KW-0952">
    Extinct organism protein
   </a>
  </li>
  <li>
   <a href="http://www.uniprot.org/keywords/KW-0951">
    Familial hemophagocytic lymphohistiocytosis
   </a>
  </li>
 </ul>
 <p>
  Modified keywords:
 </p>
 <ul>
  <li>
   Anhidrotic ectodermal dysplasia -&gt;
   <a href="http://www.uniprot.org/keywords/KW-0038">
    Ectodermal dysplasia
   </a>
  </li>
 </ul>
</div>