<div>
 <h2>
  Headlines
 </h2>
 <h3>
  Acanthamoeba polyphaga mimivirus, a "giant" virus in UniProtKB/Swiss-Prot
 </h3>
 <p>
  <a href="http://www.uniprot.org/taxonomy/212035">
   Mimivirus
  </a>
  (for
      mimicking microbe) is a new viral genus containing a single identified species,
      Acanthamoeba polyphaga mimivirus (APMV), discovered by Didier Raoult's lab in
      1992 within the amoeba Acanthamoeba polyphaga while working on Legionellosis.
      The virion has a non-enveloped, icosahedral capsid with a diameter of 400 nm
      and protein filaments projecting from its surface. The capsid contains the
      internal core surrounded by an internal lipid layer. Its linear,
  <a href="http://view.ncbi.nlm.nih.gov/pubmed/15486256">
   double-
       stranded DNA genome
  </a>
  is roughly 1.2 million bp in length, the largest viral
      genome known so far. Its replication cycle, genome and capsid structure place
      it into the nucleocytoplasmic large DNA viruses (
  <a href="http://view.ncbi.nlm.nih.gov/pubmed/16494962">
   NCLDVs
  </a>
  ), which include amongst others the poxviruses and
      iridoviruses.
 </p>
 <p>
  This virus is amazing in many ways. It is the largest virus ever isolated,
      with a genome size and complexity comparable to that of a small bacterium. A
      thorough bioinformatics analysis carried out by the group of Jean-Michel
      Claverie uncovered 909 potential protein-coding genes. Some of these proteins
      belong to families that are shared with all or some NCLDVs, many have
      eukaryotic counterparts and there are quite a number of
  <a href="http://view.ncbi.nlm.nih.gov/pubmed/16971431">
   ORFans
  </a>
  (no sequence similarity to proteins from other
      genomes). It was a surprise to find an appreciable number of genes coding for
      proteins involved in metabolism, DNA repair pathways and, most surprising,
      genes encoding a partially functional protein translation apparatus. Mimivirus
      does indeed encode
  <a href="http://view.ncbi.nlm.nih.gov/pubmed/17855524">
   four
       aminoacyl-tRNA synthetases
  </a>
  (
  <a href="http://www.uniprot.org/uniprot/Q5UQ59">
   ArgRS
  </a>
  ,
  <a href="http://www.uniprot.org/uniprot/Q5UP36">
   CysRS
  </a>
  ,
  <a href="http://www.uniprot.org/uniprot/Q5UR82">
   MetRS
  </a>
  ,
  <a href="http://www.uniprot.org/uniprot/Q5UPJ7">
   TyrRS
  </a>
  ), as well as various translation initiation,
      elongation and termination factors. It is very intriguing to find, in a virus,
      genes corresponding to central components of the protein translation machinery,
      a biochemical process widely thought to be an exclusive signature of cellular
      organisms.
 </p>
 <p>
  The discovery of this amazing virus has lead to the concept
      of
  <a href="http://view.ncbi.nlm.nih.gov/pubmed/16469402">
   "giant"
       virus
  </a>
  and implies that there is an overlap in terms of particle dimension,
      genome size, and genetic complexity between the viral and cellular organism worlds.
 </p>
 <p>
  A special effort has been made in UniProtKB/Swiss-Prot database to provide
      the complete, fully annotated mimivirus proteome. We have also integrated all
      proteomics and structural information that has been made available by the
      groups of Jean-Michel Claverie and Chantal Abergel.
 </p>
 <p>
  To get all UniProtKB mimivirus entries, click
  <a href="http://www.uniprot.org/uniprot/?query=taxonomy:212035+AND+reviewed:yes">
   here
  </a>
  .
 </p>
 <h2>
  UniProtKB News
 </h2>
 <h3>
  Format change in the ptmlist.txt document file
 </h3>
 <p>
  The ptmlist.txt document, which is available by
  <a href="ftp://ftp.uniprot.org/pub/databases/uniprot/knowledgebase/docs/ptmlist.txt">
   ftp
  </a>
  and on the
  <a href="http://www.uniprot.org/docs/ptmlist">
   Web site
  </a>
  , describes the post-translational
     modifications (PTMs) that are annotated in UniProtKB/Swiss-Prot entries in the sequence annotation section
     (Features) (FT lines in the flat file) in the subsections "Cross-link" (CROSSLNK key in the flat file),
     "Lipidation" (LIPID key in the flat file) and "Modified residue" (MOD_RES key in the flat file).
     The document was in a format that was suitable for computer applications
     (e.g. ExPASy's proteomics tools),
     but which was not very human readable. The new file format should improve this.
 </p>
 <p>
  Previous format:
 </p>
 <pre>
N,N-dimethylproline  MOD_RES P  BB Nter C2H4  28.031300  28.06  in  e:6446,7586,33682  Methylation  FT=MOD_RES%20dimethylproline&amp;wild=1  AA0066  MOD:00075
</pre>
 <p>
  New format:
 </p>
 <pre>
ID   N,N-dimethylproline
AC   PTM-0179
FT   MOD_RES
TG   Proline.
PA   Amino acid backbone.
PP   N-terminal.
CF   C2 H4
MM   28.031300
MA   28.06
LC   Intracellular localisation.
TR   Eukaryota; taxId:6446 (Sipunculus nudus), taxId:7586 (Echinodermata), taxId:33682 (Euglenozoa).
KW   Methylation.
DR   RESID:AA0066.
DR   MOD:00075.
//
</pre>
 <p>
  With the following definitions of the line types:
 </p>
 <pre>
  ---------  ---------------------------     ----------------------
  Line code  Content                         Occurrence in an entry
  ---------  ---------------------------     ----------------------
  ID         Identifier (FT description)     Once; starts a PTM entry.
  AC         Accession (PTM-xxxx)            Once.
  FT         Feature key                     Once.
  TG         Target                          Once; two targets separated
                                             by a dash in case of intrachain
                                             crosslinks.
  PA         Position of the modified        Optional, once.
             amino acid
  PP         Position of the modification    Optional, once.
             in the polypeptide
  CF         Correction formula              Optional, once.
  MM         Monoisotopic mass difference    Optional, once.
  MA         Average mass difference         Optional, once.
  LC         Cellular location               Optional, once; alternatives
                                             can be proposed.
  TR         Taxonomic range                 Optional, once or more.
  KW         Keyword                         Optional, once or more.
  DR         Cross-reference to PTM          Optional, once or more.
             databases
  //         Terminator                      Once; ends an entry.
</pre>
 <h3>
  Changes concerning cross-references to PDB
 </h3>
 <p>
  We added an additional field to the cross-reference (DR line in the flat file) to the PDB database to
     show the resolution of structures that were determined by X-ray crystallography or
     electron microscopy.
 </p>
 <p>
  For the chain names we use now the
  <a href="http://remediation.wwpdb.org/">
   remediated data
  </a>
  from
  <a href="http://www.wwpdb.org/">
   wwPDB
  </a>
  , therefore the chain names have changed for some entries.
 </p>
 <p>
  Previous format:
 </p>
 <pre>
DR   PDB; ENTRY_NAME; METHOD; CHAIN.
</pre>
 <p>
  New format:
 </p>
 <pre>
DR   PDB; ENTRY_NAME; METHOD; <strong>RESOLUTION</strong>; CHAIN.
</pre>
 <p>
  Examples:
 </p>
 <a href="http://www.uniprot.org/uniprot/Q20728">
  Q20728
 </a>
 :
 <pre>
DR   PDB; 1LPL; X-ray; 1.77 A; A=135-229.
</pre>
 <a href="http://www.uniprot.org/uniprot/Q5HEB7">
  Q5HEB7
 </a>
 :
 <pre>
DR   PDB; 2I8C; X-ray; 2.46 A; A/B=1-356.
</pre>
 <p>
  A dash indicates that we found no information about the resolution or that the field
   is not applicable (for NMR structures and theoretical models).
 </p>
 <p>
  Examples:
 </p>
 <a href="http://www.uniprot.org/uniprot/P02768">
  P02768
 </a>
 :
 <pre>
DR   PDB; 2ESG; X-ray; -; C=25-609.
</pre>
 <a href="http://www.uniprot.org/uniprot/P12872">
  P12872
 </a>
 :
 <pre>
DR   PDB; 1LBJ; NMR; -; A=26-47.
</pre>
 <a href="http://www.uniprot.org/uniprot/P0AC41">
  P0AC41
 </a>
 :
 <pre>
DR   PDB; 2AD0; Model; -; A=1-588.
</pre>
 <h3>
  Cross-references to CleanEx
 </h3>
 <p>
  Cross-references have been added to the
  <a href="http://www.cleanex.isb-sib.ch/">
   CleanEx database of gene expression profiles
  </a>
  . CleanEx is a database which provides
     access to public gene expression data via unique approved gene symbols and which
     represents heterogeneous expression data produced by different technologies in a way
     that facilitates joint analysis and cross-dataset comparison.
 </p>
 <p>
  The CleanEx database is available at
  <a href="http://www.cleanex.isb-sib.ch/">
   http://www.cleanex.isb-sib.ch/
  </a>
  .
 </p>
 <p>
  The format of the explicit links in the flat file is:
 </p>
 <table>
  <tr>
   <th>
    Resource abbreviation
   </th>
   <td>
    CleanEx
   </td>
  </tr>
  <tr>
   <th>
    Resource identifier
   </th>
   <td>
    Combination of a species code and a gene identifier.
   </td>
  </tr>
  <tr>
   <th>
    Examples
   </th>
   <td>
    <pre><a href="http://www.uniprot.org/uniprot/O08788#section_x-ref">O08788</a>:
DR   CleanEx; MM_DCTN1; -.

<a href="http://www.uniprot.org/uniprot/P78358#section_x-ref">P78358</a>:
DR   CleanEx; HS_CTAG1A; -.
DR   CleanEx; HS_CTAG1B; -.</pre>
   </td>
  </tr>
 </table>
 <h3>
  Changes concerning keywords (KW line)
 </h3>
 <p>
  Modified keyword:
 </p>
 <ul>
  <li>
   Phosphorylation -&gt;
   <a href="http://www.uniprot.org/keywords/KW-0597">
    Phosphoprotein
   </a>
  </li>
 </ul>
</div>