<div>
 <h2>
  Headlines
 </h2>
 <h3>
  Over 100 cross-references in UniProtKB/Swiss-Prot
 </h3>
 <p>
  UniProtKB/Swiss-Prot was the first biomolecular database to include cross-
      references in its entries. As of this release, we provide our users with 101
      explicit links (stored in the various distributed file formats, flat text, XML
      and RDF/XML) and 23 implicit links (available only
      from web servers, such as
  <a href="http://www.uniprot.org/">
   UniProt
  </a>
  and
  <a href="http://www.expasy.org/">
   ExPASy
  </a>
  ). Most cross-references
      can be found in the 'Cross-references' section of the entry (see for example
  <a href="http://www.uniprot.org/uniprot/Q9FK25#section_x-ref">
   Q9FK25
  </a>
  ), some
      are in the 'Sequence annotation' section (the Feature table in the flat file)
      (see for example cross-references to dbSNP in
  <a href="http://www.uniprot.org/uniprot/Q969T7#section_features">
   Q969T7
  </a>
  ).
      The
  <a href="http://www.uniprot.org/docs/dbxref">
   dbxref.txt
  </a>
  document provides a list of the databases cross-referenced in
      UniProtKB/Swiss-Prot. This document is available on the UniProt website and by
  <a href="ftp://ftp.uniprot.org/pub/databases/uniprot/knowledgebase/docs/dbxref.txt">
   ftp
  </a>
  .
 </p>
 <p>
  Additional links pepper almost every section of a UniProtKB/Swiss-Prot entry. They include
      cross-references to PubMed which are located in the 'References' section
      (see for example
  <a href="http://www.uniprot.org/uniprot/P0A790#section_ref">
   P0A790
  </a>
  ) and
      cross-references to the ENZYME database available through the EC numbers in the 'Names
      and origin' section (see for example
  <a href="http://www.uniprot.org/uniprot/Q00955#section_name">
   Q00955
  </a>
  ).
      Moreover the 'Web resources' section is dedicated to databases or web pages that are specific
      for a single protein (see for example
  <a href="http://www.uniprot.org/uniprot/P04637#section_web">
   P04637
  </a>
  ).
      Note that the dbxref.txt document does not list these 'special' links.
 </p>
 <p>
  Historically, a
  <a href="http://en.wikipedia.org/wiki/Hundred_%28division%29">
   'hundred'
  </a>
  was a geographic division referring to the amount
      of land sufficient to sustain one hundred families. With over 120 cross-references, we hope
      to sustain many more research groups in quest of protein information.
 </p>
 <h2>
  UniProtKB News
 </h2>
 <h3>
  Cross-references to HOGENOM
 </h3>
 <p>
  Cross-references have been added to
  <a href="http://pbil.univ-lyon1.fr/databases/hogenom.php">
   HOGENOM
  </a>
  , a database of homologous genes from fully sequenced organisms. HOGENOM allows to
     select sets of homologous genes among species, and to visualize multiple alignments
     and phylogenetic trees. It is as well possible to search for orthologous genes in a wide
     range of taxons. Thus HOGENOM is particularly useful for comparative sequence analysis,
     phylogeny and molecular evolution studies. More generaly, HOGENOM gives an overall view
     of what is known about a peculiar gene family.
 </p>
 <p>
  The HOGENOM database is available at
  <a href="http://pbil.univ-lyon1.fr/databases/hogenom.php">
   http://pbil.univ-lyon1.fr/databases/hogenom.php
  </a>
  .
 </p>
 <p>
  The format of the explicit links in the flat file is:
 </p>
 <table>
  <tr>
   <th>
    Resource abbreviation
   </th>
   <td>
    HOGENOM
   </td>
  </tr>
  <tr>
   <th>
    Resource identifier
   </th>
   <td>
    UniProtKB accession number.
   </td>
  </tr>
  <tr>
   <th>
    Examples
   </th>
   <td>
    <pre><a href="http://www.uniprot.org/uniprot/P0A9I1#section_x-ref">P0A9I1</a>:
DR   HOGENOM; P0A9I1; -.

<a href="http://www.uniprot.org/uniprot/P49642#section_x-ref">P49642</a>:
DR   HOGENOM; P49642; -.</pre>
   </td>
  </tr>
 </table>
 <h3>
  Cross-references to HOVERGEN
 </h3>
 <p>
  Cross-references have been added to
  <a href="http://pbil.univ-lyon1.fr/databases/hovergen.php">
   HOVERGEN
  </a>
  , a database of homologous vertebrate genes. HOVERGEN allows one to select sets
     of homologous genes among vertebrate species, and to visualize multiple alignments
     and phylogenetic trees. Thus HOVERGEN is particularly useful for comparative sequence
     analysis, phylogeny and molecular evolution studies. More generaly, HOVERGEN gives an
     overall view of what is known about a peculiar gene family.
 </p>
 <p>
  The HOVERGEN database is available at
  <a href="http://pbil.univ-lyon1.fr/databases/hovergen.php">
   http://pbil.univ-lyon1.fr/databases/hovergen.php
  </a>
  .
 </p>
 <p>
  The format of the explicit links in the flat file is:
 </p>
 <table>
  <tr>
   <th>
    Resource abbreviation
   </th>
   <td>
    HOVERGEN
   </td>
  </tr>
  <tr>
   <th>
    Resource identifier
   </th>
   <td>
    UniProtKB accession number.
   </td>
  </tr>
  <tr>
   <th>
    Examples
   </th>
   <td>
    <pre><a href="http://www.uniprot.org/uniprot/P31946#section_x-ref">P31946</a>:
DR   HOVERGEN; P31946; -.

<a href="http://www.uniprot.org/uniprot/Q91ZB4#section_x-ref">Q91ZB4</a>:
DR   HOVERGEN; Q91ZB4; -.</pre>
   </td>
  </tr>
 </table>
 <h3>
  Changes concerning keywords
 </h3>
 <p>
  New keywords:
 </p>
 <ul>
  <li>
   <a href="http://www.uniprot.org/keywords/KW-0992">
    Brugada syndrome
   </a>
  </li>
 </ul>
 <ul>
  <li>
   <a href="http://www.uniprot.org/keywords/KW-0991">
    Mental retardation
   </a>
  </li>
 </ul>
</div>