<div>
 <h2>
  Headlines
 </h2>
 <h3>
  Additional bibliography information in UniProtKB
 </h3>
 <p>
  As a comprehensive and high-quality resource of protein sequence and functional
      information, UniProtKB strives to provide comprehensive literature citations
      associated with protein sequences and their characterization. Currently about 2 thirds of the UniProtKB
      PubMed citations are found in UniProtKB/Swiss-Prot, as a result of active
      integration in the course of manual curation.
 </p>
 <p>
  In order to keep up with the explosive growth of literature and to give our
      users access to additional publications, we
      decided to integrate additional sources of literature from other annotated databases into
      UniProtKB. For this purpose we selected 5 external databases:
      Entrez Gene (
  <a href="http://www.ncbi.nlm.nih.gov/projects/GeneRIF/">
   GeneRIFs
  </a>
  ),
  <a href="http://www.yeastgenome.org/">
   SGD
  </a>
  ,
  <a href="http://www.informatics.jax.org/">
   MGI
  </a>
  ,
  <a href="http://geneticassociationdb.nih.gov/">
   GAD
  </a>
  and
  <a href="http://www.rcsb.org/pdb/home/home.do">
   PDB
  </a>
  ,
      and extracted citations that were mapped to UniProtKB entries. This additional bibliography is available
      from the 'References' section by clicking on
  <a href="http://www.uniprot.org/uniprot/P03875#section_ref">
   'Additional
       computationally mapped references'
  </a>
  .
 </p>
 <p>
  This procedure allowed the addition of about 283'000 PubMed citations in close
      to 110'000 UniProtKB entries. 85% of these references did not exist previously in
      UniProtKB.
 </p>
 <p>
  In the future, we plan to apply this pipeline to more databases that could be used as sources of
      protein bibliography, including model organism databases, such as
  <a href="http://flybase.org/">
   FlyBase
  </a>
  and
  <a href="http://www.wormbase.org/">
   WormBase
  </a>
  . We believe this additional protein bibliography
      information will allow our users to better explore the existing knowledge of their proteins
      of interest.
 </p>
</div>