<div>
 <h2>
  Headlines
 </h2>
 <h3>
  Through the Looking-Glass
 </h3>
 <p>
  All amino acids but glycine can exist in either of two
  <a href="http://en.wikipedia.org/wiki/Amino_acid#Isomerism">
   optical isomers
  </a>
  , called L-or D-amino
acids, which are mirror images of each other. However, we have been taught for decades that
proteins that occur in nature are made out of L-forms. There are some well-known exceptions, of
course, but restricted to prokaryotes. Indeed, D-forms are abundant components of the peptidoglycan
cell walls of bacteria, and are also observed in bacterial natural antibiotics, such as
actinomycin D, bacitracin or tetracycline. These latter are quite unusual peptides that are
synthesized by multienzyme complexes in a stepwise fashion without the participation of mRNA.
It has also been observed that the mammalian brain contains high levels of free
  <a href="http://www.ncbi.nlm.nih.gov/pubmed/18564180">
   D-serine
  </a>
  which appears
to be a physiological coagonist of N-methyl D-aspartate receptors (NMDARs) and, as such, may act as a
neurotransmitter in the brain, but this activity is carried out by the amino acid itself and does not occur within the context of a polypeptide.
The isolation, in the 1980s, of naturally occurring animal peptides containing D-amino acids challenged the dogma, leading to the discovery of a new post-translational modification (PTM): L- to D-isomerization.
 </p>
 <p>
  In 1981,
  <a href="http://www.ncbi.nlm.nih.gov/pubmed/7287299">
   Montecucchi et al.
  </a>
  ,
looking for enkephalin-related peptides in various amphibia, isolated dermorphin from the skin of
  <a href="http://www.uniprot.org/taxonomy/8395">
   <em>
    Phyllomedusa sauvagei
   </em>
  </a>
  .
  <a href="http://www.uniprot.org/uniprot/?query=name:dermorphin+AND+taxonomy:8395">
   Dermorphin
  </a>
  is produced
by 2 different precursors: cleavage of
  <a href="http://www.uniprot.org/uniprot/P05422">
   Dermorphin-1
  </a>
  gives rise to 4 mature dermorphins
and that of
  <a href="http://www.uniprot.org/uniprot/P05421">
   Dermorphin-2
  </a>
  to 5 mature peptides, all of which have
the identical sequence: YAFGYPS. This heptapeptide binds with high affinity
and selectivity to mu-type opioid receptors and appears to be
  <a href="http://www.ncbi.nlm.nih.gov/pubmed/7195758">
   a thousand times more potent than morphine
  </a>
  in inducing deep long-lasting analgesia when injected into mice or rats. Interestingly, the second amino acid
of dermorphin is D-alanine. A synthetic isomer, containing L-alanine at that position, is virtually
devoid of biological activity.
 </p>
 <p>
  This discovery was followed by many others.
  <a href="http://www.uniprot.org/uniprot/?query=taxonomy:%22Metazoa+[33208]%22+AND+keyword:%22D-amino+acid%22+AND+name:deltorphin">
   Deltorphins
  </a>
  ,
another class of frog opioid peptides, also characterized by a D-amino acid at position 2, were isolated.
Another amphibian,
  <a href="http://www.uniprot.org/taxonomy/8348">
   <em>
    Bombina variegata
   </em>
  </a>
  , was shown to express
antimicrobial D-amino acid-containing peptides, called
  <a href="http://www.uniprot.org/uniprot/?query=keyword:%22D-amino+acid%22+AND+taxonomy:8348">
   bombesins
  </a>
  , on its skin.
  <a href="http://www.uniprot.org/uniprot/?query=keyword:%22D-amino+acid%22+AND+taxonomy:6656">
   Arthropoda
  </a>
  ,
such as spider, lobsters and crayfish, and
  <a href="http://www.uniprot.org/uniprot/?query=keyword:%22D-amino+acid%22+taxonomy:6447">
   Mollusca
  </a>
  entered the game.
  <a href="http://www.uniprot.org/uniprot/?query=keyword:%22D-amino+acid%22+AND+taxonomy:6490">
   Cone snail peptide toxins
  </a>
  have been extensively studied in this context and they currently represent 60% of all animal
  <a href="http://www.uniprot.org/uniprot/?query=taxonomy:%22Metazoa+[33208]%22+AND+keyword:%22D-amino+acid%22&amp;by=taxonomy#6656,6447,32523">
   D-amino acid-containing proteins
  </a>
  annotated in UniProtKB/Swiss-Prot.
A single mammal appears on the list:
  <a href="http://www.uniprot.org/uniprot/?query=keyword:%22D-amino+acid%22+taxonomy:9258">
   platypus
  </a>
  with 2 peptides,
  <a href="http://www.uniprot.org/uniprot/P84715">
   C-type natriuretic peptide 39
  </a>
  and
  <a href="http://www.uniprot.org/uniprot/P82140">
   Defensin-like peptide 2/4
  </a>
  , expressed in its venom gland.
 </p>
 <p>
  Animal D-amino acid-containing proteins are synthesized on ribosomes following a classical mRNA template; unusual codons have not been observed.
In addition, some of them have been isolated from their
  <a href="http://www.ncbi.nlm.nih.gov/pubmed/7973665,8034574">
   biological source
  </a>
  with both L- and D-amino acid at the appropriate position. These observations
suggested that L- to D-amino acid isomerization is a bona fide PTM. An enzyme catalyzing the conversion of
an
  <a href="http://www.uniprot.org/uniprot/P37045">
   Omega-agatoxin-Aa4b
  </a>
  serine (at position 46 of the mature peptide,
  <a href="http://www.uniprot.org/uniprot/P37045#section_features">
   81 in the precursor
  </a>
  ) from L- to D-form
  <a href="http://www.ncbi.nlm.nih.gov/pubmed/7622482">
   has been isolated
  </a>
  from the funnel-web spider
  <em>
   Agelenopsis aperta
  </em>
  and its
  <a href="http://www.uniprot.org/uniprot/?query=q9txd8+OR+q9twh3">
   partial sequence
  </a>
  is available in UniProtKB/TrEMBL.
A similar mammalian activity
  <a href="http://www.ncbi.nlm.nih.gov/pubmed/16480722">
   has been characterized
  </a>
  from platypus venom.
 </p>
 <p>
  L- to D-amino acid isomerization presents significant advantages. The modified peptides become more resistant to protease degradation and hence much more stable.
In addition, X-ray crystallography studies have shown that the isomerization creates new structures, such as peculiar beta-turns. The creation of these new
structural elements seems crucial for interaction with specific partners, opiate receptors for instance, and may act as a switch that turns on
protein activity.
 </p>
 <p>
  L- to D-amino acid isomerization could be more frequent than initially thought. It cannot be predicted by software tools and is not detectable by
any of the standard techniques used in proteomics. It was only discovered when a synthetic peptide with the same sequence of L-amino acids
appeared to be biologically inactive. We could be facing a novel strategy of multicellular organisms to circumvent stereochemical
limitations imposed by the genetic code in an effort to increase molecular diversity.
 </p>
 <p>
  In UniProtKB, all
  <a href="http://www.uniprot.org/uniprot/?query=keyword:%22D-amino+acid%22">
   D-amino acid-containing proteins
  </a>
  can be retrieved using the keyword
  <a href="http://www.uniprot.org/keywords/KW-0208">
   'D-amino acid'
  </a>
  . To restrict the search to
  <a href="http://www.uniprot.org/uniprot/?query=taxonomy:33208+AND+keyword:%22D-amino+acid%22">
   animal proteins
  </a>
  , add
  <a href="http://www.uniprot.org/taxonomy/33208">
   'Metazoa'
  </a>
  to the taxonomy field.
 </p>
 <h2>
  UniProtKB News
 </h2>
 <h3>
  Cross-references to ArachnoServer
 </h3>
 <p>
  Cross-references have been added to
  <a href="http://www.arachnoserver.org/">
   ArachnoServer
  </a>
  ,
   a spider toxin database. ArachnoServer is a manually curated database containing information on
   the sequence, three-dimensional structure, and biological activity of protein toxins derived from
   spider venom.
 </p>
 <p>
  ArachnoServer is available at
  <a href="http://www.arachnoserver.org/">
   http://www.arachnoserver.org/
  </a>
  .
 </p>
 <p>
  The format of the explicit links in the flat file is:
 </p>
 <table>
  <tr>
   <th>
    Resource abbreviation
   </th>
   <td>
    ArachnoServer
   </td>
  </tr>
  <tr>
   <th>
    Resource identifier
   </th>
   <td>
    ArachnoServer unique identifier.
   </td>
  </tr>
  <tr>
   <th>
    Optional information 1
   </th>
   <td>
    Toxin name.
   </td>
  </tr>
  <tr>
   <th>
    Examples
   </th>
   <td>
    <pre><a href="http://www.uniprot.org/uniprot/P61232#section_x-ref">P61232</a>:
DR   ArachnoServer; AS000384; beta-hexatoxin-Mg1a.
DR   ArachnoServer; AS000417; beta-hexatoxin-Mr1a.

<a href="http://www.uniprot.org/uniprot/Q7M485#section_x-ref">Q7M485</a>:
DR   ArachnoServer; AS000160; Sphingomyelinase D (LrSicTox1) (N-terminal fragment).</pre>
   </td>
  </tr>
 </table>
 <p>
  <a href="http://www.uniprot.org/uniprot/?query=database%3AArachnoServer&amp;#38;sort=score">
   Show all the entries having a cross-reference to ArachnoServer.
  </a>
 </p>
 <h3>
  Cross-references to InParanoid
 </h3>
 <p>
  Cross-references have been added to
  <a href="http://inparanoid.sbc.su.se/">
   InParanoid
  </a>
  ,
   a database of eukaryotic ortholog groups. The InParanoid database is a
   collection of pairwise comparisons between currently 35 complete
   proteomes. The InParanoid program uses the pairwise similarity scores,
   calculated using NCBI-Blast, between two complete proteomes for constructing
   orthology groups.
 </p>
 <p>
  InParanoid is available at
  <a href="http://inparanoid.sbc.su.se/">
   http://inparanoid.sbc.su.se/
  </a>
  .
 </p>
 <p>
  The format of the explicit links in the flat file is:
 </p>
 <table>
  <tr>
   <th>
    Resource abbreviation
   </th>
   <td>
    InParanoid
   </td>
  </tr>
  <tr>
   <th>
    Resource identifier
   </th>
   <td>
    UniProtKB accession number.
   </td>
  </tr>
  <tr>
   <th>
    Example
   </th>
   <td>
    <pre><a href="http://www.uniprot.org/uniprot/P10038#section_x-ref">P10038</a>:
DR   InParanoid; P10038; -.</pre>
   </td>
  </tr>
 </table>
 <p>
  <a href="http://www.uniprot.org/uniprot/?query=database%3AInParanoid&amp;#38;sort=score">
   Show all the entries having a cross-reference to InParanoid.
  </a>
 </p>
 <h3>
  Changes concerning keywords
 </h3>
 <p>
  New keyword:
 </p>
 <ul>
  <li>
   <a href="http://www.uniprot.org/keywords/KW-1055">
    Congenital dyserythropoietic anemia
   </a>
  </li>
 </ul>
 <h3>
  Changes in
  <a href="http://www.uniprot.org/docs/subcell">
   subcellular location controlled vocabulary
  </a>
 </h3>
 <p>
  New subcellular location:
 </p>
 <ul>
  <li>
   Host multivesicular body
  </li>
 </ul>
 <h3>
  Changes concerning the
  <a href="http://www.uniprot.org/docs/ptmlist">
   controlled vocabulary for PTMs
  </a>
 </h3>
 <p>
  New terms for the feature key 'Cross-link' ('CROSSLNK' in the flat file):
 </p>
 <ul>
  <li>
   Glutamyl lysine isopeptide (Gln-Lys) (interchain with K-...)
  </li>
  <li>
   Glutamyl lysine isopeptide (Lys-Gln) (interchain with Q-...)
  </li>
 </ul>
 <p>
  Modified terms for the feature key 'Modified residue' ('MOD_RES' in the flat file):
 </p>
 <ul>
  <li>
   Glutamyl 5-glycerylphosphorylethanolamine -&gt; 5-glutamyl glycerylphosphorylethanolamine
  </li>
 </ul>
</div>