<div>
 <h2>
  Headlines
 </h2>
 <h3>
  Rotavirus: a serial killer in UniProtKB/Swiss-Prot
 </h3>
 <p>
  <a href="http://www.who.int/vaccine_research/diseases/diarrhoeal/en/index5.html">
   Rotaviruses
  </a>
  can infect humans, as well as other vertebrates. They cause severe diarrheal disease and dehydration of
      infants in both developed and developing countries. An estimated 0.6-0.8 million children
      aged 5 and under die from rotavirus-induced severe dehydrating diarrhea each year.
      Although mortality due to rotavirus infection is much higher in developing than in
      developed countries, infection frequency is remarkably similar. In temperate climates,
      rotavirus disease is seasonal, peaking in winter. Rotavirus gastroenteritis is
      transmitted by the fecal-oral route and characterized by watery stools, vomiting and
      fever. Commercially available vaccines are effective in preventing infection.
 </p>
 <p>
  The virus infects the mature enterocytes of the small intestine and induces structural
      changes in the intestinal epithelium, secretion of a viral enterotoxin by infected cells,
      impaired absorption, cellular and tight junction damage and stimulation of intestinal
      motility, leading to watery diarrhea.
 </p>
 <p>
  <a href="http://viralzone.expasy.org/all_by_species/107.html">
   Rotaviruses
  </a>
  have a segmented double-stranded RNA genome (dsRNA) protected by a three-
      layered capsid resistant to the acidic pH of the stomach. The genome is composed of 11
      segments coding for about 12 proteins. One of its features is that its dsRNA genome is
      never completely uncoated during replication. Only the outermost layer is lost following
      entry into the host cell. Replication of the viral genome thus occurs within a protective
      shell to avoid detection and degradation by the host cell.
 </p>
 <p>
  Seven different species of rotavirus have been described: A, B, C, D, E, F and G. Humans are
      primarily infected by species A, but also by species B and C. All seven species cause disease in
      other vertebrates. As of this release, sequences representative of all currently known rotavirus A,
      B, and C species have been annotated in UniProtKB/Swiss-Prot. This represents
  <a href="http://www.uniprot.org/uniprot/?query=organism:rotavirus+AND+reviewed:yes">
   480 entries
  </a>
  , from 100 distinct
      strains, 40 of which are of human origin.
 </p>
 <p>
  In addition to manually annotated sequences and functional information, we paid special
      attention to viral taxonomy. We decided to follow the recent recommendations for
  <a href="http://www.ncbi.nlm.nih.gov/pubmed/18604469,18216098?report=DocSum">
   genome-based
       classification
  </a>
  , in addition to the older antigenic classification system. This allows us to better reflect
      the frequent rearrangements (segment exchanges) that occur between strains. As a result, a detailed
      taxonomy is provided for each DNA segment/protein (see for
      instance
  <a href="http://www.uniprot.org/uniprot/Q3ZK61#section_name">
   Q3ZK61
  </a>
  ).
 </p>
 <p>
  For more detailed information on rotaviruses, see the
  <a href="http://viralzone.expasy.org/all_by_species/107.html">
   ViralZone
  </a>
  portal.
 </p>
 <h2>
  UniProtKB News
 </h2>
 <h3>
  Cross-references to PMAP-CutDB
 </h3>
 <p>
  Cross-references have been added to the
  <a href="http://www.proteolysis.org/">
   CutDB - Proteolytic event database
  </a>
  .
      PMAP-CutDB is one of the first systematic efforts to build an easily accessible collection of documented
      proteolytic events for natural proteins in vivo or in vitro. A CutDB entry is defined by a unique
      combination of these three attributes: protease, protein substrate and cleavage site.
 </p>
 <p>
  PMAP-CutDB is available at
  <a href="http://www.proteolysis.org/">
   http://www.proteolysis.org/
  </a>
  .
 </p>
 <p>
  The format of the explicit links in the flat file is:
 </p>
 <table>
  <tr>
   <th>
    Resource abbreviation
   </th>
   <td>
    PMAP-CutDB
   </td>
  </tr>
  <tr>
   <th>
    Resource identifier
   </th>
   <td>
    UniProtKB accession number.
   </td>
  </tr>
  <tr>
   <th>
    Examples
   </th>
   <td>
    <pre><a href="http://www.uniprot.org/uniprot/P02760#section_x-ref">P02760</a>:
DR   PMAP-CutDB; P02760; -.

<a href="http://www.uniprot.org/uniprot/Q02383#section_x-ref">Q02383</a>:
DR   PMAP-CutDB; Q02383; -.</pre>
   </td>
  </tr>
 </table>
 <h3>
  Removal of the ec2dtosp.txt document file.
 </h3>
 <p>
  The document
  <strong>
   ec2dtosp.txt
  </strong>
  , which listed the Escherichia coli Gene-
      protein database (ECO2DBASE) entries cross-referenced in UniProtKB/Swiss-Prot, has been
      removed.
 </p>
 <h3>
  Changes concerning keywords
 </h3>
 <p>
  New keyword:
 </p>
 <ul>
  <li>
   <a href="http://www.uniprot.org/keywords/KW-1024">
    Diamond-Blackfan anemia
   </a>
  </li>
 </ul>
</div>