<div>
 <h2>
  Headlines
 </h2>
 <h3>
  Why do we keep dubious sequences in UniProtKB? How to discard them from a protein set?
 </h3>
 <p>
  More than 99% of the protein sequences provided by UniProtKB come from the translations of coding
sequences (CDS) submitted to the
  <a href="http://www.insdc.org/">
   EMBL-Bank/GenBank/DDBJ
  </a>
  nucleotide sequence resources. These CDS
are either generated by the application of gene prediction programs to genomic DNA sequences or via
the hypothetical translation of cloned cDNAs (see
  <a href="http://www.uniprot.org/help/sequence_origin">
   FAQ
  </a>
  ).
These methods themselves provide varying
degrees of support for the existence of a protein, which may be further supplemented in some cases
by other types of evidence (such as mass spectrometry data or evidence from direct protein
sequencing).
 </p>
 <p>
  In
  <a href="http://www.uniprot.org/news/2007/07/24/release">
   July 2007
  </a>
  , a new topic was introduced into UniProtKB to indicate the
evidence for the existence of a given protein, called 'Protein existence' (PE). 5 levels of
evidence have been defined: 1. evidence at protein level (e.g. clear identification by mass
spectrometry), 2. evidence at transcript level (e.g. the existence of a putative coding cDNA), 3.
inferred by homology (a predicted protein which has been assigned membership of a defined protein
family in UniProtKB), 4. predicted (a predicted protein which has not yet been assigned membership
of a defined protein family in UniProtKB) and 5. uncertain (e.g. dubious sequences, such as those
derived from the erroneous translation of a pseudogene or non-coding RNA). Currently in UniProtKB/Swiss-Prot,
the vast majority (71%) of the entries are found in the PE3 category. PE1 and PE2 represent each approximately
13% of the total number of entries, PE4 3% and PE5 only 0.3%.
 </p>
 <p>
  Entries that are attributed an existence level of 5 (PE5) are also
tagged with the term "Putative" in the 'Protein names' section (see
for example the
  <a href="http://www.uniprot.org/uniprot/A6NMY6#section_name">
   "Putative annexin A2-like protein"
  </a>
  )
and, in the 'General annotation (Comments)' section, with a
  <a href="http://www.uniprot.org/uniprot/A6NMY6#section_comments">
   'Caution'
  </a>
  subsection warning the user of a possible
problem. The caution subsections accompanying a PE5 entry usually are of the type:
  <a href="http://www.uniprot.org/uniprot/O65469#section_comments">
   "Could be the product of a pseudogene"
  </a>
  ,
  <a href="http://www.uniprot.org/uniprot/Q9UI25#section_comments">
   "Product of a dubious CDS prediction"
  </a>
  or
  <a href="http://www.uniprot.org/uniprot/P28626#section_comments">
   "Product of a dubious gene prediction"
  </a>
  .
 </p>
 <p>
  The PE section is included in the UniProtKB search engine. It is thus possible to retrieve all entries corresponding
to a defined PE level - and thereby exclude all PE5 proteins. For human proteins this can be achieved by searching
for: (organism:"Homo sapiens (Human) [9606]" AND reviewed:yes) NOT existence:uncertain. This search allows the retrieval
of
  <a href="http://www.uniprot.org/uniprot/?query=taxonomy:9606+AND+reviewed:yes+NOT+existence:uncertain">
   19'835
  </a>
  entries,
indicating that "uncertain" proteins represent 2.4% of the total human entries. Currently PE5 entries represent
only 0.3% of all UniProtKB/Swiss-Prot.
The higher proportion of sequences identified as uncertain or dubious in
  <em>
   Homo sapiens
  </em>
  may be a
product of the continuous manual curation and review of these sequences by groups of the CCDS
consortium, such as HAVANA, as well as UniProt curators.
 </p>
 <p>
  One may ask the question: why not delete PE5 sequences from UniProtKB and provide only the most
reliable sequences? As stated above, UniProt is continuously reviewing all protein sequences. This
process can result in both the removal of some PE5 entries (in which evidence of pseudogenization
is overwhelming for instance) as well as the upgrade of other PE5 entries (such as the putative
  <em>
   E.coli
  </em>
  pseudogene
  <a href="http://www.uniprot.org/uniprot/P0CB62">
   ymiA
  </a>
  which has now been
  <a href="http://www.ncbi.nlm.nih.gov/pubmed/19121005">
   found to produce a protein product
  </a>
  and which has now acquired a PE of 1 or
the human
  <a href="http://www.uniprot.org/uniprot/Q5VTU8">
   mitochondrial ATP synthase subunit epsilon-like protein
  </a>
  ).
However, many putative pseudogene sequences may be expected to remain in UniProtKB for some
time as it can be difficult to prove the non-existence of a protein, and for certain loci some
doubts may always persist. To give our users the opportunity to work on the most complete protein
set, we have chosen to keep all PE5 sequences with the appropriate 'Caution' comments, leaving to
the users the final decision whether to retrieve them or not (using the exclusion mechanism
described above). Note that the sequences which are removed from UniProtKB can subsequently be retrieved from the
  <a href="http://www.uniprot.org/help/uniparc">
   UniParc
  </a>
  archive if so desired.
 </p>
 <p>
  Finally, please remember that the PE assignment is made at the level of the UniProtKB entry and not
at the level of individual isoform sequences; hence, dubious alternative isoform sequences cannot
be excluded from a protein set by the UniProtKB search engine. However, comments about the evidence
supporting the existence of any given isoform can be found in the 'Note:' for that isoform in the
  <a href="http://www.uniprot.org/uniprot/Q9HCH5#section_alternative">
   'Alternative products'
  </a>
  section
(which lists all protein isoforms for each entry). For instance,
isoforms that have been identified only once through large scale sequencing are tagged with the
comment
  <a href="http://www.uniprot.org/uniprot/Q9HCH5#section_alternative">
   "No experimental confirmation available"
  </a>
  .

Note that UniProt may include isoforms that contain retained introns (as these may be
physiologically relevant) as well as isoforms that contain a premature stop codon and thus could be
the target for nonsense-mediated mRNA decay (
  <a href="http://www.ncbi.nlm.nih.gov/pubmed/19859661,19359157,19162024">
   NMD
  </a>
  ).
The mechanism of NMD involves a first round of
translation before the premature stop codon is detected (often referred to as "pioneer
translation"), and so at least one protein is synthesized from each NMD target mRNA. In addition,
some of the predicted NMD targets appear to be the most abundant isoforms in certain tissues (see
for instance the human GABA-B receptor 1
  <a href="http://www.uniprot.org/uniprot/Q9UBS5#section_alternative">
   isoform 1E
  </a>
  ).
 </p>
 <p>
  For additional information, see the
  <a href="http://www.uniprot.org/docs/pe_criteria">
   document
  </a>
  describing
the criteria used to assign the PE level of entries and the UniProtKB
  <a href="http://www.uniprot.org/manual/protein_existence">
   user manual
  </a>
  .
 </p>
 <h2>
  UniProtKB News
 </h2>
 <h3>
  Cross-references to OrthoDB
 </h3>
 <p>
  Cross-references have been added to
  <a href="http://cegg.unige.ch/orthodb">
   OrthoDB
  </a>
  ,
   a database of orthologous groups. OrthoDB presents a catalog of eukaryotic orthologous
   protein-coding genes. Orthology refers to the last common ancestor of the species under
   consideration, and thus OrthoDB explicitly delineates orthologs at each radiation along
   the species phylogeny.
 </p>
 <p>
  OrthoDB is available at
  <a href="http://cegg.unige.ch/orthodb">
   http://cegg.unige.ch/orthodb
  </a>
  .
 </p>
 <p>
  The format of the explicit links in the flat file is:
 </p>
 <table>
  <tr>
   <th>
    Resource abbreviation
   </th>
   <td>
    OrthoDB
   </td>
  </tr>
  <tr>
   <th>
    Resource identifier
   </th>
   <td>
    OrthoDB cluster number.
   </td>
  </tr>
  <tr>
   <th>
    Example
   </th>
   <td>
    <pre><a href="http://www.uniprot.org/uniprot/P00915#section_x-ref">P00915</a>:
DR   OrthoDB; EOG90KBJT; -.</pre>
   </td>
  </tr>
 </table>
 <p>
  <a href="http://www.uniprot.org/uniprot/?query=database%3AOrthoDB&amp;#38;sort=score">
   Show all the entries having a cross-reference to OrthoDB.
  </a>
 </p>
 <h3>
  Cross-references to PhylomeDB
 </h3>
 <p>
  Cross-references have been added to
  <a href="http://phylomedb.org/">
   PhylomeDB
  </a>
  ,
   a database for complete collections of gene phylogenies. PhylomeDB allows users to
   interactively explore the evolutionary history of genes through the visualization
   of phylogenetic trees and multiple sequence alignments.
 </p>
 <p>
  PhylomeDB is available at
  <a href="http://phylomedb.org/">
   http://phylomedb.org/
  </a>
  .
 </p>
 <p>
  The format of the explicit links in the flat file is:
 </p>
 <table>
  <tr>
   <th>
    Resource abbreviation
   </th>
   <td>
    PhylomeDB
   </td>
  </tr>
  <tr>
   <th>
    Resource identifier
   </th>
   <td>
    UniProtKB accession number.
   </td>
  </tr>
  <tr>
   <th>
    Example
   </th>
   <td>
    <pre><a href="http://www.uniprot.org/uniprot/Q8GTR4#section_x-ref">Q8GTR4</a>:
DR   PhylomeDB; Q8GTR4; -.</pre>
   </td>
  </tr>
 </table>
 <p>
  <a href="http://www.uniprot.org/uniprot/?query=database%3APhylomeDB&amp;#38;sort=score">
   Show all the entries having a cross-reference to PhylomeDB.
  </a>
 </p>
 <h3>
  Changes concerning keywords
 </h3>
 <p>
  New keywords:
 </p>
 <ul>
  <li>
   <a href="http://www.uniprot.org/keywords/KW-1051">
    Host synapse
   </a>
  </li>
  <li>
   <a href="http://www.uniprot.org/keywords/KW-1054">
    Niemann-Pick disease
   </a>
  </li>
  <li>
   <a href="http://www.uniprot.org/keywords/KW-1052">
    Target cell membrane
   </a>
  </li>
  <li>
   <a href="http://www.uniprot.org/keywords/KW-1053">
    Target membrane
   </a>
  </li>
 </ul>
 <p>
  Deleted keywords:
 </p>
 <ul>
  <li>
   Phorbol-ester binding
  </li>
  <li>
   Plant toxin
  </li>
 </ul>
 <h3>
  Changes in
  <a href="http://www.uniprot.org/docs/subcell">
   subcellular location controlled vocabulary
  </a>
 </h3>
 <p>
  New subcellular locations:
 </p>
 <ul>
  <li>
   Host synapse
  </li>
  <li>
   Target cell membrane
  </li>
  <li>
   Target membrane
  </li>
 </ul>
</div>