<div>
 <h2>
  Headlines
 </h2>
 <h3>
  300'000 HAMAP cross-references in UniProtKB/Swiss-Prot
 </h3>
 <p>
  Bacteria and archaea can live in pretty much every environmental niche we know of. From the
       bottom of the ocean floor to arctic ice, from wastewater treatment sludge to animal-and plant-
       associated environments, bacteria and archaea are everywhere. To explore this diversity the
       number of bacterial (and to a lesser extent archaeal) genomes being sequenced is rising
       practically exponentially, giving rise to huge numbers of protein sequences that are annotated
       to varying degrees of quality. To be able to use this data appropriately quality annotation is
       however essential. In order to supply this we started the
  <a href="http://hamap.expasy.org/">
   HAMAP
  </a>
  project (High-quality
       Automatic and Manual Annotation of microbial Proteins) in 2000. In this project,
       proteins from complete bacterial and archaeal proteomes,
       together with related plastid proteins, are automatically annotated based on manually created
       annotation templates for complete protein annotation, with template-based feature propagation.
       The
  <a href="http://hamap.expasy.org/families.html">
   annotation templates
  </a>
  and much more are available on the
  <a href="http://hamap.expasy.org/">
   HAMAP website
  </a>
  .
       As of January 2008 the sequences annotated by the HAMAP
       pipeline that fulfill all of its stringent criteria have been entering automatically into the
       Swiss-Prot section of UniProtKB (see
  <a href="http://www.uniprot.org/news/2008/01/15/release">
   release 54.7 news
  </a>
  ).
 </p>
 <p>
  There are now
  <a href="http://www.uniprot.org/uniprot/?query=database:hamap">
   304'013 UniProtKB/Swiss-Prot entries
  </a>
  with a HAMAP cross-reference line to at least one
       of the
  <a href="http://hamap.expasy.org/families.html">
   1'595 HAMAP families
  </a>
  ;
  <a href="http://www.uniprot.org/uniprot/?query=database:hamap+AND+taxonomy:2">
   278'635
  </a>
  are bacterial,
  <a href="http://www.uniprot.org/uniprot/?query=database:hamap+AND+taxonomy:2157">
   14'601
  </a>
  are archaeal and
  <a href="http://www.uniprot.org/uniprot/?query=organelle:plastid+AND+database:hamap">
   10'777
  </a>
  are encoded in plastids.
       Note that some of these entries are the templates for their families
       (see for example
  <a href="http://www.uniprot.org/uniprot/P31120">
   P31120
  </a>
  ); they include extra information not
       propagated to all members (for example biophysical chemical characterization, mutagenesis experiments,
       3D structures, induction and so on) that has allowed their use as models to annotate further entries (compare
       entry
  <a href="http://www.uniprot.org/uniprot/B6I1P9">
   B6I1P9
  </a>
  containing propagated annotation based on the
  <a href="http://hamap.expasy.org/unirule/MF_01554">
   family rule MF_01554
  </a>
  with model entry
  <a href="http://www.uniprot.org/uniprot/P31120">
   P31120
  </a>
  ).
 </p>
 <p>
  This large number of semi-automatically annotated entries means that nearly 60% of
       UniProtKB/Swiss-Prot consists of HAMAP entries; add to this the approximately 30'000 other
       bacterial and archaeal entries that are not members of a HAMAP family and you find that the
  <a href="http://www.uniprot.org/uniprot/?query=(taxonomy:2+OR+taxonomy:2157)+AND+reviewed:yes">
   total
        number of bacterial and archaeal entries
  </a>
  in UniProtKB/Swiss-Prot begins to reflect their
       preponderance in nature...
 </p>
 <h2>
  UniProtKB News
 </h2>
 <h3>
  Changes in
  <a href="http://www.uniprot.org/docs/subcell">
   subcellular location controlled vocabulary
  </a>
 </h3>
 <p>
  New subcellular locations:
 </p>
 <ul>
  <li>
   Acrosome outer membrane
  </li>
  <li>
   Spindle pole
  </li>
 </ul>
 <h3>
  Changes concerning the
  <a href="http://www.uniprot.org/docs/ptmlist">
   controlled vocabulary for PTMs
  </a>
 </h3>
 <p>
  New terms for the feature key 'Cross-link' ('CROSSLNK' in the flat file):
 </p>
 <ul>
  <li>
   3-(O4'-tyrosyl)-valine (Val-Tyr)
  </li>
  <li>
   5-(methoxymethyl)thiazole-4-carboxylic acid (Val-Cys)
  </li>
  <li>
   5-methyloxazole-4-carboxylic acid (Ser-Thr)
  </li>
  <li>
   Glutamyl lysine isopeptide (Glu-Lys) (interchain with K-...)
  </li>
  <li>
   Glutamyl lysine isopeptide (Lys-Glu) (interchain with E-...)
  </li>
  <li>
   Oxazole-4-carboxylic acid (Cys-Ser)
  </li>
  <li>
   Oxazole-4-carboxylic acid (Gly-Ser)
  </li>
  <li>
   Oxazoline-4-carboxylic acid (Cys-Ser)
  </li>
  <li>
   Thiazole-4-carboxylic acid (Gly-Cys)
  </li>
 </ul>
 <p>
  New terms for the feature key 'Modified residue' ('MOD_RES' in the flat file):
 </p>
 <ul>
  <li>
   (3R)-3-hydroxyasparagine
  </li>
  <li>
   (3R)-3-hydroxyaspartate
  </li>
  <li>
   (3R,4R)-3,4-dihydroxyproline
  </li>
  <li>
   (3S)-3-hydroxyasparagine
  </li>
  <li>
   1-amino-2-propanone
  </li>
  <li>
   4,5,5'-trihydroxyleucine
  </li>
  <li>
   5-glutamyl polyglutamate
  </li>
  <li>
   5-glutamyl polyglycine
  </li>
  <li>
   5-hydroxy-3-methylproline
  </li>
  <li>
   Cyclo[(prolylserin)-O-yl] cysteinate
  </li>
  <li>
   Glutamyl 5-glycerylphosphorylethanolamine
  </li>
  <li>
   N-carbamoylalanine
  </li>
  <li>
   N6-(pyridoxal phosphate)lysine
  </li>
  <li>
   N6-(retinylidene)lysine
  </li>
  <li>
   N6-biotinyllysine
  </li>
  <li>
   N6-lipoyllysine
  </li>
  <li>
   O-(2-aminoethylphosphoryl)serine
  </li>
  <li>
   O-(2-cholinephosphoryl)serine
  </li>
  <li>
   O-(pantetheine 4'-phosphoryl)serine
  </li>
  <li>
   O-(phosphoribosyl dephospho-coenzyme A)serine
  </li>
  <li>
   O-AMP-threonine
  </li>
 </ul>
 <p>
  Deleted terms:
 </p>
 <ul>
  <li>
   2-oxazoline-4-carboxylic acid (Cys-Ser)
  </li>
  <li>
   3-hydroxy-5-methylproline
  </li>
  <li>
   5-methyloxazole (Ser-Thr)
  </li>
  <li>
   Oxazole (Cys-Ser)
  </li>
  <li>
   Oxazole (Gly-Ser)
  </li>
  <li>
   Thiazole (Gly-Cys)
  </li>
  <li>
   Thiazoline-4-carboxylic acid (Thr-Cys)
  </li>
 </ul>
</div>