<div>
 <h2>
  Headlines
 </h2>
 <h3>
  Fission yeast: the third eukaryotic complete proteome in UniProtKB/Swiss-Prot
 </h3>
 <p>
  <a href="http://www.uniprot.org/taxonomy/4896">
   <em>
    Schizosaccharomyces pombe
   </em>
  </a>
  , the fission yeast,
      was isolated in 1893 by P. Lindner from East African
      millet beer, for which it was named, 'pombe' meaning 'beer' in
      Swahili. The genus name reflects both its relationship to
  <a href="http://www.uniprot.org/taxonomy/4932">
   budding yeast
  </a>
  (-saccharomyces),
      and the most striking feature that distinguishes it
      from other yeast species, i.e. reproduction by fission (Schizo-).
      Although both
  <em>
   S.pombe
  </em>
  and
  <em>
   Saccharomyces cerevisiae
  </em>
  are yeasts, they are
      genetically as divergent from each other as both are from man.
      Unlike
  <em>
   S.cerevisiae
  </em>
  ,
  <em>
   S.pombe
  </em>
  did not acquire its fame for its beer making talents -
      beer made with
  <em>
   S.pombe
  </em>
  seems to have quite a unsavoury acidic taste -
      but for the great scientific achievements its study permitted.
 </p>
 <p>
  As mentioned above, fission yeast
      divides not by budding, but by medial fission, a process that resembles higher
      eukaryotic cell division.
      The organism grows exclusively through its cell tips and divides upon reaching the appropriate size,
      producing 2 daughter cells of equal sizes.
      Thus a simple measure of its length gives an estimate of which cell cycle phase
      the cell is in. This approach allows
      the isolation of cell cycle mutants (cdc), based on the
      presence of elongated cells due to continuous cell growth in the absence of cell
      division. This feature makes
  <em>
   S.pombe
  </em>
  a first-rate model organism to study cell division.
      The characterization of cdc mutants led to the discovery of cyclin-dependent kinases which
      was eventually awarded the
  <a href="http://nobelprize.org/nobel_prizes/medicine/laureates/2001/index.html">
   2001 Nobel Prize in Medicine
  </a>
  .
 </p>
 <p>
  In 2002,
  <em>
   S.pombe
  </em>
  was the 6th eukaryotic organism to have its
  <a href="http://view.ncbi.nlm.nih.gov/pubmed/11859360">
   genome fully sequenced
  </a>
  .
      As of this release, it is the 3rd eukaryotic organism, after
  <a href="http://www.uniprot.org/news/2007/01/09/release">
   <em>
    S.cerevisiae
   </em>
  </a>
  and
  <a href="http://www.uniprot.org/news/2008/09/02/release">
   <em>
    Homo sapiens
   </em>
  </a>
  , for which the
      complete proteome is available in UniProtKB/Swiss-Prot. This set represents
  <a href="http://www.uniprot.org/uniprot/?query=taxonomy:4896+AND+reviewed:yes+AND+keyword:KW-0181">
   4'957 manually curated protein sequence entries
  </a>
  ,
      containing data from the scientific literature and numerous
      cross-references, including links to
  <a href="http://www.genedb.org/genedb/pombe/">
   GeneDB_Spombe
  </a>
  ,
      the fission yeast community database. A list of all
  <em>
   S.pombe
  </em>
  UniProtKB/Swiss-Prot entries is available in the
  <a href="http://www.uniprot.org/docs/pombe">
   pombe.txt
  </a>
  file.
 </p>
 <p>
  The
  <em>
   S.pombe
  </em>
  proteome we provide today is not a static one. We will keep
      revisiting and updating the entries as the science develops further. Analysis of
  <em>
   S.
       pombe
  </em>
  and
  <em>
   S.cerevisiae
  </em>
  proteins, coupled with phylogenetic studies, will allow
      the identification and annotation of homologous proteins in other organisms.
 </p>
 <h2>
  UniProtKB News
 </h2>
 <h3>
  Cross-references to OMA
 </h3>
 <p>
  Cross-references have been added to the
  <a href="http://www.omabrowser.org/">
   OMA
  </a>
  project.
      The OMA project is a massive cross-comparison of complete genomes to identify the evolutionary
      relation between any pair of proteins. The main features of OMA are the large number of genomes
      from all kingdoms of life, the strict verification of orthology assignments and the determination
      of the phylogenetic relationship between any two proteins.
 </p>
 <p>
  OMA is available at
  <a href="http://www.omabrowser.org/">
   http://www.omabrowser.org/
  </a>
  .
 </p>
 <p>
  The format of the explicit links in the flat file is:
 </p>
 <table>
  <tr>
   <th>
    Resource abbreviation
   </th>
   <td>
    OMA
   </td>
  </tr>
  <tr>
   <th>
    Resource identifier
   </th>
   <td>
    UniProtKB accession number.
   </td>
  </tr>
  <tr>
   <th>
    Optional information 1
   </th>
   <td>
    OMA group fingerprint.
   </td>
  </tr>
  <tr>
   <th>
    Examples
   </th>
   <td>
    <pre><a href="http://www.uniprot.org/uniprot/P39899#section_x-ref">P39899</a>:
DR   OMA; P39899; YANTHIA.

<a href="http://www.uniprot.org/uniprot/Q9Y6C2#section_x-ref">Q9Y6C2</a>:
DR   OMA; Q9Y6C2; EGLENKP.</pre>
   </td>
  </tr>
 </table>
 <h3>
  Changes concerning keywords
 </h3>
 <p>
  New keyword:
 </p>
 <ul>
  <li>
   <a href="http://www.uniprot.org/keywords/KW-1022">
    Congenital generalized lipodystrophy
   </a>
  </li>
 </ul>
 <h3>
  Changes in
  <a href="http://www.uniprot.org/docs/subcell">
   subcellular location controlled vocabulary
  </a>
 </h3>
 <p>
  New subcellular locations:
 </p>
 <ul>
  <li>
   Host phagosome
  </li>
  <li>
   Host phagosome membrane
  </li>
  <li>
   Host presynaptic cell membrane
  </li>
  <li>
   Host synapse
  </li>
 </ul>
 <p>
  New subcellular topology:
 </p>
 <ul>
  <li>
   GPI-like-anchor
  </li>
 </ul>
</div>