<div>
 <h2>
  Headlines
 </h2>
 <h3>
  Bacillus subtilis, a Gram-positive model bacterium fully annotated in UniProtKB/Swiss-Prot
 </h3>
 <p>
  We are all aware of the importance of model bacterial systems.
  <a href="http://www.ecogene.org/">
   <em>
    Escherichia coli
   </em>
   K12
  </a>
  is the
paradigm for Gram-negative bacteria, but what of Gram-positive bacteria? There are a large variety
of these bacteria that serve us, are neutral or infect us, and model systems for these bacteria are
in demand.
 </p>
 <p>
  <a href="http://genodb.pasteur.fr/cgi-bin/WebObjects/GenoList.woa/wa/goToTaxoRank?level=Bacillus subtilis 168">
   <em>
    Bacillus subtilis
   </em>
  </a>
  ,
a rod-shaped, soil-and water-dwelling bacterium originally described as
  <em>
   Vibrio subtilis
  </em>
  in 1835 by Ehrenberg and renamed in 1872 by Cohn has served this role for over a
century.
  <em>
   B.subtilis
  </em>
  differentiates to produce endospores, can be made naturally competent for DNA
uptake and is a bacteriophage host. In the wild it has been seen to produce over 2 dozen different
antibiotics. These characteristics make it an obvious choice as a model system for bacterial
differentiation and genetics, as well as a model for other - often more dangerous - bacteria such as
  <em>
   Bacillus anthracis
  </em>
  ,
  <em>
   Mycobacterium tuberculosis
  </em>
  or
  <em>
   Staphylococcus aureus
  </em>
  . Additionally, it is used
for the production of various industrially interesting enzymes such as amylases and proteases. A
substrain,
  <em>
   B.subtilis natto
  </em>
  , is used to prepare natto, a traditional Japanese dish made from fermented
soybeans. Although
  <em>
   B.subtilis
  </em>
  is not considered pathogenic for any known organism, it has been
isolated from patients suffering from various illness such as endocarditis, pneumonia etc., and
also occasionally from spoiled food where it might be responsible for cases of food poisoning.
 </p>
 <p>
  The genome of
  <em>
   B.subtilis
  </em>
  168, a widely used laboratory strain, was sequenced by a large
international consortium in 1997 - the 6th bacterium to be fully sequenced. The sequence was updated
and reannotated in 2009 by the Institut Pasteur and the GÃ©noscope. In coordination with them we have
annotated the complete proteome, providing all 4'192
  <em>
   B.subtilis
  </em>
  proteins in UniProtKB/Swiss-Prot,
each of which has a cross-reference to the dedicated
  <em>
   B.subtilis
  </em>
  database
  <a href="http://genodb.pasteur.fr/cgi-bin/WebObjects/GenoList.woa/wa/goToTaxoRank?level=Bacillus subtilis 168">
   SubtiList/GenoList
  </a>
  as well as other
databases. A list of all
  <em>
   B.subtilis
  </em>
  UniProtKB/Swiss-Prot entries is available in the
  <a href="http://www.uniprot.org/docs/bacsu">
   bacsu.txt
  </a>
  file.
This of course provides a snapshot of the knowledge about this first fully manually annotated Gram-positive
model organism and will date easily. Despite having been so intently studied for so long,
there are many
  <em>
   B.subtilis
  </em>
  proteins about which we know very little. There will be work for years to
come for the
  <em>
   B.subtilis
  </em>
  (and larger scientific) community as these proteins and their homologues
are characterized.
 </p>
 <p>
  <a href="http://www.uniprot.org/uniprot/?query=taxonomy:1423+AND+keyword:181">
   All
   <em>
    B.subtilis
   </em>
   entries
  </a>
  can be retrieved from UniProtKB/Swiss-Prot combining the organism name "Bacillus subtilis" (or the taxonomy identifier 1423)
with the keyword 'Complete proteome' (organism:"Bacillus subtilis" AND keyword:"Complete proteome" or organism:1423 AND keyword:181).
 </p>
 <h2>
  UniProtKB News
 </h2>
 <h3>
  Cross-references to EuPathDB
 </h3>
 <p>
  Cross-references have been added to the Eukaryotic Pathogen Database Resources
  <a href="http://www.eupathdb.org/">
   EuPathDB
  </a>
  (formerly ApiDB),
   an integrated database covering the eukaryotic pathogens of the genera Cryptosporidium, Giardia, Leishmania,
   Neospora, Plasmodium, Toxoplasma, Trichomonas and Trypanosoma. While each of these groups is supported by a
   taxon-specific database built upon the same infrastructure, the EuPathDB portal offers an entry point to all
   these resources ("child databases": e.g. ToxoDB, PlasmoDB, CryptoDB...), and the opportunity to leverage
   orthology for searches across genera.
 </p>
 <p>
  EuPathDB is available at
  <a href="http://www.eupathdb.org/">
   http://www.eupathdb.org/
  </a>
  .
 </p>
 <p>
  The format of the explicit links in the flat file is:
 </p>
 <table>
  <tr>
   <th>
    Resource abbreviation
   </th>
   <td>
    EuPathDB
   </td>
  </tr>
  <tr>
   <th>
    Resource identifier
   </th>
   <td>
    Combination of the child database name and the accession number in this database concatenated by a ":".
   </td>
  </tr>
  <tr>
   <th>
    Examples
   </th>
   <td>
    <pre><a href="http://www.uniprot.org/uniprot/P84155#section_x-ref">P84155</a>:
DR   EuPathDB; TritrypDB:LmjF06.1270; -.

<a href="http://www.uniprot.org/uniprot/Q38FA5#section_x-ref">Q38FA5</a>:
DR   EuPathDB; TritrypDB:Tb09.160.2970; -.</pre>
   </td>
  </tr>
 </table>
 <p>
  <a href="http://www.uniprot.org/uniprot/?query=database:EuPathDB">
   Show all the entries having a cross-reference to EuPathDB.
  </a>
 </p>
 <h3>
  Cross-references to ProtClustDB
 </h3>
 <p>
  Cross-references have been added to Entrez Protein Clusters
  <a href="http://www.ncbi.nlm.nih.gov/sites/entrez?db=proteinclusters">
   ProtClustDB
  </a>
  ,
  a collection of related protein sequences (clusters) which consists of Reference Sequence proteins
  encoded by complete genomes. This database contains both curated and non-curated clusters. The Protein
  Clusters database provides easy access to annotation information, publications, domains, structures, and
  external links and analysis tools including multiple alignments, phylogenetic trees, and genomic neighborhoods
  (ProtMap).
 </p>
 <p>
  ProtClustDB is available at
  <a href="http://www.ncbi.nlm.nih.gov/sites/entrez?db=proteinclusters">
   http://www.ncbi.nlm.nih.gov/sites/entrez?db=proteinclusters
  </a>
  .
 </p>
 <p>
  The format of the explicit links in the flat file is:
 </p>
 <table>
  <tr>
   <th>
    Resource abbreviation
   </th>
   <td>
    ProtClustDB
   </td>
  </tr>
  <tr>
   <th>
    Resource identifier
   </th>
   <td>
    ProtClustDB accession number.
   </td>
  </tr>
  <tr>
   <th>
    Examples
   </th>
   <td>
    <pre><a href="http://www.uniprot.org/uniprot/P99178#section_x-ref">P99178</a>:
DR   ProtClustDB; PRK05431; -.

<a href="http://www.uniprot.org/uniprot/P92693#section_x-ref">P92693</a>:
DR   ProtClustDB; MTH00098; -.</pre>
   </td>
  </tr>
 </table>
 <p>
  <a href="http://www.uniprot.org/uniprot/?query=database:ProtClustDB">
   Show all the entries having a cross-reference to ProtClustDB.
  </a>
 </p>
 <h3>
  Cross-references to SUPFAM
 </h3>
 <p>
  Cross-references have been added to the Superfamily database of structural and functional annotation
  <a href="http://supfam.org">
   SUPFAM
  </a>
  , a database of structural and functional annotation for all proteins
  and genomes. The SUPFAM annotation is based on a collection of hidden Markov models, which represent
  structural protein domains at the SCOP superfamily level. A superfamily groups together domains which have an
  evolutionary relationship. The annotation is produced by scanning protein sequences from over 1,200 completely
  sequenced genomes against the hidden Markov models.
 </p>
 <p>
  SUPFAM is available at
  <a href="http://supfam.org">
   http://supfam.org
  </a>
  .
 </p>
 <p>
  The format of the explicit links in the flat file is:
 </p>
 <table>
  <tr>
   <th>
    Resource abbreviation
   </th>
   <td>
    SUPFAM
   </td>
  </tr>
  <tr>
   <th>
    Resource identifier
   </th>
   <td>
    SUPFAM superfamily identifier.
   </td>
  </tr>
  <tr>
   <th>
    Optional information 1
   </th>
   <td>
    SUPFAM superfamily domain name.
   </td>
  </tr>
  <tr>
   <th>
    Optional information 2
   </th>
   <td>
    Number of hits found.
   </td>
  </tr>
  <tr>
   <th>
    Examples
   </th>
   <td>
    <pre><a href="http://www.uniprot.org/uniprot/P08519#section_x-ref">P08519</a>:
DR   SUPFAM; SSF57440; Kringle-like; 38.
DR   SUPFAM; SSF50494; Pept_Ser_Cys; 1.

<a href="http://www.uniprot.org/uniprot/P00967#section_x-ref">P00967</a>:
DR   SUPFAM; SSF56042; AIR_synth_C; 2.
DR   SUPFAM; SSF53328; formyl_transf; 1.
DR   SUPFAM; SSF52440; PreATP-grasp-like; 1.
DR   SUPFAM; SSF55326; PurM_N-like; 2.
DR   SUPFAM; SSF51246; Rudmnt_hyb_motif; 1.</pre>
   </td>
  </tr>
 </table>
 <p>
  <a href="http://www.uniprot.org/uniprot/?query=database%3ASUPFAM&amp;#38;sort=score">
   Show all the entries having a cross-reference to SUPFAM.
  </a>
 </p>
 <h3>
  Format change in the cross-references to HOVERGEN
 </h3>
 <p>
  The format of the cross-references to the
  <a href="http://pbil.univ-lyon1.fr/databases/hovergen.php">
   HOVERGEN
  </a>
  project has changed: The resource identifier, which was a UniProtKB accession number, has been
   replaced by a HOVERGEN identifier.
 </p>
 <p>
  Example:
 </p>
 <p>
  Previous format:
 </p>
 <pre>
DR   HOVERGEN; P32754; -.
</pre>
 <p>
  New format:
 </p>
 <pre>
DR   HOVERGEN; HBG005987; -.
</pre>
 <p>
  <a href="http://www.uniprot.org/uniprot/?query=database%3AHOVERGEN&amp;#38;sort=score">
   Show all the entries having a cross-reference to HOVERGEN.
  </a>
 </p>
 <h3>
  Changes concerning keywords
 </h3>
 <p>
  New keywords:
 </p>
 <ul>
  <li>
   <a href="http://www.uniprot.org/keywords/KW-1060">
    Myofibrillar myopathy
   </a>
  </li>
  <li>
   <a href="http://www.uniprot.org/keywords/KW-1061">
    Dermonecrosis
   </a>
  </li>
  <li>
   <a href="http://www.uniprot.org/keywords/KW-1062">
    Cushing syndrome
   </a>
  </li>
  <li>
   <a href="http://www.uniprot.org/keywords/KW-1063">
    Hypotrichosis
   </a>
  </li>
 </ul>
 <h3>
  Changes concerning the
  <a href="http://www.uniprot.org/docs/ptmlist">
   controlled vocabulary for PTMs
  </a>
 </h3>
 <p>
  Modified term for the feature key 'Cross-link' ('CROSSLNK' in the flat file):
 </p>
 <p>
  New terms:
 </p>
 <ul>
  <li>
   Alanine isoaspartyl cyclopeptide (Ala-Asn)
  </li>
  <li>
   Glycyl cysteine dithioester (Cys-Gly) (interchain with G-...)
  </li>
  <li>
   Trithiocysteine (Cys-Cys)
  </li>
 </ul>
 <p>
  Modified terms for the feature key 'Lipidation' ('LIPID' in the flat file):
 </p>
 <p>
  New terms:
 </p>
 <ul>
  <li>
   N-[(12R)-12-hydroxymyristoyl]cysteine
  </li>
  <li>
   N-(12-oxomyristoyl)cysteine
  </li>
 </ul>
 <p>
  Modified terms for the feature key 'Modified residue' ('MOD_RES' in the flat file):
 </p>
 <p>
  New terms:
 </p>
 <ul>
  <li>
   S-(4-hydroxycinnamyl)cysteine
  </li>
  <li>
   S-cysteinyl cysteine
  </li>
  <li>
   Tele-(1,2,3-trihydroxypropan-2-yl)histidine
  </li>
 </ul>
</div>