<div>
 <h2>
  Headlines
 </h2>
 <h3>
  XMRV complete proteome in UniProtKB/Swiss-Prot
 </h3>
 <p>
  Despite the 118 human pathogenic viruses identified so far, our knowledge of these
pathogens is still incomplete. Several human pathologies are suspected to be
induced by unknown viruses. In this context, a new virus was isolated from human prostate in
  <a href="http://www.ncbi.nlm.nih.gov/pubmed/16609730">
   2006
  </a>
  and was
named 'Xenotropic Moloney murine leukemia virus-Related Virus' (
  <a href="http://viralzone.expasy.org/all_by_species/67.html">
   XMRV
  </a>
  ).
This retrovirus is the first representative of the
  <a href="http://viralzone.expasy.org/all_by_species/67.html">
   gammaretrovirus
  </a>
  genus
to be isolated in humans. These retroviruses are known to induce various cancers in their host and a causal link
with prostate cancer was suspected. This link was
  <a href="http://www.ncbi.nlm.nih.gov/pubmed/19805305">
   experimentally established
  </a>
  but later
  <a href="http://www.ncbi.nlm.nih.gov/pubmed/19835577">
   refuted
  </a>
  and thus remains a matter of debate.
The same virus has been recently
  <a href="http://www.ncbi.nlm.nih.gov/pubmed/19815723">
   associated with chronic fatigue syndrome
  </a>
  (CFS): XMRV has been isolated in 4% of healthy subjects, and in 67% of CFS patients. Large scale
epidemiological studies must be performed to establish with certainty whether these
correlations are relevant.
 </p>
 <p>
  Where did XMRV come from? Retroviruses identified in patients with CFS or prostate
cancer are highly related (more than 90% DNA sequence identity) to a group of mouse viruses
called xenotropic murine leukemia virus (MLV). Xenotropic MLVs
are endogenous retroviruses, i.e. the viral DNA is stably integrated in the mouse genome.
Mice produce low levels of the virus - a few infectious particles per ml of blood -
but the virus cannot reinfect mouse tissues. Instead it spreads to other species, such as humans, which is the reason for the term 'xenotropic',
meaning the virus can grow in species other than the species of origin. Therefore it makes sense to hypothesize that XMRV
is a xenotropic MLV that crossed from mice to humans.
 </p>
 <p>
  The mode of transmission of XMRV is largely unknown. It could be via transfusion, intravenous drug use, or by other blood-borne routes,
but other modes of transmission (respiratory, sexual, etc.) cannot be excluded.
 </p>
 <p>
  It will take time to answer the numerous questions raised by the discovery of XMRV. In terms of treatment,
the good news is that some of the anti-retroviral drugs used for treating AIDS can immediately be tested for their efficacy against CFS. Indeed,
  <a href="http://www.ncbi.nlm.nih.gov/pubmed/19959199">
   susceptibility of XMRV to AZT
  </a>
  has recently been
demonstrated.
 </p>
 <p>
  The
  <a href="http://www.uniprot.org/uniprot/?query=taxonomy:373193+AND+reviewed:yes">
   complete proteome of XMRV
  </a>
  has been annotated along with
  <a href="http://www.uniprot.org/uniprot/?query=taxonomy:11801+AND+reviewed:yes">
   that of the well-studied MLV
  </a>
  which is 65% (env) to 85% (gag-pol) identical and has served as a model for XMRV functional annotation.
 </p>
 <h2>
  UniProtKB News
 </h2>
 <h3>
  Cross-references to eggNOG
 </h3>
 <p>
  Cross-references have been added to
  <a href="http://eggnog.embl.de/">
   eggNOG
  </a>
  (evolutionary genealogy of genes: Non-supervised Orthologous Groups), a database of
   orthologous groups of genes. The orthologous groups are annotated with functional description
   lines (derived by identifying a common denominator for the genes based on their various
   annotations), with functional categories (i.e derived from the original COG/KOG categories).
 </p>
 <p>
  eggNOG is available at
  <a href="http://eggnog.embl.de/">
   http://eggnog.embl.de/
  </a>
  .
 </p>
 <p>
  The format of the explicit links in the flat file is:
 </p>
 <table>
  <tr>
   <th>
    Resource abbreviation
   </th>
   <td>
    eggNOG
   </td>
  </tr>
  <tr>
   <th>
    Resource identifier
   </th>
   <td>
    eggNOG cluster identifier.
   </td>
  </tr>
  <tr>
   <th>
    Example
   </th>
   <td>
    <pre><a href="http://www.uniprot.org/uniprot/P33887#section_x-ref">P33887</a>:
DR   eggNOG; maNOG10115; -.
</pre>
   </td>
  </tr>
 </table>
 <p>
  <a href="http://www.uniprot.org/uniprot/?query=database%3AeggNOG&amp;#38;sort=score">
   Show all the entries having a cross-reference to eggNOG.
  </a>
 </p>
 <h3>
  Format change in the cross-references to HAMAP
 </h3>
 <p>
  The format of the cross-references to the
  <a href="http://hamap.expasy.org/families.html">
   HAMAP
  </a>
  database has changed in order to align it with the format of other InterPro member databases.
 </p>
 <p>
  Previous format:
 </p>
 <table>
  <tr>
   <th>
    Resource abbreviation
   </th>
   <td>
    HAMAP
   </td>
  </tr>
  <tr>
   <th>
    Resource identifier
   </th>
   <td>
    HAMAP unique identifier for a protein family.
   </td>
  </tr>
  <tr>
   <th>
    Optional information 1
   </th>
   <td>
    Nature of hits found. The values are either 'fused', 'atypical', 'atypical/fused' or '-': 'fused' indicates that the family signature does not cover the entire protein; 'atypical' means that the protein is divergent in sequence or has mutated functional sites and should not be included in family datasets; 'atypical/fused' is a combination of the previous two cases; '-' is a placeholder for an empty field.
   </td>
  </tr>
  <tr>
   <th>
    Optional information 2
   </th>
   <td>
    Number of hits found, which is generally 1, rarely 2 for the fusion of identical domains/proteins.
   </td>
  </tr>
  <tr>
   <th>
    Examples
   </th>
   <td>
    <pre><a href="http://www.uniprot.org/uniprot/P12743#section_x-ref">P12743</a>:
DR   HAMAP; MF_00326; -; 1.

<a href="http://www.uniprot.org/uniprot/Q9K3D6#section_x-ref">Q9K3D6</a>:
DR   HAMAP; MF_00006; fused; 1.
DR   HAMAP; MF_01105; atypical/fused; 1.</pre>
   </td>
  </tr>
 </table>
 <p>
  New format:
 </p>
 <table>
  <tr>
   <th>
    Resource abbreviation
   </th>
   <td>
    HAMAP
   </td>
  </tr>
  <tr>
   <th>
    Resource identifier
   </th>
   <td>
    HAMAP unique identifier for a protein family signature.
   </td>
  </tr>
  <tr>
   <th>
    Optional information 1
   </th>
   <td>
    HAMAP entry name for a protein family.
   </td>
  </tr>
  <tr>
   <th>
    Optional information 2
   </th>
   <td>
    Number of hits found, which is generally 1, rarely 2 for the fusion of identical domains/proteins.
   </td>
  </tr>
  <tr>
   <th>
    Optional information 3
   </th>
   <td>
    Nature of hits found. The values are either 'fused', 'atypical', 'atypical/fused' or '-': 'fused' indicates that the family signature does not cover the entire protein; 'atypical' means that the protein is divergent in sequence or has mutated functional sites and should not be included in family datasets; 'atypical/fused' is a combination of the previous two cases; '-' is a placeholder for an empty field.
   </td>
  </tr>
  <tr>
   <th>
    Examples
   </th>
   <td>
    <pre>
DR   HAMAP; MF_00326; Ribosomal_L7Ae; 1; -.

DR   HAMAP; MF_00006; Arg_succ_lyase; 1; fused.
DR   HAMAP; MF_01105; N-acetyl_glu_synth; 1; atypical/fused.</pre>
   </td>
  </tr>
 </table>
 <p>
  <a href="http://www.uniprot.org/uniprot/?query=database%3AHAMAP&amp;#38;sort=score">
   Show all the entries having a cross-reference to HAMAP.
  </a>
 </p>
 <h3>
  Format change in the cross-references to HOGENOM
 </h3>
 <p>
  The format of the cross-references to the
  <a href="http://pbil.univ-lyon1.fr/databases/hogenom.php">
   HOGENOM
  </a>
  project has changed: The resource identifier, which was a UniProtKB accession number, has been
   replaced by a HOGENOM identifier.
 </p>
 <p>
  Example:
 </p>
 <p>
  Previous format:
 </p>
 <pre>
DR   HOGENOM; P0A9I1; -.
</pre>
 <p>
  New format:
 </p>
 <pre>
DR   HOGENOM; HBG676713; -.
</pre>
 <p>
  <a href="http://www.uniprot.org/uniprot/?query=database%3AHOGENOM&amp;#38;sort=score">
   Show all the entries having a cross-reference to HOGENOM.
  </a>
 </p>
 <h3>
  Changes concerning keywords
 </h3>
 <p>
  New keywords:
 </p>
 <ul>
  <li>
   <a href="http://www.uniprot.org/keywords/KW-1058">
    Asthma
   </a>
  </li>
  <li>
   <a href="http://www.uniprot.org/keywords/KW-1056">
    Heterotaxy
   </a>
  </li>
  <li>
   <a href="http://www.uniprot.org/keywords/KW-1057">
    Nemaline myopathy
   </a>
  </li>
 </ul>
 <h3>
  Changes in
  <a href="http://www.uniprot.org/docs/subcell">
   controlled vocabulary for subcellular locations
  </a>
 </h3>
 <p>
  New subcellular locations:
 </p>
 <ul>
  <li>
   Barrier septum
  </li>
  <li>
   Cell septum
  </li>
  <li>
   Cell tip
  </li>
  <li>
   Photoreceptor inner segment
  </li>
  <li>
   Photoreceptor outer segment
  </li>
 </ul>
 <h3>
  Changes concerning the
  <a href="http://www.uniprot.org/docs/ptmlist">
   controlled vocabulary for PTMs
  </a>
 </h3>
 <p>
  Modified terms for the feature key 'Modified residue' ('MOD_RES' in the flat file):
 </p>
 <p>
  New terms:
 </p>
 <ul>
  <li>
   5-glutamyl 2-aminoadipic acid
  </li>
  <li>
   5-glutamyl N2-lysine
  </li>
 </ul>
</div>