<div>
 <h2>
  Headline
 </h2>
 <h3>
  Fishing for new mutations in the human exome
 </h3>
 <p>
  Understanding the role of genetic variants in human health and disease is crucial in modern biology and medicine. The
  <a href="http://hapmap.ncbi.nlm.nih.gov/index.html.en">
   International HapMap Project
  </a>
  and, more recently, the
  <a href="http://www.1000genomes.org/">
   1000 Genomes Project
  </a>
  are progressively unveiling the map of human genome variation at the scale of the
  <a href="http://www.ncbi.nlm.nih.gov/pubmed/20981092">
   human population
  </a>
  , generating a flood of interesting data. Smaller research projects focused on disease-causing mutations also contribute through the development of new fruitful approaches. One of the current trends in large and small scale projects is exome sequencing. The rationale is that the clear majority of allelic variants known to underlie mendelian disorders disrupt protein-coding sequences. Restricting sequencing to exons decreases the sample size to 2-5% of that of the whole genome, thus saving time and money, while allowing the identification of missense and nonsense mutations, of small insertions and deletions (indels), as well as of splice donor and acceptor site variants. By definition, exome sequencing does not permit the discovery of mutations in non-coding, regulatory or intronic genomic regions which are known to
  <a href="http://www.ncbi.nlm.nih.gov/pubmed/20506564">
   affect disease
  </a>
  .
 </p>
 <p>
  The exome sequencing strategy is proving to be quite effective, as it has recently been used to pinpoint several genes whose mutations are associated with diseases, including
  <a href="http://www.uniprot.org/uniprot/Q02127">
   <span class="caps">
    DHODH
   </span>
  </a>
  involved in
  <a href="http://www.ncbi.nlm.nih.gov/omim/263750" title="POADS">
   postaxial acrofacial dysostosis
  </a>
  (
  <a href="http://www.ncbi.nlm.nih.gov/pubmed/19915526">
   Ng et al., 2010
  </a>
  ),
  <a href="http://www.uniprot.org/uniprot/O43379">
   <span class="caps">
    WDR62
   </span>
  </a>
  in severe cerebral cortical malformations (
  <a href="http://www.ncbi.nlm.nih.gov/pubmed/20729831">
   Bilguvar et al., 2010
  </a>
  ) and
  <a href="http://www.uniprot.org/uniprot/O14686">
   <span class="caps">
    MLL2
   </span>
  </a>
  in
  <a href="http://www.ncbi.nlm.nih.gov/omim/147920" title="KABS">
   Kabuki syndrome
  </a>
  (
  <a href="http://www.ncbi.nlm.nih.gov/pubmed/20711175">
   Ng et al., 2010
  </a>
  ).
 </p>
 <p>
  The annotation of single amino acid polymorphisms (SAPs) has always been a priority in UniProtKB/Swiss-Prot, including not only ‘neutral’ polymorphisms, resulting from normal variations among individuals, but also disease-associated mutations. Thus missense SAPs identified by the exome-sequencing strategy have been quickly annotated and integrated in the ‘Sequence annotation (Features)’ section of their respective entries (
  <a href="http://www.uniprot.org/uniprot/Q02127#section_features">
   <span class="caps">
    Q02127
   </span>
  </a>
  ,
  <a href="http://www.uniprot.org/uniprot/O43379#section_features">
   <span class="caps">
    O43379
   </span>
  </a>
  and
  <a href="http://www.uniprot.org/uniprot/O14686#section_features">
   <span class="caps">
    O14686
   </span>
  </a>
  ). The associated phenotypes are described in the ‘General annotation (Comments)’ section in ‘Involvement in disease’ (
  <a href="http://www.uniprot.org/uniprot/Q02127#section_comments">
   <span class="caps">
    Q02127
   </span>
  </a>
  ,
  <a href="http://www.uniprot.org/uniprot/O43379#section_comments">
   <span class="caps">
    O43379
   </span>
  </a>
  and
  <a href="http://www.uniprot.org/uniprot/O14686#section_comments">
   <span class="caps">
    O14686
   </span>
  </a>
  ).
 </p>
 <p>
  Over the years, we have developed a defined format to describe SAPs in the ‘Sequence annotation (Features)’ section, including
  <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/">
   dbSNP
  </a>
  accession numbers, when they exist, and links to bibliographic references. Disease-causing mutations are tagged, whenever possible, with the official abbreviation of the phenotype provided by the
  <a href="http://www.ncbi.nlm.nih.gov/sites/entrez?db=omim">
   <span class="caps">
    OMIM
   </span>
   database
  </a>
  . In addition to missense mutations, in-frame indels are also reported (
  <a href="http://www.uniprot.org/uniprot/P35453#section_features">
   <span class="caps">
    P35453
   </span>
  </a>
  ,
  <a href="http://www.uniprot.org/uniprot/P02730#section_features">
   <span class="caps">
    P02730
   </span>
  </a>
  or
  <a href="http://www.uniprot.org/uniprot/P33897#section_features">
   <span class="caps">
    P33897
   </span>
  </a>
  ). When it is not possible to represent the whole variation landscape for a given protein within the UniProtKB entry, we try and provide cross-references to specialized resources (see for instance the ‘Web resources’ section in
  <a href="http://www.uniprot.org/uniprot/P04637#section_web">
   human p53 entry
  </a>
  ). Our annotation effort does not include the representation of mutations that cause major changes to a protein sequence, such as frameshift mutations or variations at splice sites, as their deleterious effects on protein function are usually obvious.
 </p>
 <p>
  Close to 63’000 human SAPs are currently stored in UniProtKB/Swiss-Prot and about 30% of them are reported as disease-associated in the literature. SAPs selected from this pool are mapped to reference nucleotide sequences from
  <a href="http://www.ncbi.nlm.nih.gov/RefSeq/">
   RefSeq
  </a>
  and
  <a href="http://www.lrg-sequence.org/page.php">
   <span class="caps">
    LRG
   </span>
  </a>
  , following the guidelines established by the
  <a href="http://www.hgvs.org/">
   Human Genome Variation Society
  </a>
  for sequence variant designation, and submitted to
  <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/">
   dbSNP
  </a>
  (see for instance
  <a href="http://www.ncbi.nlm.nih.gov/SNP/snp_ref.cgi?rs=121908210">
   dbSNP/Swiss-Prot variant rs121908210
  </a>
  ). Thanks to a tight collaboration with Ensembl, all human variants stored in UniProtKB and characterized by a dbSNP accession number (or submitted to
  <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/">
   dbSNP
  </a>
  ) can also be accessed from the
  <a href="http://www.ensembl.org/">
   Ensembl database
  </a>
  and viewed in the context of their nucleotide sequence (see
  <a href="http://www.ensembl.org/Homo_sapiens/Variation/Summary?r=19:36037396-36038396;v=rs1269215;vdb=variation;vf=19465159">
   variant rs1269215
  </a>
  stored in UniProtKB entry
  <a href="http://www.uniprot.org/uniprot/Q9BVK8#section_features">
   <span class="caps">
    Q9BVK8
   </span>
  </a>
  ). Our ultimate goal is to spread information about protein variations to the broadest possible audience.
 </p>
 <h2>
  UniProtKB news
 </h2>
 <h3>
  Line length limit
 </h3>
 <p>
  Historically, UniProtKB flat file entries were formatted to not exceed 75 characters per line. This limitation served on one hand to display them nicely on small screens and to allow them to be processed by programs that had memory limitations. Meanwhile, computers have become more powerful and most programs have been adapted accordingly. UniProt has already made a few exceptions to the line length limit for data that cannot be wrapped, such as URLs or DOIs, or where wrapping does not increase readability, such as for protein names and a few cross-references to other databases. Especially for the latter, we have increasingly more additional information to incorporate. We will continue to wrap lines at 75 characters where it helps to increase readability, but allow for more characters where necessary. The new upper limit is 255 characters per line, as some users still depend on software with this limitation.
 </p>
 <h3>
  Changes to cross-references to RefSeq
 </h3>
 <p>
  We have introduced an additional field to the cross-reference (DR line in the flat file) to the
  <a href="http://www.ncbi.nlm.nih.gov/RefSeq/">
   <span class="caps">
    NCBI
   </span>
   Reference Sequences database
  </a>
  to show the
  <a href="http://www.ncbi.nlm.nih.gov/refseq/key.html#accession">
   RefSeq nucleotide accession number
  </a>
  .
 </p>
 <p>
  The format of the explicit links in the flat file is:
 </p>
 <pre>DR   RefSeq; <em>RefSeq protein accession number</em>; <em>RefSeq nucleotide accession number</em>.</pre>
 <p>
  <strong>
   Example:
  </strong>
  <a href="http://www.uniprot.org/uniprot/P00816#section_x-ref">
   <span class="caps">
    P00816
   </span>
  </a>
 </p>
 <p>
  Previous format in the flat file:
 </p>
 <pre>DR   RefSeq; AP_000992.1; -.<br/>DR   RefSeq; NP_414874.1; -.</pre>
 <p>
  New format:
 </p>
 <pre>DR   RefSeq; AP_000992.1; AC_000091.1.<br/>DR   RefSeq; NP_414874.1; NC_000913.2.</pre>
 <h3>
  Changes to
  <a href="http://www.uniprot.org/docs/?keywlist">
   keywords
  </a>
 </h3>
 New keywords:
 <ul>
  <li>
   <a href="http://www.uniprot.org/keywords/KW-1178">
    Actin-dependent active transport of viral material
   </a>
  </li>
  <li>
   <a href="http://www.uniprot.org/keywords/KW-1157">
    Cap snatching
   </a>
  </li>
  <li>
   <a href="http://www.uniprot.org/keywords/KW-1166">
    Caveolae-mediated endocytosis of virus by host
   </a>
  </li>
  <li>
   <a href="http://www.uniprot.org/keywords/KW-1167">
    Clathrin- and caveolae-independent endocytosis of virus by host
   </a>
  </li>
  <li>
   <a href="http://www.uniprot.org/keywords/KW-1165">
    Clathrin-mediated endocytosis of virus by host
   </a>
  </li>
  <li>
   <a href="http://www.uniprot.org/keywords/KW-1176">
    Cytoplasmic active transport of viral material
   </a>
  </li>
  <li>
   <a href="http://www.uniprot.org/keywords/KW-1169">
    Fusion of virus membrane with host cell membrane
   </a>
  </li>
  <li>
   <a href="http://www.uniprot.org/keywords/KW-1170">
    Fusion of virus membrane with host endosomal membrane
   </a>
  </li>
  <li>
   <a href="http://www.uniprot.org/keywords/KW-1168">
    Fusion of virus membrane with host membrane
   </a>
  </li>
  <li>
   <a href="http://www.uniprot.org/keywords/KW-1139">
    Helical capsid protein
   </a>
  </li>
  <li>
   <a href="http://www.uniprot.org/keywords/KW-1183">
    Host cell receptor for virus entry
   </a>
  </li>
  <li>
   <a href="http://www.uniprot.org/keywords/KW-1160">
    Initiation of viral infection
   </a>
  </li>
  <li>
   <a href="http://www.uniprot.org/keywords/KW-1153">
    Inner capsid protein
   </a>
  </li>
  <li>
   <a href="http://www.uniprot.org/keywords/KW-1154">
    Intermediate capsid protein
   </a>
  </li>
  <li>
   <a href="http://www.uniprot.org/keywords/KW-1177">
    Microtubule-dependent active transport of viral material
   </a>
  </li>
  <li>
   <a href="http://www.uniprot.org/keywords/KW-1152">
    Outer capsid protein
   </a>
  </li>
  <li>
   <a href="http://www.uniprot.org/keywords/KW-1175">
    Pilus-mediated viral adsorption onto host cell
   </a>
  </li>
  <li>
   <a href="http://www.uniprot.org/keywords/KW-1172">
    Pore-mediated penetration of viral genome into host cell
   </a>
  </li>
  <li>
   <a href="http://www.uniprot.org/keywords/KW-1179">
    Provirus integration
   </a>
  </li>
  <li>
   <a href="http://www.uniprot.org/keywords/KW-1164">
    Receptor mediated endocytosis of virus by host
   </a>
  </li>
  <li>
   <a href="http://www.uniprot.org/keywords/KW-1159">
    <span class="caps">
     RNA
    </span>
    suppression of termination
   </a>
  </li>
  <li>
   <a href="http://www.uniprot.org/keywords/KW-1158">
    <span class="caps">
     RNA
    </span>
    termination-reinitiation
   </a>
  </li>
  <li>
   <a href="http://www.uniprot.org/keywords/KW-1156">
    <span class="caps">
     RNA
    </span>
    translational shunting
   </a>
  </li>
  <li>
   <a href="http://www.uniprot.org/keywords/KW-1180">
    Syncytium formation induced by viral infection
   </a>
  </li>
  <li>
   <a href="http://www.uniprot.org/keywords/KW-1140">
    T=1 icosahedral capsid protein
   </a>
  </li>
  <li>
   <a href="http://www.uniprot.org/keywords/KW-1141">
    T=2* icosahedral capsid protein
   </a>
  </li>
  <li>
   <a href="http://www.uniprot.org/keywords/KW-1142">
    T=3 icosahedral capsid protein
   </a>
  </li>
  <li>
   <a href="http://www.uniprot.org/keywords/KW-1143">
    T=pseudo3 icosahedral capsid protein
   </a>
  </li>
  <li>
   <a href="http://www.uniprot.org/keywords/KW-1144">
    T=4 icosahedral capsid protein
   </a>
  </li>
  <li>
   <a href="http://www.uniprot.org/keywords/KW-1145">
    T=7 icosahedral capsid protein
   </a>
  </li>
  <li>
   <a href="http://www.uniprot.org/keywords/KW-1146">
    T=13 icosahedral capsid protein
   </a>
  </li>
  <li>
   <a href="http://www.uniprot.org/keywords/KW-1147">
    T=16 icosahedral capsid protein
   </a>
  </li>
  <li>
   <a href="http://www.uniprot.org/keywords/KW-1148">
    T=25 icosahedral capsid protein
   </a>
  </li>
  <li>
   <a href="http://www.uniprot.org/keywords/KW-1149">
    T=147 icosahedral capsid protein
   </a>
  </li>
  <li>
   <a href="http://www.uniprot.org/keywords/KW-1150">
    T=169 icosahedral capsid protein
   </a>
  </li>
  <li>
   <a href="http://www.uniprot.org/keywords/KW-1151">
    T=219 icosahedral capsid protein
   </a>
  </li>
  <li>
   <a href="http://www.uniprot.org/keywords/KW-1155">
    Translational shunt
   </a>
  </li>
  <li>
   <a href="http://www.uniprot.org/keywords/KW-1161">
    Viral attachment to host cell
   </a>
  </li>
  <li>
   <a href="http://www.uniprot.org/keywords/KW-1171">
    Viral genome injection through bacterial membranes
   </a>
  </li>
  <li>
   <a href="http://www.uniprot.org/keywords/KW-1182">
    Viral ionic channel
   </a>
  </li>
  <li>
   <a href="http://www.uniprot.org/keywords/KW-1162">
    Viral penetration into host cytoplasm
   </a>
  </li>
  <li>
   <a href="http://www.uniprot.org/keywords/KW-1163">
    Viral penetration into host nucleus
   </a>
  </li>
  <li>
   <a href="http://www.uniprot.org/keywords/KW-1174">
    Viral penetration via lysis of host organellar membrane
   </a>
  </li>
  <li>
   <a href="http://www.uniprot.org/keywords/KW-1173">
    Viral penetration via permeabilization of host organellar membrane
   </a>
  </li>
  <li>
   <a href="http://www.uniprot.org/keywords/KW-1181">
    Viral primary envelope fusion with host outer nuclear membrane
   </a>
  </li>
 </ul>
 <h3>
  Changes in
  <a href="http://www.uniprot.org/docs/?subcell">
   subcellular location controlled vocabulary
  </a>
 </h3>
 New subcellular locations:
 <ul>
  <li>
   <a href="http://www.uniprot.org/locations/SL-0474">
    Host glyoxysome
   </a>
  </li>
  <li>
   <a href="http://www.uniprot.org/locations/SL-0475">
    Host peroxisome
   </a>
  </li>
 </ul>
 <h3>
  Changes in the
  <a href="http://www.uniprot.org/docs/ptmlist">
   controlled vocabulary for PTMs
  </a>
 </h3>
 New term for the feature key ‘Modified residue’ (‘MOD_RES’ in the flat file):
 <ul>
  <li>
   3’-nitrotyrosine
  </li>
 </ul>
</div>