<div>
 <h2>
  Headline
 </h2>
 <h3>
  Pupylation: a ubiquitin-like tagging system in bacteria
 </h3>
 <p>
  While ubiquitin has been known for decades as a post-translationally conjugated protein degradation tag in eukaryotes, the first identified prokaryotic protein that is functionally analogous to ubiquitin, i.e. prokaryotic ubiquitin-like protein
  <a href="http://www.uniprot.org/uniprot/O33246">
   Pup
  </a>
  , has only recently been
  <a href="http://www.ncbi.nlm.nih.gov/pubmed/18832610,19028679">
   discovered in mycobacteria
  </a>
  .
 </p>
 <p>
  <a href="http://www.uniprot.org/uniprot/O33246">
   Pup
  </a>
  (64 residues) and ubiquitin (76 residues) show neither structural nor sequence homology except for a GG motif near or at the C-terminus. Although both Pup and ubiquitin are attached to the epsilon-amino group of lysine side chains in substrates and target the substrates for degradation by the proteasome, the enzymology of ubiquitination and pupylation and the chemistry of the coupling reaction appear completely different. Ubiquitin is coupled to substrates via the carboxyl group of its C-terminal glycine in a multistep reaction involving several enzymes (see
  <a href="http://www.uniprot.org/news/2010/10/05/release">
   release 2010_10 headline
  </a>
  ). In the mycobacterial pupylation pathway, the C-terminal glutamine of Pup is first deamidated to glutamate by
  <a href="http://www.uniprot.org/uniprot/O33247">
   Dop
  </a>
  (deamidase of Pup) after which it is ligated to the substrate lysine of target proteins by proteasome accessory factor A (
  <a href="http://www.uniprot.org/uniprot/P64943">
   PafA
  </a>
  ). Neither
  <a href="http://www.uniprot.org/uniprot/O33247">
   Dop
  </a>
  nor
  <a href="http://www.uniprot.org/uniprot/P64943">
   PafA
  </a>
  is similar to ubiquitin-activating enzymes. The covalently Pup-modified protein is then recognized and unfolded by the proteasomal ATPase
  <a href="http://www.uniprot.org/uniprot/P63345">
   Mpa
  </a>
  and degraded by the proteasome. The very
  <a href="http://www.ncbi.nlm.nih.gov/pubmed/20705495">
   recent discovery of a depupylase activity
  </a>
  provided by
  <a href="http://www.uniprot.org/uniprot/O33247">
   Dop
  </a>
  , able to remove conjugated Pup from target proteins in a manner analogous to the deconjugation of ubiquitin from eukaryotic proteins, strengthens the parallels between the Pup- and ubiquitin-tagging systems of prokaryotes and eukaryotes, respectively. However Mycobacterium appears to have a single Pup ligase to mediate all pupylation and a single depupylase for all pupylated substrates, in contrast to the human genome that encodes hundreds of ubiquitin ligases and dozens of deubiquitinating enzymes.
 </p>
 <p>
  Taken together, prokaryotes and eukaryotes appear to have developed distinct but parallel mechanisms to regulate protein stability by a similar proteolytic machinery: the proteasome found in all eukaryotes and archaea, and in bacteria of the class Actinobacteria, including the genus Mycobacterium.
 </p>
 <p>
  All the
  <a href="http://www.uniprot.org/uniprot/?query=keyword:833+AND+reviewed:yes+AND+taxonomy:2">
   known pupylation-related proteins
  </a>
  in bacteria have now been annotated in UniProtKB/Swiss-Prot.
 </p>
 <h2>
  UniProtKB news
 </h2>
 <h3>
  Changes concerning
  <a href="http://www.uniprot.org/docs/?keywlist">
   keywords
  </a>
 </h3>
 New keywords:
 <ul>
  <li>
   <a href="http://www.uniprot.org/keywords/KW-1135">
    Mitochondrion nucleoid
   </a>
  </li>
  <li>
   <a href="http://www.uniprot.org/keywords/KW-1136">
    Bradyzoite
   </a>
  </li>
  <li>
   <a href="http://www.uniprot.org/keywords/KW-1137">
    Tachyzoite
   </a>
  </li>
  <li>
   <a href="http://www.uniprot.org/keywords/KW-1138">
    Trophozoite
   </a>
  </li>
 </ul>
 Modified keyword:
 <ul>
  <li>
   Chromosomal protein -&gt;
   <a href="http://www.uniprot.org/keywords/KW-0158">
    Chromosome
   </a>
  </li>
 </ul>
 <h3>
  Changes in
  <a href="http://www.uniprot.org/docs/?subcell">
   subcellular location controlled vocabulary
  </a>
 </h3>
 New subcellular locations:
 <ul>
  <li>
   Filopodium tip
  </li>
  <li>
   Pseudopodium tip
  </li>
  <li>
   Bleb
  </li>
  <li>
   Phagocytic cup
  </li>
 </ul>
 <h3>
  Changes concerning the
  <a href="http://www.uniprot.org/docs/ptmlist">
   controlled vocabulary for PTMs
  </a>
 </h3>
 New terms for the feature key ‘Modified residue’ (‘MOD_RES’ in the flat file):
 <ul>
  <li>
   N,N,N-trimethylserine
  </li>
  <li>
   N,N-dimethylserine
  </li>
  <li>
   N-methylserine
  </li>
  <li>
   CysO-cysteine adduct
  </li>
 </ul>
</div>