<div>
 <h2>
  Headlines
 </h2>
 <h3>
  Nonsense-mediated mRNA decay: To be or not to be… integrated in UniProtKB
 </h3>
 <p>
  It has been known for
  <a href="http://www.ncbi.nlm.nih.gov/pubmed/388431">
   over 30 years
  </a>
  that, in yeast, nonsense mutations reduce mRNA levels and that the strength of the reduction depends on the position of the nonsense codon within the locus. This observation, followed by many others in a great variety of eukaryotic organisms, led to the concept of ‘Nonsense-mediated mRNA decay’ (NMD), ‘a surveillance mechanism that detects and degrades mRNAs with premature termination codons (PTCs), thereby preventing the production of faulty proteins’. The key question was what Mother Nature considers a ‘
  <em>
   premature
  </em>
  ’ stop. For mammals, a rule was established stating that ‘if a termination codon is more than about 50 nucleotides upstream of the final exon, it is a
  <span class="caps">
   PTC
  </span>
  and the mRNA that harbors it will be degraded’ (see
  <a href="http://www.ncbi.nlm.nih.gov/pubmed/9644970">
   Nagy and Maquat, 1998
  </a>
  ). Although we know today that
  <span class="caps">
   NMD
  </span>
  is a much more sophisticated mechanism than previously anticipated (see
  <a href="http://www.ncbi.nlm.nih.gov/pubmed/19859661,19359157,19162024,18524595">
   reviews
  </a>
  ), the ‘50 nucleotide rule’ is still used to predict potential
  <span class="caps">
   NMD
  </span>
  targets and, on this basis, some databases deleted them from their collections. Since many PTCs are generated by alternative splicing (at least
  <a href="http://www.ncbi.nlm.nih.gov/pubmed/12502788">
   one third
  </a>
  of the human alternatively spliced mRNAs contain PTCs), several alternatively spliced isoforms have disappeared from databases, victims of the ‘50 nucleotide rule’.
 </p>
 <p>
  Eukaryotic cells detect
  <span class="caps">
   PTC
  </span>
  during the first round of translation undergone by mRNAs freshly exported from the nucleus. During this ‘pioneer’ round of translation, if the ribosome terminates at a termination codon (TC) in the vicinity of the poly(A) tail,
  <a href="http://www.uniprot.org/uniprot/?query=reviewed:yes+AND+gene:pabpc1">
   <span class="caps">
    PABPC1
   </span>
  </a>
  – a poly(A)-binding protein – sends a signal which promotes proper termination of translation. This results in efficient reinitiation of the ribosome at the 5’ end of the mRNA, and the production of a stable mRNP. If the ribosome terminates at a TC that is too far away from the poly(A) tail for it to receive the
  <a href="http://www.uniprot.org/uniprot/?query=reviewed:yes+AND+gene:pabpc1">
   <span class="caps">
    PABPC1
   </span>
  </a>
  – mediated translation-termination-promoting signal, the
  <a href="http://www.uniprot.org/uniprot/?query=gene:upf1+AND+reviewed:yes">
   <span class="caps">
    UPF1
   </span>
  </a>
  protein binds to the stalled ribosome instead, thereby marking this TC as premature. Subsequently, a
  <span class="caps">
   PTC
  </span>
  -specific protein complex forms around
  <a href="http://www.uniprot.org/uniprot/?query=gene:upf1+AND+reviewed:yes">
   <span class="caps">
    UPF1
   </span>
  </a>
  , promoting
  <a href="http://www.uniprot.org/uniprot/?query=gene:upf1+AND+reviewed:yes">
   <span class="caps">
    UPF1
   </span>
  </a>
  phosphorylation and committing the mRNA to rapid degradation.
 </p>
 <p>
  It is thought that the physical distance, rather than the number of nucleotides, between a TC and the poly(A) tail is a crucial determinant in defining a TC as premature (
  <a href="http://www.ncbi.nlm.nih.gov/pubmed/18447580">
   Eberle et al., 2008
  </a>
  ). This distance depends on the 3D structure of the mRNA 3’
  <span class="caps">
   UTR
  </span>
  . This structure can be modified by altering (1) intramolecular base pairing, (2) interaction of the mRNA with
  <span class="caps">
   RNA
  </span>
  -binding proteins and (3) interactions between the involved proteins through post-translational modifications (PTMs). In other words, it can be regulated in a tissue-specific manner, during development, and by environmental cues.
 </p>
 <p>
  In higher eukaryotes, an additional level of complexity exists which links
  <span class="caps">
   PTC
  </span>
  detection and mRNA splicing. During pre-mRNA processing, the spliceosome removes intron sequences and a set of proteins called the exon-junction complex (EJC) is deposited 20-24 nucleotides upstream of the sites of intron removal. EJCs located within the
  <span class="caps">
   ORF
  </span>
  are removed from the mRNA by elongating ribosomes, and only EJCs located downstream of the TC will still be present when the first ribosome terminates. In organisms producing a large number of
  <span class="caps">
   PTC
  </span>
  -containing mRNAs by extensive alternative pre-mRNA splicing, such as humans, the
  <span class="caps">
   EJC
  </span>
  may have evolved to facilitate efficient recognition and degradation of these transcripts. An
  <span class="caps">
   EJC
  </span>
  downstream of a TC functions as an
  <span class="caps">
   NMD
  </span>
  enhancer by shortening the time window between
  <span class="caps">
   UPF1
  </span>
  binding and its phosphorylation, hence promoting mRNA degradation.
 </p>
 <p>
  <span class="caps">
   NMD
  </span>
  rarely downregulates the expression of a transcript completely. More commonly, 10-30% of the
  <span class="caps">
   PTC
  </span>
  -containing transcripts survive and may allow the production of physiologically relevant levels of protein products (
  <a href="http://www.ncbi.nlm.nih.gov/pubmed/15059251">
   Neu-Yilik et al., 2004
  </a>
  ). This is why in UniProtKB, we favour a conservative approach when dealing with protein isoforms predicted to be encoded by an
  <span class="caps">
   NMD
  </span>
  target mRNA. We do not delete them from the database, but rather tag them with the comment: ‘May be produced at very low levels due to a premature stop codon in the mRNA, leading to nonsense-mediated mRNA decay.’ For instance, in entry
  <a href="http://www.uniprot.org/uniprot/Q9HB09">
   <span class="caps">
    Q9HB09
   </span>
  </a>
  (human Bcl-2-like protein 12),
  <a href="http://www.uniprot.org/uniprot/Q9HB09#section_alternative">
   2 isoforms
  </a>
  are described, one of which has been predicted to be an
  <span class="caps">
   NMD
  </span>
  target by
  <a href="http://www.ncbi.nlm.nih.gov/pubmed/14759258">
   Hillman et al., 2004
  </a>
  (see also the
  <a href="http://www.uniprot.org/uniprot/Q9HB09#section_ref">
   ‘References’
  </a>
  section of the entry). In some cases, despite the presence of a
  <span class="caps">
   PTC
  </span>
  in the encoding mRNA, the isoform produced seems to be the predominant form, at least in some tissues (see human Gamma-aminobutyric acid type B receptor subunit 1
  <a href="http://www.uniprot.org/uniprot/Q9UBS5#section_alternative">
   isoform 1E
  </a>
  in entry
  <a href="http://www.uniprot.org/uniprot/Q9UBS5">
   <span class="caps">
    Q9UBS5
   </span>
  </a>
  ).
 </p>
 <p>
  Currently in UniProtKB/Swiss-Prot, over
  <a href="http://www.uniprot.org/uniprot/?query=%22nonsense+mediated+mrna+decay%22+AND+reviewed:yes+NOT+keyword:KW-0866">
   300 protein entries
  </a>
  describe isoforms that could be produced at low levels due to
  <span class="caps">
   NMD
  </span>
  .
  <a href="http://www.uniprot.org/uniprot/?query=keyword:KW-0866">
   228 proteins
  </a>
  from different species are directly involved in the
  <span class="caps">
   NMD
  </span>
  process itself and can be retrieved from UniProtKB with the keyword
  <a href="http://www.uniprot.org/keywords/KW-0866">
   ‘Nonsense mediated mRNA decay’
  </a>
  .
 </p>
 <h2>
  UniProtKB News
 </h2>
 <h3>
  Cross-references to
  <span class="caps">
   UCD
  </span>
  -2DPAGE
 </h3>
 <p>
  Cross-references have been added to the University College Dublin 2-DE Proteome Database, (
  <a href="http://proteomics-portal.ucd.ie:8082/cgi-bin/2d/2d.cgi">
   <span class="caps">
    UCD
   </span>
   -2DPAGE
  </a>
  ). The database
  <span class="caps">
   HSC
  </span>
  -2DPAGE,previously hosted at Harefield Hospital (and previously also cross-referenced from UniProtKB/Swiss-Prot), has been integrated into
  <span class="caps">
   UCD
  </span>
  -2DPAGE.
  <span class="caps">
   UCD
  </span>
  -2DPAGE currently contains data from
  <em>
   Canis familiaris
  </em>
  (dog),
  <em>
   Homo sapiens
  </em>
  (human),
  <em>
   Mus musculus
  </em>
  (mouse),
  <em>
   Rattus norvegicus
  </em>
  (rat) and
  <em>
   Saccharomyces cerevisiae
  </em>
  (baker’s yeast).
 </p>
 <p>
  <span class="caps">
   UCD
  </span>
  -2DPAGE is available at
  <a href="http://proteomics-portal.ucd.ie:8082/cgi-bin/2d/2d.cgi">
   http://proteomics-portal.ucd.ie:8082/cgi-bin/2d/2d.cgi
  </a>
  .
 </p>
 <p>
  The format of the explicit links in the flat file is:
 </p>
 <table>
  <tr>
   <th>
    Resource abbreviation
   </th>
   <td>
    <span class="caps">
     UCD
    </span>
    -2DPAGE
   </td>
  </tr>
  <tr>
   <th>
    Resource identifier
   </th>
   <td>
    <span class="caps">
     UCD
    </span>
    -2DPAGE accession number (in most cases the primary UniProtKB accession number)
   </td>
  </tr>
  <tr>
   <th>
    Examples
   </th>
   <td>
    <a href="http://www.uniprot.org/uniprot/P02648#section_x-ref">
     <span class="caps">
      P02648
     </span>
    </a>
    :
    <pre>DR   UCD-2DPAGE; P02648; -.</pre>
    <a href="http://www.uniprot.org/uniprot/O75112#section_x-ref">
     <span class="caps">
      O75112
     </span>
    </a>
    :
    <pre>DR   UCD-2DPAGE; O75112; -.
DR   UCD-2DPAGE; Q9Y4Z5; -.</pre>
   </td>
  </tr>
 </table>
 <p>
  <a href="http://www.uniprot.org/uniprot/?query=database:ucd-2dpage">
   Show all the entries having a cross-reference to
   <span class="caps">
    UCD
   </span>
   -2DPAGE
  </a>
  .
 </p>
 <h3>
  Changes concerning cross-references to
  <span class="caps">
   HSC
  </span>
  -2DPAGE
 </h3>
 <p>
  Cross-references to
  <span class="caps">
   HSC
  </span>
  -2DPAGE have been removed.
 </p>
 <h3>
  Changes concerning
  <a href="http://www.uniprot.org/docs/?keywlist">
   keywords
  </a>
 </h3>
 New keyword:
 <ul>
  <li>
   <a href="http://www.uniprot.org/keywords/KW-1065">
    Osteogenesis imperfecta
   </a>
  </li>
 </ul>
 <h3>
  Changes in
  <a href="http://www.uniprot.org/docs/?subcell">
   subcellular location controlled vocabulary
  </a>
 </h3>
 New subcellular location:
 <ul>
  <li>
   <a href="http://www.uniprot.org/locations/SL-0460">
    Kinocilium
   </a>
  </li>
 </ul>
 <h3>
  Changes concerning the
  <a href="http://www.uniprot.org/docs/ptmlist">
   controlled vocabulary for PTMs
  </a>
 </h3>
 New terms for the feature key ‘Cross-link’ (‘CROSSLNK’ in the flat file):
 <ul>
  <li>
   Glycyl serine ester (interchain with G-Cter in ubiquitin)
  </li>
  <li>
   Glycyl threonine ester (interchain with G-Cter in ubiquitin)
  </li>
 </ul>
 New term for the feature key ‘Modified residue’ (‘MOD_RES’ in the flat file):
 <ul>
  <li>
   2-(S-cysteinyl)pyruvic acid O-phosphothioketal
  </li>
 </ul>
</div>