<div>
 <h2>
  Headline
 </h2>
 <h3>
  Killing myself softly – Bacterial and Archaeal Type II Toxin-Antitoxin modules (TA)
 </h3>
 <p>
  Bacteria produce many kinds of toxins that attack other organisms: cholera toxin, botulinum toxin, aerolysins, insecticidal toxins, extracellular proteases, to name just a few. In the past 5 years it has become apparent that most free-living bacteria produce
  <a href="http://www.ncbi.nlm.nih.gov/pubmed/15718296">
   another kind of internal toxin
  </a>
  . These toxins are almost always encoded as bicistronic antitoxin-toxin operons (TA module), where the antitoxin is unstable and neutralizes the toxin. In type I and
  <span class="caps">
   III
  </span>
  systems the antitoxin is a small
  <span class="caps">
   RNA
  </span>
  , while in type II systems the antitoxin is a protein. If the antitoxin levels decrease then the toxin levels increase and toxic effects can be seen at the cellular level. In most type II cases studied, the antitoxin acts as an autoregulator, repressing transcription; frequently but not always, the toxin acts as a corepressor. First identified in bacteriophage and on plasmids, where their role is clearly in plasmid or phage maintenance, the role of the chromosomally encoded toxins in bacteria is hotly debated. Proposed functions include maintenance of mobile genetic elements, programmed cell death, induction of persistence (dormancy), stress response, virulence promotion in a host, and regulation of biofilm formation. The toxin’s role may in fact depend on the physiology of the organism in question. Although they are widespread in Archaea, no function for these toxins has been shown in vivo.
 </p>
 <p>
  There are many toxin families. The best characterized so far are the bacterial ribonuclease toxins which belong to the MazF, RelE, MqsR, HigB, YoeB and VapC families. Most of these toxins degrade mRNA;
  <a href="http://www.uniprot.org/uniprot/P0AE70">
   some are sequence-specific
  </a>
  ,
  <a href="http://www.uniprot.org/uniprot/P0C077">
   some work only in association with ribosomes
  </a>
  , while for others the mode is unknown.
  <a href="http://www.uniprot.org/uniprot/Q8ZM86">
   VapC
  </a>
  has been shown to degrade the anticodon loop of tRNAfMet. Other cellular functions are also toxin targets;
  <span class="caps">
   DNA
  </span>
  gyrase is targeted by the
  <a href="http://www.uniprot.org/uniprot/Q79EC5">
   ParE
  </a>
  toxin,
  <a href="http://www.uniprot.org/uniprot/P23874">
   HipA
  </a>
  toxin probably acts by inappropriately phosphorylating cellular targets,
  <a href="http://www.uniprot.org/uniprot/P0AGL5">
   RatA
  </a>
  blocks ribosomal subunit association,
  <a href="http://www.uniprot.org/uniprot/Q97QZ1">
   PezT/zeta
  </a>
  toxin corrupts peptidoglycan synthesis and
  <a href="http://www.uniprot.org/uniprot/P64524">
   CbtA
  </a>
  (formerly YeeV) toxin binds
  <a href="http://www.uniprot.org/uniprot/P0A9A6">
   FtsZ
  </a>
  and
  <a href="http://www.uniprot.org/uniprot/P0A9X4">
   MreB
  </a>
  , inhibiting them, possibly simultaneously. While the toxins form distinct families, their cognate antitoxins do not, although almost all of them have a
  <span class="caps">
   DNA
  </span>
  -binding domain, in accordance with their probable role in operon regulation. To further complicate matters, in
  <em>
   Mycobacterium tuberculosis
  </em>
  <a href="http://www.ncbi.nlm.nih.gov/pubmed/20876537">
   H37Rv cross-talk between toxins and some non-cognate antitoxins
  </a>
  has been seen, while
  <a href="http://www.ncbi.nlm.nih.gov/pubmed/20487277">
   in
   <em>
    Caulobacter crescentus
   </em>
   such cross-talk does not occur
  </a>
  . Additionally, potential new toxins are detected quite frequently.
 </p>
 <p>
  We recently performed a major update of many of the type
  <span class="caps">
   II TA
  </span>
  families in UniProtKB/Swiss-Prot, with particular attention given to the model organisms
  <em>
   Mycobacterium tuberculosis
  </em>
  strain H37Rv and
  <em>
   E.coli
  </em>
  K12 /
  <span class="caps">
   MG1655
  </span>
  . 65 TA modules have been annotated in
  <em>
   M. tuberculosis
  </em>
  and 15 in
  <em>
   E.coli
  </em>
  ; gene names for
  <em>
   M.tuberculosis
  </em>
  were assigned in collaboration with the TubercuList database. Interestingly, TA modules are more abundant in pathogens than in related non-pathogenic strains. Hence
  <em>
   Mycobacterium smegmatis
  </em>
  , a non-pathogenic mycobacterium, is only predicted to
  <a href="http://www.ncbi.nlm.nih.gov/pubmed/20011113">
   encode 3 TA modules
  </a>
  . Since January 2011 the mode of action of at least 4 toxin families has been elucidated (
  <a href="http://www.ncbi.nlm.nih.gov/pubmed/21166897">
   CbtA
  </a>
  ,
  <a href="http://www.ncbi.nlm.nih.gov/pubmed/21445328">
   PezT/zeta
  </a>
  ,
  <a href="http://www.ncbi.nlm.nih.gov/pubmed/21323758">
   RatA
  </a>
  and
  <a href="http://www.ncbi.nlm.nih.gov/pubmed/21502523">
   VapC
  </a>
  ).
 </p>
 <p>
  Although the PezT/zeta toxin and associated antitoxin module have not been predicted to exist in
  <em>
   M.tuberculosis
  </em>
  , there are indeed loci belonging to this TA module encoded in the genome. This is currently such a hot topic that integrating the data will keep us busy for quite a while yet.
 </p>
 <p>
  All manually annotated type
  <span class="caps">
   II TA
  </span>
  module entries can be retrieved from UniProtKB/Swiss-Prot using the query
  <a href='http://www.uniprot.org/uniprot/?query=annotation:(type:function+"toxin-antitoxin+(TA)+module")'>
   toxin-antitoxin (TA) module
  </a>
  .
 </p>
 <h2>
  UniProtKB News
 </h2>
 <h3>
  Provision of complete proteome data sets for
  <span class="caps">
   IPI
  </span>
  species by UniProt
 </h3>
 <p>
  Complete proteome data sets are now available for download from the
  <a href="ftp://ftp.uniprot.org/pub/databases/uniprot/current_release/knowledgebase/proteomes/">
   <span class="caps">
    FTP
   </span>
  </a>
  and
  <a href="http://www.uniprot.org/taxonomy/complete-proteomes">
   web
  </a>
  sites for the species in the International Protein Index (IPI) which is scheduled for closure this year.
  <span class="caps">
   IPI
  </span>
  is an integrated database which clusters protein sequences from different databases to provide non-redundant complete data sets for selected higher eukaryotic organisms. Since it was launched in 2001,
  <span class="caps">
   IPI
  </span>
  has covered the gaps in the gene predictions between different databases, but since then the situation has improved for many of the most-studied genomes. This is due to a close collaboration between Ensembl, RefSeq and UniProt which aims to provide a standard set of gene predictions for the genomes of interest. These new complete proteomes will therefore provide high coverage complete proteomes for
  <span class="caps">
   IPI
  </span>
  users. The complete UniProtKB proteomes will be based on existing UniProtKB sequences supplemented by missing high quality predictions imported from Ensembl.
 </p>
 <p>
  For
  <em>
   Homo sapiens
  </em>
  , a first pass annotation of the complete proteome was completed by UniProt in 2008 and all entries were incorporated into UniProtKB/Swiss-Prot. Within this UniProtKB/Swiss-Prot complete
  <em>
   H. sapiens
  </em>
  proteome, approximately 20,000 putative protein-coding genes are represented by one canonical protein sequence, with some entries describing multiple isoform sequences. Since its initial release, the UniProtKB/Swiss-Prot complete
  <em>
   H. sapiens
  </em>
  proteome has been extensively curated and the Ensembl cross-references -mapped based on sequence identity -are in the process of being manually verified. All predicted protein sequences from Ensembl (except fragments) that were found to be absent from the UniProtKB/Swiss-Prot complete
  <em>
   H. sapiens
  </em>
  proteome were imported into the unreviewed component of UniProtKB, UniProtKB/TrEMBL (see
  <a href="http://www.uniprot.org/news/2011/05/03/release">
   release 2011_05 headline
  </a>
  ). These imported UniProtKB/TrEMBL entries were tagged with the
  <a href="http://www.uniprot.org/keywords/KW-0181">
   keyword ‘Complete proteome’
  </a>
  . The aim of this import was to increase the coverage of the existing complete proteome, by supplementing it with those Ensembl protein sequences that had no UniProtKB counterpart. The resulting UniProtKB complete
  <em>
   H. sapiens
  </em>
  proteome includes both reviewed sequences from UniProtKB/Swiss-Prot (equivalent to an updated version of the complete
  <em>
   H. sapiens
  </em>
  proteome completed in 2008), now supplemented by unreviewed sequences from UniProtKB/TrEMBL. This process will enable the synchronization of the UniProt set with the
  <a href="http://www.ncbi.nlm.nih.gov/CCDS/">
   <span class="caps">
    CCDS
   </span>
   project
  </a>
  . This version of the complete
  <em>
   H. sapiens
  </em>
  proteome provides higher sequence coverage than the preceding version, but now includes sequences that have not been manually reviewed. Users can choose to opt either for this
  <a href="http://www.uniprot.org/uniprot/?query=organism:9606+keyword:181">
   expanded complete
   <em>
    H. sapiens
   </em>
   proteome
  </a>
  or a
  <a href="http://www.uniprot.org/uniprot/?query=organism:9606+keyword:181+reviewed:yes">
   reduced version that derives exclusively from UniProtKB/Swiss-Prot
  </a>
  .
 </p>
 <p>
  For the other
  <span class="caps">
   IPI
  </span>
  species (
  <a href="http://www.uniprot.org/uniprot/?query=taxonomy:10090+keyword:181">
   mouse
  </a>
  ,
  <a href="http://www.uniprot.org/uniprot/?query=taxonomy:10116+keyword:181">
   rat
  </a>
  ,
  <a href="http://www.uniprot.org/uniprot/?query=taxonomy:9031+keyword:181">
   chicken
  </a>
  ,
  <a href="http://www.uniprot.org/uniprot/?query=taxonomy:7955+keyword:181">
   zebrafish
  </a>
  ,
  <a href="http://www.uniprot.org/uniprot/?query=taxonomy:9913+keyword:181">
   cow
  </a>
  and
  <a href="http://www.uniprot.org/uniprot/?query=taxonomy:9615+keyword:181">
   dog
  </a>
  ), we added the keyword ‘Complete proteome’ to the existing UniProtKB/Swiss-Prot entries. We identified those entries in UniProtKB/TrEMBL which mapped to the complete genome in Ensembl and imported the predicted protein sequences (except fragments) from Ensembl which were found not to be present in UniProtKB. The keyword ‘Complete proteome’ was also added to these entries. As for the human counterpart, these proteomes can now be easily retrieved using this keyword. The Ensembl cross-references have been added to the UniProtKB entries on the basis of 100% sequence identity over their full length.
 </p>
 <p>
  We will expand the coverage to other species of interest in the near future and expect this will be very useful for our users as it will eliminate the need to combine data from different databases.
 </p>
 <h3>
  Changes concerning the
  <a href="http://www.uniprot.org/docs/ptmlist">
   controlled vocabulary for PTMs
  </a>
 </h3>
 New terms for the feature key ‘Cross-link’ (‘CROSSLNK’ in the flat file):
 <ul>
  <li>
   2-(3-methylbutanoyl)-5-hydroxy-oxazole-4-carbothionic acid (Leu-Cys)
  </li>
  <li>
   Proline 5-hydroxy-oxazole-4-carbothionic acid (Pro-Cys)
  </li>
 </ul>
 New term for the feature key ‘Lipidation’ (‘LIPID’ in the flat file):
 <ul>
  <li>
   S-(15-deoxy-Delta12,14-prostaglandin J2-9-yl)cysteine
  </li>
 </ul>
 <h3>
  Changes to
  <a href="http://www.uniprot.org/docs/keywlist">
   keywords
  </a>
 </h3>
 Modified keyword:
 <ul>
  <li>
   Provirus integration -&gt;
   <a href="http://www.uniprot.org/keywords/KW-1179">
    Viral genome integration
   </a>
  </li>
 </ul>
</div>