<div>
 <h2>
  Headline
 </h2>
 <h3>
  The sound of silence
 </h3>
 <p>
  Cytosine methylation is the major and best characterized epigenetic modification of metazoan
  <span class="caps">
   DNA
  </span>
  . It is implicated in long-term gene silencing, X chromosome inactivation, genomic imprinting, etc. 5-methylcytosine (5mC) is recognized by methyl-binding proteins (
  <a href="http://www.uniprot.org/uniprot/?query=name:%22Methyl-CpG-binding+domain+protein%22+reviewed:yes">
   MBDs
  </a>
  ), that in turn recruit repressive histone modifiers, such as
  <a href="http://www.uniprot.org/uniprot/?query=family:%22histone-lysine+methyltransferase+family%22">
   <span class="caps">
    H3K9
   </span>
   methyltransferases
  </a>
  , to establish a heterochromatin state.
 </p>
 <p>
  Cytosine base methylation is catalyzed by the
  <a href="http://www.uniprot.org/uniprot/?query=family:%22C5-methyltransferase+family%22+reviewed:yes">
   C5-methyltransferase
  </a>
  enzyme family.
  <a href="http://www.uniprot.org/uniprot/?query=(gene:DNMT3A+OR+gene:DNMT3B)+AND+reviewed:yes">
   <span class="caps">
    DNMT3A
   </span>
   and
   <span class="caps">
    DNMT3B
   </span>
  </a>
  methylate
  <span class="caps">
   DNA
  </span>
  de novo.
  <a href="http://www.uniprot.org/uniprot/?query=gene:Dnmt1+reviewed:yes">
   <span class="caps">
    DNMT1
   </span>
  </a>
  maintains the methylation status across cell divisions. In the absence of
  <span class="caps">
   DNMT1
  </span>
  activity,
  <span class="caps">
   DNA
  </span>
  methylation is progressively lost since methylation is not replicated onto the newly synthesized strand, leading to passive
  <span class="caps">
   DNA
  </span>
  demethylation.
 </p>
 <p>
  However, passive
  <span class="caps">
   DNA
  </span>
  demethylation cannot account for rapid demethylation that occurs in the paternal genome in the zygote within the first 4 hours following fertilization or that observed in primordial germ cells, both of which are independent of
  <span class="caps">
   DNA
  </span>
  replication. While
  <a href="http://www.uniprot.org/uniprot/?query=family:%22DNA+glycosylase+family+DEMETER+subfamily%22">
   demethylases
  </a>
  have been identified in
  <em>
   Arabidopsis thaliana
  </em>
  , the mechanism of active demethylation in mammals remained elusive (
  <a href="http://www.ncbi.nlm.nih.gov/pubmed/21811096,20683471">
   reviews
  </a>
  ).
  <a href="http://www.ncbi.nlm.nih.gov/pubmed/21778364,21817016,21490601,21460036,21750410,21451524,21552279,21460836,21817016,21295276,21321204,21722948,21278727,21496894">
   2011
  </a>
  has unveiled the central role played by
  <a href="http://www.uniprot.org/uniprot/?query=gene:(tet1+or+tet2+or+tet3)+and+reviewed:yes">
   <span class="caps">
    TET
   </span>
   family members
  </a>
  . These enzymes have already been shown to
  <a href="http://www.ncbi.nlm.nih.gov/pubmed/19372391" title="5hmC">
   to catalyze the conversion of 5mC into 5-hydroxymethylcytosine
  </a>
  . 5hmC can be further processed, either by G/T mismatch-specific thymine
  <span class="caps">
   DNA
  </span>
  glycosylase (TDG) or by deamination enzymes, such as
  <a href="http://www.uniprot.org/uniprot/?query=gene:apobec1+reviewed:yes">
   <span class="caps">
    APOBEC1
   </span>
  </a>
  and
  <a href="http://www.uniprot.org/uniprot/?query=gene:aicda+reviewed:yes">
   <span class="caps">
    AICDA
   </span>
   /AID
  </a>
  , and eventually removed and replaced by unmodified cytosine by base excision repair mechanism.
 </p>
 <p>
  Interestingly,
  <a href="http://www.uniprot.org/uniprot/?query=gene:TET1+AND+reviewed:yes">
   <span class="caps">
    TET1
   </span>
  </a>
  has a
  <a href="http://www.ncbi.nlm.nih.gov/pubmed/21490601">
   role in transcriptional repression
  </a>
  , independently of its enzymatic activity. It binds a significant proportion of Polycomb group target genes and associates and colocalizes with the
  <a href="http://www.uniprot.org/uniprot/?query=gene:SIN3A+reviewed:yes">
   <span class="caps">
    SIN3A
   </span>
  </a>
  co-repressor complex.
 </p>
 <p>
  These new exciting data pave the way for understanding transcriptional fine-tuning during embryonic development, as well as in adult organisms and will keep us busy updating UniProtKB for quite a while.
 </p>
 <h2>
  UniProtKB news
 </h2>
 <h3>
  Changes to
  <a href="http://www.uniprot.org/docs/keywlist">
   keywords
  </a>
 </h3>
 Deleted keyword:
 <ul>
  <li>
   <a href="http://www.uniprot.org/keywords/KW-1019">
    Virus reference strain
   </a>
  </li>
 </ul>
</div>