<div>
 <h2>
  Headline
 </h2>
 <h3>
  Complete proteomes for
  <em>
   Homo sapiens
  </em>
  and
  <em>
   Mus musculus
  </em>
 </h3>
 <p>
  With the imminent closure of the International Protein Index (IPI), UniProt has pledged to provide comprehensive and non-redundant complete proteomes for all species that are currently covered by this soon to be defunct resource. With this release of UniProtKB, we provide the first version of the complete proteome for
  <em>
   Mus musculus
  </em>
  and an updated version of the
  <em>
   Homo sapiens
  </em>
  set.
 </p>
 <p>
  We describe here how each of these complete proteomes is produced, and outline their major characteristics.
 </p>
 <p>
  For
  <em>
   Homo sapiens
  </em>
  , a first pass annotation of the complete proteome was completed by UniProt in 2008 and all entries were incorporated into UniProtKB/Swiss-Prot. Within this UniProtKB/Swiss-Prot complete
  <em>
   H. sapiens
  </em>
  proteome, approximately 20,000 putative protein-coding genes are represented by one canonical protein sequence, with some entries describing multiple isoform sequences. Since its initial release, the UniProtKB/Swiss-Prot complete
  <em>
   H. sapiens
  </em>
  proteome has been extensively curated and the Ensembl cross-references – mapped based on sequence identity – are in the process of being manually verified. All predicted protein sequences from Ensembl (except fragments) that were found to be absent from the UniProtKB/Swiss-Prot complete
  <em>
   H. sapiens
  </em>
  proteome were imported into the unreviewed component of UniProtKB, UniProtKB/TrEMBL. These imported UniProtKB/TrEMBL entries were tagged with the
  <a href="http://www.uniprot.org/keywords/KW-0181">
   keyword ‘Complete proteome’
  </a>
  . The aim of this import was to increase the coverage of the existing complete proteome, by supplementing it with those Ensembl protein sequences that had no UniProtKB counterpart. The resulting
  <a href="http://www.uniprot.org/uniprot/?query=organism:9606+keyword:181">
   UniProtKB complete
   <em>
    H. sapiens
   </em>
   proteome
  </a>
  includes both reviewed sequences from UniProtKB/Swiss-Prot (equivalent to an updated version of the complete
  <em>
   H. sapiens
  </em>
  proteome completed in 2008), now supplemented by unreviewed sequences from UniProtKB/TrEMBL. This process will enable the synchronization of the UniProt set with the
  <span class="caps">
   CCDS
  </span>
  project. This version of the complete
  <em>
   H. sapiens
  </em>
  proteome provides higher sequence coverage than the preceding version, but now includes sequences that have not been manually reviewed. Users can choose to opt either for this
  <a href="http://www.uniprot.org/uniprot/?query=organism:9606+keyword:181">
   expanded complete
   <em>
    H. sapiens
   </em>
   proteome
  </a>
  or a
  <a href="http://www.uniprot.org/uniprot/?query=organism:9606+keyword:181+reviewed:yes">
   reduced version that derives exclusively from UniProtKB/Swiss-Prot
  </a>
  .
 </p>
 <p>
  For
  <em>
   Mus musculus
  </em>
  , we added the keyword ‘Complete proteome’ to the existing UniProtKB/Swiss-Prot mouse entries.  We identified those entries in UniProtKB/TrEMBL which mapped to the complete genome in Ensembl and imported the predicted protein sequences (except fragments) from Ensembl which were found not to be present in UniProtKB. The keyword ‘Complete proteome’ was also added to these entries. As for the human counterpart, the
  <a href="http://www.uniprot.org/uniprot/?query=taxonomy:10090+keyword:181">
   UniProtKB complete
   <em>
    Mus musculus
   </em>
   proteome
  </a>
  can now be easily retrieved using this keyword. The Ensembl cross-references have been added to the UniProtKB entries on the basis of 100% sequence identity over their full length.
 </p>
 <p>
  There has been a deliberate introduction of redundancy into the proteomes based on the complete genomes to ensure that all alternative protein variants and isoforms are presented in the set. Over time, these will be merged with the parent entry as is
  <a href="http://www.uniprot.org/help/redundancy">
   UniProtKB/Swiss-Prot curation policy
  </a>
  . We are also evaluating the fragment sequences predicted in the Ensembl complete proteomes for future incorporation.
 </p>
 <p>
  We very much welcome the feedback of the community on our efforts. We expect to make the remaining
  <span class="caps">
   IPI
  </span>
  species (
  <em>
   Gallus gallus
  </em>
  ,
  <em>
   Bos taurus
  </em>
  ,
  <em>
   Danio rerio
  </em>
  ,
  <em>
   Arabidopsis thaliana
  </em>
  and
  <em>
   Rattus norvegicus
  </em>
  ) and some additional species of interest (
  <em>
   Sus scrofa
  </em>
  ,
  <em>
   Canis familiaris
  </em>
  ) available soon.
 </p>
 <h2>
  UniProtKB News
 </h2>
 <h3>
  Changes to the taxonomy of UniProtKB entries from the model fungal organisms
  <em>
   Saccharomyces cerevisiae
  </em>
  (YEAST) and
  <em>
   Schizosaccharomyces pombe
  </em>
  (SCHPO)
 </h3>
 <p>
  Historically, UniProt assigned species identification codes (i.e. the 5-letter mnemonic that forms the second part of the composite UniProtKB entry name) at species-level. All sequences of a given protein from a single species were merged into a single UniProtKB/Swiss-Prot entry, which could therefore contain sequences from many different strains of that species. Discrepancies between individual sequences were annotated as “conflicts” or “variants” in the feature table of the entry.
 </p>
 <p>
  The number of complete genome submissions from different strains of individual species is now increasing at an ever accelerating rate and this data has elucidated that distinct strains of many species can exhibit considerable differences in both gene content and within individual shared genes. The sheer amount of complete proteome data and the associated variability between proteomes means that manual merging of individual strains is no longer sustainable, and this approach has been discontinued. UniProt now assigns mnemonic codes at strain-level for complete genome sequences. We are in the process of reassigning many proteomes corresponding to known strains from a species-level taxonomic identifier (defined by the
  <span class="caps">
   NCBI
  </span>
  _TaxID) to the appropriate strain-level taxonomic identifier. In parallel we are also resolving some of the most significant historical cases of strain-level merging.
 </p>
 <p>
  Here we describe the changes which accompanied the reassignment of the proteomes of the model fungal organisms
  <em>
   Saccharomyces cerevisiae
  </em>
  (YEAST) and
  <em>
   Schizosaccharomyces pombe
  </em>
  (SCHPO) to a strain-level taxonomic identifier.
 </p>
 <p>
  We have changed the taxonomy of all UniProtKB entries of the
  <em>
   S. cerevisiae
  </em>
  complete proteome from strain S288c (the reference genome sequence stored at
  <span class="caps">
   SGD
  </span>
  ) from
  <span class="caps">
   NCBI
  </span>
  _TaxID=4932 to
  <span class="caps">
   NCBI
  </span>
  _TaxID=559292:
 </p>
 <pre>
OS   Saccharomyces cerevisiae (Baker's yeast).
OC   Eukaryota; Fungi; Dikarya; Ascomycota; Saccharomycotina;
OC   Saccharomycetes; Saccharomycetales; Saccharomycetaceae; Saccharomyces.
OX   NCBI_TaxID=4932;
</pre>
 to
 <pre>
OS   Saccharomyces cerevisiae (strain ATCC 204508 / S288c) (Baker's yeast).
OC   Eukaryota; Fungi; Dikarya; Ascomycota; Saccharomycotina;
OC   Saccharomycetes; Saccharomycetales; Saccharomycetaceae; Saccharomyces.
OX   NCBI_TaxID=559292;
</pre>
 <p>
  To facilitate entry recognition and tracking of the entries, those of strain S288c (NCBI_TaxID=559292) have kept the mnemonic
  <span class="caps">
   YEAST
  </span>
  .
 </p>
 <p>
  All
  <em>
   S. cerevisiae
  </em>
  entries not originating from the genome strain S288c, or one of the other completely sequenced strains (strain
  <span class="caps">
   RM11
  </span>
  -1a (YEAS1), strain
  <span class="caps">
   JAY291
  </span>
  (YEAS2), strain
  <span class="caps">
   AWRI1631
  </span>
  (YEAS6), strain
  <span class="caps">
   YJM789
  </span>
  (YEAS7) and Lalvin
  <span class="caps">
   EC1118
  </span>
  (YEAS8)), remain at a species level taxonomic identifier (NCBI_TaxID=4932), for which the new mnemonic
  <span class="caps">
   YEASX
  </span>
  was created (e.g.
  <span class="caps">
   MAL62
  </span>
  _YEASX for
  <a href="http://www.uniprot.org/uniprot/P07265">
   <span class="caps">
    P07265
   </span>
  </a>
  ).
 </p>
 <p>
  A similar procedure was applied to the
  <em>
   S. pombe
  </em>
  proteome. We have changed the taxonomy of all UniProtKB entries of the
  <em>
   S. pombe
  </em>
  complete proteome from strain 972 (the reference genome sequence stored at
  <em>
   S. pombe
  </em>
  GeneDB) from
  <span class="caps">
   NCBI
  </span>
  _TaxID=4896 to
  <span class="caps">
   NCBI
  </span>
  _TaxID=284812:
 </p>
 <pre>
OS   Schizosaccharomyces pombe (Fission yeast).
OC   Eukaryota; Fungi; Dikarya; Ascomycota; Taphrinomycotina;
OC   Schizosaccharomycetes; Schizosaccharomycetales;
OC   Schizosaccharomycetaceae; Schizosaccharomyces.
OX   NCBI_TaxID=4896;
</pre>
 to
 <pre>
OS   Schizosaccharomyces pombe (strain ATCC 38366 / 972) (Fission yeast).
OC   Eukaryota; Fungi; Dikarya; Ascomycota; Taphrinomycotina;
OC   Schizosaccharomycetes; Schizosaccharomycetales;
OC   Schizosaccharomycetaceae; Schizosaccharomyces.
OX   NCBI_TaxID=284812;
</pre>
 <p>
  All
  <em>
   S. pombe
  </em>
  entries not originating from the genome strain 972 retain a species-level taxonomic identifier (NCBI_TaxID=4896), for which the new mnemonic
  <span class="caps">
   SCHPM
  </span>
  was created.
 </p>
 <h3>
  Changes concerning the
  <a href="http://www.uniprot.org/docs/ptmlist">
   controlled vocabulary for PTMs
  </a>
 </h3>
 New terms for the feature key ‘Cross-link’ (‘CROSSLNK’ in the flat file):
 <ul>
  <li>
   1-(tryptophan-3-yl)-tryptophan (Trp-Trp) (interchain)
  </li>
  <li>
   S-(2-aminovinyl)-L-cysteine (Cys-Cys)
  </li>
 </ul>
 New terms for the feature key ‘Modified residue’ (‘MOD_RES’ in the flat file):
 <ul>
  <li>
   (E)-2,3-didehydrobutyrine
  </li>
  <li>
   (Z)-2,3-didehydrobutyrine
  </li>
  <li>
   5’-chlorotryptophan
  </li>
  <li>
   L-allo-isoleucine
  </li>
 </ul>
</div>