<div>
 <h2>
  Headline
 </h2>
 <h3>
  An old-timer, but still trendy: 10’000 entries for Arabidopsis thaliana in UniProtKB/Swiss-Prot
 </h3>
 <p>
  <a href="http://www.uniprot.org/taxonomy/3702">
   <em>
    Arabidopsis thaliana
   </em>
  </a>
  belongs to the Brassicaceae family that includes the well-known dietary staples cauliflower, broccoli, cabbage, turnip, radish, canola and mustard. The split between the Arabidopsis group and the other crops of the genus Brassica has been estimated at around
  <a href="http://www.ncbi.nlm.nih.gov/pubmed/20921408">
   43 million years ago
  </a>
  .
  <em>
   A. thaliana
  </em>
  has been widely studied since the 1980s, when the development of
  <a href="http://www.ncbi.nlm.nih.gov/pubmed/16453562">
   T-DNA mediated transformation
  </a>
  made the generation of mutants and their study relatively easy. Since then,
  <em>
   A. thaliana
  </em>
  has been become the model of choice for the study of many biological processes of flowering plants, as well as those specific to the Brassica species.
 </p>
 <p>
  Its utility as a model organism was further boosted by the completion of the
  <a href="http://www.ncbi.nlm.nih.gov/pubmed/11130711">
   whole genome sequence
  </a>
  , which revealed a relatively small genome with a low level of duplication compared to other flowering plants, and by the inception of a number of complementary efforts to sequence the transcriptome.
 </p>
 <p>
  In 2001, the Swiss-Prot group created the Plant Proteome Annotation Program (
  <a href="http://www.uniprot.org/program/Plants">
   <span class="caps">
    PPAP
   </span>
  </a>
  ) whose main focus is the annotation of proteins and protein families of
  <em>
   A. thaliana
  </em>
  and rice. One decade on, we have now annotated over
  <a href="http://www.uniprot.org/uniprot/?query=organism:3702+AND+reviewed:yes">
   10’000
   <em>
    A. thaliana
   </em>
   entries
  </a>
  in UniProtKB/Swiss-Prot. This corresponds to around 36% of the
  <em>
   A. thaliana
  </em>
  proteome according to
  <a href="http://www.arabidopsis.org/doc/news/breaking_news/140">
   version 10
   <em>
    Arabidopsis thaliana
   </em>
   genome annotation
  </a>
  from The Arabidopsis Information Resource (
  <a href="http://www.arabidopsis.org/">
   <span class="caps">
    TAIR
   </span>
  </a>
  ), which estimates at 27’416 the number of protein-coding genes in this organism. According to the Multinational Arabidopsis Steering Committee (MASC)
  <a href="http://www.arabidopsis.org/portals/masc/masc_docs/masc_reports.jsp">
   report 2010
  </a>
  , at least one third of these genes still have no known function, so the work of experimentally characterizing and of annotating each gene is far from finished.
 </p>
 <p>
  <a href="http://www.uniprot.org/uniprot/?query=organism:3702+AND+reviewed:yes">
   All manually annotated
   <em>
    A.thaliana
   </em>
   entries
  </a>
  can be retrieved from UniProtKB/Swiss-Prot using the organism name “
  <em>
   Arabidopsis thaliana
  </em>
  ” (or the taxonomy identifier 3702), with the restriction: “reviewed:yes”.
 </p>
 <h2>
  UniProtKB news
 </h2>
 <h3>
  Cross-references to Allergome
 </h3>
 <p>
  Cross-references have been added to
  <a href="http://www.allergome.org/">
   Allergome
  </a>
  , a platform for allergen knowledge.
 </p>
 <p>
  Allergome is available at
  <a href="http://www.allergome.org/">
   http://www.allergome.org/
  </a>
  .
 </p>
 <p>
  The format of the explicit link in the flat file is:
 </p>
 <table>
  <tr>
   <th>
    Resource abbreviation
   </th>
   <td>
    Allergome
   </td>
  </tr>
  <tr>
   <th>
    Resource identifier
   </th>
   <td>
    Allergome unique identifier
   </td>
  </tr>
  <tr>
   <th>
    Optional information 1
   </th>
   <td>
    Allergen name
   </td>
  </tr>
  <tr>
   <th>
    Example
   </th>
   <td>
    <a href="http://www.uniprot.org/uniprot/O76821#section_x-ref">
     <span class="caps">
      O76821
     </span>
    </a>
    <pre>DR   Allergome; 2; Aca s 13.
DR   Allergome; 3051; Aca s 13.0101.</pre>
   </td>
  </tr>
 </table>
 <h3>
  Changes to
  <a href="http://www.uniprot.org/docs/keywlist">
   keywords
  </a>
 </h3>
 New keyword:
 <ul>
  <li>
   <a href="http://www.uniprot.org/keywords/KW-1184">
    Jasmonic acid signaling pathway
   </a>
  </li>
 </ul>
 Modified keyword:
 <ul>
  <li>
   Envelope protein -&gt;
   <a href="http://www.uniprot.org/keywords/KW-0261">
    Viral envelope protein
   </a>
  </li>
 </ul>
 Deleted keywords:
 <ul>
  <li>
   Core protein
  </li>
  <li>
   Fiber protein
  </li>
  <li>
   Fusion protein
  </li>
  <li>
   Hexon protein
  </li>
  <li>
   Hexon-associated protein
  </li>
  <li>
   Phage recognition
  </li>
 </ul>
 <h3>
  Changes in the
  <a href="http://www.uniprot.org/docs/ptmlist">
   controlled vocabulary for PTMs
  </a>
 </h3>
 New term for the feature key ‘Modified residue’ (‘MOD_RES’ in the flat file):
 <ul>
  <li>
   N,N-dimethylleucine
  </li>
 </ul>
</div>