<div>
 <h2>
  Headline
 </h2>
 <h3>
  Of serpents, humans and pain
 </h3>
 <p>
  Pain can be viewed as an indispensable communication tool to warn us that something is wrong, and to help us minimize physical harm to our body. Congenital insensitivity to pain leads to severe problems. Although pain is very useful, persistent pain can turn into a nightmare and the spontaneous reaction of most persons is to seek relief – often by reaching for painkillers. Understanding the mechanism of
  <a href="http://medical-dictionary.thefreedictionary.com/nociception">
   nociception
  </a>
  could help develop treatments that provide relief for millions of people.
 </p>
 <p>
  Surprisingly, a
  <a href="http://www.ncbi.nlm.nih.gov/pubmed/22094702">
   hint
  </a>
  may come from a predator: the
  <a href="http://www.uniprot.org/taxonomy/1114302" title="_Micrurus tener tener_">
   Texas coral snake
  </a>
  . This beautiful snake
  <a href="http://upload.wikimedia.org/wikipedia/commons/c/cb/Micrurus_tener.jpg">
   with black, yellow and red banding
  </a>
  lives in the southern United States and throughout most of Mexico. In the absence of antivenom treatment, the fatality rate of coral snake envenomations is estimated at 10%. Death is primarily due to respiratory or cardiovascular failure. In addition, coral snake bite causes excruciating and unremitting pain.
 </p>
 <p>
  The culprit is MitTx, a venom toxin active as a heterodimer made of
  <a href="http://www.uniprot.org/uniprot/G9I929">
   MitTx-alpha
  </a>
  and
  <a href="http://www.uniprot.org/uniprot/G9I930">
   MitTx-beta
  </a>
  .
  <a href="http://www.uniprot.org/uniprot/G9I929">
   MitTx-alpha
  </a>
  contains a
  <span class="caps">
   BPTI
  </span>
  /Kunitz domain, found in many protease inhibitors.
  <a href="http://www.uniprot.org/uniprot/G9I930">
   MitTx-beta
  </a>
  belongs to the phospholipase A2 (PLA2) family, but it lacks critical catalytic residues normally found in the active site of related
  <span class="caps">
   PLA2
  </span>
  enzymes and has been shown to be inactive as a phospholipase. The MitTx heterodimer activates acid-sensing ion channels (ASICs). ASICs are voltage-independent channels expressed in neurons and activated by acid. They are preferentially permeable to Na+, but to a lesser extent can also conduct other cations, such as Ca2+, K+ and Li+ and H+. Physiologically ASICs can be triggered by tissue injuries, inflammation or build-up of lactic acid. This alert system is hijacked by coral snake venom. Whereas protons elicit very transient responses, those evoked by MitTx are dramatically prolonged, reflecting both lack of desensitization and slow reversibility after washout.
 </p>
 <p>
  At neutral pH, the most robust toxin-evoked responses are observed with the
  <a href="http://www.uniprot.org/uniprot/?query=gene:ACCN2">
   <span class="caps">
    ACCN2
   </span>
  </a>
  ASIC subtype. However, if the extracellular pH drops below neutrality, the toxin becomes an excellent
  <a href="http://www.uniprot.org/uniprot/?query=gene:ACCN3">
   <span class="caps">
    ACCN3
   </span>
  </a>
  agonist, essentially enhancing the potency of protons by three orders of magnitude.
 </p>
 <p>
  <a href="http://www.uniprot.org/taxonomy/129461">
   Brazilian coral snake
  </a>
  venom also activates
  <span class="caps">
   ACCN2
  </span>
  expressing cells. This very channel had already been shown to be targeted by the
  <a href="http://www.uniprot.org/uniprot/P60514">
   PcTx1
  </a>
  toxin from the
  <a href="http://www.uniprot.org/taxonomy/179874">
   Trinidad chevron tarantula
  </a>
  .
  <a href="http://www.ncbi.nlm.nih.gov/pubmed/10829030">
   In this case
  </a>
  , the toxin does not activate the channel by itself, but rather serves as a functional antagonist of proton-evoked responses by locking the channel in a desensitized state.
 </p>
 <p>
  Animal toxins often act on very restricted targets and have proven to be extremely useful tools for basic research. The identification of MitTx should allow further investigation the role of ASICs in pain signaling, and eventually the development of new analgesics.
 </p>
 <p>
  For more information on toxins in UniProtKB, see the
  <a href="http://www.uniprot.org/program/Toxins">
   Animal toxin annotation program
  </a>
  .
 </p>
 <h2>
  UniProtKB news
 </h2>
 <h3>
  Complete proteomes for Ensembl Genomes species
 </h3>
 <p>
  The source of the UniProtKB complete proteomes are genomes in
  <a href="http://www.insdc.org/">
   <span class="caps">
    INSDC
   </span>
  </a>
  and
  <a href="http://www.ensembl.org/index.html">
   Ensembl
  </a>
  and now, to further increase the taxonomic coverage, species from
  <a href="http://www.ensemblgenomes.org/">
   Ensembl Genomes
  </a>
  will also be incorporated. Ensembl Genomes aims to work with all sections of the scientific community to represent the best annotation for every genome. Its role varies according to the species, from displaying the genome assembly, gene prediction and functional annotation, through to providing a portal through which genomic data from model organism and community databases can be visualised and analysed in their wider context, and also integrated with other data stored in the core repositories maintained by the
  <span class="caps">
   EBI
  </span>
  .
 </p>
 <p>
  The new species are:
  <br/>
  <a href="http://metazoa.ensembl.org/Caenorhabditis_japonica/Info/Index">
   <em>
    Caenorhabditis japonica
   </em>
  </a>
  <br/>
  <a href="http://protists.ensembl.org/Phytophthora_ramorum/Info/Index/">
   <em>
    Phytophthora ramorum
   </em>
  </a>
  <br/>
  <a href="http://metazoa.ensembl.org/Pristionchus_pacificus/Info/Index/">
   <em>
    Pristionchus pacificus
   </em>
  </a>
  <br/>
  <a href="http://metazoa.ensembl.org/Strongylocentrotus_purpuratus/Info/Index">
   <em>
    Strongylocentrotus purpuratus
   </em>
  </a>
 </p>
 <p>
  All predicted protein sequences from an Ensembl Genome are mapped to their UniProtKB counterparts under stringent conditions: 100% identity over 100% of the length of the two sequences is required. Any sequence found to be absent from UniProtKB is imported into the unreviewed component of UniProtKB, UniProtKB/TrEMBL. All UniProtKB entries that map to an Ensembl Genome are used to build the proteome; they are tagged with the keyword
  <a href="http://www.uniprot.org/keywords/KW-0181">
   Complete proteome
  </a>
  and an Ensembl Genomes cross-reference is added.
 </p>
 <p>
  We very much welcome the feedback of the community on our efforts. In future UniProt releases, we expect to make proteomes for the remaining Ensembl Genomes species currently absent from UniProtKB.
 </p>
 <h3>
  Update of Complete proteomes with Ensembl release 66
 </h3>
 <p>
  <a href="http://www.ensembl.org/info/website/news.html">
   Ensembl release 66
  </a>
  was made available at the end of February 2012 and, in response, the appropriate complete proteomes have been updated in UniProtKB. Of note, the human reference proteome has grown in size by just over 8,000 new UniProtKB entries.  This growth is a consequence of the following updates:
 </p>
 <ul>
  <li>
   Incorporation of the latest set of cDNAs from the
   <a href="http://www.ebi.ac.uk/ena/">
    European Nucleotide Archive
   </a>
   and
   <a href="http://www.ncbi.nlm.nih.gov/RefSeq/">
    <span class="caps">
     NCBI
    </span>
    RefSeq
   </a>
   . A total of 224,907 cDNAs are aligned to the current genome showing an increase of 491 cDNAs compared to release 65.
  </li>
  <li>
   New
   <a href="http://www.ncbi.nlm.nih.gov/CCDS/">
    <span class="caps">
     CCDS
    </span>
   </a>
   import – the updated gene set includes 26,437 transcript models.
  </li>
  <li>
   The patches for GRCh37.p6 were annotated using a combination of manual annotation, annotation projected from the primary assembly and annotation derived from cDNA and protein alignment evidence.
  </li>
  <li>
   Update of Havana manual annotation representing data present in
   <a href="http://vega.sanger.ac.uk/Homo_sapiens/Info/Index">
    Vega release 46
   </a>
   which includes
   <a href="http://www.gencodegenes.org/">
    <span class="caps">
     GENCODE
    </span>
    release 11
   </a>
   .
  </li>
 </ul>
 <p>
  The proteomes of 35 chordate species are now fully synchronised with Ensembl 66. The species are:
  <br/>
  <a href="http://www.uniprot.org/uniprot/?query=organism:9646+AND+keyword:KW-0181">
   <em>
    Ailuropoda melanoleuca
   </em>
   (Giant panda)
  </a>
  <br/>
  <a href="http://www.uniprot.org/uniprot/?query=organism:28377+AND+keyword:KW-0181">
   <em>
    Anolis carolinensis
   </em>
   (American chameleon)
  </a>
  <br/>
  <a href="http://www.uniprot.org/uniprot/?query=organism:9913+AND+keyword:KW-0181">
   <em>
    Bos taurus
   </em>
   (Cow)
  </a>
  <br/>
  <a href="http://www.uniprot.org/uniprot/?query=organism:9483+AND+keyword:KW-0181">
   <em>
    Callithrix jacchus
   </em>
   (White-tufted-ear marmoset)
  </a>
  <br/>
  <a href="http://www.uniprot.org/uniprot/?query=organism:9615+AND+keyword:KW-0181">
   <em>
    Canis familiaris
   </em>
   (Dog)
  </a>
  <br/>
  <a href="http://www.uniprot.org/uniprot/?query=organism:10141+AND+keyword:KW-0181">
   <em>
    Cavia porcellus
   </em>
   (Guinea pig)
  </a>
  <br/>
  <a href="http://www.uniprot.org/uniprot/?query=organism:7719+AND+keyword:KW-0181">
   <em>
    Ciona intestinalis
   </em>
   (Transparent sea squirt)
  </a>
  <br/>
  <a href="http://www.uniprot.org/uniprot/?query=organism:51511+AND+keyword:KW-0181">
   <em>
    Ciona savignyi
   </em>
   (Pacific transparent sea squirt)
  </a>
  <br/>
  <a href="http://www.uniprot.org/uniprot/?query=organism:7955+AND+keyword:KW-0181">
   <em>
    Danio rerio
   </em>
   (Zebrafish)
  </a>
  <br/>
  <a href="http://www.uniprot.org/uniprot/?query=organism:9796+AND+keyword:KW-0181">
   <em>
    Equus caballus
   </em>
   (Horse)
  </a>
  <br/>
  <a href="http://www.uniprot.org/uniprot/?query=organism:9031+AND+keyword:KW-0181">
   <em>
    Gallus gallus
   </em>
   (Chicken)
  </a>
  <br/>
  <a href="http://www.uniprot.org/uniprot/?query=organism:69293+AND+keyword:KW-0181">
   <em>
    Gasterosteus aculeatus
   </em>
   (Three-spined stickleback)
  </a>
  <br/>
  <a href="http://www.uniprot.org/uniprot/?query=organism:9595+AND+keyword:KW-0181">
   <em>
    Gorilla gorilla
   </em>
   (Lowland gorilla)
  </a>
  <br/>
  <a href="http://www.uniprot.org/uniprot/?query=organism:9606+AND+keyword:KW-0181">
   <em>
    Homo sapiens
   </em>
   (Human)
  </a>
  <br/>
  <a href="http://www.uniprot.org/uniprot/?query=organism:7897+AND+keyword:KW-0181">
   <em>
    Latimeria chalumnae
   </em>
   (West Indian ocean coelacanth)
  </a>
  <br/>
  <a href="http://www.uniprot.org/uniprot/?query=organism:9785+AND+keyword:KW-0181">
   <em>
    Loxodonta africana
   </em>
   (African elephant)
  </a>
  <br/>
  <a href="http://www.uniprot.org/uniprot/?query=organism:9544+AND+keyword:KW-0181">
   <em>
    Macaca mulatta
   </em>
   (Rhesus macaque)
  </a>
  <br/>
  <a href="http://www.uniprot.org/uniprot/?query=organism:9103+AND+keyword:KW-0181">
   <em>
    Meleagris gallopavo
   </em>
   (Common turkey)
  </a>
  <br/>
  <a href="http://www.uniprot.org/uniprot/?query=organism:13616+AND+keyword:KW-0181">
   <em>
    Monodelphis domestica
   </em>
   (Gray short-tailed opossum)
  </a>
  <br/>
  <a href="http://www.uniprot.org/uniprot/?query=organism:10090+AND+keyword:KW-0181">
   <em>
    Mus musculus
   </em>
   (Mouse)
  </a>
  <br/>
  <a href="http://www.uniprot.org/uniprot/?query=organism:59463+AND+keyword:KW-0181">
   <em>
    Myotis lucifugus
   </em>
   (Little brown bat)
  </a>
  <br/>
  <a href="http://www.uniprot.org/uniprot/?query=organism:61853+AND+keyword:KW-0181">
   <em>
    Nomascus leucogenys
   </em>
   (Northern white-cheeked gibbon)
  </a>
  <br/>
  <a href="http://www.uniprot.org/uniprot/?query=organism:9258+AND+keyword:KW-0181">
   <em>
    Ornithorhynchus anatinus
   </em>
   (Duckbill platypus)
  </a>
  <br/>
  <a href="http://www.uniprot.org/uniprot/?query=organism:9986+AND+keyword:KW-0181">
   <em>
    Oryctolagus cuniculus
   </em>
   (Rabbit)
  </a>
  <br/>
  <a href="http://www.uniprot.org/uniprot/?query=organism:8090+AND+keyword:KW-0181">
   <em>
    Oryzias latipes
   </em>
   (Medaka fish)
  </a>
  <br/>
  <a href="http://www.uniprot.org/uniprot/?query=organism:30611+AND+keyword:KW-0181">
   <em>
    Otolemur garnettii
   </em>
   (Garnett’s greater bushbaby)
  </a>
  <br/>
  <a href="http://www.uniprot.org/uniprot/?query=organism:9598+AND+keyword:KW-0181">
   <em>
    Pan troglodytes
   </em>
   (Chimpanzee)
  </a>
  <br/>
  <a href="http://www.uniprot.org/uniprot/?query=organism:9601+AND+keyword:KW-0181">
   <em>
    Pongo abelii
   </em>
   (Sumatran orangutan)
  </a>
  <br/>
  <a href="http://www.uniprot.org/uniprot/?query=organism:10116+AND+keyword:KW-0181">
   <em>
    Rattus norvegicus
   </em>
   (Rat)
  </a>
  <br/>
  <a href="http://www.uniprot.org/uniprot/?query=organism:9305+AND+keyword:KW-0181">
   <em>
    Sarcophilus harrisii
   </em>
   (Tasmanian devil)
  </a>
  <br/>
  <a href="http://www.uniprot.org/uniprot/?query=organism:9823+AND+keyword:KW-0181">
   <em>
    Sus scrofa
   </em>
   (Pig)
  </a>
  <br/>
  <a href="http://www.uniprot.org/uniprot/?query=organism:59729+AND+keyword:KW-0181">
   <em>
    Taeniopygia guttata
   </em>
   (Zebra finch)
  </a>
  <br/>
  <a href="http://www.uniprot.org/uniprot/?query=organism:31033+AND+keyword:KW-0181">
   <em>
    Takifugu rubripes
   </em>
   (Japanese pufferfish)
  </a>
  <br/>
  <a href="http://www.uniprot.org/uniprot/?query=organism:99883+AND+keyword:KW-0181">
   <em>
    Tetraodon nigroviridis
   </em>
   (Spotted green pufferfish)
  </a>
  <br/>
  <a href="http://www.uniprot.org/uniprot/?query=organism:8364+AND+keyword:KW-0181">
   <em>
    Xenopus tropicalis
   </em>
   (Western clawed frog)
  </a>
 </p>
 <h3>
  Update to the
  <em>
   Tetraodon nigroviridis
  </em>
  proteome
 </h3>
 <p>
  The
  <em>
   Tetraodon nigroviridis
  </em>
  complete proteome has been updated with data from
  <a href="http://www.ensembl.org/Tetraodon_nigroviridis/Info/Index">
   Ensembl release 66
  </a>
  . Until now the proteome has reflected the
  <a href="http://www.genoscope.cns.fr/externe/tetranew/">
   Genoscope gene model annotations
  </a>
  provided within the whole genome shotgun project (accession
  <a href="https://www.ebi.ac.uk/ena/browser/view/CAAE00000000">
   <span class="caps">
    CAAE00000000
   </span>
  </a>
  ) that were made available in March 2007. The proteome has been updated to reflect the annotations of the genome using Ensembl’s more conservative, evidence-based pipeline. Although a consequence of this update is a slightly reduced proteome size, the gene model predictions are high-quality and fit well into the Ensembl Compara gene trees. An example of an Ensembl sourced protein sequence is entry
  <a href="http://www.uniprot.org/uniprot/H3C526">
   <span class="caps">
    H3C526
   </span>
  </a>
  .
 </p>
 <h3>
  Cross-references to EvolutionaryTrace
 </h3>
 <p>
  Cross-references have been added to EvolutionaryTrace, which ranks amino acid residues in a protein sequence by their relative evolutionary importance.
 </p>
 <p>
  EvolutionaryTrace is available at
  <a href="http://mammoth.bcm.tmc.edu/ETserver.html">
   http://mammoth.bcm.tmc.edu/ETserver.html
  </a>
 </p>
 <p>
  The format of the explicit links in the flat file is:
 </p>
 <table>
  <tr>
   <th>
    Resource abbreviation
   </th>
   <td>
    EvolutionaryTrace
   </td>
  </tr>
  <tr>
   <th>
    Resource identifier
   </th>
   <td>
    UniProtKB accession number
   </td>
  </tr>
  <tr>
   <th>
    Example
   </th>
   <td>
    <a href="http://www.uniprot.org/uniprot/P06611#section_x-ref">
     <span class="caps">
      P06611
     </span>
    </a>
    :
    <pre>DR   EvolutionaryTrace; P06611; -.</pre>
   </td>
  </tr>
 </table>
 <p>
  <a href="http://www.uniprot.org/uniprot/?query=database:evolutionarytrace">
   Show all the entries having a cross-reference to EvolutionaryTrace.
  </a>
 </p>
 <h3>
  Changes to the
  <a href="http://www.uniprot.org/docs/ptmlist">
   controlled vocabulary for PTMs
  </a>
 </h3>
 New terms for the feature key ‘Modified residue’ (‘MOD_RES’ in the flat file):
 <ul>
  <li>
   3-hydroxyhistidine
  </li>
  <li>
   (3S)-3-hydroxyaspartate
  </li>
  <li>
   (5R)-5-hydroxylysine
  </li>
  <li>
   (5S)-5-hydroxylysine
  </li>
 </ul>
</div>