<div>
 <h2>
  Headline
 </h2>
 <h3>
  Thiamine thiazole synthase: enzyme, catalyst or co-substrate?
 </h3>
 <p>
  Thiamine is a cofactor essential for many biochemical reactions in all living beings. Humans depend on their diet to supply it as vitamin B1, while bacteria, plants and yeast can make their own. They do so by coupling two precursor molecules: a sulfur-containing ring structure known as a thiazole and a nitrogenous pyrimidine.
 </p>
 <p>
  In eukaryotes, it has been known for some time that
  <a href="http://www.uniprot.org/uniprot/?query=name:%22Thiamine+thiazole+synthase*%22">
   thiamine thiazole synthase
  </a>
  catalyzes thiazole biosynthesis, but the source of the thiazole sulfur at the heart of the reaction remained elusive. A recent
  <a href="http://www.ncbi.nlm.nih.gov/pubmed/22031445">
   publication
  </a>
  unveiled a very unusual mechanism, whereby a sulfide ion is transferred from a conserved cysteine of thiamine thiazole synthase itself to become part of the thiazole precursor in
  <em>
   Saccharomyces cerevisiae
  </em>
  . This transfer is strictly dependent on the presence of Fe(2+). The donor cysteine,
  <a href="http://www.uniprot.org/uniprot/P32318#section_features">
   Cys-205
  </a>
  , is irreversibly converted to dehydroalanine, leading to the inactivation of the enzyme. Surprisingly the inactivated protein is not degraded, but accumulates in the cell where it can form up to
  <a href="http://www.ncbi.nlm.nih.gov/pubmed/15544818">
   about 1.5% of total cellular protein
  </a>
  . Could it have another physiological function? This has yet to be explored, but it has been suggested to play a role in
  <a href="http://www.ncbi.nlm.nih.gov/pubmed/9367751">
   mitochondrial
   <span class="caps">
    DNA
   </span>
   damage tolerance
  </a>
  .
 </p>
 <p>
  Although very rare, the use of a protein as a metabolic reagent has already been observed. The best characterized example is
  <a href="http://www.uniprot.org/uniprot/?query=name:2.1.1.63+reviewed:yes">
   methylated-DNA--protein-cysteine methyltransferase
  </a>
  , which repairs O-6 alkylated guanine lesions in
  <span class="caps">
   DNA
  </span>
  by stoichiometrically transferring the alkyl group to a cysteine residue in the enzyme. Here again we face a suicidal reaction, the enzyme being irreversibly inactivated. Interestingly, the
  <a href="http://www.ncbi.nlm.nih.gov/pubmed/3009022,3162236">
   inactive enzyme serves as a signal to induce other
   <span class="caps">
    DNA
   </span>
   repair enzymes
  </a>
  .
 </p>
 <p>
  Can such proteins be considered as “enzymes”? An enzyme is defined as a
  <a href="http://medical-dictionary.thefreedictionary.com/enzyme">
   protein that catalyzes chemical reactions of other substances without itself being destroyed or altered upon completion of the reactions
  </a>
  . Thiamine thiazole synthase functions as a “one-shot” reagent, so therefore does not comply with the definition. At most it can be considered as a catalyst, i.e. a reagent which promotes a reaction and may act repeatedly or only once.
 </p>
 <p>
  Such an unusual mechanism has led to some inconsistencies. The Enzyme Commission attributed the
  <a href="http://enzyme.expasy.org/EC/2.1.1.63">
   EC number 2.1.1.63
  </a>
  to methylated-DNA--protein-cysteine methyltransferases, mentioning the ambiguity of this attribution: “This enzyme catalyzes only one turnover and therefore is not strictly catalytic.” Actually the protein is a catalyst, but it is not strictly an enzyme. The later decision not to provide an EC number to thiamine thiazole synthases is more consistent in view of the definition of an enzyme. This inconsistency is also visible in UniProtKB entries which show EC numbers in the ‘Names and origin’ section of
  <a href="http://www.uniprot.org/uniprot/?query=name:2.1.1.63+reviewed:yes">
   methylated-DNA--protein-cysteine methyltransferases
  </a>
  , but not in that of
  <a href="http://www.uniprot.org/uniprot/?query=name:%22Thiamine+thiazole+synthase*%22">
   thiamine thiazole synthases
  </a>
  .
 </p>
 <p>
  As of this release,
  <a href="http://www.uniprot.org/uniprot/?query=name:%22Thiamine+thiazole+synthase*%22">
   thiamine thiazole synthases
  </a>
  , have been updated in UniProtKB/Swiss-Prot and a new post-translational modification,
  <a href="http://www.uniprot.org/docs/ptmlist" title="Cys">
   2,3-didehydroalanine
  </a>
  has been introduced.
 </p>
 <h2>
  UniProtKB news
 </h2>
 <h3>
  Update to the human proteome
 </h3>
 <p>
  The human reference proteome has been updated with data from Ensembl release 65.
  <a href="http://www.ensembl.org/info/website/news.html">
   Ensembl 65
  </a>
  has numerous updates to the human genome including an update of Havana manual annotation representing data present in
  <a href="http://vega.sanger.ac.uk/Homo_sapiens/Info/Index">
   Vega release 45
  </a>
  . As a result, the human reference proteome has increased in size by over 7,000 entries. These new entries correspond to fragment entries that have transcription evidence captured by Havana and as such they are considered valid members of the proteome. Two examples of these fragment entries are
  <a href="http://www.uniprot.org/uniprot/H0Y5B1">
   <span class="caps">
    H0Y5B1
   </span>
  </a>
  and
  <a href="http://www.uniprot.org/uniprot/H0Y653">
   <span class="caps">
    H0Y653
   </span>
  </a>
  .
 </p>
 <h3>
  Change of the cross-reference GeneDB_Spombe to PomBase
 </h3>
 <p>
  The
  <a href="http://old.genedb.org/genedb/pombe/">
   Schizosaccharomyces pombe GeneDB
  </a>
  was replaced by
  <a href="http://www.pombase.org/">
   PomBase
  </a>
  , the new model organism database for the fission yeast Schizosaccharomyces pombe. We have therefore changed the corresponding resource abbreviation from
  <code>
   GeneDB_SPombe
  </code>
  to
  <code>
   PomBase
  </code>
  .
 </p>
 <h3>
  Change of the category of the cross-reference KO
 </h3>
 <p>
  The KO database has been moved from the category “Family and domain databases” to the category “Phylogenomic databases”.
 </p>
 <h3>
  Removal of the cross-reference
  <span class="caps">
   NMPDR
  </span>
 </h3>
 <p>
  Cross-references to
  <span class="caps">
   NMPDR
  </span>
  have been removed.
 </p>
 <h3>
  Changes to the
  <a href="http://www.uniprot.org/docs/ptmlist">
   controlled vocabulary for PTMs
  </a>
 </h3>
 New terms for the feature key ‘Modified residue’ (‘MOD_RES’ in the flat file):
 <ul>
  <li>
   2,3-didehydroalanine (Cys)
  </li>
  <li>
   N6-malonyllysine
  </li>
 </ul>
</div>