<div>
 <h2>
  Headline
 </h2>
 <h3>
  Major progress in adenovirus annotation
 </h3>
 <p>
  Adenoviruses were first isolated by
  <a href="http://www.ncbi.nlm.nih.gov/pubmed/13134217">
   Wallace Rowe
  </a>
  in 1953 from adenoid tissue of sick children. These viruses infect a wide range of vertebrates, including humans. Infectious virions are spread primarily via respiratory droplets, however they can also be spread by fecal routes. Most infections with Human Adenovirus (HAdV) result in upper respiratory tract diseases; they account for about 10% of acute respiratory infections in children. They can also cause fever, diarrhea, pink eye (conjunctivitis), bladder infection (cystitis), rash illness, etc.
 </p>
 <p>
  <a href="http://viralzone.expasy.org/all_by_species/183.html">
   HAdV
  </a>
  are medium-sized (90-100 nm), non-enveloped icosahedral viruses composed of a capsid and a double-stranded linear
  <span class="caps">
   DNA
  </span>
  genome. The viral genome is approximately 36kb long. It encodes 37 proteins which are produced by complex alternative splicing of 6 mRNA transcription units. The viral genome replicates in the host cell nucleus, but never integrates into the host genome. This is the reason why adenoviruses are widely used in
  <a href="http://www.ncbi.nlm.nih.gov/pubmed/22378183">
   gene therapy and anticancer virus vector trials
  </a>
  .
 </p>
 <p>
  The
  <a href="http://gsc.jcvi.org/projects/gsc/adenovirus/index.php">
   <span class="caps">
    JCVI
   </span>
   adenovirus project
  </a>
  recently resulted in the sequencing of 150 new HAdV genomes. In order to support the annotation of these new genomes, the community needs a high quality set of data that can serve as a reference. In this context, a collaboration including UniProt,
  <span class="caps">
   NCBI
  </span>
  , JCVI and several field experts has been initiated to update reference adenovirus genomes and proteomes. Gene predictions have been corrected with the most recent proteomic and cDNA sequencing data. This major collaborative effort has resulted in a consistent and up-to-date annotation of the viral genome in
  <span class="caps">
   NCBI
  </span>
  <a href="http://www.ncbi.nlm.nih.gov/refseq/">
   RefSeq
  </a>
  and of the HAdV
  <a href="http://www.uniprot.org/uniprot/?query=accession:P24935+or+accession:P03279+or+accession:P03274+or+accession:P03280+or+accession:P03282+or+accession:P03277+or+accession:P03276+or+accession:P14269+or+accession:P03267+or+accession:P03264+or+accession:P03261+or+accession:P03254+or+accession:P03244+or+accession:P03247+or+accession:P27311+or+accession:P68976+or+accession:Q910M3+or+accession:P68978+or+accession:P15133+or+accession:P03250+or+accession:P03239+or+accession:P03242+or+accession:P0DJX0+or+accession:P03241+or+accession:P03240+or+accession:P03238+or+accession:P03263+or+accession:P68950+or+accession:P03272+or+accession:P0DJX1+or+accession:P03262+or+accession:P03252+or+accession:P24939+or+accession:P24932+or+accession:P03275+or+accession:P03269+or+accession:P0DJX2">
   reference proteome in UniProtKB/Swiss-Prot
  </a>
  .
 </p>
 <h2>
  UniProtKB news
 </h2>
 <h3>
  Removal of
  <span class="caps">
   MEDLINE
  </span>
  identifiers
 </h3>
 <p>
  We have removed the
  <span class="caps">
   MEDLINE
  </span>
  identifiers from the bibliographic database cross-references of literature citations since they have been superceded by
  <a href="http://www.ncbi.nlm.nih.gov/pubmed">
   PubMed
  </a>
  identifiers. The valid bibliographic database names and their associated identifiers are now:
 </p>
 <table>
  <tr>
   <th>
    Name
   </th>
   <th>
    Identifier
   </th>
  </tr>
  <tr>
   <td>
    PubMed
   </td>
   <td>
    PubMed Unique Identifier (PMID)
   </td>
  </tr>
  <tr>
   <td>
    <span class="caps">
     DOI
    </span>
   </td>
   <td>
    Digital Object Identifier (DOI)
   </td>
  </tr>
  <tr>
   <td>
    <span class="caps">
     AGRICOLA
    </span>
   </td>
   <td>
    AGRICOLA Unique Identifier
   </td>
  </tr>
 </table>
</div>