<div>
 <h2>
  Headline
 </h2>
 <h3>
  Latest from the prokaryotic world: bacterial Cas9, a new tool for genome engineering
 </h3>
 <p>
  The
  <a href="http://www.uniprot.org/news/2012/09/05/release">
   <span class="caps">
    CRISPR
   </span>
   system
  </a>
  (Clustered Regularly Interspaced Short Palindromic Repeat) is a bacterial and archaeal,
  <span class="caps">
   RNA
  </span>
  -based adaptive immune system, which degrades invading genetic material. Very briefly, invading viruses or plasmids are recognized by their complementarity to
  <span class="caps">
   CRISPR RNA
  </span>
  (crRNA) and degraded by dedicated nucleases.
 </p>
 <p>
  There are 3 major
  <span class="caps">
   CRISPR
  </span>
  systems, with a growing number of recognized subtypes depending on the Cas proteins (CRISPR-associated proteins) used to affect the various steps of crRNA generation and
  <a href="http://www.ncbi.nlm.nih.gov/pubmed/21552286">
   invading nucleic acid destruction
  </a>
  . In type I and
  <span class="caps">
   III CRISPR
  </span>
  systems, different specialized Cas endonucleases generate crRNAs, which then assemble with other Cas proteins to create large crRNA-protein complexes that recognize and degrade invading nucleic acids complementary to the crRNA. Type
  <span class="caps">
   II CRISPR
  </span>
  systems are a little different. In these systems, correct processing of pre-crRNA requires a
  <a href="http://www.ncbi.nlm.nih.gov/pubmed/21455174">
   trans-encoded small
   <span class="caps">
    RNA
   </span>
   (tracrRNA), endogenous RNase
   <span class="caps">
    III
   </span>
   and the Cas9 protein
  </a>
  . The tracrRNA serves as a guide for RNase
  <span class="caps">
   III
  </span>
  -aided processing of pre-crRNA. Subsequently the Cas9/crRNA/tracrRNA complex endonucleolytically cleaves linear or circular
  <a href="http://www.ncbi.nlm.nih.gov/pubmed/22745249,22949671">
   dsDNA target complementary to the crRNA
  </a>
  . Degradation requires the Cas9 protein and both
  <span class="caps">
   RNA
  </span>
  species. Thus, in type
  <span class="caps">
   II CRISPR
  </span>
  systems, crRNA-guided degradation of
  <span class="caps">
   DNA
  </span>
  relies upon a single protein. This discovery has implications beyond the world of bacteria. Expressing Cas9 with specifically chosen crRNA should allow site-specific genome modifications, knocking-out genes on demand not only in bacteria where it is already relatively simple to do so, but also in higher organisms, such as vertebrates.
 </p>
 <p>
  And indeed it works! In 2
  <a href="http://www.ncbi.nlm.nih.gov/pubmed/23287718,23287722">
   back-to-back Science articles
  </a>
  published online in January of this year, Streptococcus pyogenes strain
  <span class="caps">
   SF370
  </span>
  Cas9 endonuclease was codon-optimized and targeted to the nucleus in human or mouse cells. In
  <a href="http://www.ncbi.nlm.nih.gov/pubmed/23287718">
   one article
  </a>
  , RNase
  <span class="caps">
   III
  </span>
  was engineered in a similar fashion while the tracrRNA and pre-crRNA were expressed either separately or as a hybrid molecule, while in
  <a href="http://www.ncbi.nlm.nih.gov/pubmed/23287722">
   the other
  </a>
  , only a hybrid crRNA-tracrRNA was expressed. In both papers, various gene targets were cloned into the crRNA locus, leading to site-specific target cleavage which was subsequently repaired by either nonhomologous end-joining or homologous recombination. While the efficiency of the process varies, introducing multiple targets within a single gene or targeting multiple genes at a time is feasible, allowing for comparatively easy manipulation of a genome of interest. Additionally,
  <a href="http://www.ncbi.nlm.nih.gov/pubmed/23287722">
   no toxicity
  </a>
  has been observed upon expression in human cells.
 </p>
 <p>
  A similar approach has been successfully used not only in
  <a href="http://www.ncbi.nlm.nih.gov/pubmed/23360965">
   other bacteria
  </a>
  , but also in
  <a href="http://www.ncbi.nlm.nih.gov/pubmed/23360964">
   zebrafish
  </a>
  , as well as in different
  <a href="http://www.ncbi.nlm.nih.gov/pubmed/23360966,23386978">
   human cell lines
  </a>
  .
 </p>
 <p>
  The work described above has been carried out using
  <a href="http://www.uniprot.org/uniprot/Q99ZW2">
   Cas9 from Streptococcus pyogenes strain
   <span class="caps">
    SF370
   </span>
  </a>
  , and the corresponding UniProtKB/Swiss-Prot entry has been updated, as have been experimentally characterized orthologous proteins in other bacteria (
  <a href="http://www.uniprot.org/uniprot/G3ECR1">
   Streptococcus thermophilus strain
   <span class="caps">
    DGCC7710
   </span>
  </a>
  ,
  <a href="http://www.uniprot.org/uniprot/Q03JI6">
   Streptococcus thermophilus strain
   <span class="caps">
    ATCC BAA
   </span>
   -491 /
   <span class="caps">
    LMD
   </span>
   -9
  </a>
  and
  <a href="http://www.uniprot.org/uniprot/Q927P4">
   Listeria innocua serovar 6a strain
   <span class="caps">
    CLIP 11262
   </span>
  </a>
  ). Additionally, a new
  <a href="http://hamap.expasy.org/">
   <span class="caps">
    HAMAP
   </span>
  </a>
  rule has been made for the Cas9 family (
  <a href="http://hamap.expasy.org/unirule/MF_01480">
   MF_01480
  </a>
  ).
 </p>
 <h2>
  UniProtKB news
 </h2>
 <h3>
  Cross-references to ChiTaRS
 </h3>
 <p>
  Cross-references have been added to ChiTaRS, a database of human, mouse and fruit fly chimeric transcripts and
  <span class="caps">
   RNA
  </span>
  -sequencing data.
 </p>
 <p>
  ChiTaRS is available at
  <a href="http://chitars.bioinfo.cnio.es/">
   http://chitars.bioinfo.cnio.es/
  </a>
 </p>
 <p>
  The format of the explicit links in the flat file is:
 </p>
 <table>
  <tr>
   <th>
    Resource abbreviation
   </th>
   <td>
    ChiTaRS
   </td>
  </tr>
  <tr>
   <th>
    Resource identifier
   </th>
   <td>
    gene name
   </td>
  </tr>
  <tr>
   <th>
    Optional information 1
   </th>
   <td>
    organism name
   </td>
  </tr>
  <tr>
   <th>
    Example
   </th>
   <td>
    <a href="http://www.uniprot.org/uniprot/P16320#section_x-ref">
     <span class="caps">
      P16320
     </span>
    </a>
    :
    <pre>DR   ChiTaRS; ATP6AP1; drosophila.</pre>
   </td>
  </tr>
 </table>
 <p>
  <a href="http://www.uniprot.org/uniprot/?query=database%3Achitars&amp;sort=score">
   Show all the entries having a cross-reference to ChiTaRS
  </a>
  .
 </p>
 <h3>
  Cross-references to
  <span class="caps">
   SABIO
  </span>
  -RK
 </h3>
 <p>
  Cross-references have been added to
  <span class="caps">
   SABIO
  </span>
  -RK, a database of biochemical reaction kinetics.
 </p>
 <p>
  <span class="caps">
   SABIO
  </span>
  -RK is available at
  <a href="http://sabiork.h-its.org/">
   http://sabiork.h-its.org/
  </a>
 </p>
 <p>
  The format of the explicit links in the flat file is:
 </p>
 <table>
  <tr>
   <th>
    Resource abbreviation
   </th>
   <td>
    <span class="caps">
     SABIO
    </span>
    -RK
   </td>
  </tr>
  <tr>
   <th>
    Resource identifier
   </th>
   <td>
    UniProtKB accession number
   </td>
  </tr>
  <tr>
   <th>
    Example
   </th>
   <td>
    <a href="http://www.uniprot.org/uniprot/P10172#section_x-ref">
     <span class="caps">
      P10172
     </span>
    </a>
    :
    <pre>DR   SABIO-RK; P10172; -.</pre>
   </td>
  </tr>
 </table>
 <p>
  <a href="http://www.uniprot.org/uniprot/?query=database%3Asabio-rk&amp;sort=score">
   Show all the entries having a cross-reference to
   <span class="caps">
    SABIO
   </span>
   -RK
  </a>
  .
 </p>
 <h3>
  Removal of the cross-reference to 8 2D gel databases
 </h3>
 <p>
  Cross-references to 2DBase-Ecoli, Aarhus/Ghent-2DPAGE,
  <span class="caps">
   ANU
  </span>
  -2DPAGE, Cornea-2DPAGE,
  <span class="caps">
   PHCI
  </span>
  -2DPAGE,
  <span class="caps">
   PMMA
  </span>
  -2DPAGE, Siena-2DPAGE, and Rat-heart-2DPAGE have been removed.
 </p>
 <h3>
  Removal of the cross-reference to
  <span class="caps">
   AGD
  </span>
 </h3>
 <p>
  Cross-references to
  <span class="caps">
   AGD
  </span>
  have been removed.
 </p>
 <h3>
  Gene3D
 </h3>
 <p>
  The Gene3D database no longer provides names for their signatures. The entry name that has been displayed in the cross-references was therefore replaced by a dash (’-’).
 </p>
 <p>
  Examples:
  <a href="http://www.uniprot.org/uniprot/Q12933#section_x-ref">
   <span class="caps">
    Q12933
   </span>
  </a>
  :
 </p>
 Previous format:
 <pre>
DR   Gene3D; 2.60.210.10; TRAF-type; 1.
DR   Gene3D; 3.30.40.10; Znf_RING/FYVE/PHD; 1.
</pre>
 New format:
 <pre>
DR   Gene3D; 2.60.210.10; -; 1.
DR   Gene3D; 3.30.40.10; -; 1.
</pre>
 <h3>
  Changes to
  <a href="http://www.uniprot.org/docs/keywlist">
   keywords
  </a>
 </h3>
 New keyword:
 <ul>
  <li>
   <a href="http://www.uniprot.org/keywords/KW-1212">
    Corneal dystrophy
   </a>
  </li>
  <li>
   <a href="http://www.uniprot.org/keywords/KW-1211">
    Schizophrenia
   </a>
  </li>
 </ul>
 Modified keywords:
 <ul>
  <li>
   Ionic channel -&gt;
   <a href="http://www.uniprot.org/keywords/KW-0407">
    Ion channel
   </a>
  </li>
  <li>
   Ionic channel inhibitor -&gt;
   <a href="http://www.uniprot.org/keywords/KW-0872">
    Ion channel impairing toxin
   </a>
  </li>
  <li>
   Viral ionic channel -&gt;
   <a href="http://www.uniprot.org/keywords/KW-1182">
    Viral ion channel
   </a>
  </li>
 </ul>
</div>