<h2 id="Headline">
 Headline
</h2>
<h3 id="A_flounder____on_the_rocks_">
 A flounder... on the rocks!
</h3>
<p>
 Some organisms, such as certain vertebrates, plants, fungi and bacteria, have to resist low, subzero temperatures. Their survival relies upon the production of antifreeze molecules. Some insects, like the beetle
 <a href="http://en.wikipedia.org/wiki/Upis%5Fceramboides">
  <em>
   Upis ceramboides
  </em>
 </a>
 , tolerate freezing to -60°C in midwinter thanks to the production of a compound, called
 <a href="http://www.ncbi.nlm.nih.gov/pubmed/22029271,21955117,19934038">
  xylomannan
 </a>
 , made of a sugar and a fatty acid and located in cell membranes. However, most organisms use antifreeze proteins (AFPs). All
 <a href="http://en.wikipedia.org/wiki/Antifreeze%5Fprotein">
  AFPs
 </a>
 act by binding to small ice crystals to inhibit growth that would otherwise be fatal, but each type of AFP seems to arrive at this end by a different route.
</p>
<p>
 <a href="http://www.uniprot.org/taxonomy/8265">
  <em>
   Pseudopleuronectes americanus
  </em>
 </a>
 , commonly called 'winter flounder', is a very common variety of flounder in North America. It lives in cold water and survives thanks to the expression of the AFP Maxi. The
 <a href="http://www.ncbi.nlm.nih.gov/pubmed/24531972">
  3D structure
 </a>
 of the Maxi protein has been recently elucidated, unveiling some very unusual features.
</p>
<p>
 Maxi belongs to the
 <a href="http://www.uniprot.org/uniprot/?query=family:type-I%2BAFP%2Bfamily">
  type-I AFP family
 </a>
 and consists of a homodimer. Each monomer folds exactly in half so that its N-and C-termini are side by side, hence the dimer looks like a 4-helix rod. It is composed of tandem 11-residue repeats that exhibit the [T/I]-x3-A-x3-A-x2 motif, where x is any residue. The conserved threonine/isoleucine and alanine residues in this motif have been shown
 <a href="http://www.ncbi.nlm.nih.gov/pubmed/10601644">
  to bind ice
 </a>
 in monomeric type-I AFPs. In the 3D structure, the internal space generated by the packing of the 4 helices in the 11-residue repeat regions is just wide enough to accommodate a single layer of water. Amazingly, the water layer that occupies the gap consists of over 400 molecules forming an extensive, mainly polypentagonal network. As is the case for most globular proteins, Maxi internal residues are nonpolar, mainly alanines, which obviously is far from optimal for hydrophilic contacts. To overcome this problem, Maxi takes advantage of its backbone carboxyl groups to anchor water molecules and the whole structure is stabilized by water-mediated hydrogen bonding rather than by direct protein association. The positioned water molecules extend outwards between all 4 helices from the core to the surface and they form a network of ordered molecules at the periphery. As a result, this rather hydrophobic protein remains highly solvated and freely soluble in flounder blood under physiological conditions, i.e. at low temperatures. When the temperature rises above 16°C, Maxi irreversibly denatures.
</p>
<p>
 Another surprise came from the observation that the predicted ice-binding residues, expected to face the protein exterior, actually occur on the inward-pointing surfaces of all 4 helices where they cooperate to form and anchor the interior ordered waters.
 <a href="http://www.ncbi.nlm.nih.gov/pubmed/24531972">
  How then does Maxi bind to ice?
 </a>
 The current working hypothesis is that the positioned water molecules that extend outwards may form a network available to merge and freeze with the quasi-liquid layer on the surface of ice.
</p>
<p>
 As of this release, the
 <a href="http://www.uniprot.org/uniprot/B1P0S1">
  winter flounder antifreeze protein Maxi
 </a>
 has been annotated and integrated into UniProtKB/Swiss-Prot. All antifreeze proteins available in UniProtKB/Swiss-Prot can be retrieved with the keyword
 <a href="http://www.uniprot.org/uniprot/?query=keyword:KW-0047">
  'Antifreeze protein'
 </a>
 .
</p>
<h2 id="UniProtKB_news">
 UniProtKB news
</h2>
<h3 id="Update_of_ECO_mapping_for_evidence">
 Update of ECO mapping for evidence
</h3>
<p>
 In 2011, we have started to use the
 <a href="http://bioportal.bioontology.org/ontologies/ECO">
  Evidence Codes Ontology
 </a>
 (ECO) to describe the evidence for UniProtKB annotations. Since then, this ontology has been extended and the GO Consortium has published a
 <a href="http://purl.obolibrary.org/obo/eco/gaf-eco-mapping.txt">
  mapping
 </a>
 of their
 <a href="http://www.geneontology.org/GO.evidence.shtml">
  GO evidence codes
 </a>
 to ECO. We have adapted our mapping to ECO accordingly to have equivalent evidence codes for UniProtKB and GO annotations. How this affects different UniProtKB distribution formats is described below.
</p>
<h4 id="XML_and_DAS_format">
 XML and DAS format
</h4>
<p>
 In these two formats, ECO codes are used to describe the evidence for UniProtKB annotations. In the
 <a href="http://www.uniprot.org/docs/uniprot.xsd">
  UniProtKB XML format
 </a>
 , an evidence is represented by an
 <code>
  evidence
 </code>
 element with a
 <code>
  type
 </code>
 attribute whose value is an ECO code. In the
 <a href="http://www.ebi.ac.uk/das-srv/uniprot/das/uniprot">
  DAS (features) representation of UniProtKB
 </a>
 , an evidence is represented by a
 <code>
  METHOD
 </code>
 element with an optional
 <a href="http://www.biodas.org/documents/spec-1.6.html#ontology">
  <code>
   cvId
  </code>
  attribute
 </a>
 whose value is an ECO code.
</p>
<p>
 The table below shows the mapping of previous to new ECO codes.
</p>
<table>
 <tr>
  <th>
   Previous ECO code
  </th>
  <th>
   New ECO code
  </th>
 </tr>
 <tr>
  <td>
   <a href="http://bioportal.bioontology.org/ontologies/50043/?p=terms&amp;conceptid=ECO:0000001">
    ECO:0000001
   </a>
  </td>
  <td>
   <a href="http://bioportal.bioontology.org/ontologies/50043/?p=terms&amp;conceptid=ECO:0000305">
    ECO:0000305
   </a>
  </td>
 </tr>
 <tr>
  <td>
   <a href="http://bioportal.bioontology.org/ontologies/50043/?p=terms&amp;conceptid=ECO:0000006">
    ECO:0000006
   </a>
  </td>
  <td>
   <a href="http://bioportal.bioontology.org/ontologies/50043/?p=terms&amp;conceptid=ECO:0000269">
    ECO:0000269
   </a>
  </td>
 </tr>
 <tr>
  <td>
   <a href="http://bioportal.bioontology.org/ontologies/50043/?p=terms&amp;conceptid=ECO:0000034">
    ECO:0000034
   </a>
  </td>
  <td>
   <a href="http://bioportal.bioontology.org/ontologies/50043/?p=terms&amp;conceptid=ECO:0000303">
    ECO:0000303
   </a>
  </td>
 </tr>
 <tr>
  <td>
   <a href="http://bioportal.bioontology.org/ontologies/50043/?p=terms&amp;conceptid=ECO:0000044">
    ECO:0000044
   </a>
  </td>
  <td>
   <a href="http://bioportal.bioontology.org/ontologies/50043/?p=terms&amp;conceptid=ECO:0000250">
    ECO:0000250
   </a>
  </td>
 </tr>
 <tr>
  <td>
   <a href="http://bioportal.bioontology.org/ontologies/50043/?p=terms&amp;conceptid=ECO:0000203">
    ECO:0000203
   </a>
  </td>
  <td>
   <a href="http://bioportal.bioontology.org/ontologies/50043/?p=terms&amp;conceptid=ECO:0000501">
    ECO:0000501
   </a>
   and
   <a href="http://bioportal.bioontology.org/ontologies/50043/?p=terms&amp;conceptid=ECO:0000256">
    ECO:0000256
   </a>
  </td>
 </tr>
</table>
<p>
 The codes
 <a href="http://bioportal.bioontology.org/ontologies/50043/?p=terms&amp;conceptid=ECO:0000312">
  ECO:0000312
 </a>
 and
 <a href="http://bioportal.bioontology.org/ontologies/50043/?p=terms&amp;conceptid=ECO:0000313">
  ECO:0000313
 </a>
 remain unchanged.
</p>
<p>
 In the future, we will also use
 <a href="http://bioportal.bioontology.org/ontologies/50043/?p=terms&amp;conceptid=ECO:0000255">
  ECO:0000255
 </a>
 for UniProtKB annotations.
</p>
<h4 id="RDF_format">
 RDF format
</h4>
<p>
 In the
 <a href="ftp://ftp.uniprot.org/pub/databases/uniprot/current%5Frelease/rdf/core.owl">
  UniProtKB RDF format
 </a>
 , ECO codes are used to describe the evidence
 <br/>
 for UniProtKB and GO annotations. An evidence is represented by an
 <code>
  evidence
 </code>
 property whose value is an ECO code. The
 <code>
  evidence
 </code>
 property is part of an
 <code>
  attribution
 </code>
 object which is assigned to a UniProtKB or GO annotation via
 <a href="http://www.w3.org/TR/rdf-primer/#reification">
  reification
 </a>
 .
</p>
<p>
 The table below shows the mapping of previous to new ECO codes.
</p>
<table>
 <tr>
  <th>
   GO evidence code
  </th>
  <th>
   Previous ECO code
  </th>
  <th>
   New ECO code
  </th>
 </tr>
 <tr>
  <td>
   EXP
  </td>
  <td>
   <a href="http://bioportal.bioontology.org/ontologies/50043/?p=terms&amp;conceptid=ECO:0000006">
    ECO:0000006
   </a>
  </td>
  <td>
   <a href="http://bioportal.bioontology.org/ontologies/50043/?p=terms&amp;conceptid=ECO:0000269">
    ECO:0000269
   </a>
  </td>
 </tr>
 <tr>
  <td>
   IBA
  </td>
  <td>
   <a href="http://bioportal.bioontology.org/ontologies/50043/?p=terms&amp;conceptid=ECO:0000308">
    ECO:0000308
   </a>
  </td>
  <td>
   <a href="http://bioportal.bioontology.org/ontologies/50043/?p=terms&amp;conceptid=ECO:0000318">
    ECO:0000318
   </a>
  </td>
 </tr>
 <tr>
  <td>
   IBD
  </td>
  <td>
   <a href="http://bioportal.bioontology.org/ontologies/50043/?p=terms&amp;conceptid=ECO:0000214">
    ECO:0000214
   </a>
  </td>
  <td>
   <a href="http://bioportal.bioontology.org/ontologies/50043/?p=terms&amp;conceptid=ECO:0000319">
    ECO:0000319
   </a>
  </td>
 </tr>
 <tr>
  <td>
   IC
  </td>
  <td>
   <a href="http://bioportal.bioontology.org/ontologies/50043/?p=terms&amp;conceptid=ECO:0000001">
    ECO:0000001
   </a>
  </td>
  <td>
   <a href="http://bioportal.bioontology.org/ontologies/50043/?p=terms&amp;conceptid=ECO:0000305">
    ECO:0000305
   </a>
  </td>
 </tr>
 <tr>
  <td>
   IDA
  </td>
  <td>
   <a href="http://bioportal.bioontology.org/ontologies/50043/?p=terms&amp;conceptid=ECO:0000002">
    ECO:0000002
   </a>
  </td>
  <td>
   <a href="http://bioportal.bioontology.org/ontologies/50043/?p=terms&amp;conceptid=ECO:0000314">
    ECO:0000314
   </a>
  </td>
 </tr>
 <tr>
  <td>
   IEA
  </td>
  <td>
   <a href="http://bioportal.bioontology.org/ontologies/50043/?p=terms&amp;conceptid=ECO:0000203">
    ECO:0000203
   </a>
  </td>
  <td>
   <a href="http://bioportal.bioontology.org/ontologies/50043/?p=terms&amp;conceptid=ECO:0000501">
    ECO:0000501
   </a>
  </td>
 </tr>
 <tr>
  <td>
   IEP
  </td>
  <td>
   <a href="http://bioportal.bioontology.org/ontologies/50043/?p=terms&amp;conceptid=ECO:0000008">
    ECO:0000008
   </a>
  </td>
  <td>
   <a href="http://bioportal.bioontology.org/ontologies/50043/?p=terms&amp;conceptid=ECO:0000270">
    ECO:0000270
   </a>
  </td>
 </tr>
 <tr>
  <td>
   IGC
  </td>
  <td>
   <a href="http://bioportal.bioontology.org/ontologies/50043/?p=terms&amp;conceptid=ECO:0000177">
    ECO:0000177
   </a>
  </td>
  <td>
   <a href="http://bioportal.bioontology.org/ontologies/50043/?p=terms&amp;conceptid=ECO:0000317">
    ECO:0000317
   </a>
  </td>
 </tr>
 <tr>
  <td>
   IGI
  </td>
  <td>
   <a href="http://bioportal.bioontology.org/ontologies/50043/?p=terms&amp;conceptid=ECO:0000011">
    ECO:0000011
   </a>
  </td>
  <td>
   <a href="http://bioportal.bioontology.org/ontologies/50043/?p=terms&amp;conceptid=ECO:0000316">
    ECO:0000316
   </a>
  </td>
 </tr>
 <tr>
  <td>
   IKR
  </td>
  <td>
   <a href="http://bioportal.bioontology.org/ontologies/50043/?p=terms&amp;conceptid=ECO:0000216">
    ECO:0000216
   </a>
  </td>
  <td>
   <a href="http://bioportal.bioontology.org/ontologies/50043/?p=terms&amp;conceptid=ECO:0000320">
    ECO:0000320
   </a>
  </td>
 </tr>
 <tr>
  <td>
   IMP
  </td>
  <td>
   <a href="http://bioportal.bioontology.org/ontologies/50043/?p=terms&amp;conceptid=ECO:0000015">
    ECO:0000015
   </a>
  </td>
  <td>
   <a href="http://bioportal.bioontology.org/ontologies/50043/?p=terms&amp;conceptid=ECO:0000315">
    ECO:0000315
   </a>
  </td>
 </tr>
 <tr>
  <td>
   IPI
  </td>
  <td>
   <a href="http://bioportal.bioontology.org/ontologies/50043/?p=terms&amp;conceptid=ECO:0000021">
    ECO:0000021
   </a>
  </td>
  <td>
   <a href="http://bioportal.bioontology.org/ontologies/50043/?p=terms&amp;conceptid=ECO:0000353">
    ECO:0000353
   </a>
  </td>
 </tr>
 <tr>
  <td>
   IRD
  </td>
  <td>
   <a href="http://bioportal.bioontology.org/ontologies/50043/?p=terms&amp;conceptid=ECO:0000215">
    ECO:0000215
   </a>
  </td>
  <td>
   <a href="http://bioportal.bioontology.org/ontologies/50043/?p=terms&amp;conceptid=ECO:0000321">
    ECO:0000321
   </a>
  </td>
 </tr>
 <tr>
  <td>
   ISA
  </td>
  <td>
   <a href="http://bioportal.bioontology.org/ontologies/50043/?p=terms&amp;conceptid=ECO:0000200">
    ECO:0000200
   </a>
  </td>
  <td>
   <a href="http://bioportal.bioontology.org/ontologies/50043/?p=terms&amp;conceptid=ECO:0000247">
    ECO:0000247
   </a>
  </td>
 </tr>
 <tr>
  <td>
   ISM
  </td>
  <td>
   <a href="http://bioportal.bioontology.org/ontologies/50043/?p=terms&amp;conceptid=ECO:0000202">
    ECO:0000202
   </a>
  </td>
  <td>
   <a href="http://bioportal.bioontology.org/ontologies/50043/?p=terms&amp;conceptid=ECO:0000255">
    ECO:0000255
   </a>
  </td>
 </tr>
 <tr>
  <td>
   ISO
  </td>
  <td>
   <a href="http://bioportal.bioontology.org/ontologies/50043/?p=terms&amp;conceptid=ECO:0000201">
    ECO:0000201
   </a>
  </td>
  <td>
   <a href="http://bioportal.bioontology.org/ontologies/50043/?p=terms&amp;conceptid=ECO:0000266">
    ECO:0000266
   </a>
  </td>
 </tr>
 <tr>
  <td>
   ISS
  </td>
  <td>
   <a href="http://bioportal.bioontology.org/ontologies/50043/?p=terms&amp;conceptid=ECO:0000044">
    ECO:0000044
   </a>
  </td>
  <td>
   <a href="http://bioportal.bioontology.org/ontologies/50043/?p=terms&amp;conceptid=ECO:0000250">
    ECO:0000250
   </a>
  </td>
 </tr>
 <tr>
  <td>
   NAS
  </td>
  <td>
   <a href="http://bioportal.bioontology.org/ontologies/50043/?p=terms&amp;conceptid=ECO:0000034">
    ECO:0000034
   </a>
  </td>
  <td>
   <a href="http://bioportal.bioontology.org/ontologies/50043/?p=terms&amp;conceptid=ECO:0000303">
    ECO:0000303
   </a>
  </td>
 </tr>
 <tr>
  <td>
   ND
  </td>
  <td>
   <a href="http://bioportal.bioontology.org/ontologies/50043/?p=terms&amp;conceptid=ECO:0000035">
    ECO:0000035
   </a>
  </td>
  <td>
   <a href="http://bioportal.bioontology.org/ontologies/50043/?p=terms&amp;conceptid=ECO:0000307">
    ECO:0000307
   </a>
  </td>
 </tr>
 <tr>
  <td>
   RCA
  </td>
  <td>
   <a href="http://bioportal.bioontology.org/ontologies/50043/?p=terms&amp;conceptid=ECO:0000053">
    ECO:0000053
   </a>
  </td>
  <td>
   <a href="http://bioportal.bioontology.org/ontologies/50043/?p=terms&amp;conceptid=ECO:0000245">
    ECO:0000245
   </a>
  </td>
 </tr>
 <tr>
  <td>
   TAS
  </td>
  <td>
   <a href="http://bioportal.bioontology.org/ontologies/50043/?p=terms&amp;conceptid=ECO:0000033">
    ECO:0000033
   </a>
  </td>
  <td>
   <a href="http://bioportal.bioontology.org/ontologies/50043/?p=terms&amp;conceptid=ECO:0000304">
    ECO:0000304
   </a>
  </td>
 </tr>
</table>
<h3 id="Cross_references_for_isoform_sequences__RefSeq">
 Cross-references for isoform sequences: RefSeq
</h3>
<p>
 We have added isoform-specific cross-references to the
 <a href="http://www.ncbi.nlm.nih.gov/refseq/">
  RefSeq
 </a>
 database. The format of these cross-references is as described in release
 <a href="http://www.uniprot.org/news/2014/03/19/release">
  2014_03
 </a>
 .
</p>
<h3 id="Cross_references_to_MaxQB">
 Cross-references to MaxQB
</h3>
<p>
 Cross-references have been added to
 <a href="http://maxqb.biochem.mpg.de/mxdb/">
  MaxQB
 </a>
 , a database of large proteomics projects.
</p>
<p>
 MaxQB is available at
 <a href="http://maxqb.biochem.mpg.de/mxdb/">
  http://maxqb.biochem.mpg.de/mxdb/
 </a>
 .
</p>
<p>
 The format of the explicit links is:
</p>
<table>
 <tr>
  <th>
   Resource abbreviation
  </th>
  <td>
   MaxQB
  </td>
 </tr>
 <tr>
  <th>
   Resource identifier
  </th>
  <td>
   UniProtKB accession number.
  </td>
 </tr>
</table>
<p>
 Example:
 <a href="http://www.uniprot.org/uniprot/Q6ZSR9#section%5Fx-ref">
  Q6ZSR9
 </a>
</p>
<p>
 <a href="http://www.uniprot.org/uniprot/?query=database%3Amaxqb&amp;sort=score">
  Show all entries having a cross-reference to MaxQB.
 </a>
</p>
<h4 id="Text_format">
 Text format
</h4>
<p>
 Example:
 <a href="http://www.uniprot.org/uniprot/Q6ZSR9.txt">
  Q6ZSR9
 </a>
</p>
<pre>DR   MaxQB; Q6ZSR9; -.
</pre>
<h4 id="XML_format">
 XML format
</h4>
<p>
 Example:
 <a href="http://www.uniprot.org/uniprot/Q6ZSR9.xml">
  Q6ZSR9
 </a>
</p>
<pre>&lt;dbReference type="MaxQB" id="Q6ZSR9"/&gt;
</pre>
<h3 id="Removal_of_the_cross_references_to_ProtClustDB">
 Removal of the cross-references to ProtClustDB
</h3>
<p>
 Cross-references to ProtClustDB have been removed.
</p>
<h3 id="Changes_to_the__a_href___docs_humdisease__controlled_vocabulary_of_human_diseases__a_">
 Changes to the
 <a href="http://www.uniprot.org/docs/humdisease">
  controlled vocabulary of human diseases
 </a>
</h3>
<p>
 New diseases:
</p>
<ul>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-04043">
   Bone marrow failure syndrome 2
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-04044">
   Dowling-Degos disease 4
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-04040">
   Leukoencephalopathy with ataxia
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-04038">
   Mental retardation, autosomal recessive 41
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-04042">
   Morbid obesity and spermatogenic failure
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-04039">
   Neurodegeneration with brain iron accumulation 6
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-04037">
   Neuropathy, hereditary sensory, 1F
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-04046">
   Poikiloderma, hereditary fibrosing, with tendon contractures, myopathy, and pulmonary fibrosis
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-04035">
   Short-rib thoracic dysplasia 10 with or without polydactyly
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-04036">
   Short-rib thoracic dysplasia 11 with or without polydactyly
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-04045">
   Spastic paraplegia 61, autosomal recessive
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-04041">
   Warburg micro syndrome 4
  </a>
 </li>
</ul>
<p>
 Modified diseases:
</p>
<ul>
 <li>
  Asphyxiating thoracic dystrophy 2 -&gt;
  <a href="http://www.uniprot.org/diseases/DI-01192">
   Short-rib thoracic dysplasia 2 with or without polydactyly
  </a>
 </li>
 <li>
  Asphyxiating thoracic dystrophy 3 -&gt;
  <a href="http://www.uniprot.org/diseases/DI-02583">
   Short-rib thoracic dysplasia 3 with or without polydactyly
  </a>
 </li>
 <li>
  Asphyxiating thoracic dystrophy 4 -&gt;
  <a href="http://www.uniprot.org/diseases/DI-03067">
   Short-rib thoracic dysplasia 4 with or without polydactyly
  </a>
 </li>
 <li>
  Asphyxiating thoracic dystrophy 5 -&gt;
  <a href="http://www.uniprot.org/diseases/DI-03325">
   Short-rib thoracic dysplasia 5 with or without polydactyly
  </a>
 </li>
 <li>
  Bone marrow failure, familial -&gt;
  <a href="http://www.uniprot.org/diseases/DI-03471">
   Bone marrow failure syndrome 1
  </a>
 </li>
 <li>
  Mainzer-Saldino syndrome -&gt;
  <a href="http://www.uniprot.org/diseases/DI-03475">
   Short-rib thoracic dysplasia 9 with or without polydactyly
  </a>
 </li>
 <li>
  Short rib-polydactyly syndrome 2A -&gt;
  <a href="http://www.uniprot.org/diseases/DI-03018">
   Short-rib thoracic dysplasia 6 with or without polydactyly
  </a>
 </li>
 <li>
  Short rib-polydactyly syndrome 5 -&gt;
  <a href="http://www.uniprot.org/diseases/DI-03182">
   Short-rib thoracic dysplasia 7 with or without polydactyly
  </a>
 </li>
 <li>
  Short rib-polydactyly syndrome 6 -&gt;
  <a href="http://www.uniprot.org/diseases/DI-03926">
   Short-rib thoracic dysplasia 8 with or without polydactyly
  </a>
 </li>
</ul>
<p>
 Deleted diseases:
</p>
<ul>
 <li>
  Short rib-polydactyly syndrome 2B
 </li>
 <li>
  Short rib-polydactyly syndrome 3
 </li>
</ul>
<h2 id="UniParc_news">
 UniParc news
</h2>
<h3 id="UniParc_cross_references_with_multiple_taxonomy_identifiers">
 UniParc cross-references with multiple taxonomy identifiers
</h3>
<p>
 The UniParc XML format uses
 <code>
  dbReference
 </code>
 elements to represent cross-references to external database records that contain the same sequence as the UniParc record. Additional information about an external database record is provided with different types of
 <code>
  property
 </code>
 child elements, e.g. the species is represented with a
 <code>
  property
 </code>
 of the type
 <code>
  "NCBI_taxonomy_id"
 </code>
 that stores an
 <a href="http://www.ncbi.nlm.nih.gov/taxonomy">
  NCBI taxonomy identifier
 </a>
 in its
 <code>
  value
 </code>
 attribute. In the past, all external database records described a single species.
</p>
<p>
 Example:
</p>
<pre>&lt;dbReference type="REFSEQ" id="ZP_06545872" version_i="1" active="Y" version="1" created="2010-03-07" last="2013-07-18"&gt;
  &lt;property type="NCBI_GI" value="289827083"/&gt;
  &lt;property type="NCBI_taxonomy_id" value="496064"/&gt;
&lt;/dbReference&gt;
&lt;dbReference type="REFSEQ" id="ZP_18488583" version_i="1" active="Y" version="1" created="2012-11-25" last="2013-07-18"&gt;
  &lt;property type="NCBI_GI" value="425085490"/&gt;
  &lt;property type="NCBI_taxonomy_id" value="1203546"/&gt;
&lt;/dbReference&gt;
</pre>
<p>
 With the introduction of
 <a href="ftp://ftp.ncbi.nlm.nih.gov/refseq/release/announcements/WP-proteins-06.10.2013.pdf">
  WP-accessions in the NCBI Reference Sequence Project (RefSeq) database
 </a>
 , UniParc needs to represent more than one species per
 <code>
  dbReference
 </code>
 element.
</p>
<p>
 Example:
</p>
<pre>&lt;dbReference type="REFSEQ" id="WP_001144069" version_i="1" active="Y" version="1" created="2013-07-19" last="2013-11-12"&gt;
  &lt;property type="NCBI_GI" value="447066813"/&gt;
  &lt;property type="NCBI_taxonomy_id" value="496064"/&gt;
  &lt;property type="NCBI_taxonomy_id" value="1203546"/&gt;
&lt;/dbReference&gt;
</pre>
<p>
 This change did not affect the
 <a href="http://www.uniprot.org/docs/uniparc.xsd">
  UniParc XSD
 </a>
 , but may nevertheless require code changes.
</p>