<h2 id="Headline">
 Headline
</h2>
<h3 id="Small_is_beautiful__and_useful_">
 Small is beautiful (and useful)
</h3>
<p>
 In large scale studies, small proteins tend to be overlooked. They are difficult to predict using software tools and they often escape detection by mass spectrometry. When cDNA sequences are submitted, short coding sequences (CDS) are only rarely annotated and hence do not appear in any protein databases, including UniProtKB/TrEMBL or GenPept, and their nucleotide sequences can be tagged as 'non-coding RNAs'. In UniProtKB/Swiss-Prot, we are aware of the problem, but we are often reluctant to annotate uncharacterized small ORFs, fearing to introduce imaginary sequences in a database we wish to be as reliable as possible. That is why we are thrilled when new data become available that allow us to fill the gap.
</p>
<p>
 This happened a few months ago, with the publication of
 <a href="http://www.ncbi.nlm.nih.gov/pubmed/24316148,24407481">
  2 articles
 </a>
 that brought the
 <a href="https://www.ebi.ac.uk/ena/browser/view/AK092578">
  'noncoding transcript' AK092578
 </a>
 under the spotlight.
 <a href="http://www.ncbi.nlm.nih.gov/pubmed/24407481">
  Pauli et al.
 </a>
 were investigating inductive events during early embryogenesis in
 <a href="http://www.uniprot.org/taxonomy/7955">
  zebrafish
 </a>
 . In order to find new signaling peptides, they sequenced RNAs extracted from embryos at different developmental stages and combined this approach with ribosome profiling to select for transcripts most likely to be translated. This led to the discovery of 399 novel coding genes. 28 of them contained a signal peptide, but no transmembrane domain, making them good candidates for signaling proteins. Pauli et al. focused their attention on one of them,
 <a href="http://www.uniprot.org/uniprot/P0DMC2">
  apela
 </a>
 , that they called
 <em>
  toddler
 </em>
 , encoded by AK092578, so far considered to be a
 <a href="http://www.ncrna.org/glocal/cgi-bin/hgGene?hgg%5Fgene=uc003iqv.1&amp;hgg%5Fprot=&amp;hgg%5Fchrom=chr4&amp;hgg%5Fstart=166017785&amp;hgg%5Fend=166038125&amp;hgg%5Ftype=knownGene&amp;db=hg18&amp;hgsid=152223">
  noncoding transcript
 </a>
 . A few weeks earlier,
 <a href="http://www.ncbi.nlm.nih.gov/pubmed/24316148">
  Chng et al.
 </a>
 had already published the identification of the same protein, which they named
 <em>
  elabela
 </em>
 .
</p>
<p>
 Apela is a highly conserved protein among vertebrates; this conservation is particularly striking in the 30 amino acid long mature peptide, the last 13 residues being nearly invariant in all vertebrate species studied. Apela is expressed in the zygote, with a peak during gastrulation, and becomes undetectable by 4 days post-fertilization. Its disruption leads to a dramatic phenotype, including small or absent hearts, posterior accumulation of blood cells, malformed pharyngeal endoderm, and abnormal left-right positioning and formation of the liver. Most mutant embryos eventually die between 5 and 7 days of development. Interestingly, this phenotype was reminiscent of that observed for apelin receptor (aplnr) deficiency.
</p>
<p>
 The pathway leading to aplnr activation that could explain the observed mutant phenotype remained unsolved for several years. Indeed,
 <a href="http://www.ncbi.nlm.nih.gov/pubmed/17336906">
  aplnr disruption in zebrafish
 </a>
 demonstrated that aplnr was required prior to the onset of gastrulation for proper cardiac morphogenesis, but its known ligand, apln, was not expressed until midgastrulation, too late to play a role in such a very early event. Along the same line, it had been reported that
 <a href="http://www.ncbi.nlm.nih.gov/pubmed/19767528">
  Aplnr mutant animals
 </a>
 were not born in the expected Mendelian ratio, and many showed cardiovascular developmental defects, while Apln-deficient mice were viable, fertile, and showed normal development. Taken together, these observations suggested that Aplnr might have yet another ligand, expressed very early in embryonic development. The newly discovered apela protein seemed to fulfill the conditions and, using different strategies, both groups convincingly showed that apela is indeed aplnr's first ligand.
</p>
<p>
 <a href="http://www.uniprot.org/uniprot/?query=gene:apela+and+reviewed:yes">
  Human, mouse and zebrafish Apela orthologs
 </a>
 have been updated accordingly and these entries are now available.
</p>
<h2 id="UniProtKB_news">
 UniProtKB news
</h2>
<h3 id="Evidence_in_the_UniProtKB_flat_file_format">
 Evidence in the UniProtKB flat file format
</h3>
<p>
 The
 <a href="http://www.uniprot.org/manual/evidences">
  evidence
 </a>
 for annotations in UniProtKB entries has been available for several years in the XML and RDF representation of the data and we have now added this information also to the text format (aka flat file format).
</p>
<h4 id="Representation_of_evidence">
 Representation of evidence
</h4>
<p>
 This section describes how evidence is represented, independently of the context in which they can be found.
</p>
<p>
 An individual evidence description consists of a mandatory evidence type, represented by a code from the
 <a href="http://www.evidenceontology.org/">
  Evidence Codes Ontology
 </a>
 (ECO) and, where applicable, the source of the data which is usually another database record that is represented by the database name and record identifier, but in the case of publications that are not in PubMed we indicate instead the corresponding UniProtKB reference number.
</p>
<p>
 Examples:
</p>
<ul>
 <li>
  An evidence type without source:
  <code>
   {type}
  </code>
  , e.g.
  <pre>{ECO:0000305}
{ECO:0000250}
{ECO:0000255}
</pre>
 </li>
</ul>
<ul>
 <li>
  An evidence type with source:
  <code>
   {type|source}
  </code>
  , e.g.
  <pre>{ECO:0000269|PubMed:10433554}
{ECO:0000303|Ref.6}
{ECO:0000305|PubMed:16683188}
{ECO:0000250|UniProtKB:Q8WUF5}
{ECO:0000312|EMBL:BAG16761.1}
{ECO:0000313|EMBL:BAG16761.1}
{ECO:0000255|HAMAP-Rule:MF_00205}
{ECO:0000256|HAMAP-Rule:MF_00205}
{ECO:0000244|PDB:1K83}
{ECO:0000213|PDB:1K83}
</pre>
 </li>
</ul>
<ul>
 <li>
  Several evidence attributions:
  <code>
   {type|source, type|source, ...}
  </code>
  , e.g.
  <pre>{ECO:0000269|PubMed:10433554, ECO:0000303|Ref.6}
</pre>
 </li>
</ul>
<h4 id="Change_of_the_representation_of_different_line_and_annotation_types">
 Change of the representation of different line and annotation types
</h4>
<p>
 This section describes in which line and annotation types evidence may be found and where it is placed. We use here the symbolic representation
 <code>
  {evidence}
 </code>
 as a placeholder for all evidence representations that are described in the previous section.
</p>
<h5 id="DE_lines">
 DE lines
</h5>
<p>
 Evidence may be found at the end of
 <a href="http://www.uniprot.org/news/2008/07/22/release">
  subcategory fields
 </a>
 , e.g.
</p>
<pre>DE   RecName: Full=Palmitoyl-protein thioesterase-dolichyl pyrophosphate phosphatase fusion 1 {evidence};
DE   Contains:
DE     RecName: Full=Palmitoyl-protein thioesterase {evidence};
DE              Short=PPT {evidence};
DE              EC=3.1.2.22 {evidence};
DE     AltName: Full=Palmitoyl-protein hydrolase {evidence};
DE   Contains:
DE     RecName: Full=Dolichyldiphosphatase {evidence};
DE              EC=3.6.1.43 {evidence};
DE     AltName: Full=Dolichyl pyrophosphate phosphatase {evidence};
DE   Flags: Precursor;
</pre>
<h5 id="GN_lines">
 GN lines
</h5>
<p>
 Evidence may be found after each gene designation, e.g.
</p>
<pre>GN   Name=cysA1 {evidence}; Synonyms=cysA {evidence};
GN   OrderedLocusNames=Rv3117 {evidence}, MT3199 {evidence};
GN   ORFNames=MTCY164.27 {evidence};
GN   and
GN   Name=cysA2 {evidence}; OrderedLocusNames=Rv0815c {evidence}, MT0837
GN   {evidence}; ORFNames=MTV043.07c {evidence};
</pre>
<h5 id="OG_lines">
 OG lines
</h5>
<p>
 Evidence may be found after an organelle or plasmid, e.g.
</p>
<pre>OG   Mitochondrion {evidence}.
</pre>
<pre>OG   Plasmid pWR100 {evidence}, Plasmid pINV_F6_M1382 {evidence}, and
OG   Plasmid pCP301 {evidence}.
</pre>
<h5 id="OX_lines">
 OX lines
</h5>
<p>
 Evidence may be found after the taxonomy identifier, e.g.
</p>
<pre>OX   NCBI_TaxID=9606 {evidence};
</pre>
<h5 id="RN_lines">
 RN lines
</h5>
<p>
 Evidence may be found after the reference number, e.g.
</p>
<pre>RN   [1] {evidence}
</pre>
<h5 id="RC_lines">
 RC lines
</h5>
<p>
 Evidence may be found after each value, e.g.
</p>
<pre>RC   STRAIN=C57BL/6J {evidence}, and DBA/2J {evidence}; TISSUE=Brain
RC   {evidence};
</pre>
<h5 id="KW_lines">
 KW lines
</h5>
<p>
 Evidence may be found after each keyword, e.g.
</p>
<pre>KW   ATP-binding {evidence}; Cell cycle {evidence}; Cell division {evidence};
KW   DNA replication {evidence};
</pre>
<h5 id="CC_lines">
 CC lines
</h5>
<p>
 The evidence location depends on the annotation type.
</p>
<p>
 <strong>
  Unstructured annotations:
 </strong>
</p>
<p>
 Evidence may initially be found at the end of the annotations because this is how they have historically been attributed, e.g.
</p>
<pre>CC   -!- FUNCTION: Possesses kinase activity. May be involved in
CC       trafficking and/or processing of RNA. {evidence}.
</pre>
<p>
 At a later time, we intend to start attributing evidence at a more fine-grained level by placing them behind the sentences or paragraphs to which they apply, e.g.
</p>
<pre>CC   -!- FUNCTION: Possesses kinase activity. {evidence}. May be involved
CC       in trafficking and/or processing of RNA. {evidence}.
</pre>
<p>
 <strong>
  Structured annotations:
 </strong>
</p>
<p>
 <ins>
  ALTERNATIVE PRODUCTS:
 </ins>
</p>
<p>
 Evidence may be found behind the values of the
 <code>
  Name=
 </code>
 and
 <code>
  Synonyms=
 </code>
 fields. It may also be found in
 <code>
  Comment=
 </code>
 and
 <code>
  Note=
 </code>
 fields where it is placed as in unstructured annotations, e.g.
</p>
<pre>CC   -!- ALTERNATIVE PRODUCTS:
CC       Event=Alternative splicing; Named isoforms=13;
CC         Comment=Additional isoforms seem to exist. {evidence};
CC       Name=1 {evidence}; Synonyms=LST1/A {evidence};
CC         IsoId=O00453-1; Sequence=Displayed;
..
CC       Name=12;
CC         IsoId=O00453-12; Sequence=VSP_047367;
CC         Note=No experimental confirmation available. {evidence};
</pre>
<p>
 <ins>
  BIOPHYSICOCHEMICAL PROPERTIES:
 </ins>
</p>
<p>
 In the structured subtopics
 <code>
  Absorption
 </code>
 and
 <code>
  Kinetic parameters
 </code>
 evidence may be found at the end of the
 <code>
  Abs(max)=
 </code>
 ,
 <code>
  KM=
 </code>
 and
 <code>
  Vmax=
 </code>
 fields. It may also be found in
 <code>
  Note=
 </code>
 fields and the unstructured subtopics
 <code>
  pH dependence
 </code>
 ,
 <code>
  Redox potential
 </code>
 and
 <code>
  Temperature dependence
 </code>
 , where it is placed as in unstructured annotations, e.g.
</p>
<pre>CC   -!- BIOPHYSICOCHEMICAL PROPERTIES:
CC       Absorption:
CC         Abs(max)=465 nm {evidence};
CC         Note=The above maximum is for the oxidized form. Shows a maximal
CC         peak at 330 nm in the reduced form. These absorption peaks are
CC         for the tryptophylquinone cofactor. {evidence};
CC       Kinetic parameters:
CC         KM=5.4 uM for tyramine {evidence};
CC         Vmax=17 umol/min/mg enzyme {evidence};
CC         Note=The enzyme is substrate inhibited at high substrate
CC         concentrations (Ki=1.08 mM for tyramine). {evidence};

CC   -!- BIOPHYSICOCHEMICAL PROPERTIES:
CC       pH dependence:
CC         Optimum pH is 7-8 for ATPase activity. Is more active at pH 8 to
CC         10 than at pH 5.5. {evidence};
CC       Temperature dependence:
CC         Optimum temperature is 80 degrees Celsius for ATPase activity.
CC         {evidence};
</pre>
<p>
 <ins>
  RNA EDITING:
 </ins>
</p>
<p>
 Evidence may be found behind the modified positions as well as in the optional
 <code>
  Note=
 </code>
 field where it is placed as in unstructured annotations, e.g.
</p>
<pre>CC   -!- RNA EDITING: Modified_positions=207 {evidence}; Note=Partially
CC       edited. Target of Adar. {evidence};
</pre>
<p>
 (Please note that we have taken this occasion to make an additional small format change to this annotation type: We have replaced the full-stop at the end of the annotation with a semi-colon to be consistent with other structured annotation types that consist of a list of
 <code>
  Field=Value;
 </code>
 items.)
</p>
<p>
 <ins>
  MASS SPECTROMETRY:
 </ins>
</p>
<p>
 In MASS SPECTROMETRY annotations the same evidence applies to all fields (incl. the optional
 <code>
  Note=
 </code>
 field) and all evidence attributions are thus displayed in a separate field instead of adding them at the end of each field. A new
 <code>
  Evidence=
 </code>
 field has replaced the previously existing
 <code>
  Source=
 </code>
 field, e.g.
</p>
<pre>CC   -!- MASS SPECTROMETRY: Mass=2189.4; Method=Electrospray; Range=167-
CC       186; Note=Monophosphorylated.; Evidence={evidence};
</pre>
<p>
 <ins>
  SEQUENCE CAUTION:
 </ins>
</p>
<p>
 In SEQUENCE CAUTION annotations the same evidence applies to all fields (incl. the optional
 <code>
  Note=
 </code>
 field) and all evidence is thus displayed in a separate new
 <code>
  Evidence=
 </code>
 field instead of being added at the end of each field, e.g.
</p>
<pre>CC   -!- SEQUENCE CAUTION:
CC       Sequence=AAL25396.1; Type=Miscellaneous discrepancy; Note=Intron retention.; Evidence={evidence};
CC       Sequence=ABF70206.1; Type=Miscellaneous discrepancy; Note=Intron retention.; Evidence={evidence};
CC       Sequence=CAA32567.1; Type=Erroneous gene model prediction; Evidence={evidence};
CC       Sequence=CAA32568.1; Type=Erroneous gene model prediction; Evidence={evidence};
</pre>
<p>
 <ins>
  SUBCELLULAR LOCATION:
 </ins>
</p>
<p>
 Evidence may be found at the same places where previously the non-experimental qualifiers
 <code>
  By similarity
 </code>
 ,
 <code>
  Probable
 </code>
 and
 <code>
  Potential
 </code>
 were displayed (see
 <a href="http://www.uniprot.org/news/2007/10/23/release">
  Syntax modification of the 'Subcellular location' subtopic
 </a>
 ) as well as in the optional
 <code>
  Note=
 </code>
 field where it is placed as in unstructured annotations, e.g.
</p>
<pre>CC   -!- SUBCELLULAR LOCATION: Golgi apparatus, trans-Golgi network
CC       membrane {evidence}; Multi-pass membrane protein {evidence}.
CC       Note=Predominantly found in the trans-Golgi network (TGN). Not
CC       redistributed to the plasma membrane in response to elevated
CC       copper levels. {evidence}.
CC   -!- SUBCELLULAR LOCATION: Isoform 2: Cytoplasm {evidence}.
CC   -!- SUBCELLULAR LOCATION: WND/140 kDa: Mitochondrion {evidence}.
</pre>
<p>
 <ins>
  DISEASE:
 </ins>
</p>
<p>
 Evidence may be found at end of the disease description as well as in the optional
 <code>
  Note=
 </code>
 field where it is placed as in unstructured annotations, e.g.
</p>
<pre>CC   -!- DISEASE: Sarcoidosis 1 (SS1) [MIM:181000]: An idiopathic,
CC       systemic, inflammatory disease characterized by the formation of
CC       immune granulomas in involved organs. Granulomas predominantly
CC       invade the lungs and the lymphatic system, but also skin, liver,
CC       spleen, eyes and other organs may be involved. {evidence}.
CC       Note=Disease susceptibility is associated with variations
CC       affecting the gene represented in this entry. {evidence}.
</pre>
<h5 id="FT_lines">
 FT lines
</h5>
<p>
 Evidence may be found at the end of the feature description, e.g.
</p>
<pre>FT   VARIANT     341    341       P -&gt; L (in AH2; strongly reduced
FT                                activity). {evidence}.
FT                                /FTId=VAR_065665.
FT   CONFLICT     52     53       RT -&gt; KI (in Ref. 8; AAD14329).
FT                                {evidence}.
</pre>
<h3 id="Changes_to_the__a_href___docs_humdisease__controlled_vocabulary_of_human_diseases__a_">
 Changes to the
 <a href="http://www.uniprot.org/docs/humdisease">
  controlled vocabulary of human diseases
 </a>
</h3>
<p>
 New diseases:
</p>
<ul>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-04153">
   Amelogenesis imperfecta, hypomaturation type, 2A5
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-04138">
   Arthrogryposis, distal, 3
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-04150">
   Bleeding disorder, platelet-type 18
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-04152">
   Cholestasis, progressive familial intrahepatic, 4
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-04144">
   Ciliary dyskinesia, primary, 29
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-04145">
   Epileptic encephalopathy, early infantile, 24
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-04142">
   Facial dysmorphism, lens dislocation, anterior segment abnormalities, and spontaneous filtering blebs
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-04139">
   Giant axonal neuropathy 2, autosomal dominant
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-04148">
   Hypotrichosis 12
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-04143">
   Limb-girdle muscular dystrophy 1F
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-04140">
   Marden-Walker syndrome
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-04136">
   Melanoma, cutaneous malignant 10
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-04149">
   Mental retardation, autosomal dominant 28
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-04146">
   Microphthalmia, syndromic, 14
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-04147">
   Myopathy, tubular aggregate, 2
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-04141">
   Neu-Laxova syndrome
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-04137">
   Spastic ataxia 1, autosomal dominant
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-04151">
   Tatton-Brown-Rahman syndrome
  </a>
 </li>
</ul>
<p>
 Modified diseases:
</p>
<ul>
 <li>
  Choreoathetosis hypothyroidism, and neonatal respiratory distress -&gt;
  <a href="http://www.uniprot.org/diseases/DI-01345">
   Choreoathetosis and congenital hypothyroidism with or without pulmonary dysfunction
  </a>
 </li>
 <li>
  Giant axonal neuropathy -&gt;
  <a href="http://www.uniprot.org/diseases/DI-01658">
   Giant axonal neuropathy 1, autosomal recessive
  </a>
 </li>
 <li>
  Pallister-Hall syndrome -&gt;
  <a href="http://www.uniprot.org/diseases/DI-02122">
   Pallister-Hall syndrome 1
  </a>
 </li>
</ul>
<p>
 Deleted disease:
</p>
<ul>
 <li>
  Glycogen storage disease 14
 </li>
</ul>
<h3 id="Changes_to__a_href___docs_keywlist__keywords__a_">
 Changes to
 <a href="http://www.uniprot.org/docs/keywlist">
  keywords
 </a>
</h3>
<p>
 New keyword:
</p>
<ul>
 <li>
  <a href="http://www.uniprot.org/keywords/KW-1265">
   Chloride channel impairing toxin
  </a>
 </li>
</ul>
<p>
 Modified keyword:
</p>
<ul>
 <li>
  Chloride channel inhibitor -&gt;
  <a href="http://www.uniprot.org/keywords/KW-0870">
   Voltage-gated chloride channel impairing toxin
  </a>
 </li>
</ul>