<h2 id="Headline">
 Headline
</h2>
<h3 id="Lark_or_owl__PER3_is_the_answer">
 Lark or owl? PER3 is the answer
</h3>
<p>
 Unless you are like Napoleon who never needed more than 4 hours of sleep at a stretch, being both an early bird and a night owl, you certainly have a diurnal preference. It is not a simple matter of taste, it is a matter of genetics, involving the PER3 gene.
</p>
<p>
 In humans, the PER3 gene exists in 2 versions: a short one and a long one. The length variation depends upon the number of 18 amino-acid tandem repeats in the protein's C-terminus:
 <a href="http://www.ncbi.nlm.nih.gov/pubmed/12841365">
  4 in the short version, 5 in the long one
 </a>
 . Roughly
 <a href="http://www.ncbi.nlm.nih.gov/pubmed/16275768">
  10% of the population is homozygous for the long allele
 </a>
 (PER3 5/5) and 50% for the short allele (PER3 4/4). This polymorphism correlates significantly with extreme diurnal preference, the
 <a href="http://www.ncbi.nlm.nih.gov/pubmed/12841365,15700718">
  longer allele being associated with morningness
 </a>
 and the shorter allele with eveningness. In addition, PER3 5/5 individuals are
 <a href="http://www.ncbi.nlm.nih.gov/pubmed/24439663">
  more vulnerable to sleep deprivation
 </a>
 than their PER3 4/4 counterparts, exhibiting greater cognitive performance impairment. When allowed to take naps, PER3 5/5 individuals show a greater ability to sleep independently of circadian phase, suggesting that the polymorphism modifies the sleep homeostatic response without influencing circadian parameters.
</p>
<p>
 The molecular mechanism of this behavioral difference is not known and there was no animal model to investigate it until recently. Indeed, the 18 amino-acid polymorphism does not exist in non-primate mammals. Earlier this year,
 <a href="http://www.ncbi.nlm.nih.gov/pubmed/24577121">
  Hasan et al.
 </a>
 published a study in which they created 2 knock-in mice. These mice contained a "humanized" PER3 exon 18 with either the 4-repeat or 5-repeat allele. The transgenic mice exhibited a phenotypic response to sleep deprivation and recovery consistent with the observations made in humans. 816 genes were differentially expressed in the cortex of Per3 4/4 and Per3 5/5 mice and a similar amount in the hypothalamus. At least some of these genes seem to be involved in the regulation of, or response to, sleep, as well as in neuronal development and function. For instance, some isoforms of the
 <a href="http://www.uniprot.org/uniprot/Q9Z2Y3">
  Homer1
 </a>
 gene, a marker of sleep homeostasis, were up-regulated in the Per3 5/5 compared to the Per3 4/4 hypothalamus.
</p>
<p>
 With this tool in hand, we may be in a position to start identifying the genetic control of sleep architecture in humans and maybe unveil if Napoleon's sleep ability was a true genetic oddity, the result of his iron will or just a historical myth.
</p>
<p>
 As of this release, the
 <a href="http://www.uniprot.org/uniprot/P56645">
  human PER3 entry
 </a>
 has been updated in UniProtKB/Swiss-Prot.
</p>
<h2 id="UniProtKB_news">
 UniProtKB news
</h2>
<h3 id="Cross_references_to_CCDS">
 Cross-references to CCDS
</h3>
<p>
 Cross-references have been added to
 <a href="http://www.ncbi.nlm.nih.gov/CCDS/">
  CCDS
 </a>
 , the Consensus CDS project.
</p>
<p>
 CCDS is available at
 <a href="http://www.ncbi.nlm.nih.gov/CCDS/">
  http://www.ncbi.nlm.nih.gov/CCDS/CcdsBrowse.cgi
 </a>
 .
</p>
<p>
 The format of the explicit links is:
</p>
<table>
 <tr>
  <th>
   Resource abbreviation
  </th>
  <td>
   CCDS
  </td>
 </tr>
 <tr>
  <th>
   Resource identifier
  </th>
  <td>
   CCDS identifier
  </td>
 </tr>
</table>
<p>
 Cross-references to CCDS may be isoform-specific. The general format of isoform-specific cross-references was described in release
 <a href="http://www.uniprot.org/news/2014/03/19/release">
  2014_03
 </a>
 .
</p>
<p>
 Example:
 <a href="http://www.uniprot.org/uniprot/O70554#section%5Fx-ref">
  O70554
 </a>
</p>
<p>
 <a href="http://www.uniprot.org/uniprot/?query=database%3Accds&amp;sort=score">
  Show all entries having a cross-reference to CCDS.
 </a>
</p>
<h4 id="Text_format">
 Text format
</h4>
<p>
 Examples:
</p>
<p>
 <a href="http://www.uniprot.org/uniprot/O70554.txt">
  O70554
 </a>
 <pre>DR   CCDS; CCDS38509.1; -.
</pre>
</p>
<p>
 <a href="http://www.uniprot.org/uniprot/P00750.txt">
  P00750
 </a>
 <pre>DR   CCDS; CCDS6126.1; -. [P00750-1]
DR   CCDS; CCDS6127.1; -. [P00750-3]
</pre>
</p>
<h4 id="XML_format">
 XML format
</h4>
<p>
 Examples:
</p>
<p>
 <a href="http://www.uniprot.org/uniprot/O70554.xml">
  O70554
 </a>
 <pre>&lt;dbReference type="CCDS" id="CCDS38509.1"/&gt;
</pre>
</p>
<p>
 <a href="http://www.uniprot.org/uniprot/P00750.xml">
  P00750
 </a>
 <pre>&lt;dbReference type="CCDS" id="CCDS6126.1"&gt;
  &lt;molecule id="P00750-1"/&gt;
&lt;/dbReference&gt;
&lt;dbReference type="CCDS" id="CCDS6127.1"&gt;
  &lt;molecule id="P00750-3"/&gt;
&lt;/dbReference&gt;
</pre>
</p>
<h3 id="Cross_references_to_GeneReviews">
 Cross-references to GeneReviews
</h3>
<p>
 Cross-references have been added to
 <a href="http://www.ncbi.nlm.nih.gov/books/NBK1116/">
  GeneReviews
 </a>
 , a resource of expert-authored, peer-reviewed disease descriptions.
</p>
<p>
 GeneReviews is available at
 <a href="http://www.ncbi.nlm.nih.gov/books/NBK1116/">
  http://www.ncbi.nlm.nih.gov/books/NBK1116/
 </a>
 .
</p>
<p>
 The format of the explicit links is:
</p>
<table>
 <tr>
  <th>
   Resource abbreviation
  </th>
  <td>
   GeneReviews
  </td>
 </tr>
 <tr>
  <th>
   Resource identifier
  </th>
  <td>
   GeneReviews identifier
  </td>
 </tr>
</table>
<p>
 Example:
 <a href="http://www.uniprot.org/uniprot/O00555#section%5Fx-ref">
  O00555
 </a>
</p>
<p>
 <a href="http://www.uniprot.org/uniprot/?query=database%3Agenereviews&amp;sort=score">
  Show all entries having a cross-reference to GeneReviews.
 </a>
</p>
<h4 id="Text_format">
 Text format
</h4>
<p>
 Example:
 <a href="http://www.uniprot.org/uniprot/O00555.txt">
  O00555
 </a>
</p>
<pre>DR   GeneReviews; CACNA1A; -.
</pre>
<h4 id="XML_format">
 XML format
</h4>
<p>
 Example:
 <a href="http://www.uniprot.org/uniprot/O00555.xml">
  O00555
 </a>
</p>
<pre>&lt;dbReference type="GeneReviews" id="CACNA1A"/&gt;
</pre>
<h3 id="Changes_to_the__a_href___docs_humdisease__controlled_vocabulary_of_human_diseases__a_">
 Changes to the
 <a href="http://www.uniprot.org/docs/humdisease">
  controlled vocabulary of human diseases
 </a>
</h3>
<p>
 New diseases:
</p>
<ul>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-04047">
   Alzheimer disease 19
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-04052">
   Auriculocondylar syndrome 3
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-04065">
   Boucher-Neuhauser syndrome
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-04066">
   Coloboma, ocular, with or without hearing impairment, cleft lip/palate, and/or mental retardation
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-04048">
   Foveal hypoplasia 2
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-04049">
   Hyperphosphatasia with mental retardation syndrome 4
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-04050">
   Immunodeficiency 20
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-04067">
   Mental retardation, autosomal dominant 23
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-04059">
   Mental retardation, autosomal recessive 27
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-04058">
   Myopathy with extrapyramidal signs
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-04062">
   Pentosuria
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-04055">
   Polyarteritis nodosa
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-04053">
   Question mark ears, isolated
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-04060">
   Retinitis pigmentosa 23
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-04064">
   Retinitis pigmentosa 68
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-04068">
   Retinitis pigmentosa 69
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-04063">
   Richieri-Costa-Pereira syndrome
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-04051">
   Schwannomatosis 2
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-04056">
   Spastic paraplegia 64, autosomal recessive
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-04057">
   Spastic paraplegia 63, autosomal recessive
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-04054">
   Spinocerebellar ataxia, autosomal recessive, 15
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-04061">
   Spondylometaphyseal dysplasia with cone-rod dystrophy
  </a>
 </li>
</ul>
<p>
 Modified diseases:
</p>
<ul>
 <li>
  Cerebellar ataxia and hypogonadotropic hypogonadism -&gt;
  <a href="http://www.uniprot.org/diseases/DI-03788">
   Gordon Holmes syndrome
  </a>
 </li>
 <li>
  Foveal hypoplasia -&gt;
  <a href="http://www.uniprot.org/diseases/DI-01624">
   Foveal hypoplasia 1
  </a>
 </li>
 <li>
  Retinopathy exudative with bone marrow failure -&gt;
  <a href="http://www.uniprot.org/diseases/DI-00998">
   Dyskeratosis congenita, autosomal dominant, 5
  </a>
 </li>
 <li>
  Schwannomatosis -&gt;
  <a href="http://www.uniprot.org/diseases/DI-02287">
   Schwannomatosis 1
  </a>
 </li>
</ul>
<h3 id="Changes_to_the__a_href___docs_ptmlist__controlled_vocabulary_for_PTMs__a_">
 Changes to the
 <a href="http://www.uniprot.org/docs/ptmlist">
  controlled vocabulary for PTMs
 </a>
</h3>
<p>
 New term for the feature key 'Modified residue' ('MOD_RES' in the flat file):
</p>
<ul>
 <li>
  L-isoglutamyl histamine
 </li>
</ul>
<p>
 Modified term for the feature key 'Modified residue' ('MOD_RES' in the flat file):
</p>
<ul>
 <li>
  N6-crotonyl-L-lysine -&gt; N6-crotonyllysine
 </li>
</ul>
<h3 id="Changes_to__a_href___docs_keywlist__keywords__a_">
 Changes to
 <a href="http://www.uniprot.org/docs/keywlist">
  keywords
 </a>
</h3>
<p>
 New keywords:
</p>
<ul>
 <li>
  <a href="http://www.uniprot.org/keywords/KW-1261">
   Bacterial host gene expression shutoff by virus
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/keywords/KW-1263">
   Bacterial host transcription shutoff by virus
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/keywords/KW-1262">
   Eukaryotic host gene expression shutoff by virus
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/keywords/KW-1264">
   Viral host range expansion by genetic inversion
  </a>
 </li>
</ul>
<p>
 Modified keywords:
</p>
<ul>
 <li>
  Abi system evasion by virus -&gt;
  <a href="http://www.uniprot.org/keywords/KW-1259">
   Evasion of bacteria-mediated translation shutoff by virus
  </a>
 </li>
 <li>
  Host transcription shutoff by virus -&gt;
  <a href="http://www.uniprot.org/keywords/KW-1191">
   Eukaryotic host transcription shutoff by virus
  </a>
 </li>
 <li>
  Host translation shutoff by virus -&gt;
  <a href="http://www.uniprot.org/keywords/KW-1193">
   Eukaryotic host translation shutoff by virus
  </a>
 </li>
 <li>
  Inhibition of host translation factors by virus -&gt;
  <a href="http://www.uniprot.org/keywords/KW-1075">
   Inhibition of eukaryotic host translation factors by virus
  </a>
 </li>
 <li>
  Inhibition of host transcription initiation by virus -&gt;
  <a href="http://www.uniprot.org/keywords/KW-1111">
   Inhibition of eukaryotic host transcription initiation by virus
  </a>
 </li>
 <li>
  Viral primary attachment to host cell -&gt;
  <a href="http://www.uniprot.org/keywords/KW-1233">
   Viral attachment to host adhesion receptor
  </a>
 </li>
</ul>
<h2 id="UniParc_news">
 UniParc news
</h2>
<h3 id="UniParc_cross_references_with_protein_and_gene_names">
 UniParc cross-references with protein and gene names
</h3>
<p>
 The UniParc XML format uses
 <code>
  dbReference
 </code>
 elements to represent cross-references to external database records that contain the same sequence as the UniParc record. Additional information about an external database record is provided with different types of
 <code>
  property
 </code>
 child elements. We have introduced two new types,
 <code>
  "protein_name"
 </code>
 and
 <code>
  "gene_name"
 </code>
 , to show the preferred protein and gene name of external database records that provide this information. In this release we have added names for cross-references to UniProtKB and
 <a href="http://www.ncbi.nlm.nih.gov/RefSeq/">
  RefSeq
 </a>
 . For UniProtKB entries that have several protein or gene names, UniParc shows only the main one, which is the same name that is shown in the UniProtKB FASTA format. We will soon add names for cross-references to
 <a href="http://www.ebi.ac.uk/ena/">
  ENA
 </a>
 ,
 <a href="http://www.ensembl.org/">
  Ensembl
 </a>
 ,
 <a href="http://www.ensemblgenomes.org/">
  EnsemblGenomes
 </a>
 and model organism databases (
 <a href="http://flybase.bio.indiana.edu/">
  FlyBase
 </a>
 ,
 <a href="http://www.yeastgenome.org/">
  SGD
 </a>
 ,
 <a href="http://www.arabidopsis.org/">
  TAIR
 </a>
 ,
 <a href="http://www.wormbase.org/">
  WormBase
 </a>
 ).
</p>
<p>
 Examples:
</p>
<pre>&lt;dbReference type="UniProtKB/Swiss-Prot" id="P05067" version_i="3" active="Y" version="3" created="1991-11-01" last="2014-02-19"&gt;
  &lt;property type="NCBI_GI" value="112927"/&gt;
  &lt;property type="NCBI_taxonomy_id" value="9606"/&gt;
  &lt;property type="protein_name" value="Amyloid beta A4 protein"/&gt;
  &lt;property type="gene_name" value="APP"/&gt;
&lt;/dbReference&gt;
...
&lt;dbReference type="UniProtKB/Swiss-Prot protein isoforms" id="P05067-2" version_i="1" active="Y" created="2003-03-28" last="2014-02-19"&gt;
  &lt;property type="NCBI_taxonomy_id" value="9606"/&gt;
  &lt;property type="protein_name" value="Isoform APP305 of Amyloid beta A4 protein"/&gt;
  &lt;property type="gene_name" value="APP"/&gt;
&lt;/dbReference&gt;
</pre>
<p>
 This change did not affect the
 <a href="http://www.uniprot.org/docs/uniparc.xsd">
  UniParc XSD
 </a>
 , but may nevertheless require code changes.
</p>
<h2 id="FTP_site_news">
 FTP site news
</h2>
<p>
 Every folder on our
 <a href="ftp://ftp.uniprot.org">
  FTP server
 </a>
 now contains a file called RELEASE.metalink that specifies the size and MD5 checksum of every file in that folder, e.g.
 <br/>
 <a href="ftp://ftp.uniprot.org/pub/databases/uniprot/knowledgebase/RELEASE.metalink">
  ftp://ftp.uniprot.org/pub/databases/uniprot/knowledgebase/RELEASE.metalink
 </a>
</p>
<p>
 <a href="http://en.wikipedia.org/wiki/Metalink">
  Metalink
 </a>
 is an extensible metadata file format that describes one or more computer files available for download. It facilitates file verification and recovery from data corruption and lists alternate download sources (mirror URIs).
</p>
<p>
 Various command line download tools, e.g.
 <a href="http://curl.haxx.se/">
  cURL
 </a>
 version 7.30 or higher and
 <a href="http://aria2.sourceforge.net/">
  aria2
 </a>
 , support metalink.
</p>
<p>
 Example: The following command will download all files in the
 <code>
  current_release/
 </code>
 folder and verify their MD5 checksums:
</p>
<pre>curl --metalink ftp://ftp.uniprot.org/pub/databases/uniprot/current_release/RELEASE.metalink</pre>
<p>
 They will be downloaded from one of the alternative locations mentioned in the metalink file. If one FTP server goes down during a download, programs can automatically switch to another mirror location. Some programs can also download segments from several FTP locations at the same time, which can make downloads much faster.
</p>
<p>
 Please note that UniProt can be downloaded from the consortium member FTP sites at three different geographical locations:
</p>
<p>
 USA:
 <a href="ftp://ftp.uniprot.org/pub/databases/uniprot">
  ftp://ftp.uniprot.org/pub/databases/uniprot
 </a>
 <br/>
 UK:
 <a href="ftp://ftp.ebi.ac.uk/pub/databases/uniprot">
  ftp://ftp.ebi.ac.uk/pub/databases/uniprot
 </a>
 <br/>
 Switzerland:
 <a href="ftp://ftp.expasy.org/databases/uniprot">
  ftp://ftp.expasy.org/databases/uniprot
 </a>
</p>
<p>
 This information can be found in our
 <a href="http://www.uniprot.org/help/metalink">
  FAQ
 </a>
 .
</p>