<h2 id="Headline">
 Headline
</h2>
<h3 id="Mouse_attacks_">
 Mouse attacks!
</h3>
<p>
 In the arid lands of Arizona lives a fierce predator whose howls pierce the desert night, terrifying its prey. This predator is... a mouse,
 <a href="http://www.uniprot.org/taxonomy/38674">
  <em>
   Onychomys torridus
  </em>
 </a>
 , also called the grasshopper mouse. It may sound like a tale looming straight from the imagination of Tim Burton or
 <a href="https://www.youtube.com/watch?v=XcxKIJTb3Hg">
  Monthy Python
 </a>
 , but this mouse really exists. It is carnivorous and it regularly
 <a href="https://www.youtube.com/watch?v=-E35OMfL4A4">
  howls
 </a>
 just before a kill, although the emitted sound is more a sustained whistle than the actual howl of a wolf. Its prey is no less astonishing, including crickets, other rodents, tarantulas and bark scorpions (
 <a href="http://www.uniprot.org/taxonomy/218467">
  <em>
   Centruroides sculpturatus
  </em>
 </a>
 ).
</p>
<p>
 Bark scorpions are not easy prey. They are venomous and inflict intensely painful, sometimes lethal stings. Surprisingly grasshopper mice do not seem to be seriously bothered by that, and it takes little time before the scorpion is captured, killed and
 <a href="https://www.youtube.com/watch?v=HPR7TEyPQHI">
  eaten
 </a>
 . How can
 <em>
  O. torridus
 </em>
 ignore the venom, while common house mice are sensitive to it? Overall, grasshopper mice do feel pain normally, but when they are injected with scorpion venom or with a physiological saline solution in their hind paws, they are much more irritated by the control saline solution than by the venom. In grasshopper mice, bark scorpion venom acts as an analgesic.
</p>
<p>
 Venom from Buthidae scorpions initiates acute pain in sensitive mammals, such as house mice, rats and humans, by activating the voltage-gated sodium channel
 <a href="http://www.uniprot.org/uniprot/?query=gene:Scn9a+AND+reviewed:yes">
  Nav1.7/SCN9A
 </a>
 , but has no effect on the
 <a href="http://www.uniprot.org/uniprot/?query=gene:Scn10a+AND+reviewed:yes">
  Nav1.8/SCN10A
 </a>
 sodium channel.
 <a href="http://www.ncbi.nlm.nih.gov/pubmed/24159039">
  Recent experiments by Rowe et al.
 </a>
 on freshly isolated
 <em>
  O. torridus
 </em>
 <a href="http://en.wikipedia.org/wiki/Nociceptor">
  sensory neurons
 </a>
 showed that, in this species, the venom strongly inhibits Nav1.8/SCN10A Na+ currents. These Na+ currents are necessary for action potential sustained firing and propagation. By inhibiting Nav1.8/SCN10A, the scorpion venom blocks pain transmission to the central nervous system, and hence induces analgesia. The diametrically opposed response of rodents towards scorpion venom seems to be due to only 2 residues within the
 <a href="http://www.uniprot.org/uniprot/P0DMA5#section%5Fseq">
  Nav1.8/SCN10A sequence
 </a>
 . In
 <em>
  O. torridus
 </em>
 , a glutamate residue is found at position 859 (E-859) and a glutamine residue at position 862 (Q-862), while in species known to be sensitive to the venom, these positions are reversed: Q-859 and E-862. Site-directed mutagenesis of these 2 residues in the
 <em>
  O. torridus
 </em>
 sequence (Q859E/E862Q) abolished venom sensitivity. Conversely, mutation of the glutamine position in
 <em>
  Mus musculus
 </em>
 (Q861E) conferred inhibition by
 <em>
  C. sculpturatus
 </em>
 venom.
</p>
<p>
 Pain sensitivity is essential for survival, since it helps avoid damaging situations. Hence any change in pain perception has to be finely tuned in order not to be deleterious. O. torridus has evolved a brilliant strategy allowing it to exploit an abundant food resource in its environment, i.e. bark scorpions, while keeping intact its ability to feel the necessary pain.
</p>
<p>
 Persistent pain can turn into a nightmare and improving our understanding of pain signaling may be a tremendous help in the discovery of new analgesic drugs. Nav1.7/SCN9A is already under close investigation as a potential target for pain prevention. The new and very exciting study by Rowe et al. shows that the Nav1.8/SCN10A channel also plays a crucial key role in the transmission of pain signals and may be an interesting target for analgesic development.
</p>
<p>
 As of this release, the fully annotated
 <a href="http://www.uniprot.org/uniprot/P0DMA5">
  <em>
   O. torridus
  </em>
  Nav1.8/SCN10A protein
 </a>
 is available in UniProtKB/Swiss-Prot.
</p>
<h2 id="UniProtKB_news">
 UniProtKB news
</h2>
<h3 id="Removal_of_the_cross_references_to_IPI">
 Removal of the cross-references to IPI
</h3>
<p>
 Cross-references to IPI have been removed.
</p>
<p>
 IPI has closed in 2011. The last release is archived at
 <a href="ftp://ftp.ebi.ac.uk/pub/databases/IPI">
  ftp://ftp.ebi.ac.uk/pub/databases/IPI
 </a>
 .
</p>
<p>
 The
 <a href="http://www.ensembl.org/">
  Ensembl
 </a>
 and
 <a href="http://ensemblgenomes.org/">
  Ensembl Genomes
 </a>
 projects offer access to genomic data from vertebrate and non-vertebrate species respectively.
</p>
<p>
 Complete proteome data is available from
 <a href="http://www.uniprot.org/taxonomy/complete-proteomes">
  UniProtKB
 </a>
 .
</p>
<p>
 An archive of the last mapping table between UniProtKB and IPI is archived at
 <a href="ftp://ftp.uniprot.org/pub/databases/uniprot/previous%5Freleases/release-2014%5F01/">
  ftp://ftp.uniprot.org/pub/databases/uniprot/previous_releases/release-2014_01/
 </a>
 .
</p>
<h3 id="Documents_and_RSS_feeds_for_UniProt_Forthcoming_changes_and_News">
 Documents and RSS feeds for UniProt Forthcoming changes and News
</h3>
<p>
 We have replaced the documents sp_soon.htm ("UniProt Knowledgebase - Forthcoming changes") and xml_soon.htm ("UniProt Knowledgebase - Forthcoming changes in XML") by a searchable section
 <a href="http://www.uniprot.org/changes">
  Forthcoming changes
 </a>
 on our website to announce planned changes for all UniProt data sets and file formats in one place and to provide a common
 <a href="http://www.uniprot.org/changes/?query=&amp;format=rss">
  RSS feed
 </a>
 . The same information can also be downloaded from our
 <a href="ftp://ftp.uniprot.org/pub/databases/uniprot/current%5Frelease/changes.html">
  FTP site
 </a>
 .
</p>
<p>
 Changes that have been implemented are described in our "News archive", which can be searched in the
 <a href="http://www.uniprot.org/news">
  News
 </a>
 section of our website, followed via an
 <a href="http://www.uniprot.org/news/?query=&amp;format=rss">
  RSS feed
 </a>
 and downloaded from the
 <a href="ftp://ftp.uniprot.org/pub/databases/uniprot/current%5Frelease/news.html">
  FTP site
 </a>
 . These news include the historical contents of sp_news.htm ("What's new?"), but not that of xml_news.htm ("What's new in XML?"). The latter file was renamed to
 <a href="ftp://ftp.uniprot.org/pub/databases/uniprot/previous%5Freleases/xml%5Fnews%5Fprior%5F2014%5F01.html">
  xml_news_prior_2014_01.html
 </a>
 to archive the XML changes that were implemented before 2014. This file will no longer be updated.
</p>
<p>
 We have generated symbolic links on the FTP site for the files that have been replaced to give everyone time to update their FTP download procedures to the new files' locations:
</p>
<ul>
 <li>
  sp_soon.htm and xml_soon.htm -&gt;
  <a href="ftp://ftp.uniprot.org/pub/databases/uniprot/current%5Frelease/changes.html">
   ftp://ftp.uniprot.org/pub/databases/uniprot/current_release/changes.html
  </a>
 </li>
</ul>
<ul>
 <li>
  sp_news.htm and xml_news.htm -&gt;
  <a href="ftp://ftp.uniprot.org/pub/databases/uniprot/current%5Frelease/news.html">
   ftp://ftp.uniprot.org/pub/databases/uniprot/current_release/news.html
  </a>
 </li>
</ul>
<h3 id="New_version_of_DASty">
 New version of DASty
</h3>
<p>
 Our DAS web client
 <a href="http://www.ebi.ac.uk/dasty/">
  DASty
 </a>
 has been redesigned. DASty provides a visual representation of the compilation of protein annotations from different third-party sources. This allows users to get a global overview of all protein annotation available for their protein of interest, from UniProt as well as other sources. The "Third-party data" link that is available on each UniProtKB entry now leads to this new version of DASty. Any bookmarks should be updated accordingly. For instance, the "Third-party data" link for UniProt accession
 <a href="http://www.uniprot.org/uniprot/P05067">
  P05067
 </a>
 now links to http://www.ebi.ac.uk/dasty/client/index.html?q=P05067
</p>
<h3 id="Changes_to_the__a_href___docs_humdisease__controlled_vocabulary_of_human_diseases__a_">
 Changes to the
 <a href="http://www.uniprot.org/docs/humdisease">
  controlled vocabulary of human diseases
 </a>
</h3>
<p>
 New diseases:
</p>
<ul>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-03937">
   Alacrima, achalasia, and mental retardation syndrome
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-03940">
   Amyotrophic lateral sclerosis 19
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-03950">
   Candidiasis, familial, 8
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-03928">
   Ceroid lipofuscinosis, neuronal, 12
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-03936">
   Ciliary dyskinesia, primary, 26
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-03938">
   Ciliary dyskinesia, primary, 27
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-03944">
   Ciliary dyskinesia, primary, 28
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-03946">
   Cole disease
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-03947">
   Corneal dystrophy, Fuchs endothelial, 8
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-03953">
   Craniosynostosis 5
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-03930">
   Deafness, dystonia, and cerebral hypomyelination
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-03942">
   Hemochromatosis 5
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-03941">
   Immunodeficiency 13
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-03927">
   Mental retardation, autosomal dominant 21
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-03943">
   Mental retardation, autosomal recessive 37
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-03939">
   Mental retardation, autosomal recessive 38
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-03949">
   Mental retardation, X-linked 98
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-03931">
   Metacarpal 4-5 fusion
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-03952">
   Microphthalmia, isolated, with coloboma, 8
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-03951">
   Microphthalmia, syndromic, 12
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-03934">
   Mitochondrial DNA depletion syndrome 12, cardiomyopathic type
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-03925">
   Neurodegeneration with optic atrophy, childhood-onset
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-03935">
   Orofaciodigital syndrome 5
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-03948">
   Parkinsonism with spasticity, X-linked
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-03945">
   Phelan-McDermid syndrome
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-03929">
   Rhabdomyosarcoma, embryonal, 2
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-03926">
   Short rib-polydactyly syndrome 6
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-03932">
   Spinocerebellar ataxia 19
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-03933">
   Spinocerebellar ataxia 26
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-03954">
   Split-hand/foot malformation with long bone deficiency 3
  </a>
 </li>
</ul>
<p>
 Modified diseases:
</p>
<ul>
 <li>
  Albinism, oculocutaneous, 5 -&gt;
  <a href="http://www.uniprot.org/diseases/DI-03749">
   Albinism, oculocutaneous, 7
  </a>
 </li>
 <li>
  Amyotrophic lateral sclerosis 19 -&gt;
  <a href="http://www.uniprot.org/diseases/DI-03881">
   Amyotrophic lateral sclerosis 20
  </a>
 </li>
 <li>
  Basal ganglia calcification, idiopathic, 3 -&gt;
  <a href="http://www.uniprot.org/diseases/DI-03407">
   Basal ganglia calcification, idiopathic, 1
  </a>
 </li>
 <li>
  Capillary malformations, congenital, 1 -&gt;
  <a href="http://www.uniprot.org/diseases/DI-03786">
   Capillary malformations, congenital
  </a>
 </li>
 <li>
  CARD11 immunodeficiency -&gt;
  <a href="http://www.uniprot.org/diseases/DI-03761">
   Immunodeficiency 11
  </a>
 </li>
 <li>
  Dentinogenesis imperfecta 1 -&gt;
  <a href="http://www.uniprot.org/diseases/DI-01479">
   Dentinogenesis imperfecta, Shields type 2
  </a>
 </li>
 <li>
  Dentinogenesis imperfecta Shields type 3 -&gt;
  <a href="http://www.uniprot.org/diseases/DI-01478">
   Dentinogenesis imperfecta, Shields type 3
  </a>
 </li>
 <li>
  Hemochromatosis -&gt;
  <a href="http://www.uniprot.org/diseases/DI-01714">
   Hemochromatosis 1
  </a>
 </li>
 <li>
  Immune dysfunction with T-cell inactivation due to calcium entry defect 1 -&gt;
  <a href="http://www.uniprot.org/diseases/DI-01021">
   Immunodeficiency 9
  </a>
 </li>
 <li>
  Immune dysfunction with T-cell inactivation due to calcium entry defect 2 -&gt;
  <a href="http://www.uniprot.org/diseases/DI-02551">
   Immunodeficiency 10
  </a>
 </li>
 <li>
  Mitochondrial DNA depletion syndrome 1 -&gt;
  <a href="http://www.uniprot.org/diseases/DI-01984">
   Mitochondrial DNA depletion syndrome 1, MNGIE type
  </a>
 </li>
 <li>
  Rhabdomyosarcoma 1 -&gt;
  <a href="http://www.uniprot.org/diseases/DI-02267">
   Rhabdomyosarcoma, embryonal, 1
  </a>
 </li>
</ul>
<h3 id="Changes_to_the__a_href___docs_ptmlist__controlled_vocabulary_for_PTMs__a_">
 Changes to the
 <a href="http://www.uniprot.org/docs/ptmlist">
  controlled vocabulary for PTMs
 </a>
</h3>
<p>
 Modified terms for the feature key 'Cross-link' ('CROSSLNK' in the flat file):
</p>
<ul>
 <li>
  1-(tryptophan-3-yl)-tryptophan (Trp-Trp) (interchain) -&gt; 1-(tryptophan-3-yl)-tryptophan (Trp-Trp) (interchain with W-...)
 </li>
 <li>
  5'-tyrosyl-5'-aminotyrosine (Tyr-Tyr) (interchain) -&gt; 5'-tyrosyl-5'-aminotyrosine (Tyr-Tyr) (interchain with Y-...)
 </li>
 <li>
  Glycyl threonine ester (Gly-Thr) (interchain with G-...) -&gt; Glycyl threonine ester (Gly-Thr) (interchain with T-...)
 </li>
</ul>
<h3 id="Changes_to__a_href___docs_keywlist__keywords__a_">
 Changes to
 <a href="http://www.uniprot.org/docs/keywlist">
  keywords
 </a>
</h3>
<p>
 New keywords:
</p>
<ul>
 <li>
  <a href="http://www.uniprot.org/keywords/KW-1259">
   Abi system evasion by virus
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/keywords/KW-1231">
   Capsid inner membrane protein
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/keywords/KW-1232">
   Capsid decoration protein
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/keywords/KW-1257">
   CRISPR-cas system evasion by virus
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/keywords/KW-1238">
   Degradation of host capsule during virus entry
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/keywords/KW-1235">
   Degradation of host cell envelope components during virus entry
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/keywords/KW-1247">
   Degradation of host chromosome by virus
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/keywords/KW-1237">
   Degradation of host lipopolysaccharides during virus entry
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/keywords/KW-1236">
   Degradation of host peptidoglycans during virus entry
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/keywords/KW-1256">
   DNA end degradation evasion by virus
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/keywords/KW-1239">
   Fusion of virus membrane with host outer membrane
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/keywords/KW-1248">
   Inhibition of host DNA replication by virus
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/keywords/KW-1252">
   Latency-replication switch
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/keywords/KW-1254">
   Modulation of host virulence by virus
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/keywords/KW-1258">
   Restriction-modification system evasion by virus
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/keywords/KW-1260">
   Superinfection exclusion
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/keywords/KW-1240">
   Viral attachment to host cell flagellum
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/keywords/KW-1234">
   Viral attachment to host entry receptor
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/keywords/KW-1226">
   Viral base plate protein
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/keywords/KW-1242">
   Viral contractile tail ejection system
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/keywords/KW-1255">
   Viral exotoxin
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/keywords/KW-1249">
   Viral extrusion
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/keywords/KW-1253">
   Viral genome circularization
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/keywords/KW-1250">
   Viral genome excision
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/keywords/KW-1251">
   Viral latency
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/keywords/KW-1243">
   Viral long flexible tail ejection system
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/keywords/KW-1241">
   Viral penetration into host cytoplasm via pilus retraction
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/keywords/KW-1233">
   Viral primary attachment to host cell
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/keywords/KW-1244">
   Viral short tail ejection system
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/keywords/KW-1245">
   Viral tail assembly
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/keywords/KW-1246">
   Viral tail fiber assembly
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/keywords/KW-1230">
   Viral tail fiber protein
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/keywords/KW-1227">
   Viral tail protein
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/keywords/KW-1229">
   Viral tail sheath protein
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/keywords/KW-1228">
   Viral tail tube protein
  </a>
 </li>
</ul>
<p>
 Modified keywords:
</p>
<ul>
 <li>
  Capsid maturation -&gt;
  <a href="http://www.uniprot.org/keywords/KW-0917">
   Virion maturation
  </a>
 </li>
 <li>
  Phage lysis protein -&gt;
  <a href="http://www.uniprot.org/keywords/KW-0578">
   Host cell lysis by virus
  </a>
 </li>
 <li>
  Pilus-mediated viral adsorption onto host cell -&gt;
  <a href="http://www.uniprot.org/keywords/KW-1175">
   Viral attachment to host cell pilus
  </a>
 </li>
 <li>
  Viral capsid scaffolding protein -&gt;
  <a href="http://www.uniprot.org/keywords/KW-0118">
   Viral capsid assembly
  </a>
 </li>
 <li>
  Viral genome injection through bacterial membranes -&gt;
  <a href="http://www.uniprot.org/keywords/KW-1171">
   Viral genome ejection through host cell envelope
  </a>
 </li>
 <li>
  Viral penetration via permeabilization of host organellar membrane -&gt;
  <a href="http://www.uniprot.org/keywords/KW-1173">
   Viral penetration via permeabilization of host membrane
  </a>
 </li>
</ul>
<p>
 Deleted keyword:
</p>
<ul>
 <li>
  Phage maturation
 </li>
</ul>