<h2 id="Headline">
 Headline
</h2>
<h3 id="Epigenetics_in_the_spotlight">
 Epigenetics in the spotlight
</h3>
<p>
 In its active form, folate, commonly known as vitamin B9, is a methyl carrier, essential for the biosynthesis of methionine and nucleic acids, most notably thymine, but also purine bases. Methionine synthesis involves first the activation of methionine synthase (
 <a href="http://www.uniprot.org/uniprot/?query=gene:MTR+AND+name:Methionine+synthase+AND+reviewed:yes">
  MTR
 </a>
 ) by methionine synthase reductase (
 <a href="http://www.uniprot.org/uniprot/?query=gene:MTRR+AND+reviewed:yes">
  MTRR
 </a>
 ) and then the MTR-catalyzed conversion of homocysteine into methionine concomitant with conversion of 5-methyltetrahydrofolate into tetrahydrofolate. Methionine can be further modified into S-adenosyl methionine which serves as a methyl donor in the biosynthesis of cysteine, carnitine, taurine, lecithin, and phospholipids, among others.
</p>
<p>
 Folate deficiency can result in many health problems, the most notable one being neural tube defects in developing embryos, but the molecular mechanism linking folate metabolism to development remains poorly understood. This is what prompted
 <a href="http://www.ncbi.nlm.nih.gov/pubmed/24074862">
  Padmanabhan et al.
 </a>
 to create an animal model to study the impact of abnormal folate metabolism. These authors produced a mouse that contained a gene trap vector inserted in Mtrr gene intron 9. Wild-type Mtrr mRNA was still produced in spite of the insertion, but at lower levels, and folate metabolism was impaired.
</p>
<p>
 When mid-gestation embryos from heterozygous intercrosses were analyzed, it appeared that about half of them displayed developmental defects typical of folate deficiency, ranging from developmental delay to neural tube and heart defects. Surprisingly, wild-type embryos were affected to a similar extent as embryos bearing the mutated gene. Inheritance of the phenotype was not dependent upon the parental genotype, but instead upon that of the maternal grandparents. In other words, Mtrr mutations in either maternal grandparent disrupted the development of their grandchildren, even when the parents and the conceptus were wild-type. These congenital abnormalities persisted in wild-type progeny in generations 4 and 5 of Mtrr mutant maternal ancestors.
</p>
<p>
 What could be the mechanism of this peculiar mode of inheritance? The answer is not yet definite. Because folate plays a key role in one-carbon metabolism, the authors investigated DNA methylation. As expected, global DNA hypomethylation was observed in livers, uteri and placentas. Imprinted loci (differentially methylated regions or DMRs) in wild-type placentas of mid-gestation embryos from heterozygous maternal grandparents were also analyzed. A large proportion of the DMRs assessed in placentas of severely affected embryos had CpG site methylation levels that were statistically different from unrelated wild-type C57BL/6 mice. Surprisingly however, the majority of these sites were hypermethylated and the associated genes down-regulated. There was a positive correlation between epigenetic instability and the severity of the phenotype. Hence, epigenetic instability leading to the misexpression of certain genes may be the cause of developmental phenotypes.
</p>
<p>
 Epigenetic heredity has been reported for
 <a href="http://www.ncbi.nlm.nih.gov/pubmed/16724059">
  Kit
 </a>
 and
 <a href="http://www.ncbi.nlm.nih.gov/pubmed/19820183">
  Sox9
 </a>
 genes. In this case, heredity was mediated by RNA, a mechanism rather unlikely for the Mtrr mutations described above. The RNA-mediated heredity observed for Kit and Sox9
 <a href="http://www.ncbi.nlm.nih.gov/pubmed/23717211">
  required the presence of the tRNA-methyltransferase TRDMT1/DNMT2
 </a>
 . Hence, for both phenomena, it seems that the common feature may be methylation, either at the DNA or RNA level.
</p>
<p>
 While awaiting further exciting discoveries in the field of epigenetics, we have already updated
 <a href="http://www.uniprot.org/uniprot/?query=gene:dnmt2+and+reviewed:yes">
  MTRR entries
 </a>
 with the current knowledge and made them available.
</p>
<h2 id="UniProtKB_news">
 UniProtKB news
</h2>
<h3 id="Change_of_the_cross_references_to_PROSITE_and_HAMAP">
 Change of the cross-references to PROSITE and HAMAP
</h3>
<p>
 The format of the cross-references to the
 <a href="http://prosite.expasy.org/">
  PROSITE
 </a>
 and
 <a href="http://hamap.expasy.org/">
  HAMAP
 </a>
 databases has been simplified in order to align it with the format of other
 <a href="http://www.ebi.ac.uk/interpro/">
  InterPro
 </a>
 member databases.
</p>
<h4 id="Text_format">
 Text format
</h4>
<p>
 <strong>
  Changes for PROSITE:
 </strong>
</p>
<p>
 The optional qualifiers
 <code>
  "UNKNOWN"
 </code>
 ,
 <code>
  "FALSE_NEG"
 </code>
 and
 <code>
  "PARTIAL"
 </code>
 have been removed. Only matches above the threshold were kept, i.e. cross-references with a
 <code>
  "FALSE_NEG"
 </code>
 or
 <code>
  "PARTIAL"
 </code>
 qualifier have been removed.
</p>
<p>
 Examples:
</p>
<p>
 <a href="http://www.uniprot.org/uniprot/A1RHR2.txt">
  A1RHR2
 </a>
 :
</p>
<p>
 Previous format:
</p>
<pre>DR   PROSITE; PS51257; PROKAR_LIPOPROTEIN; UNKNOWN_1.
DR   PROSITE; PS00922; TRANSGLYCOSYLASE; FALSE_NEG.
</pre>
<p>
 New format:
</p>
<pre>DR   PROSITE; PS51257; PROKAR_LIPOPROTEIN; 1.
</pre>
<p>
 <a href="http://www.uniprot.org/uniprot/O02781.txt">
  O02781
 </a>
 :
</p>
<p>
 Previous format:
</p>
<pre>DR   PROSITE; PS00237; G_PROTEIN_RECEP_F1_1; PARTIAL.
DR   PROSITE; PS50262; G_PROTEIN_RECEP_F1_2; 1.
</pre>
<p>
 New format:
</p>
<pre>DR   PROSITE; PS50262; G_PROTEIN_RECEP_F1_2; 1.
</pre>
<p>
 <strong>
  Changes for HAMAP:
 </strong>
</p>
<p>
 The optional field that described the nature of signature hits (
 <code>
  "atypical"
 </code>
 ,
 <code>
  "fused"
 </code>
 or
 <code>
  "atypical/fused"
 </code>
 ) has been removed. Only matches above the threshold were kept, i.e.
 <code>
  "atypical"
 </code>
 and
 <code>
  "atypical/fused"
 </code>
 cross-references have been removed if their match score was below the threshold.
</p>
<p>
 Example:
</p>
<p>
 <a href="http://www.uniprot.org/uniprot/Q9K3D6.txt">
  Q9K3D6
 </a>
 :
</p>
<p>
 Previous format:
</p>
<pre>DR   HAMAP; MF_00006; Arg_succ_lyase; 1; fused.
DR   HAMAP; MF_01105; N-acetyl_glu_synth; 1; atypical/fused.
</pre>
<p>
 New format:
</p>
<pre>DR   HAMAP; MF_00006; Arg_succ_lyase; 1.
</pre>
<h4 id="XML_format">
 XML format
</h4>
<p>
 <strong>
  Changes for PROSITE:
 </strong>
</p>
<p>
 The optional values
 <code>
  "UNKNOWN"
 </code>
 ,
 <code>
  "FALSE_NEG"
 </code>
 and
 <code>
  "PARTIAL"
 </code>
 that were stored in a
 <code>
  property
 </code>
 of type
 <code>
  match status
 </code>
 have been removed, so that the
 <code>
  match status
 </code>
 value has become an integer. Only matches above the threshold were kept, i.e.
 <code>
  "FALSE_NEG"
 </code>
 and
 <code>
  "PARTIAL"
 </code>
 cross-references have been removed.
</p>
<p>
 Examples:
</p>
<p>
 <a href="http://www.uniprot.org/uniprot/A1RHR2.xml">
  A1RHR2
 </a>
 :
</p>
<p>
 Previous format:
</p>
<pre>&lt;dbReference type="PROSITE" id="PS51257"&gt;
  &lt;property type="entry name" value="PROKAR_LIPOPROTEIN"/&gt;
  &lt;property type="match status" value="UNKNOWN_1"/&gt;
&lt;/dbReference&gt;
&lt;dbReference type="PROSITE" id="PS00922"&gt;
  &lt;property type="entry name" value="TRANSGLYCOSYLASE"/&gt;
  &lt;property type="match status" value="FALSE_NEG"/&gt;
&lt;/dbReference&gt;
</pre>
<p>
 New format:
</p>
<pre>&lt;dbReference type="PROSITE" id="PS51257"&gt;
  &lt;property type="entry name" value="PROKAR_LIPOPROTEIN"/&gt;
  &lt;property type="match status" value="1"/&gt;
&lt;/dbReference&gt;
</pre>
<p>
 <a href="http://www.uniprot.org/uniprot/O02781.xml">
  O02781
 </a>
 :
</p>
<p>
 Previous format:
</p>
<pre>&lt;dbReference type="PROSITE" id="PS00237"&gt;
  &lt;property type="entry name" value="G_PROTEIN_RECEP_F1_1"/&gt;
  &lt;property type="match status" value="PARTIAL"/&gt;
&lt;/dbReference&gt;
&lt;dbReference type="PROSITE" id="PS50262"&gt;
  &lt;property type="entry name" value="G_PROTEIN_RECEP_F1_2"/&gt;
  &lt;property type="match status" value="1"/&gt;
&lt;/dbReference&gt;
</pre>
<p>
 New format:
</p>
<pre>&lt;dbReference type="PROSITE" id="PS50262"&gt;
  &lt;property type="entry name" value="G_PROTEIN_RECEP_F1_2"/&gt;
  &lt;property type="match status" value="1"/&gt;
&lt;/dbReference&gt;
</pre>
<p>
 <strong>
  Changes for HAMAP:
 </strong>
</p>
<p>
 The optional
 <code>
  property
 </code>
 of type
 <code>
  flag
 </code>
 that described the nature of signature hits (
 <code>
  "atypical"
 </code>
 ,
 <code>
  "fused"
 </code>
 or
 <code>
  "atypical/fused"
 </code>
 ) has been removed. Only matches above the threshold were kept, i.e.
 <code>
  "atypical"
 </code>
 and
 <code>
  "atypical/fused"
 </code>
 cross-references have been removed if their match score was below the threshold.
</p>
<p>
 Example:
</p>
<p>
 <a href="http://www.uniprot.org/uniprot/Q9K3D6.xml">
  Q9K3D6
 </a>
 :
</p>
<p>
 Previous format:
</p>
<pre>&lt;dbReference type="HAMAP" id="MF_00006"&gt;
  &lt;property type="entry name" value="Arg_succ_lyase"/&gt;
  &lt;property type="flag" value="fused"/&gt;
  &lt;property type="match status" value="1"/&gt;
&lt;/dbReference&gt;
&lt;dbReference type="HAMAP" id="MF_01105"&gt;
  &lt;property type="entry name" value="N-acetyl_glu_synth"/&gt;
  &lt;property type="flag" value="atypical/fused"/&gt;
  &lt;property type="match status" value="1"/&gt;
&lt;/dbReference&gt;
</pre>
<p>
 New format:
</p>
<pre>&lt;dbReference type="HAMAP" id="MF_00006"&gt;
  &lt;property type="entry name" value="Arg_succ_lyase"/&gt;
  &lt;property type="match status" value="1"/&gt;
&lt;/dbReference&gt;
</pre>
<p>
 These changes did not affect the XSD, but may nevertheless require code changes.
</p>
<h3 id="Cross_references_to_TreeFam">
 Cross-references to TreeFam
</h3>
<p>
 Cross-references have been added to
 <a href="http://www.treefam.org">
  TreeFam
 </a>
 , a database composed of phylogenetic trees inferred from animal genomes.
</p>
<p>
 TreeFam is available at
 <a href="http://www.treefam.org">
  http://www.treefam.org
 </a>
 .
</p>
<p>
 The format of the explicit links is:
</p>
<table>
 <tr>
  <th>
   Resource abbreviation
  </th>
  <td>
   TreeFam
  </td>
 </tr>
 <tr>
  <th>
   Resource identifier
  </th>
  <td>
   TreeFam unique identifier.
  </td>
 </tr>
</table>
<p>
 Example:
 <a href="http://www.uniprot.org/uniprot/Q8CFE6#section%5Fx-ref">
  Q8CFE6
 </a>
</p>
<p>
 <a href="http://www.uniprot.org/uniprot/?query=database%3Atreefam&amp;sort=score">
  Show all entries having a cross-reference to TreeFam.
 </a>
</p>
<h4 id="Text_format">
 Text format
</h4>
<p>
 Example:
 <a href="http://www.uniprot.org/uniprot/Q8CFE6.txt">
  Q8CFE6
 </a>
</p>
<pre>DR   TreeFam; TF328787; -.
</pre>
<h4 id="XML_format">
 XML format
</h4>
<p>
 Example:
 <a href="http://www.uniprot.org/uniprot/Q8CFE6.xml">
  Q8CFE6
 </a>
</p>
<pre>&lt;dbReference type="TreeFam" id="TF328787"/&gt;
</pre>
<h3 id="Cross_references_to_BioGrid">
 Cross-references to BioGrid
</h3>
<p>
 Cross-references have been added to
 <a href="http://thebiogrid.org">
  BioGrid
 </a>
 , a public database that archives and disseminates genetic and protein interaction data from model organisms and humans.
</p>
<p>
 BioGrid is available at
 <a href="http://thebiogrid.org">
  http://thebiogrid.org
 </a>
 .
</p>
<p>
 The format of the explicit links is:
</p>
<table>
 <tr>
  <th>
   Resource abbreviation
  </th>
  <td>
   BioGrid
  </td>
 </tr>
 <tr>
  <th>
   Resource identifier
  </th>
  <td>
   BioGrid unique identifier.
  </td>
 </tr>
 <tr>
  <th>
   Optional information 1
  </th>
  <td>
   Number of interactions.
  </td>
 </tr>
</table>
<p>
 Example:
 <a href="http://www.uniprot.org/uniprot/O46201#section%5Fx-ref">
  O46201
 </a>
</p>
<p>
 <a href="http://www.uniprot.org/uniprot/?query=database%3Abiogrid&amp;sort=score">
  Show all entries having a cross-reference to BioGrid.
 </a>
</p>
<h4 id="Text_format">
 Text format
</h4>
<p>
 Example:
 <a href="http://www.uniprot.org/uniprot/O46201.txt">
  O46201
 </a>
</p>
<pre>DR   BioGrid; 69392; 1.
</pre>
<h4 id="XML_format">
 XML format
</h4>
<p>
 Example:
 <a href="http://www.uniprot.org/uniprot/O46201.xml">
  O46201
 </a>
</p>
<pre>&lt;dbReference type="BioGrid" id="69392"&gt;
  &lt;property type="interactions" value="1"/&gt;
&lt;/dbReference&gt;
</pre>
<h3 id="Changes_to_the__a_href___docs_humdisease__controlled_vocabulary_of_human_diseases__a_">
 Changes to the
 <a href="http://www.uniprot.org/docs/humdisease">
  controlled vocabulary of human diseases
 </a>
</h3>
<p>
 New diseases:
</p>
<ul>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-03995">
   Activated PI3K-delta syndrome
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-03955">
   Angioedema induced by ACE inhibitors
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-03977">
   Arthrogryposis, mental retardation, and seizures
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-03985">
   Asparagine synthetase deficiency
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-03969">
   Deafness and myopia
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-03966">
   Deafness, autosomal dominant, 41
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-03957">
   Deafness, autosomal recessive, 76
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-03992">
   Deafness, onychodystrophy, osteodystrophy, mental retardation, and seizures syndrome
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-03972">
   Diamond-Blackfan anemia 12
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-03960">
   Ehlers-Danlos syndrome, musculocontractural type 2
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-03973">
   Episodic pain syndrome, familial, 2
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-03978">
   Episodic pain syndrome, familial, 3
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-03968">
   Erythroderma, congenital, with palmoplantar keratoderma, hypotrichosis, and hyper IgE
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-03975">
   Hyperprolactinemia
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-03976">
   Immunodeficiency, common variable, 9
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-03979">
   Immunodeficiency, common variable, 10
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-03989">
   Klippel-Feil syndrome 2, autosomal recessive
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-03959">
   Leukemia, acute lymphoblastic, 3
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-03993">
   Megalencephaly, autosomal recessive
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-03970">
   Mental retardation, autosomal dominant 22
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-03963">
   Mental retardation, autosomal recessive 39
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-03965">
   Microtia with or without hearing impairment
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-03981">
   Multiple fibroadenomas of the breast
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-03974">
   Native American myopathy
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-03980">
   Nephrotic syndrome 9
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-03987">
   Neuronopathy, distal hereditary motor, 2D
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-03988">
   Neuropathy, hereditary sensory and autonomic, 7
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-03986">
   Otofaciocervical syndrome 2
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-03961">
   Parkinson disease 19, juvenile-onset
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-03964">
   Parkinson disease 20, early-onset
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-03958">
   Periventricular nodular heterotopia 6
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-03984">
   Prader-Willi-like syndrome
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-03962">
   Reticulate acropigmentation of Kitamura
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-03990">
   Retinitis pigmentosa 67
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-03991">
   Rienhoff syndrome
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-03971">
   Spastic paraplegia 43, autosomal recessive
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-03994">
   Spinocerebellar ataxia, autosomal recessive, 7
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-03967">
   Telangiectasia, hereditary hemorrhagic, 5
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-03956">
   Testicular anomalies with or without congenital heart disease
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-03982">
   Van Maldergem syndrome 1
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-03983">
   Van Maldergem syndrome 2
  </a>
 </li>
</ul>
<p>
 Modified diseases:
</p>
<ul>
 <li>
  Adenosine monophosphate deaminase deficiency muscle type -&gt;
  <a href="http://www.uniprot.org/diseases/DI-00039">
   Myopathy due to myoadenylate deaminase deficiency
  </a>
 </li>
 <li>
  Choroid plexus papilloma -&gt;
  <a href="http://www.uniprot.org/diseases/DI-01346">
   Papilloma of choroid plexus
  </a>
 </li>
 <li>
  Dyschromatosis symmetrica hereditaria 1 -&gt;
  <a href="http://www.uniprot.org/diseases/DI-01510">
   Dyschromatosis symmetrica hereditaria
  </a>
 </li>
 <li>
  Ehlers-Danlos syndrome, musculocontractural type -&gt;
  <a href="http://www.uniprot.org/diseases/DI-02810">
   Ehlers-Danlos syndrome, musculocontractural type 1
  </a>
 </li>
 <li>
  Epilepsy with neurodevelopmental defects -&gt;
  <a href="http://www.uniprot.org/diseases/DI-03169">
   Epilepsy, focal, with speech disorder and with or without mental retardation
  </a>
 </li>
 <li>
  Episodic pain syndrome, familial -&gt;
  <a href="http://www.uniprot.org/diseases/DI-03683">
   Episodic pain syndrome, familial, 1
  </a>
 </li>
 <li>
  Familial hypobetalipoproteinemia 1 -&gt;
  <a href="http://www.uniprot.org/diseases/DI-01587">
   Hypobetalipoproteinemia, familial, 1
  </a>
 </li>
 <li>
  Familial hypobetalipoproteinemia 2 -&gt;
  <a href="http://www.uniprot.org/diseases/DI-03014">
   Hypobetalipoproteinemia, familial, 2
  </a>
 </li>
 <li>
  Hereditary hemorrhagic telangiectasia 1 -&gt;
  <a href="http://www.uniprot.org/diseases/DI-01716">
   Telangiectasia, hereditary hemorrhagic, 1
  </a>
 </li>
 <li>
  Hereditary hemorrhagic telangiectasia 2 -&gt;
  <a href="http://www.uniprot.org/diseases/DI-01717">
   Telangiectasia, hereditary hemorrhagic, 2
  </a>
 </li>
 <li>
  Hereditary sensory and autonomic neuropathy 1A -&gt;
  <a href="http://www.uniprot.org/diseases/DI-00547">
   Neuropathy, hereditary sensory and autonomic, 1A
  </a>
 </li>
 <li>
  Hereditary sensory and autonomic neuropathy 1C -&gt;
  <a href="http://www.uniprot.org/diseases/DI-02943">
   Neuropathy, hereditary sensory and autonomic, 1C
  </a>
 </li>
 <li>
  Hereditary sensory and autonomic neuropathy 2A -&gt;
  <a href="http://www.uniprot.org/diseases/DI-00548">
   Neuropathy, hereditary sensory and autonomic, 2A
  </a>
 </li>
 <li>
  Hereditary sensory and autonomic neuropathy 2B -&gt;
  <a href="http://www.uniprot.org/diseases/DI-02529">
   Neuropathy, hereditary sensory and autonomic, 2B
  </a>
 </li>
 <li>
  Hereditary sensory and autonomic neuropathy 3 -&gt;
  <a href="http://www.uniprot.org/diseases/DI-01566">
   Neuropathy, hereditary sensory and autonomic, 3
  </a>
 </li>
 <li>
  Hereditary sensory and autonomic neuropathy 5 -&gt;
  <a href="http://www.uniprot.org/diseases/DI-00549">
   Neuropathy, hereditary sensory and autonomic, 5
  </a>
 </li>
 <li>
  Hereditary sensory and autonomic neuropathy 6 -&gt;
  <a href="http://www.uniprot.org/diseases/DI-03461">
   Neuropathy, hereditary sensory and autonomic, 6
  </a>
 </li>
 <li>
  Hereditary sensory neuropathy 1D -&gt;
  <a href="http://www.uniprot.org/diseases/DI-03056">
   Neuropathy, hereditary sensory, 1D
  </a>
 </li>
 <li>
  Hereditary sensory neuropathy 1E -&gt;
  <a href="http://www.uniprot.org/diseases/DI-03189">
   Neuropathy, hereditary sensory, 1E
  </a>
 </li>
 <li>
  Hereditary sensory neuropathy 2C -&gt;
  <a href="http://www.uniprot.org/diseases/DI-03263">
   Neuropathy, hereditary sensory, 2C
  </a>
 </li>
 <li>
  Hereditary sensory neuropathy, autosomal recessive, with spastic paraplegia -&gt;
  <a href="http://www.uniprot.org/diseases/DI-01256">
   Neuropathy, hereditary sensory, with spastic paraplegia, autosomal recessive
  </a>
 </li>
 <li>
  Keratoderma, palmoplantar, striate 1 -&gt;
  <a href="http://www.uniprot.org/diseases/DI-00895">
   Palmoplantar keratoderma 1, striate, focal, or diffuse
  </a>
 </li>
 <li>
  Klippel-Feil syndrome 1 -&gt;
  <a href="http://www.uniprot.org/diseases/DI-02534">
   Klippel-Feil syndrome 1, autosomal dominant
  </a>
 </li>
 <li>
  Klippel-Feil syndrome 3 -&gt;
  <a href="http://www.uniprot.org/diseases/DI-02973">
   Klippel-Feil syndrome 3, autosomal dominant
  </a>
 </li>
 <li>
  Molybdenum cofactor deficiency type A -&gt;
  <a href="http://www.uniprot.org/diseases/DI-01989">
   Molybdenum cofactor deficiency, complementation group A
  </a>
 </li>
 <li>
  Molybdenum cofactor deficiency type B -&gt;
  <a href="http://www.uniprot.org/diseases/DI-01990">
   Molybdenum cofactor deficiency, complementation group B
  </a>
 </li>
 <li>
  Molybdenum cofactor deficiency type C -&gt;
  <a href="http://www.uniprot.org/diseases/DI-01991">
   Molybdenum cofactor deficiency, complementation group C
  </a>
 </li>
 <li>
  Neuroaxonal neurodegeneration, infantile, with facial dysmophism -&gt;
  <a href="http://www.uniprot.org/diseases/DI-03902">
   Hypotonia, infantile, with psychomotor retardation and characteristic facies
  </a>
 </li>
 <li>
  Otofaciocervical syndrome -&gt;
  <a href="http://www.uniprot.org/diseases/DI-02112">
   Otofaciocervical syndrome 1
  </a>
 </li>
 <li>
  Pancreatitis -&gt;
  <a href="http://www.uniprot.org/diseases/DI-01731">
   Pancreatitis, hereditary
  </a>
 </li>
 <li>
  Split-hand/foot malformation 1, with sensorineural hearing loss -&gt;
  <a href="http://www.uniprot.org/diseases/DI-03391">
   Split-hand/foot malformation 1 with sensorineural hearing loss, autosomal recessive
  </a>
 </li>
</ul>
<h3 id="Changes_to_the__a_href___docs_ptmlist__controlled_vocabulary_for_PTMs__a_">
 Changes to the
 <a href="http://www.uniprot.org/docs/ptmlist">
  controlled vocabulary for PTMs
 </a>
</h3>
<p>
 New terms for the feature key 'Modified residue' ('MOD_RES' in the flat file):
</p>
<ul>
 <li>
  N-methylglycine
 </li>
 <li>
  N,N-dimethylglycine
 </li>
 <li>
  N,N,N-trimethylglycine
 </li>
</ul>
<p>
 Deleted term:
</p>
<ul>
 <li>
  3-hydroxyhistidine
 </li>
</ul>
<h2 id="UniRef_news">
 UniRef news
</h2>
<h3 id="Revision_of_the_UniParc_records_used_in_the_UniRef_databases">
 Revision of the UniParc records used in the UniRef databases
</h3>
<p>
 We have stopped importing UniParc records that correspond to Ensembl proteomes sequences in the UniRef databases, as the relevant sequences are now part of UniProtKB. Previously, some sequences from Ensembl proteomes (e.g. from Human, Chicken, Cow) were missing from UniProtKB, but we have recently completed their import into UniProtKB (see
 <a href="http://www.uniprot.org/proteomes">
  FAQ
 </a>
 ) and thus no longer need to import them via UniParc. The UniRef databases will continue to include UniParc records from the
 <a href="https://www.ncbi.nlm.nih.gov/refseq/">
  RefSeq
 </a>
 and
 <a href="http://www.wwpdb.org/">
  PDB
 </a>
 databases that are not in UniProtKB to ensure a complete sequence space coverage.
</p>