<h2 id="Headline">
 Headline
</h2>
<h3 id="Pseudo_allergy__real_progress">
 Pseudo-allergy, real progress
</h3>
<p>
 Do you sniffle and sneeze as trees start to bloom and the pollen gets airborne? Your mast cells are to blame. These cells reside at strategic anatomical positions, such as skin, gastrointestinal tract and lung, and provide us with a first line of defence against potential harm from our environment. Besides their beneficial functions, mast cells can also react to compounds that do not represent any threat to our health, such as pollen. This process begins with the interaction of an antigen with immunoglobulin E (IgE) bound to high affinity Fc epsilon receptors at the mast cell surface. It ends with the release of histamine and various inflammatory and immunomodulatory substances, which causes allergy. Most adverse reactions to peptidergic and small molecule therapeutic agents, collectively called basic secretagogues, also rely on mast cell stimulation, but do not correlate with IgE antibody titer. They proceed through a different, not yet fully understood, IgE-independent mechanism called pseudo-allergy, that eventually also leads the release of granule-stored histamine. In human, MRGPRX2 has been proposed, among others, to serve as a receptor for basic secretagogues, but until recently there was no direct proof of its involvement.
</p>
<p>
 Earlier this year,
 <a href="http://www.ncbi.nlm.nih.gov/pubmed/25517090">
  McNeil et al.
 </a>
 showed that "basic secretagogues activate mouse mast cells
 <em>
  in vitro
 </em>
 and
 <em>
  in vivo
 </em>
 through a single receptor,
 <a href="http://www.uniprot.org/uniprot/?query=Q3KNA1">
  Mrgprb2
 </a>
 , the ortholog of the human G-protein-coupled receptor
 <a href="http://www.uniprot.org/uniprot/?query=Q96LB1">
  MRGPRX2
 </a>
 ". The first achievement of this study was to prove the orthology of these 2 genes, which was not an easy task. In humans, MRGPRX2 is found in a cluster with 3 other MRGPRX family members. This cluster is dramatically expanded in mouse, with 22 potential protein-coding genes that show comparable sequence identity to MRGPRX2. To establish orthology, the authors used 2 criteria: expression pattern (expression in mast cells) and pharmacology (some 16 compounds were tested for mast cell activation). Then Mrgprb2a knockout mice were created. Gene targeting was performed using a zinc-finger-nuclease-based strategy, as classical homologous recombination approach was impossible in this genomic locus due to too many repetitive sequences. The null animals showed no visible phenotype in normal conditions, but didn't produce any pseudo-allergic reaction in response to small-molecule therapeutic drugs. Secretagogue-induced histamine release, inflammation and airway contraction were abolished.
</p>
<p>
 This elegant study does not deal simply with the identification of "just another receptor". It addresses an issue that may concern all of us at some point in our lives. Basic secretagogues are compounds that are frequently encountered either in natural fluids, such as the
 <a href="https://en.wikipedia.org/wiki/Mastoparan">
  wasp venom toxin mastoparan
 </a>
 , or in various drugs, such as cationic peptidergic drugs, antibiotics (fluoroquinolone family), neuromuscular blocking agents, etc. These latter are routinely used in surgery to reduce unwanted muscle movement and are responsible for nearly
 <a href="http://www.ncbi.nlm.nih.gov/pubmed/21497888">
  60% of allergic reactions in a surgical setting
 </a>
 . The majority of these compounds activate mast cells in an Mrgprb2-dependent manner. The animal model created by McNeil et al. could then be used for pre-clinical testing of new drugs in order to minimize pseudo-allergic risks. In addition, the identification a motif common to several Mrgprb2 agonists may allow the prediction of side effects of clinically used compounds.
</p>
<p>
 As of this release,
 <a href="http://www.uniprot.org/uniprot/?query=gene:mrgprx2+NOT+organism:10090+AND+reviewed:yes">
  primate MRGPRX2
 </a>
 and mouse
 <a href="http://www.uniprot.org/uniprot/?query=Q3KNA1">
  Mrgprb2
 </a>
 entries have been updated and are publicly available.
</p>
<h2 id="UniProt_service_news">
 UniProt service news
</h2>
<h3 id="Programmatic_access_to_UniProt_with_sparql_uniprot_org">
 Programmatic access to UniProt with sparql.uniprot.org
</h3>
<p>
 We are happy to announce the public release of the UniProt SPARQL endpoint at
 <a href="http://sparql.uniprot.org">
  sparql.uniprot.org
 </a>
 , where you can also find links to the documentation of the UniProt RDF data model and an interactive query interface with sample queries to get you started.
</p>
<p>
 For those unfamiliar with
 <a href="https://en.wikipedia.org/wiki/SPARQL">
  SPARQL
 </a>
 , this is a
 <a href="https://en.wikipedia.org/wiki/World%5FWide%5FWeb%5FConsortium">
  W3C
 </a>
 standardized query language for the
 <a href="https://en.wikipedia.org/wiki/Semantic%5FWeb">
  Semantic Web
 </a>
 . If you know SQL, it will look familiar to you and you can do similar types of queries with it. SPARQL also allows you to query and combine data from a variety of SPARQL endpoints, providing a valuable low-cost alternative to building your own data warehouse. You can combine UniProt data from sparql.uniprot.org with that from the SPARQL endpoints hosted by the
 <a href="https://www.ebi.ac.uk/rdf/">
  EBI's RDF platform
 </a>
 , the
 <a href="http://snorql.nextprot.org">
  SIB's neXtProt
 </a>
 SPARQL endpoint, etc.
</p>
<p>
 We look forward to feedback from the community to help us improve this service further.
</p>
<h2 id="UniProtKB_news">
 UniProtKB news
</h2>
<h3 id="Addition_of_human_somatic_protein_altering_variants_from_COSMIC">
 Addition of human somatic protein altering variants from COSMIC
</h3>
<p>
 The Catalogue of Somatic Mutations in Cancer (
 <a href="http://cancer.sanger.ac.uk/cosmic">
  COSMIC
 </a>
 ) is a database of manually curated somatic variants from peer reviewed publications and genome-wide studies. UniProt, in collaboration with COSMIC, have integrated COSMIC release v71 protein altering variants into the homo_sapiens_variation.txt.gz file. The COSMIC variants provide the standard information found in the homo_sapiens_variation.txt.gz file and additional information on the primary tissue(s) the variant was found in within the Phenotype/Disease field.
</p>
<h3 id="Changes_to_the__a_href___docs_humdisease__humdisease_txt__a__file">
 Changes to the
 <a href="http://www.uniprot.org/docs/humdisease">
  humdisease.txt
 </a>
 file
</h3>
<p>
 We have added cross-references to
 <a href="http://www.ncbi.nlm.nih.gov/medgen/">
  MedGen
 </a>
 to the
 <a href="http://www.uniprot.org/docs/humdisease">
  humdisease.txt
 </a>
 file. MedGen, the NCBI portal to information about human genetic disorders, conveys multiple disease names, medical terms and information for the same disorder from various sources into a specific concept. Each MedGen concept has a Concept Unique Identifier (CUI) that allows computational access to global disease information. Together with disease nomenclature, this includes disease definitions, clinical findings, available clinical and research tests, molecular resources, professional guidelines, original and review literature, consumer resources, clinical trials, and Web links to other related resources. MedGen is a valuable resource to allow UniProtKB users to access an extensive range of biomedical data.
</p>
<h3 id="Changes_to_the__a_href___docs_humdisease__controlled_vocabulary_of_human_diseases__a_">
 Changes to the
 <a href="http://www.uniprot.org/docs/humdisease">
  controlled vocabulary of human diseases
 </a>
</h3>
<p>
 New diseases:
</p>
<ul>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-04405">
   Anhidrosis, isolated, with normal sweat glands
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-04407">
   Cardiomyopathy, familial hypertrophic 24
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-04416">
   Deafness, autosomal dominant, 67
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-04417">
   Deafness, autosomal dominant, 40
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-04424">
   Dyskeratosis congenita, autosomal recessive, 6
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-04408">
   Dystonia 26, myoclonic
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-04412">
   Epileptic encephalopathy, early infantile, 29
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-04413">
   Epileptic encephalopathy, early infantile, 30
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-04414">
   Epileptic encephalopathy, early infantile, 31
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-04415">
   Epileptic encephalopathy, early infantile, 32
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-04423">
   Immunodeficiency 39
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-04406">
   Kaufman oculocerebrofacial syndrome
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-04409">
   Linear skin defects with multiple congenital anomalies 3
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-04422">
   Lissencephaly 7, with cerebellar hypoplasia
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-04404">
   Maturity-onset diabetes of the young 13
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-04418">
   Mental retardation, autosomal dominant 34
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-04419">
   Mental retardation, autosomal dominant 35
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-04420">
   Mental retardation, autosomal dominant 36
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-04421">
   Mental retardation, autosomal dominant 37
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-04411">
   Microcephaly and chorioretinopathy, autosomal recessive, 3
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-04393">
   Myasthenic syndrome, congenital, 2A, slow-channel
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-04398">
   Myasthenic syndrome, congenital, 2C, associated with acetylcholine receptor deficiency
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-04394">
   Myasthenic syndrome, congenital, 3A, slow-channel
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-04395">
   Myasthenic syndrome, congenital, 3B, fast-channel
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-04399">
   Myasthenic syndrome, congenital, 3C, associated with acetylcholine receptor deficiency
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-04397">
   Myasthenic syndrome, congenital, 4A, slow-channel
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-04396">
   Myasthenic syndrome, congenital, 4B, fast-channel
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-04400">
   Myasthenic syndrome, congenital, 9, associated with acetylcholine receptor deficiency
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-04401">
   Myasthenic syndrome, congenital, 11, associated with acetylcholine receptor deficiency
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-04402">
   Myasthenic syndrome, congenital, 17
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-04403">
   Myasthenic syndrome, congenital, 18
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-04410">
   Myopathy, lactic acidosis, and sideroblastic anemia 3
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-04425">
   Parkinson disease 21
  </a>
 </li>
</ul>
<p>
 Modified diseases:
</p>
<ul>
 <li>
  Cardiomyopathy, dilated 1C -&gt;
  <a href="http://www.uniprot.org/diseases/DI-00212">
   Cardiomyopathy, dilated 1C, with or without left ventricular non-compaction
  </a>
 </li>
 <li>
  Ehlers-Danlos syndrome 1 -&gt;
  <a href="http://www.uniprot.org/diseases/DI-00436">
   Ehlers-Danlos syndrome, classic type
  </a>
 </li>
 <li>
  Fertile eunuch syndrome -&gt;
  <a href="http://www.uniprot.org/diseases/DI-01614">
   Hypogonadotropic hypogonadism 23 without anosmia
  </a>
 </li>
 <li>
  Isolated follicle-stimulating hormone deficiency -&gt;
  <a href="http://www.uniprot.org/diseases/DI-01840">
   Hypogonadotropic hypogonadism 24 without anosmia
  </a>
 </li>
 <li>
  Microphthalmia, isolated, with cataract, 2 -&gt;
  <a href="http://www.uniprot.org/diseases/DI-00757">
   Optic disk anomalies with retinal and/or macular dystrophy
  </a>
 </li>
 <li>
  Myasthenia, limb-girdle, familial -&gt;
  <a href="http://www.uniprot.org/diseases/DI-00494">
   Myasthenic syndrome, congenital, 10
  </a>
 </li>
 <li>
  Myasthenic syndrome, congenital, acetazolamide-responsive -&gt;
  <a href="http://www.uniprot.org/diseases/DI-00365">
   Myasthenic syndrome, congenital, 16
  </a>
 </li>
 <li>
  Myasthenic syndrome, congenital, associated with acetylcholine receptor deficiency -&gt;
  <a href="http://www.uniprot.org/diseases/DI-00369">
   Myasthenic syndrome, congenital, 4C, associated with acetylcholine receptor deficiency
  </a>
 </li>
 <li>
  Myasthenic syndrome, congenital, associated with episodic apnea -&gt;
  <a href="http://www.uniprot.org/diseases/DI-00370">
   Myasthenic syndrome, congenital, 6, presynaptic
  </a>
 </li>
 <li>
  Myasthenic syndrome, congenital, Engel type -&gt;
  <a href="http://www.uniprot.org/diseases/DI-00366">
   Myasthenic syndrome, congenital, 5
  </a>
 </li>
 <li>
  Myasthenic syndrome, congenital, fast-channel -&gt;
  <a href="http://www.uniprot.org/diseases/DI-00367">
   Myasthenic syndrome, congenital, 1B, fast-channel
  </a>
 </li>
 <li>
  Myasthenic syndrome, congenital, slow-channel -&gt;
  <a href="http://www.uniprot.org/diseases/DI-00368">
   Myasthenic syndrome, congenital, 1A, slow-channel
  </a>
 </li>
 <li>
  Myasthenic syndrome, congenital, with pre- and postsynaptic defects -&gt;
  <a href="http://www.uniprot.org/diseases/DI-04109">
   Myasthenic syndrome, congenital, 8
  </a>
 </li>
 <li>
  Myasthenic syndrome, congenital, with tubular aggregates, 1 -&gt;
  <a href="http://www.uniprot.org/diseases/DI-03084">
   Myasthenic syndrome, congenital, 12
  </a>
 </li>
 <li>
  Myasthenic syndrome, congenital, with tubular aggregates, 2 -&gt;
  <a href="http://www.uniprot.org/diseases/DI-03511">
   Myasthenic syndrome, congenital, 13
  </a>
 </li>
 <li>
  Myasthenic syndrome, congenital, with tubular aggregates, 3 -&gt;
  <a href="http://www.uniprot.org/diseases/DI-04340">
   Myasthenic syndrome, congenital, 14
  </a>
 </li>
 <li>
  Myasthenic syndrome, congenital, without tubular aggregates -&gt;
  <a href="http://www.uniprot.org/diseases/DI-04339">
   Myasthenic syndrome, congenital, 15
  </a>
 </li>
 <li>
  Myasthenic syndrome, presynaptic, congenital, with or without motor neuropathy -&gt;
  <a href="http://www.uniprot.org/diseases/DI-04255">
   Myasthenic syndrome, congenital, 7, presynaptic
  </a>
 </li>
 <li>
  Robinow syndrome autosomal dominant -&gt;
  <a href="http://www.uniprot.org/diseases/DI-03227">
   Robinow syndrome, autosomal dominant 1
  </a>
 </li>
 <li>
  Tenascin-X deficiency -&gt;
  <a href="http://www.uniprot.org/diseases/DI-01097">
   Ehlers-Danlos syndrome due to tenascin X deficiency
  </a>
 </li>
</ul>
<p>
 Deleted diseases:
</p>
<ul>
 <li>
  Blepharophimosis-ptosis-intellectual disability syndrome
 </li>
 <li>
  Ehlers-Danlos syndrome 2
 </li>
</ul>