<h2 id="Headline">
 Headline
</h2>
<h3 id="The_smell_of_the_sea_in_UniProtKB">
 The smell of the sea in UniProtKB
</h3>
<p>
 Memories left by a walk on the seashore bring into play all our senses, of which smell is not the least. This characteristic 'smell of the sea' is carried by a little molecule,
 <a href="https://www.ebi.ac.uk/chebi/searchId.do?chebiId=17437">
  dimethylsulfide
 </a>
 (DMS), which is an enzymatic cleavage product of
 <a href="https://www.ebi.ac.uk/chebi/searchId.do;jsessionid=E2E9DFC0243113BD50DDC22170E8DAA4?chebiId=CHEBI:16457">
  dimethylsulfoniopropionate
 </a>
 (DMSP).
</p>
<p>
 DMSP is one of the most abundant organic molecules in the world, with
 <a href="http://www.ncbi.nlm.nih.gov/pubmed/26113706">
  a billion tons made and turned over every year
 </a>
 . It is produced by marine macroalgae, as well as by single-cell
 <a href="https://en.wikipedia.org/wiki/Phytoplankton">
  phytoplankton
 </a>
 species, such as diatoms, dinoflagellates and haptophytes, and occurs at high concentrations in their cytoplasm. The physiological function of DMSP is not yet fully established. It is thought to function as an osmolyte. It has also been proposed to serve as a cryoprotectant in polar algae. DMSP enzymatic cleavage products, DMS and acrylate, are quite effective at scavenging free radicals and other reactive oxygen species. Hence they may serve as an
 <a href="http://www.ncbi.nlm.nih.gov/pubmed/12124622">
  antioxidant system
 </a>
 .
</p>
<p>
 In healthy growing phytoplankton, DMSP freely diffuses in the cytoplasm, and only minute quantities are released. This amount is sufficient to attract
 <a href="https://en.wikipedia.org/wiki/Zooplankton">
  zooplankton
 </a>
 which start feeding on algae. Organisms grazed upon or infected by viruses as well as stressed or senescent cells release greater amount of DMSP, which is taken up by
 <a href="https://en.wikipedia.org/wiki/Bacterioplankton">
  bacterioplankton
 </a>
 , metabolized into DMS and used as a
 <a href="http://www.sciencedirect.com/science/article/pii/S138511010000023X">
  source of carbon and sulfur
 </a>
 . DMS is not only used by seawater microorganisms, it is also volatile and a small fraction of it is released into the atmosphere where it creates an olfactory landscape providing
 <a href="http://www.ncbi.nlm.nih.gov/pubmed/16709918,17148193">
  seabirds
 </a>
 with orientation cues to potential food supplies. In the atmosphere, DMS is oxidized to sulfuric acid and becomes an important source of sulfate aerosols. These act as condensation nuclei, causing water molecules to coalesce and cloud to form. The cycle is closed when rain brings back the sulfur-containing particles into the ocean. Interestingly, phytoplankton appear to convert DMSP into DMS very rapidly when they are stressed by
 <a href="http://onlinelibrary.wiley.com/doi/10.1029/2004GL019581/abstract">
  UV radiation
 </a>
 . The local increase in volatile DMS increases cloud formation, hence decreasing direct sun light exposure and relieving stress. Through this mechanism, plankton may shape local weather for their own benefit.
</p>
<p>
 DMS release by seaweed was described
 <a href="http://www.ncbi.nlm.nih.gov/pubmed/16745792">
  in 1935
 </a>
 and
 <a href="http://www.ncbi.nlm.nih.gov/pubmed/18101461">
  DMSP was identified as its precursor
 </a>
 almost 70 years ago, but the enzyme catalyzing the reaction remained elusive until last June. Using classical biochemical approaches, as well as genomic and proteomic analyses,
 <a href="http://www.ncbi.nlm.nih.gov/pubmed/26113722">
  Alcombri et al.
 </a>
 identified
 <a href="http://www.uniprot.org/uniprot/P0DN21">
  ALMA1
 </a>
 from the chloroplastic membrane fraction of the coccolithophore alga
 <a href="http://www.uniprot.org/taxonomy/2903">
  <em>
   Emiliania huxleyi
  </em>
 </a>
 , an abundant bloom-forming marine phytoplankton. This enzyme is a redox-sensitive homotetramer, that belongs to the
 <a href="http://www.uniprot.org/uniprot/?query=family:aspartate%2Fglutamate+racemases+family+AND+reviewed:yes">
  aspartate/glutamate racemase superfamily
 </a>
 and catalyzes DMSP cleavage into DMS and acrylate. Phylogenetic studies show the presence of numerous ALMA1 homologs in major, globally distributed phytoplankton taxa and in other marine organisms. This major discovery paves the way for future investigations on the physiological role of DMS and may allow quantification of the relative biogeochemical contribution of algae and bacteria to global DMS production.
</p>
<p>
 If you want to take a deep, though virtual breath of sea smell, you can visit
 <a href="http://www.uniprot.org/uniprot/?query=gene:alma1+and+reviewed:yes">
  ALMA1 entries
 </a>
 that are available to you as of this release.
</p>
<h2 id="UniProtKB_news">
 UniProtKB news
</h2>
<h3 id="Cross_references_to_WBParaSite">
 Cross-references to WBParaSite
</h3>
<p>
 Cross-references have been added to WBParaSite, an open access resource providing access to the genome sequences, genome browsers, semi-automatic annotation and comparative genomics analysis of parasitic worms (helminths). WormBase ParaSite is closely integrated with and complementary to the main WormBase resource, the central focus of which is the model nematode Caenorhabditis elegans and its close relatives.
</p>
<p>
 WBParaSite is available at
 <a href="http://parasite.wormbase.org">
  http://parasite.wormbase.org
 </a>
 .
</p>
<p>
 The format of the explicit links is:
</p>
<table>
 <tr>
  <th>
   Resource abbreviation
  </th>
  <td>
   WBParaSite
  </td>
 </tr>
 <tr>
  <th>
   Resource identifier
  </th>
  <td>
   Transcript identifier
  </td>
 </tr>
 <tr>
  <th>
   Optional information 1
  </th>
  <td>
   Protein identifier
  </td>
 </tr>
 <tr>
  <th>
   Optional information 2
  </th>
  <td>
   Gene identifier
  </td>
 </tr>
</table>
<p>
 Cross-references to WBParaSite may be isoform-specific. The general format of isoform-specific cross-references was described in release
 <a href="http://www.uniprot.org/news/2014/03/19/release">
  2014_03
 </a>
 .
</p>
<p>
 Example:
 <a href="http://www.uniprot.org/uniprot/A8PGQ3#sequences">
  A8PGQ3
 </a>
</p>
<p>
 <a href="http://www.uniprot.org/uniprot/?query=database:wbparasite&amp;sort=score">
  Show all entries having a cross-reference to WBParaSite.
 </a>
</p>
<h4 id="Text_format">
 Text format
</h4>
<p>
 Example:
 <a href="http://www.uniprot.org/uniprot/A8PGQ3.txt">
  A8PGQ3
 </a>
</p>
<pre>DR   WBParaSite; Bm6838; Bm6838; WBGene00227099.
</pre>
<h4 id="XML_format">
 XML format
</h4>
<p>
 Example:
 <a href="http://www.uniprot.org/uniprot/A8PGQ3.xml">
  A8PGQ3
 </a>
</p>
<pre>&lt;dbReference type="WBParaSite" id="Bm6838"&gt;
  &lt;property type="protein sequence ID" value="Bm6838"/&gt;
  &lt;property type="gene ID" value="WBGene00227099"/&gt;
&lt;/dbReference&gt;
</pre>
<h4 id="RDF_format">
 RDF format
</h4>
<p>
 Example:
 <a href="http://www.uniprot.org/uniprot/A8PGQ3.ttl">
  A8PGQ3
 </a>
</p>
<pre>uniprot:A8PGQ3
  rdfs:seeAlso &lt;http://purl.uniprot.org/wbparasite/Bm6838&gt; .
&lt;http://purl.uniprot.org/wbparasite/Bm6838&gt;
  rdf:type up:Transcript_Resource ;
  up:database &lt;http://purl.uniprot.org/database/WBParaSite&gt; ;
  up:translatedTo &lt;http://purl.uniprot.org/wbparasite/Bm6838&gt; ;
  up:transcribedFrom &lt;http://purl.uniprot.org/wbparasite/WBGene00227099&gt; .
</pre>
<h3 id="Removal_of_the_cross_references_to_CYGD">
 Removal of the cross-references to CYGD
</h3>
<p>
 Cross-references to CYGD have been removed.
</p>
<h3 id="Changes_to_the__a_href___docs_humdisease__controlled_vocabulary_of_human_diseases__a_">
 Changes to the
 <a href="http://www.uniprot.org/docs/humdisease">
  controlled vocabulary of human diseases
 </a>
</h3>
<p>
 New diseases:
</p>
<ul>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-04458">
   46,XY sex reversal 10
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-04490">
   Achondroplasia, severe, with developmental delay and acanthosis nigricans
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-04483">
   Acrofacial dysostosis, Cincinnati type
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-04480">
   Al-Raqad syndrome
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-04454">
   Autoimmune interstitial lung, joint, and kidney disease
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-04468">
   Ayme-Gripp syndrome
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-04476">
   B-cell expansion with NFKB and T-cell anergy
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-04453">
   Basal ganglia calcification, idiopathic, 6
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-04474">
   Basel-Vanagaite-Smirin-Yosef syndrome
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-04487">
   Bethlem myopathy 2
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-04473">
   Candidiasis, familial, 9
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-04496">
   Charcot-Marie-Tooth disease 2V
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-04489">
   Ciliary dyskinesia, primary, 32
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-04460">
   Combined oxidative phosphorylation deficiency 25
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-04479">
   Congenital disorder of glycosylation 1Z
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-04449">
   Dystonia 27
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-04492">
   Epidermolysis bullosa simplex with nail dystrophy
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-04469">
   Epilepsy, familial adult myoclonic, 2
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-04463">
   Epilepsy, familial temporal lobe, 7
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-04482">
   Epilepsy, familial temporal lobe, 8
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-04447">
   Epileptic encephalopathy, early infantile, 33
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-04455">
   Familial adenomatous polyposis 3
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-04462">
   Fanconi anemia complementation group T
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-04471">
   Frontotemporal dementia and/or amyotrophic lateral sclerosis 3
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-04472">
   Frontotemporal dementia and/or amyotrophic lateral sclerosis 4
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-04494">
   Growth restriction, severe, with distinctive facies
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-04464">
   Hypertension and brachydactyly syndrome
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-04456">
   Hypomagnesemia, seizures, and mental retardation
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-04465">
   Hypotrichosis-lymphedema-telangiectasia-renal defect syndrome
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-04461">
   Immunodeficiency 40
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-04495">
   Joubert syndrome 23
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-04450">
   Leukodystrophy, hypomyelinating, 10
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-04497">
   Leukodystrophy, hypomyelinating, 11
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-04481">
   Mental retardation, autosomal recessive 50
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-04491">
   Microcephaly 15, primary, autosomal recessive
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-04459">
   Microphthalmia, isolated, with coloboma, 10
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-04457">
   Myoclonic-atonic epilepsy
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-04466">
   Myopathy, myosin storage, autosomal recessive
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-04493">
   Neuropathy, hereditary sensory and autonomic, 8
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-04451">
   Nevus spilus
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-04470">
   Pontocerebellar hypoplasia 3
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-04488">
   Progressive external ophthalmoplegia with mitochondrial DNA deletions, autosomal recessive 2
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-04485">
   Retinitis pigmentosa 72
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-04475">
   Spastic paraplegia 74, autosomal recessive
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-04467">
   Spermatogenic failure, X-linked, 2
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-04448">
   Spinocerebellar ataxia 41
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-04452">
   Spitz nevus
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-04486">
   Ullrich congenital muscular dystrophy 2
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-04484">
   Vitreoretinopathy, exudative 6
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-04477">
   Zimmermann-Laband syndrome 1
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-04478">
   Zimmermann-Laband syndrome 2
  </a>
 </li>
</ul>
<p>
 Modified diseases:
</p>
<ul>
 <li>
  Bethlem myopathy -&gt;
  <a href="http://www.uniprot.org/diseases/DI-00188">
   Bethlem myopathy 1
  </a>
 </li>
 <li>
  Eagle-Barrett syndrome -&gt;
  <a href="http://www.uniprot.org/diseases/DI-03312">
   Prune belly syndrome
  </a>
 </li>
 <li>
  Ullrich congenital muscular dystrophy -&gt;
  <a href="http://www.uniprot.org/diseases/DI-01110">
   Ullrich congenital muscular dystrophy 1
  </a>
 </li>
</ul>
<h2 id="UniParc_news">
 UniParc news
</h2>
<h3 id="UniParc_cross_reference_types_changes">
 UniParc cross-reference types changes
</h3>
<p>
 UniParc and UniProtKB entries both contain cross-references to external databases. For consistency reasons we have adjusted the names of these databases in UniParc to the ones in UniProtKB. In particular we have changed the following types of cross-references in UniParc:
</p>
<table>
 <tr>
  <th>
   Old type
  </th>
  <th>
   New type
  </th>
 </tr>
 <tr>
  <td>
   ENSEMBL
  </td>
  <td>
   Ensembl
  </td>
 </tr>
 <tr>
  <td>
   FLYBASE
  </td>
  <td>
   FlyBase
  </td>
 </tr>
 <tr>
  <td>
   H_INV
  </td>
  <td>
   H-InvDB
  </td>
 </tr>
 <tr>
  <td>
   REFSEQ
  </td>
  <td>
   RefSeq
  </td>
 </tr>
 <tr>
  <td>
   TAIR_ARABIDOPSIS
  </td>
  <td>
   TAIR
  </td>
 </tr>
 <tr>
  <td>
   WORMBASE
  </td>
  <td>
   WormBase
  </td>
 </tr>
 <tr>
  <td>
   WormBase ParaSite
  </td>
  <td>
   WBParaSite
  </td>
 </tr>
</table>
<p>
 Example:
</p>
<p>
 Previous XML:
</p>
<pre>&lt;dbReference type="WormBase ParaSite" id="A_03330" version_i="1" active="Y" created="2014-09-12" last="2015-07-09"&gt;
  &lt;property type="NCBI_taxonomy_id" value="6185"/&gt;
&lt;/dbReference&gt;
</pre>
<p>
 New XML:
</p>
<pre>&lt;dbReference type="WBParaSite" id="A_03330" version_i="1" active="Y" created="2014-09-12" last="2015-07-09"&gt;
  &lt;property type="NCBI_taxonomy_id" value="6185"/&gt;
&lt;/dbReference&gt;
</pre>