<h2 id="Headline">
 Headline
</h2>
<h3 id="Butterfly_fashion__all_they_need_is_cortex">
 Butterfly fashion: all they need is cortex
</h3>
<p>
 Butterfly and moth wing patterns fulfill various functions, such as mate attraction, thermal regulation, and protection by concealment, mimicry or warning. Patterns are produced by
 <a href="http://www.learnaboutbutterflies.com/Anatomy%203.htm">
  a dust-like layer of tiny colored scales
 </a>
 that cover an otherwise transparent membrane. Scales can be pigmented with melanins resulting in black and brown colors. Blue, red and iridescence are usually created by the microstructure of the scales, resulting in the scattering of light. Each scale is produced by a single cell on the wing surface.
</p>
<p>
 Wing pattern and color can change in order to adapt to environmental changes. The classical example of such a phenomenon is provided by
 <a href="http://www.uniprot.org/taxonomy/82595">
  <em>
   Biston betularia
  </em>
 </a>
 . This moth used to camouflage itself against lichen-covered tree trunks. Its peppered white wings makes it almost invisible on this background. With the advent of the industrial revolution in the 19th century in Britain, trunks turned soot black and so did
 <em>
  Biston betularia
 </em>
 . The new melanic morph was described for the first time in Manchester in 1848 and called
 <em>
  carbonaria
 </em>
 . It spread all over England and its frequency was over 90% in the 1950s. Several years after the Clean Air Act, in the early 1970s, its frequency started to drop again and nowadays the
 <a href="https://www.ncbi.nlm.nih.gov/pubmed/14737825">
  maximum is evaluated less than 50% and in most places below 10%
 </a>
 .
</p>
<p>
 The mutation that gave rise to
 <em>
  Biston betularia
 </em>
 industrial melanism
 <a href="https://www.ncbi.nlm.nih.gov/pubmed/27251284">
  has just been identified
 </a>
 . It is the insertion of a large, tandemly repeated, transposable element into the first intron of the
 <em>
  cort
 </em>
 gene, which results in increased gene expression. The transposition event is thought to have occurred around 1819, which is consistent with the historical record. Surprisingly, the
 <em>
  cort
 </em>
 gene does not encode a transcription factor that would be involved in the expression of pigmentation genes. Its only known function has been reported in
 <em>
  Drosophila
 </em>
 , where the
 <em>
  cort
 </em>
 -encoded protein
 <a href="http://www.uniprot.org/uniprot/Q960N3">
  cortex
 </a>
 is a cell-cycle regulator,
 <a href="https://www.ncbi.nlm.nih.gov/pubmed/11252055,10683177,10924478,17251266,18020708">
  required for the completion of meiosis in oocytes
 </a>
 . In
 <a href="http://www.uniprot.org/taxonomy/884341">
  <em>
   Heliconius numata tarapotensis
  </em>
 </a>
 and
 <a href="http://www.uniprot.org/taxonomy/171916">
  Heliconius melpomene rosina
 </a>
 , 2 butterfly species, cortex is expressed in final instar larval hindwing discs, in
 <a href="https://www.ncbi.nlm.nih.gov/pubmed/27251285">
  regions fated to become black in the adult wing
 </a>
 . Although cortex function in the regulation of pigmentation patterning is yet unknown, the current hypothesis is that it may regulate scale cell development.
</p>
<p>
 In other latitudes, butterflies escape from predators not by concealment, but by warning that they are unpalatable with bright and distinctive wing colors. Within a given area, experienced birds have been "educated" to avoid certain patterns. This pattern recognition varies upon geographical locations. As a result, in a given area, a number of butterfly species, edible or not, mimic each other and have the same color pattern, even though they may be only distantly related, while Lepidopteria of the same species found in other locations may exhibit very different patterns. A
 <a href="https://www.ncbi.nlm.nih.gov/pubmed/27251285">
  recent study
 </a>
 focused on different
 <a href="http://www.uniprot.org/taxonomy/33416">
  <em>
   Heliconius
  </em>
 </a>
 species living in South America. The result was quite striking. In these species too, the
 <em>
  cort
 </em>
 gene appeared to be a major regulator of color and pattern. This result suggests that the recruitment of cortex to wing patterning may have occurred before the major diversification of the Lepidoptera. This gene has repeatedly been targeted by natural selection to generate both cryptic, as in
 <em>
  Biston betularia
 </em>
 , and
 <a href="https://en.wikipedia.org/wiki/Aposematism">
  aposematic
 </a>
 , as in
 <em>
  Heliconius
 </em>
 genus, patterns.
</p>
<p>
 As of this release, UniProtKB/Swiss-Prot
 <a href="http://www.uniprot.org/uniprot/?query=citation:(id:27251285)%20OR%20citation:(id:27251284)&amp;fil=reviewed%3Ayes">
  <em>
   Biston betularia
  </em>
  ,
  <em>
   Heliconius melpomene
  </em>
  and
  <em>
   Heliconius erato
  </em>
  cortex entries
 </a>
 have been updated with this new knowledge and are publicly available.
</p>
<h2 id="UniProtKB_news">
 UniProtKB news
</h2>
<h3 id="Cross_references_to_Conserved_Domains_Database">
 Cross-references to Conserved Domains Database
</h3>
<p>
 Cross-references have been added to the Conserved Domains Database (CDD), a protein annotation resource that consists of a collection of well-annotated multiple sequence alignment models for ancient domains and full-length proteins.
</p>
<p>
 CDD is available at
 <a href="https://www.ncbi.nlm.nih.gov/cdd">
  https://www.ncbi.nlm.nih.gov/cdd
 </a>
 .
</p>
<p>
 The format of the explicit links is:
</p>
<table>
 <tr>
  <th>
   Resource abbreviation
  </th>
  <td>
   CDD
  </td>
 </tr>
 <tr>
  <th>
   Resource identifier
  </th>
  <td>
   CDD identifier
  </td>
 </tr>
 <tr>
  <th>
   Optional information 1
  </th>
  <td>
   CDD model name
  </td>
 </tr>
 <tr>
  <th>
   Optional information 2
  </th>
  <td>
   Number of hits
  </td>
 </tr>
</table>
<p>
 Example:
 <a href="http://www.uniprot.org/uniprot/Q196W5#family%5Fand%5Fdomains">
  Q196W5
 </a>
</p>
<p>
 <a href="http://www.uniprot.org/uniprot/?query=database:cdd&amp;sort=score">
  Show all entries having a cross-reference to CDD.
 </a>
</p>
<h4 id="Text_format">
 Text format
</h4>
<p>
 Example:
 <a href="http://www.uniprot.org/uniprot/Q196W5.txt">
  Q196W5
 </a>
</p>
<pre>DR   CDD; cd04278; ZnMc_MMP; 1.
</pre>
<h4 id="XML_format">
 XML format
</h4>
<p>
 Example:
 <a href="http://www.uniprot.org/uniprot/Q196W5.xml">
  Q196W5
 </a>
</p>
<pre>&lt;dbReference type="CDD" id="cd04278"&gt;
  &lt;property type="entry name" value="ZnMc_MMP"/&gt;
  &lt;property type="match status" value="1"/&gt;
&lt;/dbReference&gt;
</pre>
<h4 id="RDF_format">
 RDF format
</h4>
<p>
 Example:
 <a href="http://www.uniprot.org/uniprot/Q196W5.ttl">
  Q196W5
 </a>
</p>
<pre>uniprot:Q196W5
  rdfs:seeAlso &lt;http://purl.uniprot.org/cdd/cd04278&gt; .
&lt;http://purl.uniprot.org/cdd/cd04278&gt;
  rdf:type up:Resource ;
  up:database &lt;http://purl.uniprot.org/database/CDD&gt; ;
  rdfs:comment "ZnMc_MMP" .
</pre>
<h3 id="Change_of_the_cross_references_to_VectorBase">
 Change of the cross-references to VectorBase
</h3>
<p>
 We have modified our cross-references to the
 <a href="http://www.vectorbase.org/">
  VectorBase
 </a>
 database. We now use the VectorBase Transcript identifier as the primary resource identifier, while showing the VectorBase Protein and Gene identifiers in additional fields.
</p>
<p>
 VectorBase is available at
 <a href="http://vectorbase.org">
  http://vectorbase.org
 </a>
 .
</p>
<p>
 The new format of the explicit links is:
</p>
<table>
 <tr>
  <th>
   Resource abbreviation
  </th>
  <td>
   VectorBase
  </td>
 </tr>
 <tr>
  <th>
   Resource identifier
  </th>
  <td>
   Transcript identifier
  </td>
 </tr>
 <tr>
  <th>
   Optional information 1
  </th>
  <td>
   Protein identifier
  </td>
 </tr>
 <tr>
  <th>
   Optional information 2
  </th>
  <td>
   Gene identifier
  </td>
 </tr>
</table>
<p>
 Example:
 <a href="http://www.uniprot.org/uniprot/A7UVJ5#sequences">
  A7UVJ5
 </a>
</p>
<p>
 <a href="http://www.uniprot.org/uniprot/?query=database:vectorbase&amp;sort=score">
  Show all entries having a cross-reference to VectorBase.
 </a>
</p>
<h4 id="Text_format">
 Text format
</h4>
<p>
 Example:
 <a href="http://www.uniprot.org/uniprot/A7UVJ5.txt">
  A7UVJ5
 </a>
</p>
<p>
 Previous format:
</p>
<pre>DR   VectorBase; AGAP001789. Anopheles gambiae.
</pre>
<p>
 New format:
</p>
<pre>DR   VectorBase; AGAP001789-RA; AGAP001789-PA; AGAP001789.
</pre>
<h4 id="XML_format">
 XML format
</h4>
<p>
 Example:
 <a href="http://www.uniprot.org/uniprot/A7UVJ5.xml">
  A7UVJ5
 </a>
</p>
<p>
 Previous format:
</p>
<pre>&lt;dbReference type="VectorBase" id="AGAP001789"&gt;
  &lt;property type="organism name" value="Anopheles gambiae"/&gt;
&lt;/dbReference&gt;
</pre>
<p>
 New format:
</p>
<pre>&lt;dbReference type="VectorBase" id="AGAP001789-RA"&gt;
  &lt;property type="protein sequence ID" value="AGAP001789-PA"/&gt;
  &lt;property type="gene ID" value="AGAP001789"/&gt;
&lt;/dbReference&gt;
</pre>
<p>
 This change does not affect the XSD, but may nevertheless require code changes.
</p>
<h4 id="RDF_format">
 RDF format
</h4>
<p>
 Example:
 <a href="http://www.uniprot.org/uniprot/A7UVJ5.ttl">
  A7UVJ5
 </a>
</p>
<p>
 Previous format:
</p>
<pre>uniprot:A7UVJ5
  rdfs:seeAlso &lt;http://purl.uniprot.org/vectorbase/AGAP001789&gt; .
&lt;http://purl.uniprot.org/vectorbase/AGAP001789&gt;
  rdf:type up:Resource ;
  up:database &lt;http://purl.uniprot.org/database/VectorBase&gt; ;
  rdfs:comment "Anopheles gambiae" .
</pre>
<p>
 New format:
</p>
<pre>uniprot:A7UVJ5
  rdfs:seeAlso &lt;http://purl.uniprot.org/vectorbase/AGAP001789-RA&gt; .
&lt;http://purl.uniprot.org/vectorbase/AGAP001789-RA&gt;
  rdf:type up:Transcript_Resource ;
  up:database &lt;http://purl.uniprot.org/database/VectorBase&gt; ;
  up:translatedTo &lt;http://purl.uniprot.org/vectorbae/AGAP001789-PA&gt; ;
  up:transcribedFrom &lt;http://purl.uniprot.org/vectorbase/AGAP001789&gt; .
</pre>
<h3 id="Change_of_the_cross_references_to_WormBase">
 Change of the cross-references to WormBase
</h3>
<p>
 Cross-references to WormBase may now be isoform-specific. The general format of isoform-specific cross-references was described in release
 <a href="http://www.uniprot.org/news/2014/03/19/release">
  2014_03
 </a>
 .
</p>
<p>
 Example:
 <a href="http://www.uniprot.org/uniprot/P00167">
  P00167
 </a>
</p>
<h3 id="Changes_to_the__a_href___docs_humdisease__controlled_vocabulary_of_human_diseases__a_">
 Changes to the
 <a href="http://www.uniprot.org/docs/humdisease">
  controlled vocabulary of human diseases
 </a>
</h3>
<p>
 New diseases:
</p>
<ul>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-04717">
   Agammaglobulinemia 8, autosomal dominant
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-04715">
   Bartter syndrome, type 5, antenatal, transient
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-04724">
   Chorea, childhood-onset, with psychomotor retardation
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-04718">
   Coffin-Siris syndrome 5
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-04729">
   Deafness, autosomal dominant, 66
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-04728">
   Deafness, autosomal dominant, 70
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-04727">
   Deafness, autosomal recessive, 105
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-04726">
   Hypercalcemia, infantile, 2
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-04716">
   Mental retardation, autosomal dominant 41
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-04731">
   Mental retardation, autosomal dominant 42
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-04722">
   Premature ovarian failure 11
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-04723">
   Premature ovarian failure 12
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-04725">
   Retinitis pigmentosa and erythrocytic microcytosis
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-04721">
   Spermatogenic failure, 15
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-04713">
   Spinocerebellar ataxia, autosomal recessive, 22
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-04714">
   Spinocerebellar ataxia, autosomal recessive, 23
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-04730">
   Spondyloepimetaphyseal dysplasia, Genevieve type
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-04719">
   Thrombocytopenia 6
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-04720">
   Trichothiodystrophy 6, non-photosensitive
  </a>
 </li>
</ul>
<h2 id="UniProt_website_news">
 UniProt website news
</h2>
<h3 id="Peptide_search_tool">
 Peptide search tool
</h3>
<p>
 We have introduced a new tool called
 <a href="http://www.uniprot.org/peptidesearch">
  Peptide search
 </a>
 that is available from a link in the header of the UniProt website. You can enter one or several peptide sequences (for example from a proteomics experiment) into the search field and the tool quickly finds all UniProtKB sequences that exactly match one of your query sequences. Searches can be restricted to a taxonomic subset of UniProtKB to decrease the search time. The tool returns a results page showing the matched UniProtKB entries in a design consistent with the UniProtKB text search results page, including filters on the left, results on the right and an option to customise the results table through the 'Columns' button.
</p>
<h3 id="Publications_view_added_to_UniProtKB_entries">
 Publications view added to UniProtKB entries
</h3>
<p>
 UniProt Knowledgebase (UniProtKB) protein entries now have a dedicated view of publications relevant for a protein. UniProtKB contains more than 350,000 unique publications, with over 210,000 of these fully curated in UniProtKB/Swiss-Prot and the remainder imported in UniProtKB/TrEMBL. This set is complemented by more than 640,000 additional publications that have been computationally mapped from other resources to UniProtKB entries. The publications annotated in UniProtKB have previously been displayed in the main 'Entry' view and a link provided access to a separate page that listed the computationally mapped publications. We have now combined all publications into a new 'Publications' view that can be accessed from a link under the 'Display' heading on the left hand side of a UniProtKB page. In this view you can filter the publications list by source and categories that are based on the type of data a publication contains about the protein (such as function, interaction, sequence, etc.) or the number of proteins it describes ('small scale' vs 'large scale'), see for example
 <a href="http://www.uniprot.org/uniprot/P10276/publications">
  P10276
 </a>
 .
</p>