<h3 id="Another_one__antibiotic__bites_the_dust">
 Another one (antibiotic) bites the dust
</h3>
<p>
 Polymyxin E (also known as colistin) and other polymyxin antibiotics are among our
 <a href="http://www.ncbi.nlm.nih.gov/pubmed/26603171">
  last-resort drugs against multi-drug resistant Gram-negative bacteria
 </a>
 such as
 <em>
  Klebsiella pneumoniae
 </em>
 ,
 <em>
  Pseudomonas aeruginosa
 </em>
 and
 <em>
  Acinetobacter
 </em>
 .
</p>
<p>
 The initial target of
 <a href="http://www.ncbi.nlm.nih.gov/pubmed/19874036">
  polymyxin antibiotics
 </a>
 is the lipopolysaccharide layer (LPS) of the Gram-negative bacterial outer membrane. LPS has two 2-keto-3-deoxyoctonoic acid units bound to lipid A, which itself consists of 2 glucosamine units with attached fatty acyl chains and a phosphate group on each sugar. Lipid A acts as a hydrophobic anchor, in which the tight packing of the fatty acyl chains helps to stabilize the overall outer membrane structure. The positively charged L-2,4-diaminobutyric acid residues of polymyxins interact with the negatively charged phosphate groups on lipid A. The amphipathic antibiotics are thought to form pores that permeabilize the outer membrane. The polymyxins would then insert into and disrupt the inner membrane, leading to further pore formation. There is also some evidence that polymyxins have other intracellular targets.
</p>
<p>
 As the initial contact of polymyxin antibiotics is with lipid A, resistance often occurs via its modification, frequently masking its negative charge. Before August 2015 a number of chromosomal resistance loci were known, but no resistance had been identified on a more easily transferred plasmid. During a routine surveillance of commensal
 <em>
  Escherichia coli
 </em>
 for antibiotic resistance, scientists in China identified
 <a href="http://www.uniprot.org/uniprot/A0A0R6L508#names%5Fand%5Ftaxonomy">
  mcr1
 </a>
 , a plasmid-encoded gene which encodes a protein of the phosphoethanolamine transferase family. The gene confers both
 <a href="http://www.ncbi.nlm.nih.gov/pubmed/26603172">
  colistin and polymyxin B resistance
 </a>
 by modifying lipid A, and probably originated in
 <a href="https://en.wikipedia.org/wiki/Paenibacillus">
  <em>
   Paenibacillus
  </em>
 </a>
 . This would seem logical as
 <em>
  Paenibacillus
 </em>
 is the natural source of
 <a href="http://www.ncbi.nlm.nih.gov/pubmed/16561391">
  polymyxin antibiotics
 </a>
 .
</p>
<p>
 The gene was first identified from a pig farm in Shanghai in July 2013. Retrospective screening of isolated
 <em>
  E.coli
 </em>
 plasmids in China showed an alarming rise in its presence in pork, ranging from 6% in 2011 to 22% in 2014. The gene has also been detected in chicken meat in China, rising from 5% in 2011 to 28% in 2014. Screening hospital inpatients in 2014 showed both
 <em>
  E.coli
 </em>
 and
 <em>
  K.pneumoniae
 </em>
 mcr1-containing plasmid; 1.4% from
 <em>
  E.coli
 </em>
 , 0.7% from
 <em>
  K.pneumoniae
 </em>
 . The gene was also detected in
 <em>
  E.coli
 </em>
 genomes from Malaysia. An
 <em>
  in situ
 </em>
 test in mice showed that the gene was indeed able to confer colistin resistance. The original plasmid can transfer to other
 <em>
  E.coli
 </em>
 cells via conjugation, but only via transformation into
 <em>
  K.pneumoniae
 </em>
 or
 <em>
  P.aeruginosa
 </em>
 ; it is stable in the absence of selective pressure.
</p>
<p>
 Since the publication of the paper identifying mcr1 on-line November 15, 2105, numerous papers have appeared reporting retrospective screening for the gene. So far its earliest isolation is from a French calf in 2005, in which a worrying
 <a href="http://www.ncbi.nlm.nih.gov/pubmed/26774244">
  co-localization with a wide-spectrum beta-lactamase resistance gene was also reported
 </a>
 . The gene has been
 <a href="http://www.ncbi.nlm.nih.gov/pubmed/26711359,26711360,26711361,26774239">
  found in human fecal samples
 </a>
 dating from 2012 on, in Europe, Africa, South America and Asia. It was found in
 <em>
  E.coli
 </em>
 isolated from
 <a href="http://www.ncbi.nlm.nih.gov/pubmed/26774242">
  pigs in Germany
 </a>
 in 2010, from
 <a href="http://www.ncbi.nlm.nih.gov/pubmed/26774247">
  Belgian calves
 </a>
 in 2011-2012, in
 <a href="http://www.ncbi.nlm.nih.gov/pubmed/26603172,26711359,26711360,26711362,26711363">
  European food samples
 </a>
 from June 2011 on, and from
 <a href="http://www.ncbi.nlm.nih.gov/pubmed/26711360,26774239,26774240,26774245,26774248">
  animal feces in Asia
 </a>
 . The gene is not always isolated from the same plasmid background, and mcr1 is often associated with mobile genetic elements, probably aiding its dispersal.
</p>
<p>
 In short, the gene has been slowly spreading around the world since before we were even aware of its existence. Colistin has been used in agriculture since the 1950s and is widely used in China, which is probably contributing to its steady dissemination. There are
 <a href="http://www.ema.europa.eu/ema/index.jsp?curl=pages/news%5Fand%5Fevents/news/2016/01/news%5Fdetail%5F002455.jsp&amp;mid=WC0b01ac058004d5c1">
  increasingly urgent calls
 </a>
 for its agricultural use to be reevaluated before resistance spreads even further.
</p>
<p>
 As of this release,
 <a href="http://www.uniprot.org/uniprot/A0A0R6L508">
  Mcr-1
 </a>
 has been annotated and is available in UniProtKB/Swiss-Prot.
</p>
<h3 id="Cross_references_to_SwissPalm">
 Cross-references to SwissPalm
</h3>
<p>
 Cross-references have been added to SwissPalm, a manually curated resource to study protein S-palmitoylation. It encompasses S-palmitoylated protein hits from more than 50 species and provides curated information and filters that increase the confidence in true positive hits. SwissPalm integrates predictions of S-palmitoylated cysteine scores, orthologs and isoform multiple alignments.
</p>
<p>
 SwissPalm is available at
 <a href="http://swisspalm.epfl.ch/">
  http://swisspalm.epfl.ch/
 </a>
 .
</p>
<p>
 The format of the explicit links is:
</p>
<table>
 <tr>
  <th>
   Resource abbreviation
  </th>
  <td>
   SwissPalm
  </td>
 </tr>
 <tr>
  <th>
   Resource identifier
  </th>
  <td>
   UniProtKB accession number.
  </td>
 </tr>
</table>
<p>
 Example:
 <a href="http://www.uniprot.org/uniprot/Q13530">
  Q13530
 </a>
</p>
<p>
 <a href="http://www.uniprot.org/uniprot/?query=database%3Aswisspalm&amp;sort=score">
  Show all entries having a cross-reference to SwissPalm.
 </a>
</p>
<h4 id="Text_format">
 Text format
</h4>
<p>
 Example:
 <a href="http://www.uniprot.org/uniprot/Q13530.txt">
  Q13530
 </a>
</p>
<pre>DR   SwissPalm; Q13530; -.
</pre>
<h4 id="XML_format">
 XML format
</h4>
<p>
 Example:
 <a href="http://www.uniprot.org/uniprot/Q13530.xml">
  Q13530
 </a>
</p>
<pre>&lt;dbReference type="SwissPalm" id="Q13530"/&gt;
</pre>
<h4 id="RDF_format">
 RDF format
</h4>
<p>
 Example:
 <a href="http://www.uniprot.org/uniprot/Q13530.ttl">
  Q13530
 </a>
</p>
<pre>uniprot:Q13530
  rdfs:seeAlso &lt;http://purl.uniprot.org/swisspalm/Q13530&gt; .
&lt;http://purl.uniprot.org/swisspalm/Q13530&gt;
  rdf:type up:Resource ;
  up:database &lt;http://purl.uniprot.org/database/SwissPalm&gt; .
</pre>
<h3 id="Change_of_the_cross_references_to_Gramene">
 Change of the cross-references to Gramene
</h3>
<p>
 We have modified our cross-references to the
 <a href="http://www.gramene.org/">
  Gramene
 </a>
 database.
</p>
<p>
 The new format of the explicit links is:
</p>
<table>
 <tr>
  <th>
   Resource abbreviation
  </th>
  <td>
   Gramene
  </td>
 </tr>
 <tr>
  <th>
   Resource identifier
  </th>
  <td>
   Transcript identifier
  </td>
 </tr>
 <tr>
  <th>
   Optional information 1
  </th>
  <td>
   Protein identifier
  </td>
 </tr>
 <tr>
  <th>
   Optional information 2
  </th>
  <td>
   Gene identifier
  </td>
 </tr>
</table>
<p>
 Cross-references to Gramene may be isoform-specific. The general format of isoform-specific cross-references was described in release
 <a href="http://www.uniprot.org/news/2014/03/19/release">
  2014_03
 </a>
 .
</p>
<p>
 The Gramene database has also been moved from the category "Organism-specific databases" to the category "Genome annotation databases".
</p>
<p>
 Example:
 <a href="http://www.uniprot.org/uniprot/Q10DK7#sequences">
  Q10DK7
 </a>
</p>
<p>
 <a href="http://www.uniprot.org/uniprot/?query=database:gramene&amp;sort=score">
  Show all entries having a cross-reference to Gramene.
 </a>
</p>
<h4 id="Text_format">
 Text format
</h4>
<p>
 Example:
 <a href="http://www.uniprot.org/uniprot/Q10DK7.txt">
  Q10DK7
 </a>
</p>
<p>
 Previous format:
</p>
<pre>DR   Gramene; Q10DK7; -.
</pre>
<p>
 New format:
</p>
<pre>DR   Gramene; OS03T0727600-01; OS03T0727600-01; OS03G0727600.
</pre>
<h4 id="XML_format">
 XML format
</h4>
<p>
 Example:
 <a href="http://www.uniprot.org/uniprot/Q10DK7.xml">
  Q10DK7
 </a>
</p>
<p>
 Previous format:
</p>
<pre>&lt;dbReference type="Gramene" id="Q10DK7"/&gt;
</pre>
<p>
 New format:
</p>
<pre>&lt;dbReference type="Gramene" id="OS03T0727600-01"&gt;
  &lt;property type="protein sequence ID" value="OS03T0727600-01"/&gt;
  &lt;property type="gene ID" value="OS03G0727600"/&gt;
&lt;/dbReference&gt;
</pre>
<p>
 This change does not affect the XSD, but may nevertheless require code changes.
</p>
<h4 id="RDF_format">
 RDF format
</h4>
<p>
 Example:
 <a href="http://www.uniprot.org/uniprot/Q10DK7.ttl">
  Q10DK7
 </a>
</p>
<p>
 Previous format:
</p>
<pre>uniprot:Q10DK7
  rdfs:seeAlso &lt;http://purl.uniprot.org/gramene/Q10DK7&gt; .
&lt;http://purl.uniprot.org/gramene/Q10DK7&gt;
  rdf:type up:Resource ;
  up:database &lt;http://purl.uniprot.org/database/Gramene&gt; .
</pre>
<p>
 New format:
</p>
<pre>uniprot:Q10DK7
  rdfs:seeAlso &lt;http://purl.uniprot.org/gramene/OS03T0727600-01&gt; .
&lt;http://purl.uniprot.org/gramene/OS03T0727600-01&gt;
  rdf:type up:Transcript_Resource ;
  up:database &lt;http://purl.uniprot.org/database/Gramene&gt; ;
  up:translatedTo &lt;http://purl.uniprot.org/gramene/OS03T0727600-01&gt; ;
  up:transcribedFrom &lt;http://purl.uniprot.org/gramene/OS03G0727600&gt; .
</pre>
<h3 id="Removal_of_the_cross_references_to_GeneFarm">
 Removal of the cross-references to GeneFarm
</h3>
<p>
 Cross-references to GeneFarm have been removed.
</p>
<h3 id="Removal_of_the_cross_references_to_GenoList">
 Removal of the cross-references to GenoList
</h3>
<p>
 Cross-references to GenoList have been removed.
</p>
<h3 id="Changes_to_the__a_href___docs_humdisease__controlled_vocabulary_of_human_diseases__a_">
 Changes to the
 <a href="http://www.uniprot.org/docs/humdisease">
  controlled vocabulary of human diseases
 </a>
</h3>
<p>
 New diseases:
</p>
<ul>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-04574">
   Charcot-Marie-Tooth disease 2W
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-04581">
   Epilepsy, progressive myoclonic 10
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-04577">
   Epileptic encephalopathy, early infantile, 34
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-04578">
   Epileptic encephalopathy, early infantile, 35
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-04579">
   Joubert syndrome 24
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-04582">
   Mental retardation, X-linked, syndromic, 11
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-04583">
   Mitral valve prolapse 2
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-04570">
   Porokeratosis 1, multiple types
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-04571">
   Porokeratosis 7, multiple types
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-04569">
   Porokeratosis 9, multiple types
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-04573">
   Ritscher-Schinzel syndrome 2
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-04572">
   Seizures, cortical blindness, microcephaly syndrome
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-04575">
   Senior-Loken syndrome 9
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-04580">
   Spastic tetraplegia, thin corpus callosum, and progressive microcephaly
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-04576">
   Smith-Kingsmore syndrome
  </a>
 </li>
</ul>
<p>
 Modified diseases:
</p>
<ul>
 <li>
  Disseminated superficial actinic porokeratosis 1 -&gt;
  <a href="http://www.uniprot.org/diseases/DI-01490">
   Porokeratosis 3, multiple types
  </a>
 </li>
 <li>
  Elliptocytosis 4 -&gt;
  <a href="http://www.uniprot.org/diseases/DI-00448">
   Ovalocytosis, Southeast Asian
  </a>
 </li>
 <li>
  Hypercatabolic hypoproteinemia -&gt;
  <a href="http://www.uniprot.org/diseases/DI-01765">
   Immunodeficiency 43
  </a>
 </li>
 <li>
  Optic atrophy 8 -&gt;
  <a href="http://www.uniprot.org/diseases/DI-04381">
   Optic atrophy 9
  </a>
 </li>
 <li>
  Ritscher-Schinzel syndrome -&gt;
  <a href="http://www.uniprot.org/diseases/DI-04011">
   Ritscher-Schinzel syndrome 1
  </a>
 </li>
</ul>
<p>
 Deleted diseases
</p>
<ul>
 <li>
  Periventricular nodular heterotopia 4
 </li>
 <li>
  Transposition of the great arteries dextro-looped 2
 </li>
</ul>
<h2 id="UniProt_website_news">
 UniProt website news
</h2>
<h3 id="UniProt_feature_viewer_added_to_UniProtKB_entries">
 UniProt feature viewer added to UniProtKB entries
</h3>
<p>
 UniProt provides
 <a href="http://www.uniprot.org/help/sequence%5Fannotation">
  sequence annotations
 </a>
 , a.k.a. protein features, to describe regions or sites of biological interest;
 <a href="http://www.uniprot.org/help/structure%5Fsection">
  secondary structure regions
 </a>
 ,
 <a href="http://www.uniprot.org/help/domain">
  domains
 </a>
 ,
 <a href="http://www.uniprot.org/help/ptm%5Fprocessing%5Fsection">
  post-translational modifications
 </a>
 and
 <a href="http://www.uniprot.org/help/binding">
  binding sites
 </a>
 among others, play a critical role in the understanding of what the protein does. With the growth in biological data, integration and visualization becomes increasingly important for exposing different data aspects that might be otherwise hidden, unclear or difficult to grasp.
</p>
<p>
 Hence we are introducing the UniProt feature viewer, a BioJS component bringing together protein sequence features in one compact view. Similar to genome viewers, the viewer uses tracks to display different protein features providing an intuitive picture of co-localized elements. Each track can be expanded to reveal a more in-depth view of the underlying data. The variant track offers a novel visualization and presents UniProt curated
 <a href="http://www.uniprot.org/help/variant">
  natural variants
 </a>
 along with imported variants from large-scale studies (such as 1000 Genomes and COSMIC).
</p>
<p>
 The UniProt feature viewer is available for every UniProtKB protein entry through the 'Feature viewer' link under the 'Display' heading on the left hand side.
</p>
<p>
 If you would like to include the feature viewer in your own website or resource, you can find instructions in our
 <a href="http://ebi-uniprot.github.io/biojs-vis-proteinFeaturesViewer/developerGuide.html">
  technical documentation
 </a>
 .
</p>