<h2 id="Headline">
 Headline
</h2>
<h3 id="From_mouth_to_gut__a_new_mechanism_for_fimbria_assembly">
 From mouth to gut, a new mechanism for fimbria assembly
</h3>
<p>
 Fighting the oral microbiome is a daily task. Ineffective oral hygiene leads not only to dental caries, but also to inflammatory gum diseases, such as gingivitis. In some cases, gingivitis can worsen and turn into periodontitis, which involves the chronic destruction of connective tissues, including that of the alveolar bone around the teeth, and consequently loosening and subsequent loss of teeth. We are not all equally affected by periodontal diseases. There are marked differences in disease progression rate and severity, reflecting personal susceptibility, diversity in virulence among the microorganism species (and subspecies) and environmental conditions. Despite these variables,
 <a href="https://www.ncbi.nlm.nih.gov/pubmed/26903954">
  <em>
   Porphyromonas gingivalis
  </em>
 </a>
 is now recognized as a major contributor to periodontitis. This Gram-negative black-pigmented anaerobic rod resides in subgingival biofilms and harbors an arsenal of virulence factors, among which are fimbriae (also called pili). Described for the first time in the
 <a href="https://www.ncbi.nlm.nih.gov/pubmed/13295908">
  early 1950s
 </a>
 , fimbriae are non-flagellar appendages, formed by the assembly of proteins called pilins at the bacterial surface. They are often involved in the initial adhesion of the bacteria to host tissues during colonization, and also in biofilm formation, cell motility (
 <a href="https://www.youtube.com/watch?v=yGMSQNBDq48">
  twitching mobility
 </a>
 ), and transport of proteins and DNA across cell membranes. There are major (long) and minor (short) fimbriae, both containing a structural, stalk-forming subunit (
 <a href="http://www.uniprot.org/uniprot/?query=gene:fima+and+taxonomy:Porphyromonas+gingivalis+and+reviewed:yes">
  FimA
 </a>
 for the major fimbriae,
 <a href="http://www.uniprot.org/uniprot/?query=gene:mfa1+and+taxonomy:Porphyromonas+gingivalis+and+reviewed:yes">
  Mfa1
 </a>
 for the minor fimbriae) and 3 accessory subunits (
 <a href="http://www.uniprot.org/uniprot/?query=gene:fimc+and+taxonomy:Porphyromonas+gingivalis+and+reviewed:yes">
  FimC
 </a>
 ,
 <a href="http://www.uniprot.org/uniprot/?query=gene:fimd+and+taxonomy:Porphyromonas+gingivalis+and+reviewed:yes">
  FimD
 </a>
 and
 <a href="http://www.uniprot.org/uniprot/?query=gene:fime+and+taxonomy:Porphyromonas+gingivalis+and+reviewed:yes">
  FimE
 </a>
 for the major fimbriae;
 <a href="http://www.uniprot.org/uniprot/?query=gene:mfa3+and+taxonomy:Porphyromonas+gingivalis+and+reviewed:yes">
  Mfa3
 </a>
 ,
 <a href="http://www.uniprot.org/uniprot/?query=gene:mfa4+and+taxonomy:Porphyromonas+gingivalis+and+reviewed:yes">
  Mfa4
 </a>
 and
 <a href="http://www.uniprot.org/uniprot/?query=gene:mfa5+and+taxonomy:Porphyromonas+gingivalis">
  Mfa5
 </a>
 for the minor fimbriae) thought to form the fimbria tip. The last subunit is
 <a href="http://www.uniprot.org/uniprot/?query=gene:fimb+and+taxonomy:Porphyromonas+gingivalis+and+reviewed:yes">
  FimB
 </a>
 (major fimbriae) or
 <a href="http://www.uniprot.org/uniprot/?query=gene:mfa2+and+taxonomy:Porphyromonas+gingivalis+and+reviewed:yes">
  Mfa2
 </a>
 (minor fimbriae), which anchors the pilus to the outer membrane.
</p>
<p>
 A
 <a href="https://www.ncbi.nlm.nih.gov/pubmed/27062925">
  very thorough study
 </a>
 published last April, combining X-ray structure, biochemical and mutational analyses, sheds new light on the fimbria assembly mechanism in several bacteria from the
 <a href="http://www.uniprot.org/taxonomy/200643">
  <em>
   Bacteroidia
  </em>
  class
 </a>
 , including
 <em>
  P. gingivalis
 </em>
 . The assembly occurs from tip to base. A tip pilin monomer is incorporated first, followed by stalk-forming structural pilin subunits and finally an anchor pilin at the base. Tip and structural pilins are synthesized in the cytoplasm as lipoprotein precursors, and exported into the periplasm using the Sec pathway. In the periplasm, they are folded and become lipidated at the N-terminus. The modified pilins are then exported across the outer membrane. During this process, they undergo a cleavage that releases the lipid moiety and several amino acids from the N-terminus, creating a groove. At this stage, mature structural pilins adopt an extended "open" conformation, allowing the assembly of the fimbriae where a C-terminal extension binds to the N-terminal groove of the previous subunit, a little like interlocking Lego bricks. The tip pilins exhibit a similar N-terminal groove to accommodate the C-terminal extension from structural pilin, but their C-terminus remains buried. Anchor pilins do not undergo cleavage and remain tethered to the outer membrane. As for structural pilin subunits, their C-terminus is involved in their incorporation into fimbriae.
</p>
<p>
 Although fimbria assembly has been studied in numerous phylogenetically distinct bacteria, until this recent publication, very little was known about pilin structure and assembly in human-associated
 <em>
  Bacteroidales
 </em>
 members. The reported mechanism was hitherto unseen, but it could be widespread. Indeed, FimA proteins represent a large and diverse superfamily, which is highly represented in the gut microbiome, suggesting that they may confer adaptive advantages in bacterial colonization of this environment.
</p>
<p>
 Close to
 <a href="http://www.uniprot.org/uniprot/?query=B2RH54+OR+B2RHG4+OR+B2RHG1+OR+B2RHG2+OR+B2RHG3+OR+P59914+OR+Q51822+OR+Q51827+OR+P81363+OR+Q51825+OR+B2RHG5+OR+Q9S0W8+OR+Q51826+OR+Q93R80+OR+B2RH59+OR+P0DOA1+OR+B2RH57+OR+Q7MXK0+OR+B2RH58+OR+A0PA81+OR+O32390+OR+O32388+OR+O32389+OR+A7LXW1+OR+Q7MT55+OR+A0PA72">
  30 entries
 </a>
 have been updated in UniProtKB/Swiss-Prot to include these new findings. The entries can be consulted just as well before or after brushing your teeth!
</p>
<h2 id="UniProtKB_news">
 UniProtKB news
</h2>
<h3 id="Changes_to_the__a_href___docs_humdisease__controlled_vocabulary_of_human_diseases__a_">
 Changes to the
 <a href="http://www.uniprot.org/docs/humdisease">
  controlled vocabulary of human diseases
 </a>
</h3>
<p>
 New diseases:
</p>
<ul>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-04825">
   Alazami-Yuan syndrome
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-04829">
   Band heterotopia
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-04830">
   Bardet-Biedl syndrome 20
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-04822">
   Ciliary dyskinesia, primary, 34
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-04827">
   Ciliary dyskinesia, primary, 35
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-04817">
   Congenital bilateral aplasia of the vas deferens, X-linked
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-04832">
   Epilepsy, familial focal, with variable foci 2
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-04831">
   Epilepsy, familial focal, with variable foci 3
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-04837">
   Epileptic encephalopathy, early infantile, 41
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-04836">
   Epileptic encephalopathy, early infantile, 42
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-04835">
   Epileptic encephalopathy, early infantile, 43
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-04840">
   Familial adenomatous polyposis 4
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-04841">
   Growth retardation, intellectual developmental disorder, hypotonia, and hepatopathy
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-04819">
   Joubert syndrome 27
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-04820">
   Joubert syndrome 28
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-04818">
   Macular dystrophy, patterned, 3
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-04823">
   Mental retardation, autosomal recessive 56
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-04814">
   Mental retardation, X-linked 103
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-04815">
   Mental retardation, X-linked 104
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-04816">
   Mental retardation, X-linked 105
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-04821">
   Microcephaly 17, primary, autosomal recessive
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-04834">
   Myopathy, myofibrillar, 7
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-04826">
   Orofaciodigital syndrome 15
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-04833">
   Peeling skin syndrome 5
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-04824">
   Retinitis pigmentosa 76
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-04838">
   Sessile serrated polyposis cancer syndrome
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-04839">
   Thauvin-Robinet-Faivre syndrome
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-04828">
   Vascular malformation, primary intraosseous
  </a>
 </li>
</ul>
<h2 id="RDF_news">
 RDF news
</h2>
<h3 id="Change_of_URIs_for_Ensembl_and_Ensembl_Genomes">
 Change of URIs for Ensembl and Ensembl Genomes
</h3>
<p>
 For historic reasons, UniProt had to generate URIs to cross-reference databases that did not have an RDF representation. Our policy is to replace these by the URIs generated by the cross-referenced database once it starts to distribute an RDF representation of its data.
</p>
<p>
 We have therefore updated the URIs for the
 <a href="http://www.ensembl.org/">
  Ensembl
 </a>
 and
 <a href="http://www.ensemblgenomes.org/">
  Ensembl Genomes
 </a>
 databases from
</p>
<pre>http://purl.uniprot.org/ensembl/&lt;identifier&gt;
http://purl.uniprot.org/ensemblbacteria/&lt;identifier&gt;
http://purl.uniprot.org/ensemblfungi/&lt;identifier&gt;
http://purl.uniprot.org/ensemblmetazoa/&lt;identifier&gt;
http://purl.uniprot.org/ensemblplants/&lt;identifier&gt;
http://purl.uniprot.org/ensemblprotists/&lt;identifier&gt;
</pre>
<p>
 to
</p>
<ul>
 <li>
  <pre>http://rdf.ebi.ac.uk/resource/ensembl/&lt;identifier&gt;</pre>
  for genes
 </li>
 <li>
  <pre>http://rdf.ebi.ac.uk/resource/ensembl.transcript/&lt;identifier&gt;</pre>
  for transcripts
 </li>
 <li>
  <pre>http://rdf.ebi.ac.uk/resource/ensembl.protein/&lt;identifier&gt;</pre>
  for proteins
 </li>
</ul>