<h2 id="Headline">
 Headline
</h2>
<h3 id="cGAMP__a_welcome_stowaway">
 cGAMP, a welcome stowaway
</h3>
<p>
 We are often amazed by the strategies deployed by viruses to trick our defences, but our immune system does not lag behind and it can also fool viral invaders. The detection of viruses by the innate immune system relies on the detection of intracellular DNA by pattern recognition receptors, including cyclic guanosine monophosphate (GMP) adenosine monophosphate (AMP) synthase (
 <a href="http://www.uniprot.org/uniprot/?query=gene:mb21d1+AND+reviewed:yes">
  cGAS
 </a>
 , also called MB21D1). In response to cytosolic DNA, this enzyme synthesizes 2'3'-cyclic GMP-AMP (cGAMP), which then binds to
 <a href="http://www.uniprot.org/uniprot/?query=gene:tmem173+AND+reviewed:yes">
  STING
 </a>
 (also called TMEM173), an endoplasmic reticulum transmembrane protein, leading to the activation of the type I interferon (IFN) response, thereby inducing an antiviral state.
</p>
<p>
 Last year,
 <a href="http://www.ncbi.nlm.nih.gov/pubmed/26229115">
  Gentili et al.
 </a>
 made a puzzling observation. To study cGAS function, they transduced human monocyte-derived dendritic cells with a cGAS-expressing lentivirus. As expected, the cells were strongly activated, but the stimulatory property of the cGAS-encoding lentivirus did not correlate with the transduction efficiency. This led to the hypothesis that it was not cGAS itself that was responsible for the activation of the infected cells, but some other stimulatory signal, which was transferred by the viral vector. Indeed, when dendritic cells were challenged with virus-like particles (VLPs) that did not themselves encode cGAS, but were produced in the presence of cGAS, the cells were stimulated. This effect was abolished when VLPs were produced in the presence of a catalytically inactive cGAS mutant. Concomitantly,
 <a href="http://www.ncbi.nlm.nih.gov/pubmed/26229117">
  Bridgeman et al.
 </a>
 found that the incubation of macrophages, epithelial cells or lung fibroblasts with lentiviral particles collected from cells overexpressing cGAS led to the STING-dependent up-regulation of type I interferons and interferon-stimulated genes. All this evidence pointed to cGAMP as the stimulatory signal and indeed both groups identified the dinucleotide in the viral particles, by mass spectrometry, not only in their experimental system, but also in more physiological settings, using a herpes virus (MCMV) and a poxvirus (Modified Vaccinia Anakara virus). It is yet unclear whether the incorporation of cGAMP into virus particles is a selective host-directed process or simply a consequence of random fluid-phase uptake of cytosolic material into viral particles.
</p>
<p>
 cGAMP has
 <a href="http://www.ncbi.nlm.nih.gov/pubmed/24077100">
  previously been shown
 </a>
 to diffuse through gap junctions, thereby alerting non-infected neighboring cells to pathogen threat. The discovery by Gentili et al. and Bridgeman et al. suggests that cells located far from the initial infection site may also benefit from cGAMP transfer and initiate rapid antiviral responses bypassing the need for cGAS activation.
</p>
<p>
 Although the downstream fate of the dinucleotide does not directly depend on cGAS enzyme activity, this piece of information has been introduced into
 <a href="http://www.uniprot.org/uniprot/?query=gene:mb21d1+AND+reviewed:yes">
  cGAS
 </a>
 entries as of this release.
</p>
<h3 id="Cross_references_to_CollecTF">
 Cross-references to CollecTF
</h3>
<p>
 Cross-references have been added to the CollecTF database of bacterial transcription factor binding sites. CollecTF stores data on experimentally-validated TFBS and places special emphasis on providing a transparent curation process that captures the experimental support for sites as reported by authors in peer-reviewed publications.
</p>
<p>
 CollecTF is available at
 <a href="http://www.collectf.org">
  http://www.collectf.org
 </a>
 .
</p>
<p>
 The format of the explicit links is:
</p>
<table>
 <tr>
  <th>
   Resource abbreviation
  </th>
  <td>
   CollecTF
  </td>
 </tr>
 <tr>
  <th>
   Resource identifier
  </th>
  <td>
   CollecTF identifier
  </td>
 </tr>
</table>
<p>
 Example:
 <a href="http://www.uniprot.org/uniprot/A0KST7#expression">
  A0KST7
 </a>
</p>
<p>
 <a href="http://www.uniprot.org/uniprot/?query=database:collectf&amp;sort=score">
  Show all entries having a cross-reference to CollecTF.
 </a>
</p>
<h4 id="Text_format">
 Text format
</h4>
<p>
 Example:
 <a href="http://www.uniprot.org/uniprot/A0KST7.txt">
  A0KST7
 </a>
</p>
<pre>DR   CollecTF; EXPREG_00000150; -.
</pre>
<h4 id="XML_format">
 XML format
</h4>
<p>
 Example:
 <a href="http://www.uniprot.org/uniprot/A0KST7.xml">
  A0KST7
 </a>
</p>
<pre>&lt;dbReference type="CollecTF" id="EXPREG_00000150"/&gt;
</pre>
<h4 id="RDF_format">
 RDF format
</h4>
<p>
 Example:
 <a href="http://www.uniprot.org/uniprot/A0KST7.ttl">
  A0KST7
 </a>
</p>
<pre>uniprot:A0KST7
  rdfs:seeAlso &lt;http://purl.uniprot.org/collectf/EXPREG_00000150&gt; .
&lt;http://purl.uniprot.org/collectf/EXPREG_00000150&gt;
  rdf:type up:Resource ;
  up:database &lt;http://purl.uniprot.org/database/CollecTF&gt; .
</pre>
<h3 id="Cross_references_to_GeneDB">
 Cross-references to GeneDB
</h3>
<p>
 Cross-references have been added to GeneDB pathogen genome database from Sanger Institute. GeneDB provides access to the latest sequence data and annotation/curation for the whole range of organisms sequenced by the Sanger Pathogen group.
</p>
<p>
 GeneDB is available at
 <a href="http://www.genedb.org">
  http://www.genedb.org
 </a>
 .
</p>
<p>
 The format of the explicit links is:
</p>
<table>
 <tr>
  <th>
   Resource abbreviation
  </th>
  <td>
   GeneDB
  </td>
 </tr>
 <tr>
  <th>
   Resource identifier
  </th>
  <td>
   GeneDB identifier
  </td>
 </tr>
</table>
<p>
 Example:
 <a href="http://www.uniprot.org/uniprot/Q8WPT5#cross-references">
  Q8WPT5
 </a>
</p>
<p>
 <a href="http://www.uniprot.org/uniprot/?query=database:genedb&amp;sort=score">
  Show all entries having a cross-reference to GeneDB.
 </a>
</p>
<h4 id="Text_format">
 Text format
</h4>
<p>
 Example:
 <a href="http://www.uniprot.org/uniprot/Q8WPT5.txt">
  Q8WPT5
 </a>
</p>
<pre>DR    GeneDB; H25N7.01:pep; -.
</pre>
<h4 id="XML_format">
 XML format
</h4>
<p>
 Example:
 <a href="http://www.uniprot.org/uniprot/Q8WPT5.xml">
  Q8WPT5
 </a>
</p>
<pre>&lt;dbReference type="GeneDB" id="H25N7.01:pep"/&gt;
</pre>
<h4 id="RDF_format">
 RDF format
</h4>
<p>
 Example:
 <a href="http://www.uniprot.org/uniprot/Q8WPT5.ttl">
  Q8WPT5
 </a>
</p>
<pre>uniprot:Q8WPT5
  rdfs:seeAlso &lt;http://purl.uniprot.org/genedb/H25N7.01:pep&gt; .
&lt;http://purl.uniprot.org/genedb/H25N7.01:pep&gt;
  rdf:type up:Resource ;
  up:database &lt;http://purl.uniprot.org/database/GeneDB&gt; .
</pre>
<h3 id="Cross_references_to_iPTMnet">
 Cross-references to iPTMnet
</h3>
<p>
 Cross-references have been added to iPTMnet integrated resource for PTMs in systems biology context. iPTMnet connects multiple disparate bioinformatics tools and systems text mining, data mining, analysis and visualization tools, and databases and ontologies into an integrated resource to address the knowledge gaps in exploring and discovering PTM networks. iPTMnet database currently contains phosphorylation information.
</p>
<p>
 iPTMnet is available at
 <a href="http://pir.georgetown.edu/iPTMnet">
  http://pir.georgetown.edu/iPTMnet
 </a>
 .
</p>
<p>
 The format of the explicit links is:
</p>
<table>
 <tr>
  <th>
   Resource abbreviation
  </th>
  <td>
   iPTMnet
  </td>
 </tr>
 <tr>
  <th>
   Resource identifier
  </th>
  <td>
   UniProtKB accession number.
  </td>
 </tr>
</table>
<p>
 Example:
 <a href="http://www.uniprot.org/uniprot/Q15796">
  Q15796
 </a>
</p>
<p>
 <a href="http://www.uniprot.org/uniprot/?query=database%3Aiptmnet&amp;sort=score">
  Show all entries having a cross-reference to iPTMnet.
 </a>
</p>
<h4 id="Text_format">
 Text format
</h4>
<p>
 Example:
 <a href="http://www.uniprot.org/uniprot/Q15796.txt">
  Q15796
 </a>
</p>
<pre>DR   iPTMnet; Q15796; -.
</pre>
<h4 id="XML_format">
 XML format
</h4>
<p>
 Example:
 <a href="http://www.uniprot.org/uniprot/Q15796.xml">
  Q15796
 </a>
</p>
<pre>&lt;dbReference type="iPTMnet" id="Q15796"/&gt;
</pre>
<h4 id="RDF_format">
 RDF format
</h4>
<p>
 Example:
 <a href="http://www.uniprot.org/uniprot/Q15796.ttl">
  Q15796
 </a>
</p>
<pre>uniprot:Q15796
  rdfs:seeAlso &lt;http://purl.uniprot.org/iptmnet/Q15796&gt; .
&lt;http://purl.uniprot.org/iptmnet/Q15796&gt;
  rdf:type up:Resource ;
  up:database &lt;http://purl.uniprot.org/database/iPTMnet&gt; .
</pre>
<h3 id="Changes_to_the__a_href___docs_humdisease__controlled_vocabulary_of_human_diseases__a_">
 Changes to the
 <a href="http://www.uniprot.org/docs/humdisease">
  controlled vocabulary of human diseases
 </a>
</h3>
<p>
 New diseases:
</p>
<ul>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-04567">
   Bulimia nervosa 2
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-04568">
   Anorexia nervosa 2
  </a>
 </li>
</ul>
<p>
 Modified diseases:
</p>
<ul>
 <li>
  Acromesomelic chondrodysplasia, with genital anomalies -&gt;
  <a href="http://www.uniprot.org/diseases/DI-00033">
   Acromesomelic dysplasia, Demirhan type
  </a>
 </li>
 <li>
  Dystonia 11 -&gt;
  <a href="http://www.uniprot.org/diseases/DI-00418">
   Dystonia 11, myoclonic
  </a>
 </li>
 <li>
  Growth retardation developmental delay coarse facies early death -&gt;
  <a href="http://www.uniprot.org/diseases/DI-02561">
   Growth retardation, developmental delay, and facial dysmorphism
  </a>
 </li>
 <li>
  Myopathy, reducing body, X-linked, childhood-onset -&gt;
  <a href="http://www.uniprot.org/diseases/DI-02435">
   Reducing body myopathy, X-linked 1B, with late childhood or adult onset
  </a>
 </li>
 <li>
  Myopathy, reducing body, X-linked, early-onset, severe -&gt;
  <a href="http://www.uniprot.org/diseases/DI-02458">
   Reducing body myopathy, X-linked 1A, severe, with infantile or early childhood onset
  </a>
 </li>
</ul>
<h3 id="Changes_to_the__a_href___docs_ptmlist__controlled_vocabulary_for_PTMs__a_">
 Changes to the
 <a href="http://www.uniprot.org/docs/ptmlist">
  controlled vocabulary for PTMs
 </a>
</h3>
<p>
 New term for the feature key 'Modified residue' ('MOD_RES' in the flat file):
</p>
<ul>
 <li>
  ADP-ribosyl aspartic acid
 </li>
</ul>