<h2 id="Headline">
 Headline
</h2>
<h3 id="From_the_Zika_forest_to_the_Amazon__news_from_a_viral_wanderer">
 From the Zika forest to the Amazon, news from a viral wanderer
</h3>
<p>
 In 2015, a large outbreak in Brazil put the
 <a href="http://viralzone.expasy.org/all%5Fby%5Fspecies/6756.html">
  Zika virus
 </a>
 in the spotlight. Most people who become infected with Zika virus do not become sick and for those who do, the illness is generally mild. However, in some cases, complications can be quite severe. In addition, microcephaly has been reported in some babies born to mothers infected with Zika virus during pregnancy, pointing to the virus as an emerging human pathogen.
</p>
<p>
 Although the Zika virus owes its worldwide infamy to its wandering to the Western hemisphere, it has been circulating in Africa for a long time before. It was first discovered in Uganda, in 1947 in rhesus monkeys living in the Zika Forest (after which it was named), and subsequently in humans in 1952. It is an RNA virus of the flavivirus genus, which also includes dengue, yellow fever and West Nile viruses. Like its relatives, it is transmitted by
 <em>
  Aedes
 </em>
 mosquitoes originally in endemic regions of central Africa. Taking advantage of modern means of transportation, it started spreading, first in Micronesia in 2007, then French Polynesia in 2013, and Brazil and Central America in 2014.
</p>
<p>
 As it has long been considered insignificant, the Zika virus has not been extensively studied and most of our current knowledge has been inferred from other viruses of the same genus. The Zika virus entry into target cells can be triggered by binding to
 <a href="http://www.uniprot.org/uniprot/P30530">
  AXL
 </a>
 and
 <a href="http://www.uniprot.org/uniprot/Q06418">
  TYRO3
 </a>
 . Interestingly, these proteins are also
 <a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=PMID:%2026085147">
  involved in Ebola virus and Lassa virus entry in human cells
 </a>
 . Attachment to the host receptors is followed by internalization by a process called 'apoptotic mimicry' whereby the virus manages to be recognized by the target cell as an apoptotic body. After fusion of the virus membrane with the host endosomal membrane, the RNA genome is released into the cytoplasm. Flaviviruses are remarkable in that their genome encodes a single polyprotein that inserts into the endoplasmic reticulum (ER) membrane forming a complex pattern. This polyprotein is subsequently cleaved into 13 molecules by viral and host peptidases. The non-structural proteins form membrane spherules, presumably to protect the double stranded RNA intermediate of viral replication. The genomic viral RNA is replicated and translated, leading to creation of new Zika virions in the ER. The virions bud by hijacking the host endosomal sorting complex required for transport (ESCRT) system. They are transported to the Golgi apparatus, where further maturation occurs. Eventually fusion-competent virions are released by exocytosis.
</p>
<p>
 As of this release, a
 <a href="http://www.uniprot.org/uniprot/Q32ZE1">
  Zika virus reference proteome
 </a>
 has been manually curated in UniProtKB, where it can be safely visited.
</p>
<p>
 A page dedicated to Zika has also been created in
 <a href="http://viralzone.expasy.org/all%5Fby%5Fspecies/6756.html">
  ViralZone
 </a>
 to offer a global view of how this particular virus functions and provides access to other databases.
</p>
<h3 id="Cross_references_to_EPD">
 Cross-references to EPD
</h3>
<p>
 Cross-references have been added to EPD, the Encyclopedia of Proteome Dynamics, a resource that contains data from multiple, large-scale proteomics experiments aimed at characterising proteome dynamics in both human cells and model organisms.
</p>
<p>
 EPD is available at
 <a href="https://www.peptracker.com/epd/analytics/">
  https://www.peptracker.com/epd/analytics/
 </a>
 .
</p>
<p>
 The format of the explicit links is:
</p>
<table>
 <tr>
  <th>
   Resource abbreviation
  </th>
  <td>
   EPD
  </td>
 </tr>
 <tr>
  <th>
   Resource identifier
  </th>
  <td>
   UniProtKB accession number.
  </td>
 </tr>
</table>
<p>
 Example:
 <a href="http://www.uniprot.org/uniprot/P00451">
  P00451
 </a>
</p>
<p>
 <a href="http://www.uniprot.org/uniprot/?query=database%3Aepd&amp;sort=score">
  Show all entries having a cross-reference to EPD.
 </a>
</p>
<h4 id="Text_format">
 Text format
</h4>
<p>
 Example:
 <a href="http://www.uniprot.org/uniprot/P00451.txt">
  P00451
 </a>
</p>
<pre>DR   EPD; P00451; -.
</pre>
<h4 id="XML_format">
 XML format
</h4>
<p>
 Example:
 <a href="http://www.uniprot.org/uniprot/P00451.xml">
  P00451
 </a>
</p>
<pre>&lt;dbReference type="EPD" id="P00451"/&gt;
</pre>
<h4 id="RDF_format">
 RDF format
</h4>
<p>
 Example:
 <a href="http://www.uniprot.org/uniprot/P00451.ttl">
  P00451
 </a>
</p>
<pre>uniprot:P00451
  rdfs:seeAlso &lt;http://purl.uniprot.org/epd/P00451&gt; .
&lt;http://purl.uniprot.org/epd/P00451&gt;
  rdf:type up:Resource ;
  up:database &lt;http://purl.uniprot.org/database/EPD&gt; .
</pre>
<h3 id="Cross_references_to_TopDownProteomics">
 Cross-references to TopDownProteomics
</h3>
<p>
 Cross-references have been added to TopDownProteomics, a resource from the Consortium for Top Down Proteomics that hosts top down proteomics data presenting validated proteoforms to the scientific community.
</p>
<p>
 TopDownProteomics is available at
 <a href="http://repository.topdownproteomics.org/">
  http://repository.topdownproteomics.org/
 </a>
 .
</p>
<p>
 The format of the explicit links is:
</p>
<table>
 <tr>
  <th>
   Resource abbreviation
  </th>
  <td>
   TopDownProteomics.
  </td>
 </tr>
 <tr>
  <th>
   Resource identifier
  </th>
  <td>
   UniProtKB accession number.
  </td>
 </tr>
</table>
<p>
 Example:
 <a href="http://www.uniprot.org/uniprot/P10599">
  P10599
 </a>
</p>
<p>
 <a href="http://www.uniprot.org/uniprot/?query=database%3Atopdownproteomics&amp;sort=score">
  Show all entries having a cross-reference to TopDownProteomics.
 </a>
</p>
<p>
 Cross-references to TopDownProteomics may be isoform-specific. The general format of isoform-specific cross-references was described in release
 <a href="http://www.uniprot.org/news/2014/03/19/release">
  2014_03
 </a>
 .
</p>
<h4 id="Text_format">
 Text format
</h4>
<p>
 Example:
 <a href="http://www.uniprot.org/uniprot/P10599.txt">
  P10599
 </a>
</p>
<pre>DR   TopDownProteomics; P10599-1; -. [P10599-1]
DR   TopDownProteomics; P10599-2; -. [P10599-2]
</pre>
<h4 id="XML_format">
 XML format
</h4>
<p>
 Example:
 <a href="http://www.uniprot.org/uniprot/P10599.xml">
  P10599
 </a>
</p>
<pre>&lt;dbReference type="TopDownProteomics" id="P10599-1"&gt;
  &lt;molecule id="P10599-1"/&gt;
&lt;/dbReference&gt;
&lt;dbReference type="TopDownProteomics" id="P10599-2"&gt;
  &lt;molecule id="P10599-2"/&gt;
&lt;/dbReference&gt;
</pre>
<h4 id="RDF_format">
 RDF format
</h4>
<p>
 Example:
 <a href="http://www.uniprot.org/uniprot/P10599.ttl">
  P10599
 </a>
</p>
<pre>uniprot:P10599
  rdfs:seeAlso &lt;http://purl.uniprot.org/topdownproteomics/P10599-1&gt; ,
    &lt;http://purl.uniprot.org/topdownproteomics/P10599-2&gt; .

&lt;http://purl.uniprot.org/topdownproteomics/P10599-1&gt;
  rdf:type up:Resource ;
  up:database &lt;http://purl.uniprot.org/database/TopDownProteomics&gt; .
&lt;#_5030303735300040&gt;
  rdf:type rdf:Statement ;
  rdf:subject &lt;P10599&gt; ;
  rdf:predicate rdfs:seeAlso ;
  rdf:object &lt;http://purl.uniprot.org/topdownproteomics/P10599-1&gt; ;
  up:sequence isoform:P00750-1 .
&lt;http://purl.uniprot.org/topdownproteomics/P10599-2&gt;
  rdf:type up:Resource ;
  up:database &lt;http://purl.uniprot.org/database/TopDownProteomics&gt; .
&lt;#_5030303735300040&gt;
  rdf:type rdf:Statement ;
  rdf:subject &lt;P10599&gt; ;
  rdf:predicate rdfs:seeAlso ;
  rdf:object &lt;http://purl.uniprot.org/topdownproteomics/P10599-2&gt; ;
  up:sequence isoform:P00750-2
</pre>
<h3 id="Changes_to_the__a_href___docs_humdisease__controlled_vocabulary_of_human_diseases__a_">
 Changes to the
 <a href="http://www.uniprot.org/docs/humdisease">
  controlled vocabulary of human diseases
 </a>
</h3>
<p>
 New diseases:
</p>
<ul>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-04588">
   Charcot-Marie-Tooth disease 2X
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-04589">
   Charcot-Marie-Tooth disease 2Y
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-04590">
   Charcot-Marie-Tooth disease 2Z
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-04591">
   Charcot-Marie-Tooth disease 4K
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-04584">
   Fundus albipunctatus
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-04585">
   Immunodeficiency 44
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-04586">
   Immunodeficiency 45
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-04587">
   Spondyloepimetaphyseal dsyplasia, Faden-Alkuraya type
  </a>
 </li>
</ul>