<h2 id="Headline">
 Headline
</h2>
<h3 id="Sheep_in_wolves__clothing__human_variant_reannotation_in_UniProtKB_Swiss_Prot_with_ExAC">
 Sheep in wolves' clothing: human variant reannotation in UniProtKB/Swiss-Prot with ExAC
</h3>
<p>
 Annotation of sequence variants has always been an important part of the curation of human proteins in UniProtKB/Swiss-Prot. As of this release, about 76,500 variants are annotated in the knowledgebase. 99% of them are single amino acid polymorphisms (SAPs), the rest are small indels. 38% of the SAPs are associated with a genetic disorder. This high percentage of rare SAPs reflects our strategy to prioritize the annotation of disease-causing and/or functionally characterized variants reported in peer-reviewed scientific literature. Most are annotated as involved in diseases (as disease-causing agents, susceptibility factors or disease modifiers), but for some, the role in the phenotype is not clear, although they have been found in patients and not (yet?) in healthy individuals. These variants are called Variants of Unknown Significance (VUS). In the 'good old days', we were quite confident and we associated SAPs with diseases provided some criteria were met, such as cosegregation of the mutation with the phenotype, and absence of the mutation in a reasonably high number of healthy controls. At that time, 100 control individuals, ethnically matched if possible, seemed acceptable. Those days are gone. Nowadays, these simple criteria have been changed for a real roadmap, based on guidelines developed by
 <a href="https://www.ncbi.nlm.nih.gov/pubmed/25741868">
  Richards et al.
 </a>
 The stumbling block remains the frequency of a given variant in the population in view of the occurrence of the disease. In other words, if a variant is not found in healthy individuals, is it because it is pathogenic, or simply not looked for hard enough? In this context, the high-quality sequence of almost 61,000
 <a href="https://en.wikipedia.org/wiki/Exome">
  exomes
 </a>
 provided by the
 <a href="https://www.ncbi.nlm.nih.gov/pubmed/27535533">
  Exome Aggregation Consortium
 </a>
 (ExAC) is a major achievement.
</p>
<p>
 ExAC aims to aggregate and harmonize exome sequencing data from a wide variety of large-scale sequencing projects, and to make summary data available for the wider scientific community. The sequence of 60,706 exomes from unrelated individuals is currently available on the ExAC
 <a href="http://exac.broadinstitute.org/">
  website
 </a>
 . Surprisingly, each ExAC exome donor harbored on average 54 mutations reported to be disease-causing in
 <a href="http://www.hgmd.cf.ac.uk/ac/index.php">
  HGMD
 </a>
 or
 <a href="https://www.ncbi.nlm.nih.gov/clinvar/">
  ClinVar
 </a>
 . The pathogenicity of most of them (41) could be ruled out due to high allele frequency. Take for instance the gene
 <a href="http://www.uniprot.org/uniprot/Q9UBY8">
  CLN8
 </a>
 . Mutations in this gene have been shown to cause
 <a href="http://www.uniprot.org/uniprot/Q9UBY8#pathology%5Fand%5Fbiotech">
  neuronal ceroid lipofuscinosis-8 (CLN8)
 </a>
 , an autosomal recessive neurodegenerative disorder with an onset age of 2 to 7 years. In view of the clinical synopsis, no 'healthy' adult homozygous for any disease-causing mutation is expected. ExAC observed 93 individuals homozygous for the
 <a href="http://www.uniprot.org/uniprot/Q9UBY8#VAR%5F066927">
  p.Pro229Ala
 </a>
 variant, which
 <a href="https://www.ncbi.nlm.nih.gov/pubmed/21990111">
  had formerly been reported to be pathogenic
 </a>
 . An analogous result was obtained for the variant
 <a href="http://www.uniprot.org/uniprot/P10070#VAR%5F032977">
  p.Met1444Ile
 </a>
 in
 <a href="http://www.uniprot.org/uniprot/P10070">
  GLI2
 </a>
 . This mutation was
 <a href="https://www.ncbi.nlm.nih.gov/pubmed/17096318">
  reported to cause
 </a>
 holoprosencephaly-9 (
 <a href="http://www.uniprot.org/uniprot/P10070#pathology%5Fand%5Fbiotech">
  HPE9
 </a>
 ), an autosomal dominant disorder characterized by a wide phenotypic spectrum of brain developmental defects. Although HPE9 has variable expressivity and incomplete penetrance, the presence of this mutation in 20 homozygous individuals analyzed by ExAC lead to its reclassification as a benign polymorphism.
</p>
<p>
 The ExAC publication has a fruitful impact on our annotation. First, 38 variants (in 36 gene entries) reported in UniProtKB/Swiss-Prot and thought to be pathogenic have been reclassified as either benign polymorphisms or VUS. Second, the ExAC database has become an invaluable tool for curators, helping them to tag
 <a href="http://www.uniprot.org/docs/humsavar">
  human variants
 </a>
 with the appropriate status 'Disease' (disease-associated), 'Polymorphism' (innocuous) or 'Unclassified' (i.e. VUS). Third, we are learning to be more and more cautious when annotating new variants. The result is an increased number of VUS in UniProtKB/Swiss-Prot (currently representing about 20% of the total number of variants identified in patients). Old variants will be progressively confirmed or reclassified as new knowledge becomes available.
</p>
<p>
 As of this release, the
 <a href="http://www.uniprot.org/uniprot/?query=citation:(id:27535533)&amp;sort=score">
  variants updated thanks to ExAC data
 </a>
 are available in UniProtKB/Swiss-Prot.
</p>
<p>
 The UniProt team wishes you a Happy New Year!
</p>
<h3 id="Cross_references_to_SFLD">
 Cross-references to SFLD
</h3>
<p>
 Cross-references have been added to the Structure Function Linkage Database (SFLD), a resource that links evolutionarily related sequences and structures from mechanistically diverse superfamilies of enzymes to their chemical reactions.
</p>
<p>
 SFLD is available at
 <a href="http://sfld.rbvi.ucsf.edu/django/">
  http://sfld.rbvi.ucsf.edu/django/
 </a>
 .
</p>
<p>
 The format of the explicit links is:
</p>
<table>
 <tr>
  <th>
   Resource abbreviation
  </th>
  <td>
   SFLD
  </td>
 </tr>
 <tr>
  <th>
   Resource identifier
  </th>
  <td>
   SFLD identifier
  </td>
 </tr>
 <tr>
  <th>
   Optional information 1
  </th>
  <td>
   SFLD model name
  </td>
 </tr>
 <tr>
  <th>
   Optional information 2
  </th>
  <td>
   Number of hits
  </td>
 </tr>
</table>
<p>
 Example:
 <a href="http://www.uniprot.org/uniprot/P00877#family%5Fand%5Fdomains">
  P00877
 </a>
</p>
<p>
 <a href="http://www.uniprot.org/uniprot/?query=database:sfld&amp;sort=score">
  Show all entries having a cross-reference to SFLD.
 </a>
</p>
<h4 id="Text_format">
 Text format
</h4>
<p>
 Example:
 <a href="http://www.uniprot.org/uniprot/P00877.txt">
  P00877
 </a>
</p>
<pre>DR   SFLD; SFLDS00014; RuBisCO; 1.
</pre>
<h4 id="XML_format">
 XML format
</h4>
<p>
 Example:
 <a href="http://www.uniprot.org/uniprot/P00877.xml">
  P00877
 </a>
</p>
<pre>&lt;dbReference type="SFLD" id="SFLDS00014"&gt;
  &lt;property type="entry name" value="RuBisCO"/&gt;
  &lt;property type="match status" value="1"/&gt;
&lt;/dbReference&gt;
</pre>
<h4 id="RDF_format">
 RDF format
</h4>
<p>
 Example:
 <a href="http://www.uniprot.org/uniprot/P00877.ttl">
  P00877
 </a>
</p>
<pre>uniprot:P00877
  rdfs:seeAlso &lt;http://purl.uniprot.org/sfld/SFLDS00014&gt; .
&lt;http://purl.uniprot.org/sfld/SFLDS00014&gt;
  rdf:type up:Resource ;
  up:database &lt;http://purl.uniprot.org/database/SFLD&gt; ;
  rdfs:comment "RuBisCO" ;
  up:signatureSequenceMatch &lt;http://purl.uniprot.org/isoforms/P00877-1#SFLD_SFLDS00014_match_1&gt; .
</pre>
<h3 id="Changes_to_the__a_href___docs_humdisease__controlled_vocabulary_of_human_diseases__a_">
 Changes to the
 <a href="http://www.uniprot.org/docs/humdisease">
  controlled vocabulary of human diseases
 </a>
</h3>
<p>
 New diseases:
</p>
<ul>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-04842">
   Aortic aneurysm, familial thoracic 10
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-04853">
   Cardiospondylocarpofacial syndrome
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-04854">
   Dyskinesia, seizures, and intellectual developmental disorder
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-04848">
   Ehlers-Danlos syndrome, periodontal type, 1
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-04849">
   Ehlers-Danlos syndrome, periodontal type, 2
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-04843">
   Epileptic encephalopathy, early infantile, 44
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-04844">
   Epileptic encephalopathy, early infantile, 45
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-04845">
   Epileptic encephalopathy, early infantile, 46
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-04846">
   Epileptic encephalopathy, early infantile, 47
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-04852">
   Frontometaphyseal dysplasia 2
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-04850">
   Mental retardation, X-linked, syndromic, Bain type
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-04851">
   Mental retardation, X-linked, syndromic, Borck type
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-04856">
   Short stature, rhizomelic, with microcephaly, micrognathia, and developmental delay
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-04857">
   Sifrim-Hitz-Weiss syndrome
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-04855">
   Sotos syndrome 3
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-04847">
   Spinocerebellar ataxia, autosomal recessive, 24
  </a>
 </li>
</ul>
<h3 id="Changes_to_the__a_href___docs_ptmlist__controlled_vocabulary_for_PTMs__a_">
 Changes to the
 <a href="http://www.uniprot.org/docs/ptmlist">
  controlled vocabulary for PTMs
 </a>
</h3>
<p>
 New terms for the feature key 'Modified residue' ('MOD_RES' in the flat file):
</p>
<ul>
 <li>
  O-UMP-histidine
 </li>
 <li>
  O-UMP-serine
 </li>
 <li>
  O-UMP-threonine
 </li>
</ul>
<h3 id="Changes_to__a_href___docs_keywlist__keywords__a_">
 Changes to
 <a href="http://www.uniprot.org/docs/keywlist">
  keywords
 </a>
</h3>
<p>
 Deleted keyword:
</p>
<ul>
 <li>
  Cyclosporin
 </li>
</ul>
<h2 id="UniRef_news">
 UniRef news
</h2>
<h3 id="Change_of_the_UniRef_FASTA_header">
 Change of the UniRef FASTA header
</h3>
<p>
 We have added the
 <a href="https://www.ncbi.nlm.nih.gov/taxonomy">
  NCBI taxonomy identifier
 </a>
 of the common taxon of a UniRef cluster to the
 <a href="http://www.uniprot.org/help/fasta-headers">
  UniRef FASTA header
 </a>
 , which now has the format:
</p>
<pre>&gt;UniqueIdentifier ClusterName n=Members Tax=TaxonName TaxID=TaxonIdentifier RepID=RepresentativeMember
</pre>
<p>
 Where:
</p>
<ul>
 <li>
  <em>
   UniqueIdentifier
  </em>
  is the primary accession number of the UniRef cluster.
 </li>
 <li>
  <em>
   ClusterName
  </em>
  is the name of the UniRef cluster.
 </li>
 <li>
  <em>
   Members
  </em>
  is the number of UniRef cluster members.
 </li>
 <li>
  <em>
   TaxonName
  </em>
  is the scientific name of the lowest common taxon shared by all UniRef cluster members.
 </li>
 <li>
  <em>
   TaxonIdentifier
  </em>
  is the NCBI taxonomy identifier of the lowest common taxon shared by all UniRef cluster members.
 </li>
 <li>
  <em>
   RepresentativeMember
  </em>
  is the entry name of the representative member of the UniRef cluster.
 </li>
</ul>
<p>
 Example:
 <pre>&gt;UniRef50_Q9K794 Putative AgrB-like protein n=2 Tax=Bacillus TaxID=1386 RepID=AGRB_BACHD
MLERLALTLAHQVKALNAEETESVEVLTFGFTIILHYLFTLLLVLAVGLLHGEIWLFLQI
ALSFTFMRVLTGGAHLDHSIGCTLLSVLFITAISWVPFANNYAWILYGISGGLLIWKYAP
YYEAHQVVHTEHWERRKKRIAYILIVLFIILAMLMSTQGLVLGVLLQGVLLTPIGLKVTR
QLNRFILKGGETNEENS
</pre>
</p>
<p>
 This addresses the issue that scientific taxon names can be ambiguous. Example: "Bacillus" refers to both a genus of
 <a href="http://www.uniprot.org/taxonomy/1386">
  bacteria
 </a>
 as well as a genus of
 <a href="http://www.uniprot.org/taxonomy/55087">
  insects
 </a>
 .
</p>