<h2 id="Headline">
 Headline
</h2>
<h3 id="A_certain_taste_for_light">
 A certain taste for light
</h3>
<p>
 In most organisms, light perception is essential for survival. It not only mediates image-forming vision, but also performs other functions, such as phototaxis and circadian rhythm. Light-sensing function is carried out by photoreceptors, of which only 2 types are known in metazoans: opsins and cryptochromes. They are typically composed of two moieties: a protein and a prosthetic chromophore, the latter is responsible for light absorption. Consequently, photoreceptor denaturation, which targets the protein moiety, does not abolish light absorption, although it shifts absorbance peaks to different wavelengths. Photoreceptor activation by light induces a signaling pathway, called phototransduction, which involves the activation of a G-protein, the modulation of cGMP levels and ultimately a change in the permeability of cyclic nucleotide-gated channels.
</p>
<p>
 It has been long thought that
 <a href="http://www.uniprot.org/taxonomy/6239">
  <em>
   Caenorhabditis elegans
  </em>
 </a>
 , an eyeless, soil-dwelling nematode, could not sense light. This assumption turned out to be erroneous. Not only does
 <a href="https://www.ncbi.nlm.nih.gov/pubmed/18604203,18687026">
  <em>
   C.elegans
  </em>
  sense light, but it vigorously escapes from it
 </a>
 . This behaviour is elicited only in response to blue or shorter wavelengths of light, with maximal responsiveness to UV light. This mechanism may have evolved to protect the animal against prolonged direct sunlight exposure that paralyzes and eventually kills it. Indeed worms appear to spend much of their time above ground, living on small surface-dwelling animals or their carcasses and may therefore be frequently exposed to direct sunlight. From the very beginning of the discovery of phototransduction in
 <em>
  C.elegans
 </em>
 , it was obvious that
 <a href="https://www.ncbi.nlm.nih.gov/pubmed/18687026,20436480">
  the lite-1 gene was involved in this process
 </a>
 , as its heterologous expression in muscle cells was sufficient to confer light responsiveness on these cells that were normally unresponsive. Lite-1 was also shown to act upstream of G proteins, but its exact function remained unclear. Is it a bone fide photoreceptor? Or is it just sensing light-produced chemicals? Like opsins, which are the most common photoreceptor proteins in metazoan photoreceptor cells,
 <a href="http://www.uniprot.org/uniprot/Q17990">
  lite-1
 </a>
 contains a 7-transmembrane domain. However, it does not share any sequence similarity with opsins and its topology is opposite to conventional 7-transmembrane receptors, with its N-terminus located intracellularly and its C-terminus extracellularly. In fact, lite-1 belongs to the insect gustatory receptor family of chemoreceptors, rather than opsin family. To clarify its role,
 <a href="https://www.ncbi.nlm.nih.gov/pubmed/27863243">
  Gong et al.
 </a>
 purified lite-1 and showed that it directly absorbs photons with an efficiency 10 to 100 times that of all known photoreceptors, capturing both UVA and UVB light. Interestingly, absorption of UVA and UVB light can be separated. For instance, mutations at residues Ala-332 and Ser-226 disrupt UVA absorption, but do not affect UVB absorption. In addition, prolonged light illumination, which bleaches conventional photoreceptors, abolishes lite-1 absorption of UVA, but does not affect that of UVB, which appears to be more stable and relatively resistant to photobleaching.
</p>
<p>
 Another remarkable lite-1 feature is that it loses all photoabsorption abilities upon denaturation, suggesting that this activity strictly depends on its conformation and not upon the presence of a chromophore. Mutational analysis pointed at 2 tryptophan residues (
 <a href="http://www.uniprot.org/uniprot/Q17990#pathology%5Fand%5Fbiotech">
  Trp-77 and Trp-328
 </a>
 ) that are required for the absorption of both UVA and UVB light. In order to confirm the importance of these residues, Gong et al. introduced 'Trp-77' by mutagenesis at the equivalent position in a structurally related gustatory receptor, called
 <a href="http://www.uniprot.org/uniprot/O18280">
  gur-3
 </a>
 , which contains 'Trp-328', but is not photosensitive. Amazingly,
 <a href="http://www.uniprot.org/uniprot/O18280#pathology%5Fand%5Fbiotech">
  mutated gur-3
 </a>
 absorbs UVB light with an efficiency of about 30% of that of lite-1. All these observations indicate that lite-1 is a bona fide photoreceptor of a novel type.
</p>
<p>
 <em>
  C.elegans
 </em>
 <a href="http://www.uniprot.org/uniprot/Q17990">
  lite-1
 </a>
 entry has been updated and is publicly available as of this release.
</p>
<h2 id="UniProtKB_news">
 UniProtKB news
</h2>
<h3 id="Extension_of_controlled_vocabulary_for_PTM_to_glycosylation_sites">
 Extension of controlled vocabulary for PTM to glycosylation sites
</h3>
<p>
 Our
 <a href="http://www.uniprot.org/docs/ptmlist">
  controlled vocabulary for post-translational modification
 </a>
 , so far used to standardize the annotation of
 <a href="http://www.uniprot.org/help/mod%5Fres">
  modified residues
 </a>
 ,
 <a href="http://www.uniprot.org/help/lipid">
  lipidation sites
 </a>
 and
 <a href="http://www.uniprot.org/help/crosslnk">
  protein cross-links
 </a>
 , has been extended to include terms for
 <a href="http://www.uniprot.org/help/carbohyd">
  glycosylation sites
 </a>
 .
</p>
<h3 id="Change_of_the_nomenclature_for_glycosylation_sites">
 Change of the nomenclature for glycosylation sites
</h3>
<p>
 We have introduced a change to the nomenclature for glycosylation sites.
</p>
<p>
 We previously described the occurrence of the attachment of a glycan (mono- or polysaccharide) to an amino-acid residue with the following elements:
</p>
<ul>
 <li>
  The type of linkage (C-, N-, O- or S-linked) to the protein
 </li>
 <li>
  The abbreviation of the reducing terminal sugar (shown between parentheses): If three dots '...' follow the abbreviation, this indicates an extension of the carbohydrate chain. Conversely the absence of dots means that a monosaccharide is linked.
 </li>
</ul>
<p>
 To this we have added:
</p>
<ul>
 <li>
  The name of the glycosylated amino acid
 </li>
</ul>
<p>
 The new nomenclature is thus composed of three elements:
</p>
<p>
 <code>
  &lt;linkage type&gt; (&lt;reducing carbohydrate&gt;) &lt;amino acid name&gt;.
 </code>
</p>
<p>
 The valid values have been added to our
 <a href="http://www.uniprot.org/docs/ptmlist">
  controlled vocabulary for post-translational modifications
 </a>
 and applied to all
 <a href="http://www.uniprot.org/manual/carbohyd">
  Glycosylation
 </a>
 annotations.
</p>
<p>
 Example:
 <a href="http://www.uniprot.org/uniprot/Q9HCN3.txt">
  Q9HCN3
 </a>
</p>
<p>
 Previous nomenclature:
</p>
<pre>FT   CARBOHYD    144    144       N-linked (GlcNAc...).
</pre>
<p>
 New nomenclature:
</p>
<pre>FT   CARBOHYD    144    144       N-linked (GlcNAc...) asparagine.
</pre>
<p>
 Note that this information about the type of glycosylation can be complemented by
</p>
<ul>
 <li>
  the name of the modified protein form,
 </li>
 <li>
  information on whether the modification is carried out by a host protein,
 </li>
 <li>
  the frequency of the modification or the relationship with another feature ('partial', 'alternate', 'transient'),
 </li>
 <li>
  evidence attribution
 </li>
</ul>
<p>
 as documented for
 <a href="http://www.uniprot.org/help/mod%5Fres">
  modified residues
 </a>
 .
</p>
<h3 id="Changes_to_the__a_href___docs_ptmlist__controlled_vocabulary_for_PTMs__a_">
 Changes to the
 <a href="http://www.uniprot.org/docs/ptmlist">
  controlled vocabulary for PTMs
 </a>
</h3>
<p>
 New terms for the feature key 'Glycosylation' ('CARBOHYD' in the flat file):
</p>
<ul>
 <li>
  C-linked (Man) hydroxytryptophan
 </li>
 <li>
  C-linked (Man) tryptophan
 </li>
 <li>
  N-linked (DATDGlc) asparagine
 </li>
 <li>
  N-linked (GalNAc) asparagine
 </li>
 <li>
  N-linked (GalNAc...) asparagine
 </li>
 <li>
  N-linked (GalNAc...) (glycosaminoglycan) asparagine
 </li>
 <li>
  N-linked (Glc) arginine
 </li>
 <li>
  N-linked (Glc) asparagine
 </li>
 <li>
  N-linked (Glc) (glycation) histidine
 </li>
 <li>
  N-linked (Glc) (glycation) isoleucine
 </li>
 <li>
  N-linked (Glc) (glycation) lysine
 </li>
 <li>
  N-linked (Glc) (glycation) valine
 </li>
 <li>
  N-linked (Glc...) arginine
 </li>
 <li>
  N-linked (Glc...) asparagine
 </li>
 <li>
  N-linked (GlcNAc) arginine
 </li>
 <li>
  N-linked (GlcNAc) asparagine
 </li>
 <li>
  N-linked (GlcNAc...) arginine
 </li>
 <li>
  N-linked (GlcNAc...) asparagine
 </li>
 <li>
  N-linked (GlcNAc...) (complex) arginine
 </li>
 <li>
  N-linked (GlcNAc...) (complex) asparagine
 </li>
 <li>
  N-linked (GlcNAc...) (high mannose) arginine
 </li>
 <li>
  N-linked (GlcNAc...) (high mannose) asparagine
 </li>
 <li>
  N-linked (GlcNAc...) (hybrid) arginine
 </li>
 <li>
  N-linked (GlcNAc...) (hybrid) asparagine
 </li>
 <li>
  N-linked (GlcNAc...) (keratan sulfate) arginine
 </li>
 <li>
  N-linked (GlcNAc...) (keratan sulfate) asparagine
 </li>
 <li>
  N-linked (GlcNAc...) (paucimannose) arginine
 </li>
 <li>
  N-linked (GlcNAc...) (paucimannose) asparagine
 </li>
 <li>
  N-linked (GlcNAc...) (polylactosaminoglycan) arginine
 </li>
 <li>
  N-linked (GlcNAc...) (polylactosaminoglycan) asparagine
 </li>
 <li>
  N-linked (Hex) arginine
 </li>
 <li>
  N-linked (Hex) asparagine
 </li>
 <li>
  N-linked (Hex) tryptophan
 </li>
 <li>
  N-linked (Hex...) arginine
 </li>
 <li>
  N-linked (Hex...) asparagine
 </li>
 <li>
  N-linked (HexNAc) arginine
 </li>
 <li>
  N-linked (HexNAc) asparagine
 </li>
 <li>
  N-linked (HexNAc...) arginine
 </li>
 <li>
  N-linked (HexNAc...) asparagine
 </li>
 <li>
  N-linked (Lac) (glycation) lysine
 </li>
 <li>
  N-linked (Man) tryptophan
 </li>
 <li>
  O-linked (Ara) hydroxyproline
 </li>
 <li>
  O-linked (Ara...) hydroxyproline
 </li>
 <li>
  O-linked (DADDGlc) serine
 </li>
 <li>
  O-linked (DATDGlc) serine
 </li>
 <li>
  O-linked (GATDGlc) serine
 </li>
 <li>
  O-linked (Fuc) serine
 </li>
 <li>
  O-linked (Fuc) threonine
 </li>
 <li>
  O-linked (Fuc...) serine
 </li>
 <li>
  O-linked (Fuc...) threonine
 </li>
 <li>
  O-linked (FucNAc) serine
 </li>
 <li>
  O-linked (FucNAc...) serine
 </li>
 <li>
  O-linked (Gal) hydroxylysine
 </li>
 <li>
  O-linked (Gal) hydroxyproline
 </li>
 <li>
  O-linked (Gal) serine
 </li>
 <li>
  O-linked (Gal) threonine
 </li>
 <li>
  O-linked (Gal...) hydroxylysine
 </li>
 <li>
  O-linked (Gal...) hydroxyproline
 </li>
 <li>
  O-linked (Gal...) serine
 </li>
 <li>
  O-linked (Gal...) threonine
 </li>
 <li>
  O-linked (GalNAc) serine
 </li>
 <li>
  O-linked (GalNAc...) serine
 </li>
 <li>
  O-linked (GalNAc...) (keratan sulfate) serine
 </li>
 <li>
  O-linked (GalNAc) threonine
 </li>
 <li>
  O-linked (GalNAc...) threonine
 </li>
 <li>
  O-linked (GalNAc...) (keratan sulfate) threonine
 </li>
 <li>
  O-linked (GalNAc) tyrosine
 </li>
 <li>
  O-linked (GalNAc...) tyrosine
 </li>
 <li>
  O-linked (Glc) hydroxylysine
 </li>
 <li>
  O-linked (Glc) serine
 </li>
 <li>
  O-linked (Glc...) serine
 </li>
 <li>
  O-linked (Glc) tyrosine
 </li>
 <li>
  O-linked (Glc...) tyrosine
 </li>
 <li>
  O-linked (GlcA) serine
 </li>
 <li>
  O-linked (GlcNAc) hydroxyproline
 </li>
 <li>
  O-linked (GlcNAc...) hydroxyproline
 </li>
 <li>
  O-linked (GlcNAc) serine
 </li>
 <li>
  O-linked (GlcNAc...) serine
 </li>
 <li>
  O-linked (GlcNAc) threonine
 </li>
 <li>
  O-linked (GlcNAc...) threonine
 </li>
 <li>
  O-linked (GlcNAc) tyrosine
 </li>
 <li>
  O-linked (GlcNAc...) tyrosine
 </li>
 <li>
  O-linked (GlcNAc1P) serine
 </li>
 <li>
  O-linked (GlcNAc6P) serine
 </li>
 <li>
  O-linked (Man) serine
 </li>
 <li>
  O-linked (Man...) serine
 </li>
 <li>
  O-linked (Man...) (keratan sulfate) serine
 </li>
 <li>
  O-linked (Man) threonine
 </li>
 <li>
  O-linked (Man...) threonine
 </li>
 <li>
  O-linked (Man...) (keratan sulfate) threonine
 </li>
 <li>
  O-linked (Man1P) serine
 </li>
 <li>
  O-linked (Man1P...) serine
 </li>
 <li>
  O-linked (Man6P) threonine
 </li>
 <li>
  O-linked (Man6P...) threonine
 </li>
 <li>
  O-linked (Xyl) serine
 </li>
 <li>
  O-linked (Xyl...) serine
 </li>
 <li>
  O-linked (Xyl...) (chondroitin sulfate) serine
 </li>
 <li>
  O-linked (Xyl...) (dermatan sulfate) serine
 </li>
 <li>
  O-linked (Xyl...) (heparan sulfate) serine
 </li>
 <li>
  O-linked (Xyl...) (glycosaminoglycan) serine
 </li>
 <li>
  O-linked (Xyl...) (keratan sulfate) threonine
 </li>
 <li>
  O-linked (Xyl...) (glycosaminoglycan) threonine
 </li>
 <li>
  O-linked (Hex) hydroxylysine
 </li>
 <li>
  O-linked (Hex...) hydroxylysine
 </li>
 <li>
  O-linked (Hex) hydroxyproline
 </li>
 <li>
  O-linked (Hex...) hydroxyproline
 </li>
 <li>
  O-linked (Hex) serine
 </li>
 <li>
  O-linked (Hex...) serine
 </li>
 <li>
  O-linked (Hex) threonine
 </li>
 <li>
  O-linked (Hex...) threonine
 </li>
 <li>
  O-linked (Hex) tyrosine
 </li>
 <li>
  O-linked (Hex...) tyrosine
 </li>
 <li>
  O-linked (HexNAc) hydroxyproline
 </li>
 <li>
  O-linked (HexNAc...) hydroxyproline
 </li>
 <li>
  O-linked (HexNAc) serine
 </li>
 <li>
  O-linked (HexNAc...) serine
 </li>
 <li>
  O-linked (HexNAc) threonine
 </li>
 <li>
  O-linked (HexNAc...) threonine
 </li>
 <li>
  O-linked (HexNAc) tyrosine
 </li>
 <li>
  O-linked (HexNAc...) tyrosine
 </li>
 <li>
  S-linked (Gal) cysteine
 </li>
 <li>
  S-linked (Gal...) cysteine
 </li>
 <li>
  S-linked (Glc) cysteine
 </li>
 <li>
  S-linked (Glc...) cysteine
 </li>
 <li>
  S-linked (GlcNAc) cysteine
 </li>
 <li>
  S-linked (GlcNAc...) cysteine
 </li>
 <li>
  S-linked (Hex) cysteine
 </li>
 <li>
  S-linked (Hex...) cysteine
 </li>
 <li>
  S-linked (HexNAc) cysteine
 </li>
 <li>
  S-linked (HexNAc...) cysteine
 </li>
</ul>
<p>
 New terms for the feature key 'Modified residue' ('MOD_RES' in the flat file):
</p>
<ul>
 <li>
  Cysteine sulfonic acid (-SO3H)
 </li>
</ul>
<h3 id="Changes_to_the__a_href___docs_humdisease__controlled_vocabulary_of_human_diseases__a_">
 Changes to the
 <a href="http://www.uniprot.org/docs/humdisease">
  controlled vocabulary of human diseases
 </a>
</h3>
<p>
 New diseases:
</p>
<ul>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-04955">
   Bosma arhinia microphthalmia syndrome
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-04952">
   Congenital heart defects, dysmorphic facial features, and intellectual developmental disorder
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-04953">
   Congenital heart defects and ectodermal dysplasia
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-04956">
   Coronary artery disease
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-04954">
   Grange syndrome
  </a>
 </li>
</ul>
<p>
 Modified diseases:
</p>
<ul>
 <li>
  Cerebroretinal microangiopathy with calcifications and cysts -&gt;
  <a href="http://www.uniprot.org/diseases/DI-03394">
   Cerebroretinal microangiopathy with calcifications and cysts 1
  </a>
 </li>
 <li>
  Corneal intraepithelial dyskeratosis and ectodermal dysplasia -&gt;
  <a href="http://www.uniprot.org/diseases/DI-03762">
   Palmoplantar carcinoma, multiple self-healing
  </a>
 </li>
 <li>
  Deafness, sensorineural, with hypertrophic cardiomyopathy -&gt;
  <a href="http://www.uniprot.org/diseases/DI-01013">
   Deafness, autosomal dominant 22, with hypertrophic cardiomyopathy
  </a>
 </li>
 <li>
  Familial isolated hyperparathyroidism -&gt;
  <a href="http://www.uniprot.org/diseases/DI-01589">
   Hyperparathyroidism 1
  </a>
 </li>
 <li>
  Hyperparathyroidism-jaw tumor syndrome -&gt;
  <a href="http://www.uniprot.org/diseases/DI-01780">
   Hyperparathyroidism 2 with jaw tumors
  </a>
 </li>
 <li>
  Mental retardation, X-linked, SYP-related -&gt;
  <a href="http://www.uniprot.org/diseases/DI-02522">
   Mental retardation, X-linked 96
  </a>
 </li>
 <li>
  Neurodegeneration with optic atrophy, childhood-onset -&gt;
  <a href="http://www.uniprot.org/diseases/DI-03925">
   Spastic paraplegia 79, autosomal recessive
  </a>
 </li>
</ul>
<p>
 Deleted diseases
</p>
<ul>
 <li>
  Cardiomyopathy, dilated 1T
 </li>
 <li>
  Sarcoidosis early-onset
 </li>
</ul>
<h2 id="UniRef_news">
 UniRef news
</h2>
<h3 id="Addition_of_GO_annotation_to_UniRef90_and_UniRef50_clusters">
 Addition of GO annotation to UniRef90 and UniRef50 clusters
</h3>
<p>
 We have started to compute Gene Ontology (GO) annotations for UniRef90 and UniRef50 clusters: A GO term is assigned to a cluster when it is found in all UniProtKB members that are annotated with this term, or when it is a common ancestor of at least one GO term of each such member.
</p>
<p>
 The UniRef XML format now represents the GO annotations with
 <code>
  property
 </code>
 elements. We have introduced three new types:
 <code>
  "GO Molecular Function"
 </code>
 ,
 <code>
  "GO Biological Process"
 </code>
 ,
 <code>
  "GO Cellular Component"
 </code>
 . The values of these
 <code>
  property
 </code>
 elements are GO identifiers.
</p>
<p>
 Example:
</p>
<pre>&lt;entry id="UniRef50_B0KJL7" updated="2017-03-15"
  &lt;name&gt;Cluster: Animal haem peroxidase&lt;/name&gt;
  ...
  &lt;property type="GO Molecular Function" value="GO:0004601"/&gt;
  &lt;property type="GO Biological Process" value="GO:0006979"/&gt;
  ...
</pre>
<p>
 This change does not affect the XSD, but may nevertheless require code changes.
</p>