<h2 id="Headline">
 Headline
</h2>
<h3 id="Sexual_reproduction__good_ideas_shared_with_viruses">
 Sexual reproduction: good ideas shared with viruses
</h3>
<p>
 Sexual reproduction is a brilliant eukaryotic invention that allows
 <a href="https://www.ncbi.nlm.nih.gov/pubmed/18606131">
  the reassortment of alleles through recombination
 </a>
 . The first step is the formation of haploid male and female gametes that unite to form a new individual. Most gametes unite by membrane fusion, a process mediated by specialized proteins, called fusogens. The study of these proteins is difficult, since they are often scarce. The few identified so far are clade-specific, such as
 <a href="http://www.uniprot.org/uniprot/?query=name:bindin+and+reviewed:yes">
  bindin
 </a>
 in echinoderms or
 <a href="http://www.uniprot.org/uniprot/?query=gene:IZUMO1+and+reviewed:yes">
  izumo
 </a>
 in mammals, suggesting that each clade has evolved its own fusion strategy. This is at least what was thought until
 <a href="https://www.ncbi.nlm.nih.gov/pubmed/18367645,19888453">
  the discovery of hapless-2
 </a>
 (HAP2[1]), also called generative cell specific-1 (GCS1).
</p>
<p>
 Hapless-2 is a single-span transmembrane protein located at the gamete cell surface, typically at mating structures. It is essential for gamete fusion in the green alga
 <em>
  Chlamydomonas reinhardtii
 </em>
 , but also in other plants, including
 <em>
  Arabidopsis thaliana
 </em>
 , and
 <em>
  Lilium longiflorum
 </em>
 and in protozoans, such as
 <em>
  Plasmodium berghei
 </em>
 or
 <em>
  Tetrahymena thermophila
 </em>
 . A thorough eukaryotic genome examination reveals the existence of this gene in many major eukaryotic taxa, from slime molds to the honey bee. It is however not present in fungi, nor in most animals, including humans. The wide evolutionary distribution of hapless-2 suggests it was present in the last eukaryotic common ancestor and lost in some clades later on. Disruption of hapless-2 blocks gamete fusion, but not adhesion to gametes of the opposite mating type (or sex), suggesting that gamete adhesion relies on proteins that are species-specific, but that fusion itself is mediated by an ancestral common gene product.
</p>
<p>
 Earlier this year, the 3D-structure of
 <a href="https://www.ncbi.nlm.nih.gov/pubmed/28235200">
  <em>
   Chlamydomonas reinhardtii
  </em>
  hapless-2 was unraveled
 </a>
 . The secondary and tertiary structures of the ectodomain are almost identical to viral class II proteins, such as the envelope protein E of
 <a href="http://viralzone.expasy.org/24?outline=all%5Fby%5Fspecies">
  flaviviruses
 </a>
 , with which hapless-2 shares very low identity at the amino acid level, and which are also involved in membrane fusion. FÃ©dry et al. hypothesize that these fusion proteins most certainly derived from a common ancestor, whose gene has likely been transferred via horizontal exchange.
</p>
<p>
 Like the flavivirus class II proteins, the hapless-2 ectodomain trimerizes concomitantly with insertion into the membrane of the partner gamete. The trigger for trimerization of hapless-2 is not yet known, although acidification, which drives trimerization of flavivirus class II proteins in late endosomes, is not required.
</p>
<p>
 Information gained from the 3D structure of hapless-2 may help in the development of
 <a href="https://www.ncbi.nlm.nih.gov/pubmed/19596419,24042109">
  transmission-blocking vaccines
 </a>
 (TBVs), a new strategy to fight malaria (and other protozoan diseases). Successful transmission of
 <em>
  Plasmodium
 </em>
 from humans to mosquitoes relies on hapless-2-dependent fusion of the parasite gametes and fertilization, which occurs rapidly after ingestion by the mosquito. If TBVs could be designed to induce anti-hapless-2 antibodies in human hosts, these would be ingested by
 <em>
  Anopheles
 </em>
 mosquitoes along with blood
 <em>
  Plasmodium
 </em>
 gametocytes. The initial gamete fusion step could be prevented and the deadly cycle of transmission blocked. This approach has already been tested in model animals and, although the preliminary results look promising, they are not yet sufficient for clinical development. The identification of new peptides, that are both functionally crucial and immunogenic, may prove very helpful in the design of efficient anti-malaria TBVs.
</p>
<p>
 As of this release,
 <a href="http://www.uniprot.org/uniprot/?query=name:%22hapless-2%22+and+reviewed:yes">
  hapless-2 UniProtKB/Swiss-Prot entries
 </a>
 have been created and are publicly available.
</p>
<p>
 fn1. The acronym HAP2 is somewhat unfortunate, since this protein has nothing to do with the yeast HAP2 transcription factor. These are the mysterious ways of nomenclature, which sometimes may be quite confusing...
</p>
<h2 id="UniProtKB_news">
 UniProtKB news
</h2>
<h3 id="Modification_of_cross_references_to_PATRIC">
 Modification of cross-references to PATRIC
</h3>
<p>
 We have modified our cross-references to the
 <a href="https://www.patricbrc.org">
  PATRIC
 </a>
 database in order to reflect the new PATRIC primary identifier scheme. The earlier identifier scheme used simple numeric ids, e.g.
 <pre>32117610</pre>
 which were replaced by more informative primary identifiers such as
 <pre>fig|1427269.3.peg.1028.</pre>
</p>
<h4 id="Text_format">
 Text format
</h4>
<p>
 Example:
 <a href="http://www.uniprot.org/uniprot/Q9ZNI1.txt">
  Q9ZNI1
 </a>
</p>
<p>
 Previous format:
</p>
<pre>DR   PATRIC; 19579917; VBIStaAur99865_1117.
</pre>
<p>
 New format:
</p>
<pre>DR   PATRIC; fig|93061.5.peg.1117; -.
</pre>
<h4 id="XML_format">
 XML format
</h4>
<p>
 Example:
 <a href="http://www.uniprot.org/uniprot/Q9ZNI1.xml">
  Q9ZNI1
 </a>
</p>
<p>
 Previous format:
 <pre>&lt;dbReference type="PATRIC" id="19579917"&gt;
  &lt;property type="gene designation" value="VBIStaAur99865_1117"/&gt;
&lt;/dbReference&gt;
</pre>
</p>
<p>
 New format:
</p>
<pre>&lt;dbReference type="PATRIC" id="fig|93061.5.peg.1117"/&gt;
</pre>
<h4 id="RDF_format">
 RDF format
</h4>
<p>
 Example:
 <a href="http://www.uniprot.org/uniprot/Q9ZNI1.ttl">
  Q9ZNI1
 </a>
</p>
<p>
 Previous format:
</p>
<pre>uniprot:Q9ZNI1
  rdfs:seeAlso &lt;http://purl.uniprot.org/patric/19579917&gt; .
&lt;http://purl.uniprot.org/patric/19579917&gt;
  rdf:type up:Resource ;
  up:database &lt;http://purl.uniprot.org/database/PATRIC&gt; ;
  rdfs:comment "VBIStaAur99865_1117" .
</pre>
<p>
 New format:
</p>
<pre>uniprot:Q9ZNI1
  rdfs:seeAlso &lt;http://purl.uniprot.org/patric/fig%7C93061.5.peg.1117&gt; .
&lt;http://purl.uniprot.org/patric/fig%7C93061.5.peg.1117&gt;
  rdf:type up:Resource ;
  up:database &lt;http://purl.uniprot.org/database/PATRIC&gt; .
</pre>
<h3 id="New_file_linking_deleted_entries_to_their_subsequently_reinstated_versions">
 New file linking deleted entries to their subsequently reinstated versions
</h3>
<p>
 Since release 2015_04, we are applying at each release a procedure to
 <a href="http://www.uniprot.org/help/proteome%5Fredundancy">
  identify highly redundant proteomes
 </a>
 within selected species groups using a combination of manual and automatic methods. This procedure prevents the creation of UniProtKB/TrEMBL entries from these redundant proteomes, but also means that a huge number of previously existing entries had to be
 <a href="http://www.uniprot.org/news/2015/04/01/release">
  deleted from UniProtKB
 </a>
 when the procedure was put in place.
</p>
<p>
 It may happen that proteomes that were identified as redundant are later reinstated as non-redundant, e.g. a proteome for a strain used as a model by a significant community or with proteins that have been crystallized. In the past, it has also happened on rare occasions that entries were deleted but later reinstated for other reasons. In such cases, the UniProtKB entries are created anew, with new accession numbers.
</p>
<p>
 To help users to link deleted to subsequently reinstated entries, we are introducing a file that maps old to new accession numbers via their
 <a href="http://www.uniprot.org/faq/sequence%5Forigin">
  protein_ids
 </a>
 . This file is available (in compressed format) by FTP at
</p>
<p>
 <a href="ftp://ftp.uniprot.org/pub/databases/uniprot/current%5Frelease/knowledgebase/complete/docs/reinstated%5Fmap.txt.gz">
  ftp://ftp.uniprot.org/pub/databases/uniprot/current_release/knowledgebase/complete/docs/reinstated_map.txt.gz
 </a>
</p>
<p>
 This mapping will also be used to make queries for obsolete identifiers on the UniProt website more meaningful.
</p>
<h3 id="Changes_to_the__a_href___docs_humdisease__controlled_vocabulary_of_human_diseases__a_">
 Changes to the
 <a href="http://www.uniprot.org/docs/humdisease">
  controlled vocabulary of human diseases
 </a>
</h3>
<p>
 New diseases:
</p>
<ul>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-04972">
   Anauxetic dysplasia 2
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-04967">
   Autoinflammation with arthritis and dyskeratosis
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-04965">
   Avascular necrosis of the femoral head, primary 2
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-04960">
   Bardet-Biedl syndrome 21
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-04977">
   Basan syndrome
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-04971">
   Congenital disorder of glycosylation 2Q
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-04974">
   Cutis laxa, autosomal recessive, 2C
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-04975">
   Cutis laxa, autosomal recessive, 2D
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-04958">
   Diamond-Blackfan anemia 16
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-04959">
   Diamond-Blackfan anemia 17
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-04968">
   Ectodermal dysplasia 13, hair/tooth type
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-04961">
   Epileptic encephalopathy, early infantile, 53
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-04962">
   Epileptic encephalopathy, early infantile, 54
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-04966">
   Hyperphenylalaninemia, mild, non-BH4-deficient
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-04963">
   Myasthenic syndrome, congenital, 22
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-04969">
   Neurodevelopmental disorder with epilepsy, cataracts, feeding difficulties, and delayed brain myelination
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-04964">
   Peroxisome biogenesis disorder 10B
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-04973">
   Pseudo-TORCH syndrome 2
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-04970">
   Sclerosing cholangitis, neonatal
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-04957">
   Short-rib thoracic dysplasia 17 with or without polydactyly
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-04976">
   Spastic paraplegia 23
  </a>
 </li>
</ul>
<p>
 Modified diseases:
</p>
<ul>
 <li>
  Avascular necrosis of the femoral head, primary -&gt;
  <a href="http://www.uniprot.org/diseases/DI-02197">
   Avascular necrosis of femoral head, primary, 1
  </a>
 </li>
 <li>
  Rippling muscle disease -&gt;
  <a href="http://www.uniprot.org/diseases/DI-02270">
   Rippling muscle disease 2
  </a>
 </li>
 <li>
  Mental retardation, X-linked, ZNF711-related -&gt;
  <a href="http://www.uniprot.org/diseases/DI-02523">
   Mental retardation, X-linked 97
  </a>
 </li>
 <li>
  Band-like calcification with simplified gyration and polymicrogyria -&gt;
  <a href="http://www.uniprot.org/diseases/DI-02925">
   Pseudo-TORCH syndrome 1
  </a>
 </li>
</ul>
<h3 id="Changes_to_the__a_href___docs_ptmlist__controlled_vocabulary_for_PTMs__a_">
 Changes to the
 <a href="http://www.uniprot.org/docs/ptmlist">
  controlled vocabulary for PTMs
 </a>
</h3>
<p>
 New term for the feature key 'Cross-link' ('CROSSLNK' in the flat file):
</p>
<ul>
 <li>
  Cyclopeptide (Glu-Asn)
 </li>
</ul>
<p>
 New term for the feature key 'Modified residue' ('MOD_RES' in the flat file):
</p>
<ul>
 <li>
  S-methylmethionine
 </li>
</ul>
<p>
 Deleted term
</p>
<ul>
 <li>
  N-acetylated lysine
 </li>
</ul>
<h3 id="Changes_in__a_href___docs_subcell__subcellular_location_controlled_vocabulary__a_">
 Changes in
 <a href="http://www.uniprot.org/docs/subcell">
  subcellular location controlled vocabulary
 </a>
</h3>
<p>
 New subcellular location:
</p>
<ul>
 <li>
  <a href="http://www.uniprot.org/locations/SL-0490">
   Midbody ring
  </a>
 </li>
</ul>
<h3 id="Changes_to__a_href___docs_keywlist__keywords__a_">
 Changes to
 <a href="http://www.uniprot.org/docs/keywlist">
  keywords
 </a>
</h3>
<p>
 New keyword:
</p>
<ul>
 <li>
  <a href="http://www.uniprot.org/keywords/KW-1274">
   Primary mitochondrial disease
  </a>
 </li>
</ul>