<h2 id="Headline">
 Headline
</h2>
<h3 id="Escaping_friendly_fire">
 Escaping friendly fire
</h3>
<p>
 During the first hours of an infection, our safety relies almost entirely on the innate immune system, and predominantly on neutrophils. The encounter between neutrophils and invading microbes leads to neutrophil activation and to the engulfment of pathogens into intracellular phagosomes, where exposure to high concentrations of reactive oxygen species (ROS) and antimicrobial peptides eventually kill them. Neutrophils defend us not only in life but also in death, when they release chromatin and granule proteins that together form extracellular fibers, called
 <a href="https://www.ncbi.nlm.nih.gov/pubmed/15001782">
  'neutrophil extracellular traps' or NETs, which catch and prevent the spread of microorganisms
 </a>
 . NETs are covered with antimicrobial compounds, such as cathelicidin peptides, as well as
 <a href="https://www.ncbi.nlm.nih.gov/pubmed/13598820">
  histones
 </a>
 , which can also effectively neutralize intruders. This process is so efficient that
 <a href="https://www.ncbi.nlm.nih.gov/pubmed/16488874,24233725">
  extracellular DNases able to catalyze NET disruption serve as virulence factors
 </a>
 in several pathogenic bacteria, such as in group A
 <em>
  Streptococcus
 </em>
 .
</p>
<p>
 NETs are a double-edged sword and have to be regulated very tightly. Indeed, free extracellular DNA is a potent trigger of autoimmune response, such as that encountered in
 <a href="https://www.omim.org/entry/152700">
  systemic lupus erythematosus (SLE)
 </a>
 that is characterized by circulating anti-DNA antibodies. NETs can also
 <a href="https://www.ncbi.nlm.nih.gov/pubmed/22451716,29191910">
  initiate vascular occlusion
 </a>
 in a fibrin-independent manner. In other words, NETs are not an innocuous therapy in the middle/long term and the host has to get rid of them quickly. Timely removal of NET chromatin by
 <a href="https://www.ncbi.nlm.nih.gov/pubmed/20439745,27293190">
  DNases DNASE1 and DNASE1L3 has been shown to play a crucial role in the prevention of autoimmunity
 </a>
 . However, it was not known until recently what mechanism was involved in NET clearance under inflammatory conditions. This issue was addressed by
 <a href="https://www.ncbi.nlm.nih.gov/pubmed/29191910">
  Jimenez-Alcazar and colleagues
 </a>
 . They created knockout mice lacking both DNASE1 and DNASE1L3. Mutant animals were treated with granulocyte colony-stimulating factor (G-CSF) to induce chronic neutrophilia, a condition mimicking acute inflammation. While wild-type mice showed no sign of distress, all double knockout animals exhibited features of infection-induced thrombotic microangiopathies (TMAs) and died within 6 days. This phenotype could be reversed by the reintroduction of DNASE1 or DNASE1L3, but not by an anti-thrombotic treatment, further supporting the idea that NETs can clog vessels by themselves. TMAs are a well-known complication encountered by patients suffering from systemic bacterial infections. Analysis of lungs from patients with acute respiratory distress syndrome and/or sepsis revealed numerous NET-derived clots in their blood vessels. It is too early yet to propose DNase treatment for TMA patients, but at least it opens new therapeutic perspectives.
</p>
<p>
 As of this release,
 <a href="http://www.uniprot.org/uniprot/?query=(gene:dnase1l3+OR+gene:dnase1)+AND+reviewed:yes+AND+taxonomy:mammalia">
  murine DNASE1 and DNASE1L3 and their orthologs
 </a>
 in other mammalian species have been updated and are now publicly available.
</p>
<h2 id="UniProtKB_news">
 UniProtKB news
</h2>
<h3 id="UniProtKB_FASTA_headers__Addition_of_NCBI_taxonomy_identifier">
 UniProtKB FASTA headers: Addition of NCBI taxonomy identifier
</h3>
<p>
 In order to avoid ambiguities and simplify parsing, we have added the NCBI taxonomy identifier to UniProtKB
 <a href="http://www.uniprot.org/help/fasta-headers">
  FASTA headers
 </a>
 .
</p>
<p>
 Previous format:
</p>
<pre>&gt;db|UniqueIdentifier|EntryName ProteinName OS=OrganismName [GN=GeneName ]PE=ProteinExistence SV=SequenceVersion
</pre>
<p>
 New format:
</p>
<pre>&gt;db|UniqueIdentifier|EntryName ProteinName OS=OrganismName OX=OrganismIdentifier [GN=GeneName ]PE=ProteinExistence SV=SequenceVersion
</pre>
<p>
 Where:
</p>
<ul>
 <li>
  <em>
   db
  </em>
  is 'sp' for UniProtKB/Swiss-Prot and 'tr' for UniProtKB/TrEMBL.
 </li>
 <li>
  <em>
   UniqueIdentifier
  </em>
  is the primary
  <a href="http://www.uniprot.org/help/accession%5Fnumbers">
   accession number
  </a>
  of the UniProtKB entry.
 </li>
 <li>
  <em>
   EntryName
  </em>
  is the
  <a href="http://www.uniprot.org/help/entry%5Fname">
   entry name
  </a>
  of the UniProtKB entry.
 </li>
 <li>
  <em>
   ProteinName
  </em>
  is the
  <a href="http://www.uniprot.org/help/protein%5Fnames">
   recommended name
  </a>
  of the UniProtKB entry as annotated in the
  <code>
   RecName
  </code>
  field. For UniProtKB/TrEMBL entries without a
  <code>
   RecName
  </code>
  field, the
  <code>
   SubName
  </code>
  field is used. In case of multiple
  <code>
   SubNames
  </code>
  , the first one is used. The 'precursor' attribute is excluded, 'Fragment' is included with the name if applicable.
 </li>
 <li>
  <em>
   OrganismName
  </em>
  is the
  <a href="http://www.uniprot.org/help/organism-name">
   scientific name of the organism
  </a>
  of the UniProtKB entry.
 </li>
 <li>
  <strong>
   <em>
    OrganismIdentifier
   </em>
   is the
   <a href="http://www.uniprot.org/help/taxonomic%5Fidentifier">
    unique identifier of the source organism, assigned by the NCBI
   </a>
   .
  </strong>
 </li>
 <li>
  <em>
   GeneName
  </em>
  is the first
  <a href="http://www.uniprot.org/help/gene%5Fname">
   gene name
  </a>
  of the UniProtKB entry. If there is no gene name,
  <code>
   OrderedLocusName
  </code>
  or
  <code>
   ORFname
  </code>
  , the
  <code>
   GN
  </code>
  field is not listed.
 </li>
 <li>
  <em>
   ProteinExistence
  </em>
  is the numerical value describing the
  <a href="http://www.uniprot.org/help/protein%5Fexistence">
   evidence for the existence
  </a>
  of the protein.
 </li>
 <li>
  <em>
   SequenceVersion
  </em>
  is the
  <a href="http://www.uniprot.org/help/entry%5Fhistory">
   version number
  </a>
  of the sequence.
 </li>
</ul>
<p>
 Examples:
</p>
<pre>&gt;sp|Q8I6R7|ACN2_ACAGO Acanthoscurrin-2 (Fragment) OS=Acanthoscurria gomesiana OX=115339 GN=acantho2 PE=1 SV=1
&gt;sp|P27748|ACOX_CUPNH Acetoin catabolism protein X OS=Cupriavidus necator (strain ATCC 17699 / H16 / DSM 428 / Stanier 337) OX=381666 GN=acoX PE=4 SV=2
&gt;sp|P04224|HA22_MOUSE H-2 class II histocompatibility antigen, E-K alpha chain OS=Mus musculus OX=10090 PE=1 SV=1

&gt;tr|Q3SA23|Q3SA23_9HIV1 Protein Nef (Fragment) OS=Human immunodeficiency virus 1  OX=11676 GN=nef PE=3 SV=1
&gt;tr|Q8N2H2|Q8N2H2_HUMAN cDNA FLJ90785 fis, clone THYRO1001457, moderately similar to H.sapiens protein kinase C mu OS=Homo sapiens OX=9606 PE=2 SV=1
</pre>
<p>
 The same modification has been applied to FASTA headers of
 <strong>
  alternative isoforms in UniProtKB/Swiss-Prot
 </strong>
 ), where the new format is:
</p>
<pre>&gt;sp|IsoID|EntryName Isoform IsoformName of ProteinName OS=OrganismName OX=OrganismIdentifier[ GN=GeneName]
</pre>
<p>
 Example:
</p>
<pre>&gt;sp|Q4R572-2|1433B_MACFA Isoform Short of 14-3-3 protein beta/alpha OS=Macaca fascicularis OX=9541 GN=YWHAB
</pre>
<h3 id="Cross_references_to_CarbonylDB">
 Cross-references to CarbonylDB
</h3>
<p>
 Cross-references have been added to the CarbonylDB database, a resource of protein carbonylation sites.
</p>
<p>
 CarbonylDB is available at
 <a href="http://digbio.missouri.edu/CarbonylDB/">
  http://digbio.missouri.edu/CarbonylDB/
 </a>
 .
</p>
<p>
 The format of the explicit links is:
</p>
<table>
 <tr>
  <th>
   Resource abbreviation
  </th>
  <td>
   CarbonylDB
  </td>
 </tr>
 <tr>
  <th>
   Resource identifier
  </th>
  <td>
   UniProtKB accession number
  </td>
 </tr>
</table>
<p>
 Example:
 <a href="http://www.uniprot.org/uniprot/P02768">
  P02768
 </a>
</p>
<p>
 <a href="http://www.uniprot.org/uniprot/?query=database:carbonyldb&amp;sort=score">
  Show all entries having a cross-reference to CarbonylDB.
 </a>
</p>
<h4 id="Text_format">
 Text format
</h4>
<p>
 Example:
 <a href="http://www.uniprot.org/uniprot/P02768.txt">
  P02768
 </a>
</p>
<pre>DR   CarbonylDB; P02768; -.
</pre>
<h4 id="XML_format">
 XML format
</h4>
<p>
 Example:
 <a href="http://www.uniprot.org/uniprot/P02768.xml">
  P02768
 </a>
</p>
<pre>&lt;dbReference type="CarbonylDB" id="P02768"/&gt;
</pre>
<h4 id="RDF_format">
 RDF format
</h4>
<p>
 Example:
 <a href="http://www.uniprot.org/uniprot/P02768.ttl">
  P02768
 </a>
</p>
<pre>uniprot:P02768
  rdfs:seeAlso &lt;http://purl.uniprot.org/carbonyldb/P02768&gt; .
&lt;http://purl.uniprot.org/carbonyldb/P02768&gt;
  rdf:type up:Resource ;
  up:database &lt;http://purl.uniprot.org/database/CarbonylDB&gt; .
</pre>
<h3 id="Changes_to_the__a_href___docs_humdisease__controlled_vocabulary_of_human_diseases__a_">
 Changes to the
 <a href="http://www.uniprot.org/docs/humdisease">
  controlled vocabulary of human diseases
 </a>
</h3>
<p>
 New diseases:
</p>
<ul>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-05168">
   Choroidal dystrophy, central areolar, 1
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-05158">
   Coffin-Siris syndrome 6
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-05147">
   Combined immunodeficiency and megaloblastic anemia with or without hyperhomocysteinemia
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-05146">
   Deafness, autosomal dominant 34, with or without inflammation
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-05166">
   Ehlers-Danlos syndrome, arthrochalasia type, 2
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-05145">
   Epileptic encephalopathy, early infantile, 57
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-05138">
   Erythrokeratodermia variabilis et progressiva 5
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-05159">
   Geleophysic dysplasia 3
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-05165">
   Glucocorticoid deficiency 5
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-05160">
   Glycosylphosphatidylinositol biosynthesis defect 15
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-05164">
   Glycosylphosphatidylinositol biosynthesis defect 16
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-05140">
   Immunodeficiency, common variable, 14
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-05136">
   Joubert syndrome 31
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-05134">
   Joubert syndrome 32
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-05135">
   Joubert syndrome 33
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-05148">
   Joubert syndrome 34
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-05128">
   Fanconi anemia, complementation group W
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-05142">
   Kleefstra syndrome 2
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-05125">
   Leukodystrophy, progressive, early childhood-onset
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-05131">
   Mental retardation, autosomal dominant 48
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-05132">
   Mental retardation, autosomal dominant 49
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-05151">
   Mental retardation, autosomal dominant 50
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-05152">
   Mental retardation, autosomal dominant 51
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-05153">
   Mental retardation, autosomal dominant 52
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-05154">
   Mental retardation, autosomal dominant 53
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-05155">
   Mental retardation, autosomal dominant 54
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-05133">
   Mental retardation, autosomal recessive 61
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-05156">
   Mental retardation, X-linked, syndromic, Houge type
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-05157">
   Microcephaly 19, primary, autosomal recessive
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-05139">
   Myopathy, centronuclear, 6, with fiber-type disproportion
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-05150">
   Myopia 26, X-linked, female-limited
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-05129">
   Nephrotic syndrome 16
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-05163">
   Neurodevelopmental disorder with ataxic gait, absent speech, and decreased cortical white matter
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-05137">
   Neurodevelopmental disorder with dysmorphic facies and distal limb anomalies
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-05161">
   Neurodevelopmental disorder with microcephaly, seizures, and cortical atrophy
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-05162">
   Neurodevelopmental disorder with severe motor impairment and absent language
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-05126">
   Optic atrophy 5
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-05149">
   Renal hypodysplasia/aplasia 3
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-05130">
   Retinitis pigmentosa 80
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-05141">
   Short stature, hearing loss, retinitis pigmentosa, and distinctive facies
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-05143">
   Spinocerebellar ataxia 45
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-05144">
   Spinocerebellar ataxia 46
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-05167">
   Spondylometaphyseal dysplasia, corner fracture type
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-05127">
   Uruguay faciocardiomusculoskeletal syndrome
  </a>
 </li>
</ul>
<p>
 Modified diseases:
</p>
<ul>
 <li>
  Brain malformations and urinary tract defects -&gt;
  <a href="http://www.uniprot.org/diseases/DI-04979">
   Brain malformations with or without urinary tract defects
  </a>
 </li>
 <li>
  Cytosolic phosphoenolpyruvate carboxykinase deficiency -&gt;
  <a href="http://www.uniprot.org/diseases/DI-01470">
   Phosphoenolpyruvate carboxykinase deficiency, cytosolic
  </a>
 </li>
 <li>
  Galloway-Mowat syndrome -&gt;
  <a href="http://www.uniprot.org/diseases/DI-04306">
   Galloway-Mowat syndrome 1
  </a>
 </li>
 <li>
  Kleefstra syndrome -&gt;
  <a href="http://www.uniprot.org/diseases/DI-01348">
   Kleefstra syndrome 1
  </a>
 </li>
</ul>
<h3 id="Changes_to_the__a_href___docs_ptmlist__controlled_vocabulary_for_PTMs__a_">
 Changes to the
 <a href="http://www.uniprot.org/docs/ptmlist">
  controlled vocabulary for PTMs
 </a>
</h3>
<p>
 New term for the feature key 'Modified residue' ('MOD_RES' in the flat file):
</p>
<ul>
 <li>
  O-AMP-serine
 </li>
</ul>
<h2 id="RDF_news">
 RDF news
</h2>
<h3 id="Change_of_URIs_for_OrthoDB">
 Change of URIs for OrthoDB
</h3>
<p>
 For historic reasons, UniProt had to generate URIs to cross-reference databases that did not have an RDF representation. Our policy is to replace these by the URIs generated by the cross-referenced database once it starts to distribute an RDF representation of its data.
</p>
<p>
 The URIs for the
 <a href="http://www.orthodb.org/">
  OrthoDB
 </a>
 database have therefore been updated from:
</p>
<pre>http://purl.uniprot.org/orthodb/&lt;ID&gt;
</pre>
<p>
 to:
</p>
<pre>http://purl.orthodb.org/odbgroup/&lt;ID&gt;
</pre>
<p>
 If required for backward compatibility, you can use the following query to add the old URIs:
</p>
<pre>PREFIX owl:&lt;http://www.w3.org/2002/07/owl#&gt;
PREFIX up:&lt;http://purl.uniprot.org/core/&gt;
INSERT
{
   ?protein rdfs:seeAlso ?old .
   ?old owl:sameAs ?new .
   ?old up:database &lt;http://purl.uniprot.org/database/orthodb&gt; .
}
WHERE
{
   ?protein rdfs:seeAlso ?new .
   ?new up:database &lt;http://purl.uniprot.org/database/OrthoDB&gt; .
   BIND(iri(concat('http://purl.uniprot.org/orthodb/', substr(str(?new),31))) AS ?old)
}
</pre>
<p>
 The dereferencing of existing http://purl.uniprot.org/orthodb/&lt;ID&gt; URIs will be maintained.
</p>