<h2 id="Headline">
 Headline
</h2>
<h3 id="Thicker_than_water">
 Thicker than water
</h3>
<p>
 We know about blood types and their incompatibility; transfusing someone who is O- with AB+ blood can be lethal. The
 <a href="http://www.uniprot.org/uniprot/P16442">
  ABO
 </a>
 alleles present on chromosome 9 decide our blood type. The A and B antigens are a set of red blood cell surface carbohydrates ending in α-1,3-linked N-acetylgalactosamine and α-1,3-linked galactose respectively, while type O blood has neither of these cell surface sugars. Sequence variations in the ABO gene determine if the encoded protein has α-1,3-N-acetylgalactosaminyltransferase activity and makes type A blood, or if it has α-1,3-galactosyltransferase activity and makes type B blood. When both alleles are present, we make type AB blood. Deletion of a single G nucleotide in the ABO gene leads to a truncated inactive product and type O blood, which has the non-modified H antigen.
</p>
<p>
 To improve the usability of blood, people have tried for years to find a way to enzymatically convert A or B blood to type O; it seems an obvious way to increase the supply of universal donor (which would still require Rhesus matching). While
 <a href="https://www.ncbi.nlm.nih.gov/pubmed/6274021,11099655,15618227,17401360">
  such enzymes have been found
 </a>
 , they are not yet ideal, as they either work at high concentration or have very specific buffer requirements, not met by blood.
</p>
<p>
 By screening human fecal metagenomic libraries,
 <a href="https://www.ncbi.nlm.nih.gov/pubmed/31182795">
  Rahfeld et al.
 </a>
 have isolated a pair of enzymes from the obligate gut anaerobe
 <em>
  Flavonifractor plautii
 </em>
 that efficiently converts the A to H antigen (type O). The first enzyme (A type blood N-acetyl-alpha-D-galactosamine deacetylase,
 <a href="http://www.uniprot.org/uniprot/P0DTR4">
  ADAC
 </a>
 ) deacylates all A antigen subtypes tested (and there are many), while the second enzyme (A type blood alpha-D-galactosamine galactosaminidase,
 <a href="http://www.uniprot.org/uniprot/P0DTR5">
  AGAL
 </a>
 ) removes the residual galactosamine moiety. This reaction can occur on red blood cells and in blood, as opposed to a buffer system, and at low enzyme concentration, and thus shows promise for uses in blood production. Further testing is underway, and we still need a way to remove the B antigen, but this could well help increase the flexibility of our blood supply. It still won’t solve the world shortage of blood, only more donors can do that...
</p>
<p>
 As of this release,
 <a href="http://www.uniprot.org/uniprot/?query=id:P0DTR4+OR+id:P0DTR5">
  ADAC and AGAL
 </a>
 have been annotated and are available in UniProtKB/Swiss-Prot.
</p>
<h2 id="UniProtKB_news">
 UniProtKB news
</h2>
<h3 id="Change_of_FT_and_CC_sections_in_UniProtKB_text_format">
 Change of FT and CC sections in UniProtKB text format
</h3>
<p>
 We have changed the format of the FT and CC section of the UniProtKB text files. The
 <a href="http://www.uniprot.org/news/2019/12/18/release#text%5Fft">
  changes of the FT section
 </a>
 likely affects all parsers, and software will have to be adapted accordingly. The
 <a href="http://www.uniprot.org/news/2019/12/18/release#text%5Fcc">
  changes of the CC section
 </a>
 are smaller, but may also require code adaptations depending on the CC annotation types that you parse.
</p>
<p>
 The motivation for this change is described in the section "Functional annotation of different gene products in UniProtKB/Swiss-Prot" below, where you can also find the technical details and examples under the heading
 <a href="http://www.uniprot.org#text%5Fformat">
  Text format
 </a>
 .
</p>
<h3 id="line_length">
 Change of line length in UniProtKB text format
</h3>
<p>
 Historically, the lines of the UniProtKB text format have been wrapped at 75 characters for technical reasons (terminal screen size and data processing capabilities). When these technical restrictions vanished, we introduced exceptions for data like URLs, protein names and cross-references where line wrapping does not improve readability. These lines can be up to
 <a href="http://www.uniprot.org/news/2010/11/30/release">
  255 characters
 </a>
 long, but most lines are still wrapped at 75 characters for readability. We have now
 <strong>
  increased the maximum number of characters for wrapped lines to 80
 </strong>
 in the context of the
 <a href="http://www.uniprot.org#text%5Fft">
  format change of the FT section of the UniProtKB text format
 </a>
 for the functional annotation of different gene products in UniProtKB/Swiss-Prot described below.
</p>
<h3 id="Functional_annotation_of_different_gene_products_in_UniProtKB_Swiss_Prot">
 Functional annotation of different gene products in UniProtKB/Swiss-Prot
</h3>
<p>
 To reduce
 <a href="http://www.uniprot.org/help/redundancy">
  database redundancy
 </a>
 , the UniProtKB/Swiss-Prot policy is to describe, whenever possible, all protein products that are encoded by one gene in a given species in a single entry. This includes
 <a href="http://www.uniprot.org/help/alternative%5Fproducts">
  isoforms
 </a>
 generated by alternative promoter usage, alternative splicing, alternative initiation and ribosomal frameshifting. We assign a
 <a href="http://www.uniprot.org/help/alternative%5Fproducts#iso%5Fname">
  name
 </a>
 and a
 <a href="http://www.uniprot.org/help/alternative%5Fproducts#iso%5Fid">
  unique identifier
 </a>
 to each isoform and choose one of them to be the
 <a href="http://www.uniprot.org/help/canonical%5Fand%5Fisoforms">
  canonical sequence
 </a>
 that is shown in the UniProtKB text and XML format (the RDF format shows all sequences). All
 <a href="http://www.uniprot.org/help/sequence%5Fannotation">
  positional annotations
 </a>
 in the entry referred to this canonical sequence until this release. Some gene products are precursors that are processed by proteolytic cleavage to generate the biologically active product(s). These products are described by their location on the sequence, a name and a
 <a href="http://www.uniprot.org/help/sequence%5Fannotation#annotation%5Fid">
  unique identifier
 </a>
 .
</p>
<p>
 When isoforms, or products of proteolytic cleavage, are known to differ in their function or other characteristics, we generally describe this in the text of the respective annotations. To make this information also accessible to software applications, we adapted the UniProtKB text format to describe the product to which an annotation applies in a computer-processable way. The schemas of the XML and RDF format already supported this and required no changes. The following sections describe the changes for the text format and how the data is represented in the XML and RDF format.
</p>
<h4 id="text_format">
 Text format
</h4>
<p>
 Isoforms are described in ALTERNATIVE PRODUCTS annotations in the CC section. The products of proteolytic cleavage are described in PEPTIDE and CHAIN annotations in the FT section. All three annotation types provide a name (
 <code>
  &lt;ProductName&gt;
 </code>
 ) and a unique ID (
 <code>
  &lt;ProductId&gt;
 </code>
 ) for the product that they describe:
</p>
<ul>
 <li>
  ALTERNATIVE PRODUCTS annotations show the name of an isoform in the
  <code>
   Name
  </code>
  field and its ID in the
  <code>
   IsoId
  </code>
  field.
  <pre>CC   -!- ALTERNATIVE PRODUCTS:
...
CC       <code style="color: green;">Name=</code><code style="color: red;">&lt;ProductName&gt;</code><code style="color: green;">;</code>
CC         <code style="color: green;">IsoId=</code><code style="color: red;">&lt;ProductId&gt;</code><code style="color: green;">;</code> Sequence=Displayed;
</pre>
 </li>
 <li>
  PEPTIDE and CHAIN annotations showed the name of a proteolytic cleavage product in the
  <code>
   &lt;Description&gt;
  </code>
  field and its ID in the
  <code>
   FTId
  </code>
  field in the previous text format:
  <pre>FT   CHAIN       &lt;B&gt;    &lt;E&gt;       <code style="color: red;">&lt;ProductName&gt;</code><code style="color: green;">.</code>
FT                                <code style="color: green;">/FTId=</code><code style="color: red;">&lt;ProductId&gt;</code><code style="color: green;">.</code>
</pre>
  In the new text format that is described in more details in the
  <a href="http://www.uniprot.org#text%5Fft">
   FT section
  </a>
  they are shown in the
  <code>
   /note=
  </code>
  and
  <code>
   /id=
  </code>
  qualifiers, respectively:
  <pre>FT   CHAIN           &lt;B&gt;..&lt;E&gt;
FT                   <code style="color: green;">/note="</code><code style="color: red;">&lt;ProductName&gt;</code><code style="color: green;">"</code>
FT                   <code style="color: green;">/id="</code><code style="color: red;">&lt;ProductId&gt;</code><code style="color: green;">"</code>
</pre>
 </li>
</ul>
<p>
 Example:
 <a href="http://www.uniprot.org/uniprot/O60443.txt">
  O60443
 </a>
</p>
<pre>CC   -!- ALTERNATIVE PRODUCTS:
CC       Event=Alternative splicing; Named isoforms=3;
CC       <code style="color: green;">Name=</code><code style="color: red;">1</code><code style="color: green;">;</code> Synonyms=Long;
CC         <code style="color: green;">IsoId=</code><code style="color: red;">O60443-1</code><code style="color: green;">;</code> Sequence=Displayed;
CC       <code style="color: green;">Name=</code><code style="color: red;">2</code><code style="color: green;">;</code> Synonyms=Short;
CC         <code style="color: green;">IsoId=</code><code style="color: red;">O60443-2</code><code style="color: green;">;</code> Sequence=VSP_004190;
CC         Note=No experimental confirmation available.;
CC       <code style="color: green;">Name=</code><code style="color: red;">3</code><code style="color: green;">;</code>
CC         <code style="color: green;">IsoId=</code><code style="color: red;">O60443-3</code><code style="color: green;">;</code> Sequence=VSP_044276;
...
FT   CHAIN         1    496       <code style="color: red;">Gasdermin-E</code><code style="color: green;">.</code>
FT                                <code style="color: green;">/FTId=</code><code style="color: red;">PRO_0000148178</code><code style="color: green;">.</code>
FT   CHAIN         1    270       <code style="color: red;">Gasdermin-E, N-terminal</code><code style="color: green;">.</code>
FT                                {ECO:0000269|PubMed:27281216,
FT                                ECO:0000305|PubMed:28459430}.
FT                                <code style="color: green;">/FTId=</code><code style="color: red;">PRO_0000442786</code><code style="color: green;">.</code>
FT   CHAIN       271    496       <code style="color: red;">Gasdermin-E, C-terminal</code><code style="color: green;">.</code>
FT                                {ECO:0000305|PubMed:28459430}.
FT                                <code style="color: green;">/FTId=</code><code style="color: red;">PRO_0000442787</code><code style="color: green;">.</code>
</pre>
<h5 id="text_cc">
 CC section
</h5>
<p>
 The annotation types in the CC section describe a product by its name (isoform names are prefixed with the term "Isoform"). In the format descriptions below this name is represented by
 <strong>
  <code>
   &lt;ProductName&gt;
  </code>
 </strong>
 . Different products are described in separate annotations (see
 <a href="http://www.uniprot.org#function">
  FUNCTION
 </a>
 and
 <a href="http://www.uniprot.org#bpc">
  BIOPHYSICOCHEMICAL PROPERTIES
 </a>
 examples).
</p>
<p>
 All annotation types of the CC section start with:
</p>
<pre>CC   -!- <code style="color: grey;">&lt;TYPE&gt;</code>:
</pre>
<p>
 Where
 <code>
  &lt;TYPE&gt;
 </code>
 is a value from the controlled vocabulary of annotation types.
</p>
<p>
 In some annotation types the content of the annotation used to directly follow the
 <code>
  &lt;TYPE&gt;
 </code>
 , and lines were wrapped at 75 chars:
</p>
<pre>CC   -!- <code style="color: grey;">&lt;TYPE&gt;</code>: <code style="color: grey;">&lt;Content&gt;</code>
</pre>
<p>
 In the new format a
 <strong>
  <code>
   &lt;ProductName&gt;
  </code>
 </strong>
 may be added between the
 <code>
  &lt;TYPE&gt;
 </code>
 and the
 <code>
  &lt;Content&gt;
 </code>
 and lines are wrapped at 80 chars (see
 <a href="http://www.uniprot.org/help/line%5Flength%5Fchange">
  Change of line length in UniProtKB text format
 </a>
 ):
</p>
<pre>CC   -!- <code style="color: grey;">&lt;TYPE&gt;</code>: [<code style="color: grey;">&lt;ProductName&gt;</code>]: <code style="color: grey;">&lt;Content&gt;
</code></pre>
<p>
 The
 <strong>
  <code>
   &lt;ProductName&gt;
  </code>
 </strong>
 is surrounded by square brackets and separated by a colon from the
 <code>
  &lt;Content&gt;
 </code>
 to make it possible to parse it with a
 <a href="https://en.wikibooks.org/wiki/Regular%5FExpressions/POSIX-Extended%5FRegular%5FExpressions">
  POSIX ERE
 </a>
 like this one:
</p>
<pre>^CC   -!- ([^:]+):(?: \[(.+?)\]:)? (.+)
</pre>
<p>
 Where $1=
 <code>
  &lt;TYPE&gt;
 </code>
 , $2=
 <code>
  &lt;ProductName&gt;
 </code>
 , $3=
 <code>
  &lt;Content&gt;
 </code>
 .
</p>
<p>
 In annotation types where the content is structured as a list of different fields that are formatted according to custom rules for better readability, the annotation content starts on a new line:
</p>
<pre>CC   -!- <code style="color: grey;">&lt;TYPE&gt;</code>:
CC       <code style="color: grey;">&lt;Content&gt;</code>
</pre>
<p>
 In the new format a
 <strong>
  <code>
   &lt;ProductName&gt;
  </code>
 </strong>
 may be added after the
 <code>
  &lt;TYPE&gt;
 </code>
 and this line is not wrapped (i.e. it may in rare cases exceed 80 chars).
</p>
<pre>CC   -!- <code style="color: grey;">&lt;TYPE&gt;</code>: [<code style="color: grey;">&lt;ProductName&gt;</code>]:
CC       <code style="color: grey;">&lt;Content&gt;</code>
</pre>
<p>
 The format of the
 <code>
  &lt;Content&gt;
 </code>
 remains unchanged.
</p>
<p>
 A
 <code>
  &lt;ProductName&gt;
 </code>
 cannot be added to ALTERNATIVE PRODUCTS and INTERACTION annotations. The INTERACTION format will be adapted in a different way to describe binary interactions that involve isoforms and/or products of proteolytic cleavage (see
 <a href="http://www.uniprot.org/help/interaction%5Fchange">
  Change of annotation topic 'Interaction'
 </a>
 ).
</p>
<p>
 <em>
  Please note that the previous text format of SUBCELLULAR LOCATION, COFACTOR and MASS SPECTROMETRY annotations already allowed to specify a product name/ID, but we have adapted it to be consistent with all other annotation types.
 </em>
</p>
<p>
 Representative examples for different annotation types are shown here:
</p>
<p>
 <ins>
  <a id="function">
   FUNCTION
  </a>
 </ins>
</p>
<p>
 Example:
 <a href="http://www.uniprot.org/uniprot/Q96F85.txt">
  Q96F85
 </a>
</p>
<pre>CC   -!- ALTERNATIVE PRODUCTS:
CC       Event=Alternative splicing; Named isoforms=2;
CC       <code style="color: green;">Name=</code><code style="color: red;">1</code><code style="color: green;">;</code> Synonyms=CRIP1a;
CC         <code style="color: green;">IsoId=</code><code style="color: red;">Q96F85-1</code><code style="color: green;">;</code> Sequence=Displayed;
CC       <code style="color: green;">Name=</code><code style="color: red;">2</code><code style="color: green;">;</code> Synonyms=CRIP1b;
CC         <code style="color: green;">IsoId=</code><code style="color: red;">Q96F85-2</code><code style="color: green;">;</code> Sequence=VSP_035598;
</pre>
<p>
 Previous format:
</p>
<pre>CC   -!- FUNCTION: <code style="color: blue;">Isoform 1</code> suppresses cannabinoid receptor CNR1-mediated
CC       tonic inhibition of voltage-gated calcium channels. <code style="color: blue;">Isoform 2</code> does
CC       not have this effect. {ECO:0000269|PubMed:17895407}.
</pre>
<p>
 New format:
</p>
<pre>CC   -!- FUNCTION: <code style="color: green;">[</code><code style="color: red;">Isoform 1</code><code style="color: green;">]:</code> Suppresses cannabinoid receptor CNR1-mediated
CC       tonic inhibition of voltage-gated calcium channels.
CC       {ECO:0000269|PubMed:17895407}.
CC   -!- FUNCTION: <code style="color: green;">[</code><code style="color: red;">Isoform 2</code><code style="color: green;">]:</code> Does not suppress cannabinoid receptor CNR1-
CC       mediated tonic inhibition of voltage-gated calcium channels.
CC       {ECO:0000269|PubMed:17895407}.
</pre>
<p>
 <ins>
  DISEASE
 </ins>
</p>
<p>
 Example:
 <a href="http://www.uniprot.org/uniprot/P35555.txt">
  P35555
 </a>
</p>
<pre>FT   CHAIN      2732   2871       <code style="color: red;">Asprosin</code><code style="color: green;">.</code> {ECO:0000305|PubMed:27087445,
FT                                ECO:0000305|PubMed:9817919}.
FT                                <code style="color: green;">/FTId=</code><code style="color: red;">PRO_0000436882</code><code style="color: green;">.</code>
</pre>
<p>
 Previous format:
</p>
<pre>CC   -!- DISEASE: Marfan lipodystrophy syndrome (MFLS) [MIM:616914]: A
CC       syndrome characterized by congenital ...
CC       Note=The disease is caused by mutations affecting the gene
CC       represented in this entry. <code style="color: blue;">Asprosin</code>: Mutations specifically affect
CC       Asprosin, a hormone peptide present at the C-terminus of
CC       Fibrillin-1 chain, which is cleaved from Fibrillin-1 following
CC       secretion (PubMed:27087445). {ECO:0000269|PubMed:27087445}.
</pre>
<p>
 New format:
</p>
<pre>CC   -!- DISEASE: <code style="color: green;">[</code><code style="color: red;">Asprosin</code><code style="color: green;">]:</code> Marfan lipodystrophy syndrome (MFLS) [MIM:616914]:
CC       A syndrome characterized by congenital ...
CC       Note=The disease is caused by mutations affecting the gene represented
CC       in this entry. {ECO:0000269|PubMed:27087445}.
</pre>
<p>
 <ins>
  SUBCELLULAR LOCATION
 </ins>
</p>
<p>
 <em>
  Please note that the
  <a href="http://www.uniprot.org/news/2007/10/23/release">
   previous text format of SUBCELLULAR LOCATION annotations
  </a>
  already allowed to describe a product by its name in the optional first field. To be consistent with all other annotation types we have added square brackets around the product name.
 </em>
</p>
<p>
 Example:
 <a href="http://www.uniprot.org/uniprot/Q13421.txt">
  Q13421
 </a>
</p>
<pre>CC   -!- ALTERNATIVE PRODUCTS:
...
CC       <code style="color: green;">Name=</code><code style="color: red;">3</code><code style="color: green;">;</code> Synonyms=SMRP;
CC         <code style="color: green;">IsoId=</code><code style="color: red;">Q13421-2</code><code style="color: green;">;</code> Sequence=VSP_021059, VSP_021060;
...
FT   CHAIN        37    286       <code style="color: red;">Megakaryocyte-potentiating factor</code><code style="color: green;">.</code>
FT                                <code style="color: green;">/FTId=</code><code style="color: red;">PRO_0000253560</code><code style="color: green;">.</code>
</pre>
<p>
 Previous format:
</p>
<pre>CC   -!- SUBCELLULAR LOCATION: Cell membrane; Lipid-anchor, GPI-anchor.
CC       Golgi apparatus.
CC   -!- SUBCELLULAR LOCATION: <code style="color: red;">Megakaryocyte-potentiating factor</code><code style="color: green;">:</code> Secreted.
CC   -!- SUBCELLULAR LOCATION: <code style="color: red;">Isoform 3</code><code style="color: green;">:</code> Secreted.
</pre>
<p>
 New format:
</p>
<pre>CC   -!- SUBCELLULAR LOCATION: Cell membrane; Lipid-anchor, GPI-anchor. Golgi
CC       apparatus.
CC   -!- SUBCELLULAR LOCATION: <code style="color: green;">[</code><code style="color: red;">Megakaryocyte-potentiating factor</code><code style="color: green;">]:</code> Secreted.
CC   -!- SUBCELLULAR LOCATION: <code style="color: green;">[</code><code style="color: red;">Isoform 3</code><code style="color: green;">]:</code> Secreted.
</pre>
<p>
 <ins>
  <a id="mass_spectrometry">
   MASS SPECTROMETRY
  </a>
 </ins>
</p>
<p>
 <em>
  Please note that the previous text format of MASS SPECTROMETRY annotations already allowed to describe a product (by its sequence range and an optional
  <a href="http://www.uniprot.org/help/alternative%5Fproducts#iso%5Fid">
   isoform ID
  </a>
  ) in the
  <code>
   Range
  </code>
  field. To be consistent with all other annotation types we have replaced the
  <code>
   Range
  </code>
  field by a
  <code>
   &lt;ProductName&gt;
  </code>
  field.
 </em>
</p>
<p>
 Example:
 <a href="http://www.uniprot.org/uniprot/P09493.txt">
  P09493
 </a>
</p>
<pre>CC   -!- ALTERNATIVE PRODUCTS:
...
CC       <code style="color: green;">Name=</code><code style="color: red;">3</code><code style="color: green;">;</code> Synonyms=Fibroblast, TM3;
CC         <code style="color: green;">IsoId=</code><code style="color: red;">P09493-3</code><code style="color: green;">;</code> Sequence=VSP_006577, VSP_006579;
</pre>
<p>
 Previous format:
</p>
<pre>CC   -!- MASS SPECTROMETRY: Mass=32875.93; Method=MALDI; <code style="color: green;">Range=</code><code style="color: red;">1-284</code>
CC       <code style="color: green;">(<code style="color: red;">P09493-3</code>)</code><code style="color: green;">;</code> Evidence={ECO:0000269|PubMed:11840567};
</pre>
<p>
 New format:
</p>
<pre>CC   -!- MASS SPECTROMETRY: <code style="color: green;">[</code><code style="color: red;">Isoform 3</code><code style="color: green;">]:</code> Mass=32875.93; Method=MALDI;
CC       Evidence={ECO:0000269|PubMed:11840567};
</pre>
<p>
 <ins>
  RNA EDITING
 </ins>
</p>
<p>
 Example:
 <a href="http://www.uniprot.org/uniprot/Q9P225.txt">
  Q9P225
 </a>
</p>
<pre>CC   -!- ALTERNATIVE PRODUCTS:
...
CC       <code style="color: green;">Name=</code><code style="color: red;">3</code><code style="color: green;">;</code>
CC         <code style="color: green;">IsoId=</code><code style="color: red;">Q9P225-3</code><code style="color: green;">;</code> Sequence=VSP_031913, VSP_031914, VSP_031915;
</pre>
<p>
 Previous format:
</p>
<pre>CC   -!- RNA EDITING: Modified_positions=Not_applicable; Note=Exon 13
CC       included in <code style="color: blue;">isoform 3</code> is extensively edited in brain.
CC       {ECO:0000269|PubMed:20835228};
</pre>
<p>
 New format:
</p>
<pre>CC   -!- RNA EDITING: <code style="color: green;">[</code><code style="color: red;">Isoform 3</code><code style="color: green;">]:</code> Modified_positions=Not_applicable; Note=Exon
CC       13 is extensively edited in brain. {ECO:0000269|PubMed:20835228};
</pre>
<p>
 <ins>
  WEB RESOURCE
 </ins>
</p>
<p>
 Example:
 <a href="http://www.uniprot.org/uniprot/P50570.txt">
  P50570
 </a>
</p>
<pre>CC   -!- ALTERNATIVE PRODUCTS:
...
CC       <code style="color: green;">Name=</code><code style="color: red;">1</code><code style="color: green;">;</code>
CC         <code style="color: green;">IsoId=</code><code style="color: red;">P50570-1</code><code style="color: green;">;</code> Sequence=Displayed;
</pre>
<p>
 Previous format:
</p>
<pre>CC   -!- WEB RESOURCE: Name=The UMD-DNM2-<code style="color: blue;">isoform 1</code> mutations database;
CC       URL="http://www.umd.be/DNM2/";
</pre>
<p>
 New format:
</p>
<pre>CC   -!- WEB RESOURCE: <code style="color: green;">[</code><code style="color: red;">Isoform 1</code><code style="color: green;">]:</code> Name=The UMD-DNM2-isoform 1 mutations
CC       database;
CC       URL="http://www.umd.be/DNM2/";
</pre>
<p>
 <ins>
  CATALYTIC ACTIVITY
 </ins>
</p>
<p>
 Example:
 <a href="http://www.uniprot.org/uniprot/Q2YHF0.txt">
  Q2YHF0
 </a>
</p>
<pre>FT   CHAIN      1475   2092       <code style="color: red;">Serine protease NS3</code><code style="color: green;">.</code>
FT                                {ECO:0000250|UniProtKB:P29990}.
FT                                <code style="color: green;">/FTId=</code><code style="color: red;">PRO_0000268140</code><code style="color: green;">.</code>
...
FT   CHAIN      2488   3387       <code style="color: red;">RNA-directed RNA polymerase NS5</code><code style="color: green;">.</code>
FT                                {ECO:0000250|UniProtKB:P29990}.
FT                                <code style="color: green;">/FTId=</code><code style="color: red;">PRO_0000268144</code><code style="color: green;">.</code>
</pre>
<p>
 Previous format:
</p>
<pre>CC   -!- CATALYTIC ACTIVITY:
CC       Reaction=Selective hydrolysis of -Xaa-Xaa-|-Yaa- bonds in which
CC         each of the Xaa can be either Arg or Lys and Yaa can be either
CC         Ser or Ala.; EC=3.4.21.91;
CC   -!- CATALYTIC ACTIVITY:
CC       Reaction=a ribonucleoside 5'-triphosphate + RNA(n) = diphosphate +
CC         RNA(n+1); Xref=Rhea:RHEA:21248, Rhea:RHEA-COMP:11128, Rhea:RHEA-
CC         COMP:11129, ChEBI:CHEBI:33019, ChEBI:CHEBI:61557,
CC         ChEBI:CHEBI:83400; EC=2.7.7.48; Evidence={ECO:0000255|PROSITE-
CC         ProRule:PRU00539};
</pre>
<p>
 New format:
</p>
<pre>CC   -!- CATALYTIC ACTIVITY: <code style="color: green;">[</code><code style="color: red;">Serine protease NS3</code><code style="color: green;">]:</code>
CC       Reaction=Selective hydrolysis of -Xaa-Xaa-|-Yaa- bonds in which each of
CC         the Xaa can be either Arg or Lys and Yaa can be either Ser or Ala.;
CC         EC=3.4.21.91;
CC   -!- CATALYTIC ACTIVITY: <code style="color: green;">[</code><code style="color: red;">RNA-directed RNA polymerase NS5</code><code style="color: green;">]:</code>
CC       Reaction=a ribonucleoside 5'-triphosphate + RNA(n) = diphosphate +
CC         RNA(n+1); Xref=Rhea:RHEA:21248, Rhea:RHEA-COMP:11128, Rhea:RHEA-
CC         COMP:11129, ChEBI:CHEBI:33019, ChEBI:CHEBI:61557, ChEBI:CHEBI:83400;
CC         EC=2.7.7.48; Evidence={ECO:0000255|PROSITE-ProRule:PRU00539};
</pre>
<p>
 <ins>
  COFACTOR
 </ins>
</p>
<p>
 <em>
  Please note that the
  <a href="http://www.uniprot.org/news/2014/11/26/release">
   previous text format of COFACTOR annotations
  </a>
  already allowed to describe a product by its name in the optional first field. To be consistent with all other annotation types we have added square brackets around the product name.
 </em>
</p>
<p>
 Example:
 <a href="http://www.uniprot.org/uniprot/P26662.txt">
  P26662
 </a>
</p>
<pre>FT   CHAIN      1027   1657       <code style="color: red;">Serine protease NS3</code><code style="color: green;">.</code> {ECO:0000255}.
FT                                <code style="color: green;">/FTId=</code><code style="color: red;">PRO_0000037644</code><code style="color: green;">.</code>
FT   CHAIN      1658   1711       <code style="color: red;">Non-structural protein 4A</code><code style="color: green;">.</code> {ECO:0000255}.
FT                                <code style="color: green;">/FTId=</code><code style="color: red;">PRO_0000037645</code><code style="color: green;">.</code>
</pre>
<p>
 Previous format:
</p>
<pre>CC   -!- COFACTOR: <code style="color: red;">Serine protease NS3</code><code style="color: green;">:</code>
CC       Name=Zn(2+); Xref=ChEBI:CHEBI:29105;
CC         Evidence={ECO:0000269|PubMed:9060645};
CC       Note=Binds 1 zinc ion. {ECO:0000269|PubMed:9060645};
CC   -!- COFACTOR: <code style="color: red;">Non-structural protein 5A</code><code style="color: green;">:</code>
CC       Name=Zn(2+); Xref=ChEBI:CHEBI:29105; Evidence={ECO:0000250};
CC       Note=Binds 1 zinc ion in the NS5A N-terminal domain.
CC       {ECO:0000250};
</pre>
<p>
 New format:
</p>
<pre>CC   -!- COFACTOR: <code style="color: green;">[</code><code style="color: red;">Serine protease NS3</code><code style="color: green;">]:</code>
CC       Name=Zn(2+); Xref=ChEBI:CHEBI:29105;
CC         Evidence={ECO:0000269|PubMed:9060645};
CC       Note=Binds 1 zinc ion. {ECO:0000269|PubMed:9060645};
CC   -!- COFACTOR: <code style="color: green;">[</code><code style="color: red;">Non-structural protein 5A</code><code style="color: green;">]:</code>
CC       Name=Zn(2+); Xref=ChEBI:CHEBI:29105; Evidence={ECO:0000250};
CC       Note=Binds 1 zinc ion in the NS5A N-terminal domain. {ECO:0000250};
</pre>
<p>
 <ins>
  <a id="bpc">
   BIOPHYSICOCHEMICAL PROPERTIES
  </a>
 </ins>
</p>
<p>
 Example:
 <a href="http://www.uniprot.org/uniprot/Q9ULC5.txt">
  Q9ULC5
 </a>
</p>
<pre>CC   -!- ALTERNATIVE PRODUCTS:
...
CC       <code style="color: green;">Name=</code><code style="color: red;">1</code><code style="color: green;">;</code> Synonyms=ACSL5b, ACSL5-fl;
CC         <code style="color: green;">IsoId=</code><code style="color: red;">Q9ULC5-1</code><code style="color: green;">;</code> Sequence=Displayed;
...
CC       <code style="color: green;">Name=</code><code style="color: red;">3</code><code style="color: green;">;</code> Synonyms=ACSL5delta20;
CC         <code style="color: green;">IsoId=</code><code style="color: red;">Q9ULC5-4</code><code style="color: green;">;</code> Sequence=VSP_038233;
</pre>
<p>
 Previous format:
</p>
<pre>CC   -!- BIOPHYSICOCHEMICAL PROPERTIES:
CC       Kinetic parameters:
CC         KM=0.11 uM for palmitic acid (<code style="color: blue;">isoform 1</code> at pH 7.5)
CC         {ECO:0000269|PubMed:17681178};
CC         KM=0.38 uM for palmitic acid (<code style="color: blue;">isoform 1</code> at pH 9.5)
CC         {ECO:0000269|PubMed:17681178};
CC         KM=0.04 uM for palmitic acid (<code style="color: blue;">isoform 3</code> at pH 7.5)
CC         {ECO:0000269|PubMed:17681178};
CC         KM=0.15 uM for palmitic acid (<code style="color: blue;">isoform 3</code> at pH 8.5)
CC         {ECO:0000269|PubMed:17681178};
CC       pH dependence:
CC         Optimum pH is 9.5 (<code style="color: blue;">isoform 1</code>), 7.5-8.5 (<code style="color: blue;">isoform 3</code>).
CC         {ECO:0000269|PubMed:17681178};
</pre>
<p>
 New format:
</p>
<pre>CC   -!- BIOPHYSICOCHEMICAL PROPERTIES: <code style="color: green;">[</code><code style="color: red;">Isoform 1</code><code style="color: green;">]:</code>
CC       Kinetic parameters:
CC         KM=0.11 uM for palmitic acid (at pH 7.5)
CC         {ECO:0000269|PubMed:17681178};
CC         KM=0.38 uM for palmitic acid (at pH 9.5)
CC         {ECO:0000269|PubMed:17681178};
CC       pH dependence:
CC         Optimum pH is 9.5. {ECO:0000269|PubMed:17681178};
CC   -!- BIOPHYSICOCHEMICAL PROPERTIES: <code style="color: green;">[</code><code style="color: red;">Isoform 3</code><code style="color: green;">]:</code>
CC       Kinetic parameters:
CC         KM=0.04 uM for palmitic acid (at pH 7.5)
CC         {ECO:0000269|PubMed:17681178};
CC         KM=0.15 uM for palmitic acid (at pH 8.5)
CC         {ECO:0000269|PubMed:17681178};
CC       pH dependence:
CC         Optimum pH is 7.5-8.5. {ECO:0000269|PubMed:17681178};
</pre>
<p>
 <ins>
  SEQUENCE CAUTION
 </ins>
</p>
<p>
 Example:
 <a href="http://www.uniprot.org/uniprot/Q9NQS3.txt">
  Q9NQS3
 </a>
</p>
<p>
 Previous format:
</p>
<pre>CC   -!- ALTERNATIVE PRODUCTS:
...
CC       <code style="color: green;">Name=</code><code style="color: red;">1</code><code style="color: green;">;</code>
CC         <code style="color: green;">IsoId=</code><code style="color: red;">Q9NQS3-1</code><code style="color: green;">;</code> Sequence=Displayed;
...
CC       <code style="color: green;">Name=</code><code style="color: red;">3</code><code style="color: green;">;</code>
CC         <code style="color: green;">IsoId=</code><code style="color: red;">Q9NQS3-3</code><code style="color: green;">;</code> Sequence=VSP_046893, VSP_046894;
CC         Note=<code style="color: blue;">Ref.2 (BAC11404) sequence differs from that shown due to</code>
CC         <code style="color: blue;">erroneous termination (Truncated C-terminus).</code> {ECO:0000305};
...
CC   -!- SEQUENCE CAUTION:
CC       Sequence=AAH17572.1; Type=Erroneous initiation; Note=Truncated N-terminus.; Evidence={ECO:0000305};
</pre>
<p>
 New format:
</p>
<pre>CC   -!- ALTERNATIVE PRODUCTS:
...
CC       <code style="color: green;">Name=</code><code style="color: red;">1</code><code style="color: green;">;</code>
CC         <code style="color: green;">IsoId=</code><code style="color: red;">Q9NQS3-1</code><code style="color: green;">;</code> Sequence=Displayed;
...
CC       <code style="color: green;">Name=</code><code style="color: red;">3</code><code style="color: green;">;</code>
CC         <code style="color: green;">IsoId=</code><code style="color: red;">Q9NQS3-3</code><code style="color: green;">;</code> Sequence=VSP_046893, VSP_046894;
...
CC   -!- SEQUENCE CAUTION: <code style="color: green;">[</code><code style="color: red;">Isoform 1</code><code style="color: green;">]:</code>
CC       Sequence=AAH17572.1; Type=Erroneous initiation; Note=Truncated N-terminus.; Evidence={ECO:0000305};
CC   -!- SEQUENCE CAUTION: <code style="color: green;">[</code><code style="color: red;">Isoform 3</code><code style="color: green;">]:</code>
CC       Sequence=BAC11404.1; Type=Erroneous termination; Note=Truncated C-terminus.; Evidence={ECO:0000305};
</pre>
<h5 id="text_ft">
 FT section
</h5>
<p>
 Note: The format descriptions make use of
 <a href="https://en.wikibooks.org/wiki/Regular%5FExpressions/POSIX-Extended%5FRegular%5FExpressions">
  POSIX ERE
 </a>
 syntax.
</p>
<p>
 All
 <a href="http://www.uniprot.org/help/sequence%5Fannotation">
  positional annotations
 </a>
 in the FT section previously referred to the
 <a href="http://www.uniprot.org/help/canonical%5Fand%5Fisoforms">
  canonical sequence
 </a>
 that is shown in the UniProtKB entry. This was the text format of these annotation types:
</p>
<pre>FT   <code style="color: grey;">&lt;TYPE&gt;</code>      <code style="color: grey;">&lt;B&gt;</code>    <code style="color: grey;">&lt;E&gt;</code>       (<code style="color: grey;">&lt;Description&gt;</code>.)?( {<code style="color: grey;">&lt;Evidences&gt;</code>}.)?
(FT                                /FTId=<code style="color: grey;">&lt;Id&gt;</code>.)?
</pre>
<p>
 Where
</p>
<ul>
 <li>
  <code>
   &lt;TYPE&gt;
  </code>
  is a value from the controlled vocabulary of positional annotation types.
 </li>
 <li>
  <code>
   &lt;B&gt;
  </code>
  and
  <code>
   &lt;E&gt;
  </code>
  are amino acid positions on the canonical sequence. For most annotation types, they are the begin and end position of a sequence range, but they have other semantics for some types (e.g. CROSSLNK and DISULFID).
 </li>
 <li>
  <code>
   &lt;Description&gt;
  </code>
  may provide information in addition to that conveyed by the
  <code>
   &lt;TYPE&gt;
  </code>
  and the location
  <code>
   &lt;B&gt;
  </code>
  and
  <code>
   &lt;E&gt;
  </code>
  . This field is mandatory for some annotation types and optional for others.
 </li>
 <li>
  <code>
   &lt;Evidences&gt;
  </code>
  are optional and added between curly braces.
 </li>
 <li>
  <code>
   &lt;Id&gt;
  </code>
  is a
  <a href="http://www.uniprot.org/help/sequence%5Fannotation#annotation%5Fid">
   unique annotation identifier
  </a>
  that is mandatory for some annotation types, including CHAIN and PEPTIDE where it corresponds to the
  <code>
   &lt;ProductId&gt;
  </code>
  .
 </li>
</ul>
<p>
 We have modified this format in order to describe amino acid positions on isoforms sequences. The new format is inspired by the
 <a href="http://www.insdc.org/files/feature%5Ftable.html">
  INSDC's feature table format
 </a>
 to enable code reuse:
</p>
<pre>FT   <code style="color: grey;">&lt;TYPE&gt;</code>          <code style="color: grey;">&lt;Location&gt;</code>
(FT                   /<code style="color: grey;">&lt;Qualifier&gt;</code>(="<code style="color: grey;">&lt;Value&gt;</code>")?)*
</pre>
<p>
 Where
</p>
<ul>
 <li>
  <code>
   &lt;TYPE&gt;
  </code>
  is a value from the controlled vocabulary of positional annotation types.
 </li>
 <li>
  <code>
   &lt;Location&gt;
  </code>
  is a sequence location on the canonical or an isoform sequence. We will use for now only a subset of the
  <a href="http://www.insdc.org/files/feature%5Ftable.html#3.4">
   INSDC Location
  </a>
  types: A
  <code>
   &lt;Location&gt;
  </code>
  must be either a single
  <code>
   &lt;Position&gt;
  </code>
  or a range of
  <code>
   &lt;Position&gt;
  </code>
  that may optionally be preceded by an
  <a href="http://www.uniprot.org/help/alternative%5Fproducts#iso%5Fid">
   isoform ID
  </a>
  . The
  <code>
   &lt;
  </code>
  and
  <code>
   &gt;
  </code>
  symbols may be used with begin and end positions to indicate that the begin or end point is beyond the specified amino acid position. Please note that we have to extend the INSDC Location format with the
  <code>
   ?
  </code>
  symbol to allow us to represent all existing UniProtKB locations. This symbol may precede a
  <code>
   &lt;Position&gt;
  </code>
  to indicate that the exact position is unsure, or it may substitute the
  <code>
   &lt;Position&gt;
  </code>
  when the position is unknown.
  <pre>(<code style="color: grey;">&lt;IsoformId&gt;</code>:)?((&lt;|?)?<code style="color: grey;">&lt;Position&gt;</code>|?)(..((&gt;|?)?<code style="color: grey;">&lt;Position&gt;</code>|?))?
</pre>
 </li>
 <li>
  /
  <code>
   &lt;Qualifier&gt;
  </code>
  may provide information in addition to that conveyed by the
  <code>
   &lt;TYPE&gt;
  </code>
  and
  <code>
   &lt;Location&gt;
  </code>
  . While we will follow the format of the
  <a href="http://www.insdc.org/files/feature%5Ftable.html#3.3">
   INSDC Qualifiers
  </a>
  , we will introduce our own
  <code>
   &lt;Qualifier&gt;
  </code>
  types where necessary. For this format change, we will represent the existing data with 3 qualifiers:
  <ul>
   <li>
    <code>
     /note=
    </code>
    will show the content of the current
    <code>
     &lt;Description&gt;
    </code>
    field.
   </li>
   <li>
    <code>
     /evidence=
    </code>
    will show the content of the current
    <code>
     &lt;Evidences&gt;
    </code>
    field.
   </li>
   <li>
    <code>
     /id=
    </code>
    will show the content of the current
    <code>
     /FTId=
    </code>
    field.
   </li>
  </ul>
 </li>
</ul>
<p>
 In a future format change, we may introduce more
 <code>
  &lt;Location&gt;
 </code>
 and
 <code>
  &lt;Qualifier&gt;
 </code>
 types to structure the description of positional annotations further.
</p>
<p>
 Lines are wrapped at 80 chars (see section
 <a href="http://www.uniprot.org#line%5Flength">
  Change of line length in UniProtKB text format
 </a>
 above).
</p>
<p>
 Example:
 <a href="http://www.uniprot.org/uniprot/P84077.txt">
  P84077
 </a>
</p>
<p>
 <em>
  This example illustrates the format change with a selection of representative positional annotation types that refer to the canonical sequence.
 </em>
</p>
<p>
 Previous format:
</p>
<pre>FT   INIT_MET      1      1       <code style="color: brown;">Removed</code><code style="color: green;">. {</code><code style="color: orange;">ECO:0000244|PubMed:19413330,</code>
FT                                <code style="color: orange;">ECO:0000244|PubMed:22223895,</code>
FT                                <code style="color: orange;">ECO:0000269|PubMed:25255805,</code>
FT                                <code style="color: orange;">ECO:0000269|PubMed:25807930</code><code style="color: green;">}.</code>
FT   CHAIN         2    181       <code style="color: red;">ADP-ribosylation factor 1</code><code style="color: green;">.</code>
FT                                <code style="color: green;">/FTId=</code><code style="color: red;">PRO_0000207378</code><code style="color: green;">.</code>
...
FT   NP_BIND     126    129       <code style="color: brown;">GTP</code><code style="color: green;">. {</code><code style="color: orange;">ECO:0000244|PDB:1HUR,</code>
FT                                <code style="color: orange;">ECO:0000244|PDB:1RE0,</code>
FT                                <code style="color: orange;">ECO:0000244|PDB:1U81,</code>
FT                                <code style="color: orange;">ECO:0000244|PDB:3O47, ECO:0000305</code><code style="color: green;">}.</code>
...
FT   VARIANT      35     35       <code style="color: brown;">Y -&gt; H (in PVNH8; decreased interaction</code>
FT                                <code style="color: brown;">with GGA3; dbSNP:rs879036238)</code><code style="color: green;">.</code>
FT                                <code style="color: green;">{</code><code style="color: orange;">ECO:0000269|PubMed:28868155</code><code style="color: green;">}.</code>
FT                                <code style="color: green;">/FTId=</code><code style="color: red;">VAR_081272</code><code style="color: green;">.</code>
...
FT   HELIX         6      9       <code style="color: green;">{</code><code style="color: orange;">ECO:0000244|PDB:1HUR</code><code style="color: green;">}.</code>
</pre>
<p>
 New format:
</p>
<pre>FT   INIT_MET        1
FT                   <code style="color: green;">/note="</code><code style="color: brown;">Removed</code><code style="color: green;">"</code>
FT                   <code style="color: green;">/evidence="</code><code style="color: orange;">ECO:0000244|PubMed:19413330,</code>
FT                   <code style="color: orange;">ECO:0000244|PubMed:22223895, ECO:0000269|PubMed:25255805,</code>
FT                   <code style="color: orange;">ECO:0000269|PubMed:25807930</code><code style="color: green;">"</code>
FT   CHAIN           2..181
FT                   <code style="color: green;">/note="</code><code style="color: red;">ADP-ribosylation factor 1</code><code style="color: green;">"</code>
FT                   <code style="color: green;">/id="</code><code style="color: red;">PRO_0000207378</code><code style="color: green;">"</code>
...
FT   NP_BIND         126..129
FT                   <code style="color: green;">/note="</code><code style="color: brown;">GTP</code><code style="color: green;">"</code>
FT                   <code style="color: green;">/evidence="</code><code style="color: orange;">ECO:0000244|PDB:1HUR, ECO:0000244|PDB:1RE0,</code>
FT                   <code style="color: orange;">ECO:0000244|PDB:1U81, ECO:0000244|PDB:3O47, ECO:0000305</code><code style="color: green;">"</code>
...
FT   VARIANT         35
FT                   <code style="color: green;">/note="</code><code style="color: brown;">Y -&gt; H (in PVNH8; decreased interaction with GGA3;</code>
FT                   <code style="color: brown;">dbSNP:rs879036238)</code><code style="color: green;">"</code>
FT                   <code style="color: green;">/evidence="</code><code style="color: orange;">ECO:0000269|PubMed:28868155</code><code style="color: green;">"</code>
FT                   <code style="color: green;">/id="</code><code style="color: red;">VAR_081272</code><code style="color: green;">"</code>
...
FT   HELIX           6..9
FT                   <code style="color: green;">/evidence="</code><code style="color: orange;">ECO:0000244|PDB:1HUR</code><code style="color: green;">"</code>
</pre>
<p>
 Example:
 <a href="http://www.uniprot.org/uniprot/P0C551.txt">
  P0C551
 </a>
</p>
<p>
 <em>
  This example illustrates the use of the
  <code>
   &lt;
  </code>
  and
  <code>
   ?
  </code>
  symbols in UniProtKB locations.
 </em>
</p>
<p>
 Previous format:
</p>
<pre>FT   SIGNAL       &lt;1      ?       <code style="color: green;">{</code><code style="color: orange;">ECO:0000250</code><code style="color: green;">}.</code>
FT   PROPEP        ?     17       <code style="color: green;">{</code><code style="color: orange;">ECO:0000250</code><code style="color: green;">}.</code>
FT                                <code style="color: green;">/FTId=</code><code style="color: red;">PRO_0000293097</code><code style="color: green;">.</code>
FT   CHAIN        18    142       <code style="color: red;">Acidic phospholipase A2 KBf-grIB</code><code style="color: green;">.</code>
FT                                <code style="color: green;">/FTId=</code><code style="color: red;">FTId=PRO_0000293098</code><code style="color: green;">.</code>
</pre>
<p>
 New format:
</p>
<pre>FT   SIGNAL          &lt;1..?
FT                   <code style="color: green;">/evidence="</code><code style="color: orange;">ECO:0000250</code><code style="color: green;">"</code>
FT   PROPEP          ?..17
FT                   <code style="color: green;">/evidence="</code><code style="color: orange;">ECO:0000250</code><code style="color: green;">"</code>
FT                   <code style="color: green;">/id="</code><code style="color: red;">PRO_0000293097</code><code style="color: green;">"</code>
FT   CHAIN           18..142
FT                   <code style="color: green;">/note="</code><code style="color: red;">Acidic phospholipase A2 KBf-grIB</code><code style="color: green;">"</code>
FT                   <code style="color: green;">/id="</code><code style="color: red;">PRO_0000293098</code><code style="color: green;">"</code>
</pre>
<p>
 Example:
 <a href="http://www.uniprot.org/uniprot/P12821.txt">
  P12821
 </a>
</p>
<p>
 <em>
  This example illustrates how positional annotations for isoforms are represented.
 </em>
</p>
<p>
 Previous format:
</p>
<pre>CC   -!- ALTERNATIVE PRODUCTS:
CC       ...
CC       <code style="color: green;">Name=</code><code style="color: red;">Testis-specific</code><code style="color: green;">;</code> Synonyms=ACE-T;
CC         <code style="color: green;">IsoId=</code><code style="color: red;">P12821-3</code>, P22966-1<code style="color: green;">;</code>
CC         Sequence=VSP_035120, VSP_035121;
CC         Note=<code style="color: blue;">Variant in position: 32:S-&gt;P (in dbSNP:rs4317). Variant in</code>
CC         <code style="color: blue;">position: 49:S-&gt;G (in dbSNP:rs4318).</code>;
...
FT   VARIANT     154    154       <code style="color: brown;">A -&gt; T (in dbSNP:rs13306087)</code><code style="color: green;">.</code>
FT                                <code style="color: green;">/FTId=</code><code style="color: red;">VAR_029139</code><code style="color: green;">.</code>
</pre>
<p>
 New format:
</p>
<pre>CC   -!- ALTERNATIVE PRODUCTS:
CC       ...
CC       <code style="color: green;">Name=</code><code style="color: red;">Testis-specific</code><code style="color: green;">;</code> Synonyms=ACE-T;
CC         <code style="color: green;">IsoId=</code><code style="color: red;">P12821-3</code>, P22966-1<code style="color: green;">;</code>
CC         Sequence=VSP_035120, VSP_035121;
...
FT   VARIANT         154
FT                   <code style="color: green;">/note="</code><code style="color: brown;">A -&gt; T (in dbSNP:rs13306087)</code><code style="color: green;">"</code>
FT                   <code style="color: green;">/id="</code><code style="color: red;">VAR_029139</code><code style="color: green;">"</code>
...
FT   VARIANT         <code style="color: red;">P12821-3</code><code style="color: green;">:</code>32
FT                   <code style="color: green;">/note="</code><code style="color: brown;">S -&gt; P (in dbSNP:rs4317)</code><code style="color: green;">"</code>
FT                   <code style="color: green;">/id="</code><code style="color: red;">VAR_x</code><code style="color: green;">"</code>
FT   VARIANT         <code style="color: red;">P12821-3</code><code style="color: green;">:</code>49
FT                   <code style="color: green;">/note="</code><code style="color: brown;">S -&gt; G (in dbSNP:rs4318)</code><code style="color: green;">"</code>
FT                   <code style="color: green;">/id="</code><code style="color: red;">VAR_y</code><code style="color: green;">"</code>
</pre>
<h4 id="XML_format">
 XML format
</h4>
<p>
 The
 <a href="http://www.uniprot.org/docs/uniprot.xsd">
  UniProtKB XSD
 </a>
 already allowed to describe the product to which an annotation applies and required no changes.
</p>
<p>
 Isoforms are described in "alternative products" annotations. The products of proteolytic cleavage are described in "peptide" and "chain" annotations. All three annotation types provide a name (
 <code>
  &lt;ProductName&gt;
 </code>
 ) and/or a unique ID (
 <code>
  &lt;ProductId&gt;
 </code>
 ) for the product that they describe:
</p>
<ul>
 <li>
  "alternative products" annotations describe each isoform by an
  <code>
   isoform
  </code>
  element of
  <code>
   isoformType
  </code>
  . The
  <code>
   isoformType
  </code>
  describes the product IDs and names with sequences of
  <code>
   id
  </code>
  and
  <code>
   name
  </code>
  elements (where the first element in each sequence is the main product ID/name).
  <pre>&lt;comment type="alternative products"&gt;
  ...
  &lt;isoform&gt;
    <code style="color: green;">&lt;id&gt;</code><code style="color: red;">&lt;ProductId&gt;</code><code style="color: green;">&lt;/id&gt;</code>
    &lt;id&gt;&lt;OldProductId&gt;&lt;/id&gt;
    <code style="color: green;">&lt;name&gt;</code><code style="color: red;">&lt;ProductName&gt;</code><code style="color: green;">&lt;/name&gt;</code>
    &lt;name&gt;&lt;AlternativeProductName&gt;&lt;/name&gt;
    ...
  &lt;/isoform&gt;
  ...
&lt;/comment&gt;
</pre>
 </li>
</ul>
<ul>
 <li>
  "peptide" and "chain" annotations show the name and ID of a proteolytic cleavage product in the
  <code>
   description
  </code>
  and
  <code>
   id
  </code>
  attributes of the
  <code>
   featureType
  </code>
  .
  <pre>&lt;feature type="chain" <code style="color: green;">description="</code><code style="color: red;">&lt;ProductName&gt;</code><code style="color: green;">" id="</code><code style="color: red;">&lt;ProductId&gt;</code><code style="color: green;">"</code>&gt;
...
&lt;/feature&gt;
</pre>
 </li>
</ul>
<h5 id="xml_comment">
 commentType
</h5>
<p>
 The
 <code>
  commentType
 </code>
 has two ways to indicate that the annotation applies to a specific product:
</p>
<ul>
 <li>
  An optional
  <code>
   molecule
  </code>
  element of
  <code>
   moleculeType
  </code>
  allows to describe a product by its name or/and unique ID. It is currently only used for "subcellular location" and "cofactor" annotations (see examples below). In the future it may be used for all annotations that are represented by
  <code>
   commentType
  </code>
  .
 </li>
</ul>
<ul>
 <li>
  An optional sequence of
  <code>
   location
  </code>
  elements of
  <code>
   locationType
  </code>
  allows to describe the sequence coordinates of an annotation. The
  <code>
   locationType
  </code>
  has an optional
  <code>
   sequence
  </code>
  attribute that is only set (to an
  <a href="http://www.uniprot.org/help/alternative%5Fproducts#iso%5Fid">
   isoform ID
  </a>
  ) when the coordinates are not for the
  <a href="http://www.uniprot.org/help/canonical%5Fand%5Fisoforms">
   canonical sequence
  </a>
  . Sequence coordinates may currently be given for "rna editing", "sequence caution" and "mass spectrometry" annotations. In the future
  <a href="http://www.uniprot.org/help/sequence%5Fcaution%5Fpositions%5Fchange">
   sequence caution
  </a>
  and
  <a href="http://www.uniprot.org#mass_spectrometry">
   mass spectrometry
  </a>
  annotations will no longer describe sequence coordinates.
 </li>
</ul>
<p>
 <ins>
  subcellular location
 </ins>
</p>
<p>
 Example:
 <a href="http://www.uniprot.org/uniprot/Q13421.xml">
  Q13421
 </a>
</p>
<pre>&lt;comment type="alternative products"&gt;
  ...
  &lt;isoform&gt;
    <code style="color: green;">&lt;id&gt;</code><code style="color: red;">Q13421-2</code><code style="color: green;">&lt;/id&gt;</code>
    <code style="color: green;">&lt;name&gt;</code><code style="color: red;">3</code><code style="color: green;">&lt;/name&gt;</code>
    &lt;name&gt;SMRP&lt;/name&gt;
    &lt;sequence type="described" ref="VSP_021059 VSP_021060"/&gt;
    ...
  &lt;/isoform&gt;
  ...
&lt;/comment&gt;
...
&lt;feature type="chain" <code style="color: green;">description="</code><code style="color: red;">Megakaryocyte-potentiating factor</code><code style="color: green;">"</code>
                      <code style="color: green;">id="</code><code style="color: red;">PRO_0000253560</code><code style="color: green;">"</code>&gt;
  &lt;location&gt;
    &lt;begin position="37"/&gt;
    &lt;end position="286"/&gt;
  &lt;/location&gt;
&lt;/feature&gt;
</pre>
<pre>&lt;comment type="subcellular location"&gt;
  &lt;subcellularLocation&gt;
    &lt;location&gt;Cell membrane&lt;/location&gt;
    &lt;topology&gt;Lipid-anchor&lt;/topology&gt;
    &lt;topology&gt;GPI-anchor&lt;/topology&gt;
  &lt;/subcellularLocation&gt;
  &lt;subcellularLocation&gt;
    &lt;location&gt;Golgi apparatus&lt;/location&gt;
  &lt;/subcellularLocation&gt;
&lt;/comment&gt;
&lt;comment type="subcellular location"&gt;
  <code style="color: green;">&lt;molecule&gt;</code><code style="color: red;">Megakaryocyte-potentiating factor</code><code style="color: green;">&lt;/molecule&gt;</code>
  &lt;subcellularLocation&gt;
    &lt;location&gt;Secreted&lt;/location&gt;
  &lt;/subcellularLocation&gt;
&lt;/comment&gt;
&lt;comment type="subcellular location"&gt;
  <code style="color: green;">&lt;molecule&gt;</code><code style="color: red;">Isoform 3</code><code style="color: green;">&lt;/molecule&gt;</code>
  &lt;subcellularLocation&gt;
    &lt;location&gt;Secreted&lt;/location&gt;
  &lt;/subcellularLocation&gt;
&lt;/comment&gt;
</pre>
<p>
 <ins>
  cofactor
 </ins>
</p>
<p>
 Example:
 <a href="http://www.uniprot.org/uniprot/P26662.xml">
  P26662
 </a>
</p>
<pre>&lt;feature type="chain" <code style="color: green;">description="</code><code style="color: red;">Serine protease NS3</code><code style="color: green;">"</code>
                      <code style="color: green;">id="</code><code style="color: red;">PRO_0000037644</code><code style="color: green;">"</code> evidence="4"&gt;
  &lt;location&gt;
    &lt;begin position="1027"/&gt;
    &lt;end position="1657"/&gt;
  &lt;/location&gt;
&lt;/feature&gt;
&lt;feature type="chain" <code style="color: green;">description="</code><code style="color: red;">Non-structural protein 4A</code><code style="color: green;">"</code>
                      <code style="color: green;">id="</code><code style="color: red;">PRO_0000037645</code><code style="color: green;">"</code> evidence="4"&gt;
  &lt;location&gt;
    &lt;begin position="1658"/&gt;
    &lt;end position="1711"/&gt;
  &lt;/location&gt;
&lt;/feature&gt;
</pre>
<pre>&lt;comment type="cofactor"&gt;
  <code style="color: green;">&lt;molecule&gt;</code><code style="color: red;">Serine protease NS3</code><code style="color: green;">&lt;/molecule&gt;</code>
  &lt;cofactor evidence="14"&gt;
    &lt;name&gt;Zn(2+)&lt;/name&gt;
    &lt;dbReference type="ChEBI" id="CHEBI:29105"/&gt;
  &lt;/cofactor&gt;
  &lt;text evidence="14"&gt;Binds 1 zinc ion.&lt;/text&gt;
&lt;/comment&gt;
&lt;comment type="cofactor"&gt;
  <code style="color: green;">&lt;molecule&gt;</code><code style="color: red;">Non-structural protein 5A</code><code style="color: green;">&lt;/molecule&gt;</code>
  &lt;cofactor evidence="3"&gt;
    &lt;name&gt;Zn(2+)&lt;/name&gt;
    &lt;dbReference type="ChEBI" id="CHEBI:29105"/&gt;
  &lt;/cofactor&gt;
  &lt;text evidence="3"&gt;Binds 1 zinc ion in the NS5A N-terminal domain.&lt;/text&gt;
&lt;/comment&gt;
</pre>
<h5 id="xml_feature">
 featureType
</h5>
<p>
 The
 <code>
  featureType
 </code>
 has a mandatory
 <code>
  location
 </code>
 element of
 <code>
  locationType
 </code>
 to describe the sequence coordinates of an annotation.
</p>
<p>
 Example:
 <a href="http://www.uniprot.org/uniprot/P84077.xml">
  P84077
 </a>
</p>
<pre>&lt;feature type="initiator methionine" description="Removed" evidence="6 7 21 22"&gt;
  &lt;location&gt;
    &lt;position position="1"/&gt;
  &lt;/location&gt;
&lt;/feature&gt;
&lt;feature type="chain" description="ADP-ribosylation factor 1" id="PRO_0000207378"&gt;
  &lt;location&gt;
    &lt;begin position="2"/&gt;
    &lt;end position="181"/&gt;
  &lt;/location&gt;
&lt;/feature&gt;
...
&lt;feature type="nucleotide phosphate-binding region" description="GTP" evidence="1 2 3 4 25"&gt;
  &lt;location&gt;
    &lt;begin position="126"/&gt;
    &lt;end position="129"/&gt;
  &lt;/location&gt;
&lt;/feature&gt;
...
&lt;feature type="sequence variant" description="In PVNH8; decreased interaction with GGA3; dbSNP:rs879036238." id="VAR_081272" evidence="23"&gt;
  &lt;original&gt;Y&lt;/original&gt;
  &lt;variation&gt;H&lt;/variation&gt;
  &lt;location&gt;
    &lt;position position="35"/&gt;
  &lt;/location&gt;
&lt;/feature&gt;
...
&lt;feature type="helix" evidence="1"&gt;
  &lt;location&gt;
    &lt;begin position="6"/&gt;
    &lt;end position="9"/&gt;
  &lt;/location&gt;
&lt;/feature&gt;
</pre>
<p>
 The
 <code>
  locationType
 </code>
 has an optional
 <code>
  sequence
 </code>
 attribute that is only set (to an
 <a href="http://www.uniprot.org/help/alternative%5Fproducts#iso%5Fid">
  isoform ID
 </a>
 ) when the coordinates are not for the
 <a href="http://www.uniprot.org/help/canonical%5Fand%5Fisoforms">
  canonical sequence
 </a>
 .
</p>
<p>
 Example:
 <a href="http://www.uniprot.org/uniprot/P12821.xml">
  P12821
 </a>
</p>
<p>
 Previous representation:
</p>
<pre>&lt;comment type="alternative products"&gt;
  ...
  &lt;isoform&gt;
    <code style="color: green;">&lt;id&gt;</code><code style="color: red;">P12821-3</code><code style="color: green;">&lt;/id&gt;</code>
    &lt;id&gt;P22966-1&lt;/id&gt;
    <code style="color: green;">&lt;name&gt;</code><code style="color: red;">Testis-specific</code><code style="color: green;">&lt;/name&gt;</code>
    &lt;name&gt;ACE-T&lt;/name&gt;
    &lt;sequence type="described" ref="VSP_035120 VSP_035121"/&gt;
    &lt;text&gt;<code style="color: blue;">Variant in position: 32:S-&gt;P (in dbSNP:rs4317). Variant in position: 49:S-&gt;G (in dbSNP:rs4318)</code>.&lt;/text&gt;
  &lt;/isoform&gt;
  ...
&lt;/comment&gt;
...
&lt;feature type="sequence variant" description="In dbSNP:rs13306087." id="VAR_029139"&gt;
  &lt;original&gt;A&lt;/original&gt;
  &lt;variation&gt;T&lt;/variation&gt;
  &lt;location&gt;
    &lt;position position="154"/&gt;
  &lt;/location&gt;
&lt;/feature&gt;
</pre>
<p>
 New representation:
</p>
<pre>&lt;comment type="alternative products"&gt;
  ...
  &lt;isoform&gt;
    <code style="color: green;">&lt;id&gt;</code><code style="color: red;">P12821-3</code><code style="color: green;">&lt;/id&gt;</code>
    &lt;id&gt;P22966-1&lt;/id&gt;
    <code style="color: green;">&lt;name&gt;</code><code style="color: red;">Testis-specific</code><code style="color: green;">&lt;/name&gt;</code>
    &lt;name&gt;ACE-T&lt;/name&gt;
    &lt;sequence type="described" ref="VSP_035120 VSP_035121"/&gt;
  &lt;/isoform&gt;
  ...
&lt;/comment&gt;
...
&lt;feature type="sequence variant" description="In dbSNP:rs13306087." id="VAR_029139"&gt;
  &lt;original&gt;A&lt;/original&gt;
  &lt;variation&gt;T&lt;/variation&gt;
  &lt;location&gt;
    &lt;position position="154"/&gt;
  &lt;/location&gt;
&lt;/feature&gt;
...
&lt;feature type="sequence variant" description="In dbSNP:rs4317." id="VAR_x"&gt;
  &lt;original&gt;S&lt;/original&gt;
  &lt;variation&gt;P&lt;/variation&gt;
  &lt;location <code style="color: green;">sequence="</code><code style="color: red;">P12821-3</code><code style="color: green;">"</code>&gt;
    &lt;position position="32"/&gt;
  &lt;/location&gt;
&lt;/feature&gt;
&lt;feature type="sequence variant" description="In dbSNP:rs4318." id="VAR_y"&gt;
  &lt;original&gt;S&lt;/original&gt;
  &lt;variation&gt;G&lt;/variation&gt;
  &lt;location <code style="color: green;">sequence="</code><code style="color: red;">P12821-3</code><code style="color: green;">"</code>&gt;
    &lt;position position="49"/&gt;
  &lt;/location&gt;
&lt;/feature&gt;
</pre>
<h4 id="rdf_product">
 RDF format
</h4>
<p>
 The
 <a href="ftp://ftp.uniprot.org/pub/databases/uniprot/current%5Frelease/rdf/core.owl">
  UniProt RDF schema ontology
 </a>
 already allowed to describe the product to which an annotation applies and required no changes for this purpose.
</p>
<p>
 The RDF format has a single hierarchy of
 <a href="http://purl.uniprot.org/core/Annotation">
  <code>
   Annotation
  </code>
 </a>
 classes with various intermediary classes. The subclass
 <a href="http://purl.uniprot.org/core/Sequence%5FAnnotation">
  <code>
   Sequence_Annotation
  </code>
 </a>
 groups all classes that refer to a location on a protein sequence. This location is represented with
 <a href="https://doi.org/10.1186/s13326-016-0067-z">
  FALDO
 </a>
 and always indicates the FALDO
 <code>
  reference
 </code>
 sequence for the location (the RDF format makes no special case for a
 <a href="http://www.uniprot.org/help/canonical%5Fand%5Fisoforms">
  canonical sequence
 </a>
 ). Annotations that do not refer to a specific location on a protein sequence, but that apply to a given product, describe the sequence of this product with a
 <a href="http://purl.uniprot.org/core/sequence">
  <code>
   sequence
  </code>
 </a>
 property. The object of this property may be a
 <a href="http://purl.uniprot.org/core/Sequence">
  <code>
   Sequence
  </code>
 </a>
 or a
 <a href="http://purl.uniprot.org/core/Chain%5FAnnotation">
  <code>
   Chain_Annotation
  </code>
 </a>
 /
 <a href="http://purl.uniprot.org/core/Peptide%5FAnnotation">
  <code>
   Peptide_Annotation
  </code>
 </a>
 that describes a sequence that is the product of proteolytic processing.
</p>
<p>
 Please note that the change of
 <a href="http://www.uniprot.org#mass_spectrometry">
  mass spectrometry
 </a>
 annotations required an adaptation of the hierarchy of
 <a href="http://purl.uniprot.org/core/Annotation">
  <code>
   Annotation
  </code>
 </a>
 classes: The
 <a href="http://purl.uniprot.org/core/Mass%5FSpectrometry%5FAnnotation">
  <code>
   Mass_Spectrometry_Annotation
  </code>
 </a>
 class no longer is an
 <code>
  rdfs:subClassOf
 </code>
 of the
 <a href="http://purl.uniprot.org/core/Sequence%5FAnnotation">
  <code>
   Sequence_Annotation
  </code>
 </a>
 class, but a direct
 <code>
  rdfs:subClassOf
 </code>
 of the
 <a href="http://purl.uniprot.org/core/Annotation">
  <code>
   Annotation
  </code>
 </a>
 class.
</p>
<p>
 Example:
 <a href="http://www.uniprot.org/uniprot/Q13421.ttl">
  Q13421
 </a>
</p>
<pre>@prefix up: &lt;http://purl.uniprot.org/core/&gt; .
@prefix uniprot: &lt;http://purl.uniprot.org/uniprot/&gt; .
@prefix isoform: &lt;http://purl.uniprot.org/isoforms/&gt; .
@prefix annotation: &lt;http://purl.uniprot.org/annotation/&gt; .
@prefix faldo: &lt;http://biohackathon.org/resource/faldo#&gt; .

uniprot:Q13421
  <code style="color: green;">up:annotation</code>
    <code style="color: red;">annotation:PRO_0000253560</code> ,
    &lt;Q13421#SIPADAC7D651EFC09CC&gt; ,
    &lt;Q13421#SIP307BEB951103B073&gt; ,
    &lt;Q13421#SIPB6746E472B99B031&gt; ,
    ...
  <code style="color: green;">up:sequence</code>
    <code style="color: red;">isoform:Q13421-1</code> ,
    <code style="color: red;">isoform:Q13421-3</code> ,
    <code style="color: red;">isoform:Q13421-2</code> ,
    <code style="color: red;">isoform:Q13421-4</code> ;

<code style="color: red;">annotation:PRO_0000253560</code>
  rdf:type up:Chain_Annotation ;
  rdfs:comment "<code style="color: red;">Megakaryocyte-potentiating factor</code>" ;
  up:range range:22853569102360878tt37tt286 .
range:22853569102360878tt37tt286
  rdf:type faldo:Region ;
  faldo:begin position:22853569102360878tt37 ;
  faldo:end position:22853569102360878tt286 .
position:22853569102360878tt37
  rdf:type faldo:Position , faldo:ExactPosition ;
  <code style="color: green;">faldo:position</code> <code style="color: red;">37</code> ;
  <code style="color: green;">faldo:reference</code> <code style="color: red;">isoform:Q13421-1</code> .
position:22853569102360878tt286
  rdf:type faldo:Position , faldo:ExactPosition ;
  <code style="color: green;">faldo:position</code> <code style="color: red;">286</code> ;
  <code style="color: green;">faldo:reference</code> <code style="color: red;">isoform:Q13421-1</code> .

&lt;Q13421#SIPADAC7D651EFC09CC&gt;
  rdf:type up:Subcellular_Location_Annotation ;
  up:locatedIn &lt;Q13421#SIP04927440DF8EB941&gt; ,
               &lt;Q13421#SIP727DF431EB6C89EC&gt; .

&lt;Q13421#SIP307BEB951103B073&gt;
  rdf:type up:Subcellular_Location_Annotation ;
  up:locatedIn &lt;Q13421#SIPD59D33F5047A94FD&gt; ;
  <code style="color: green;">up:sequence</code> <code style="color: red;">annotation:PRO_0000253560</code> .

&lt;Q13421#SIPB6746E472B99B031&gt;
  rdf:type up:Subcellular_Location_Annotation ;
  up:locatedIn &lt;Q13421#SIPD59D33F5047A94FD&gt; ;
  <code style="color: green;">up:sequence</code> <code style="color: red;">isoform:Q13421-2</code> .

<code style="color: red;">isoform:Q13421-1</code>
  rdf:type up:Simple_Sequence ;
  up:modified "2006-10-17"^^xsd:date ;
  up:version 2 ;
  up:precursor true ;
  up:mass 68986 ;
  up:crc64Checksum "FA17E3609B6CC9CA"^^xsd:token ;
  up:name "1" ;
  rdf:value "MALPTARPLLGSCGTPALGSLLFLLFSL ... LLASTLA" .
<code style="color: red;">isoform:Q13421-3</code>
  rdf:type up:Modified_Sequence ;
  up:name "2" ;
  up:basedOn isoform:Q13421-1 ;
  up:modification annotation:VSP_021059 .
  rdf:value "MALPTARPLLGSCGTPALGSLLFLLFSL ... LLASTLA" ;
<code style="color: red;">isoform:Q13421-2</code>
  rdf:type up:Modified_Sequence ;
  up:name "3" , "SMRP" ;
  up:basedOn isoform:Q13421-1 ;
  up:modification annotation:VSP_021059 , annotation:VSP_021060 .
  rdf:value "MALPTARPLLGSCGTPALGSLLFLLFSL ... LRAPLPC" ;
<code style="color: red;">isoform:Q13421-4</code>
  rdf:type up:Modified_Sequence ;
  up:name "4" ;
  up:basedOn isoform:Q13421-1 ;
  up:modification annotation:VSP_021058 , annotation:VSP_021059 .
  rdf:value "MALPTARPLLGSCGTPALGSLLFLLFSL ... LLASTLA" ;
</pre>
<h3 id="Cross_references_to_RNAct">
 Cross-references to RNAct
</h3>
<p>
 Cross-references have been added to RNAct, a database of protein-RNA interaction predictions for model organisms with supporting experimental data.
</p>
<p>
 RNAct is available at
 <a href="https://rnact.crg.eu">
  https://rnact.crg.eu
 </a>
 .
</p>
<p>
 The format of the explicit links is:
</p>
<table>
 <tr>
  <th>
   Resource abbreviation
  </th>
  <td>
   RNAct
  </td>
 </tr>
 <tr>
  <th>
   Resource identifier
  </th>
  <td>
   UniProtKB accession number
  </td>
 </tr>
 <tr>
  <th>
   Optional information 1
  </th>
  <td>
   Molecule type
  </td>
 </tr>
</table>
<p>
 Example:
 <a href="http://www.uniprot.org/uniprot/Q9Y2I1">
  Q9Y2I1
 </a>
</p>
<p>
 <a href="http://www.uniprot.org/uniprot/?query=database:(type:rnact)&amp;sort=score">
  Show all entries having a cross-reference to RNAct.
 </a>
</p>
<h4 id="Text_format">
 Text format
</h4>
<p>
 Example:
 <a href="http://www.uniprot.org/uniprot/Q9Y2I1.txt">
  Q9Y2I1
 </a>
</p>
<pre>DR   RNAct; Q9Y2I1; protein.
</pre>
<h4 id="XML_format">
 XML format
</h4>
<p>
 Example:
 <a href="http://www.uniprot.org/uniprot/Q9Y2I1.xml">
  Q9Y2I1
 </a>
</p>
<pre>&lt;dbReference type="RNAct" id="Q9Y2I1"&gt;
   &lt;property type="molecule type" value="protein"/&gt;
&lt;/dbreference&gt;
</pre>
<h4 id="RDF_format">
 RDF format
</h4>
<p>
 Example:
 <a href="http://www.uniprot.org/uniprot/Q9Y2I1.ttl">
  Q9Y2I1
 </a>
</p>
<pre>uniprot:Q9Y2I1
  rdfs:seeAlso &lt;http://purl.uniprot.org/rnact/Q9Y2I1&gt; .
&lt;http://purl.uniprot.org/rnact/Q9Y2I1&gt;
  rdf:type up:Resource ;
  up:database &lt;http://purl.uniprot.org/database/RNAct&gt; ;
  rdfs:comment "protein" .
</pre>
<h3 id="Change_of_the_cross_references_to_Pharos">
 Change of the cross-references to Pharos
</h3>
<p>
 We have introduced an additional field in the cross-references to the
 <a href="https://pharos.nih.gov">
  Pharos
 </a>
 database to indicate the development status of a target. Targets are categorized into four
 <a href="http://juniper.health.unm.edu/tcrd/">
  development/druggability levels
 </a>
 (TDLs), ranging from
 <em>
  Tclin
 </em>
 for approved drugs with known mechanisms of action, to
 <em>
  Tdark
 </em>
 for targets about which virtually nothing is known.
</p>
<h4 id="Text_format">
 Text format
</h4>
<p>
 Example:
 <a href="http://www.uniprot.org/uniprot/P33151.txt">
  P33151
 </a>
</p>
<pre>DR   Pharos; P33151; Tbio.
</pre>
<h4 id="XML_format">
 XML format
</h4>
<p>
 Example:
 <a href="http://www.uniprot.org/uniprot/P33151.xml">
  P33151
 </a>
</p>
<pre>&lt;dbReference type="Pharos" id="P33151"&gt;
  &lt;property type="development level" value="Tbio"/&gt;
&lt;/dbReference&gt;
</pre>
<p>
 This change does not affect the XSD, but may nevertheless require code changes.
</p>
<h4 id="RDF_format">
 RDF format
</h4>
<p>
 Example:
 <a href="http://www.uniprot.org/uniprot/P33151.ttl">
  P33151
 </a>
</p>
<pre>uniprot:P33151
  rdfs:seeAlso &lt;http://purl.uniprot.org/pharos/P33151&gt; .
&lt;http://purl.uniprot.org/pharos/P33151&gt;
  rdf:type up:Resource ;
  up:database &lt;http://purl.uniprot.org/database/Pharos&gt; ;
  rdfs:comment "Tbio" .
</pre>
<h3 id="Removal_of_the_cross_references_to_PMAP_CutDB">
 Removal of the cross-references to PMAP-CutDB
</h3>
<p>
 Cross-references to PMAP-CutDB have been removed.
</p>
<h3 id="Changes_to_the__a_href___docs_humdisease__controlled_vocabulary_of_human_diseases__a_">
 Changes to the
 <a href="http://www.uniprot.org/docs/humdisease">
  controlled vocabulary of human diseases
 </a>
</h3>
<p>
 New diseases:
</p>
<ul>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-05648">
   Congenital disorder of glycosylation 1CC
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-05665">
   Epilepsy, idiopathic generalized 16
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-05652">
   Epileptic encephalopathy, early infantile, 78
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-05653">
   Epileptic encephalopathy, early infantile, 79
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-05661">
   Epileptic encephalopathy, early infantile, 80
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-05663">
   Glycosylphosphatidylinositol biosynthesis defect 21
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-05659">
   Hypothyroidism, congenital, non-goitrous, 7
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-05650">
   Hypothyroidism, congenital, non-goitrous, 8
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-05651">
   Hypothyroidism, congenital, non-goitrous, 9
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-05662">
   Intellectual developmental disorder 60 with seizures
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-05649">
   Mental retardation, X-linked, syndromic, 36
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-05654">
   Mitochondrial DNA depletion syndrome 17
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-05660">
   Myopathy, congenital, progressive, with scoliosis
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-05664">
   Nephrotic syndrome 21
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-05655">
   Neurodevelopmental disorder with ataxia, hypotonia, and microcephaly
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-05656">
   Neurodevelopmental disorder with cataracts, poor growth, and dysmorphic facies
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-05657">
   Neurodevelopmental disorder with cerebellar hypoplasia and spasticity
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-05667">
   Neurodevelopmental disorder with hypotonia and variable intellectual and behavioral abnormalities
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-05658">
   Neurodevelopmental disorder with structural brain anomalies and dysmorphic facies
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-05666">
   Spastic tetraplegia and axial hypotonia, progressive
  </a>
 </li>
</ul>
<h3 id="Changes_to_the__a_href___docs_ptmlist__controlled_vocabulary_for_PTMs__a_">
 Changes to the
 <a href="http://www.uniprot.org/docs/ptmlist">
  controlled vocabulary for PTMs
 </a>
</h3>
<p>
 New term for the feature key 'Cross-link' ('CROSSLNK' in the flat file):
</p>
<ul>
 <li>
  6-(S-cysteinyl)-8alpha-(pros-histidyl)-FAD (Cys-His)
 </li>
</ul>
<h3 id="Changes_to__a_href___docs_keywlist__keywords__a_">
 Changes to
 <a href="http://www.uniprot.org/docs/keywlist">
  keywords
 </a>
</h3>
<p>
 Deleted keyword:
</p>
<ul>
 <li>
  Complete proteome
 </li>
</ul>
<h2 id="Proteomes_changes">
 Proteomes changes
</h2>
<p>
 The UniProt
 <a href="http://www.uniprot.org/proteomes/">
  Proteomes portal
 </a>
 is offering protein sequence sets obtained from the translation of sequenced genomes. Published genomes from
 <a href="https://www.ncbi.nlm.nih.gov/genome">
  NCBI Genome
 </a>
 used to be brought into UniProt if they satisfied the following criteria:
</p>
<ul>
 <li>
  The genome is annotated and a set of coding sequences is available.
 </li>
 <li>
  The number of predicted coding sequences falls within a statistically significant range of published proteomes from neighbouring species.
 </li>
</ul>
<p>
 We have changed these criteria to publish all proteomes that can be derived from NCBI genomes that are not considered to be low quality assemblies. We now use a subset of the
 <a href="https://www.ncbi.nlm.nih.gov/assembly/help/anomnotrefseq/">
  RefSeq reasons to exclude a genome assembly
 </a>
 to determine which proteomes to bring into UniProtKB and we give the reason(s) why a proteome is excluded from UniProtKB. We also provide
 <a href="http://www.uniprot.org/help/assessing%5Fproteomes">
  two metrics
 </a>
 to help users to assess the quality of a proteome:
</p>
<ul>
 <li>
  A score obtained with the
  <a href="https://busco.ezlab.org">
   BUSCO
  </a>
  software.
 </li>
 <li>
  A score based on the number of coding sequences expected based on neighbouring species, "Complete Proteome Detector (CPD)".
 </li>
</ul>
<p>
 The "Complete proteome" keyword was removed from all UniProtKB entries. Individual proteomes can be retrieved from the UniProt website by their unique proteome identifier, e.g.
 <a href="http://www.uniprot.org/proteomes/UP000005640">
  UP000005640
 </a>
 .
</p>