<h2 id="Headline">
 Headline
</h2>
<h3 id="Inflammation__the_Good__the_Bad_and_the_Ugly">
 Inflammation: the Good, the Bad and the Ugly
</h3>
<p>
 In a world filled with (microbial) foes, inflammation is our best friend, helping us fight against invading microorganisms, promoting wound healing and tissue regeneration. When it gets out of hand, however, it becomes our worst enemy. For instance, inflammation is required for mice to fight
 <a href="https://viralzone.expasy.org/6?outline=all%5Fby%5Fspecies">
  Influenza virus A
 </a>
 (IAV) infections, but
 <a href="https://www.ncbi.nlm.nih.gov/pubmed/32200799">
  animals challenged with high doses of virus
 </a>
 experience elevated levels of inflammation, destruction of airway epithelia and eventually die, unless they are deficient in
 <a href="http://www.uniprot.org/uniprot/?query=accession:Q8NB16+OR+accession:Q9D2Y4">
  MLKL protein
 </a>
 , an endogenous protein that induces cell death in response to orthomyxovirus infection.
</p>
<p>
 <a href="https://www.ncbi.nlm.nih.gov/pubmed/32200799">
  The mechanism leading to this fatal issue has been recently elucidated
 </a>
 . When IAV infects cells and replicates, newly formed IAV RNA duplexes can adopt the Z-confirmation. These very peculiar RNAs are recognized by
 <a href="http://www.uniprot.org/uniprot/?query=accession:Q9H171+OR+accession:Q9QY24+OR+accession:Q8VDA5">
  ZBP1 protein
 </a>
 as a pathogen-associated molecular pattern (PAMP) and initiate a cascade of reactions. Activated ZBP1 recruits
 <a href="http://www.uniprot.org/uniprot/?query=accession:Q9Y572+OR+accession:Q9QZL0+OR+accession:Q9Z2P5">
  RIPK3
 </a>
 , which in turn
 <a href="https://pubmed.ncbi.nlm.nih.gov/27321907/">
  can induce one of two forms of programmed cell death
 </a>
 , apoptosis and pro-inflammatory necroptosis. In the presence of RIPK1, apoptosis is favored. In the absence of RIPK1, RIPK3 phosphorylates
 <a href="http://www.uniprot.org/uniprot/?query=accession:Q8NB16+OR+accession:Q9D2Y4">
  MLKL
 </a>
 , which then induces necroptosis. Programmed cell death is an effective mechanism of
 <a href="https://www.ncbi.nlm.nih.gov/pubmed/32200799">
  IAV clearance that not only eliminates infected cells to limit virus spread but also serves to catalyze adaptive immune responses
 </a>
 . When cell death is unrestrained, or when the mode of cell death is primarily necrotic, then injury and severe illness ensue, despite virus clearance.
</p>
<p>
 The dark side of inflammation also includes chronic inflammatory diseases and although progress has been made in this field, the causes that initiate pathogenic inflammatory responses still remain elusive. Could ZBP1 be a player in this deadly game? In other words, can ZBP1 be activated in the absence of any viral infections? An answer to these questions came from a recent publication by
 <a href="https://www.ncbi.nlm.nih.gov/pubmed/32296175">
  Jiao et al.
 </a>
 . The authors showed that endogenous retroelements (EREs) produce double-stranded RNAs (dsRNAs) adopting a Z-confirmation. These dsRNAs are recognized by ZBP1 and activate it. EREs derive either from ancient retroviral infections or from active retrotransposons and constitute over half of the human and mouse genomes. Most dsRNAs identified by Jiao et al. originate from B2 and Alu short interspersed nuclear elements, followed by long interspersed nuclear elements (specifically, L1 elements) and long terminal repeat elements. Hence, these old mates of ours may still be able to mimic viral infections. Indeed in RIPK1 knockout mice, deregulation of ERE expression may trigger ZBP1-dependent skin inflammation. This mechanism may be relevant for the pathogenesis of inflammatory pathologies in humans, particularly in patients with variations in proteins that inhibit the ZBP1-MLKL-induced necroptotic pathway, such as RIPK1.
</p>
<p>
 As of this release,
 <a href="http://www.uniprot.org/uniprot/?query=accession:Q9H171+OR+accession:Q9QY24+OR+accession:Q8VDA5">
  ZBP1
 </a>
 ,
 <a href="http://www.uniprot.org/uniprot/?query=accession:Q9Y572+OR+accession:Q9QZL0+OR+accession:Q9Z2P5">
  RIPK3
 </a>
 , and
 <a href="http://www.uniprot.org/uniprot/?query=accession:Q8NB16+OR+accession:Q9D2Y4">
  MLKL
 </a>
 have been updated and are available in UniProtKB/Swiss-Prot.
</p>
<h2 id="UniProtKB_news">
 UniProtKB news
</h2>
<h3 id="Cross_references_to_GlyGen">
 Cross-references to GlyGen
</h3>
<p>
 Cross-references have been added to the GlyGen Computational and Informatics Resource for Glycoscience. GlyGen retrieves information from multiple international data sources, integrates and harmonizes this data and makes it searchable.
</p>
<p>
 GlyGen is available at
 <a href="https://www.glygen.org">
  https://www.glygen.org
 </a>
 .
</p>
<p>
 The format of the explicit links is:
</p>
<table>
 <tr>
  <th>
   Resource abbreviation
  </th>
  <td>
   GlyGen
  </td>
 </tr>
 <tr>
  <th>
   Resource identifier
  </th>
  <td>
   UniProtKB accession number
  </td>
 </tr>
 <tr>
  <th>
   Optional information 1
  </th>
  <td>
   Glycosylation details
  </td>
 </tr>
</table>
<p>
 Example:
 <a href="http://www.uniprot.org/uniprot/P07766">
  P07766
 </a>
</p>
<p>
 <a href="http://www.uniprot.org/uniprot/?query=database:glygen&amp;sort=score">
  Show all entries having a cross-reference to GlyGen.
 </a>
</p>
<h4 id="Text_format">
 Text format
</h4>
<p>
 Example:
 <a href="http://www.uniprot.org/uniprot/P14210.txt">
  P14210
 </a>
</p>
<pre>DR   GlyGen; P14210; 5 sites, 8 N-linked glycans (4 sites), 1 O-linked glycan (1 site).
</pre>
<h4 id="XML_format">
 XML format
</h4>
<p>
 Example:
 <a href="http://www.uniprot.org/uniprot/P14210.xml">
  P14210
 </a>
</p>
<pre>&lt;dbReference type="GlyGen" id="P14210"&gt;
  &lt;property type="glycosylation" value="5 sites, 8 N-linked glycans (4 sites), 1 O-linked glycan (1 site)"/&gt;
&lt;/dbReference&gt;
</pre>
<h4 id="RDF_format">
 RDF format
</h4>
<p>
 Example:
 <a href="http://www.uniprot.org/uniprot/P14210.ttl">
  P14210
 </a>
</p>
<pre>uniprot:P07766
  rdfs:seeAlso &lt;http://purl.uniprot.org/glygen/P14210&gt; .
&lt;http://purl.uniprot.org/glygen/P14210&gt;
  rdf:type up:Resource ;
  up:database &lt;http://purl.uniprot.org/database/GlyGen&gt; ;
  rdfs:comment "5 sites, 8 N-linked glycans (4 sites), 1 O-linked glycan (1 site)" .
</pre>
<h3 id="Cross_references_to_PathwayCommons">
 Cross-references to PathwayCommons
</h3>
<p>
 Cross-references have been added to the PathwayCommons, a resource that aims to collect and disseminate biological pathway and interaction data. Data is collected from partner databases and is represented in the BioPAX standard. By representing data in BioPAX, Pathway Commons is able to provide a detailed representation of a variety of biological concepts including: Biochemical reactions; gene regulatory networks; and genetic interactions; transport and catalysis events; and physical interactions involving proteins, DNA, RNA and small molecules and complexes.
</p>
<p>
 PathwayCommons is available at
 <a href="http://www.pathwaycommons.org">
  http://www.pathwaycommons.org
 </a>
 .
</p>
<p>
 The format of the explicit links is:
</p>
<table>
 <tr>
  <th>
   Resource abbreviation
  </th>
  <td>
   PathwayCommons
  </td>
 </tr>
 <tr>
  <th>
   Resource identifier
  </th>
  <td>
   UniProtKB accession number
  </td>
 </tr>
</table>
<p>
 Example:
 <a href="http://www.uniprot.org/uniprot/P01042">
  P01042
 </a>
</p>
<p>
 <a href="http://www.uniprot.org/uniprot/?query=database:pathwaycommons&amp;sort=score">
  Show all entries having a cross-reference to PathwayCommons.
 </a>
</p>
<h4 id="Text_format">
 Text format
</h4>
<p>
 Example:
 <a href="http://www.uniprot.org/uniprot/P01042.txt">
  P01042
 </a>
</p>
<pre>DR   PathwayCommons; P01042; -.
</pre>
<h4 id="XML_format">
 XML format
</h4>
<p>
 Example:
 <a href="http://www.uniprot.org/uniprot/P01042.xml">
  P01042
 </a>
</p>
<pre>&lt;dbReference type="PathwayCommons" id="P01042"/&gt;
</pre>
<h4 id="RDF_format">
 RDF format
</h4>
<p>
 Example:
 <a href="http://www.uniprot.org/uniprot/P01042.ttl">
  P01042
 </a>
</p>
<pre>uniprot:P01042
  rdfs:seeAlso &lt;http://purl.uniprot.org/pathwaycommons/P01042&gt; .
&lt;http://purl.uniprot.org/pathwaycommons/P01042&gt;
  rdf:type up:Resource ;
  up:database &lt;http://purl.uniprot.org/database/PathwayCommons&gt; .
</pre>
<h3 id="Change_to_the_cross_references_to_GlyConnect">
 Change to the cross-references to GlyConnect
</h3>
<p>
 We have introduced an additional field in the cross-references to the
 <a href="https://glyconnect.expasy.org">
  GlyConnect protein glycosylation database
 </a>
 , a platform integrating sources of information to help characterize the molecular components of protein glycosylation. This allows us to provide some additional information, e.g. about the nature of the glycans and the number of glycosylation sites.
</p>
<p>
 The format of the explicit links is:
</p>
<table>
 <tr>
  <th>
   Resource abbreviation
  </th>
  <td>
   GlyConnect
  </td>
 </tr>
 <tr>
  <th>
   Resource identifier
  </th>
  <td>
   Resource identifier
  </td>
 </tr>
 <tr>
  <th>
   Optional information 1
  </th>
  <td>
   Glycosylation details
  </td>
 </tr>
</table>
<h4 id="Text_format">
 Text format
</h4>
<p>
 Example:
 <a href="http://www.uniprot.org/uniprot/P12763.txt">
  P12763
 </a>
</p>
<p>
 Previous format:
</p>
<pre>DR   GlyConnect; 22; -.
</pre>
<p>
 New format:
</p>
<pre>DR   GlyConnect; 22; 49 N-Linked glycans (3 sites), 13 O-Linked glycans (6 sites).
</pre>
<h4 id="XML_format">
 XML format
</h4>
<p>
 Example:
 <a href="http://www.uniprot.org/uniprot/P12763.xml">
  P12763
 </a>
</p>
<p>
 Previous format:
</p>
<pre>&lt;dbReference type="GlyConnect" id="22"/&gt;
</pre>
<p>
 New format:
</p>
<pre>&lt;dbReference type="GlyConnect" id="22"&gt;
  &lt;property type="glycosylation" value="49 N-Linked glycans (3 sites), 13 O-Linked glycans (6 sites)"/&gt;
&lt;/dbReference&gt;
</pre>
<p>
 This change does not affect the XSD, but may nevertheless require code changes.
</p>
<h4 id="RDF_format">
 RDF format
</h4>
<p>
 Example:
 <a href="http://www.uniprot.org/uniprot/P12763.ttl">
  P12763
 </a>
</p>
<p>
 Previous format
</p>
<pre>uniprot:P12763
  rdfs:seeAlso &lt;http://purl.uniprot.org/glyconnect/22&gt; .
&lt;http://purl.uniprot.org/glyconnect/22&gt;
  rdf:type &lt;http://purl.uniprot.org/core/Resource&gt; ;
  &lt;http://purl.uniprot.org/core/database&gt; &lt;http://purl.uniprot.org/database/GlyConnect&gt; .
</pre>
<p>
 New format:
</p>
<pre>uniprot:P12763
  rdfs:seeAlso &lt;http://purl.uniprot.org/glyconnect/22&gt; .
&lt;http://purl.uniprot.org/glyconnect/22&gt;
  rdf:type up:Resource ;
  up:database &lt;http://purl.uniprot.org/database/GlyConnect&gt; ;
  rdfs:comment "49 N-Linked glycans (3 sites), 13 O-Linked glycans (6 sites)" .
</pre>
<h3 id="New_automatic_annotation_system_ARBA__Association_Rule_Based_Annotator_">
 New automatic annotation system ARBA (Association-Rule-Based Annotator)
</h3>
<p>
 In the UniProt
 <a href="http://www.uniprot.org/help/automatic%5Fannotation">
  Automatic Annotation pipeline
 </a>
 which enhances unreviewed (UniProtKB/TrEMBL) records by enriching them with automatic classification and annotation, the SAAS (Statistical Automatic Annotation System) component has been superseded by the more powerful
 <a href="http://www.uniprot.org/help/arba">
  ARBA (Association-Rule-Based Annotator)
 </a>
 . ARBA is a multiclass learning system trained on expertly annotated entries in UniProtKB/Swiss-Prot. ARBA uses rule mining techniques to generate concise annotation models with the highest representativeness and coverage for annotation, based on the properties of
 <a href="https://www.ebi.ac.uk/interpro">
  InterPro
 </a>
 group membership and taxonomy. ARBA employs a data exclusion set that censors data not suitable for computational annotation (such as specific biophysical or chemical properties) and generates human-readable rules for each release. ARBA rules can be
 <a href="http://www.uniprot.org/arba">
  browsed and searched
 </a>
 via the website.
</p>
<h3 id="Changes_to_the__a_href___docs_humdisease__controlled_vocabulary_of_human_diseases__a_">
 Changes to the
 <a href="http://www.uniprot.org/docs/humdisease">
  controlled vocabulary of human diseases
 </a>
</h3>
<p>
 New diseases:
</p>
<ul>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-05803">
   46,XY sex reversal 11
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-05819">
   Al-Gazali syndrome
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-05812">
   Alopecia-mental retardation syndrome 4
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-05799">
   Anauxetic dysplasia 3
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-05817">
   Autoinflammation with episodic fever and lymphadenopathy
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-05796">
   Bone marrow failure syndrome 6
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-05808">
   Combined oxidative phosphorylation deficiency 40
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-05809">
   Combined oxidative phosphorylation deficiency 41
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-05810">
   Combined oxidative phosphorylation deficiency 42
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-05811">
   Combined oxidative phosphorylation deficiency 43
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-05820">
   Cone-rod dystrophy 14
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-05804">
   Congenital disorder of glycosylation 2R
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-05814">
   Diets-Jongmans syndrome
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-05807">
   Epilepsy, early-onset, with or without developmental delay
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-05802">
   Epileptic encephalopathy, early infantile, 85, with or without midline brain defects
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-05791">
   Genitourinary and/or brain malformation syndrome
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-05801">
   Holoprosencephaly 13, X-linked
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-05797">
   Hypervalinemia and hyperleucine-isoleucinemia
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-05798">
   Hypogonadotropic hypogonadism 25 with anosmia
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-05815">
   Immunodeficiency 66
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-05788">
   Intellectual developmental disorder with poor growth and with or without seizures or ataxia
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-05790">
   Mitochondrial DNA depletion syndrome 18
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-05816">
   Muscular dystrophy, limb-girdle, autosomal recessive 26
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-05793">
   Myopathy, congenital, with respiratory insufficiency and bone fractures
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-05794">
   Myopathy, congenital proximal, with minicore lesions
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-05805">
   Nabais Sa-de Vries syndrome 1
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-05806">
   Nabais Sa-de Vries syndrome 2
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-05789">
   Pontocerebellar hypoplasia, hypotonia, and respiratory insufficiency syndrome, neonatal lethal
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-05784">
   Respiratory papillomatosis, juvenile recurrent, congenital
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-05818">
   Retinal dystrophy with leukodystrophy
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-05792">
   Rhizomelic limb shortening with dysmorphic features
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-05785">
   Sandestig-Stefanova syndrome
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-05783">
   Spinocerebellar ataxia, autosomal recessive, 28
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-05795">
   T-cell immunodeficiency with thymic aplasia
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-05787">
   T-cell lymphopenia, infantile, with or without nail dystrophy, autosomal dominant
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-05786">
   Triokinase and FMN cyclase deficiency syndrome
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-05813">
   Vertebral, cardiac, renal, and limb defects syndrome 3
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-05800">
   Wieacker-Wolff syndrome, female-restricted
  </a>
 </li>
</ul>
<h3 id="Changes_to_the__a_href___docs_ptmlist__controlled_vocabulary_for_PTMs__a_">
 Changes to the
 <a href="http://www.uniprot.org/docs/ptmlist">
  controlled vocabulary for PTMs
 </a>
</h3>
<p>
 New term for the feature key 'Cross-link' ('CROSSLNK' in the flat file):
</p>
<ul>
 <li>
  Isoaspartyl glycine isopeptide (Gly-Asp)
 </li>
</ul>
<p>
 New terms for the feature key 'Modified residue' ('MOD_RES' in the flat file):
</p>
<ul>
 <li>
  2-oxo-5,5-dimethylhexanoate
 </li>
 <li>
  3-methylisoleucine
 </li>
 <li>
  3-methylvaline
 </li>
 <li>
  3-methyl-D-valine
 </li>
 <li>
  N4-methyl-D-asparagine
 </li>
 <li>
  3-hydroxyvaline (Thr)
 </li>
 <li>
  (3S)-3-methylglutamine
 </li>
 <li>
  3-hydroxy-D-valine
 </li>
 <li>
  (3R)-N4-methyl-3-hydroxy-D-asparagine
 </li>
 <li>
  3,3-dimethylmethionine
 </li>
 <li>
  Diphosphoserine
 </li>
 <li>
  Diphosphothreonine
 </li>
</ul>
<p>
 Modified terms for the feature key 'Modified residue' ('MOD_RES' in the flat file):
</p>
<ul>
 <li>
  Lactic acid -&gt; D-lactate
 </li>
 <li>
  3-hydroxyvaline -&gt; 3-hydroxyvaline (Val)
 </li>
</ul>