<h3 id="_Almost__all_about_that_CBASS">
 (Almost) all about that CBASS
</h3>
<p>
 Bacteria and archaea, like every other living organisms, must defend themselves against viral (bacteriophage or phage) attack. It is estimated that there are
 <a href="https://pubmed.ncbi.nlm.nih.gov/16163346">
  10
  <sup>
   31
  </sup>
  bacteriophages on the planet
 </a>
 found in all habitats; sea water, animal gastrointestinal tracts, oceanic basement... A number of anti-phage defense mechanisms are known: restriction nucleases, an Argonaute-like system, CRISPR, and more are being discovered as we explore less well-known archaea and bacteria. A new phage defense mechanism has been recently discovered and is being characterized, the
 <a href="https://pubmed.ncbi.nlm.nih.gov/32839535">
  CBASS system
 </a>
 (Cyclic oligonucleotide-Based Anti-phage Signalling System).
</p>
<p>
 At the heart of each CBASS system is a cyclic nucleotide synthase. These belong to the cGAS/DncV-like nucleotidyltransferase family (abbreviated CD-NTase), and
 <a href="https://pubmed.ncbi.nlm.nih.gov/31932165,30837338,30787435,32877915,32544385">
  they make cyclic di-, tri-, and possibly even tetranucleotides
 </a>
 . The cyclic nucleotides activate an effector protein, encoded adjacently to the CD-NTase. The effectors have a variety of activities: nucleases that degrade all DNA, including viral DNA (
 <a href="https://pubmed.ncbi.nlm.nih.gov/31932164,31932165">
  NucC
 </a>
 ,
 <a href="https://pubmed.ncbi.nlm.nih.gov/32544385">
  Cap4
 </a>
 ), an NAD+ hydrolase that presumably depletes cellular NAD+ (
 <a href="https://pubmed.ncbi.nlm.nih.gov/32877915">
  Cap12
 </a>
 ), phospholipases that are capable of degrading the cell membrane (
 <a href="https://pubmed.ncbi.nlm.nih.gov/31533127,29891656,30787435/">
  CapV
 </a>
 ,
 <a href="https://pubmed.ncbi.nlm.nih.gov/30787435">
  CapE
 </a>
 ), and transmembrane proteins that may form pores (
 <a href="https://pubmed.ncbi.nlm.nih.gov/32877915">
  Cap13
 </a>
 ) have all been observed, although not all have been characterized yet.
</p>
<p>
 Transforming CBASS into bacteria without a natural CBASS system
 <a href="https://pubmed.ncbi.nlm.nih.gov/31932165,31533127,32544385">
  protects the population against phages
 </a>
 . The effectors lead to cell death before the phage have completed an infection cycle, thus protecting adjacent uninfected cells from infection. There are at least 4 types of CBASS systems,
 <a href="https://pubmed.ncbi.nlm.nih.gov/32839535">
  classified on basis of the presence or absence of ancillary genes encoded in the same locus
 </a>
 . Type I systems have no extra genes. In type II systems, the ancillary proteins Cap2 and Cap3 are not necessary for protection against all phages, but
 <a href="https://pubmed.ncbi.nlm.nih.gov/31533127">
  they enlarge the range of phages against which the system is active
 </a>
 . In type III systems, the ancillary proteins Cap6 and Cap7 control the activity of the system; Cap7 is required to activate the CD-NTase, while Cap6 prevents the Cap7-CD-NTase association. A subtype has a second Cap7-like gene called Cap8
 <a href="https://pubmed.ncbi.nlm.nih.gov/31932165">
  which may act as a scaffold for complex assembly
 </a>
 . Type IV systems are rare and so far uncharacterized. They occur mostly in Firmicutes and archaea.
</p>
<p>
 At the request of Dr. Philip Kranzusch, the lead author on many of these papers,
 <a href="http://www.uniprot.org/uniprot/?query=cbass%20reviewed:yes">
  characterized CBASS proteins
 </a>
 have been annotated. We'd like to thank Dr. Kranzusch, members of his lab, and Dr. R. Sorek for answering annotation questions. We also thank you for your interest in our knowledgebase and remind you that we are always looking for your input on our entries.
</p>
<h2 id="UniProtKB_news">
 UniProtKB news
</h2>
<h3 id="Change_of_the_cross_references_to_EuPathDB__renamed_to_VEuPathDB">
 Change of the cross-references to EuPathDB: renamed to VEuPathDB
</h3>
<p>
 We have updated our cross-references to reflect the name change of EuPathDB into
 <a href="https://veupathdb.org/">
  VEuPathDB
 </a>
 , following the inclusion of genomes of invertebrate vectors of human pathogens.
</p>
<h3 id="Removal_of_the_cross_references_to_VectorBase">
 Removal of the cross-references to VectorBase
</h3>
<p>
 Direct cross-references to VectorBase have been removed, as VectorBase is now part of the
 <a href="https://veupathdb.org/">
  VEuPathDB
 </a>
 resources.
</p>
<h3 id="Removal_of_the_cross_references_to_UniCarbKB">
 Removal of the cross-references to UniCarbKB
</h3>
<p>
 Cross-references to UniCarbKB have been removed.
</p>
<h3 id="Changes_to_the__a_href___docs_humdisease__controlled_vocabulary_of_human_diseases__a_">
 Changes to the
 <a href="http://www.uniprot.org/docs/humdisease">
  controlled vocabulary of human diseases
 </a>
</h3>
<p>
 New diseases:
</p>
<ul>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-05905">
   Autoinflammation, immune dysregulation, and eosinophilia
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-05918">
   Coenzyme Q10 deficiency, primary, 9
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-05913">
   Combined oxidative phosphorylation deficiency 48
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-05914">
   Combined oxidative phosphorylation deficiency 49
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-05915">
   Combined oxidative phosphorylation deficiency 50
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-05908">
   DEEAH syndrome
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-05904">
   Hemophagocytic lymphohistiocytosis, familial, 6
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-05901">
   Ichthyosis, lamellar, autosomal dominant
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-05917">
   IFAP syndrome 2
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-05898">
   Immunodeficiency 73B with defective neutrophil chemotaxis and lymphopenia
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-05899">
   Immunodeficiency 73C with defective neutrophil chemotaxis and hypogammaglobulinemia
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-05903">
   Intellectual developmental disorder, X-linked, syndromic, Armfield type
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-05906">
   Intellectual developmental disorder with seizures and language delay
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-05919">
   Intellectual developmental disorder with epilepsy, behavioral abnormalities, and coarse facies
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-05907">
   Mitochondrial complex I deficiency, nuclear type 35
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-05928">
   Mitochondrial complex IV deficiency, nuclear type 3
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-05929">
   Mitochondrial complex IV deficiency, nuclear type 4
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-05930">
   Mitochondrial complex IV deficiency, nuclear type 7
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-05931">
   Mitochondrial complex IV deficiency, nuclear type 8
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-05932">
   Mitochondrial complex IV deficiency, nuclear type 10
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-05933">
   Mitochondrial complex IV deficiency, nuclear type 11
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-05934">
   Mitochondrial complex IV deficiency, nuclear type 12
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-05935">
   Mitochondrial complex IV deficiency, nuclear type 14
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-05936">
   Mitochondrial complex IV deficiency, nuclear type 15
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-05937">
   Mitochondrial complex IV deficiency, nuclear type 16
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-05938">
   Mitochondrial complex IV deficiency, nuclear type 17
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-05939">
   Mitochondrial complex IV deficiency, nuclear type 18
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-05940">
   Mitochondrial complex IV deficiency, nuclear type 19
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-05941">
   Mitochondrial complex IV deficiency, nuclear type 20
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-05942">
   Mitochondrial complex IV deficiency, nuclear type 21
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-05900">
   Muscular dystrophy-dystroglycanopathy congenital with impaired intellectual development B15
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-05924">
   Myopathy, epilepsy, and progressive cerebral atrophy
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-05925">
   Myopathy, myofibrillar, 10
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-05909">
   Neurodevelopmental disorder with dysmorphic facies, impaired speech, and hypotonia
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-05922">
   Neurodevelopmental disorder with progressive spasticity and brain white matter abnormalities
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-05911">
   Oocyte maturation defect 8
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-05912">
   Oocyte maturation defect 9
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-05921">
   Optic atrophy 13 with retinal and foveal abnormalities
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-05902">
   Pallister-Hall-like syndrome
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-05916">
   Rajab interstitial lung disease with brain calcifications 2
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-05910">
   Retinitis pigmentosa 90
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-05923">
   Spastic paraplegia 83, autosomal recessive
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-05927">
   Spermatogenic failure 44
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-05926">
   Spinal muscular atrophy, infantile, James type
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/diseases/DI-05920">
   Vissers-Bodmer syndrome
  </a>
 </li>
</ul>
<p>
 Modified diseases:
</p>
<ul>
 <li>
  Familial hemophagocytic lymphohistiocytosis 2 -&gt;
  <a href="http://www.uniprot.org/diseases/DI-01573">
   Hemophagocytic lymphohistiocytosis, familial, 2
  </a>
 </li>
 <li>
  Familial hemophagocytic lymphohistiocytosis 3 -&gt;
  <a href="http://www.uniprot.org/diseases/DI-01574">
   Hemophagocytic lymphohistiocytosis, familial, 3
  </a>
 </li>
 <li>
  Familial hemophagocytic lymphohistiocytosis 4 -&gt;
  <a href="http://www.uniprot.org/diseases/DI-01575">
   Hemophagocytic lymphohistiocytosis, familial, 4
  </a>
 </li>
 <li>
  Familial hemophagocytic lymphohistiocytosis 5 -&gt;
  <a href="http://www.uniprot.org/diseases/DI-02796">
   Hemophagocytic lymphohistiocytosis, familial, 5
  </a>
 </li>
 <li>
  Neutrophil immunodeficiency syndrome -&gt;
  <a href="http://www.uniprot.org/diseases/DI-02051">
   Immunodeficiency 73A with defective neutrophil chemotaxis and leukocytosis
  </a>
 </li>
</ul>
<h3 id="Changes_in__a_href___docs_subcell__subcellular_location_controlled_vocabulary__a_">
 Changes in
 <a href="http://www.uniprot.org/docs/subcell">
  subcellular location controlled vocabulary
 </a>
</h3>
<p>
 New subcellular locations:
</p>
<ul>
 <li>
  <a href="http://www.uniprot.org/locations/SL-0535">
   Autolysosome
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/locations/SL-0536">
   Autolysosome membrane
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/locations/SL-0537">
   Autolysosome lumen
  </a>
 </li>
 <li>
  <a href="http://www.uniprot.org/locations/SL-0538">
   Ascus epiplasm
  </a>
 </li>
</ul>
<h3 id="Changes_to_the_humsavar_txt_file_and_related__a_href___docs_keywlist__keywords__a_">
 Changes to the humsavar.txt file and related
 <a href="http://www.uniprot.org/docs/keywlist">
  keywords
 </a>
</h3>
<p>
 The
 <a href="http://www.uniprot.org/docs/humsavar">
  humsavar.txt
 </a>
 file lists all missense variants annotated in UniProtKB/Swiss-Prot human entries. It provides a variant classification which is intended for research purposes only, not for clinical and diagnostic use. Variants were previously classified into the categories
 <code>
  Disease
 </code>
 ,
 <code>
  Polymorphisms
 </code>
 and
 <code>
  Unclassified
 </code>
 . We have renamed these categories to follow the terminology recommended by the American College of Medical Genetics and Genomics/Association for Molecular Pathology (ACMG/AMP) (Richards et al.
 <a href="https://pubmed.ncbi.nlm.nih.gov/25741868/">
  PubMed:25741868
 </a>
 ):
</p>
<table>
 <tr>
  <th>
   Previous category
  </th>
  <th>
   New category
  </th>
  <th>
   Description
  </th>
 </tr>
 <tr>
  <td>
   Disease
  </td>
  <td>
   LP/P
  </td>
  <td>
   likely pathogenic or pathogenic
  </td>
 </tr>
 <tr>
  <td>
   Polymorphism
  </td>
  <td>
   LB/B
  </td>
  <td>
   likely benign or benign
  </td>
 </tr>
 <tr>
  <td>
   Unclassified
  </td>
  <td>
   US
  </td>
  <td>
   uncertain significance
  </td>
 </tr>
</table>
<p>
 Previous format:
</p>
<pre>Main      Swiss-Prot             AA             Type of
gene name AC         FTId        change         variant       dbSNP          Disease name
_________ __________ ___________ ______________ _____________ ______________ _____________________
...
CC2D2A    Q9P2K1     VAR_075698  p.Trp1182Arg   Disease       rs386833755    Joubert syndrome 9 (JBTS9) [MIM:612285]
CC2D2A    Q9P2K1     VAR_076881  p.Ser117Arg    Unclassified  rs186264635    Joubert syndrome 9 (JBTS9) [MIM:612285]
CC2D2A    Q9P2K1     VAR_076882  p.Lys507Glu    Polymorphism  rs144439937    Joubert syndrome 9 (JBTS9) [MIM:612285]
...
</pre>
<p>
 New format:
</p>
<pre>Main      Swiss-Prot             AA             Variant
gene name AC         FTId        change         category dbSNP          Disease name
_________ __________ ___________ ______________ ________ ______________ _____________________
...
CC2D2A    Q9P2K1     VAR_075698  p.Trp1182Arg   LP/P     rs386833755    Joubert syndrome 9 (JBTS9) [MIM:612285]
CC2D2A    Q9P2K1     VAR_076881  p.Ser117Arg    US       rs186264635    Joubert syndrome 9 (JBTS9) [MIM:612285]
CC2D2A    Q9P2K1     VAR_076882  p.Lys507Glu    LB/B     rs144439937    Joubert syndrome 9 (JBTS9) [MIM:612285]
...
</pre>
<p>
 Also in line with the ACMG/AMP guidelines, we have at the same time deprecated the keyword "Polymorphism" and renamed the keyword "Disease mutation" to
 <a href="http://www.uniprot.org/keywords/KW-0225">
  "Disease variant"
 </a>
 . This was done because the terms 'polymorphism' and 'mutation', which have been widely used, often lead to confusion due to incorrect assumptions of pathogenic and benign effects, respectively.
</p>
<p>
 Entries with variant annotations can be retrieved on the UniProt website with the query
 <a href="http://www.uniprot.org/uniprot/?query=annotation:%28type:variant%29">
  <code>
   annotation:(type:variant)
  </code>
 </a>
 .
</p>
<h2 id="UniProt_FTP_and_website_news">
 UniProt FTP and website news
</h2>
<h3 id="Changes_to_the_FTP_repository_for_reference_proteomes">
 Changes to the FTP repository for reference proteomes
</h3>
<p>
 We currently distribute the UniProt reference proteomes on our
 <a href="https://ftp.uniprot.org/pub/databases/uniprot/current%5Frelease/knowledgebase/reference%5Fproteomes/">
  FTP site
 </a>
 in four taxonomic division folders (Archaea, Bacteria, Eukaryota and Viruses) and provide, for each proteome, its sequences in FASTA format and mappings from UniProt identifiers and gene names to those found in other databases. Starting from this release, we also publish the full protein records for a proteome in the UniProtKB text and XML format, and we have at the same time introduced a subfolder for each proteome that groups all its files in order to reduce the number of files in the taxonomic division folders.
</p>
<p>
 Example:
 <a href="http://www.uniprot.org/proteomes/UP000005640">
  UP000005640
 </a>
</p>
<p>
 Previous FTP folder and files:
</p>
<p>
 https://ftp.uniprot.org/pub/databases/uniprot/current_release/knowledgebase/reference_proteomes/
 <strong>
  Eukaryota/
 </strong>
</p>
<ul>
 <li>
  UP000005640_9606.fasta.gz
 </li>
 <li>
  UP000005640_9606_additional.fasta.gz
 </li>
 <li>
  UP000005640_9606_DNA.fasta.gz
 </li>
 <li>
  UP000005640_9606_DNA.miss.gz
 </li>
 <li>
  UP000005640_9606.idmapping.gz
 </li>
 <li>
  UP000005640_9606.gene2acc.gz
 </li>
</ul>
<p>
 New FTP folder and files:
</p>
<p>
 https://ftp.uniprot.org/pub/databases/uniprot/current_release/knowledgebase/reference_proteomes/
 <strong>
  Eukaryota/UP000005640/
 </strong>
</p>
<ul>
 <li>
  <strong>
   UP000005640_9606.dat.gz
  </strong>
  —
  <em>
   UniProtKB text format
  </em>
 </li>
 <li>
  <strong>
   UP000005640_9606.xml.gz
  </strong>
  —
  <em>
   UniProtKB XML format
  </em>
 </li>
 <li>
  UP000005640_9606.fasta.gz
 </li>
 <li>
  UP000005640_9606_additional.fasta.gz
 </li>
 <li>
  UP000005640_9606_DNA.fasta.gz
 </li>
 <li>
  UP000005640_9606_DNA.miss.gz
 </li>
 <li>
  UP000005640_9606.idmapping.gz
 </li>
 <li>
  UP000005640_9606.gene2acc.gz
 </li>
</ul>
<h3 id="Browser_support_change__End_of_support_for_Microsoft_Internet_Explorer">
 Browser support change: End of support for Microsoft Internet Explorer
</h3>
<p>
 We have phased out support for Microsoft's Internet Explorer (IE 11), since our user base for this web browser is continuously decreasing and has significantly fallen below 1.5%. Please switch to a more recent and
 <a href="http://www.uniprot.org/help/browser%5Fsupport">
  supported browser
 </a>
 in order to enjoy the full functionalities of the UniProt website.
</p>
<p>
 Even if you are not using IE 11, it may be a good idea to look at the list of the browsers we support and consider switching to the most recent version of
 <a href="https://whatismybrowser.org">
  your browser
 </a>
 for the best user experience and security.
</p>